### Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2

Li et al. *bioRxiv*. [@doi:10.1101/2020.02.25.965434]

#### Keywords
- SARS-CoV-2
- immune-informatics
- vaccine design
- T cell epitope
- B cell epitope

#### Main Findings

This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.

#### Limitations

While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.

#### Significance

This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*
