<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title> • covidpreprints</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="">
<meta property="og:description" content="covidpreprints">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">covidpreprints</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/timeline.html">Timeline</a>
</li>
<li>
  <a href="../articles/further_info.html">Further information</a>
</li>
<li>
  <a href="../articles/resources.html">Resources</a>
</li>
<li>
  <a href="../articles/about.html">About</a>
</li>
<li>
  <a href="../articles/preprint_plot.html">Preprints posted to bioRxiv and medRxiv</a>
</li>
<li>
  <a href="../articles/mount_sinai.html">Mount Sinai peer-reviews of COVID-19 preprints</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/coatesj/shiny-server/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="mount_sinai_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip></h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/coatesj/shiny-server/blob/master/vignettes/mount_sinai.Rmd"><code>vignettes/mount_sinai.Rmd</code></a></small>
      <div class="hidden name"><code>mount_sinai.Rmd</code></div>

    </div>

    
    
<p>The following is a collection of the Mount Sinai Immunology researchers (<span class="citation">@MountSinaiNYC</span>). We thank the team for their important efforts in reviewing critical literature during such a turbulent period and for allowing us to link to these reviews. See <a href="https://observablehq.com/@ismms-himc/covid-19-sars-cov-2-preprints-from-medrxiv-and-biorxiv" class="uri">https://observablehq.com/@ismms-himc/covid-19-sars-cov-2-preprints-from-medrxiv-and-biorxiv</a> for more. Please use Crt+F to search.</p>
<div id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody" class="section level3">
<h3 class="hasAnchor">
<a href="#potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody" class="anchor"></a>Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h3>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation">[@doi:10.1101/2020.01.28.923011]</span></p>
<div id="keywords" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords" class="anchor"></a>Keywords</h4>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
</div>
<div id="summary" class="section level4">
<h4 class="hasAnchor">
<a href="#summary" class="anchor"></a>Summary</h4>
<p>Considering the relatively high identity of the receptor binding domain (RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this study aims to assess the cross-reactivity of several anti-SARS-CoV monoclonal antibodies with 2019-nCoV. The results showed that the SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
</div>
<div id="main-findings" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings" class="anchor"></a>Main Findings</h4>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in silico</em> and compared with the SARS-CoV RBD structure. The models predicted very similar RBD-ACE2 interactions for both viruses. The binding capacity of representative SARS-CoV-RBD specific monoclonal antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was then investigated by ELISA and their binding kinetics studied using biolayer interferometry. The analysis showed that only CR3022 was able to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did not interfere with ACE2 binding. Antibodies m396 and CR3014, which target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein.</p>
</div>
<div id="limitations" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations" class="anchor"></a>Limitations</h4>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus residues, which drastically impact the cross-reactivity of antibodies described for other B beta-coronaviruses, including SARS-CoV. This study claims that CR3022 antibody could be a potential candidate for therapy. However, none of the antibodies assayed in this work showed cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for the replication of this virus. Furthermore, neutralization assays with 2019-nCoV virus or pseudovirus were not performed. Although the use of neutralizing antibodies is an interesting approach, these results suggest that it is critical the development of novel monoclonal antibodies able to specifically bind 2019-nCoV spike protein.</p>
</div>
<div id="credit" class="section level4">
<h4 class="hasAnchor">
<a href="#credit" class="anchor"></a>Credit</h4>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov" class="section level3">
<h3 class="hasAnchor">
<a href="#integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov" class="anchor"></a>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h3>
<p>He et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.20020206]</span></p>
<div id="keywords-1" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-1" class="anchor"></a>Keywords</h4>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
</div>
<div id="main-findings-1" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-1" class="anchor"></a>Main Findings</h4>
<p>The authors used bioinformatics tools to identify features of ACE2 expression in the lungs of different patent groups: healthy, smokers, patients with chronic airway disease (i.e., COPD) or asthma. They used gene expression data publicly available from GEO that included lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small airway epithelium of long-term smokers vs non-smokers (p&lt;0.001). Consistently, there was a trend of higher ACE2 expression in the bronchial airway epithelial cells 24h post-acute smoking exposure (p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h post SARS-Cov infection (p=0.026; n=3 at each time point) was also detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high and low ACE2 expression groups. “High” ACE2 expression was associated with the following GO pathways: innate and adaptive immune responses, B cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8 cytokines. The authors speculate that a high basal ACE2 expression will increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6 cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell activation and T-cell cytokine production. It is unclear whether those changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates in epithelial cells infected with SARS-Cov infection. They state that in healthy donors ACE2 expression did not correlate with the immune cell infiltration. However, in SARS-Cov samples, at 48h they found that ACE2 correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs. Again, while authors claim significance, the corresponding correlation coefficients and p-values are not presented in the text or figures. In addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1 and PRKCD were found to be central (Hub proteins) in the regulating network of cytokine secretion after coronavirus infection. Authors conclude this indicates that these molecules were critically important in ACE2-induced inflammatory response. Additionally, authors speculate that the increased expression of ACE2 affected RPS3 and SRC, which were the two hub genes involved in viral replication and inflammatory response.</p>
</div>
<div id="limitations-1" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-1" class="anchor"></a>Limitations</h4>
<p>The methods section is very limited and does not describe any of the statistical analyses; and description of the construction of the regulatory protein networks is also limited. For the findings in Figures 2 authors claim significance, which is not supported by p-values or coefficients. For the sample selection, would be useful if sample sizes and some of the patients’ demographics (e.g. age) were described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it is not clear what was the cut off for ‘high’ expression and how it was determined. Additionally, further laboratory studies are warranted to confirm that high ACE2 gene expression would have high correlation with the amount of ACE2 protein on cell surface. For the GO pathway analysis significance was set at p&lt;0.05, but not adjusted for multiple comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only included data on SARS-Cov and any conclusions about SARS-CoV2 are limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two might not be cited correctly: GSE37758 (“A spergillus niger: Control (fructose) vs. steam-exploded sugarcane induction (SEB)”" was used in this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment of Organ Donors to Prevent Postischemic Renal Allograft Failure: A Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
</div>
<div id="credit-1" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-1" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus" class="anchor"></a>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h3>
<p>Liang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.20020289]</span></p>
<div id="keywords-2" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-2" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
</div>
<div id="main-findings-2" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-2" class="anchor"></a>Main Findings</h4>
<p>This study examined the incidence of diarrhea in patients infected with SARS-CoV-2 across three recently published cohorts and found that there are statistically significant differences by Fisher’s exact test. They report that this could be due to subjective diagnosis criterion for diarrhea or from patients first seeking medical care from gastroenterologist. In order to minimize nosocomial infections arising from unsuspected patients with diarrhea and gain comprehensive understanding of transmission routes for this viral pathogen, they compared the transcriptional levels of ACE2 of various human tissues from NCBI public database as well as in small intestine tissue from CD57BL/6 mice using single cell sequencing. They show that ACE2 expression is not only increased in the human small intestine, but demonstrate a particular increase in mice enterocytes positioned on the surface of the intestinal lining exposed to viral pathogens. Given that ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate diarrhea, their data suggests the small intestine as a potential transmission route and diarrhea as a potentially underestimated symptom in COVID19 patients that must be carefully monitored. Interestingly, however, they show that ACE2 expression level is not elevated in human lung tissue.</p>
</div>
<div id="limitations-2" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-2" class="anchor"></a>Limitations</h4>
<p>Although this study demonstrates a statistical difference in the incidence of diarrhea across three separate COVID19 patient cohorts, their conclusions are limited by a small sample size. Specifically, the p-value computed by Fisher’s exact test is based on a single patient cohort of only six cases of which 33% are reported to have diarrhea, while the remaining two larger cohorts with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively. Despite showing significance, they would need to acquire larger sample sizes and cohorts to minimize random variability and draw meaningful conclusions. Furthermore, they do not address why ACE2 expression level is not elevated in human lung tissue despite it being a major established route of transmission for SARS-CoV-2. It could be helpful to validate this result by looking at ACE2 expression in mouse lung tissue. Finally, although this study is descriptive and shows elevated ACE2 expression in small intestinal epithelial cells, it does not establish a mechanistic link to SARS-CoV-2 infection of the host. Overall, their claim that infected patients exhibiting diarrhea pose an increased risk to hospital staff needs to be further substantiated.</p>
</div>
<div id="significance" class="section level4">
<h4 class="hasAnchor">
<a href="#significance" class="anchor"></a>Significance</h4>
<p>This study provides a possible transmission route and a potentially underappreciated clinical symptom for SARS-CoV-2 for better clinical management and control of COVID19.</p>
</div>
<div id="credit-2" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-2" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus-1" class="section level3">
<h3 class="hasAnchor">
<a href="#diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus-1" class="anchor"></a>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h3>
<p>Liang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.20020289v2]</span></p>
<div id="keywords-3" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-3" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
</div>
<div id="main-findings-3" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-3" class="anchor"></a>Main Findings</h4>
<p><strong><em>:</em></strong> This study examined the incidence of diarrhea in patients infected with SARS-CoV-2 across three recently published cohorts and found that there are statistically significant differences by Fisher’s exact test. They report that this could be due to subjective diagnosis criterion for diarrhea or from patients first seeking medical care from gastroenterologist. In order to minimize nosocomial infections arising from unsuspected patients with diarrhea and gain comprehensive understanding of transmission routes for this viral pathogen, they compared the transcriptional levels of ACE2 of various human tissues from NCBI public database as well as in small intestine tissue from CD57BL/6 mice using single cell sequencing. They show that ACE2 expression is not only increased in the human small intestine, but demonstrate a particular increase in mice enterocytes positioned on the surface of the intestinal lining exposed to viral pathogens. Given that ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate diarrhea, their data suggests the small intestine as a potential transmission route and diarrhea as a potentially underestimated symptom in COVID19 patients that must be carefully monitored. Interestingly, however, they show that ACE2 expression level is not elevated in human lung tissue.</p>
</div>
<div id="limitations-3" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-3" class="anchor"></a>Limitations</h4>
<p>Although this study demonstrates a statistical difference in the incidence of diarrhea across three separate COVID19 patient cohorts, their conclusions are limited by a small sample size. Specifically, the p-value computed by Fisher’s exact test is based on a single patient cohort of only six cases of which 33% are reported to have diarrhea, while the remaining two larger cohorts with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively. Despite showing significance, they would need to acquire larger sample sizes and cohorts to minimize random variability and draw meaningful conclusions. Furthermore, they do not address why ACE2 expression level is not elevated in human lung tissue despite it being a major established route of transmission for SARS-CoV-2. It could be helpful to validate this result by looking at ACE2 expression in mouse lung tissue. Finally, although this study is descriptive and shows elevated ACE2 expression in small intestinal epithelial cells, it does not establish a mechanistic link to SARS-CoV-2 infection of the host. Overall, their claim that infected patients exhibiting diarrhea pose an increased risk to hospital staff needs to be further substantiated.</p>
</div>
<div id="significance-1" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-1" class="anchor"></a>Significance</h4>
<p>This study provides a possible transmission route and a potentially underappreciated clinical symptom for SARS-CoV-2 for better clinical management and control of COVID19.</p>
</div>
<div id="credit-3" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-3" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection" class="anchor"></a>Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h3>
<p>Chai et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.931766]</span></p>
<div id="keywords-4" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-4" class="anchor"></a>Keywords</h4>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
</div>
<div id="summary-1" class="section level4">
<h4 class="hasAnchor">
<a href="#summary-1" class="anchor"></a>Summary</h4>
<p>Using both publicly available scRNA-seq dataset of liver samples from colorectal patients and scRNA-sequencing of four liver samples from healthy volunteers, the authors show that ACE2 is significantly enriched in the majority of cholangiocytes (59.7 %) but not in hepatocytes (2.6%).</p>
</div>
<div id="main-findings-4" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-4" class="anchor"></a>Main Findings</h4>
<p>Using bioinformatics approaches of RNASeq analysis, this study reveals that ACE2 dominates in cholangiocytes and is present at very low levels in hepatocytes.</p>
</div>
<div id="limitations-4" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-4" class="anchor"></a>Limitations</h4>
<p>The study does not provide mechanistic insights into how SARS-CoV-2 can infect and replicate in cholangiocytes and the types of intrinsic anti-viral responses induced by cholangiocytes when infected. In addition, because the study relies on the assumption that SARS-CoV-2 infects cells only through ACE2, it cannot discount the possibility that the virus can infect hepatocytes through mechanisms other than ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were performed on healthy liver samples, one cannot draw any definitive conclusions about gene expression states (including ACE2 expression in liver cell types) in system-wide inflammatory contexts.</p>
</div>
<div id="significance-2" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-2" class="anchor"></a>Significance</h4>
<p>This article with other studies on liver damage in COVID patients suggests that liver damage observed in COVID patients is more due to inflammatory cytokines than direct infection of the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which was demonstrated by human liver ductal organoid study (<span class="citation">[@doi:10.1101/2020.03.16.990317]</span>), published clinical data show no significant increase in bile duct injury related indexes (i.e. alkaline phosphatase, gamma-glutamyl transpeptidase and total bilirubin). In sum, it underscores the importance of future studies characterizing cellular responses of extra-pulmonary organs in the context of COVID or at least in viral lung infections..</p>
</div>
<div id="credit-4" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-4" class="anchor"></a>Credit</h4>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism" class="anchor"></a>ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h3>
<p>Wang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.05.20020545]</span></p>
<div id="keywords-5" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-5" class="anchor"></a>Keywords</h4>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
</div>
<div id="main-findings-5" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-5" class="anchor"></a>Main Findings</h4>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
</div>
<div id="limitations-5" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-5" class="anchor"></a>Limitations</h4>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
</div>
<div id="significance-3" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-3" class="anchor"></a>Significance</h4>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
</div>
<div id="credit-5" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-5" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice" class="section level3">
<h3 class="hasAnchor">
<a href="#the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice" class="anchor"></a>The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h3>
<p>Bao et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.07.939389]</span></p>
<div id="keywords-6" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-6" class="anchor"></a>Keywords</h4>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
</div>
<div id="main-findings-6" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-6" class="anchor"></a>Main Findings</h4>
<p>Using a transgenic human Angiotensin-converting enzyme 2 (hACE2) mouse that has previously been shown susceptible to infection by SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain coronavirus. The model includes interstitial hyperplasia in lung tissue, moderate inflammation in bronchioles and blood vessels, and histology consistent with viral pneumonia at 3 days post infection. Wildtype did not experience these symptoms. In addition, viral antigen and hACE2 receptor were found to co-localize the lung by immunofluorescence 3-10 days post infection only in the hACE2 infected mice.</p>
</div>
<div id="limitations-6" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-6" class="anchor"></a>Limitations</h4>
<p>The characterization of the infection remains incomplete, as well as lacking characterization of the immune response other than the presence of a single antiviral antibody. Though they claim to fulfill Koch’s postulates, they only isolate the virus and re-infect Vero cells, rather than naive mice.</p>
</div>
<div id="significance-4" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-4" class="anchor"></a>Significance</h4>
<p>This paper establishes a murine model for 2019-nCoV infection with symptoms consistent with viral pneumonia. Though not fully characterized, this model allows in vivo analysis of viral entry and pathology that is important for the development of vaccines and antiviral therapeutics.</p>
</div>
<div id="credit-6" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-6" class="anchor"></a>Credit</h4>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="caution-on-kidney-dysfunctions-of-2019-ncov-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#caution-on-kidney-dysfunctions-of-2019-ncov-patients" class="anchor"></a>Caution on Kidney Dysfunctions of 2019-nCoV Patients</h3>
<p>Li et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span></p>
<div id="keywords-7" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-7" class="anchor"></a>Keywords</h4>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine, proteinuria, albuminuria, CT</p>
</div>
<div id="main-findings-7" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-7" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators of the kidney–including urine protein, blood urea nitrogen (BUN), plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the first day of admission, and 63% developed proteinuria during their stay in hospital; and 19% of patients had high plasma creatinine, especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal kidney damage; indicate that inflammation and edema of the renal parenchyma very common.</p></li>
</ul>
</div>
<div id="limitations-7" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-7" class="anchor"></a>Limitations</h4>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in blood/plasma/urine. Will be worth correlating disease progression with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only rationale for viral-damage in kidney and resultant pathologic immune response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
</div>
<div id="significance-5" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-5" class="anchor"></a>Significance</h4>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the idea that renal impairment/injury is a key risk factor in 2019-nCoV patients similar to what has been reported for SARS-CoV<span class="citation">[@doi:10.1111/j.1523-1755.2005.67130.x]</span>; this may be one of the major causes of virally-induced damage and contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder epithelia <span class="citation">[@doi:10.1101/2020.02.08.939892]</span> support these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying potential interventions including continuous renal replacement therapies (CRRT) for protecting kidney functions as early as possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
</div>
<div id="credit-7" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-7" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus" class="anchor"></a>Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h3>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.08.939553]</span></p>
<div id="keywords-8" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-8" class="anchor"></a>Keywords</h4>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
</div>
<div id="main-findings-8" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-8" class="anchor"></a>Main Findings</h4>
<p>This study harnesses bioinformatic profiling to predict the potential of COV2 viral proteins to be presented on MHC I and II and to form linear B-cell epitopes. These estimates suggest a T-cell antigenic profile distinct from SARS-CoV or MERS-CoV, identify focused regions of the virus with a high density of predicted epitopes, and provide preliminary evidence for adaptive immune pressure in the genetic evolution of the virus.</p>
</div>
<div id="limitations-8" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-8" class="anchor"></a>Limitations</h4>
<p>While the study performs a comprehensive analysis of potential epitopes within the virus genome, the analysis relies solely on bioinformatic prediction to examine MHC binding affinity and B-cell epitope potential and does not capture the immunogenicity or recognition of these epitopes. Future experimental validation in data from patients infected with SARS-CoV-2 will be important to validate and refine these findings. Thus some of the potential conclusions stated, including viral evolution toward lower immunogenicity or a dominant role for CD4+ T-cells rather than CD8+ T-cells in viral clearance, require further valiadtion.</p>
</div>
<div id="significance-6" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-6" class="anchor"></a>Significance</h4>
<p>These findings may help direct peptide vaccine design toward relevant epitopes and provide intriguing evidence of viral evolution in response to immune pressure.</p>
</div>
<div id="credit-8" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-8" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection" class="anchor"></a>Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h3>
<p>Lin et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.08.939892]</span></p>
<div id="keywords-9" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-9" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
</div>
<div id="main-findings-9" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-9" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV patients, authors used published kidney and bladder cell atlas data (GSE131685, GSE108097; 3 healthy donors each) as well as an unpublished kidney single-cell RNA-Seq data (in-house from 2 transplant donors) to evaluate ACE2 gene expressions in all cell types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of proximal tubule cells of kidney, with 5%-15% of both straight and convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial cells, noting expression from around 1.5% of cells in the outer layer umbrella cells of the bladder epithelium and decreasing in the basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not express ACE2.</p></li>
</ul>
</div>
<div id="limitations-9" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-9" class="anchor"></a>Limitations</h4>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other genes) from a small healthy-donor dataset, and will benefit from supporting data in (expired) patient samples to show functional viral damage. ACE2 transcript does not necessarily translate to viral permissiveness in kidney/bladder epithelia or cytokine release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to analyze kidney/bladder epithelial ACE2 expression under inflammatory conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell identification would be useful in these datasets.</p></li>
</ul>
</div>
<div id="significance-7" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-7" class="anchor"></a>Significance</h4>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal tubules and in bladder umbrella cells <span class="citation">[@doi:10.1002/path.1570]</span>, such cells in direct contact with viral particles are likely to be highly sensitive to viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of patients and associate with higher mortality <span class="citation">[@doi:10.1111/j.1523-1755.2005.67130.x; @doi:10.7326/m13-2486]</span>, thus worth understanding kidney damage and resultant immune response in SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and resultant inflammatory responses in these tissue in addition to reported infectivity in the lung and digestive system, which is supported by clinical data showing acute and early kidney complications in 2019-nCoV patients <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
</div>
<div id="credit-9" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-9" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage" class="section level3">
<h3 class="hasAnchor">
<a href="#neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage" class="anchor"></a>Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h3>
<p>Liu et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.10.20021584]</span></p>
<div id="keywords-10" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-10" class="anchor"></a>Keywords</h4>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
</div>
<div id="main-findings-10" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-10" class="anchor"></a>Main Findings</h4>
<p>This study aimed to find prognostic biomarkers of COVID-19 pneumonia severity. Sixty-one (61) patients with COVID-19 treated in January at a hospital in Beijing, China were included. On average, patients were seen within 5 days from illness onset. Samples were collected on admission; and then patients were monitored for the development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17) according to symptoms on admission and compared for different clinical/laboratory features. “Moderate/severe” patients were significantly older (median of 56 years old, compared to 41 years old). Whereas comorbidies rates were largely similar between the groups, except for hypertension, which was more frequent in the severe group (p= 0.056). ‘Severe’ patients had higher counts of neutrophils, and serum glucose levels; but lower lymphocyte counts, sodium and serum chlorine levels. The ratio of neutrophils to lymphocytes (NLR) was also higher for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial infections (and antibiotic treatment) and received more intensive respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the best predictors of the severe disease. Predictive cutoffs for a severe illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
</div>
<div id="limitations-10" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-10" class="anchor"></a>Limitations</h4>
<p>Identification of early biomarkers is important for making clinical decisions, but large sample size and validation cohorts are necessary to confirm findings. It is worth noting that patients classified as “mild” showed pneumonia by imaging and fever, and in accordance with current classifications this would be consistent with “moderate” cases. Hence it would be more appropriate to refer to the groups as “moderate” vs “severe/critical”. Furthermore, there are several limitations that could impact the interpretation of the results: e.g. classification of patients was based on symptoms presented on admission and not based on disease progression, small sample size, especially the number of ‘severe’ cases (with no deaths among these patients). Given the small sample size, the proposed NLR and age cut offs might not hold for a slightly different set of patients. For example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR were 3.2 and 5.5, respectively <span class="citation">[@doi:10.1093/cid/ciaa248]</span>.</p>
</div>
<div id="credit-10" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-10" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp" class="section level3">
<h3 class="hasAnchor">
<a href="#characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp" class="anchor"></a>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h3>
<p>Wan et. al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.10.20021832]</span></p>
<div id="keywords-11" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-11" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
</div>
<div id="main-findings-11" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-11" class="anchor"></a>Main Findings</h4>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients with mild disease and 21 with severe disease. CD8+T cells and CD4+T cells were significantly reduced in both cohort. particularly in severe patients. The cytokines IL6 and IL10 were significantly elevated in severe patients as compared to mild. No significant differences were observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and cytokine levels of IL6 and IL10 can be used to predict progression of disease from Mild to severe Cov-2 infection.</p>
</div>
<div id="limitations-11" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-11" class="anchor"></a>Limitations</h4>
<p>The study demonstrates in a limited cohort similar associations to several other reported studies. The authors didn’t compare the changes in lymphocyte and cytokine with healthy individual (Covid-19 Negative) rather used an internal standard value. The recently preprint in LANCET shows The degree of lymphopenia and a pro-inflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity <span class="citation">[@doi:10.2139/ssrn.3539682]</span>.</p>
</div>
<div id="significance-8" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-8" class="anchor"></a>Significance</h4>
<p>This translational data identifies key cytokines and lymphopenia associated with disease severity although mechanism and key cellular players are still unknown. Higher level IL-6 production in severe patient suggests potential role of Tocilizumab (anti-IL6R) biologic although clinical trial will be necessary.</p>
</div>
<div id="credit-11" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-11" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp-1" class="section level3">
<h3 class="hasAnchor">
<a href="#characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp-1" class="anchor"></a>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h3>
<p>Wan et. al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.10.20021832]</span></p>
<div id="keywords-12" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-12" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
</div>
<div id="main-findings-12" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-12" class="anchor"></a>Main Findings</h4>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients with mild disease and 21 with severe disease. CD8+T cells and CD4+T cells were significantly reduced in both cohort. particularly in severe patients. The cytokines IL6 and IL10 were significantly elevated in severe patients as compared to mild. No significant differences were observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and cytokine levels of IL6 and IL10 can be used to predict progression of disease from Mild to severe Cov-2 infection.</p>
</div>
<div id="limitations-12" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-12" class="anchor"></a>Limitations</h4>
<p>The study demonstrates in a limited cohort similar associations to several other reported studies. The authors didn’t compare the changes in lymphocyte and cytokine with healthy individual (Covid-19 Negative) rather used an internal standard value. The recently preprint in LANCET shows The degree of lymphopenia and a pro-inflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity <span class="citation">[@doi:10.2139/ssrn.3539682]</span>.</p>
</div>
<div id="significance-9" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-9" class="anchor"></a>Significance</h4>
<p>This translational data identifies key cytokines and lymphopenia associated with disease severity although mechanism and key cellular players are still unknown. Higher level IL-6 production in severe patient suggests potential role of Tocilizumab (anti-IL6R) biologic although clinical trial will be necessary.</p>
</div>
<div id="credit-12" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-12" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china" class="anchor"></a>Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h3>
<p>Li et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.11.20022053]</span></p>
<div id="keywords-13" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-13" class="anchor"></a>Keywords</h4>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
</div>
<div id="major-findings" class="section level4">
<h4 class="hasAnchor">
<a href="#major-findings" class="anchor"></a>Major Findings</h4>
<p>These authors looked at 17 hospitalized patients with COVID-19 confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between January 22 and February 10 and the final data were collected on February 11. Of the 17 patients, 12 remained hospitalized while 5 were discharged after meeting national standards. The authors observed no differences based on the sex of the patients but found that the discharged patients were younger in age (p = 0.026) and had higher lymphocyte counts (p = 0.005) and monocyte counts (p = 0.019) upon admission.</p>
</div>
<div id="limitations-13" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-13" class="anchor"></a>Limitations</h4>
<p>This study is limited in the sample size of the study and the last data collection point was only one day after some of the patients were admitted.</p>
</div>
<div id="significance-10" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-10" class="anchor"></a>Significance</h4>
<p>These findings have been somewhat supported by subsequent studies that show that older age and an immunocompromised state are more likely to result in a more severe clinical course with COVID-19. However, other studies have been published that report on larger numbers of cases.</p>
</div>
<div id="credit-13" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-13" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection" class="anchor"></a>ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h3>
<p><span class="citation">[@doi:10.1101/2020.02.12.20022418]</span></p>
<div id="keywords-14" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-14" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-13" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-13" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585, Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to show 3~10% of 2019-nCoV patients present with abnormal renal function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells, Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
</div>
<div id="limitations-14" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-14" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy cohorts; does not necessarily translate to actual viral tropism and permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal dataset if SARS-CoV-2 infection changes sperm quality or testicular inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
</div>
<div id="significance-11" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-11" class="anchor"></a>Significance</h4>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies <span class="citation">[@doi:10.1101/2020.02.08.939892]</span> and clinical reports of abnormal renal function or even kidney damage in patients infected with 2019-nCoV <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the virus to testicular tissues and indicates potential risks for male fertility. Viral orchitis reported for SARS-CoV previously <span class="math display">\[1\]</span>, but no clear evidence so far of infertility in SARS, MERS or CoVID-19 patients.</p></li>
</ul>
</div>
<div id="credit-14" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-14" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus" class="anchor"></a>Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h3>
<p><span class="citation">[@doi:10.1101/2020.02.12.945576]</span></p>
<div id="keywords-15" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-15" class="anchor"></a>Keywords</h4>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
</div>
<div id="main-findings-14" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-14" class="anchor"></a>Main Findings</h4>
<p>The authors of this study sought to characterize the immune mechanism causing severe pulmonary disease and mortality in 2019-nCoV (COVID-19) patients. Peripheral blood was collected from hospitalized ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from healthy controls (n=10) in The First Affiliated Hospital of University of Science and Technology China (Hefei, Anhui). Immune analysis was conducted by flow cytometry. 2019-nCoV patients had decreased lymphocyte, monocyte, and CD4 T cell counts compared to healthy controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4 T cells of 2019-nCoV patients expressed higher levels of activation markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3) than those of healthy controls. CD4 cells of ICU patients expressed significantly higher levels of OX40, PD-1, and Tim3 than those of non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T cells co-expressing GM-CSF and IL-6 compared to healthy controls, while ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T cells than non-ICU patients. The CD4 T cells of nCoV patients and healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of activation markers CD69, CD38, and CD44, as well as exhaustion markers PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients expressed higher levels of GM-CSF than those of non-ICU patients and healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any group. There were no differences in numbers of NK cells or B cells in 2019-nCoV patients and healthy controls, nor was there any GM-CSF or IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory monocytes were significantly increased in nCoV patients compared to healthy controls; in particular, patients in the ICU had greater percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high levels of IL-6. These may enter pulmonary circulation and damage lung tissue while initiating the cytokine storm that causes mortality in severe cases. This is consistent with the cytokine storm seen in similar coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators seen in patients with SARS-CoV-1 and MERS-CoV.</p>
</div>
<div id="limitations-15" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-15" class="anchor"></a>Limitations</h4>
<p>Though the results of this study open questions for further investigation, this is an early study on a small cohort of patients, and as such there are a number of limitations. The study included only 12 ICU patients and 21 non-ICU patients, and ideally would be repeated with a much larger patient cohort. Though the authors make claims about differences in lymphocyte and monocyte counts between patients and healthy controls, they did not report baseline laboratory findings for the control group. Additionally, severity of disease was classified based on whether or not patients were in the ICU. It would be interesting to contextualize the authors’ immunological findings with more specific metrics of disease severity or time course. Noting mortality, time from disease onset, pre-existing conditions, or severity of lung pathology in post-mortem tissue samples would paint a fuller picture of how to assess risk level and the relationship between severity of disease and immunopathology. Another limitation is the selection of cytokines and immune markers for analysis, as the selection criteria were based on the cell subsets and cytokine storm typically seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and immune profiling may reveal novel therapeutic targets that were not included in this study.</p>
</div>
<div id="significance-12" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-12" class="anchor"></a>Significance</h4>
<p>This study identifies potential therapeutic targets that could prevent acute respiratory disease syndrome (ARDS) and mortality in patients most severely affected by COVID-19. The authors propose testing monoclonal antibodies against IL6-R or GM-CSF to block recruitment of inflammatory monocytes and the subsequent cytokine storm in these patients.</p>
</div>
<div id="credit-15" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-15" class="anchor"></a>Credit</h4>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis" class="anchor"></a>Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h3>
<p>Qian et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.14.20021535]</span></p>
<div id="keywords-16" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-16" class="anchor"></a>Keywords</h4>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
</div>
<div id="main-findings-15" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-15" class="anchor"></a>Main Findings</h4>
<p>The authors performed a meta analysis of literature on clinical, laboratory and radiologic characteristics of patients presenting with pneumonia related to SARSCoV2 infection, published up to Feb 6 2020. They found that symptoms that were mostly consistent among studies were sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise and muscle pain were highly variable across studies. Leukopenia (mostly lymphocytopenia) and increased white blood cells were highly variable across studies. They identified three most common patterns seen on CT scan, but there was high variability across studies. Consistently across the studies examined, the authors found that about 75% of patients need supplemental oxygen therapy, about 23% mechanical ventilation and about 5% extracorporeal membrane oxygenation (ECMO). The authors calculated a staggering pooled mortality incidence of 78% for these patients.</p>
</div>
<div id="limitations-16" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-16" class="anchor"></a>Limitations</h4>
<p>The authors mention that the total number of studies included in this meta analysis is nine, however they also mentioned that only three studies reported individual patient data. It is overall unclear how many patients in total were included in their analysis. This is mostly relevant as they reported an incredibly high mortality (78%) and mention an absolute number of deaths of 26 cases overall. It is not clear from their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area, which somewhat limits the overall generalizability and applicability of these results.</p>
</div>
<div id="significance-13" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-13" class="anchor"></a>Significance</h4>
<p>This meta analysis offers some important data for clinicians to refer to when dealing with patients with COVID-19 and specifically with pneumonia. It is very helpful to set expectations about the course of the disease.</p>
</div>
<div id="credit-16" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-16" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients" class="anchor"></a>Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h3>
<p>Liu et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.20023671]</span></p>
<div id="keywords-17" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-17" class="anchor"></a>Keywords</h4>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
</div>
<div id="main-findings-16" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-16" class="anchor"></a>Main Findings</h4>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan including 27 mild cases and 13 severe cases of COVID-19. They performed a 16-day kinetic analysis of peripheral blood from time of disease onset. Patients in the severe group were older (medium age of 59.7, compared to 48.7 in mild group) and more likely to have hypertension as a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients and 84.6% of the severe patients. Lymphopenia was due to low T cell count, specially CD8 T cells. Severe patients showed higher neutrophil counts and an increase of cytokines in the serum (IL2, IL6, IL10 and IFNγ). The authors measured several other clinical laboratory parameters were also higher in severe cases compared to mild, but concluded that neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to identify the severe cases compared to other receiver operating characteristic (ROC).</p>
</div>
<div id="limitations-17" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-17" class="anchor"></a>Limitations</h4>
<p>This was a small cohort (N=40), and two of the patients initially included in the severe group (N=13) passed away and were excluded from the analysis due to lack of longitudinal data. However, it would be most important to be able to identify patients with severe disease with higher odds of dying. It seems that the different time points analyzed relate to hospital admission, which the authors describe as disease onset. The time between first symptoms and first data points is not described. It would have been important to analyze how the different measured parameters change according to health condition, and not just time (but that would require a larger cohort). The predictive value of N8R compared to the more commonly used NLR needs to be assessed in other independent and larger cohorts. Lastly, it is important to note that pneumonia was detected in patients included in the “mild” group, but according to the Chinese Clinical Guidance for COVID-19 Pneumonia Diagnosis and Treatment (7th edition) this group should be considered “moderate”.</p>
</div>
<div id="significance-14" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-14" class="anchor"></a>Significance</h4>
<p>Lymphopenia and cytokine storm have been described to be detrimental in many other infections including SARS-CoV1 and MERS-CoV. However, it was necessary to confirm that this dramatic immune response was also observed in the SARS-CoV2 infected patients. These results and further validation of the N8R ratio as a predictor of disease severity will contribute for the management of COVID19 patients and potential development of therapies.</p>
</div>
<div id="credit-17" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-17" class="anchor"></a>Credit</h4>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019" class="anchor"></a>Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h3>
<p>Chen et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.20023903]</span></p>
<div id="keywords-18" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-18" class="anchor"></a>Keywords</h4>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
</div>
<div id="main-findings-17" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-17" class="anchor"></a>Main Findings</h4>
<p>This study retrospectively evaluated clinical, laboratory, hematological, biochemical and immunologic data from 21 subjects admitted to the hospital in Wuhan, China (late December/January) with confirmed SARS-CoV-2 infection. The aim of the study was to compare ‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old) COVID-19 cases. Disease severity was defined by patients’ blood oxygen level and respiratory output. They were classified as ‘severe’ if SpO2 93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had higher CRP and ferritin, alanine and aspartate aminotransferases, and lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells and total B cells; and lower total lymphocytes, CD4 and CD8 T cells (except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No significant differences were observed for IL-8, counts of NK cells, CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
</div>
<div id="limitations-18" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-18" class="anchor"></a>Limitations</h4>
<p>Several potential limitations should be noted: 1) Blood samples were collected 2 days post hospital admission and no data on viral loads were available; 2) Most patients were administered medications (e.g. corticosteroids), which could have affected lymphocyte counts. Medications are briefly mentioned in the text of the manuscript; authors should include medications as part of Table 1. 3) ‘Severe’ cases were significantly older and 4/11 ‘severe’ patients died within 20 days. Authors should consider a sensitivity analysis of biomarkers with the adjustment for patients’ age.</p>
</div>
<div id="significance-15" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-15" class="anchor"></a>Significance</h4>
<p>Although the sample size was small, this paper presented a broad range of clinical, biochemical, and immunologic data on patients with COVID-19. One of the main findings is that SARS-CoV-2 may affect T lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy production. Potentially, diminished T lymphocytes and elevated cytokines can serve as biomarkers of severity of COVID-19.</p>
</div>
<div id="credit-18" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-18" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development" class="anchor"></a>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h3>
<p>Sun et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.951723]</span></p>
<div id="keywords-19" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-19" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
</div>
<div id="main-findings-18" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-18" class="anchor"></a>Main Findings</h4>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S) protein receptor binding domain (RBD) and interactions with ACE2 using computational modeling, and interrogated cross-reactivity and cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the same human receptor ACE2, the receptor binding motif (RBM) is only 50% homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based on the previous crystal structure data of SARS-CoV, and measurement of binding affinities against human ACE2 using recombinant SARS-CoV and SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1 peptides to ACE2, explaining the similar transmissibility of SARS-CoV and SARS-CoV-2 and consistent with previous data (Wall et al Cell 2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal antibodies were about two-order of magnitude less efficient to neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing monoclonal antibodies against SARS-CoV had poor binding and neutralizing activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding monoclonal antibodies show some efficiency to bind and neutralize SARS-CoV-2. The results suggest that that antibodies to more conserved regions outside the RBM motif might possess better cross-protective neutralizing activities between two strains.</p>
</div>
<div id="limitations-19" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-19" class="anchor"></a>Limitations</h4>
<p>It would have been helpful to show the epitopes recognized by the monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to make predictions for induction of broadly neutralizing antibodies. The data on monoclonal antibody competition with ACE2 for binding to SARS-CoV RBD should have also included binding on SARS-CoV2, especially for the three monoclonal antibodies that showed neutralization activity for SARS-CoV2. Because of the less homology in RBM sequences between viruses, it still may be possible that these antibodies would recognize the ACE2 RBD in SARS-CoV-2.</p>
</div>
<div id="significance-16" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-16" class="anchor"></a>Significance</h4>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1 or RBD protein could induce monoclonal antibodies to cross-bind and cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these types of antibodies could be found in human survivors or in the asymptomatic populations as well, it might suggest that exposure to previous Coronavirus strains could have induced cross-neutralizing antibodies and resulted in the protection from severe symptoms in some cases of SARS-CoV2.</p>
</div>
<div id="credit-19" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-19" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine" class="section level3">
<h3 class="hasAnchor">
<a href="#protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine" class="anchor"></a>Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h3>
<p>Chen et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.17.951939]</span></p>
<div id="keywords-20" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-20" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
</div>
<div id="main-findings-19" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-19" class="anchor"></a>Main Findings</h4>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and week 4) with SV8000 carrying the information to express a S1-orf8 fusion protein and the N protein from the BJ01 strain of SARS-CoV-1. By week 8, immunized animals had signs of immunological protection (IgG and neutralization titers) against SARS-CoV-1 and were protected against challenge with the PUMC-1 strain, with fewer detectable symptoms of respiratory distress, lower viral load, shorter periods of viral persistence, and less pathology in the lungs compared to non-immunized animals.</p>
</div>
<div id="limitations-20" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-20" class="anchor"></a>Limitations</h4>
<p>The authors should write clearer descriptions of the methods used in this article. They do not describe how the IgG titers or neutralization titers were determined. There are some issues with the presentation of data, for example, in Figure 1a, y-axis should not be Vmax; forming cells and 1d would benefit from showing error bars. Furthermore, although I inferred that the animals were challenged at week 8, the authors did not explicitly detail when the animals were challenged. The authors should explain the design of their vaccine, including the choice of antigens and vector. The authors also do not include a description of the ethical use of animals in their study.</p>
</div>
<div id="significance-17" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-17" class="anchor"></a>Significance</h4>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of possible implications for the current SARS-CoV-2 pandemic. Could a similar vaccine be designed to protect against SARS-CoV-2 and would the concerns regarding emerging viral mutations that the authors describe as a limitation for SARS-CoV-1 also be true in the context of SARS-CoV-2?</p>
</div>
<div id="credit-20" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-20" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19" class="anchor"></a>Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h3>
<p><span class="citation">[@doi:10.1101/2020.02.18.20024364]</span></p>
<div id="keywords-21" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-21" class="anchor"></a>Keywords</h4>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
</div>
<div id="main-findings-20" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-20" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 522 COVID patients and 40 healthy controls from two hospitals in Wuhan, China, authors show both age-dependent and clinical severity-dependent decrease in T cell numbers with elderly patients and patients who are in ICU-care showing the most dramatic decrease in T cell counts. Cytokine profiling of COVID patients reveal that TNF-α, IL-6 and IL-10 are increased in infected patients with patients in the ICU showing the highest levels. Interestingly, these three cytokine levels were inversely correlated with T cell counts and such inverse relationship was preserved throughout the disease progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and flow cytometry of stained peripheral blood of 14 patients and 3 healthy volunteers demonstrate that T cells of COVID patients have increased expression of PD-1 with patients in ICU having the highest number of CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
</div>
<div id="limitations-21" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-21" class="anchor"></a>Limitations</h4>
<p>Compared to the number of patients, number of control (n= 40) is small and is not controlled for age. Additional data linking inflammatory cytokines and the quality of the adaptive response including humoral and antigen specific T cell response is much needed. T cell exhaustion study relies on marker-dependent labeling of T cell functionality of a very limited sample size (n=17)—a functional/mechanistic study of these T cells from PBMCs would have bolstered their claims.</p>
</div>
<div id="significance-18" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-18" class="anchor"></a>Significance</h4>
<p>Limited but contains interesting implications. It is already known in literature that in the context of acute respiratory viral infections CD8 T cells exhibit exhaustion-like phenotypes which further underscores the importance of mechanistic studies that can elucidate how COVID infection leads to lymphopenia and T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10) correlate with or affect effective viral immunity and what types of cells produce these cytokines? Answering that question will help us refine our targets for immune-modulatory therapies especially in patients suffering from cytokine storms.</p>
</div>
<div id="credit-21" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-21" class="anchor"></a>Credit</h4>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china" class="anchor"></a>Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.20025239]</span></p>
<div id="keywords-22" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-22" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
</div>
<div id="main-findings-21" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-21" class="anchor"></a>Main Findings</h4>
<p>Most common chronic conditions among 25 patients that died from COVID-19 related respiratory failure were hypertension (64%) and diabetes (40%). Disease progression was marked by progressive organ failure, starting first with lung dysfunction, then heart (e.g. increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN, Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88% also had lymphopenia. General markers of inflammation were also increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
</div>
<div id="limitations-22" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-22" class="anchor"></a>Limitations</h4>
<p>The limitations of this study include small sample size and lack of measurements for some tests for several patients. This study would also have been stronger with comparison of the same measurements to patients suffering from less severe disease to further validate and correlate proposed biomarkers with disease severity.</p>
</div>
<div id="significance-19" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-19" class="anchor"></a>Significance</h4>
<p>This study identifies chronic conditions (i.e. hypertension and diabetes) that strongly correlates with disease severity. In addition to general markers of inflammation, the authors also identify concomitant neutrophilia and lymphopenia among their cohort of patients. This is a potentially interesting immunological finding because we would typically expect increased lymphocytes during a viral infection. Neutrophilia may also be contributing to cytokine storm. In addition, PCT was elevated in 90.5% of patients, suggesting a role for sepsis or secondary bacterial infection in COVID-19 related respiratory failure.</p>
</div>
<div id="credit-22" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-22" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china" class="anchor"></a>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.20025288]</span></p>
<div id="keywords-23" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-23" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
</div>
<div id="main-findings-22" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-22" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over 4 weeks for correlation with renal injury. Comorbidities included chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in blood urea or creatinine, and 7.2% of patients with no prior kidney disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal replacement therapy (CRRT) alongside CoVID-19 treatment. Renal functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without progression to ARDS or worsening of CRF.</p></li>
</ul>
</div>
<div id="limitations-23" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-23" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney abnormalities not tabulated separately, making overall data hard to interpret. It will be critical to separately examine kidney function (BUN, urine creatinine and eGFR) in patients that developed any kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will be useful to correlate how treatment modality influences kidney function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low instances of kidney damage<span class="citation">[@doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.06.20020974]</span>, but those studies did not track longitudinal urine samples for acute renal injury markers and viral shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of CoVID-19 status; it will be important to compare clinical parameters of these patients (n=5) with virus-naïve CRF patients (none in this study) to make any meaningful conclusions.</p></li>
</ul>
</div>
<div id="significance-20" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-20" class="anchor"></a>Significance</h4>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and not associated with high mortaility, in stark contrast with a concurrent study <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>. If supported by further studies, this argues kidney impairment is secondary to cytokine storm/inflammation-induced organ failure, and not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of CoVID-19 patients to conclude if kidney function actively disrupted due to viral infection.</p></li>
</ul>
</div>
<div id="credit-23" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-23" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov" class="section level3">
<h3 class="hasAnchor">
<a href="#potential-t-cell-and-b-cell-epitopes-of-2019-ncov" class="anchor"></a>Potential T-cell and B-Cell Epitopes of 2019-nCoV</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.955484]</span></p>
<div id="keywords-24" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-24" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
</div>
<div id="main-findings-23" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-23" class="anchor"></a>Main Findings</h4>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and MARIA, to identify regions within the COVID-19 genome that are presentable by HLA. They identify 405 viral epitopes that are presentable on MHC-I and MHC-II and validate using known epitopes from SARS-CoV. To determine whether immune surveillance drives viral mutations to evade MHC presentation, the authors analyzed 68 viral genomes from 4 continents. They identified 93 point mutations that occurred preferentially in regions predicted to be presented by MHC-I (p=0.02) suggesting viral evolution to evade CD8 T-cell mediated killing. 2 nonsense mutations were also identified that resulted in loss of presentation of an associated antigen (FGDSVEEVL) predicted to be good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the authors used homology modeling to the SARS-CoV’s spike protein (S protein) to determine the putative structure of the CoV2 spike protein. They used Discotope2 to identify antibody binding sites on the protein surface in both the down and up conformations of the S protein. The authors validate this approach by first identifying antibody binding site in SARS-CoV S protein. In both the down and up conformation of the CoV2 S protein, the authors identified a potential antibody binding site on the S protein receptor binding domain (RBD) of the ACE2 receptor (residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike proteins may be important for antibody binding, the authors note that SARS-CoV has larger attack surfaces than CoV2. These results were later validated on published crystal structures of the CoV2 S protein RBD and human ACE2. Furthermore, analysis of 68 viral genomes did not identify any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine candidates across the viral genome that can be presented by multiple HLA alleles. Several of the peptides showed homology to SARS-CoV T-cell and B-cell epitopes.</p>
</div>
<div id="limitations-24" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-24" class="anchor"></a>Limitations</h4>
<p>While the authors used computational methods of validation, primarily through multiple comparisons to published SARS-CoV structures and epitopes, future work should include experimental validation of putative T-cell and B-cell epitopes.</p>
</div>
<div id="significance-21" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-21" class="anchor"></a>Significance</h4>
<p>The authors identified potential T-cell and B-cell epitopes that may be good candidates for peptide based vaccines against CoV2. They also made interesting observations in comparing SARS-CoV and CoV2 potential antibody binding sites, noting that SARS-CoV had larger attack surfaces for potential neutralizing antibody binding. One of the highlights of this paper was the authors’ mutation analysis of 68 viral genomes from 4 continents. This analysis not only validated their computational method for identifying T-cell epitopes, but showed that immune surveillance likely drives viral mutation in MHC-I binding peptides. The smaller attack surface may point to potential mechanisms of immune evasion by CoV2. However, absence of mutations in the RBD of CoV2 and the small number of mutations in peptides presentable to T cells suggests that vaccines against multiple epitopes could still elicit robust immunity against CoV2.</p>
</div>
<div id="credit-24" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-24" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>a.<strong>Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</strong></p>
<p>b. <strong>binding affinity, antigenicity, neutralizing antibody</strong></p>
<p>c. <strong>The authors highlight a human angiotensin-converting enzyme 2 (hACE2), as a potential receptor used by the current Severe Acute respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that allows the virus target human cells. This virus-host interaction facilitates the infection of human cells with a high affinity comparable with SARS-CoV. The authors propose this mechanism as a probable explanation of the efficient transmission of SARS-CoV-2 between humans. Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry into VeroE6 infected cells.</strong></p>
<p>d. <strong>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>, 2020. The authors used green monkey (VeroE6) and hamster (BHK) cell lines in the experiments to drive its conclusions to humans; however, it is well known the caucasian colon adenocarcinoma human cell line (CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as well. In humans, ACE2 protein is highly expressed in the gastrointestinal tract, which again, makes the CaCo-2 cell line suitable for the following SARS-CoV-2 studies.</strong></p>
<p>e. <strong>The results propose a functional receptor used by SARS-CoV-2 to infect humans worldwide and defining two distinct conformations of spike (S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study might help establish a precedent for initial drug design and treatment of the current global human coronavirus epidemic.</strong></p>
</div>
</div>
<div id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein" class="section level3">
<h3 class="hasAnchor">
<a href="#structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein" class="anchor"></a>Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h3>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.19.956581]</span> now <span class="citation">[@doi:10.1016/j.cell.2020.02.058]</span></p>
<div id="keywords-25" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-25" class="anchor"></a>Keywords</h4>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
</div>
<div id="main-findings-24" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-24" class="anchor"></a>Main Findings</h4>
<p>The authors highlight a human angiotensin-converting enzyme 2 (hACE2), as a potential receptor used by the current Severe Acute respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that allows the virus target human cells. This virus-host interaction facilitates the infection of human cells with a high affinity comparable with SARS-CoV. The authors propose this mechanism as a probable explanation of the efficient transmission of SARS-CoV-2 between humans. Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry into VeroE6 infected cells.**</p>
</div>
<div id="limitations-25" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-25" class="anchor"></a>Limitations</h4>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>, 2020. The authors used green monkey (VeroE6) and hamster (BHK) cell lines in the experiments to drive its conclusions to humans; however, it is well known the caucasian colon adenocarcinoma human cell line (CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as well. In humans, ACE2 protein is highly expressed in the gastrointestinal tract, which again, makes the CaCo-2 cell line suitable for the following SARS-CoV-2 studies.</p>
</div>
<div id="significance-22" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-22" class="anchor"></a>Significance</h4>
<p>The results propose a functional receptor used by SARS-CoV-2 to infect humans worldwide and defining two distinct conformations of spike (S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study might help establish a precedent for initial drug design and treatment of the current global human coronavirus epidemic.</p>
</div>
<div id="credit-25" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-25" class="anchor"></a>Credit</h4>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19" class="anchor"></a>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h3>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.20.20025841]</span></p>
<div id="keywords-26" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-26" class="anchor"></a>Keywords</h4>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
</div>
<div id="main-findings-25" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-25" class="anchor"></a>Main Findings</h4>
<p>The authors characterized the immune response in peripheral blood of a 47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces samples, but not in urine, rectal swab, whole blood or throat swab. 7 days after symptom onset, the nasopharyngeal swab test turned negative, at day 10 the radiography infiltrates were cleared and at day 13 the patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of <strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK frequencies and decreased monocyte frequencies compared to healthy counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9 and was associated with higher production of cytotoxic mediators by CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3 SNP-rs12252-C/C variant</strong> in this patient, which is associated with higher susceptibility to influenza virus.</p>
</div>
<div id="limitations-26" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-26" class="anchor"></a>Limitations</h4>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their lungs <span class="citation">[@doi:10.1101/2020.02.23.20026690]</span>. Monitoring monocyte proportions in blood earlier in the disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is not sufficient to rule out NK cell activation upon SARS-CoV2 infection. In response to influenza A virus, NK cells express higher levels of activation markers CD69 and CD38, proliferate better and display higher cytotoxicity <span class="citation">[@doi:10.3389/fimmu.2019.01116]</span>. Assessing these parameters in COVID-19 patients is required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP associated with severe outcome following influenza infection. The association between this SNP and outcome upon SARS-CoV2 infection should be further investigated.</p>
</div>
<div id="significance-23" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-23" class="anchor"></a>Significance</h4>
<p>This study is among the first to describe the appearance of COVID-19-binding IgG and IgM antibodies upon infection. The emergence of new serological assays might contribute to monitor more precisely the seroconversion kinetics of COVID-19 patients <span class="citation">[@doi:10.1101/2020.03.17.20037713]</span>. Further association studies between IFITM3 SNP-rs12252-C/C variant and clinical data might help to refine the COVID-19 outcome prediction tools.</p>
</div>
<div id="credit-26" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-26" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing" class="section level3">
<h3 class="hasAnchor">
<a href="#the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing" class="anchor"></a>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h3>
<p>Liao et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.23.20026690]</span></p>
<div id="keywords-27" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-27" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-26" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-26" class="anchor"></a>Main Findings</h4>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild cases, n=3 severe cases). Data was compared to previously generated scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell organization between mild and severe disease. Specifically, they found that transcriptional clusters annotated as tissue resident alveolar macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup> inflammatory macrophages dominated the BAL of patients with severe COVID19 diseases. They show that inflammatory macrophages upregulated interferon-signaling genes, monocytes recruiting chemokines including CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine TGF-β, while alveolar macrophages expressed lipid metabolism genes, such as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients. Clonally expanded CD8 T cells were enriched in mild cases suggesting that CD8 T cells contribute to viral clearance as in Flu infection, whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but considered here as ambient contaminations.</p>
</div>
<div id="limitations-27" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-27" class="anchor"></a>Limitations</h4>
<p>These results are based on samples from 6 patients and should therefore be confirmed in the future in additional patients. Longitudinal monitoring of BAL during disease progression or resolution would have been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in patients towards inflammatory macrophages should be investigated in future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The composition of the “proliferating” cluster and how these cells differ from conventional T cell clusters should be assessed. NK and CD8 T cell transcriptomic profile, in particular the expression of cytotoxic mediator and immune checkpoint transcripts, should be compared between healthy and diseased lesions.</p>
</div>
<div id="significance-24" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-24" class="anchor"></a>Significance</h4>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that contributes to severe lung tissue damage and ARDS <span class="citation">[@doi:10/ggjfnn]</span>. Accumulation of monocyte-derived inflammatory macrophages at the expense of Alveolar macrophages known to play an anti-inflammatory role following respiratory viral infection, in part through the PPARγ pathway <span class="citation">[@doi:10.3389/fimmu.2018.01777; @doi:10.1128/JVI.00030-19]</span> are likely contributing to lung tissue injuries. These data suggest that reduction of monocyte accumulation in the lung tissues could help modulate COVID-19-induced inflammation. Further analysis of lymphoid subsets is required to understand the contribution of adaptive immunity to disease outcome.</p>
</div>
<div id="credit-27" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-27" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia" class="section level3">
<h3 class="hasAnchor">
<a href="#can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia" class="anchor"></a>Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h3>
<p>Pan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.25.20024711]</span></p>
<div id="keywords-28" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-28" class="anchor"></a>Keywords</h4>
<ul>
<li>Routine laboratory testing</li>
</ul>
</div>
<div id="main-findings-27" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-27" class="anchor"></a>Main Findings</h4>
<p>In an attempt to use standard laboratory testing for the discrimination between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual community acquired pneumonia (CAP), the authors compared laboratory testing results of 84 NCIP patients with those of a historical group of 316 CAP patients from 2018 naturally COVID-19 negative. The authors describe significantly lower white blood- as well as red blood- and platelet counts in NCIP patients. When analyzing differential blood counts, lower absolute counts were measured in all subsets of NCIP patients. With regard to clinical chemistry parameters, they found increased AST and bilirubin in NCIP patients as compared to CAP patients.</p>
</div>
<div id="limitations-28" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-28" class="anchor"></a>Limitations</h4>
<p>The authors claim to describe a simple method to rapidly assess a pre-test probability for NCIP. However, the study has substantial weakpoints. The deviation in clinical laboratory values in NCIP patients described here can usually be observed in severely ill patients. The authors do not comment on how severely ill the patients tested here were in comparison to the historical control. Thus, the conclusion that the tests discriminate between CAP and NCIP lacks justification.</p>
</div>
<div id="significance-25" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-25" class="anchor"></a>Significance</h4>
<p>The article strives to compare initial laboratory testing results in patients with COVID-19 pneumonia as compared to patients with a usual community acquired pneumonia. The implications of this study for the current clinical situation seem restricted due to a lack in clinical information and the use of a control group that might not be appropriate.</p>
</div>
<div id="credit-28" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-28" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia" class="anchor"></a>Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h3>
<p><span class="citation">[@doi:10.1101/2020.02.25.20025643]</span></p>
<div id="keywords-29" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-29" class="anchor"></a>Keywords</h4>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
</div>
<div id="main-findings-28" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-28" class="anchor"></a>Main Findings</h4>
<p>This study is a cross-sectional analysis of 100 patients with COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol style="list-style-type: decimal">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest; oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in arterial blood ≤ 300mmHg, progression of disease on imaging to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires mechanical ventilation; shock; other organ failure that requires treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status. Disease severity was significantly associated with levels of IL-2R, IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total WBC count, lymphocyte count, neutrophil count, and eosinophil count were also significantly correlated with disease status. Since this is a retrospective, cross-sectional study of clinical laboratory values, these data may be extrapolated for clinical decision making, but without studies of underlying cellular causes of these changes this study does not contribute to a deeper understanding of SARS-CoV-2 interactions with the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was significantly different from the mean ages of patients designated as severe or critical (p &lt; 0.001). The mean patient age was not significantly different between the severe and critical groups. However, IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B), WBC count, and neutrophil count (Figure 4A, 4B) were all significantly elevated in the critical group compared to severe. These data suggest underlying differences in COVID-19 progression that is unrelated to age.</p>
</div>
<div id="significance-26" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-26" class="anchor"></a>Significance</h4>
<p>Given the inflammatory profile outlined in this study, patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any elevations in the inflammatory biomarkers listed above, should be closely monitored for potential progression to critical status.</p>
</div>
<div id="credit-29" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-29" class="anchor"></a>Credit</h4>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design" class="section level3">
<h3 class="hasAnchor">
<a href="#an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design" class="anchor"></a>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h3>
<p>Herst et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.25.963546]</span></p>
<div id="keywords-30" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-30" class="anchor"></a>Keywords</h4>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
</div>
<div id="main-findings-29" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-29" class="anchor"></a>Main Findings</h4>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52 located in a conserved region of ebolavirus (EBOV) nucleocapsid protein (NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV (maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which was commonly targeted by CD8+ T cells in a group of EBOV survivors carrying the HLA-A*30:01:01 allele. To justify the testing of specific vaccine epitopes in a mouse challenge setting, the authors cite known examples of human pathogen-derived peptide antigens that are also recognized by C57BL/6 mice, as well as existing data surrounding known mouse immunogenicity of peptides related to this EBOV NP region. Testing 3 distinct 9mer peptides over an 11 amino-acid window and comparing to vaccination with the 11mer with a T-cell reactivity readout demonstrated that optimizing peptide length and position for immunogenicity may be crucial, likely due to suboptimal peptide processing and MHC-class-I loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4 ligand) was added to the injectate solution. A second peptide consisting of a predicted MHC-II epitope from the EBOV VG19 protein was added using a separate population of microspheres, and the formulation was injected by intraperitoneal administration. The vaccine was protective against a range of maEBOV doses up to at least 10,000 PFU. Survival was anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the authors state that MHC binding algorithms predict NP44-52 to be a strong binder of a set of more common HLA-A*02 alleles. The authors predict that a peptide vaccine based on the proposed formulation could elicit responses in up to 50% of people in Sudan or 30% of people in North America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for HLA-binding 9mers identified 53 peptides with predicted binding affinity &lt; 500nM, including peptides that are predicted to bind to HLA-class-I alleles of 97% of humans, 7 of which have previously been tested <em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain cytokines and disease severity, specifically IL6 which relates to multiple trial therapies. Prediction of HLA-class-I binding of SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine targeting conserved regions of SARS-CoV-2 NP although further validation in previously infected patient samples will be essential.</p>
</div>
<div id="credit-30" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-30" class="anchor"></a>Credit</h4>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</strong></p>
<p>Keywords: Peptide vaccine; Ebolavirus; nucleocapsid; epitope; vaccine design; microsphere</p>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52 located in a conserved region of ebolavirus (EBOV) nucleocapsid protein (NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV (maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which was commonly targeted by CD8+ T cells in a group of EBOV survivors carrying the HLA-A*30:01:01 allele. To justify the testing of specific vaccine epitopes in a mouse challenge setting, the authors cite known examples of human pathogen-derived peptide antigens that are also recognized by C57BL/6 mice, as well as existing data surrounding known mouse immunogenicity of peptides related to this EBOV NP region. Testing 3 distinct 9mer peptides over an 11 amino-acid window and comparing to vaccination with the 11mer with a T-cell reactivity readout demonstrated that optimizing peptide length and position for immunogenicity may be crucial, likely due to suboptimal peptide processing and MHC-class-I loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4 ligand) was added to the injectate solution. A second peptide consisting of a predicted MHC-II epitope from the EBOV VG19 protein was added using a separate population of microspheres, and the formulation was injected by intraperitoneal administration. The vaccine was protective against a range of maEBOV doses up to at least 10,000 PFU. Survival was anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the authors state that MHC binding algorithms predict NP44-52 to be a strong binder of a set of more common HLA-A*02 alleles. The authors predict that a peptide vaccine based on the proposed formulation could elicit responses in up to 50% of people in Sudan or 30% of people in North America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for HLA-binding 9mers identified 53 peptides with predicted binding affinity &lt; 500nM, including peptides that are predicted to bind to HLA-class-I alleles of 97% of humans, 7 of which have previously been tested <em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain cytokines and disease severity, specifically IL6 which relates to multiple trial therapies. Prediction of HLA-class-I binding of SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine targeting conserved regions of SARS-CoV-2 NP although further validation in previously infected patient samples will be essential.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2" class="anchor"></a>Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h3>
<p>Li et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.25.965434]</span></p>
<div id="keywords-31" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-31" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
</div>
<div id="main-findings-30" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-30" class="anchor"></a>Main Findings</h4>
<p>This study employs a series of bioinformatic pipelines to identify T and B cell epitopes on spike (S) protein of SARS-CoV-2 and assess their properties for vaccine potential. To identify B cell epitopes, they assessed structural accessibility, hydrophilicity, and beta-turn and flexibility which are all factors that promote their targeting by antibodies. To identify T cell epitopes, they filtered for peptides with high antigenicity score and capacity to bind 3 or more MHC alleles. Using the protein digest server, they also demonstrated that their identified T and B cell epitopes are stable, having multiple non-digesting enzymes per epitope. Epitopes were also determined to be non-allergenic and non-toxin as assessed by Allergen FP 1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the strength of epitope-HLA interaction via PepSite. Overall, they predict four B cell and eleven T cell epitopes (two MHC I and nine MHC II binding) to pass stringent computational thresholds as candidates for vaccine development. Furthermore, they performed sequence alignment between all identified SARS-CoV-2 S protein mutations and predicted epitopes, and showed that the epitopes are conserved across 134 isolates from 38 locations worldwide. However, they report that these conserved epitopes may soon become obsolete given the known mutation rate of related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the urgency of anti-viral vaccine development.</p>
</div>
<div id="limitations-29" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-29" class="anchor"></a>Limitations</h4>
<p>While spike (S) protein may have a critical role in viral entry into host cells and their epitope prediction criterion were comprehensive, this study did not examine other candidate SARS-CoV-2 proteins. This point is particularly important given that a single epitope may not be sufficient to induce robust immune memory, and recent approaches involve multi-epitope vaccine design. Furthermore, their study only included a direct implementation of various published methods, but did not validate individual bioinformatic tools with controls to demonstrate robustness. Finally, it is critical that these predicted epitopes are experimentally validated before any conclusions can be drawn about their potential as vaccine candidates or their clinical efficacy.</p>
</div>
<div id="significance-27" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-27" class="anchor"></a>Significance</h4>
<p>This study provides a computational framework to rapidly identify epitopes that may serve as potential vaccine candidates for treating SARS-CoV-2.</p>
</div>
<div id="credit-31" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-31" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title:</em></strong> Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2, immune-informatics, vaccine design, T cell epitope, B cell epitope</p>
<p><strong><em>Main findings:</em></strong> This study employs a series of bioinformatic pipelines to identify T and B cell epitopes on spike (S) protein of SARS-CoV-2 and assess their properties for vaccine potential. To identify B cell epitopes, they assessed structural accessibility, hydrophilicity, and beta-turn and flexibility which are all factors that promote their targeting by antibodies. To identify T cell epitopes, they filtered for peptides with high antigenicity score and capacity to bind 3 or more MHC alleles. Using the protein digest server, they also demonstrated that their identified T and B cell epitopes are stable, having multiple non-digesting enzymes per epitope. Epitopes were also determined to be non-allergenic and non-toxin as assessed by Allergen FP 1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the strength of epitope-HLA interaction via PepSite. Overall, they predict four B cell and eleven T cell epitopes (two MHC I and nine MHC II binding) to pass stringent computational thresholds as candidates for vaccine development. Furthermore, they performed sequence alignment between all identified SARS-CoV-2 S protein mutations and predicted epitopes, and showed that the epitopes are conserved across 134 isolates from 38 locations worldwide. However, they report that these conserved epitopes may soon become obsolete given the known mutation rate of related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the urgency of anti-viral vaccine development.</p>
<p><strong><em>Limitations of the Study:</em></strong> While spike (S) protein may have a critical role in viral entry into host cells and their epitope prediction criterion were comprehensive, this study did not examine other candidate SARS-CoV-2 proteins. This point is particularly important given that a single epitope may not be sufficient to induce robust immune memory, and recent approaches involve multi-epitope vaccine design. Furthermore, their study only included a direct implementation of various published methods, but did not validate individual bioinformatic tools with controls to demonstrate robustness. Finally, it is critical that these predicted epitopes are experimentally validated before any conclusions can be drawn about their potential as vaccine candidates or their clinical efficacy.</p>
<p><strong><em>Relevance:</em></strong> This study provides a computational framework to rapidly identify epitopes that may serve as potential vaccine candidates for treating SARS-CoV-2.</p>
</div>
</div>
<div id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis" class="anchor"></a>The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h3>
<p>Wang Jr. et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.26.20026989]</span></p>
<div id="keywords-32" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-32" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
</div>
<div id="main-findings-31" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-31" class="anchor"></a>Main Findings</h4>
<p>This study describes the occurrence of a cytokine release syndrome-like (CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time from first symptom to acute respiratory distress syndrome (ARDS) was 10 days. All patients had decreased CD3, CD4 and CD8 cells, and a significant increase of serum IL-6. Furthermore, 91% had decreased NK cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell counts. All of these parameters correlated with the area of pulmonary inflammation in CT scan images. Mechanical ventilation increased the numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and improving the immunological parameters.</p>
</div>
<div id="limitations-30" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-30" class="anchor"></a>Limitations</h4>
<p>The number of patients included in this retrospective single center study is small (n=11), and the follow-up period very short (25 days). Eight of the eleven patients were described as having CRSL, and were treated by intubation (7) or ECMO (2). Nine patients were still in the intensive care unit at the time of publication of this article, so their disease outcome is unknown.</p>
</div>
<div id="significance-28" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-28" class="anchor"></a>Significance</h4>
<p>The authors define a cytokine release syndrome-like toxicity in patients with COVID-19 with clinical radiological and immunological criteria: 1) decrease of circulating CD4, CD8 and NK cells; 2) substantial increase of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue damage. This event seems to occur very often in critically ill patients with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the peripheral blood preceded other laboratory alterations, thus, IL-6 might be an early biomarker for the severity of COVID-19 pneumonia. The manuscript will require considerable editing for organization and clarity.</p>
</div>
<div id="credit-32" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-32" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china" class="anchor"></a>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h3>
<p>Huang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.27.20029009]</span></p>
<div id="keywords-33" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-33" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
</div>
<div id="main-findings-32" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-32" class="anchor"></a>Main Findings</h4>
<p>This is a simple study reporting clinical characteristics of patients who did not survive COVID-19. All patients (mean age=69.22 years) had acute respiratory distress syndrome (ARDS) and their median time from onset to ARDS was 11 days. The median time from onset to death was 17 days. Most patients were older male (70% male) with co-morbidities and only 11 % were smokers. 75% patients showed bilateral pneumonia. Many patients had chronic diseases, including hypertension (58.33%). cardiovascular disease (22.22%) and diabetes (19.44%). Typical clinical feature measured in these patients includes lymphopenia and elevated markers of inflammation.</p>
</div>
<div id="limitations-31" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-31" class="anchor"></a>Limitations</h4>
<p>As noted by the authors, the conclusions of this study are very limited because this is single-centered study focusing on a small cohort of patients who did not survive. Many clinical parameters observed by the authors (such* as increase levels of serum CRP, PCT, IL-6) have also been described in other COVID19 patients who survived the infection</p>
</div>
<div id="significance-29" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-29" class="anchor"></a>Significance</h4>
<p>This study is essentially descriptive and may be useful for clinical teams monitoring COVID19 patients.</p>
</div>
<div id="credit-33" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-33" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Edited By:</p>
<p><strong>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</strong></p>
<p>Huang et al</p>
<p><strong>doi:</strong> <a href="https://doi.org/10.1101/2020.02.27.20029009" class="uri">https://doi.org/10.1101/2020.02.27.20029009</a></p>
<p>Quick Link <a href="https://www.medrxiv.org/content/10.1101/2020.02.27.20029009v2" class="uri">https://www.medrxiv.org/content/10.1101/2020.02.27.20029009v2</a></p>
<p><em>Keywords:</em> Clinical Characteristics, Non Survivors, retrospective study</p>
<p><em>Main Findings:</em> This is a simple study reporting clinical characteristics of patients who did not survive COVID-19. All patients (mean age=69.22 years) had acute respiratory distress syndrome (ARDS) and their median time from onset to ARDS was 11 days. The median time from onset to death was 17 days. Most patients were older male (70% male) with co-morbidities and only 11 % were smokers. 75% patients showed bilateral pneumonia. Many patients had chronic diseases, including hypertension (58.33%). cardiovascular disease (22.22%) and diabetes (19.44%). Typical clinical feature measured in these patients includes lymphopenia and elevated markers of inflammation.</p>
<p><em>Limitations: As noted by the authors, the conclusions of this study are very limited because this is single-centered study focusing on a small cohort of patients who did not survive. Many clinical parameters observed by the authors (such</em> as increase levels of serum CRP, PCT, IL-6) have also been described in other COVID19 patients who survived the infection</p>
<p><em>Relevance:</em> This study is essentially descriptive and may be useful for clinical teams monitoring COVID19 patients.</p>
</div>
</div>
<div id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019" class="anchor"></a>Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h3>
<p><span class="citation">[@doi:10.1101/2020.02.28.20028514]</span></p>
<div id="keywords-34" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-34" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
</div>
<div id="main-findings-33" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-33" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 85 hospitalized COVID patients in a Beijing hospital, authors showed that patients with elevated ALT levels (n = 33) were characterized by significantly higher levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia compared to their counterparts with normal ALT levels. Proportion of severe and critical patients in the ALT elevation group was significantly higher than that of normal ALT group. Multivariate logistic regression performed on clinical factors related to ALT elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count (&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a finding that led the authors to suggest cytokine storm as a major mechanism of liver damage.</p>
</div>
<div id="limitations-32" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-32" class="anchor"></a>Limitations</h4>
<p>The article’s most attractive claim that liver damage seen in COVID patients is caused by cytokine storm (rather than direct infection of the liver) hinges solely on their multivariate regression analysis. Without further mechanistic studies a) demonstrating how high levels of inflammatory cytokines can induce liver damage and b) contrasting types of liver damage incurred by direct infection of the liver vs. system-wide elevation of inflammatory cytokines, their claim remains thin. It is also worth noting that six of their elevated ALT group (n=33) had a history of liver disease (i.e. HBV infection, alcoholic liver disease, fatty liver) which can confound their effort to pin down the cause of hepatic injury to COVID.</p>
</div>
<div id="significance-30" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-30" class="anchor"></a>Significance</h4>
<p>Limited. This article confirms a rich body of literature describing liver damage and lymphopenia in COVID patients.</p>
</div>
<div id="credit-34" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-34" class="anchor"></a>Credit</h4>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients" class="anchor"></a>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h3>
<p><span class="citation">[@doi:10.1101/2020.02.29.20029520]</span></p>
<div id="keywords-35" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-35" class="anchor"></a>Keywords</h4>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
</div>
<div id="main-findings-34" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-34" class="anchor"></a>Main Findings</h4>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines for Covid19 diagnosis and treatment version 6 were included in this study. Patients were further sub-divided into three groups based on clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR with no or mild clinical symptoms (fever; respiratory; radiological abnormalities); (2) severe, at least one of the following: shortness of breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%, Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary damage); and (3) critically ill, at least one additional complicating factor: respiratory failure with need for mechanical ventilation; systemic shock; multi-organ failure and transfer to ICU. Serum samples and throat-swaps were collected from all 48 patients enrolled. SARS-CoV-2 RNA was assessed by qPCR with positive results being defined as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified using a commercially available detection kit. Briefly, patient characteristics in this study confirm previous reports suggesting that higher age and comorbidities are significant risk factors of clinical severity. Of note, 5 out of 48 of patients (10.41%), all in the critically ill category, were found to have detectable serum SARS-CoV-2 RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these patients were found to be substantially higher and this correlated with the presence of detectable SARS-CoV-2 RNA levels. The authors hypothesize that viral RNA might be released from acutely damages tissues in moribund patients during the course of Covid19 and that RNaemia along with IL-6 could potentially be used as a prognostic marker.</p>
</div>
<div id="limitations-33" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-33" class="anchor"></a>Limitations</h4>
<p>While this group’s report generally confirms some of the major findings of a more extensive study, published in early February 2020, <span class="citation">[@doi:10/ggjfnn]</span>, there are limitations that should be taken into account. First, the number of patients enrolled is relatively small; second, interpretation of these data would benefit from inclusion of information about study specifics as well as providing relevant data on the clinical course of these patients other than the fact that some were admitted to ICU (i.e. demographics on how many patients needed respiratory support, dialysis, APACHE Ii/III or other standard ICU scores as robust prognostic markers for mortality etc). It also remains unclear at which time point the serum samples were taken, i.e. whether at admission, when the diagnosis was made or during the course of the hospital stay (and potentially after onset of therapy, which could have affected both IL-6 and RNA levels). The methods section lacks important information on the qPCR protocol employed, including primers and cycling conditions used. From a technical point of view, Ct values &gt;35 seem somewhat non-specific (although Ct &lt;40 was defined as the CDC cutoff as well) indicating that serum RNA levels are probably very low, therefore stressing the need for highly specific primers and high qPCR efficiency. In addition, the statistical tests used (t-tests, according to the methods section) do not seem appropriate as the organ-specific data such as BUN and troponin T values seem to be not normally distributed across groups (n= 5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations and the differences in patient sample size, it is difficult to believe that these data are statistically significantly different.</p>
</div>
<div id="significance-31" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-31" class="anchor"></a>Significance</h4>
<p>This study is very rudimentary and lacks a lot of relevant clinical details. However, it corroborates some previously published observations regarding RNAemia and IL-6 by another group. Generally, regarding future studies, it would be important to address the question of IL-6 and other inflammatory cytokine dynamics in relation to Covid19 disease kinetics (high levels of IL-6, IL-8 and plasma leukotriene were shown to have prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med. 2015 <span class="citation">[@doi:10.1186/s40169-015-0065-2]</span>).</p>
</div>
<div id="credit-35" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-35" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study" class="section level3">
<h3 class="hasAnchor">
<a href="#lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study" class="anchor"></a>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h3>
<p><span class="citation">[@doi:10.1101/2020.03.01.20029074]</span></p>
<div id="keywords-36" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-36" class="anchor"></a>Keywords</h4>
<ul>
<li>Lymphopenia</li>
</ul>
</div>
<div id="main-findings-35" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-35" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 162 COVID patients from a local hospital in Wuhan, China, the authors show an inverse correlation between lymphocyte % (LYM%) of patients and their disease severity. The authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40 moderate) throughout the disease progression with fatal cases showing no recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The temporal data of LYM % in COVID patients was used to construct a Time-Lymphocyte% model which is used to categorize and predict patients’ disease severity and progression. The model was validated using 92 hospitalized cases and kappa statistic test was used to assess agreement between predicted disease severity and the assigned clinical severity (k = 0.49).</p>
</div>
<div id="limitations-34" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-34" class="anchor"></a>Limitations</h4>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as a predictive model for clinical severity is very simple in its construction and derives from correlative data of 162 patients. In order for the model to be of use, it needs validation using a far more robust data set and possibly a mechanistic study on how COVID leads to lymphopenia in the first place. In addition, it should be noted that no statistical test assessing significance of LYM % values between disease severities was performed.</p>
</div>
<div id="significance-32" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-32" class="anchor"></a>Significance</h4>
<p>This article is of limited significance as it simply reports similar descriptions of COVID patients made in previous literature that severe cases are characterized by lymphopenia.</p>
</div>
<div id="credit-36" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-36" class="anchor"></a>Credit</h4>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019" class="anchor"></a>The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.01.20029769]</span></p>
<div id="keywords-37" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-37" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
</div>
<div id="main-findings-36" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-36" class="anchor"></a>Main Findings</h4>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe and 11 non-severe). Patients with severe symptoms at admission (baseline) showed obvious lymphocytopenia and significantly increased interleukin-6 (IL-6) and CRP, which was positively correlated with symptoms severity. IL-6 at baseline positively correlates with CRP, LDH, ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed significant reduction of IL-6 in remission cases.</p>
</div>
<div id="limitations-35" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-35" class="anchor"></a>Limitations</h4>
<p>Limited sample size at baseline, especially for the non-severe leads to question on representativeness. The longitudinal study method is not described in detail and suffers from non-standardized treatment. Limited panel of pro-inflammatory cytokine was analyzed. Patients with severe disease show a wide range of altered blood composition and biomarkers of inflammation, as well as differences in disease course (53.6% were cured, about 10% developed acute respiratory distress syndrome). The authors comment on associations between IL-6 levels and outcomes, but these were not statistically significant (maybe due to the number of patients, non-standardized treatments, etc.) and data is not shown. Prognostic biomarkers could have been better explored. Study lacks multivariate analysis.</p>
</div>
<div id="significance-33" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-33" class="anchor"></a>Significance</h4>
<p>IL-6 could be used as a pharmacodynamic marker of disease severity. Cytokine Release Syndrome (CRS) is a well-known side effect for CAR-T cancer therapy and there are several effective drugs to manage CRS. Drugs used to manage CRS could be tested to treat the most severe cases of COVID19.</p>
</div>
<div id="credit-37" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-37" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china" class="anchor"></a>Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h3>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.01.20029785]</span></p>
<div id="keywords-38" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-38" class="anchor"></a>Keywords</h4>
<ul>
<li>Routine laboratory testing</li>
</ul>
</div>
<div id="main-findings-37" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-37" class="anchor"></a>Main Findings</h4>
<p>The authors of this study provide a comprehensive analysis of clinical laboratory assessments in 75 patients (median age 47 year old) hospitalized for Corona virus infection in China measuring differential blood counts including T-cell subsets (CD4, CD8), coagulation function, basic blood chemistry, of infection-related biomarkers including CRP, Procalcitonin (PCT) (Precursor of calcitonin that increases during bacterial infection or tissue injury), IL-6 and erythrocyte sedimentation rate as well as clinical parameters. Among the most common hematological changes they found increased neutrophils, reduced CD4 and CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and PCT levels were significantly increased in infected patients suggesting that increased IL-6 may correlate well with disease severity in COVID-19 infections</p>
</div>
<div id="limitations-36" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-36" class="anchor"></a>Limitations</h4>
<p>The authors performed an early assessment of clinical standard parameters in patients infected with COVID-19. Overall, the number of cases (75) is rather low and the snapshot approach does not inform about dynamics and thus potential relevance in the assessment of treatment options in this group of patients.</p>
</div>
<div id="significance-34" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-34" class="anchor"></a>Significance</h4>
<p>The article summarizes provides a good summary of some of the common changes in immune cells inflammatory cytokines in patients with a COVID-19 infection and. Understanding how these changes can help predict severity of disease and guide therapy including IL-6 cytokine receptor blockade using Tocilizumab or Sarilumab will be important to explore.</p>
</div>
<div id="credit-38" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-38" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome" class="section level3">
<h3 class="hasAnchor">
<a href="#exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome" class="anchor"></a>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h3>
<p>Yang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.02.20029975]</span></p>
<div id="keywords-39" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-39" class="anchor"></a>Keywords</h4>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
</div>
<div id="summary-2" class="section level4">
<h4 class="hasAnchor">
<a href="#summary-2" class="anchor"></a>Summary</h4>
<p>Plasma cytokine analysis (48 cytokines) was performed on COVID-19 patient plasma samples, who were sub-stratified as severe (N=34), moderate (N=19), and compared to healthy controls (N=8). Patients were monitored for up to 24 days after illness onset: viral load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests, and epidemiological/clinical characteristics of patients were reported.</p>
</div>
<div id="main-findings-38" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-38" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function (PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001 and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic insufficiency, renal insufficiency).</p></li>
</ul>
</div>
<div id="limitations-37" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-37" class="anchor"></a>Limitations</h4>
<p>Collection time of clinical data and lab results not reported directly (likely 4 days (2,6) after illness onset), making it very difficult to determine if cytokines were predictive of patient outcome or reflective of patient compensatory immune response (likely the latter). Small N for cytokine analysis (N=2 fatal and N=5 severe/critical, and N=7 moderate or discharged). Viral treatment strategy not clearly outlined.</p>
</div>
<div id="expanded-results" class="section level4">
<h4 class="hasAnchor">
<a href="#expanded-results" class="anchor"></a>Expanded Results</h4>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory manifestations, and radiological findings consistent with pneumonia while severe patients had one or more of the following: 1) respiratory distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300 mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3, and IL-Ra</strong> as measure by area under the curve analysis during sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif. upregulated throughout observation timecourse, and IP-10 increased and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF, MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as a potential treatment for amelioration of COVID-19 (and associated ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li>
<p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/ severe patients</strong></p>
<ul>
<li>
<strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or eosino/basophil % measured</li>
</ul>
</li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and LDH were not significantly different between groups and were not compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly different (chronic heart/lung/renal/liver disease, diabetes, or cancer) and patient time courses (onset to admission and onset to viral tx) also not significantly different – 4 days (2, 6) on average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74) vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%), myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
</div>
<div id="significance-35" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-35" class="anchor"></a>Significance</h4>
<p>Outline of pathological time course (implicating innate immunity esp.) and identification key cytokines associated with disease severity and prognosis (+ comorbidities). Anti-IP-10 as a possible therapeutic intervention (ex: Eldelumab).</p>
</div>
<div id="credit-39" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-39" class="anchor"></a>Credit</h4>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019" class="anchor"></a>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h3>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.02.20030189]</span></p>
<div id="keywords-40" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-40" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
</div>
<div id="main-findings-39" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-39" class="anchor"></a>Main Findings</h4>
<p>This study examined antibody responses in the blood of COVID-19 patients during the early SARS CoV2 outbreak in China. Total 535 plasma samples were collected from 173 patients (51.4% female) and were tested for seroconversion rate using ELISA. Authors also compared the sensitivity of RNA and antibody tests over the course of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody (Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. Overall, the seroconversion of Ab was significantly quicker than that of IgM (p = 0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates between critical and non-critical patients did not reveal any significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease onset, reached 90% at day 13 and 100% at later time points (15-39 days). In contrast, RNA was only detectable in 45.5% of samples at days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6% (2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable total Ab titers respectively Combining RNA and antibody tests significantly raised the sensitivity for detecting COVID-19 patients in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative COVID-19 patients (n=9) since onset of disease revealed that antibodies may not be sufficient to clear the virus. It should be noted that increases in of antibody titers were not always accompanied by RNA clearance.</p></li>
</ul>
</div>
<div id="limitations-38" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-38" class="anchor"></a>Limitations</h4>
<p>Because different types of ELISA assays were used for determining antibody concentrations at different time points after disease onset, sequential seroconversion of total Ab, IgM and IgG may not represent actual temporal differences but rather differences in the affinities of the assays used. Also, due to the lack of blood samples collected from patients in the later stage of illness, how long the antibodies could last remain unknown. For investigative dynamics of antibodies, more samples were required.</p>
</div>
<div id="significance-36" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-36" class="anchor"></a>Significance</h4>
<p>Total and IgG antibody titers could be used to understand the epidemiology of SARS CoV-2 infection and to assist in determining the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological and RNA testing in the diagnosis and clinical management of COVID-19 patients. The understanding of antibody responses and their half-life during and after SARS CoV2 infection is important and warrants further investigations.</p>
</div>
<div id="credit-40" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-40" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients" class="anchor"></a>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<p>Chen et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.03.20030437]</span></p>
<div id="keywords-41" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-41" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
</div>
<div id="main-findings-40" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-40" class="anchor"></a>Main Findings</h4>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14 women) using standard laboratory tests and flow cytometry. All patients were treated with antibiotics. Twenty-four of the 25 patients were also treated with anti-viral Umefinovir and 14 of the patients were treated with corticosteroids. 14 patients became negative for the virus after 8-14 days of treatment. The same treatment course was extended to 15-23 days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of infection. Patients with hypertension, diabetes, malignancy or chronic liver disease were all unable to clear the virus at day 14, though not statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also reported. B cell, CD4 and CD8 T cell counts were only increased upon treatment in patients who cleared the virus. NK cell frequencies remained unchanged after treatment in all the patients.</p>
</div>
<div id="limitations-39" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-39" class="anchor"></a>Limitations</h4>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup> treatment, and 43% of all patients who cleared the virus were treated with corticosteroids. Corticosteroids have strong effects on the immune compartment in blood <span class="citation">[@doi:10.1038/srep23002]</span>. The authors should have accounted for corticosteroid treatment when considering changes in T, NK and B cell frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following treatment would have provided insights into the COVID-19 cytokine storm biology. Patients with higher baseline levels of IL-6 have been reported to have lower CD8 and CD4 T cell frequencies <span class="citation">[@doi:10.1101/2020.03.01.20029785]</span>. Correlating IL-6 with cell counts before and after treatment would thus have also been of interest. The report of the laboratory measures in table 2 is incomplete and should include the frequencies of patients with increased/decreased levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data does not support NK cell restoration upon treatment in patients who cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup> treatment course in those patients.</p>
</div>
<div id="significance-37" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-37" class="anchor"></a>Significance</h4>
<p>Previous reports suggest an association between disease severity and elevated IL-6 or pro-calcitonin concentrations in COVID-19 patients <span class="citation">[@doi:10.1101/2020.02.29.20029520; @doi:10.1016/j.cca.2020.03.004]</span>. IL-6 receptor blockade is also being administered to patients enrolled in clinical trials (NCT04317092). This report thus contributes to highlight elevated concentrations of these analytes in COVID-19 patients. Mechanisms underlying the association between viral clearance and restoration of the T cell and B cell frequencies suggests viral-driven immune dysregulation, which needs to be investigated in further studies.</p>
</div>
<div id="credit-41" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-41" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study" class="anchor"></a>Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study </h3>
<p>Xu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.03.20030668]</span></p>
<div id="keywords-42" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-42" class="anchor"></a>Keywords</h4>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
</div>
<div id="main-findings-41" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-41" class="anchor"></a>Main Findings:</h4>
<p>This work analyses laboratory and clinical data from 45 patients treated in the in ICU in a single province in China. Overall, 44% of the patients were intubated within 3 days of ICU admission with only 1 death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ Failure Assessment (SOFA) score (clinical score, the higher the more critical state), LDH levels are positively correlated to SOFA score. Overall, older patients (&gt;60yo), with high SOFA score, high LDH levels and low lymphocytes levels at ICU admission are at higher risk of intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to limited sample size no conclusion can be made.</p>
</div>
<div id="limitations-40" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-40" class="anchor"></a>Limitations</h4>
<p>While the study offers important insights into disease course and clinical lab correlates of outcome, the cohort is relatively small and is likely skewed towards a less-severe population compared to other ICU reports given the outcomes observed. Analysis of laboratory values and predictors of outcomes in larger cohorts will be important to make triage and treatment decisions. As with many retrospective analyses, pre-infection data is limited and thus it is not possible to understand whether lymphopenia was secondary to underlying comorbidities or infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of convalescent plasma administration.</p>
</div>
<div id="significance-38" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-38" class="anchor"></a>Significance</h4>
<p>This clinical data enables the identification of at-risk patients and gives guidance for research for treatment options. Indeed, further work is needed to better understand the causes of the lymphopenia and its correlation with outcome.</p>
</div>
<div id="credit-42" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-42" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2" class="anchor"></a>Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h3>
<p><span class="citation">[@doi:10.1101/2020.03.03.962332]</span></p>
<div id="keywords-43" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-43" class="anchor"></a>Keywords</h4>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
</div>
<div id="main-findings-42" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-42" class="anchor"></a>Main Findings</h4>
<p>Using in silico bioinformatic tools, this study identified putative antigenic B-cell epitopes and HLA restricted T-cell epitopes from the spike, envelope and membrane proteins of SARS-CoV-2, based on the genome sequence available on the NCBI database. T cell epitopes were selected based on predicted affinity for the more common HLA-I alleles in the Chinese population. Subsequently, the authors designed vaccine peptides by bridging selected B-cell epitopes and adjacent T-cell epitopes. Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface of the virion and had a high antigenicity score. A total of 499 T-cell epitopes were predicted. Based on the 19 B-cell epitopes and their 121 adjacent T-cell epitopes, 17 candidate vaccine peptides were designed. Additionally, another 102 vaccine peptides containing T-cell epitopes only were generated. Based on the epitope counts and HLA score, 13 of those were selected. Thus, a total of 30 peptide vaccine candidates were designed.</p>
</div>
<div id="limitations-41" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-41" class="anchor"></a>Limitations</h4>
<p>While this study provides candidates for the development of vaccines against SARS-CoV-2, in vitro and in vivo trials are required to validate the immunogenicity of the selected B and T cell epitopes. This could be done using serum and cells from CoV-2-exposed individuals, and in preclinical studies. The implication of this study for the current epidemic are thus limited. Nevertheless, further research on this field is greatly needed.</p>
</div>
<div id="credit-43" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-43" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china" class="anchor"></a>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h3>
<p>Cao et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.04.20030395]</span></p>
<div id="keywords-44" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-44" class="anchor"></a>Keywords</h4>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
</div>
<div id="main-findings-43" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-43" class="anchor"></a>Main Findings</h4>
<p>This single-center cohort study analyzes the clinical and laboratory features of 198 patients with confirmed COVID-19 infection in Shanghai, China and correlated these parameters with clinical disease severity, including subsequent intensive care unit (ICU) admission. 19 cases (9.5%) required ICU admission after developing respiratory failure or organ dysfunction. Age, male sex, underlying cardiovascular disease, and high symptom severity (high fever, dyspnea) were all significantly correlated with ICU admission. Additionally, ICU admission was more common in patients who presented with lymphopenia and elevated neutrophil counts, among other laboratory abnormalities. Flow cytometric analysis revealed that patients admitted to the ICU had significantly reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+ leukocyte populations compared to the cohort of patients not requiring ICU admission.</p>
</div>
<div id="limitations-42" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-42" class="anchor"></a>Limitations</h4>
<p>The limitations of this study include the relatively small sample size and lack of longitudinal testing. The authors also did not assess whether respiratory comorbidity – such as asthma or chronic obstructive lung disease – in addition to immunosuppression affected ICU admission likelihood.</p>
</div>
<div id="significance-39" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-39" class="anchor"></a>Significance</h4>
<p>COVID-19 has already sickened thousands across the globe, though the severity of these infections is markedly diverse, ranging from mild symptoms to respiratory failure requiring maximal intervention. Understanding what clinical, laboratory, and immunologic factors predict the clinical course of COVID-19 infection permits frontline providers to distribute limited medical resources more effectively.</p>
</div>
<div id="credit-44" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-44" class="anchor"></a>Credit</h4>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
</div>
</div>
<div id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing" class="section level3">
<h3 class="hasAnchor">
<a href="#serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing" class="anchor"></a>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h3>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.04.20030916]</span></p>
<div id="keywords-45" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-45" class="anchor"></a>Keywords</h4>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
</div>
<div id="main-finding" class="section level4">
<h4 class="hasAnchor">
<a href="#main-finding" class="anchor"></a>Main Finding</h4>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by automated chemiluminescent immunoassay in the patients who had been already confirmed as positive by nucleic acid detection, while single positivity of IgM or IgG were detected in a very few cases in the other population including 225 non-COVID-19 cases. In addition to the increase of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was detected in three patients with COVID-19 within a very short of time (0-1 day).</p>
</div>
<div id="limitations-43" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-43" class="anchor"></a>Limitations</h4>
<p>The limitation of this study is only 3 confirmed COVID-19 cases were included, so that the relationship between anti-2019-nCov antibodies and disease progression might not be clearly defined. Another limitation is that they did not show the course of 2019-nCov specific antibodies in the cases with positive for COVID-19 but without clinical symptoms.</p>
</div>
<div id="significance-40" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-40" class="anchor"></a>Significance</h4>
<p>The detection of anti-2019-nCov antibodies can be an alternative method to diagnose and treat COVID-19 more comprehensively by distinguish non COVID-19 patients. It may be helpful to understand the course of individual cases with COVID-19 to predict the prognosis if more cases will be evaluated.</p>
</div>
<div id="credit-45" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-45" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection" class="anchor"></a>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h3>
<p><span class="citation">[@doi:10.1101/2020.03.04.20031120]</span></p>
<div id="keywords-46" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-46" class="anchor"></a>Keywords</h4>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
</div>
<div id="main-finding-1" class="section level4">
<h4 class="hasAnchor">
<a href="#main-finding-1" class="anchor"></a>Main Finding</h4>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19 patients and characterizing tissue damage and viral presence in post-mortem kidney samples from 6 Covid-19 patients, the authors conclude that significant damage occurs to the kidney, following Covid-19 infection. This is in contrast to the SARS infection from the 2003 outbreak. They determine this damage to be more prevalent in patients older than 60 years old, as determined by analysis of eGFR. H&amp;E and IHC analysis in 6 Covid-19 patients revealed that damage was in the tubules, not the glomeruli of the kidneys and suggested that macrophage accumulation and C5b-9 deposition are key to this process. </p>
</div>
<div id="limitations-44" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-44" class="anchor"></a>Limitations</h4>
<p>Severe limitations include that the H&amp;E and IHC samples were performed on post-mortem samples of unknown age, thus we cannot assess how/if age correlates with kidney damage, upon Covid-19 infection. Additionally, eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and proteinuria are amongst other measurements that could have been obtained from patient records. An immune panel of the blood was not performed to assess immune system activation. Additionally, patients are only from one hospital. </p>
</div>
<div id="significance-41" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-41" class="anchor"></a>Significance</h4>
<p>This report makes clear that kidney damage is prevalent in Covid-19 patients and should be accounted for. </p>
</div>
<div id="credit-46" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-46" class="anchor"></a>Credit</h4>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
</div>
</div>
<div id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients" class="anchor"></a>COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h3>
<p>Song et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.05.20031906]</span></p>
<div id="keywords-47" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-47" class="anchor"></a>Keywords</h4>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
</div>
<div id="main-findings-44" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-44" class="anchor"></a>Main Findings</h4>
<p>The aim of this study was to identify diagnostic or prognostic criteria which could identify patients with COVID-19 and predict patients who would go on to develop severe respiratory disease. The authors use EMR data from individuals taking a COVID-19 test at Zhejiang hospital, China in late January/Early February. A large number of clinical parameters were different between individuals with COVID-19 and also between ‘severe’ and ‘non-severe’ infections and the authors combine these into a multivariate linear model to derive a weighted score, presumably intended for clinical use.</p>
</div>
<div id="limitations-45" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-45" class="anchor"></a>Limitations</h4>
<p>Unfortunately, the paper is lacking a lot of crucial information, making it impossible to determine the importance or relevance of the findings. Most importantly, the timings of the clinical measurements are not described relative to the disease course, so it is unclear if the differences between ‘severe’ and ‘non-severe’ infections are occurring before progression to severe disease (which would make them useful prognostic markers), or after (which would not).</p>
</div>
<div id="significance-42" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-42" class="anchor"></a>Significance</h4>
<p>This paper is one of many retrospective studies coming from hospitals in China studying individuals with COVID-19. Because of the sparse description of the study design, this paper offers little new information. However, studies like this could be very valuable and we would strongly encourage the authors to revise this manuscript to include more information about the timeline of clinical measurements in relation to disease onset and more details of patient outcomes.</p>
</div>
<div id="credit-47" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-47" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease" class="section level3">
<h3 class="hasAnchor">
<a href="#ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease" class="anchor"></a>LY6E impairs coronavirus fusion and confers immune control of viral disease</h3>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.05.979260]</span></p>
<div id="keywords-48" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-48" class="anchor"></a>Keywords</h4>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
</div>
<div id="main-findings-45" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-45" class="anchor"></a>Main Findings</h4>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes for antiviral activity against endemic human coronavirus HCoV-229E (associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular infection of Huh7 cells, a human hepatoma cell line susceptible to HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em> experiments including both stable Ly6E overexpression and CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E reduces cellular infection by various other coronaviruses including human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis virus (MHV). Their experiments suggest that this effect is dependent on Ly6E inhibition of CoV strain-specific spike protein-mediated membrane fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup> mice (referred to as functional wild-type mice) with transgenic <em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type and Ly6E HSC ko mice of both sexes were infected intraperitoneally with varying doses of the natural murine coronavirus MHV, generally causing a wide range of diseases in mice including hepatitis, enteritis and encephalomyelitis. Briefly, compared to wild-type controls, mice lacking hematopoietic cell-expressed Ly6E were found to present with a more severe disease phenotype as based on serum ALT levels (prognostic of liver damage), liver histopathology, and viral titers in the spleen. Moreover, bulk RNAseq analysis of infected liver and spleen tissues indicated changes in gene expression pathways related to tissue damage and antiviral immune responses as well as a reduction of genes associated with type I IFN response and inflammation. Finally, the authors report substantial differences in the numbers of hepatic and splenic APC subsets between wild-type and knockout mice following MHV infection and show that Ly6E-deficient B cells and to a lesser extent also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
</div>
<div id="limitations-46" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-46" class="anchor"></a>Limitations</h4>
<p>Experiments and data in this study are presented in an overall logical and coherent fashion; however, some observations and the conclusions drawn are problematic and should be further addressed &amp; discussed by the authors. Methodological &amp; formal limitations include relatively low replicate numbers as well as missing technical replicates for some <em>in vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission of “outliers” in Fig. legend 2 without an apparent rationale as to why this approach was chosen; the lack of detection of actual Ly6E protein levels in Ly6E HSC ko or wild-type mice; and most importantly, missing information on RNAseq data collection &amp; analysis in the method section and throughout the paper. A more relevant concern though is that the interpretation of the experimental data presented and the language used tend to overrate and at times overgeneralize findings: for example, while the authors demonstrate statistically significant, Ly6E-mediated reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it remains unclear whether a viral titer reduction by one log decade would be of actual biological relevance in face of high viral titers <em>in vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation in the spleen (~1.5 log decades) but not liver of the ko mice (other tissue not evaluated), and while ko mice presented with only modestly increased liver pathology, both male and female ko mice exhibited significantly higher mortality. Thus, the manuscript tile statement that “Ly6E … confers immune control of viral disease” is supported by only limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E overexpression) were not performed. Of additional note here, tissue tropism and virulence differ greatly among various MHV strains and isolates whereas dose, route of infection, age, genetic background and sex of the mice used may additionally affect disease outcome and phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A, <a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/ 83/18/9258</a>). Observations attributed to hematopoietic stem cell-specific Ly6E deletion could therefore be influenced by the different genetic backgrounds of floxed and cre mice used, and although it appears that littermates wt and ko littermates were used in the experiments, the potentially decisive impact of strain differences should at least have been discussed. Along these lines, it should also be taken into account that the majority of human coronaviruses cause respiratory symptoms, which follow a different clinical course engaging other primary cellular mediators than the hepatotropic murine MHV disease studied here. It therefore remains highly speculative how the findings reported in this study will translate to human disease and it would therefore be important to test other routes of MHV infection and doses that have been described to produce a more comparable phenotype to human coronavirus disease (<em>cf.</em> Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/ 83/18/9258</a>). Another important shortcoming of this study is the lack of any information on functional deficits or changes in Ly6E-deficient immune cells and how this might relate to the phenotype observed. Overall, the <em>in vitro</em> experiments are more convincing than the <em>in vivo</em> studies which appear somewhat limited.</p>
</div>
<div id="significance-43" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-43" class="anchor"></a>Significance</h4>
<p>Despite some shortcomings, the experiments performed in this study suggest a novel and somewhat unexpected role of Ly6E in the protection against coronaviruses across species. These findings are of relevance and should be further explored in ongoing research on potential coronavirus therapies. Yet an important caveat pertains to the authors’ suggestion that “therapeutic mimicking of Ly6E action” may constitute a first line of defense against novel coronaviruses since their own prior work demonstrated that Ly6E can enhance rather than curtail infection with influenza A and other viruses.</p>
</div>
<div id="credit-48" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-48" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients" class="anchor"></a>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.06.20031856]</span></p>
<div id="keywords-49" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-49" class="anchor"></a>Keywords</h4>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
</div>
<div id="main-findings-46" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-46" class="anchor"></a>Main Findings</h4>
<p>While RT-PCR is being used currently to routinely diagnose infection with SARS-CoV-2, there are significant limitations to the use of a nucleic acid test that lead to a high false-negative rate. This article describes ELISAs that can measure IgM and IgG antibodies against the N protein of SARS-CoV-2 to test samples from 238 patients (153 positive by RT-PCR and 85 negative by RT-PCR) at different times after symptom onset. The positivity rate of the IgM and/or IgG ELISAs was greater than that of the RT-PCR (81.5% compared to 64.3%) with similar positive rates in the confirmed and suspected cases (83% and 78.8%, respectively), suggesting that many of the suspected but RT-PCR-negative cases were also infected. The authors also found that the ELISAs have higher positive rates later after symptom onset while RT-PCR is more effective as a diagnostic test early during the infection.</p>
</div>
<div id="limitations-47" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-47" class="anchor"></a>Limitations</h4>
<p>I cannot identify any limitations to this study.</p>
</div>
<div id="significance-44" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-44" class="anchor"></a>Significance</h4>
<p>The authors make a strong case for using a combination of ELISA and RT-PCR for diagnosis of infection with SARS-CoV-2, especially considering the dynamics of positivity rates of RT-PCR and ELISA. Fewer false-negative diagnoses would improve infection control and patient management.</p>
</div>
<div id="credit-49" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-49" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2" class="anchor"></a>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.06.980037]</span></p>
<div id="keywords-50" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-50" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
</div>
<div id="main-findings-47" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-47" class="anchor"></a>Main Findings</h4>
<p>Whole genome sequencing-based comparisons of the 2003 Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019 SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with this, the authors tested cross-reactivity of murine monoclonal antibodies (mAbs) previously generated against the SARS-CoV spike (S) glycoprotein involved in viral entry. One of the screened mAb, 1A9, was able to bind and cross-neutralize multiple strains of SARS-CoV, as well as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was generated using an immunogenic fragment in the S2 subunit of SARS-CoV and binds through a novel epitope within the S2 subunit at amino acids 1111-1130. It is important to note that CD8+ T lymphocyte epitopes overlap with these residues, suggesting that S2 subunit could be involved in inducing both, humoral and cell-mediated immunity.</p>
</div>
<div id="limitations-48" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-48" class="anchor"></a>Limitations</h4>
<p>The authors used previously generated mouse mAbs against the S protein in SARS-CoV expressed in mammalian cell line. Future experimental validation using COVID-19 patient samples is needed to validate these findings. In addition, the results of these studies are predominantly based on in vitro experiments and so, evaluating the effects of the mAb 1A9 in an animal model infected with this virus will help us better understand the host immune responses in COVID-19 and potential therapeutic vaccines.</p>
</div>
<div id="significance-45" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-45" class="anchor"></a>Significance</h4>
<p>This study identified mAbs that recognize the new coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in developing diagnostic assays for COVID-19.</p>
</div>
<div id="credit-50" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-50" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>SARS-CoV-2 is sensitive to type I interferon pretreatment</strong></p>
<p>Lokugamage et al. 2020</p>
<p>Keywords: Treatment, cytokines, interferons</p>
<p>Summary</p>
<p>Lokugamage et al have found that SARS-CoV-2 has sensitivity to type I interferon treatment <em>in vitro</em>. They report a significant difference in interferon stimulated genes (ISGs) and transcription factor expression between SARS-CoV and SARS-CoV-2 infected VERO cells pretreated with IFN-α. They attribute these differences to significant changes in the viral genes ORF6 and ORF 3b, which have been reported to function as interferon antagonist in SARS-CoV. They suggest using interferon type I treatment as a potential pathway to further analyze in animal models and humans.</p>
<p>Caveats</p>
<p>The study was carried out <em>in vitro</em> in an African monkey fibroblast kidney cell line. They show sensitivity to IFN in cells infected <em>in vitro</em> and do not mention any other pathway.</p>
<p>[]{#_heading=h.gjdgxs .anchor}This is extrapolated from sequence homology of SARS-CoV and SARS-CoV-2 proteins. The study does not test the ability of ORF6 and ORF 3b to inhibit the IFN pathway directly.</p>
<p>Importance of findings</p>
<p>Interferon type I therapies are readily available and have been tested against several viral infections such as SARS-CoV without success. Nevertheless, these results suggest that the difference in homologies between viral proteins ORF6 and ORF 3b of SARS-CoV and SARS-CoV-2 is enough to disrupt this viral defense mechanism.</p>
<p>[]{#_heading=h.1b4b0g63ddt .anchor}<em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population" class="section level3">
<h3 class="hasAnchor">
<a href="#mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population" class="anchor"></a>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h3>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.08.20031229]</span></p>
<div id="keywords-51" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-51" class="anchor"></a>Keywords</h4>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
</div>
<div id="main-findings-48" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-48" class="anchor"></a>Main Findings</h4>
<p>Retrospective study of the clinical characteristics of 752 patients infected with COVID-19 at Chinese PLA General Hospital, Peking Union Medical College Hospital, and affiliated hospitals at Shanghai University of medicine &amp; Health Sciences. This study is the first one that compares PB from healthy controls from the same regions in Shanghai and Beijing, and infected COVID-19 patients to standardize a reference range of WBCs of people at high risk.</p>
</div>
<div id="limitations-49" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-49" class="anchor"></a>Limitations</h4>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T cells- correlated with mortality (WBCs are significantly lower in severe or critical UCI patients vs mild ones). Based on 14,117 normal controls in Chinese Han population (ranging in age from 18-86) it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500 cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this study also reported that the levels of D-dimer, C-reactive protein and IL-6 were elevated in COVID-19 pts., indicating clot formation, severe inflammation and cytokine storm.</p>
</div>
<div id="significance-46" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-46" class="anchor"></a>Significance</h4>
<p>This study sets a threshold to identify patients at risk by analyzing their levels of leukocytes, which is an easy and fast approach to stratify individuals that require hospitalization. Although the study is limited (only counts of WBC are analyzed and not its profile) the data is solid and statistically robust to correlate levels of lymphopenia with mortality.</p>
</div>
<div id="credit-51" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-51" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19" class="anchor"></a>Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h3>
<p>Chen et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.09.20033068]</span></p>
<div id="keywords-52" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-52" class="anchor"></a>Keywords</h4>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
</div>
<div id="main-findings-49" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-49" class="anchor"></a>Main Findings</h4>
<p>This is a retrospective study involving 101 death cases with COVID-19 in Wuhan Jinyintan Hospital. The aim was to describe clinical, epidemiological and laboratory features of fatal cases in order to identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by clinical diagnostics. Males dominated the number of deaths and the average age was 65.46 years. All patients died of respiratory failure and multiple organs failure, except one (acute coronary syndrome). The predominant comorbidities were hypertension (42.57%) and diabetes (22.77%). 25.74% of the patients presented more than two underlying diseases. 82% of patients presented myocardial enzymes abnormalities at admission and further increase in myocardial damage indicators with disease progression: patients with elevated Troponin I progressed faster to death. Alterations in coagulation were also detected. Indicators of liver and kidney damage increased 48 hours before death. The authors studied the deceased patients’ blood type and presented the following results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O (18.19%), which is inconsistent with the distribution in Han population in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%), followed by cough and dyspnea. The medium time from onset of symptoms to acute respiratory distress syndrome (ARDS) development was 12 days. Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented abnormal lung imaging at admission and most had double-lung abnormalities. Related to the laboratorial findings some inflammatory indicators gradually increased during the disease progression, such as IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive protein (CRP), while platelets numbers decreased. The authors also reported an initial lymphopenia that was followed by an increase in the lymphocytes numbers. Neutrophil count increased with disease progression.</p>
<p>The patients received different treatments such as antiviral drugs (60.40%), glucocorticoids, thymosin and immunoglobulins. All patients received antibiotic treatment and some received antifungal drugs. All patients received oxygen therapy (invasive or non-invasive ones).</p>
</div>
<div id="limitations-50" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-50" class="anchor"></a>Limitations</h4>
<p>This study involves just fatal patients, lacking comparisons with other groups of patients e.g. patients that recovered from COVID-19. The authors didn’t discuss the different approaches used for treatments and how these may affect the several parameters measured. The possible relationship between the increase of inflammatory indicators and morbidities of COVID-19 are not discussed.</p>
</div>
<div id="significance-47" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-47" class="anchor"></a>Significance</h4>
<p>This study has the largest cohort of fatal cases reported so far. The authors show that COVID-19 causes fatal respiratory distress syndrome and multiple organ failure. This study highlights prevalent myocardial damage and indicates that cardiac function of COVID-19 patients should be carefully monitored. The data suggest that Troponin I should be further investigated as an early indicator of patients with high risk of accelerated health deterioration. Secondary bacterial and fungal infections were frequent in critically ill patients and these need to be carefully monitored in severe COVID-19 patients. Differences in blood type distribution were observed, suggesting that type A is detrimental while type O is protective – but further studies are needed to confirm these findings and elucidate if blood type influences infection or disease severity. Several inflammatory indicators (neutrophils, PCT, CRP and IL-6, D-dimer) increased according to disease severity and should be assessed as biomarkers and to better understand the biology of progression to severe disease.</p>
</div>
<div id="credit-52" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-52" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell membrane during infection. The spike (S) protein plays an essential role in this step (1). This work sheds light on the particularities of the S protein from SARS-CoV2 and compares it to previously existing coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for hACE2 compared to SARS-CoV. Previous work by this group had established the potential of a previous version of the peptide (EK1) in reducing the infectivity of several coronavirus by blocking the S protein interactions with the host cell (2). In this work, authors present an improved EK1C4 version of this peptide. In vitro experiments on human cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2 and other coronavirus. Administration of EK1C4 in prophylaxis or shortly after exposure to the HCoV-OC43 coronavirus protected newborn mice from virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be related to its increased infectivity compared to CoV-S. However, a previous report suggested that the enhanced cell-to-cell fusion mediated by this furin cleavage might not necessarily translate into enhanced viral infectivity (3). The mutation of this site, or inhibition of furin in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if this mechanism is actually involved in SARS-CoV2 pathogenesis, and potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is based on the coronavirus HCoV-OC43. Further work should aim to establish the potential of EK1C4 for prevention of SARS-CoV-2 replication in more relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is, however, acknowledged, that experimentation in vitro included both SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been previously examined (2), the potential side effects and pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a paramount element of epidemic control, specially in the time before a vaccine can be readily available. Similar strategies have been followed in the past in response to threats of this nature (4,5). This work’s relevance is limited by the absence of relevant animal models of SARS-CoV-2 infection. Further pharmacological analyses would also be necessary to ensure the safety of EK1C4 in animal models before a potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike protein and acquisition of fusion competence. Vol. 4, Viruses. Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S. Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide That Targets Viral Attachment to Cells. J Virol. 2006 Dec 15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human antimicrobial peptides as therapeutics for viral infections. Vol. 11, Viruses. MDPI AG; 2019.</p>
</div>
</div>
<div id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility" class="section level3">
<h3 class="hasAnchor">
<a href="#relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility" class="anchor"></a>Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h3>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.11.20031096]</span></p>
<div id="keywords-53" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-53" class="anchor"></a>Keywords</h4>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
</div>
<div id="main-findings-50" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-50" class="anchor"></a>Main Findings</h4>
<p>These authors compared the ABO blood group of 2,173 patients with RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the ABO blood group distribution in unaffected people in the same cities from previous studies (2015 and 2010 for Wuhan and Shenzhen, respectively). They found that people with blood group A are statistically over-represented in the number of those infected and who succumb to death while those with blood group O are statistically underrepresented with no influence of age or sex.</p>
</div>
<div id="limitations-51" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-51" class="anchor"></a>Limitations</h4>
<p>This study compares patients with COVID-19 to the general population but relies on data published 5 and 10 years ago for the control. The mechanisms that the authors propose may underlie the differences they observed require further study.</p>
</div>
<div id="significance-48" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-48" class="anchor"></a>Significance</h4>
<p>Risk stratification based on blood group may be beneficial for patients and also healthcare workers in infection control. Additionally, investigating the mechanism behind these findings could lead to better developing prophylactic and therapeutic targets for COVID-19.</p>
</div>
<div id="credit-53" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-53" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15" class="section level3">
<h3 class="hasAnchor">
<a href="#the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15" class="anchor"></a>The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h3>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.11.987016]</span></p>
<div id="keywords-54" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-54" class="anchor"></a>Keywords</h4>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
</div>
<div id="main-findings-51" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-51" class="anchor"></a>Main Findings</h4>
<p>This study reconsiders the use of inhaled corticosteroids in the treatment of pneumonia by coronavirus. Corticosteroids were associated with increased mortality for SARS in 2003 and for MERS in 2013, probably due to that fact that systemic corticosteroids suppress the innate immune system, resulting in increased viral replication. However, some steroid compounds might block coronavirus replication. The authors screened steroids from a chemical library and assessed the viral growth suppression and drug cytotoxicity. Ciclesonide demonstrated low cytotoxicity and potent suppression of MERS-CoV viral growth. The commonly used systemic steroids cortisone, prednisolone and dexamethasone did not suppress viral growth, nor did the commonly used inhaled steroid fluticasone. To identify the drug target of virus replication, the authors conducted 11 consecutive MERS-CoV passages in the presence of ciclesonide or mometasone, and they could generate a mutant virus that developed resistance to ciclesonide, but not to mometasone. Afterwards, they performed next-generation sequencing and identified an amino acid substitution in nonstructural protein 15 (NSP15) as the predicted mechanism for viral resistance to ciclesonide. The authors were able to successfully generate a recombinant virus carrying that amino acid substitution, which overcome the antiviral effect of ciclesonide, suggesting that ciclosenide interacts with NSP15. The mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone is different from that of ciclesonide. Lastly, the effects of ciclesonide and mometason on suppressing the replication of SARS-CoV-2 were evaluated. Both compounds were found to suppress viral replication with a similar efficacy to lopinavir.</p>
</div>
<div id="limitations-52" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-52" class="anchor"></a>Limitations</h4>
<p>Most of the experiments, including the identification of the mutation in NSP15 were conducted with MERS-CoV. This is not the closest related virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the initial experiments with SARS-CoV, or preferably SARS-CoV-2, is essential. The manuscript should address this and, therefore, it will require considerable editing for organization and clarity. Also, in terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein contains several predicted B and T cell immunogenic epitopes that are shared with other coronaviruses, some studies have shown critical differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct cytotoxic effects of viral replication. To evaluate this in other cell lines, including human airway epithelial cells, is crucial, as the infectivity of coronavirus strains greatly varies in different cell lines,</p>
</div>
<div id="significance-49" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-49" class="anchor"></a>Significance</h4>
<p>Nevertheless, these findings encourage evaluating ciclesonide and mometasone as better options for patients with COVID-19 in need of inhaled steroids, especially as an alternative to other corticosteroids that have been shown to increase viral replication in vitro. This should be evaluated in future clinical studies.</p>
</div>
<div id="credit-54" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-54" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#a-human-monoclonal-antibody-blocking-sars-cov-2-infection" class="anchor"></a>A human monoclonal antibody blocking SARS-CoV-2 infection **</h3>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.11.987958]</span></p>
<div id="keywords-55" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-55" class="anchor"></a>Keywords</h4>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
</div>
<div id="main-findings-52" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-52" class="anchor"></a>Main Findings</h4>
<p>The authors reported a human monoclonal antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same family of corona viruses. For identifying mAbs, supernatants of a collection of 51 hybridomas raised against the spike protein of SARS-CoV (SARS-S) were screened by ELISA for cross-reactivity against the spike protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S, one of which (47D11) exhibited cross-neutralizing activity for SARS-S and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1 isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml respectively. 47D11 mAb bound a conserved epitope on the spike receptor binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and SARS2-S with similar affinities. Interestingly, binding of 47D11 to SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2 receptor-expressing cells assayed by flow cytometry.</p>
</div>
<div id="limitations-53" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-53" class="anchor"></a>Limitations</h4>
<p>These results show that the human 47D11 antibody neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown mechanism that is different from receptor binding interference. Alternative mechanisms were proposed but these as yet remain to be tested in the context of SARS-CoV2. From a therapeutic standpoint and in the absence of in vivo data, it is unclear whether the 47D11 ab can alter the course of infection in an infected host through virus clearance or protect an uninfected host that is exposed to the virus. There is a precedent for the latter possibility as it relates to SARS-CoV that was cited by the authors and could turn out to be true for SARS-CoV2.</p>
</div>
<div id="significance-50" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-50" class="anchor"></a>Significance</h4>
<p>This study enabled the identification of novel neutralizing antibody against COV-that could potentially be used as first line of treatment in the near future to reduce the viral load and adverse effects in infected patients. In addition, neutralizing antibodies such as 47D11 represent promising reagents for developing antigen-antibody-based detection test kits and assays.</p>
</div>
<div id="credit-55" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-55" class="anchor"></a>Credit</h4>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>A human monoclonal antibody blocking SARS-CoV-2 infection </strong></p>
<p>Wang et al</p>
<p><strong>doi:</strong> <a href="https://doi.org/10.1101/2020.03.11.987958" class="uri">https://doi.org/10.1101/2020.03.11.987958</a></p>
<p>Quick Link <a href="https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1" class="uri">https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1</a></p>
<p>Keywords: Monoclonal antibodies, SARS-CoV2, cross-neutralization, potential treatment, spike receptor</p>
<p><em>Summary and key findings:</em> The authors reported a human monoclonal antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same family of corona viruses. For identifying mAbs, supernatants of a collection of 51 hybridomas raised against the spike protein of SARS-CoV (SARS-S) were screened by ELISA for cross-reactivity against the spike protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S, one of which (47D11) exhibited cross-neutralizing activity for SARS-S and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1 isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml respectively. 47D11 mAb bound a conserved epitope on the spike receptor binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and SARS2-S with similar affinities. Interestingly, binding of 47D11 to SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2 receptor-expressing cells assayed by flow cytometry.</p>
<p><em>Limitations:</em> These results show that the human 47D11 antibody neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown mechanism that is different from receptor binding interference. Alternative mechanisms were proposed but these as yet remain to be tested in the context of SARS-CoV2. From a therapeutic standpoint and in the absence of in vivo data, it is unclear whether the 47D11 ab can alter the course of infection in an infected host through virus clearance or protect an uninfected host that is exposed to the virus. There is a precedent for the latter possibility as it relates to SARS-CoV that was cited by the authors and could turn out to be true for SARS-CoV2.</p>
<p><em>Relevance:</em> This study enabled the identification of novel neutralizing antibody against COV-that could potentially be used as first line of treatment in the near future to reduce the viral load and adverse effects in infected patients. In addition, neutralizing antibodies such as 47D11 represent promising reagents for developing antigen-antibody-based detection test kits and assays.</p>
<p>Edited by K. Alexandropoulos</p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies, IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples collected from suspected COVID-19 patients reduces the sensitivity of a fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an important safety precaution in laboratory manipulation of clinical samples. However, the effect of this step on downstream SARS-CoV-2-specific serum antibody assays has not been examined. The authors tested the effect of heat inactivation (56 deg C for 30 minutes) versus no heat inactivation on a fluorescence immunochromatography assay. Heat inactivation reduced all IgM measurements by an average of 54% and most IgG measurements (22/36 samples, average reduction of 50%), consistent with the lower thermal stability of IgM than that of IgG. Heat inactivation caused a subset of IgM but not IgG readings to fall below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat inactivated vs non-inactivated sera, and the use of the same baseline for heat inactivated and non-inactivated sera. The results indicate that heat inactivation affects the quantification of SARS-CoV-2-antibody response, specially IgM, but still allows to distinguish positive specific IgG. Therefore, the effect of heat inactivation should be studied when designing assays that quantitatively associate immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19" class="anchor"></a>Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h3>
<p>Zhang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.12.20035048]</span></p>
<div id="keywords-56" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-56" class="anchor"></a>Keywords</h4>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
</div>
<div id="main-findings-53" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-53" class="anchor"></a>Main Findings</h4>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were analyzed during acute and convalescent phases (up to day 35) and correlated to the diseases’ severity. The same was done with neutrophil-to-lymphocyte ratio. High IgG levels and high neutrophil-to-lymphocyte ratio in convalescence were both independently associated to the severity of the disease. The simultaneous occurrence of both of these laboratory findings correlated even stronger to the diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly higher levels of IL-6. The authors propose that a robust IgG response leads to immune-mediated tissue damage, thus explaining the worse outcome in patients with overexuberant antibody response.</p>
</div>
<div id="limitations-54" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-54" class="anchor"></a>Limitations</h4>
<p>A main criticism is that the criteria for stratifying patients in severe vs. non-severe are not described. The only reference related to this is the difference between the percentage of patients who needed mechanical ventilation, which was greater in patients with both high IgG levels and high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels and low neutrophil-to-lymphocyte ratio was treated with mechanical ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of the most interesting findings is that an early increase in anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease. However, as only median values are shown for antibody kinetics curves, the extent of variation in acute phase cannot be assessed.</p>
</div>
<div id="significance-51" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-51" class="anchor"></a>Significance</h4>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio predict severity of COVID-19 independently of each other. An additive predictive value of both variables is noticeable. Importantly, an early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict outcome.</p>
</div>
<div id="credit-56" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-56" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques" class="section level3">
<h3 class="hasAnchor">
<a href="#reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques" class="anchor"></a>Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h3>
<p>Bao et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.13.990226]</span></p>
<div id="keywords-57" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-57" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
</div>
<div id="main-findings-54" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-54" class="anchor"></a>Main Findings</h4>
<p>This study addresses the issue or acquired immunity after a primary COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were intratracheally infected with SARS-CoV-2 and two out of the four were re-infected at 28 days post initial infection (dpi) with the same viral dose after confirming the recovery by the absence of clinical symptoms, radiological abnormalities and viral detection (2 negative RT-PCR tests). While the initial infection led the viral loads in nasal and pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3 dpi in all four monkeys, viral loads in the swabs tested negative after reinfection in the two reinfected monkeys. In addition, the necropsies from a monkey (M1) at 7 days after primary infection, and another monkey (M3) at 5 days post reinfection, revealed the histopathological damages and viral replication in the examined tissues from M1, while no viral replication as well as no histological damages were detected in the tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting the production of protective neutralizing antibodies in these monkeys. Overall, this study indicates that primary infection with SARS-CoV-2 may protect from subsequent exposure to the same virus.</p>
</div>
<div id="limitations-55" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-55" class="anchor"></a>Limitations</h4>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15 days after the onset of symptoms. In the infected monkeys in this study, virus were detected from day 1 after the infection, declining to undetectable level by day 15 post infection. It may suggest that there is a faster viral clearance mechanism in monkeys, therefore the conclusions of reinfection protection for humans need to be carefully considered. In addition, only two monkeys were re-infected in this study and the clinical signs of these monkeys were not similar: M3 did not show weight loss and M4 showed relatively higher fever on the day of infection and the day of re-challenge.</p>
</div>
<div id="significance-52" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-52" class="anchor"></a>Significance</h4>
<p>This study showed clear viral clearance and no indications of relapse or viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus macaque model. These results support the idea that patients with full recovery (two negative RT-PCR results) may also be protected from secondary SARS-CoV-2 infection. Recovered patients may be able to reintegrate to normal public life and provide protective serum perhaps even if having had a mild infection. The results are also encouraging for successful vaccine development against SARS-CoV-2.</p>
</div>
<div id="credit-57" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-57" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov" class="section level3">
<h3 class="hasAnchor">
<a href="#a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov" class="anchor"></a>A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h3>
<p><span class="citation">[@doi:10.1101/2020.03.13.991570]</span></p>
<div id="keywords-58" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-58" class="anchor"></a>Keywords</h4>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
</div>
<div id="main-findings-55" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-55" class="anchor"></a>Main Findings</h4>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing antibodies isolated from convalescent SARS-CoV patients may offer insight into cross-reactive antibodies targeting SARS-CoV-2. In particular, they find that the receptor-binding domain (RBD) of SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of SARS-CoV S protein that binds to the CR3022 neutralizing antibody. CR3022 also displays increased affinity for the “up” conformation of the SARS-CoV-2 S protein compared to the “down” conformation as it does for the SARS-CoV S protein. Therefore, the authors propose that this cross-reactive antibody may confer some degree of protection <em>in vivo</em> even if it fails to neutralize <em>in vitro</em>.</p>
</div>
<div id="limitations-56" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-56" class="anchor"></a>Limitations</h4>
<p>Although the authors offer a logical rationale for identifying cross-reactive neutralizing antibodies derived from SARS-CoV, their study using only CR3022 failed to demonstrate whether this approach will be successful. After all, CR3022 failed to neutralize <em>in vitro</em> despite the binding affinity to a similar epitope on SARS-CoV-2. They would benefit from testing more candidates and using an <em>in vivo</em> model to demonstrate their claim that protection may be possible in the absence neutralization if combinations are used <em>in vivo.</em></p>
</div>
<div id="significance-53" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-53" class="anchor"></a>Significance</h4>
<p>The ability to make use of previously characterized neutralizing antibodies for conserved epitopes can expedite drug design and treatment options.</p>
</div>
<div id="credit-58" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-58" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr" class="section level3">
<h3 class="hasAnchor">
<a href="#highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr" class="anchor"></a>Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h3>
<p>Dong et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.14.20036129]</span></p>
<div id="keywords-59" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-59" class="anchor"></a>Keywords</h4>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
</div>
<div id="main-findings-56" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-56" class="anchor"></a>Main Findings</h4>
<p>The authors present a digital PCR (dPCR) diagnostic test for SARS-CoV-2 infection. In 103 individuals that were confirmed in a follow-up to be infected, the standard qPCR test had a positivity rate of 28.2% while the dPCR test detected 87.4% of the infections by detecting an additional 61 positive cases. The authors also tested samples from close contacts (early in infection stage) and convalescing individuals (late in infection stage) and were able to detect SARS-CoV-2 nucleic acid in many more samples using dPCR compared to qPCR.</p>
</div>
<div id="limitations-57" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-57" class="anchor"></a>Limitations</h4>
<p>I did not detect limitations.</p>
</div>
<div id="significance-54" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-54" class="anchor"></a>Significance</h4>
<p>The authors make a strong case for the need for a highly sensitive and accurate confirmatory method for diagnosing COVID-19 during this outbreak and present a potential addition to the diagnostic arsenal. They propose a dPCR test that they present has a dramatically lower false negative rate than the standard RT-qPCR tests and can be especially beneficial in people with low viral load, whether they are in the earlier or later stages of infection.</p>
</div>
<div id="credit-59" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-59" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein" class="anchor"></a>SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h3>
<p>Wang et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.14.988345]</span></p>
<div id="keywords-60" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-60" class="anchor"></a>Keywords</h4>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
</div>
<div id="main-findings-57" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-57" class="anchor"></a>Main Findings</h4>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through the interaction of the viral spike protein (SP) and the immunoglobulin superfamily protein CD147 (also known as Basigin). Using an in-house developed humanized antibody against CD147 (maplazumab), they show that blocking CD147 decreases viral replication in Vero E6 cells. Using surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly, they utilize immune-election microscopy to show spike protein and CD147 localize to viral inclusion bodies of Vero E6 cells.</p>
</div>
<div id="limitations-58" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-58" class="anchor"></a>Limitations</h4>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits SARS-CoV-2 replication by testing cell growth and viral load in cells infected with SARS-CoV-2, however there are key pieces of this experiment that are missing. First, the authors fail to use a non-specific antibody control. Second, the authors claim that viral replication is inhibited, and that they test this by qPCR, however this data is <strong>not shown.</strong> To further prove specificity, the authors should introduce CD147 to non-susceptible cells and show that they become permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and SP through SPR, ELISA, and Co-IP, and this data seems generally convincing. The electron microscopy provides further correlative evidence that SARS-CoV-2 may interact with CD147 as they are both found in the same viral inclusion body. A quantification of this data would make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and statistics to show that the data are reproducible and statistically significant.</p>
</div>
<div id="significance-55" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-55" class="anchor"></a>Significance</h4>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell surface protein ACE2 for cell entry, yet ACE2 is highly expressed in heart, kidney, and intestinal cells, raising the concern that blocking ACE2 would result in harmful side effects <span class="citation">[@doi:10.1093/ndt/gfu302]</span> CD147 on the other hand is highly expressed in various tumor types, inflamed tissues, and pathogen infected cells, suggesting that the inhibition of CD147 would not result in major side effects <span class="citation">[@doi:10.1016/j.yexmp.2018.05.001; @pmid:20551248]</span> The research in this paper has resulted in an ongoing clinical trail in China to test the safety and efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov identifier NCT04275245).</p>
</div>
<div id="credit-60" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-60" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19" class="anchor"></a>Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h3>
<p>Huang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.15.20033472]</span></p>
<div id="keywords-61" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-61" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
</div>
<div id="main-findings-58" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-58" class="anchor"></a>Main Findings</h4>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of peripheral blood mononuclear cells isolated from whole blood samples of COVID-19 patients (n=10). Data was compared to scRNAseq of samples collected from patients with influenza A (n=1), acute pharyngitis (n=1), and cerebral infarction (n=1), as well as, three healthy controls. COVID-19 patients were categorized into those with moderate (n=6), severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across all COVID-19 disease levels revealed 56 different cellular subtypes, among 17 immune cell types; comparisons between each category to the normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19 patients have circulating antibodies against known viral antigens, including EBV, HIV, influenza A, and other RNA viruses. This may suggest that the immune response to SARS-CoV-2 infection elicits production of antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients with other conditions (influenza A, acute pharyngitis, and cerebral infarction), the authors identified the <strong>interferon-MAPK signaling pathway as a major response to SARS-CoV-2 infection</strong>. The authors performed quantitative real-time reverse transcriptase polymerase chain reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em> and <em>FOS</em>. These samples were collected from a separate cohort of COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild, n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19 patients and down-regulation in cured patients. <strong>The authors propose that <em>FOS</em> may be a candidate marker gene for curative COVID-19 disease</strong>.</p>
</div>
<div id="limitations-59" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-59" class="anchor"></a>Limitations</h4>
<p>The sample size of this study is limited. To further delineate differences in the immune profile of peripheral blood of COVID-19 patients, a greater sample size is needed, and longitudinal samples are needed, as well. A better understanding of the immunological interactions in cured patients, for example, would require a profile before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to potential autoimmunity and immune deficiency to distinguish different severities of COVID-19 disease. However, this requires an expanded number of samples and a more robust organization of specific immune cell subtypes that can be compared across different patients. Importantly, this criterion is likely needed to ensure greater specificity in identifying markers for COVID-19 infection and subsequent immune response.</p>
</div>
<div id="significance-56" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-56" class="anchor"></a>Significance</h4>
<p>At the single-cell level, COVID-19 disease has been characterized in the lung, but a greater understanding of systemic immunological responses is furthered in this study. Type I interferon is an important signaling molecule for the anti-viral response. The identification of the interferon-MAPK signaling pathway and the differential expression of MAPK regulators between patients of differing COVID-19 severity and compared to cured patients may underscore the importance of either immune deficiency or autoimmunity in COVID-19 disease.</p>
</div>
<div id="credit-61" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-61" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection-" class="section level3">
<h3 class="hasAnchor">
<a href="#cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection-" class="anchor"></a>Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h3>
<p>Lv et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.15.993097]</span></p>
<div id="keywords-62" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-62" class="anchor"></a>Keywords</h4>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity, cross-neutralization, antibody, human patients, mouse</p>
</div>
<div id="main-findings-59" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-59" class="anchor"></a>Main Findings</h4>
<p>The authors explore the antigenic differences between SARS-CoV-2 and SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the spike protein between SARS-CoV-2 and SARS-CoV was found to be common, mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients. Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV, with low neutralization activity. No cross-neutralization response was detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of plasma collected from mice infected or immunized with SARS-CoV-2 or SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2 S ectodomain. Cross-neutralization activity was not detected in any of the mouse plasma samples.</p>
</div>
<div id="limitations-60" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-60" class="anchor"></a>Limitations</h4>
<p>The size of each patient cohort is insufficient to accurately determine the frequency of cross-reactivity and cross-neutralization in the current SARS-CoV-2 pandemic. Recruitment of additional patients from a larger range of geographical regions and time points would also enable exploration into the effect of the genetic diversity and evolution of the SARS-CoV-2 virus on cross-reactivity. This work would also benefit from the mapping of specific epitopes for each sample. Future studies may determine whether the non-neutralizing antibody responses can confer <em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
</div>
<div id="significance-57" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-57" class="anchor"></a>Significance</h4>
<p>The cross-reactive antibody responses to S protein in the majority of SARS-CoV-2 patients is an important consideration for development of serological assays and vaccine development during the current outbreak. The limited extent of cross-neutralization demonstrated in this study indicates that vaccinating to cross-reactive conserved epitopes may have limited efficacy, presenting a key concern for the development of a more universal coronavirus vaccine to address the global health risk of novel coronavirus outbreaks.</p>
</div>
<div id="credit-62" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-62" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study" class="section level3">
<h3 class="hasAnchor">
<a href="#the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study" class="anchor"></a>The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h3>
<p>Duan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.16.20036145]</span></p>
<div id="keywords-63" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-63" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
</div>
<div id="main-findings-60" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-60" class="anchor"></a>Main Findings</h4>
<p>This is the first report to date of convalescent plasma therapy as a therapeutic against COVID-19 disease. This is a feasibility pilot study. The authors report the administration and clinical benefit of 200 mL of convalescent plasma (CP) (1:640 titer) derived from recently cured donors (CP selected among 40 donors based on high neutralizing titer and ABO compatibility) to 10 severe COVID-19 patients with confirmed viremia. The primary endpoint was the safety of CP transfusion. The secondary endpoint were clinical signs of improvement based on symptoms and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate any potential residual virus in the plasma samples, without compromising neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete disappearance of clinical symptoms, including fever, cough, shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in neutralizing antibody titers in 5 patients. Persistence of neutralizing antibody levels in 4 patients.</p></li>
</ul>
</div>
<div id="limitations-61" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-61" class="anchor"></a>Limitations</h4>
<p>It is important to note that most recipients had high neutralization titers of antibodies before plasma transfusion and even without transfusion it would be expected to see an increase in neutralizing antibodies over time. In addition to the small sample set number (n=10), there are additional limitations to this pilot study:</p>
<ol style="list-style-type: decimal">
<li><p>All patients received concurrent therapy, in addition to the CP transfusion. Therefore, it is unclear whether a combinatorial or synergistic effect between these standards of care and CP transfusion contributed to the clearance of viremia and improvement of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect to the administration of CP transfusion. So, the definitive impact of CP transfusion on immune dynamics and subsequent viral load is not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
</div>
<div id="significance-58" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-58" class="anchor"></a>Significance</h4>
<p>For the first time, a pilot study provides promising results involving the use of convalescent plasma from cured COVID-19 patients to treat others with more severe disease. The authors report that the administration of a single, high-dose of neutralizing antibodies is safe. In addition, there were encouraging results with regards to the reduction of viral load and improvement of clinical outcomes. It is, therefore, necessary to expand this type of study with more participants, in order to determine optimal dose and treatment kinetics. It is important to note that CP has been studied to treat H1N1 influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to be effective in treating these infections.</p>
</div>
<div id="credit-63" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-63" class="anchor"></a>Credit</h4>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial" class="anchor"></a>Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.20037135]</span></p>
<div id="keywords-64" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-64" class="anchor"></a>Keywords</h4>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
</div>
<div id="main-findings-61" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-61" class="anchor"></a>Main Findings</h4>
<p>This study was a single-arm, open label clinical trial with 600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients who refused participation or patients from another center not treated with HCQ were included as negative controls (n = 16). Among the patients in the treatment arm, 6 received concomitant azithromycin to prevent superimposed bacterial infection. The primary endpoint was respiratory viral loads on day 6 post enrollment, measured by nasopharyngeal swab followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n = 8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to be synergistic with HCQ, as 6/6 patients receiving combined treatment had complete viral clearance (p &lt; 0.001).</p>
</div>
<div id="limitations-62" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-62" class="anchor"></a>Limitations</h4>
<p>Despite what is outlined above, this study has a number of limitations that must be considered. First, there were originally n = 26 patients in the treatment arm, with 6 lost to follow up for the following reasons: 3 transferred to ICU, 1 discharge, 1 self-discontinued treatment d/t side effects, and 1 patient expired. Total length of clinical follow up was 14 days, but the data beyond day 6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR are listed for the control and treatment arms from D0 – D6. However, the data are not reported in a standard way, with a mix of broadly positive or negative result delineation with Ct (cycle threshold) values, the standard output of real time PCR. It is impossible to compare what is defined as a positive value between the patients in the control and treatment arms without a standardized threshold for a positive test. Further, the starting viral loads reported at D0 in the groups receiving HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs 26.8 respectively), which could explain in part the differences observed in the response to treatment between 2 groups. Finally, patients in the control arm from outside the primary medical center in this study (Marseille) did not actually have samples tested by PCR daily. Instead, positive test results from every other day were extrapolated to mean positive results on the day before and after testing as well (Table 2, footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a promising treatment avenue for COVID-19 patients. However, the limited size of the trial, and the way in which the results were reported does not allow for other medical centers to extrapolate a positive or negative result in the treatment of their own patients with HCQ +/- azithromycin. Further larger randomized clinical trials will be required to ascertain the efficacy of HCQ +/- azithromycin in the treatment of COVID-19.</p>
</div>
<div id="significance-59" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-59" class="anchor"></a>Significance</h4>
<p>Chloroquine is thought to inhibit viral infection, including SARS-Cov-2, by increasing pH within endosomes and lysosomes, altering the biochemical conditions required for viral fusion <span class="citation">[@doi:10.1038/s41422-020-0282-0; @doi:10.1016/j.imlet.2013.07.004]</span>. However, chloroquine also has immuno-modulatory effects that I think may play a role. Chloroquine has been shown to increase CTLA-4 expression at the cell surface by decreasing its degradation in the endo-lysosome pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but in conditions of increased pH, the protein machinery required for degradation is less functional <span class="citation">[@doi:10.1126/science.aaa1663]</span>. As such, more CTLA-4 remains in endosomes and is trafficked back to the cell surface. It is possible that this may also contribute to patient recovery via reduction of cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
</div>
<div id="credit-64" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-64" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids" class="section level3">
<h3 class="hasAnchor">
<a href="#recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids" class="anchor"></a>Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.990317]</span></p>
<div id="keywords-65" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-65" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
</div>
<div id="main-findings-62" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-62" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes in healthy liver (2.92% of all cells) are infectable and support SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid barrier and bile transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation (CLDN1) and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
</div>
<div id="limitations-63" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-63" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct cholangiocyte infection or due to secondary immune/cytokine effects. This study argues for direct damage as it lacks immune contexture; but further studies needed with autopsy samples or organoid-immune cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral response and alarmins secreted upon infection, and furthermore study tropism of various immune cells to conditioned media from organoids infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or alcohol/smoking/obesity-associated liver organoids respond to SARS-CoV-2 infectivity and replication, worth pursuing to experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
</div>
<div id="significance-60" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-60" class="anchor"></a>Significance</h4>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2 infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the results from this study could explain the bile acid accumulation and consequent liver damage observed.</p></li>
</ul>
</div>
<div id="credit-65" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-65" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2" class="section level3">
<h3 class="hasAnchor">
<a href="#the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2" class="anchor"></a>The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.994236]</span></p>
<div id="keywords-66" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-66" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
</div>
<div id="main-findings-63" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-63" class="anchor"></a>Main Findings</h4>
<p>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infects cells through S spike glycoprotein binding angiotensin-converting enzyme (ACE2) on host cells. S protein can bind both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a decoy that neutralizes infection. Recombinant sACE2 is now being tested in clinical trials for COVID-19. To determine if a therapeutic sACE2 with higher affinity for S protein could be designed, authors generated a library containing every amino acid substitution possible at the 117 sites spanning the binding interface with S protein. The ACE2 library was expressed in human Expi293F cells and cells were incubated with medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused to GFP. Cells with high or low affinity mutant ACE2 receptor compared to affinity of wild type ACE2 for the RBD were FACS sorted and transcripts from these sorted populations were deep sequenced. Deep mutagenesis identified numerous mutations in ACE2 that enhance RBD binding. This work serves to identify putative high affinity ACE2 therapeutics for the treatment of CoV-2.</p>
</div>
<div id="limitations-64" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-64" class="anchor"></a>Limitations</h4>
<p>The authors generated a large library of mutated ACE2, expressed them in human Expi293F cells, and performed deep mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S protein. While these data serve as a useful resource, the ability of the high affinity ACE2 mutants identified to serve as therapeutics needs further validation in terms of conformational stability when purified as well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally, authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit beneficial host immune responses, which would need further design and validation.</p>
</div>
<div id="significance-61" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-61" class="anchor"></a>Significance</h4>
<p>This study identified structural ACE2 mutants that have potential to serve as therapeutics in the treatment of SARS-CoV-2 upon further testing and validation.</p>
</div>
<div id="credit-66" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-66" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA - seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1 SARS-CoV-2 isolate. The S protein was modified to allow trimerization and increase stability. The authors compared the antibody reactivity of 59 banked human serum samples (non-exposed) and 3 serum samples from confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19 patients, and observed stronger IgG3 response than IgG1. IgM and IgA responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and samples from only three different patients to test for reactivity against the RBD domain and full-length spike protein. It will be important to follow up with a larger number of patient samples to confirm the data obtained. Furthermore, it would be interesting to assess people at different age groups and determine whether unexposed control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are limited, since antibody response should start to be detectable only one to two weeks after infection. Future studies will be required to assess how long after infection this assay allow to detect anti-CoV2 antibodies. Finally, while likely, the association of seroconversion with protective immunity against SARS-Cov-2 infection still needs to be fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2. First, it is now possible to perform serosurveys and determine who has been infected, allowing a more accurate estimate of infection prevalence and death rate. Second, if it is confirmed that re-infection does not happen (or is rare), this assay can be used as a tool to screen healthcare workers and prioritize immune ones to work with infected patients. Third, potential convalescent plasma donors can now be screened to help treating currently infected patients. Of note, this assay does not involve live virus handling. experimentally, this is an advantage as the assay does not require the precautions required by manipulation of live virus. Finally, the recombinant proteins described in this study represent new tools that can be used for further applications, including vaccine development.</p>
</div>
</div>
<div id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model" class="section level3">
<h3 class="hasAnchor">
<a href="#comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model" class="anchor"></a>COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h3>
<p><span class="citation">[@doi:10.1101/2020.03.17.995639]</span></p>
<div id="keywords-67" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-67" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
</div>
<div id="main-findings-64" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-64" class="anchor"></a>Main Findings</h4>
<p>This work assesses SARS-CoV-2 infection in young adult and aged cynomolgus macaques. 4 macaques per age group were infected with low-passage clinical sample of SARS-CoV-2 by intranasal and intratracheal administration. Viral presence was assessed in nose, throat and rectum through RT-PCR and viral culture. SARS-CoV-2 replication was confirmed in the respiratory track (including nasal samples), and it was also detected in ileum. Viral nucleocapsid detection by IHC showed infection of type I and II pneumocytes and epithelia. Virus was found to peak between 2 and 4 days after administration and reached higher levels in aged vs. young animals. The early peak is consistent with data in patients and contrasts to SARS-CoV replication. SARS-CoV-2 reached levels below detection between 8 and 21 days after inoculation and macaques established antibody immunity against the virus by day 14. There were histopathological alteration in lung, but no overt clinical signs. At day 4 post inoculation of SARS-CoV-2, two of four animals presented foci of pulmonary consolidation, with limited areas of alveolar edema and pneumonia, as well as immune cell infiltration. In sum, cynomolgus macaques are permissive to SARS- CoV-2 and develop lung pathology (less severe than SARC-CoV, but more severe than MERS-CoV).</p>
</div>
<div id="limitations-65" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-65" class="anchor"></a>Limitations</h4>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2 replication, it is unclear if the viral load reaches levels comparable to humans and there wasn’t overt clinical pathology.</p>
</div>
<div id="significance-62" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-62" class="anchor"></a>Significance</h4>
<p>The development of platforms in which to carry out relevant experimentation on SARS-CoV-2 pathophysiology is of great urgency. Cynomolgus macaques offer an environment in which viral replication can happen, albeit in a limited way and without translating into clinically relevant symptoms. Other groups are contributing to SARS-CoV2 literature using this animal model <span class="citation">[@doi:10.1101/2020.03.13.990226]</span>, potentially showing protection against reinfection in cured macaques. Therefore, this platform could be used to examine SARS-CoV2 pathophysiology while studies in other animal models are also underway <span class="citation">[@doi:10.1128/JVI.02012-06; @doi:10.1101/2020.02.07.939389]</span>.</p>
</div>
<div id="credit-67" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-67" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission" class="section level3">
<h3 class="hasAnchor">
<a href="#investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission" class="anchor"></a>Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20037994]</span></p>
<div id="keywords-68" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-68" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
</div>
<div id="main-findings-65" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-65" class="anchor"></a>Main Findings</h4>
<p>Multiple studies reported the same level of infectiousness between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that asymptomatic and undocumented carriers escape public health surveillance systems, a better mathematical model of transmission is needed to determine a more accurate estimate of the basic reproductive number (R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors developed a SEYAR dynamical model for transmission of the new coronavirus that takes into account asymptomatic and undocumented carriers. The model was validated using data reported from thirteen countries during the first three weeks of community transmission. While current studies estimate R<sub>0</sub> to be around 3, this model indicates that the value could range between 5.5 to 25.4.</p>
</div>
<div id="limitations-66" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-66" class="anchor"></a>Limitations</h4>
<p>The SEYAR model realistically depicts transmission of the virus only during the initial stages of the disease. More data is necessary to better fit the model with current trends. In addition, multiple factors (e.g. behavioral patterns, surveillance capabilities, environmental and socioeconomic factors) affect transmission of the virus and so, these factors must be taken into consideration when estimating the R<sub>0</sub>.</p>
</div>
<div id="significance-63" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-63" class="anchor"></a>Significance</h4>
<p>Public health authorities use the basic reproductive number to determine the severity of disease. An accurate estimate of R<sub>0</sub> will inform intervention strategies. This model can be applied to different locations to assess the potential impact of COVID-19.</p>
</div>
<div id="credit-68" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-68" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice" class="section level3">
<h3 class="hasAnchor">
<a href="#antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice" class="anchor"></a>Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h3>
<p>Long et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.18.20038018]</span></p>
<div id="keywords-69" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-69" class="anchor"></a>Keywords</h4>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
</div>
<div id="main-findings-66" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-66" class="anchor"></a>Main Findings</h4>
<p>This study investigated the profile of the acute antibody response against SARS-CoV-2 and provided proposals for serologic tests in clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was used to evaluate IgM and IgG seroconversion in 285 hospital admitted patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19 suspected patients that tested negative by RT-PCR. A follow up study with 63 patients was performed to investigate longitudinal effects. In addition, IgG and IgM titers were evaluated in a cohort of close contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after symptom onset. Patients varied in the order of IgM/ IgG seroconversion and there was no apparent correlation of order with age, severity, or hospitalization time. This led the authors to conclude that for diagnosis IgM and IgG should be detected simultaneously at the early phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared to non-severe patients after controlling for days post-symptom onset. Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet the WHO serological diagnosis criterion of either seroconversion or <em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests the current serological criteria may be too stringent for COVID-19 diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test) had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close contacts” who had negative RT-PCR tests were positive for IgG and/or IgM. This highlights the usefulness of a serological assay to identify asymptomatic infections and/or infections that are missed by RT-PCR.</p>
</div>
<div id="limitations-67" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-67" class="anchor"></a>Limitations</h4>
<p>This group’s report generally confirms the findings of others that have evaluated the acute antibody response to SARS-Cov-2. However, these data would benefit from inclusion of data on whether the participants had a documented history of viral infection. Moreover, serum samples that were collected prior to SARS-Cov-2 outbreak from patients with other viral infections would serve as a useful negative control for their assay. Methodological limitations include that only one serum sample per case was tested as well as the heat inactivation of serum samples prior to testing. It has previously been reported that heat inactivation interferes with the level of antibodies to SARS-Cov-2 and their protocol may have resulted in diminished quantification of IgM, specifically <span class="citation">[@doi:10.1101/2020.03.12.20034231]</span>.</p>
</div>
<div id="significance-64" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-64" class="anchor"></a>Significance</h4>
<p>Understanding the features of the antibody responses against SARS-CoV is useful in the development of a serological test for the diagnosis of COVID-19. This paper addresses the need for additional screening methods that can detect the presence of infection despite lower viral titers. Detecting the production of antibodies, especially IgM, which are produced rapidly after infection can be combined with PCR to enhance detection sensitivity and accuracy and map the full spread of infection in communities, Moreover, serologic assays would be useful to screen health care workers in order to identify those with immunity to care for patients with COVID19.</p>
</div>
<div id="credit-69" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-69" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-specific-antibody-responses-in-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-specific-antibody-responses-in-covid-19-patients" class="anchor"></a>SARS-CoV-2 specific antibody responses in COVID-19 patients</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20038059]</span></p>
<div id="keywords-70" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-70" class="anchor"></a>Keywords</h4>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
</div>
<div id="main-findings-67" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-67" class="anchor"></a>Main findings</h4>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
</div>
<div id="limitations-68" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-68" class="anchor"></a>Limitations</h4>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
</div>
<div id="significance-65" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-65" class="anchor"></a>Significance</h4>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
</div>
</div>
<div id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19" class="anchor"></a>A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h3>
<p>Belhadi et al. <span class="citation">[@doi:10.1101/2020.03.18.20038190]</span></p>
<div id="keywords-71" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-71" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
</div>
<div id="main-findings-68" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-68" class="anchor"></a>Main Findings</h4>
<p>Summary of clinical trials registered as of March7, 2020 from U.S, Chinese, Korean, Iranian and European registries. Out of the 353 studies identified, 115 were selected for data extraction. 80% of the trials were randomized with parallel assignment and the median number of planned inclusions was 63 (IRQ, 36-120). Most frequent therapies in the trials included; 1) antiviral drugs <span class="math display">\[lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)\]</span>; 2) anti-malaria drugs <span class="math display">\[chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs \[methylprednisolone
(n=5)\]</span>; and stem cell therapies (n=23). Medians of the total number of planned inclusions per trial for these therapies were also included. Stem cells and lopunavir/ritonavir were the most frequently evaluated candidate therapies (23 and 15 trials respectively), whereas remdesivir was only tested in 5 trials but these trials had the highest median number of planned inclusions per trial (400, IQR 394-453). Most of the agents used in the different trials were chosen based on preclinical assessments of antiviral activity against SARS CoV and MERS Cov corona viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological (23%); radiological (8%); or immunological (3%). The trials were classified as those that included patients with severe disease only; trials that included patients with moderate disease; and trials that included patients with severe or moderate disease.</p>
</div>
<div id="limitations-69" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-69" class="anchor"></a>Limitations</h4>
<p>The trials evaluated provided incomplete information: 23% of these were phase IV trials but the bulk of the trials (54%) did not describe the phase of the study. Only 52% of the trials (n=60) reported treatment dose and only 34% (n=39) reported the duration. A lot of the trials included a small number of patients and the trials are still ongoing, therefore no insight was provided on the outcome of the trials.</p>
</div>
<div id="significance-66" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-66" class="anchor"></a>Significance</h4>
<p>Nonetheless, this review serves as framework for identifying COVID-19 related trials, which can be expanded upon as new trials begin at an accelerated rate as the disease spreads around the world.</p>
</div>
<div id="credit-70" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-70" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19" class="anchor"></a>ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h3>
<p>Leung et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.18.20038455]</span></p>
<div id="keywords-72" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-72" class="anchor"></a>Keywords</h4>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
</div>
<div id="main-findings-69" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-69" class="anchor"></a>Main Findings</h4>
<p>In bronchial epithelial samples from 3 different cohorts of individuals, ACE-2 gene expression was found to be significantly increased in both COPD patients and smokers relative to healthy controls. Across all test subjects, ACE-2 gene expression was also highly correlated with decreased forced expiratory volume in 1 second (FEV1), which may explain the increased COVID-19 disease severity in COPD patients. Former smokers were also found to show decreased ACE2 expression relative to current smokers and had no significant difference when compared to non-smokers.</p>
</div>
<div id="limitations-70" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-70" class="anchor"></a>Limitations</h4>
<p>While the upregulation of ACE-2 is an interesting hypothesis for COVID-19 disease severity in COPD patients, this study leaves many more unanswered questions than it addresses. Further studies are required to show whether the specific cell type isolated in these studies is relevant to the pathophysiology of COVID-19. Furthermore, there is no attempt to show whether that increased ACE-2 expression contributes to greater disease severity. Does the increased ACE-2 expression lead to greater infectivity with SARS-CoV-2? There is no mechanistic explanation for why ACE-2 levels are increased in COPD patients. The authors could also have considered the impact of co-morbidities and interventions such as corticosteroids or bronchodilators on ACE-2 expression. Finally, given the extensive sequencing performed, the authors could have conducted significantly more in-depth analyses into gene signature differences.</p>
</div>
<div id="significance-67" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-67" class="anchor"></a>Significance</h4>
<p>This study attempts to address an important clinical finding that both smokers and COPD patients show increased mortality from COVID-19. The novel finding that ACE-2 expression is induced in smokers and COPD patients suggests not only a mechanism for the clinical observation, but also highlights the potential benefit of smoking cessation in reducing the risk of severe COVID-19 disease.</p>
</div>
<div id="credit-71" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-71" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="dynamic-profile-of-severe-or-critical-covid-19-cases" class="section level3">
<h3 class="hasAnchor">
<a href="#dynamic-profile-of-severe-or-critical-covid-19-cases" class="anchor"></a>Dynamic profile of severe or critical COVID-19 cases</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20038513]</span></p>
<div id="keywords-73" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-73" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
</div>
<div id="main-findings-70" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-70" class="anchor"></a>Main Findings</h4>
<p>Authors evaluate clinical correlates of 10 patients (6 male and 4 female) hospitalized for severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and received broad spectrum antibiotics and 6 patients received anti-viral drugs. Additionally, 40% of patients were co-infected with influenza A. All 10 patients developed lymphopenia, two of which developed progressive lymphopenia (PLD) and died. Peripheral blood (PB) lymphocytes were analyzed – low CD4 and CD8 counts were noted in most patients, though CD4:CD8 ratio remained normal.</p>
</div>
<div id="limitations-71" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-71" class="anchor"></a>Limitations</h4>
<p>The authors evaluated a small cohort of severe SARS-CoV-2 cases and found an association between T cell lymphopenia and adverse outcomes. However, this is an extremely small and diverse cohort (40% of patients were co-infected with influenza A). These findings need to be validated in a larger cohort. Additionally, the value of this data would be greatly increased by adding individual data points for each patient as well as by adding error bars to each of the figures.</p>
</div>
<div id="significance-68" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-68" class="anchor"></a>Significance</h4>
<p>This study provides a collection of clinical data and tracks evolution of T lymphocyte in 10 patients hospitalized for SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
</div>
<div id="credit-72" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-72" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients" class="anchor"></a>Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h3>
<p>Fu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.19.20038315]</span></p>
<div id="keywords-74" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-74" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
</div>
<div id="study-description" class="section level4">
<h4 class="hasAnchor">
<a href="#study-description" class="anchor"></a>Study description</h4>
<p>Data analyzed from 52 COVID-19 patients admitted and then discharged with COVID-19. Clinical, laboratory, and radiological data were longitudinally recorded with illness timecourse (PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up positive test (PCR+) within two weeks of discharge.</p>
</div>
<div id="main-findings-71" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-71" class="anchor"></a>Main Findings</h4>
<ul>
<li>
<p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular enlargement (42.3%), and reticulation (11.5%) were most commonly observed.</p></li>
</ul>
</li>
<li>
<p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging w different extents of inflammatory exudation in lungs, with overall tendency for improvement (except 2/7 patients that were readmitted after discharge with re-positive test) after negative PCR.</p></li>
</ul>
</li>
<li>
<p>Seven patients repeated positive RT-PCR test and were readmitted to the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery and patients with lesion progression should be given more attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in lymphocytes in one patient), but was not correlated with new lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs. re-exposure.</p></li>
</ul>
</li>
</ul>
</div>
<div id="limitations-72" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-72" class="anchor"></a>Limitations</h4>
<p>Patients sampled in this study were generally younger (65.4% &lt; 50 yrs) and less critically ill/all discharged. Small number of recovered patients (N=18). Time of follow up was relatively short.. Limited clinical information available about patients with re-positive test (except CRP and lymph tracking).</p>
</div>
<div id="extended-results" class="section level4">
<h4 class="hasAnchor">
<a href="#extended-results" class="anchor"></a>Extended Results</h4>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4% &lt; 50 yrs) and less critically ill/all discharged. After two consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37 cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+ and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in lungs, with overall tendency for improvement (except 2/7 patients that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of segmental progression CT in 2 patients (2/18 or 11.1% - re-positive 9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in lymphocytes in one patient), but was not correlated with new lesions or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
</div>
<div id="significance-69" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-69" class="anchor"></a>Significance</h4>
<p>Study tracked key clinical features associated with disease progression, recovery, and determinants of clinical diagnosis/management of COVID-19 patients.</p>
</div>
<div id="credit-73" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-73" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus" class="anchor"></a>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h3>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.19.997890]</span></p>
<div id="keywords-75" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-75" class="anchor"></a>Keywords</h4>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
</div>
<div id="main-findings-72" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-72" class="anchor"></a>Main Findings</h4>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an orally bioavailable ribonucleoside with antiviral activity against various RNA viruses including Ebola, Influenza and CoV. NHC activity introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a dose dependent <em>manner</em> that directly correlated with a decrease in viral titers. Authors show that NHC inhibited multiple genetically distinct Bat-CoV viruses in human primary epithelial cells <em>without affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>. Prophylactic oral administration of NHC in C57BL/6 mice reduce lung titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic administration of NHC in C57BL/6 mice 12 hours post infected with SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage. The degree of clinical benefit was dependent on the time of treatment initiation post infection. The authors also demonstrate that NHC reduces MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic and therapeutic settings.</p>
</div>
<div id="limitations-73" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-73" class="anchor"></a>Limitations</h4>
<p>Most of the experiments were conducted using MERS-CoV, and SARS-CoV and a few experiments were conducted using other strains of CoV as opposed to SARS-CoV-2. The authors note the core residues that make up the RNA interaction sites (which constitutes the NHC interaction sites) are highly conserved among CoV and because of this conservation their understanding is that NHC can inhibit a broad-spectrum of CoV including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have adverse effects if mutations cause the virus to become drug resistant, more infectious or speed-up immune evasion. <em>In addition, the temporal diminishing effectiveness of NHC on clinical outcome when NHC was used therapeutically is concerning. However, the longer window (7-10 days) for clinical disease onset in human patients from the time of infection compared to that of mice (24-48 hours), may associate with increased NHC effectiveness in the clinic.</em></p>
</div>
<div id="significance-70" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-70" class="anchor"></a>Significance</h4>
<p>Prophylactic or therapeutic oral administration of NHC reduces lung titers and prevents acute lung failure in C57B\6 mice infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC could turn out to be a useful drug for treating current, emerging and future corona virus outbreaks.</em>   #### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs" class="section level3">
<h3 class="hasAnchor">
<a href="#identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs" class="anchor"></a>Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h3>
<p>Sangeun Jeon et al. <span class="citation">[@doi:10.1101/2020.03.20.999730]</span></p>
<div id="keywords-76" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-76" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
</div>
<div id="main-findings-73" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-73" class="anchor"></a>Main Findings</h4>
<p>A panel of ~3,000 FDA- and IND-approved antiviral drugs were previously screened for inhibitory efficacy against SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2 (79.5%) homology. 35 of these drugs along with another 15 (suggested by infectious disease specialists) were tested in vitro for their ability to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell viability. The infected cells were scored by immunofluorescence analysis using an antibody against the N protein of SARS CoV-2. Chloroquine, lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50 values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti helminthic drug niclosamide which exhibited potent antiviral activity against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect of niclosamide against SARS and MERS-CoV have been previously documented and recent evidence suggests that in may inhibit autophagy and reduce MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat asthma and allergic rhinitis, also exhibited antiviral efficacy but with a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of ciclesonide were directed against NSP15, a viral riboendonuclease which is the molecular target of this drug.</p>
</div>
<div id="limitations-74" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-74" class="anchor"></a>Limitations</h4>
<p>The drugs were tested against SARS CoV-2 infectivity in vitro only, therefore preclinical studies in animals and clinical trials in patients will be need for validation of these drugs as therapeutic agents for COVID-19. In addition, niclosamide exhibits low adsorption pharmatokinetically which could be alleviated with further development of drug formulation to increase effective delivery of this drug to target tissues. Nonetheless, niclosamide and ciclesonide represent promising therapeutic agents against SARS CoV-2 given that other compounds tested in the same study including favipiravir (currently used in clinical trials) and atazanavir (predicted as the most potent antiviral drug by AI-inference modeling) did not exhibit antiviral activity in the current study.</p>
</div>
<div id="credit-74" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-74" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2" class="anchor"></a>Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h3>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.001628]</span></p>
<div id="keywords-77" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-77" class="anchor"></a>Keywords</h4>
<p>animal model, pulmonary infiltrates, dynamic of antibody response, cytokine</p>
</div>
<div id="main-findings-74" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-74" class="anchor"></a>Main Findings</h4>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which all showed clinical signs of infection (respiratory pattern, reduced appetite, weight loss, elevated body temperature) resulting in moderate, transient disease. Four animals were euthanized at 3dpi, the 4 others at 21 dpi. Study of viral loads in different organs showed that nose swab and throat swabs were the most sensitive, with bronchio-alveolar lavage. Interstitial pneumonia was visible in radiographies and at the histological scale too. Clinically, the macaques had similar symptoms as described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs immediately after infection but less consistent thereafter which could reflect virus administration route (intranasal, oral). Bronchoalveolar lavages performed as a measure of virus replication in the lower respiratory tract on animals maintained for 21 days, contained high viral loads in 1 and 3dpi. The majority of the animals exhibited pulmonary edema and mild to moderate interstitial pneumonia on terminal bronchioles. In addition to the lung, viral RNA could also be detected throughout the respiratory track where viral replication mainly occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and through 10dpi after which they started to recover. After infection, serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10, IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody response started around 7dpi, and the antibody titers stayed elevated until 21dpi</strong> (day of animal euthanasia).</p>
</div>
<div id="limitations-75" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-75" class="anchor"></a>Limitations</h4>
<p>The macaques were inoculated via a combination of intratracheal, intranasal, ocular and oral routes, which might not reproduce how humans get infected. Maybe this can lead to different dynamics in the host immune response. Also, the authors noted that the seroconversion was not directly followed by a decline in viral loads, as observed in covid19 patients.</p>
</div>
<div id="significance-71" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-71" class="anchor"></a>Significance</h4>
<p>This work confirms that rhesus macaques can be a good model to study Covid-19, as it has been shown by other groups <span class="citation">[@doi:10.1101/2020.03.13.990226; @doi:10.1101/2020.03.13.990036; @doi:10.1101/2020.03.17.995639]</span>. While these experiments recapitulate moderate COVID-19 in humans, the mode of inoculation via a combination of intratracheal, intranasal, ocular and oral routes, might not reproduce how humans get infected and may lead to different dynamics in the host immune response. For example, the authors noted that the seroconversion was not directly followed by a decline in viral loads, as observed in COVID-19 patients. Therefore, it will be interesting to follow their antibody titers longer and further assess the possibility/effect of reinfection in these macaques. It is essential to be able to understand the dynamic of the disease and associated immune responses, and to work on vaccine development and antiviral drug testing.</p>
</div>
<div id="credit-75" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-75" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19" class="anchor"></a>ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h3>
<p><span class="citation">[@doi:10.1101/2020.03.21.20040261]</span></p>
<div id="keywords-78" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-78" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
</div>
<div id="main-findings-75" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-75" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and network analysis of existing literature to determine ACE2 regulation in patients who have frequent COVID-19 comorbidities <span class="math display">\[eg-
cardiovascular diseases, familial pulmonary hypertension, cancer\]</span>.</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation, metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2 higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
</div>
<div id="limitations-76" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-76" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations in this study need to be confirmed by analyzing upcoming transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and autoimmunity as risk factors in CoVID-19 patients due to lack of data; would be useful to extend such analyses to those datasets when available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs validation-experiments especially in CoVID-19 lung samples to mean anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of histone marks does not necessarily mean occupancy. Would be pertinent to check cell-line data (CCLE) or actual CoVID-19 patient samples to confirm ACE2 epigenetic control.</p></li>
</ul>
</div>
<div id="significance-72" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-72" class="anchor"></a>Significance</h4>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that could promote CoVID-19 severity. Shows important data-mining strategy to find gene-networks associated with ACE2 upregulation in co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might play an important role in CoVID-19 and can be preliminary targets for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2 expression could be a new target for CoVID-19 therapy with strategies such as KDM5 demethylases.</p></li>
</ul>
</div>
<div id="credit-76" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-76" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial" class="anchor"></a>Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h3>
<p>Bian et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.20040691]</span></p>
<div id="keywords-79" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-79" class="anchor"></a>Keywords</h4>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
</div>
<div id="main-findings-76" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-76" class="anchor"></a>Main Findings</h4>
<p>This work is based on previous work by the same group that demonstrated that SARS-CoV-2can also enter host cells via CD147 (also called Basigin, part of the immunoglobulin superfamily, is expressed by many cell types) consistent with their previous work with SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17 patients receiving Meplazumab, a humanized anti-CD147 antibody, in addition to all other treatments. 11 patients were included as a control group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group (e.g. at day 14 47% vs 17% improvement rate) compared to the control patients. Also, virological clearance was more rapid with median of 3 days in the Meplazumab group vs 13 days in control group. In laboratory values, a faster normalization of lymphocyte counts in the Meplazumab group was observed, but no clear difference was observed for CRP levels.</p>
</div>
<div id="limitations-77" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-77" class="anchor"></a>Limitations</h4>
<p>While the results from the study are encouraging, this study was non-randomized, open-label and on a small number of patients, all from the same hospital. It offers evidence to perform a larger scale study. Selection bias as well as differences between treatment groups (e.g. age 51yo vs 64yo) may have contributed to results. The authors mention that there was no toxic effect to Meplazumab injection but more patient and longer-term studies are necessary to assess this.</p>
</div>
<div id="significance-73" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-73" class="anchor"></a>Significance</h4>
<p>These results seem promising as for now there are limited treatments for Covid-19 patients, but a larger cohort of patient is needed. CD147 has already been described to facilitate HIV <span class="citation">[@doi:10.1073/pnas.111583198]</span>, measles virus <span class="citation">[@doi:10.1128/JVI.02168-09]</span>, and malaria <span class="citation">[@doi:10.1038/nature10606]</span> entry into host cells. This group was the first to describe the CD147-spike route of SARS-Cov-2 entry in host cells <span class="citation">[@doi:10.1101/2020.03.14.988345]</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov could enter host cells via this transmembrane protein <span class="citation">[@doi:10.1086/427811]</span>. Further biological understanding of how SARS-CoV-2 can enter host cells and how this integrates with ACE2R route of entry is needed. Also, the specific cellular targets of the anti-CD147 antibody need to be assessed, as this protein can be expressed by many cell types and has been shown to involved in leukocytes aggregation <span class="citation">[@doi:10.1371/journal.pone.0215557]</span>. Lastly, Meplazumab is not a commercially-available drug and requires significant health resources to generate and administer which might prevent rapid development and use.</p>
</div>
<div id="credit-77" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-77" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection" class="anchor"></a>Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h3>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.990770]</span></p>
<div id="keywords-80" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-80" class="anchor"></a>Keywords</h4>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
</div>
<div id="main-findings-77" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-77" class="anchor"></a>Main Findings</h4>
<p>In this study the authors report the affinity, cross reactivity (with SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206 monoclonal antibodies engineered from isolated IgG memory B cells of patients suffering from SARS-CoV-2 infection in Wuhan, China. All patients but one recovered from disease. Interestingly, the patient that did not recover had less SARS-CoV-2 specific B cells circulating compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore, plasma from patients recognized the receptor binding domain (RBD) from SARS-CoV-2 but had little to no cross-reactivity against the RBD of related viruses SARS-CoV and MERS-CoV, suggesting significant differences between the RBDs of the different viruses. Negligible levels of cross-neutralization using pseudoviruses bearing Spike proteins of SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV specific IgG memory B cells were single cell sorted to sequence the antibody genes that were subsequently expressed as recombinant IgG1 antibodies. From this library, 206 antibodies with different binding capacities were obtained. No discernible patterns of VH usage were found in the 206 antibodies suggesting immunologically distinct responses to the infection. Nevertheless, most high-binding antibodies were derived by clonal expansion. Further analyses in one of the patient derived clones, showed that the antibodies from three different timepoints did not group together in phylogenetic analysis, suggesting selection during early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have 10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in competition assays. Thus, low Kd values alone did not predict ACE2 competing capacities. Consistent with competition assays the two antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using SPR the neutralizing antibodies were found to recognize both overlapping and distinct epitopes of the RBD of SARS-CoV-2.</p>
</div>
<div id="limitations-78" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-78" class="anchor"></a>Limitations</h4>
<ol style="list-style-type: decimal">
<li>
<p>Relatively low number of patients</p>
<ol style="list-style-type: lower-alpha">
<li>No significant conclusion can be drawn about the possible &gt; correlation between humoral response and disease severity</li>
</ol>
</li>
<li>
<p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol style="list-style-type: lower-alpha">
<li><p>Huh7 cells were used in neutralization assays with &gt; pseudoviruses, and they may not entirely mimic what happens in &gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol>
</li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
</div>
<div id="significance-74" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-74" class="anchor"></a>Significance</h4>
<p>This paper offers an explanation as to why previously isolated antibodies against SARS-CoV do not effectively block SARS-CoV-2. Also, it offers important insight into the development of humoral responses at various time points during the first weeks of the disease in small but clinically diverse group of patients. Furthermore, it provides valuable information and well characterized antibody candidates for the development of a recombinant antibody treatment for SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have variability in its effectiveness, depending on the donor’s antibody repertoire at the time of donation.</p>
</div>
<div id="credit-78" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-78" class="anchor"></a>Credit</h4>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site-" class="section level3">
<h3 class="hasAnchor">
<a href="#characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site-" class="anchor"></a>Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h3>
<p>Davidson et al. <span class="citation">[@doi:10.1101/2020.03.22.002204]</span></p>
<div id="keywords-81" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-81" class="anchor"></a>Keywords</h4>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
</div>
<div id="main-findings-78" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-78" class="anchor"></a>Main Findings</h4>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore MinION as well as tandem mass spec (MS) on Vero cells (a cell line derived from kidney cells of the African green monkey that is deficient in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike (S) protein. The deleted sequence overlaps a predicted furin cleavage site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but not SARS-CoV or the closely related bat coronavirus RaTG13, which are cleaved at S1 / S2 by other proteases <span class="citation">[@doi:10.1126/science.abb2507]</span>. Furin cleavage sites at similar positions in other viruses have been linked to increased pathogenicity and greater cell tropism <span class="citation">[@doi:10.1128/JVI.00797-10]</span>. Loss of this site in SARS-CoV-2 has also already been shown to increase viral entry into Vero but not BHK cells (which are also interferon deficient) <span class="citation">[@doi:10.1016/j.cell.2020.02.058]</span>. The authors therefore make an important contribution in demonstrating that passage in Vero cells may lead to spontaneous loss of a key pathogenicity-conferring element in SARS-CoV-2.</p>
</div>
<div id="limitations-79" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-79" class="anchor"></a>Limitations</h4>
<p>As the authors note, a similar study posted earlier by Kim et al., which also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the S protein furin cleavage site <span class="citation">[@doi:10.1101/2020.03.12.988865]</span>. It therefore remains to be determined how likely it is that this mutation spontaneously arises. A quantitative investigation using multiple experimental replicas to understand the spontaneous viral mutation rate at this site and elsewhere would be informative. Also, the mechanistic basis for the higher viral fitness conferred by loss of the furin cleavage site in Vero cells – but, evidently, not in vivo in humans, as this site is maintained in all currently sequenced circulating isolates - remains to be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’ bioinformatic pipeline required aligning transcripts to a reference to correct sequencing artifacts. This presumably made it difficult or impossible to identify other kinds of mutations, such as single nucleotide substitutions, which may occur even more frequently than the deletions identified in this work. Pairing long read sequencing with higher-accuracy short-read sequencing may be one approach to overcome this issue.</p>
</div>
<div id="significance-75" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-75" class="anchor"></a>Significance</h4>
<p>As the authors suggest, animal studies using live virus challenge may need to periodically verify the genomic integrity of the virus, or potentially risk unknowingly using a likely less-pathogenic variant of the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the SARS-CoV-2 viral transcriptome and proteome. For example, the authors found multiple versions of transcripts encoding the nucleocapsid (N) protein, each with different small internal deletions, some of which were verified for translation by MS. A number of peptides arising from translation of unexpected rearrangements of transcripts were also detected. Additionally, the authors identified phosphorylation of a number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For any cases where these have functional consequences, targeting the kinases responsible could be an avenue for drug development. Understanding the functional consequences of the mutations, transcript variations, and post translational modifications identified in this study will be important future work.</p>
</div>
<div id="credit-79" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-79" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Davidson et al. “Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.” </strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.03.22.002204v1.full.pdf"><strong><em>https://www.biorxiv.org/content/10.1101/2020.03.22.002204v1.full.pdf</em></strong></a></p>
<p><strong>Review contributors: Tim O’Donnell, Maria Kuksin</strong></p>
<p><strong>Keywords</strong>:</p>
<p>Transcription, RNA-seq, proteomics, mass spec, furin cleavage site, mutation, pathogenicity</p>
<p><strong>Main findings</strong></p>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore MinION as well as tandem mass spec (MS) on Vero cells (a cell line derived from kidney cells of the African green monkey that is deficient in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike (S) protein. The deleted sequence overlaps a predicted furin cleavage site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but not SARS-CoV or the closely related bat coronavirus RaTG13, which are cleaved at S1 / S2 by other proteases <span class="math display">\[1\]</span>. Furin cleavage sites at similar positions in other viruses have been linked to increased pathogenicity and greater cell tropism <span class="math display">\[2\]</span>. Loss of this site in SARS-CoV-2 has also already been shown to increase viral entry into Vero but not BHK cells (which are also interferon deficient) <span class="math display">\[3\]</span>. The authors therefore make an important contribution in demonstrating that passage in Vero cells may lead to spontaneous loss of a key pathogenicity-conferring element in SARS-CoV-2.</p>
<p><strong>Critical analysis</strong></p>
<p>As the authors note, a similar study posted earlier by Kim et al., which also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the S protein furin cleavage site <span class="math display">\[4\]</span>. It therefore remains to be determined how likely it is that this mutation spontaneously arises. A quantitative investigation using multiple experimental replicas to understand the spontaneous viral mutation rate at this site and elsewhere would be informative. Also, the mechanistic basis for the higher viral fitness conferred by loss of the furin cleavage site in Vero cells – but, evidently, not in vivo in humans, as this site is maintained in all currently sequenced circulating isolates - remains to be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’ bioinformatic pipeline required aligning transcripts to a reference to correct sequencing artifacts. This presumably made it difficult or impossible to identify other kinds of mutations, such as single nucleotide substitutions, which may occur even more frequently than the deletions identified in this work. Pairing long read sequencing with higher-accuracy short-read sequencing may be one approach to overcome this issue.</p>
<p><strong>Relevance</strong></p>
<p>As the authors suggest, animal studies using live virus challenge may need to periodically verify the genomic integrity of the virus, or potentially risk unknowingly using a likely less-pathogenic variant of the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the SARS-CoV-2 viral transcriptome and proteome. For example, the authors found multiple versions of transcripts encoding the nucleocapsid (N) protein, each with different small internal deletions, some of which were verified for translation by MS. A number of peptides arising from translation of unexpected rearrangements of transcripts were also detected. Additionally, the authors identified phosphorylation of a number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For any cases where these have functional consequences, targeting the kinases responsible could be an avenue for drug development. Understanding the functional consequences of the mutations, transcript variations, and post translational modifications identified in this study will be important future work.</p>
<p><span class="math display">\[1\]</span> Wrapp, D. et al. Science. <a href="doi:10.1126/science.abb2507" class="uri">doi:10.1126/science.abb2507</a></p>
<p><span class="math display">\[2\]</span> Sun et al. <a href="doi:10.1128/JVI.00797-10" class="uri">doi:10.1128/JVI.00797-10</a></p>
<p><span class="math display">\[3\]</span> Walls et al. <a href="doi:10.1016/j.cell.2020.02.058" class="uri">doi:10.1016/j.cell.2020.02.058</a></p>
<p><span class="math display">\[4\]</span> Kim et al. <a href="doi:10.1101/2020.03.12.988865" class="uri">doi:10.1101/2020.03.12.988865</a>.</p>
</div>
</div>
<div id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing" class="section level3">
<h3 class="hasAnchor">
<a href="#a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing" class="anchor"></a>A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h3>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.22.002386]</span></p>
<div id="keywords-82" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-82" class="anchor"></a>Keywords</h4>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
</div>
<div id="main-findings-79" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-79" class="anchor"></a>Main Findings</h4>
<p>Gordon et al cloned, tagged and expressed 26 of the 29 SARS-CoV-2 proteins individually in HEK293T cells and used mass spectrometry to identify protein-protein interactions. They identified 332 viral-host protein-protein interactions. Furthermore, they used these interactions to identify 66 existing drugs known to target host proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact with proteins regulated by the mTOR pathway, so mTOR inhibitors Silmitasertib and Rapamycin are possible drug candidates).</p>
</div>
<div id="limitations-80" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-80" class="anchor"></a>Limitations</h4>
<p>The main limitation of the study stems from the reductionist model: overexpression of plasmids encoding individual viral proteins in HEK293T cells. This precludes any interactions between the viral proteins, or the combined effects of multiple proteins on the host, as they are expressed individually. Moreover, HEK293T cells come from primary embryonic kidney and therefore might not reflect how SARS-CoV-2 interacts with its primary target, the lung. However, the authors found that the proteins found to interact with viral proteins in their experiments are enriched in lung tissue compared to HEK293Ts.</p>
</div>
<div id="significance-76" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-76" class="anchor"></a>Significance</h4>
<p>The authors provide a “SARS-CoV-2 interaction map,” which may provide potential hypotheses as to how the virus interacts with the host. Further, they identified existing drugs that could disrupt these host-viral interactions and curb SARS-CoV-2 infection. Although these interactions have not been validated, this paper acts as a valuable resource.</p>
</div>
<div id="credit-80" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-80" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-na%C3%AFve-and-experienced-covid-19-patients" class="anchor"></a>First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h3>
<p>Chen et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.22.20034041]</span></p>
<div id="keywords-83" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-83" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
</div>
<div id="main-findings-80" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-80" class="anchor"></a>Main Findings</h4>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a commercialized HCV protease inhibitor <span class="citation">[@doi:10.1128/AAC.00699-08]</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation">[@doi:10.14218/JCTH.2019.00033]</span>, a CYP3A4 inhibitor (which enhances the plasma concentration and bioavailabilty of Danoprevir). Two patients had never received anti-viral therapy before (=naïve), whereas nine patients were on Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir (=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a decreased hospitalization time. Patients treated with Lopinavir/Ritonavir did not have a negative PCR test, while after switching to Danoprevir/Ritonavir treatment, the first negative PCR test occurred at a median of two days.</p>
</div>
<div id="limitations-81" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-81" class="anchor"></a>Limitations</h4>
<p>The results of the study are very hard to interpret as there is no control group not receiving Danoprevir/Ritonavir treatment. This was especially true in naïve patients who seemed to have more mild symptoms before the start of the study and were younger (18 and 44yo) compared to the experienced patients (18 to 66yo). The possibility that the patients would have recovered without Danoprevir/Ritonavir treatment cannot be excluded.</p>
</div>
<div id="significance-77" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-77" class="anchor"></a>Significance</h4>
<p>The authors of the study treated patients with Danoprevir, with the rational to that this is an approved and well tolerated drug for HCV patients <span class="citation">[@doi:10.14218/JCTH.2019.00033]</span>, and that it could also target the protease from SARS-CoV-2 (essential for viral replication and transcription). Indeed, homology modelling data indicated that HCV protease inhibitors have the highest binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation">[@doi:10.1101/2020.02.05.936013]</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir might be beneficial for Covid-19 patients, additional clinical trials with more patients and with better methodology (randomization and control group) are needed to make further conclusions.</p>
</div>
<div id="credit-81" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-81" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial" class="anchor"></a>Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h3>
<p><span class="citation">[@doi:10.1101/2020.03.22.20040758]</span></p>
<div id="keywords-84" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-84" class="anchor"></a>Keywords</h4>
<ul>
<li>hydroxychloroquine</li>
</ul>
</div>
<div id="study-description-1" class="section level4">
<h4 class="hasAnchor">
<a href="#study-description-1" class="anchor"></a>Study Description</h4>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy in the treatment of COVID-19. From February 4 – February 28, 2020 142 COVID-19 positive patients were admitted to Renmin Hospital of Wuhan University. 62 patients met inclusion criteria and were enrolled in a double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol style="list-style-type: decimal">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li>
<p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note: relevant clinical references described below.)</p>
<ol style="list-style-type: lower-alpha">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt; 85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction of inspired oxygen) is used to classify the severity of acute respiratory distress syndrome (ARDS). Mild ARDS has a PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe &lt; 100.</p></li>
</ol>
</li>
<li><p>Willing to receive a random assignment to any designated treatment group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol style="list-style-type: decimal">
<li><p>Severe or critical respiratory illness (not explicitly defined, presumed to be respiratory function worse than outlined in inclusion criteria); or participation in trial does not meet patient’s maximum benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt; twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral agents, antibacterial agents, and immunoglobulin, with or without corticosteroids. Patients in the HCQ treatment group received additional oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from days 1-5 of the trial. The primary endpoint was 5 days post enrollment or a severe adverse reaction to HCQ. The primary outcome evaluated was time to clinical recovery (TTCR), defined as return to normal body temperature and cough cessation for &gt; 72h. Chest CT were imaged on days 0 and 6 of the trial for both groups; body temperature and patient reports of cough were collected 3x/day from day 0 – 6. The mean age and sex distribution between the HCQ and control arms were comparable.</p>
</div>
<div id="main-findings-81" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-81" class="anchor"></a>Main Findings</h4>
<p>There were 2 patients showing mild secondary effects of HCQ treatment. More importantly, while 4 patients in the control group progressed to severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p = 0.0008); time to cessation of cough was similarly reduced (3.1 days control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19 has a modest effect on clinical recovery (symptom relief on average 1 day earlier) but may be more potent in reducing the progression from mild to severe disease.</p>
</div>
<div id="limitations-82" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-82" class="anchor"></a>Limitations</h4>
<p>This study is limited in its inclusion of only patients with mild disease, and exclusion of those on any treatment other than the standard of care. It would also have been important to include the laboratory values of positive RT-PCR detection of SARS-CoV-2 to compare the baseline and evolution of the patients’ viral load.</p>
</div>
<div id="limitations-83" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-83" class="anchor"></a>Limitations</h4>
<p>Despite its limitations, the study design has good rigor as a double blind RCT and consistent symptom checks on each day of the trail. Now that the FDA has approved HCQ for treatment of COVID-19 in the USA, this study supports the efficacy of HCQ use early in treatment of patients showing mild symptoms, to improve time to clinical recovery, and possibly reduce disease progression. However, most of the current applications of HCQ have been in patients with severe disease and for compassionate use, which are out of the scope of the findings presented in this trial. Several additional clinical trials to examine <a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a> are now undergoing; their results will be critical to further validate these findings.</p>
</div>
<div id="credit-82" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-82" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Clinical features and the maternal and neonatal outcomes of pregnant women with coronavirus disease 2019</strong></p>
<p><strong><em>Keywords</em></strong></p>
<p>Pregnancy, SARS-CoV2, neonatal and maternal Covid-19 outcome</p>
<p><strong><em>Key findings</em></strong></p>
<p>33 pregnant woman and 28 newborns were included in this retrospective multicenter study, conducted at 5 hospitals in Wuhan and Hubei province, China, between January 1 and February 20, 2020. All women were diagnosed with Covid-19 by qPCR or viral gene sequencing based on the Chinese New Corona Pneumonia Prevention and Control Program, 6<sup>th</sup> edition, and were further subdivided into four groups based on clinical severity: (1) mild, presence of mild clinical symptoms without radiological abnormalities; (2) moderate, fever or upper respiratory symptoms as well as radiological signs of pneumonia; (3) severe, at least one of the following: shortness of breath/respiratory rate &gt;30/min, resting oxygen saturation SaO<sub>2</sub>&lt;93%, Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary damage); and (4) severe-acute, acute respiratory distress with need for mechanical ventilation; systemic shock; multi-organ failure and transfer to ICU. Maternal admission to ICU, mechanical ventilation or death were defined as primary outcomes; secondary study outcomes comprised clinical Covid-19 severity in both mothers and newborns, including development of ARDS, neonatal ICU admission as well as mortality.</p>
<p><em>Maternal characteristics and outcome:</em> 3 out of 33 women were in their second trimester of pregnancy (17, 20 and 26 weeks), and 15/33 (45.5%) had a previous history of underlying chronic health disorders including cardiovascular, cerebrovascular or nervous system disease. Common Covid-19 symptoms at presentation were fever (63.6%), dry cough (39.4%), fatigue (21.2%), and shortness of breath (21.2%). Less common symptoms included diarrhea, post-partum fever, muscle ache, sore throat and chest pain. 4 (12.1%) pregnant women had no apparent symptoms. The majority of cases were classified as mild (39.4%) or moderate (57.6%); however, one woman developed severe Covid-19. 40.6% of women were diagnosed with bilateral pneumonia, 43.8% presented with unilateral pneumonia, and 15.6% showed radiological ground-glass opacity. 87.9% of women required oxygen administration, and one (3%) woman had to be put on non-invasive mechanical ventilation (primary outcome). 81.5% of women had a C-section and only 5% had vaginal deliveries. Obstetrical complications were seen in 22.2% of women, including three cases of preterm rupture of membranes, two cases of hypertensive disorders of pregnancy, and one case of spontaneous preterm labor. Five pregnancies were ongoing at the end of the observation point of this study; one woman decided to have her pregnancy terminated. <em>Neonatal outcome:</em> Out of 28 newborns included in this study, 35.7% were born preterm at &lt;37 weeks of gestation with Apgar scores ranging from 8-10/10 at 1 min and from 9-10/10 after 5 min, indicating normal heart and respiratory rates. 17.9% of newborns were of low birth weight (not specified) and 14.3% showed signs of fetal distress (also not specified). According to the authors of this study, none of the newborns presented with clinical Covid-19 symptoms. However, one newborn, delivered at 34 weeks of gestation, was diagnosed with (apparently Covid-19 unrelated?) ARDS and transferred to NICU (secondary outcome). Of 26 newborns tested for SARS-CoV2, only one was found positive and showed radiological signs of pneumonia, but no clinical symptoms of Covid-19. It remains unclear whether this was the same case as the newborn diagnosed with ARDS. The affected newborn did not require any treatment and was discharged at 16 days post birth. In summary, the primary outcome “mechanical ventilation” in pregnant women was rare (3%), no other primary outcomes were reached. Most Covid-19 cases in pregnant women were described as mild to moderate. Only one of 28 (3.57%) newborns was diagnosed with ARDS (secondary outcome).</p>
<p><strong><em>Potential limitations</em></strong></p>
<p>Major limitations of this study are its small size and the rudimentary and at times inadequate description of patient specifics. For example, underlying health conditions that might be affecting Covid-19 outcome in pregnant women should have been clearly specified (other than being of be listed (not just &lt;37 weeks). Given that maternal infection status seemed mostly unknown at the time of birth and, more importantly, that the majority of cases in this study were clinically asymptomatic or mild to moderate, it remains unclear whether the C-sections performed were a medical necessity or elective procedures. This is of importance and should have been discussed. With regard to neonatal outcome, it is also not apparent whether the newborn found to be infected with SARS-CoV2 and the case diagnosed with ARDS were the same individual. If this was the case, it would be incorrect to refer to all newborns as asymptomatic. Additionally, it seems somewhat unlikely that a newborn with a near-perfect Apgar score would present with ARDS immediately after birth. Likewise, any individual diagnosed with ARDS would certainly be expected to receive supportive treatment including (invasive) mechanical ventilation. While it is highly relevant that overall clinical outcome in pregnant women diagnosed with Covid-19 seems better than in SARS or MERS (as discussed by the authors), it nevertheless needs to be stressed that more than 37% of newborns in this study were delivered preterm and that the obstetric complication rate of 22% seems higher than non-Covid-19 average.</p>
<p><strong><em>Overall relevance for the field</em></strong></p>
<p>Observations in this study confirm some of the findings published in a case series by Yu N et al. (Lancet Infect Dis 2020; <a href="https://doi.org/10.1016/" class="uri">https://doi.org/10.1016/</a> S1473-3099(20)30176-6). However, due to the relatively small study size of 33 pregnant women and 28 newborns, this study lacks statistical power and final conclusions on Covid-19 outcomes in pregnant women and newborns cannot be drawn. Yet, the data collected here are important and should be incorporated into larger data sets for more insight. Understanding the clinical course and effects of Covid19 in both pregnant women and newborns is essential, and while there are some recent publications on vertical SARS-CoV2 transmission between mothers and newborns (Dong L et al, JAMA March 26, 2020, <a href="doi:10.1001/jama.2020.4621" class="uri">doi:10.1001/jama.2020.4621</a>; Zeng H et al, JAMA March 26, 2020, <a href="doi:10.1001/jama.2020.4861" class="uri">doi:10.1001/jama.2020.4861</a>) as well as on neonatal infection at birth (Zeng L et al, JAMA March 26, 2020, <a href="doi:10.1001/jamapediatrics.2020.0878" class="uri">doi:10.1001/jamapediatrics.2020.0878</a>), our knowledge of how these patient subsets are affected is still very limited.</p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide, modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC complexes on Protein Data Bank, substitute homologous peptides with SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding energy for further study and publish models in an online database (<a href="https://rosettamhc.chemistry.ucsc.edu" class="uri">https://rosettamhc.chemistry.ucsc.edu</a>).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted SARS-CoV-2 peptides must be validated experimentally for immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high resolution X-ray structures on PDB, authors only simulate 9-mer and 10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a starting point, backbone conformations that deviate significantly between test and template peptides are not captured. Furthermore, Rosetta modeling protocols do not capture all possible structures and binding energy scoring does not fully recapitulate fundamental forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides that may direct a targeted, experimental validation approach toward a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results from NetMHCpan 4.0, supporting machine learning prediction with structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method of leveraging existing X-ray structures for identification of immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol style="list-style-type: decimal">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D., Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34), 4748-4763. <a href="doi:10.1021/acs.biochem.6b00444" class="uri">doi:10.1021/acs.biochem.6b00444</a></p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J., DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design. J Chem Theory Comput, 13(6), 3031-3048. <a href="doi:10.1021/acs.jctc.7b00125" class="uri">doi:10.1021/acs.jctc.7b00125</a></p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset" class="section level3">
<h3 class="hasAnchor">
<a href="#serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset" class="anchor"></a>Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h3>
<p>Lou et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.23.20041707]</span></p>
<div id="keywords-85" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-85" class="anchor"></a>Keywords</h4>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
</div>
<div id="main-findings-82" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-82" class="anchor"></a>Main Findings</h4>
<p>Currently, the diagnosis of SARS-CoV-2 infection entirely depends on the detection of viral RNA using polymerase chain reaction (PCR) assays. False negative results are common, particularly when the samples are collected from upper respiratory. Serological detection may be useful as an additional testing strategy. In this study the authors reported that a typical acute antibody response was induced during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8% (79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first detectible serology marker was total antibody followed by IgM and IgG, with a median seroconversion time of 15, 18 and 20 days-post exposure (d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was first detected at day 7d.p.e in 98.9% of the patients. Interestingly they found that viral load declined as antibody levels increased. This was in contrast to a previous study <span class="citation">[@doi:10.1101/2020.03.02.20030189]</span>, showing that increased antibody titers did not always correlate with RNA clearance (low number of patient sample).</p>
</div>
<div id="limitations-84" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-84" class="anchor"></a>Limitations</h4>
<p>Current knowledge of the antibody response to SAR-CoV-2 infection and its mechanism is not yet well elucidated. Similar to the RNA test, the absence of antibody titers in the early stage of illness could not exclude the possibility of infection. A diagnostic test, which is the aim of the authors, would not be useful at the early time points of infection but it could be used to screen asymptomatic patients or patients with mild disease at later times after exposure.</p>
</div>
<div id="significance-78" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-78" class="anchor"></a>Significance</h4>
<p>Understanding the antibody responses against SARS-CoV2 is useful in the development of a serological test for the diagnosis of COVID-19. This manuscript discussed acute antibody responses which can be deducted in plasma for diagnostic as well as prognostic purposes. Thus, patient-derived plasma with known antibody titers may be used therapeutically for treating COVID-19 patients with severe illness. <span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
</div>
<div id="credit-83" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-83" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems" class="anchor"></a>SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h3>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.004655]</span></p>
<div id="keywords-86" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-86" class="anchor"></a>Keywords</h4>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
</div>
<div id="main-findings-83" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-83" class="anchor"></a>Main Findings</h4>
<p>Given the high mortality rate of SARS-CoV-2 relative to other respiratory viruses such seasonal IAV and RSV, there may be underlying host-pathogen interactions specific to SARS-CoV-2 that predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and <em>in vitro</em> systems, the authors profiled host cell transcriptional responses to SARS-CoV-2 and to other common respiratory viruses (seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an induction of type I interferon response signaling and the upregulation of cytokine/chemokines transcripts. In comparison with IAV and RSV infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I and type III interferon expression and higher levels of two cytokines previously implicated in respiratory inflammation. Lastly, <em>in vivo</em> data from ferrets showed a reduced induction of cytokines and chemokines by SARS-CoV-2 infection relative to IAV infection.</p>
</div>
<div id="limitations-85" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-85" class="anchor"></a>Limitations</h4>
<p>While these results are promising, there are several key weaknesses of this paper. 1) As the authors point out, there is an undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease (TMPRSS2) expression in the lung epithelial cell line used for the <em>in vitro</em> studies. This raises the important question of whether viral replication actually occurs in any of the models used, which may explain the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2 treated cells. Further studies characterizing viral titers across timepoints are needed. 2) Furthermore, these studies only characterize the host response at a single dose and timepoint per virus, and it is unclear why these doses/timepoints were chosen. This leaves open the possibility that the observed differences between viruses could be due to differences in dose, timing, host response, or a combination of all of these. 3) It is unclear whether ferrets are productively infected, which cell types are infected, and the extent/timing of the clinical course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not strongly correlate and the reasons for this are unclear.</p>
</div>
<div id="significance-79" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-79" class="anchor"></a>Significance</h4>
<p>This paper describes potentially unique transcriptional signatures of host cells exposed to SARS-CoV-2. If validated, these findings may help explain clinical outcomes and could be targeted in future therapeutic interventions.</p>
</div>
<div id="potential-conflicts-of-interest-disclosure" class="section level4">
<h4 class="hasAnchor">
<a href="#potential-conflicts-of-interest-disclosure" class="anchor"></a>Potential Conflicts of Interest Disclosure</h4>
<p>The reviewers are also researchers at the Icahn School of Medicine at Mount Sinai.</p>
</div>
<div id="credit-84" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-84" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Analysis of Serologic Cross-Reactivity Between Common Human Coronaviruses and SARS-CoV-2 Using Coronavirus Antigen Microarray</strong></p>
<p><strong><em>Keywords</em></strong>: serum antibodies, cross-reactivity, antigen microarray</p>
<p><strong><em>Key Findings</em></strong></p>
<p>The authors of this study describe a common respiratory viral antigens microarray that can be used to assess the cross-reactivity of antibodies against the novel SARS-CoV-2 antigens and common human coronaviruses. Sixty-seven (67) antigens from across subtypes of coronaviruses, influenza viruses, adenoiruses, respiratory syncytial viruses and other viruses, were printed onto microarrays, probed with human sera, and analyzed. Five (5) serum samples collected before the SARS-CoV-2 outbreak were tested as part of a study that monitored acute respiratory infection cases. Four out of five (4/5) sera samples showed high IgG seroreactivity across the 4 common human coronaviruses. All sera showed low IgG seroreactivity to SARS-CoV-2.</p>
<p>The S1 domain of the spike protein is suggested to be subtype specific as it demonstrated very low cross-reactivity among the novel coronaviruses (SARS-CoV-2, SARS-CoV and MERS-CoV) and between the novel coronal viruses and common human coronaviruses. In contrast, the S2 domain of the spike protein and the nucleocapsid (NP) protein showed low levels of cross-reactivity between the coronavirus subtypes, suggesting the presence of more conserved antigenic domains. Therefore, the authors conclude that the S1 domain is an ideal candidate for virus-specific serologic assays.</p>
<p><strong><em>Importance</em></strong></p>
<p>This study provides insights into the potential cross-reactivity of common human coronavirus antibodies for SARS-CoV-2 antigens. Understanding cross-reactivity is useful because it can help in the development of sensitive and specific serological assays that can test for COVID-19. Additionally, studies such as this can help inform vaccine development and indicate which antigens would be optimal to target.</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=5) of serum samples that all came from students who were part of the same college resident community at The University of Maryland. A greatest limitation is the lack of serum samples from COVID-19 patients. Additionally, the MFI of 2019-nCov from these naïve samples (in particular sera 4) is not zero. This indicates that while relatively low, there is some cross-reactivity of common human coronavirus antibodies for SARS-CoV-2 antigens, including the S1 domain. A larger sample size and samples from SARS-CoV2 infected patients may be needed to confirm the sensitivity and specificity of the assay.</p>
</div>
</div>
<div id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china" class="anchor"></a>A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h3>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042119]</span></p>
<div id="keywords-87" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-87" class="anchor"></a>Keywords</h4>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
</div>
<div id="main-findings-84" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-84" class="anchor"></a>Main Findings</h4>
<p>377 hospitalized patients were divided into two groups: severe and non-severe pneumonia. The laboratory results of their first day of admission were retrospectively analyzed to identify predictors of disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such as high blood pressure, type 2 diabetes, coronary heart disease, and chronic obstructive pulmonary disease), the independent risk factors identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers, they showed that their multiplication <strong><span class="math display">\[(Neutrophil/lymphocyte count)
\* CRP \* D-dimer\]</span></strong> was a better predictor of disease severity, with higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value of 2.68.</p>
</div>
<div id="limitations-86" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-86" class="anchor"></a>Limitations</h4>
<p>This study included 377 hospitalized patients. Among them, 45.6% patients tested positive for SARS-Cov-2 nucleic acid test results, and others were included in the study based on clinically diagnosis even if the molecular diagnosis was negative. Thus, additional studies are needed to verify this on a larger number of covid-19 certified patients and the cutoff value might be adjusted. Also, all the patients that did not have the clinical characteristics of severe pneumonia were included in the non-severe pneumonia group, but usually patients are also divided into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more specific predictor. Another study showed that the neutrophils to CD8+ T cells ratio was a strong predictor of disease severity <span class="citation">[@doi:10.1101/2020.02.16.20023671]</span>. Another more precise study showed that the percentage of helper T cells and regulatory T cells decrease but the percentage of naïve helper T cells increases in severe cases <span class="citation">[@doi:10.1093/cid/ciaa248]</span>. Taking these subpopulations into account might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity and LDH <span class="citation">[@doi:10.1101/2020.03.03.20030668]</span> or IL6 <span class="citation">[@doi:10.1101/2020.03.08.20031229]</span> levels, but the authors here do not discuss LDH nor IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of admission, studying the dynamic of the changes in patients might also be interesting to better predict disease severity.</p>
</div>
<div id="significance-80" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-80" class="anchor"></a>Significance</h4>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a predictor of disease severity as shown by many others <span class="citation">[@doi:10.1093/cid/ciaa248; @doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.10.20021584]</span>. The novelty here is that they show that a combination with other markers can enhance the specificity and sensibility of the predictor, although the study could be improved by taking into account sub-populations of lymphocytes and more biological factors from patients such as LDH and IL6.</p>
</div>
<div id="credit-85" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-85" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study" class="section level3">
<h3 class="hasAnchor">
<a href="#metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study" class="anchor"></a>Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h3>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042283]</span></p>
<div id="keywords-88" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-88" class="anchor"></a>Keywords</h4>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
</div>
<div id="main-findings-85" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-85" class="anchor"></a>Main Findings</h4>
<p>Retrospective Study on 97 COVID-19 hospitalized patients (25 severe and 72 non-severe) analyzing clinical and laboratory parameter to predict transition from mild to severe disease based on more accessible indicators (such as fasting blood glucose, serum protein or blood lipid) than inflammatory indicators. In accordance with other studies, age and hypertension were risk factors for disease severity, and lymphopenia and increased IL-6 was observed in severe patients. The authors show that fasting blood glucose (FBG) was altered and patients with severe disease were often hyperglycemic. Data presented support that hypoproteinaemia, hypoalbuminemia, and reduction in high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease severity.</p>
</div>
<div id="limitations-87" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-87" class="anchor"></a>Limitations</h4>
<p>In this study non-severe patients were divided in two groups based on average course of the disease: mild group1 (14 days, n=28) and mild group 2 (30 days, n=44). However mild patients with a longer disease course did not show an intermediate phenotype (between mild patients with shorter disease course and severe patients), hence it is unclear whether this was a useful and how it impacted the analysis. Furthermore, the non-exclusion of co-morbidity factors in the analysis may bias the results (e.g. diabetic patients and glucose tests) It is not clear at what point in time the laboratory parameters are sampled. In table 3, it would have been interesting to explore a multivariate multiple regression. The correlation lacks of positive control to assess the specificity of the correlation to the disease vs. correlation in any inflammatory case. The dynamic study assessing the predictability of the laboratory parameter is limited to 2 patients. Hence there are several associations with disease severity, but larger studies are necessary to test the independent predictive value of these potential biomarkers.</p>
</div>
<div id="significance-81" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-81" class="anchor"></a>Significance</h4>
<p>As hospital are getting overwhelmed a set of easily accessible laboratory indicators (such as serum total protein) would potentially provide a triage methodology between potentially severe cases and mild ones. This paper also opens the question regarding metabolic deregulation and COVID-19 severity.</p>
</div>
<div id="credit-86" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-86" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#viral-kinetics-and-antibody-responses-in-patients-with-covid-19" class="anchor"></a>Viral Kinetics and Antibody Responses in Patients with COVID-19</h3>
<p><span class="citation">[@doi:10.1101/2020.03.24.20042382]</span></p>
<div id="keywords-89" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-89" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
</div>
<div id="main-findings-86" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-86" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset in sputum, nasal swabs, and stool samples. Shedding ranged from 12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as ‘strong vs weak vs non-responders’. Strong responders (with higher IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
</div>
<div id="limitations-88" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-88" class="anchor"></a>Limitations</h4>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in intestine/liver or from swallowed sputum. Transmission electron microscopy (TEM) carried out on sputum samples as proof of concept, but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver disease) that were not accounted for when tracking antibody responses. Viral kinetics and IgM/IgG titers in subsets of patients with underlying conditions/undergoing certain medication would be informative.</p></li>
</ul>
</div>
<div id="significance-82" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-82" class="anchor"></a>Significance</h4>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different disease progression and kinetics is useful to stratify patients, and for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and CoVID-19 disease severity <span class="citation">[@doi:10.1101/2020.03.20.20039495]</span>, similar to macaque studies in SARS <span class="citation">[@doi:10.1172/jci.insight.123158]</span>. It would be critical to see if anti-NP or anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar detrimental effects before clinical use.</p></li>
</ul>
</div>
<div id="credit-87" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-87" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome" class="section level3">
<h3 class="hasAnchor">
<a href="#covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome" class="anchor"></a>COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h3>
<p><span class="citation">[@doi:10.1101/2020.03.24.20042655]</span></p>
<div id="keywords-90" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-90" class="anchor"></a>Keywords</h4>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
</div>
<div id="main-findings-87" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-87" class="anchor"></a>Main Findings</h4>
<p>This study is based on flow cytometry immunophenotyping of PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The authors identify a population of abnormally large (FSC-hi) monocytes, present in COVID19 patients, but absent in PBMCs of healthy volunteers (n=16) or patients with different infections (AIDS, malaria, TB). This FSC-hi monocytic population contains classical, intermediate and non-classical (monocytes (based on CD14 and CD16 expression) that produce inflammatory cytokines (IL-6, TNF and IL-10). The authors suggest an association of FSC-hi monocytes with poor outcome and correlate a high percentage of FSC-low monocytes, or higher ratio of FSC-low/hi monocytes, with faster hospital discharge.</p>
</div>
<div id="limitations-89" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-89" class="anchor"></a>Limitations</h4>
<p>While identification of the monocytic population based on FSC is rather robust, the characterization of these cells remains weak. A comprehensive comparison of FSC-hi monocytes with FSC-low monocytes from patients and healthy controls would be of high value. It is unclear if percentages in blood are among CD45+ cells. Furthermore, it would have been important to include absolute numbers of different monocytic populations (in table 1 there are not enough samples and it is unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the monocytes, and highlight these cells as potential targets of SARS-Cov2. However, appropriate controls are needed. CD16 has high affinity to rabbit IgG and it is unclear whether the authors considered unspecific binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on monocytes needs to be assessed. Furthermore, it would be important to confirm infection of monocytes by presence of viral proteins or viral particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development of the disease and its severity, some data expected at this level are neither present nor addressed. Although the cohort is small, it does include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi monocytes in comparison to other patients? Was this apparent early in the disease course? Does this population of FSC-hi monocytes differ between ICU patients and others in terms of frequency, phenotype or cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example, in Fig. 5, according to the legend it seems that patients with FSC-high monocytes are discharged faster from the hospital. However according to description in the text, patients were grouped in high or low levels of FSC-low monocytes.</p>
</div>
<div id="significance-83" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-83" class="anchor"></a>Significance</h4>
<p>Despite the limitations of this study, the discovery of a FSC-high monocyte population in COVID-19 patients is of great interest. With similar implication, a the recent study by Zhou et al. <span class="citation">[@doi:10.1101/2020.02.12.945576]</span> identified a connection between an inflammatory CD14+CD16+ monocyte population and pulmonary immunopathology leading to deleterious clinical manifestations and even acute mortality after SARS-CoV-2 infections. Although the presence of these monocytes in the lungs has yet to be demonstrated, such results support the importance of monocytes in the critical inflammation observed in some COVID19 patients.</p>
</div>
<div id="credit-88" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-88" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study" class="anchor"></a>Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h3>
<p>Miller et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042937]</span></p>
<div id="keywords-91" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-91" class="anchor"></a>Keywords</h4>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
</div>
<div id="main-findings-88" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-88" class="anchor"></a>Main Findings</h4>
<p>The authors compared middle and high income countries that never had a universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium) and countries with a current policy (low income countries were excluded from the analysis as their number of cases and deaths might be underreported for the moment). <strong>Countries that never implement BCG vaccination have a higher mortality rate than countries which have a BCG vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next, <strong>the authors show that an earlier start of vaccination correlates with a lower number of deaths per million inhabitants</strong>. They interpret this as the vaccine protecting a larger fraction of elderly people, which are usually more affected by COVID-19. Moreover, higher number of COVID-19 <strong>cases</strong> were presented in countries that never implemented a universal BCG vaccination policy.</p>
</div>
<div id="limitations-90" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-90" class="anchor"></a>Limitations</h4>
<p>While this study aims to test an intriguing hypothesis unfortunately, the data is not sufficient at this time to accurately make any determinations. Several caveats must be noted including: not all countries are in the same stage of the pandemic, the number of cases/deaths is still changing very rapidly in a lot of countries and thus the association may only reflect exposure to the virus. This analysis would need to be re-evaluated when all the countries are passed the pandemic and more accurate numbers are available. Additionally, very few middle and high-income countries ever implemented universal BCG vaccination, which can be a source of bias (5 countries, vs 55 that have a BCG vaccine policy). Effective screening and social isolation policies also varied considerable across the countries tested and may reflect another important confounder. The authors could consider analyzing the Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more correct for exposure although testing availability will still bias this result. Variability in mortality within countries or cities with variable vaccination and similar exposure could also be appropriate although confounders will still be present.</p>
</div>
<div id="significance-84" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-84" class="anchor"></a>Significance</h4>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has been proven to be efficient in preventing childhood meningitis TB, but doesn’t prevent adult TB as efficiently. For this reason, several countries are now only recommending this vaccine for at-risk population only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has been shown to protect against several viruses and enhance innate immunity <span class="citation">[@doi:10.1016/j.cmi.2019.04.020]</span>, which could explain why it could protect against SARS-CoV-2 infection, but the exact mechanism is still unknown. <strong>Moreover, the efficiency of adult/older people vaccination and protection against Covid-19 still needs to be assessed.</strong> Regarding this, Australian researchers are starting a clinical trial of BCG vaccine for healthcare workers <span class="citation">[@url:https://www.mcri.edu.au/BRACE]</span>, to assess if it can protect them against Covid-19.</p>
</div>
<div id="credit-89" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-89" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium" class="section level3">
<h3 class="hasAnchor">
<a href="#non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium" class="anchor"></a>Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h3>
<p><span class="citation">[@doi:10.1101/2020.03.25.009084]</span></p>
<div id="keywords-92" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-92" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-89" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-89" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry) are expressed in olfactory epithelial (OE) cells, basal stem cells and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE sustentacular cells was similar to expression in the remainder of the non-nasal respiratory tract.</p></li>
</ul>
</div>
<div id="limitations-91" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-91" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is not sufficient to confirm infectivity of nasal passage, or to indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial injury leads to reduced sensitivity or increased inflammation and altered immune contexture drives neural/epithelial dysfunction. Will be critical to test this in CoVID-19 patient samples or mouse models.</p></li>
</ul>
</div>
<div id="significance-85" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-85" class="anchor"></a>Significance</h4>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells, and olfactory epithelia may serve as a viral reservoir after initial SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1, ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in other coronaviruses.</p></li>
</ul>
</div>
<div id="credit-90" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-90" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome peptide microarray. The peptide library was constructed with 966 linear peptides, each 15 amino acids long with a 5 amino acid overlap, based on the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1 nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides. Polyclonal antibodies showed some crossreactivity to other epitopes from membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a, and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies in more than 80 % COVID-19 patients were present in N, S and Orf3. It is shown that the receptor binding domain (RBD) resides on S protein and RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among six epitopes on S protein, structural analysis predicted that three epitopes were located at the surface and three epitopes were located inside of the protein. Furthermore, some IgM antibodies from 1 patient and IgG antibodies from 2 patients bound to the same epitope (residue 456-460, FRKSN) which resided within the RBD, and structural analysis determined that this epitope was located in the region of the RBD loop that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would have been informative to do the analysis over the course of the disease. The pattern of antibody recognition, especially on S protein, and the course of antibodies of different isotypes recognizing the same peptide might correlate to the clinical course in these patients. It would alos have been informative to analyze the presence of cross-reactive antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2, Wuhan-Hu-1 strain. This is a valuable information to design vaccines that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe lymphocytopenia, but the mechanism through which this depletion occurs has not been described. In order to characterize the cause and process of lymphocyte depletion in COVID-19 patients, the authors performed gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens and lymph nodes (hilar and subscapular) obtained from post-mortem autopsies of 6 patients with confirmed positive viremia and 3 healthy controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy, hemorrhaging, and necrosis with congestion of interstitial blood vessels and large accumulation of mononuclear cells and massive lymphocyte death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and subscapular LN, and capillaries of these secondary lymphoid organs expressed the ACE2 receptor and stain positive for the SARS-CoV-2 nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup> CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ) and in the marginal sinuses of LN, which suggests that these macrophages were positioned to encounter invading pathogens first and may contribute to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for persistent activation of lymphocytes via Fas::FasL interactions that would mediate activation-induced cell death (AICD). Indeed, the expression of Fas was significantly higher in virus-infected tissue than that of healthy controls, and TUNEL staining showed significant lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and TNF-α can also engage cellular apoptosis and necrosis, the authors interrogated the cytokine expression of the secondary lymphoid organs from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected splenic and lymph node tissues, compared to those of healthy controls, and immunofluorescent staining showed that IL-6 is primarily produced by the infected macrophages. <em>In vitro</em> infection of THP1 cells with SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em> expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively, the authors concluded that a combination of Fas up-regulation and IL-6 production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic MZ, subscapular LN, and the lining capillaries of the secondary lymphoid tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The findings point to the potential role of these macrophages in viral dissemination, immunopathology of these secondary lymphoid organs, hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6); moreover, the analysis of these samples using multiplexed immunohistochemistry and immunofluorescence do not necessarily provide the depth of unbiased interrogation needed to better identify the cell types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including single-cell experiments of spleen and LN from organ donors, do not indicate that ACE2 is expressed by macrophages; however, it remains possible that ACE2 expression may be triggered by type I IFN in COVID-19 patients. Importantly, the SARS-CoV-2 NP staining of the macrophages does not necessarily reflect direct infection of these macrophages; instead, positive staining only indicates that these macrophages carry SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct viral culture of macrophages isolated from the secondary lymphoid organs with SARS-CoV-2 is required to confirm the potential for direct infection of macrophages by SARS-CoV-2. Additionally, it is important to note that the low to negligible viremia reported in COVID-19 patients to-date does not favor a dissemination route via the blood, as suggested by this study, which would be necessary to explain the presence of virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized by cytokine storm in many COVID-19 patients. The contribution of this pathology to the overall fatality rate due to COVID-19, not even necessarily directly due to SARS-CoV-2 infection, is significant. A better understanding of the full effect and source of some of these major cytokines, like IL-6, as well as the deficient immune responses, like lymphocytopenia, is urgently needed. In this study, the authors report severe tissue damage in spleens and lymph nodes of COVID-19 patients and identify the role that CD169<sup>+</sup> macrophages may play in the hyperinflammation and lymphocytopenia that are both characteristic of the disease. It may, therefore, be important to note the effects that IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on splenic and LN function. It is important to note that similar observations of severe splenic and LN necrosis and inflammation in patients infected with SARS-CoV-1 further support the potential importance and relevance of this study.</p>
</div>
</div>
<div id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2" class="section level3">
<h3 class="hasAnchor">
<a href="#cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2" class="anchor"></a>Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.28.013672]</span></p>
<div id="keywords-93" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-93" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
</div>
<div id="main-findings-90" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-90" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture experiments to show that cigarette smoke causes a dose-dependent upregulation of ACE2 in mouse and human lungs (transplantation, tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+ goblet and SCGB1A1+ club cells; ACE2 presence in these cells is increased upon smoke exposure.</p></li>
</ul>
</div>
<div id="limitations-92" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-92" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including immune dysfunction, which can affect interpretation of CoV-2 susceptibility in these datasets. Ideally, analyses can control for major co-morbidities across smokers and non-smokers (immune suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral entry into goblet cells; future studies with air-liquid interface cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells from human epithelial lines, ex vivo samples or hACE2 mice will be very informative.</p></li>
</ul>
</div>
<div id="significance-86" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-86" class="anchor"></a>Significance</h4>
<ul>
<li><p>This study may partially explain why smokers are more likely to develop severe SARS-CoV-2 infections. Also, the reversibility of ACE2 expression upon smoking cessation suggests that quitting smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies implies CoV-2 susceptibility might be smoking-specific, and not fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of patients with CoV-2 co-morbidities such as hypertension <span class="citation">[@doi:10.1101/2020.03.21.20040261]</span>; these studies collectively paint a better picture of CoV-2 susceptibility before actual experiments can be carried out.</p></li>
</ul>
</div>
<div id="credit-91" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-91" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>Virus shedding patterns in nasopharyngeal and fecal specimens of1COVID-19 patients</strong></p>
<p>Zhang et al. 2020</p>
<p>Keywords: Viral shedding, testing specimens.</p>
<p><strong>Summary </strong></p>
<p>The authors tested feces, urine, plasma, nasal/throat swabs (respiratory samples) for SARS-CoV-2 virus load analysis by qPCR several days after illness onset (DAO).</p>
<ul>
<li><p>All plasma samples were negative (n=56).</p></li>
<li><p>Two urine samples were positive from patients that had severe disease and were positive at least 16-21 DAO (n=56).</p></li>
<li><p>10 of 12 cases (83.3%) tested positive for fecal samples.</p></li>
<li><p>14 of 21 cases (66.7%) were positive for respiratory samples.</p></li>
<li><p>Respiratory samples were positive for 21 DAO, nevertheless fecal samples were still positive.</p></li>
<li><p>Median duration of virus shedding was 10.0 days in respiratory samples</p></li>
<li><p>For fecal samples median shedding was 22.0 days for the feces</p></li>
<li><p>Viral titers of respiratory samples peaked at six to nine DAO. and at 14-18 DAO for fecal samples, and the highest virus titers at the peak was significantly higher for feces (105.8 copies/ml, mean 5623 copies/ml) than of respiratory samples (106.363 copies/ml, mean 2535 copies/ml).</p></li>
</ul>
<p>Importance of findings and caveats</p>
<p>Notably, these data reflect viral RNA and there is mention of infectious particles. Without these data, it is difficult to ascertain the significance of this paper. While these data demonstrate that the duration of fecal viral RNA shedding is significantly prolonged when compared to viral RNA shedding from the respiratory tract, it is critical for the field to ascertain if this represents infectious virions or not. If viable virus were to be isolated from the GI tract, that would indeed be very concerning for potential feco-oral transmission of SARS-COV2. As such, this paper does not establish this important fact.</p>
<p><em>Review by Jovani Catalan-Dibene and validated by Saurabh Mehandru as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis" class="section level3">
<h3 class="hasAnchor">
<a href="#the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis" class="anchor"></a>The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.28.20045765]</span></p>
<div id="keywords-94" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-94" class="anchor"></a>Keywords</h4>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
</div>
<div id="main-findings-91" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-91" class="anchor"></a>Main Findings</h4>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin Hospital (Wuhan University, China) were analyzed. The positive ratio was 78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N qPCR detection kit). There were no differences in the sensitivity of SARS-CO-V2 diagnosis in patients grouped according to disease severity. For example, IgG responses were detected in 93.18% of moderate cases, 100% of severe cases and 97.3% of critical cases. In sum, positive ratios were higher in antibody testing compared to RT-PCR detection, demonstrating a higher detection sensitivity of IgM-IgG testing for patients hospitalized with COVID-19 symptoms.</p>
</div>
<div id="limitations-93" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-93" class="anchor"></a>Limitations</h4>
<p>This analysis only included one-time point of 133 hospitalized patients, and the time from symptom onset was not described. There was no discussion about specificity of the tests and no healthy controls were included. It would be important to perform similar studies with more patients, including younger age groups and patients with mild symptoms as well as asymptomatic individuals. It is critical to determine how early after infection/symptom onset antibodies can be detected and the duration of this immune response.</p>
</div>
<div id="significance-87" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-87" class="anchor"></a>Significance</h4>
<p>The IgM-IgG combined testing is important to improve clinical sensitivity and diagnose COVID-19 patients. The combined antibody test shows higher sensitivity than individual IgM and IgG tests or nucleic acid-based methods, at least in patients hospitalized with symptoms.</p>
</div>
<div id="credit-92" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-92" class="anchor"></a>Credit</h4>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Clinical observations of low molecular weight heparin in relieving inflammation in COVID-19 patients: a retrospective cohort study</strong></p>
<p><strong><em>Key words</em></strong></p>
<p>Heparin, IL-6, cytokine storm, anti-inflammatory, anti-coagulant</p>
<p><strong><em>Key findings</em></strong></p>
<p>The authors investigated the use of a commercially available form of heparin, low molecular weight heparin (LMWH), as a therapeutic drug for patients with COVID-19. Previous studies showed that in addition to its anticoagulant properties, LMWH exerts anti-inflammatory effects by reducing the release of IL-6 and counteracting IL-6.</p>
<p>This was a retrospective single-center study conducted in Wuhan, China. Forty-two (42) hospitalized patients with coronavirus pneumonia were included, of which 21 underwent LMWH treatment (heparin group) and 21 did not (control). The general characteristics of the two groups of patients were statistically comparable. Both control and LMWH had the same hospitalization time and there were no critical cases in either group.</p>
<p>This study found that treatment with LMWH significantly reduced IL-6 levels in patients in the heparin group compared to the control group. However, LMWH treatment did not have an effect on the levels of other inflammatory factors: CRP, IL-2, IL-4, IL-10, TNF-α, and IFN-γ. Compared with the control group, patients in the heparin group had a significantly increased percentage of lymphocytes after treatment, further suggesting that LMWH treatment has anti-inflammatory effects and can reduce the lymphopenia associated with COVID-19.</p>
<p>Consistent with other studies in COVID-19 patients, they found that LMWH treatment can improve hypercoagulability. D-dimer and FDP levels (biomarkers of coagulation) in the heparin group significantly decreased from baseline after treatment, whereas there was no significant change in levels for the control group. Of note however, patients in the heparin group had a significantly higher level of D-dimer and FDP at baseline compared to the control group.</p>
<p><strong><em>Importance</em></strong></p>
<p>Many studies have shown that severely ill COVID-19 patients have significantly higher levels of IL-6 compared to patients with mild cases and it has been proposed that progression to severe disease may be caused by lymphopenia and cytokine storms. The anti-inflammatory effects of heparin may help prevent or reverse a cytokine storm caused by the virus and thus delay COVID-19 progression and improve overall condition in patients. The pleiotropic effects of heparin may have a greater therapeutic effect than compounds that are directed against a single target, such as an anti-IL-6 therapy. This is because COVID-19 patients commonly have complications such as coagulopathy and endothelial dysfunction leading to cardiac pathology that may be mitigated by heparin treatment (Li J, et.al; Wojnicz et.al).</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=44) and a single-center design. Double-blinded, randomized, placebo controlled clinical trials of LMWH treatment are needed to understand the possible benefit of the treatment. Additionally, this study was unable to control for the dose and days of treatment of LMWH. Identifying the correct dose and timing of LMWH is a matter of immediate interest. Of note, patients in the heparin group received two types of LMWH, enoxaparin sodium or nadroparin calcium, which have been reported to have differing anticoagulant activity. The use of different LMWHs in the heparin group warrants further explanation.</p>
<p>Another caveat of this study is that the levels of D-dimer and fibrinogen degradation products were significantly higher at baseline for patients in the heparin group compared to those in the control group. Therefore, it is difficult to decipher whether some of the positive effects of heparin treatment were due to its anti-coagulation effects or direct anti-inflammatory effects. Future studies are that delineate the anti-inflammatory functions of heparin independently of its anticoagulant properties in cases of COVID-19 would be useful.</p>
<p>Lastly, this study did not discuss any side-effects of heparin, such as the risk of bleeding. Moreover, coagulation can help to compartmentalize pathogens and reduce their invasion, therefore anticoagulant treatments like heparin may have risks and it remains to be determined which patients would benefit from this therapy.</p>
<p><strong>References:</strong></p>
<p>Li J, Li Y, Yang B, Wang H, Li L. Low-molecular-weight heparin treatment for acute lung injury/acute respiratory distress syndrome: a meta-analysis of randomized controlled trials. Int J Clin Exp Med 2018;11(2):414-422</p>
<p>Wojnicz R, Nowak J, Szyguła-Jurkiewicz B, Wilczek K, Lekston A, Trzeciak P, et al. Adjunctive therapy with low-molecular-weight heparin in patients with chronic heart failure secondary to dilated cardiomyopathy: oneyear follow-up results of the randomized trial. Am Heart J. 2006;152(4):713.e1-7</p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar epithelial cells and small intestine enterocytes, respiratory problems are the most common symptom after viral infection while intestinal symptoms are much less frequent. Thus, the authors here investigate the biology behind the observed protection of the intestinal epithelium from SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the small intestine, was shown to interact with human ACE2, with a binding affinity of 39.3 nM by biolayer interferometry (BLI). A blocking experiment using different doses of HD5 coating ACE2 showed that HD5 lowered viral spike protein S1 binding to ACE2. Further, a molecular dynamic simulation demonstrated a strong intermolecular interaction between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells were preincubated with HD5. Preincubation strongly reduced adherence of S1 to surface of cells. HD5 was effective at a concentration as low as 10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could have been tested to block ACE2-S1 binding in human lung epithelial cells as a potential treatment strategy. It would be useful to know whether HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency of viral entry and infection among ACE2-expressing cells and tissues. Specifically, the authors show that human defensin 5 produced in the small intestine is able to block binding between S1 and ACE2 necessary for viral entry into cells. The study provides a plausible explanation on why few patients show intestinal symptoms and suggests that patients with intestinal disease that decrease defensins’ production may be more susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a molecule to be exogenously administered to patients to prevent viral infection in lung epithelial cells.</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal, domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory animals (ferrets), as well as companion (cats and dogs), and domestic animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection with two SARS-CoV2 isolates, one from an environmental sample collected in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two ferrets from each group were euthanized on day 4 post infection (p.i.). AT day 4 p.i., viral RNA and infectious viruses were detected only in upper respiratory tract (nasal turbinate, upper palate, tonisls, but not in the trachea, lungs or other tissues. Viral RNA and virus titer in the remaining ferrets were monitored in nasal washes and rectal swabs on days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were detected in nasal washes until day 8 p.i. One ferret in each group developed fever and loss of appetite on days 10 and 12 p.i., however, viral RNA was practically undetactable. These two ferrets showed severe lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with CTan-h virus and three uninfected cats were placed in a cage adjacent to each of the infected cats to monitor respiratory droplet transmission. Viral RNA was detected in the upper respiratory organs from all infected cats and in one out of three exposed cats. All infected (inoculated and exposed) cats developed elevated antibodies against SARS-CoV2. Viral replication studies with juvenile cats (70-100 days) revealed massive lesions in the nasal and tracheal mucosa epithelium and lungs of two inoculated cats which died or were euthanized on day 3 p.i., and infection in one out of three exposed cats. These results indicated SARS-CoV2 could replicate in cats, that juvenile cats were more susceptible that adults, and theat SARS-CoV2 could be transmit via respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and housed with two uninoculated beagles in a room. Two virus inoculated dogs seroconverted, but others including two contact dogs were all seronegative for SARS-CoV2 and infectious virus was not detected in any swabs collected. Viral RNA was not detected in swabs from pigs, chickens, and ducks inoculated or contacted. These results indicated that dogs, pigs, chickens, and ducks might have low or no susceptibility to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and transmission in cats. Clinical and pathological analysis was not performed in cats, therefore the correlation of virus titer with symptoms severity in the adult and juvenile cats could not be determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously reported. It should be noted that this manuscript did not evaluate the transmission from cats to human. Nevertheless, it clearly showed higher susceptibility of ferrets and domestic cats to SARS-CoV-2. This data strongly indicates the need for surveillance of possible infection and transmission of SARS-CoV-2 by domestic cats.</p>
</div>
</div>
<div id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis" class="section level3">
<h3 class="hasAnchor">
<a href="#virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis" class="anchor"></a>Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h3>
<p>Li et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.31.019216]</span></p>
<div id="keywords-95" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-95" class="anchor"></a>Keywords</h4>
<ul>
<li>PBMC</li>
<li>virulence factors – interaction network – nsp9</li>
<li>nsp10 – NKRF</li>
</ul>
</div>
<div id="main-findings-92" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-92" class="anchor"></a>Main findings</h4>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong> (PPI) with two different approaches: genome wide yeast-two hybrid (Y2H) and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were characterized. A screen of <strong>viral-host PPI</strong> was also established by overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293 cells and purifying each protein complex. Interacting host proteins were then identified by liquid chromatography and tandem mass spectrometry. 251 cellular proteins were identified, such as subunits of ATPase, 40S ribosomal proteins, T complex proteins and proteasome related proteins, for a total of 631 viral-host PPI. Several interactions suggesting protein-mediated modulation of the immune response were identified, highlighting the multiple ways SARS-CoV-2 might reprogram infected cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs from healthy donors (n=6) with PBMC from COVID-19 patients with mild (n=22) or severe (n=13) symptoms. 220 proteins were found to be differentially expressed between <em>healthy donors and mild COVID-19 patients</em>, and a pathway analysis showed <strong>a general activation of the innate immune response.</strong> 553 proteins were differentially expressed between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them (95%) being downregulated in severe patients<em>.</em> Functional pathway analysis indicated a defect of T cell activation and function in severe COVID-19. There was also evidence suggesting reduced antibody secretion by B cells. Together, these results suggest a <strong>functional decline of adaptive immunity</strong>. A FACS analysis of PBMC from severe patients indicated higher levels of IL6 and IL8 but not IL17 compared to mild patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6 synthesis that was previously identified as interacting with SARS-Cov-2 nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459 cells, indicating that nsp9 and nsp10 may be directly involved in the induction of these pro-inflammatory cytokines. The authors finally argue that nsp9 and nsp10 represent potential drug targets to prevent over-production of IL6 and IL8 in infected cells, and reducing the over-activation of neutrophils.</p>
</div>
<div id="limitations-94" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-94" class="anchor"></a>Limitations</h4>
<p>First, the authors seem to have forgotten to include the extended data in the manuscript, and their proteomic data does not seem to be publicly available for the moment, which limits greatly our analysis of their results.</p>
<p>While this work provides important data on host and viral PPI, only 19 interactions were identified by Y2H system but 52 with co-IP. The authors do not comment about what could lead to such differences between the two techniques and they don’t specify whether they detected the same interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk PBMC. Consequently, protein differences likely reflect differences in cell populations rather than cell-intrinsic differences in protein expression. While this analysis is still interesting, a similar experiment performed on pre-sorted specific cell populations would allow measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to another SARS-CoV-2 interactome of host-viral PPI that had been published previously<sup>1</sup>. This study reported 332 host-virus PPI, but no interaction of viral proteins with NKRF was found. Some interactions were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However, the time point used to lyse the cells were different (40h previously vs 72h here), which could explain some of the differences.</p>
</div>
<div id="relevance" class="section level4">
<h4 class="hasAnchor">
<a href="#relevance" class="anchor"></a>Relevance</h4>
<p>The identification of many interactions between intra-viral and host-virus PPI provides an overview of host protein and pathways that are modulated by SARS-CoV-2, which can lead to the identification of potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2 induce an over-expression of IL6 and IL8 by lung epithelial cells, which recruits neutrophils and could lead to an excess in lung infiltration. Nsp9 has been shown to be essential for viral replication for SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>. Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help to develop drug inhibitors if this protein is further confirmed as being important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386. <a href="doi:10.1101/2020.03.22.002386" class="uri">doi:10.1101/2020.03.22.002386</a></p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW. Severe acute respiratory syndrome coronavirus nsp9 dimerization is essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018. <a href="doi:10.1128/JVI.01505-08" class="uri">doi:10.1128/JVI.01505-08</a></p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020. <a href="doi:10.1101/2020.03.28.013920" class="uri">doi:10.1101/2020.03.28.013920</a></p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2 genomes in order to identify conserved linear and conformational B cell epitopes found on the spike (S), envelope (E), and membrane (M) proteins. The characterization of the surface proteins in this study began with an assessment of the peptide sequences in order to identify hydrophilicity indices and protein instability indices using the Port-Param tool in ExPASy. All three surface proteins were calculated to have an instability score under 40 indicating that they were stable. Linear epitopes were identified on the basis of surface probability and antigenicity, excluding regions of glycosylation. Using BepiPred 2.0 (with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51), 4 linear B cell epitopes were predicted for the S protein, 1 epitope for the E protein, and 1 epitope for the M protein. For structural analysis, SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for input into the SWISS-MODEL server in order to generate three-dimensional structural models for the assessment of conformational epitopes. Using Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1 conformational epitope was identified for the S protein and 1 epitope was identified for the E protein, both of which are accessible on the surface of the virus. Finally, the Consurf Server was used to assess the conservation of these epitopes. All epitopes were conserved across the published SARS-CoV-2 genomes and one epitope of the spike protein was predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential linear and conformational B cell epitopes, the translational value of the epitopes described still needs extensive experimental validation to ascertain whether these elicit a humoral immune response. The conformational epitope analyses are also limited by the fact that they are based off of predicted 3D structure from homology comparisons and not direct crystal structures of the proteins themselves. Additionally, since there was not a published M protein with a high homology to SARS-CoV-2, no conformational epitopes were assessed for this protein. Finally, while evolutionary conservation is an important consideration in understanding the biology of the virus, conservation does not necessarily imply that these sites neutralize the virus or aid in non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes induce effective antibody responses to the virus, the epitopes described can be used for the development of treatments and vaccines as well as better characterize the viral structure to more deeply understand pathogenesis.</p>
<p><strong>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</strong></p>
<p><strong>ACE2; lungs; smoking; COPD; asthma; SARS-Cov; IL-1; IL-10; IL-6; IL-8;</strong></p>
<p><strong>SUMMARY:</strong> The authors used bioinformatics tools to identify features of ACE2 expression in the lungs of different patent groups: healthy, smokers, patients with chronic airway disease (i.e., COPD) or asthma. They used gene expression data publicly available from GEO that included lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small airway epithelium of long-term smokers vs non-smokers (p&lt;0.001). Consistently, there was a trend of higher ACE2 expression in the bronchial airway epithelial cells 24h post-acute smoking exposure (p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h post SARS-Cov infection (p=0.026; n=3 at each time point) was also detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high and low ACE2 expression groups. “High” ACE2 expression was associated with the following GO pathways: innate and adaptive immune responses, B cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8 cytokines. The authors speculate that a high basal ACE2 expression will increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6 cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell activation and T-cell cytokine production. It is unclear whether those changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates in epithelial cells infected with SARS-Cov infection. They state that in healthy donors ACE2 expression did not correlate with the immune cell infiltration. However, in SARS-Cov samples, at 48h they found that ACE2 correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs. Again, while authors claim significance, the corresponding correlation coefficients and p-values are not presented in the text or figures. In addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1 and PRKCD were found to be central (Hub proteins) in the regulating network of cytokine secretion after coronavirus infection. Authors conclude this indicates that these molecules were critically important in ACE2-induced inflammatory response. Additionally, authors speculate that the increased expression of ACE2 affected RPS3 and SRC, which were the two hub genes involved in viral replication and inflammatory response.</p>
<p><strong>LIMITATIONS:</strong> The methods section is very limited and does not describe any of the statistical analyses; and description of the construction of the regulatory protein networks is also limited. For the findings in Figures 2 authors claim significance, which is not supported by p-values or coefficients. For the sample selection, would be useful if sample sizes and some of the patients’ demographics (e.g. age) were described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it is not clear what was the cut off for ‘high’ expression and how it was determined. Additionally, further laboratory studies are warranted to confirm that high ACE2 gene expression would have high correlation with the amount of ACE2 protein on cell surface. For the GO pathway analysis significance was set at p&lt;0.05, but not adjusted for multiple comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only included data on SARS-Cov and any conclusions about SARS-CoV2 are limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two might not be cited correctly: GSE37758 (<em>“A spergillus niger: Control (fructose) vs. steam-exploded sugarcane induction (SEB)”</em> was used in this paper as “lung tissue” data) and GSE14700 (“<em>Steroid Pretreatment of Organ Donors to Prevent Postischemic Renal Allograft Failure: A Randomized, Controlled Tria</em>l” – was used as SARS-Cov infection data).</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell membrane during infection. The spike (S) protein plays an essential role in this step (1). This work sheds light on the particularities of the S protein from SARS-CoV2 and compares it to previously existing coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for hACE2 compared to SARS-CoV. Previous work by this group had established the potential of a previous version of the peptide (EK1) in reducing the infectivity of several coronavirus by blocking the S protein interactions with the host cell (2). In this work, authors present an improved EK1C4 version of this peptide. In vitro experiments on human cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2 and other coronavirus. Administration of EK1C4 in prophylaxis or shortly after exposure to the HCoV-OC43 coronavirus protected newborn mice from virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be related to its increased infectivity compared to CoV-S. However, a previous report suggested that the enhanced cell-to-cell fusion mediated by this furin cleavage might not necessarily translate into enhanced viral infectivity (3). The mutation of this site, or inhibition of furin in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if this mechanism is actually involved in SARS-CoV2 pathogenesis, and potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is based on the coronavirus HCoV-OC43. Further work should aim to establish the potential of EK1C4 for prevention of SARS-CoV-2 replication in more relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is, however, acknowledged, that experimentation in vitro included both SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been previously examined (2), the potential side effects and pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a paramount element of epidemic control, specially in the time before a vaccine can be readily available. Similar strategies have been followed in the past in response to threats of this nature (4,5). This work’s relevance is limited by the absence of relevant animal models of SARS-CoV-2 infection. Further pharmacological analyses would also be necessary to ensure the safety of EK1C4 in animal models before a potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike protein and acquisition of fusion competence. Vol. 4, Viruses. Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S. Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide That Targets Viral Attachment to Cells. J Virol. 2006 Dec 15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human antimicrobial peptides as therapeutics for viral infections. Vol. 11, Viruses. MDPI AG; 2019.</p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p>Developed by First Last.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
