<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title> • covidpreprints</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="">
<meta property="og:description" content="covidpreprints">
<meta property="og:image" content="https://covidpreprints.com/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-164190709-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-164190709-1');
</script>
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">covidpreprints</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/timeline.html">Timeline</a>
</li>
<li>
  <a href="../articles/further_info.html">Further information</a>
</li>
<li>
  <a href="../articles/resources.html">Resources</a>
</li>
<li>
  <a href="../articles/about.html">About</a>
</li>
<li>
  <a href="../articles/preprint_plot.html">Preprints posted to bioRxiv and medRxiv</a>
</li>
<li>
  <a href="../articles/mount_sinai.html">Mount Sinai peer-reviews of COVID-19 preprints</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/coatesj/covidpreprints/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="mount_sinai_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip></h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/coatesj/covidpreprints/blob/master/vignettes/mount_sinai.Rmd"><code>vignettes/mount_sinai.Rmd</code></a></small>
      <div class="hidden name"><code>mount_sinai.Rmd</code></div>

    </div>

    
    
<p>The following is a collection of the Mount Sinai Immunology researchers (<span class="citation">@MountSinaiNYC</span>). We thank the team for their important efforts in reviewing critical literature during such a turbulent period and for allowing us to link to these reviews. See <a href="https://observablehq.com/@ismms-himc/covid-19-sars-cov-2-preprints-from-medrxiv-and-biorxiv" class="uri">https://observablehq.com/@ismms-himc/covid-19-sars-cov-2-preprints-from-medrxiv-and-biorxiv</a> for more. Please use Crt+F to search.</p>
<div id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody" class="section level3">
<h3 class="hasAnchor">
<a href="#potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody" class="anchor"></a>Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h3>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation">[@doi:10.1101/2020.01.28.923011]</span></p>
<div id="keywords" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords" class="anchor"></a>Keywords</h4>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
</div>
<div id="summary" class="section level4">
<h4 class="hasAnchor">
<a href="#summary" class="anchor"></a>Summary</h4>
<p>Considering the relatively high identity of the receptor binding domain (RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this study aims to assess the cross-reactivity of several anti-SARS-CoV monoclonal antibodies with 2019-nCoV. The results showed that the SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
</div>
<div id="main-findings" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings" class="anchor"></a>Main Findings</h4>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in silico</em> and compared with the SARS-CoV RBD structure. The models predicted very similar RBD-ACE2 interactions for both viruses. The binding capacity of representative SARS-CoV-RBD specific monoclonal antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was then investigated by ELISA and their binding kinetics studied using biolayer interferometry. The analysis showed that only CR3022 was able to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did not interfere with ACE2 binding. Antibodies m396 and CR3014, which target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein.</p>
</div>
<div id="limitations" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations" class="anchor"></a>Limitations</h4>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus residues, which drastically impact the cross-reactivity of antibodies described for other B beta-coronaviruses, including SARS-CoV. This study claims that CR3022 antibody could be a potential candidate for therapy. However, none of the antibodies assayed in this work showed cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for the replication of this virus. Furthermore, neutralization assays with 2019-nCoV virus or pseudovirus were not performed. Although the use of neutralizing antibodies is an interesting approach, these results suggest that it is critical the development of novel monoclonal antibodies able to specifically bind 2019-nCoV spike protein.</p>
</div>
<div id="credit" class="section level4">
<h4 class="hasAnchor">
<a href="#credit" class="anchor"></a>Credit</h4>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title</strong>: Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig</p>
<p><strong>Keywords</strong>: ACE2, Ig-like protein, SARS-CoV neutralization</p>
<p><strong>Summary</strong>:</p>
<p>Angiotensin-converting enzyme 2 ACE2 is a negative regulator of the renin-angiotensin system. In the lung tissues, ACE2 is expressed on lung epithelial cells (AT2 cells) and has been identified as a receptor for SARS-CoV-1 and SARS-CoV-2<span class="math display">\[1\]</span>⁠.</p>
<p>Administration of recombinant human ACE2 has been shown to protect mice from severe acute lung injury induced by acid aspiration or sepsis and lethal avian influenza H5N1<span class="math display">\[2,3\]</span>⁠⁠.</p>
<p>Human recombinant ACE has already been shown in animal models and humans to have a fast clearance rate with a half-life of only a few hours<span class="math display">\[4[]{#ADDIN_CSL_CITATION_%7B%22citationItems%2 .anchor}\]</span>⁠, thereby limiting its therapeutic potential.</p>
<p>To address this protein stability limitation, the authors generated a fusion protein that links the extracellular domain of human ACE2 to the Fc domain of human IgG1. This fusion protein was shown to have a prolonged half-life and neutralize viruses pseudotyped with the S glycoprotein of both of SARS-CoV and 2019-nCoV in vitro, thus providing a potential therapeutic for COVID-19.</p>
<p><strong>Critical analysis</strong>: ACE2-Ig fusion proteins are promising but their ability to neutralize the virus and reduce viral load remains be tested with intact Coronaviruses in vitro and in animals before being tested in clinical trials.</p>
<p><strong>Implications for current epidemic</strong>: If neutralizing capacity can be validated with 2019-nCov viruses, this novel drug target provides a promising therapeutic strategy for the treatment of COVID-19 patients. Nonetheless, the authors mention potential cardiovascular side-effects stemming from the role of ACE2 in the renin-engiotensin system. These will need to be examined prior to the initiation of a phase I clinical trial.</p>
<p><strong>References</strong>:</p>
<p>1. Kuba K, Imai Y, Rao S, Jiang C, Penninger JM: <strong>Lessons from SARS: Control of acute lung failure by the SARS receptor ACE2</strong>. <em>J Mol Med</em> 2006, <strong>84</strong>:814–820.</p>
<p>2. Imai Y, Kuba K, Rao S, Huan Y, Guo F, Guan B, Yang P, Sarao R, Wada T, Leong-Poi H, et al.: <strong>Angiotensin-converting enzyme 2 protects from severe acute lung failure</strong>. <em>Nature</em> 2005, <strong>436</strong>:112–116.</p>
<p>3. Zou Z, Yan Y, Shu Y, Gao R, Sun Y, Li X, Ju X, Liang Z, Liu Q, Zhao Y, et al.: <strong>Angiotensin-converting enzyme 2 protects from lethal avian influenza A H5N1 infections</strong>. <em>Nat Commun</em> 2014, <strong>5</strong>:3594.</p>
<p>4. Haschke M, Schuster M, Poglitsch M, Loibner H, Salzberg M, Bruggisser M, Penninger J, Krähenbühl S: <strong>Pharmacokinetics and pharmacodynamics of recombinant human angiotensin-converting enzyme 2 in healthy human subjects</strong>. <em>Clin Pharmacokinet</em> 2013, <strong>52</strong>:783–792.</p>
<p><strong>Title</strong>: Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig</p>
<p><strong>Keywords</strong>: ACE2, Ig-like protein, SARS-CoV neutralization</p>
<p><strong>Summary</strong>:</p>
<p>Angiotensin-converting enzyme 2 ACE2 is a negative regulator of the renin-angiotensin system. In the lung tissues, ACE2 is expressed on lung epithelial cells (AT2 cells) and has been identified as a receptor for SARS-CoV-1 and SARS-CoV-2<span class="math display">\[1\]</span>⁠.</p>
<p>Administration of recombinant human ACE2 has been shown to protect mice from severe acute lung injury induced by acid aspiration or sepsis and lethal avian influenza H5N1<span class="math display">\[2,3\]</span>⁠⁠.</p>
<p>Human recombinant ACE has already been shown in animal models and humans to have a fast clearance rate with a half-life of only a few hours<span class="math display">\[4[]{#ADDIN_CSL_CITATION_%7B%22citationItems%2 .anchor}\]</span>⁠, thereby limiting its therapeutic potential.</p>
<p>To address this protein stability limitation, the authors generated a fusion protein that links the extracellular domain of human ACE2 to the Fc domain of human IgG1. This fusion protein was shown to have a prolonged half-life and neutralize viruses pseudotyped with the S glycoprotein of both of SARS-CoV and 2019-nCoV in vitro, thus providing a potential therapeutic for COVID-19.</p>
<p><strong>Critical analysis</strong>: ACE2-Ig fusion proteins are promising but their ability to neutralize the virus and reduce viral load remains be tested with intact Coronaviruses in vitro and in animals before being tested in clinical trials.</p>
<p><strong>Implications for current epidemic</strong>: If neutralizing capacity can be validated with 2019-nCov viruses, this novel drug target provides a promising therapeutic strategy for the treatment of COVID-19 patients. Nonetheless, the authors mention potential cardiovascular side-effects stemming from the role of ACE2 in the renin-engiotensin system. These will need to be examined prior to the initiation of a phase I clinical trial.</p>
<p><strong>References</strong>:</p>
<p>1. Kuba K, Imai Y, Rao S, Jiang C, Penninger JM: <strong>Lessons from SARS: Control of acute lung failure by the SARS receptor ACE2</strong>. <em>J Mol Med</em> 2006, <strong>84</strong>:814–820.</p>
<p>2. Imai Y, Kuba K, Rao S, Huan Y, Guo F, Guan B, Yang P, Sarao R, Wada T, Leong-Poi H, et al.: <strong>Angiotensin-converting enzyme 2 protects from severe acute lung failure</strong>. <em>Nature</em> 2005, <strong>436</strong>:112–116.</p>
<p>3. Zou Z, Yan Y, Shu Y, Gao R, Sun Y, Li X, Ju X, Liang Z, Liu Q, Zhao Y, et al.: <strong>Angiotensin-converting enzyme 2 protects from lethal avian influenza A H5N1 infections</strong>. <em>Nat Commun</em> 2014, <strong>5</strong>:3594.</p>
<p>4. Haschke M, Schuster M, Poglitsch M, Loibner H, Salzberg M, Bruggisser M, Penninger J, Krähenbühl S: <strong>Pharmacokinetics and pharmacodynamics of recombinant human angiotensin-converting enzyme 2 in healthy human subjects</strong>. <em>Clin Pharmacokinet</em> 2013, <strong>52</strong>:783–792.</p>
</div>
</div>
<div id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov" class="section level3">
<h3 class="hasAnchor">
<a href="#integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov" class="anchor"></a>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h3>
<p>He et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.20020206]</span></p>
<div id="keywords-1" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-1" class="anchor"></a>Keywords</h4>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
</div>
<div id="main-findings-1" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-1" class="anchor"></a>Main Findings</h4>
<p>The authors used bioinformatics tools to identify features of ACE2 expression in the lungs of different patent groups: healthy, smokers, patients with chronic airway disease (i.e., COPD) or asthma. They used gene expression data publicly available from GEO that included lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small airway epithelium of long-term smokers vs non-smokers (p&lt;0.001). Consistently, there was a trend of higher ACE2 expression in the bronchial airway epithelial cells 24h post-acute smoking exposure (p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h post SARS-Cov infection (p=0.026; n=3 at each time point) was also detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high and low ACE2 expression groups. “High” ACE2 expression was associated with the following GO pathways: innate and adaptive immune responses, B cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8 cytokines. The authors speculate that a high basal ACE2 expression will increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6 cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell activation and T-cell cytokine production. It is unclear whether those changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates in epithelial cells infected with SARS-Cov infection. They state that in healthy donors ACE2 expression did not correlate with the immune cell infiltration. However, in SARS-Cov samples, at 48h they found that ACE2 correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs. Again, while authors claim significance, the corresponding correlation coefficients and p-values are not presented in the text or figures. In addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1 and PRKCD were found to be central (Hub proteins) in the regulating network of cytokine secretion after coronavirus infection. Authors conclude this indicates that these molecules were critically important in ACE2-induced inflammatory response. Additionally, authors speculate that the increased expression of ACE2 affected RPS3 and SRC, which were the two hub genes involved in viral replication and inflammatory response.</p>
</div>
<div id="limitations-1" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-1" class="anchor"></a>Limitations</h4>
<p>The methods section is very limited and does not describe any of the statistical analyses; and description of the construction of the regulatory protein networks is also limited. For the findings in Figures 2 authors claim significance, which is not supported by p-values or coefficients. For the sample selection, would be useful if sample sizes and some of the patients’ demographics (e.g. age) were described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it is not clear what was the cut off for ‘high’ expression and how it was determined. Additionally, further laboratory studies are warranted to confirm that high ACE2 gene expression would have high correlation with the amount of ACE2 protein on cell surface. For the GO pathway analysis significance was set at p&lt;0.05, but not adjusted for multiple comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only included data on SARS-Cov and any conclusions about SARS-CoV2 are limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two might not be cited correctly: GSE37758 (“A spergillus niger: Control (fructose) vs. steam-exploded sugarcane induction (SEB)”" was used in this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment of Organ Donors to Prevent Postischemic Renal Allograft Failure: A Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
</div>
<div id="credit-1" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-1" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus" class="anchor"></a>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h3>
<p>Liang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.20020289]</span></p>
<div id="keywords-2" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-2" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
</div>
<div id="main-findings-2" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-2" class="anchor"></a>Main Findings</h4>
<p>This study examined the incidence of diarrhea in patients infected with SARS-CoV-2 across three recently published cohorts and found that there are statistically significant differences by Fisher’s exact test. They report that this could be due to subjective diagnosis criterion for diarrhea or from patients first seeking medical care from gastroenterologist. In order to minimize nosocomial infections arising from unsuspected patients with diarrhea and gain comprehensive understanding of transmission routes for this viral pathogen, they compared the transcriptional levels of ACE2 of various human tissues from NCBI public database as well as in small intestine tissue from CD57BL/6 mice using single cell sequencing. They show that ACE2 expression is not only increased in the human small intestine, but demonstrate a particular increase in mice enterocytes positioned on the surface of the intestinal lining exposed to viral pathogens. Given that ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate diarrhea, their data suggests the small intestine as a potential transmission route and diarrhea as a potentially underestimated symptom in COVID19 patients that must be carefully monitored. Interestingly, however, they show that ACE2 expression level is not elevated in human lung tissue.</p>
</div>
<div id="limitations-2" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-2" class="anchor"></a>Limitations</h4>
<p>Although this study demonstrates a statistical difference in the incidence of diarrhea across three separate COVID19 patient cohorts, their conclusions are limited by a small sample size. Specifically, the p-value computed by Fisher’s exact test is based on a single patient cohort of only six cases of which 33% are reported to have diarrhea, while the remaining two larger cohorts with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively. Despite showing significance, they would need to acquire larger sample sizes and cohorts to minimize random variability and draw meaningful conclusions. Furthermore, they do not address why ACE2 expression level is not elevated in human lung tissue despite it being a major established route of transmission for SARS-CoV-2. It could be helpful to validate this result by looking at ACE2 expression in mouse lung tissue. Finally, although this study is descriptive and shows elevated ACE2 expression in small intestinal epithelial cells, it does not establish a mechanistic link to SARS-CoV-2 infection of the host. Overall, their claim that infected patients exhibiting diarrhea pose an increased risk to hospital staff needs to be further substantiated.</p>
</div>
<div id="significance" class="section level4">
<h4 class="hasAnchor">
<a href="#significance" class="anchor"></a>Significance</h4>
<p>This study provides a possible transmission route and a potentially underappreciated clinical symptom for SARS-CoV-2 for better clinical management and control of COVID19.</p>
</div>
<div id="credit-2" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-2" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection" class="anchor"></a>Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h3>
<p>Chai et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.03.931766]</span></p>
<div id="keywords-3" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-3" class="anchor"></a>Keywords</h4>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
</div>
<div id="summary-1" class="section level4">
<h4 class="hasAnchor">
<a href="#summary-1" class="anchor"></a>Summary</h4>
<p>Using both publicly available scRNA-seq dataset of liver samples from colorectal patients and scRNA-sequencing of four liver samples from healthy volunteers, the authors show that ACE2 is significantly enriched in the majority of cholangiocytes (59.7 %) but not in hepatocytes (2.6%).</p>
</div>
<div id="main-findings-3" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-3" class="anchor"></a>Main Findings</h4>
<p>Using bioinformatics approaches of RNASeq analysis, this study reveals that ACE2 dominates in cholangiocytes and is present at very low levels in hepatocytes.</p>
</div>
<div id="limitations-3" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-3" class="anchor"></a>Limitations</h4>
<p>The study does not provide mechanistic insights into how SARS-CoV-2 can infect and replicate in cholangiocytes and the types of intrinsic anti-viral responses induced by cholangiocytes when infected. In addition, because the study relies on the assumption that SARS-CoV-2 infects cells only through ACE2, it cannot discount the possibility that the virus can infect hepatocytes through mechanisms other than ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were performed on healthy liver samples, one cannot draw any definitive conclusions about gene expression states (including ACE2 expression in liver cell types) in system-wide inflammatory contexts.</p>
</div>
<div id="significance-1" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-1" class="anchor"></a>Significance</h4>
<p>This article with other studies on liver damage in COVID patients suggests that liver damage observed in COVID patients is more due to inflammatory cytokines than direct infection of the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which was demonstrated by human liver ductal organoid study (<span class="citation">[@doi:10.1101/2020.03.16.990317]</span>), published clinical data show no significant increase in bile duct injury related indexes (i.e. alkaline phosphatase, gamma-glutamyl transpeptidase and total bilirubin). In sum, it underscores the importance of future studies characterizing cellular responses of extra-pulmonary organs in the context of COVID or at least in viral lung infections..</p>
</div>
<div id="credit-3" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-3" class="anchor"></a>Credit</h4>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism" class="anchor"></a>ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h3>
<p>Wang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.05.20020545]</span></p>
<div id="keywords-4" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-4" class="anchor"></a>Keywords</h4>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
</div>
<div id="main-findings-4" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-4" class="anchor"></a>Main Findings</h4>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
</div>
<div id="limitations-4" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-4" class="anchor"></a>Limitations</h4>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
</div>
<div id="significance-2" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-2" class="anchor"></a>Significance</h4>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
</div>
<div id="credit-4" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-4" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice" class="section level3">
<h3 class="hasAnchor">
<a href="#the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice" class="anchor"></a>The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h3>
<p>Bao et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.07.939389]</span></p>
<div id="keywords-5" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-5" class="anchor"></a>Keywords</h4>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
</div>
<div id="main-findings-5" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-5" class="anchor"></a>Main Findings</h4>
<p>Using a transgenic human Angiotensin-converting enzyme 2 (hACE2) mouse that has previously been shown susceptible to infection by SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain coronavirus. The model includes interstitial hyperplasia in lung tissue, moderate inflammation in bronchioles and blood vessels, and histology consistent with viral pneumonia at 3 days post infection. Wildtype did not experience these symptoms. In addition, viral antigen and hACE2 receptor were found to co-localize the lung by immunofluorescence 3-10 days post infection only in the hACE2 infected mice.</p>
</div>
<div id="limitations-5" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-5" class="anchor"></a>Limitations</h4>
<p>The characterization of the infection remains incomplete, as well as lacking characterization of the immune response other than the presence of a single antiviral antibody. Though they claim to fulfill Koch’s postulates, they only isolate the virus and re-infect Vero cells, rather than naive mice.</p>
</div>
<div id="significance-3" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-3" class="anchor"></a>Significance</h4>
<p>This paper establishes a murine model for 2019-nCoV infection with symptoms consistent with viral pneumonia. Though not fully characterized, this model allows in vivo analysis of viral entry and pathology that is important for the development of vaccines and antiviral therapeutics.</p>
</div>
<div id="credit-5" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-5" class="anchor"></a>Credit</h4>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="caution-on-kidney-dysfunctions-of-2019-ncov-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#caution-on-kidney-dysfunctions-of-2019-ncov-patients" class="anchor"></a>Caution on Kidney Dysfunctions of 2019-nCoV Patients</h3>
<p>Li et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span></p>
<div id="keywords-6" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-6" class="anchor"></a>Keywords</h4>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine, proteinuria, albuminuria, CT</p>
</div>
<div id="main-findings-6" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-6" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators of the kidney–including urine protein, blood urea nitrogen (BUN), plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the first day of admission, and 63% developed proteinuria during their stay in hospital; and 19% of patients had high plasma creatinine, especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal kidney damage; indicate that inflammation and edema of the renal parenchyma very common.</p></li>
</ul>
</div>
<div id="limitations-6" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-6" class="anchor"></a>Limitations</h4>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in blood/plasma/urine. Will be worth correlating disease progression with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only rationale for viral-damage in kidney and resultant pathologic immune response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
</div>
<div id="significance-4" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-4" class="anchor"></a>Significance</h4>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the idea that renal impairment/injury is a key risk factor in 2019-nCoV patients similar to what has been reported for SARS-CoV<span class="citation">[@doi:10.1111/j.1523-1755.2005.67130.x]</span>; this may be one of the major causes of virally-induced damage and contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder epithelia <span class="citation">[@doi:10.1101/2020.02.08.939892]</span> support these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying potential interventions including continuous renal replacement therapies (CRRT) for protecting kidney functions as early as possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
</div>
<div id="credit-6" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-6" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus" class="anchor"></a>Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h3>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.08.939553]</span></p>
<div id="keywords-7" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-7" class="anchor"></a>Keywords</h4>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
</div>
<div id="main-findings-7" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-7" class="anchor"></a>Main Findings</h4>
<p>This study harnesses bioinformatic profiling to predict the potential of COV2 viral proteins to be presented on MHC I and II and to form linear B-cell epitopes. These estimates suggest a T-cell antigenic profile distinct from SARS-CoV or MERS-CoV, identify focused regions of the virus with a high density of predicted epitopes, and provide preliminary evidence for adaptive immune pressure in the genetic evolution of the virus.</p>
</div>
<div id="limitations-7" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-7" class="anchor"></a>Limitations</h4>
<p>While the study performs a comprehensive analysis of potential epitopes within the virus genome, the analysis relies solely on bioinformatic prediction to examine MHC binding affinity and B-cell epitope potential and does not capture the immunogenicity or recognition of these epitopes. Future experimental validation in data from patients infected with SARS-CoV-2 will be important to validate and refine these findings. Thus some of the potential conclusions stated, including viral evolution toward lower immunogenicity or a dominant role for CD4+ T-cells rather than CD8+ T-cells in viral clearance, require further valiadtion.</p>
</div>
<div id="significance-5" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-5" class="anchor"></a>Significance</h4>
<p>These findings may help direct peptide vaccine design toward relevant epitopes and provide intriguing evidence of viral evolution in response to immune pressure.</p>
</div>
<div id="credit-7" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-7" class="anchor"></a>Credit</h4>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection" class="anchor"></a>Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h3>
<p>Lin et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.08.939892]</span></p>
<div id="keywords-8" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-8" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
</div>
<div id="main-findings-8" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-8" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV patients, authors used published kidney and bladder cell atlas data (GSE131685, GSE108097; 3 healthy donors each) as well as an unpublished kidney single-cell RNA-Seq data (in-house from 2 transplant donors) to evaluate ACE2 gene expressions in all cell types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of proximal tubule cells of kidney, with 5%-15% of both straight and convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial cells, noting expression from around 1.5% of cells in the outer layer umbrella cells of the bladder epithelium and decreasing in the basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not express ACE2.</p></li>
</ul>
</div>
<div id="limitations-8" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-8" class="anchor"></a>Limitations</h4>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other genes) from a small healthy-donor dataset, and will benefit from supporting data in (expired) patient samples to show functional viral damage. ACE2 transcript does not necessarily translate to viral permissiveness in kidney/bladder epithelia or cytokine release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to analyze kidney/bladder epithelial ACE2 expression under inflammatory conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell identification would be useful in these datasets.</p></li>
</ul>
</div>
<div id="significance-6" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-6" class="anchor"></a>Significance</h4>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal tubules and in bladder umbrella cells <span class="citation">[@doi:10.1002/path.1570]</span>, such cells in direct contact with viral particles are likely to be highly sensitive to viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of patients and associate with higher mortality <span class="citation">[@doi:10.1111/j.1523-1755.2005.67130.x; @doi:10.7326/m13-2486]</span>, thus worth understanding kidney damage and resultant immune response in SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and resultant inflammatory responses in these tissue in addition to reported infectivity in the lung and digestive system, which is supported by clinical data showing acute and early kidney complications in 2019-nCoV patients <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
</div>
<div id="credit-8" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-8" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage" class="section level3">
<h3 class="hasAnchor">
<a href="#neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage" class="anchor"></a>Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h3>
<p>Liu et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.10.20021584]</span></p>
<div id="keywords-9" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-9" class="anchor"></a>Keywords</h4>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
</div>
<div id="main-findings-9" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-9" class="anchor"></a>Main Findings</h4>
<p>This study aimed to find prognostic biomarkers of COVID-19 pneumonia severity. Sixty-one (61) patients with COVID-19 treated in January at a hospital in Beijing, China were included. On average, patients were seen within 5 days from illness onset. Samples were collected on admission; and then patients were monitored for the development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17) according to symptoms on admission and compared for different clinical/laboratory features. “Moderate/severe” patients were significantly older (median of 56 years old, compared to 41 years old). Whereas comorbidies rates were largely similar between the groups, except for hypertension, which was more frequent in the severe group (p= 0.056). ‘Severe’ patients had higher counts of neutrophils, and serum glucose levels; but lower lymphocyte counts, sodium and serum chlorine levels. The ratio of neutrophils to lymphocytes (NLR) was also higher for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial infections (and antibiotic treatment) and received more intensive respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the best predictors of the severe disease. Predictive cutoffs for a severe illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
</div>
<div id="limitations-9" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-9" class="anchor"></a>Limitations</h4>
<p>Identification of early biomarkers is important for making clinical decisions, but large sample size and validation cohorts are necessary to confirm findings. It is worth noting that patients classified as “mild” showed pneumonia by imaging and fever, and in accordance with current classifications this would be consistent with “moderate” cases. Hence it would be more appropriate to refer to the groups as “moderate” vs “severe/critical”. Furthermore, there are several limitations that could impact the interpretation of the results: e.g. classification of patients was based on symptoms presented on admission and not based on disease progression, small sample size, especially the number of ‘severe’ cases (with no deaths among these patients). Given the small sample size, the proposed NLR and age cut offs might not hold for a slightly different set of patients. For example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR were 3.2 and 5.5, respectively <span class="citation">[@doi:10.1093/cid/ciaa248]</span>.</p>
</div>
<div id="credit-9" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-9" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp" class="section level3">
<h3 class="hasAnchor">
<a href="#characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp" class="anchor"></a>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h3>
<p>Wan et. al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.10.20021832]</span></p>
<div id="keywords-10" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-10" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
</div>
<div id="main-findings-10" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-10" class="anchor"></a>Main Findings</h4>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients with mild disease and 21 with severe disease. CD8+T cells and CD4+T cells were significantly reduced in both cohort. particularly in severe patients. The cytokines IL6 and IL10 were significantly elevated in severe patients as compared to mild. No significant differences were observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and cytokine levels of IL6 and IL10 can be used to predict progression of disease from Mild to severe Cov-2 infection.</p>
</div>
<div id="limitations-10" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-10" class="anchor"></a>Limitations</h4>
<p>The study demonstrates in a limited cohort similar associations to several other reported studies. The authors didn’t compare the changes in lymphocyte and cytokine with healthy individual (Covid-19 Negative) rather used an internal standard value. The recently preprint in LANCET shows The degree of lymphopenia and a pro-inflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity <span class="citation">[@doi:10.2139/ssrn.3539682]</span>.</p>
</div>
<div id="significance-7" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-7" class="anchor"></a>Significance</h4>
<p>This translational data identifies key cytokines and lymphopenia associated with disease severity although mechanism and key cellular players are still unknown. Higher level IL-6 production in severe patient suggests potential role of Tocilizumab (anti-IL6R) biologic although clinical trial will be necessary.</p>
</div>
<div id="credit-10" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-10" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china" class="anchor"></a>Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h3>
<p>Li et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.11.20022053]</span></p>
<div id="keywords-11" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-11" class="anchor"></a>Keywords</h4>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
</div>
<div id="major-findings" class="section level4">
<h4 class="hasAnchor">
<a href="#major-findings" class="anchor"></a>Major Findings</h4>
<p>These authors looked at 17 hospitalized patients with COVID-19 confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between January 22 and February 10 and the final data were collected on February 11. Of the 17 patients, 12 remained hospitalized while 5 were discharged after meeting national standards. The authors observed no differences based on the sex of the patients but found that the discharged patients were younger in age (p = 0.026) and had higher lymphocyte counts (p = 0.005) and monocyte counts (p = 0.019) upon admission.</p>
</div>
<div id="limitations-11" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-11" class="anchor"></a>Limitations</h4>
<p>This study is limited in the sample size of the study and the last data collection point was only one day after some of the patients were admitted.</p>
</div>
<div id="significance-8" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-8" class="anchor"></a>Significance</h4>
<p>These findings have been somewhat supported by subsequent studies that show that older age and an immunocompromised state are more likely to result in a more severe clinical course with COVID-19. However, other studies have been published that report on larger numbers of cases.</p>
</div>
<div id="credit-11" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-11" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection" class="anchor"></a>ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h3>
<p><span class="citation">[@doi:10.1101/2020.02.12.20022418]</span></p>
<div id="keywords-12" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-12" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-11" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-11" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585, Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to show 3~10% of 2019-nCoV patients present with abnormal renal function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells, Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
</div>
<div id="limitations-12" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-12" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy cohorts; does not necessarily translate to actual viral tropism and permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal dataset if SARS-CoV-2 infection changes sperm quality or testicular inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
</div>
<div id="significance-9" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-9" class="anchor"></a>Significance</h4>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies <span class="citation">[@doi:10.1101/2020.02.08.939892]</span> and clinical reports of abnormal renal function or even kidney damage in patients infected with 2019-nCoV <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the virus to testicular tissues and indicates potential risks for male fertility. Viral orchitis reported for SARS-CoV previously <span class="math display">\[1\]</span>, but no clear evidence so far of infertility in SARS, MERS or CoVID-19 patients.</p></li>
</ul>
</div>
<div id="credit-12" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-12" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus" class="anchor"></a>Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h3>
<p><span class="citation">[@doi:10.1101/2020.02.12.945576]</span></p>
<div id="keywords-13" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-13" class="anchor"></a>Keywords</h4>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
</div>
<div id="main-findings-12" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-12" class="anchor"></a>Main Findings</h4>
<p>The authors of this study sought to characterize the immune mechanism causing severe pulmonary disease and mortality in 2019-nCoV (COVID-19) patients. Peripheral blood was collected from hospitalized ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from healthy controls (n=10) in The First Affiliated Hospital of University of Science and Technology China (Hefei, Anhui). Immune analysis was conducted by flow cytometry. 2019-nCoV patients had decreased lymphocyte, monocyte, and CD4 T cell counts compared to healthy controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4 T cells of 2019-nCoV patients expressed higher levels of activation markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3) than those of healthy controls. CD4 cells of ICU patients expressed significantly higher levels of OX40, PD-1, and Tim3 than those of non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T cells co-expressing GM-CSF and IL-6 compared to healthy controls, while ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T cells than non-ICU patients. The CD4 T cells of nCoV patients and healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of activation markers CD69, CD38, and CD44, as well as exhaustion markers PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients expressed higher levels of GM-CSF than those of non-ICU patients and healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any group. There were no differences in numbers of NK cells or B cells in 2019-nCoV patients and healthy controls, nor was there any GM-CSF or IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory monocytes were significantly increased in nCoV patients compared to healthy controls; in particular, patients in the ICU had greater percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high levels of IL-6. These may enter pulmonary circulation and damage lung tissue while initiating the cytokine storm that causes mortality in severe cases. This is consistent with the cytokine storm seen in similar coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators seen in patients with SARS-CoV-1 and MERS-CoV.</p>
</div>
<div id="limitations-13" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-13" class="anchor"></a>Limitations</h4>
<p>Though the results of this study open questions for further investigation, this is an early study on a small cohort of patients, and as such there are a number of limitations. The study included only 12 ICU patients and 21 non-ICU patients, and ideally would be repeated with a much larger patient cohort. Though the authors make claims about differences in lymphocyte and monocyte counts between patients and healthy controls, they did not report baseline laboratory findings for the control group. Additionally, severity of disease was classified based on whether or not patients were in the ICU. It would be interesting to contextualize the authors’ immunological findings with more specific metrics of disease severity or time course. Noting mortality, time from disease onset, pre-existing conditions, or severity of lung pathology in post-mortem tissue samples would paint a fuller picture of how to assess risk level and the relationship between severity of disease and immunopathology. Another limitation is the selection of cytokines and immune markers for analysis, as the selection criteria were based on the cell subsets and cytokine storm typically seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and immune profiling may reveal novel therapeutic targets that were not included in this study.</p>
</div>
<div id="significance-10" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-10" class="anchor"></a>Significance</h4>
<p>This study identifies potential therapeutic targets that could prevent acute respiratory disease syndrome (ARDS) and mortality in patients most severely affected by COVID-19. The authors propose testing monoclonal antibodies against IL6-R or GM-CSF to block recruitment of inflammatory monocytes and the subsequent cytokine storm in these patients.</p>
</div>
<div id="credit-13" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-13" class="anchor"></a>Credit</h4>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis" class="anchor"></a>Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h3>
<p>Qian et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.14.20021535]</span></p>
<div id="keywords-14" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-14" class="anchor"></a>Keywords</h4>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
</div>
<div id="main-findings-13" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-13" class="anchor"></a>Main Findings</h4>
<p>The authors performed a meta analysis of literature on clinical, laboratory and radiologic characteristics of patients presenting with pneumonia related to SARSCoV2 infection, published up to Feb 6 2020. They found that symptoms that were mostly consistent among studies were sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise and muscle pain were highly variable across studies. Leukopenia (mostly lymphocytopenia) and increased white blood cells were highly variable across studies. They identified three most common patterns seen on CT scan, but there was high variability across studies. Consistently across the studies examined, the authors found that about 75% of patients need supplemental oxygen therapy, about 23% mechanical ventilation and about 5% extracorporeal membrane oxygenation (ECMO). The authors calculated a staggering pooled mortality incidence of 78% for these patients.</p>
</div>
<div id="limitations-14" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-14" class="anchor"></a>Limitations</h4>
<p>The authors mention that the total number of studies included in this meta analysis is nine, however they also mentioned that only three studies reported individual patient data. It is overall unclear how many patients in total were included in their analysis. This is mostly relevant as they reported an incredibly high mortality (78%) and mention an absolute number of deaths of 26 cases overall. It is not clear from their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area, which somewhat limits the overall generalizability and applicability of these results.</p>
</div>
<div id="significance-11" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-11" class="anchor"></a>Significance</h4>
<p>This meta analysis offers some important data for clinicians to refer to when dealing with patients with COVID-19 and specifically with pneumonia. It is very helpful to set expectations about the course of the disease.</p>
</div>
<div id="credit-14" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-14" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients" class="anchor"></a>Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h3>
<p>Liu et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.20023671]</span></p>
<div id="keywords-15" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-15" class="anchor"></a>Keywords</h4>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
</div>
<div id="main-findings-14" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-14" class="anchor"></a>Main Findings</h4>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan including 27 mild cases and 13 severe cases of COVID-19. They performed a 16-day kinetic analysis of peripheral blood from time of disease onset. Patients in the severe group were older (medium age of 59.7, compared to 48.7 in mild group) and more likely to have hypertension as a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients and 84.6% of the severe patients. Lymphopenia was due to low T cell count, specially CD8 T cells. Severe patients showed higher neutrophil counts and an increase of cytokines in the serum (IL2, IL6, IL10 and IFNγ). The authors measured several other clinical laboratory parameters were also higher in severe cases compared to mild, but concluded that neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to identify the severe cases compared to other receiver operating characteristic (ROC).</p>
</div>
<div id="limitations-15" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-15" class="anchor"></a>Limitations</h4>
<p>This was a small cohort (N=40), and two of the patients initially included in the severe group (N=13) passed away and were excluded from the analysis due to lack of longitudinal data. However, it would be most important to be able to identify patients with severe disease with higher odds of dying. It seems that the different time points analyzed relate to hospital admission, which the authors describe as disease onset. The time between first symptoms and first data points is not described. It would have been important to analyze how the different measured parameters change according to health condition, and not just time (but that would require a larger cohort). The predictive value of N8R compared to the more commonly used NLR needs to be assessed in other independent and larger cohorts. Lastly, it is important to note that pneumonia was detected in patients included in the “mild” group, but according to the Chinese Clinical Guidance for COVID-19 Pneumonia Diagnosis and Treatment (7th edition) this group should be considered “moderate”.</p>
</div>
<div id="significance-12" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-12" class="anchor"></a>Significance</h4>
<p>Lymphopenia and cytokine storm have been described to be detrimental in many other infections including SARS-CoV1 and MERS-CoV. However, it was necessary to confirm that this dramatic immune response was also observed in the SARS-CoV2 infected patients. These results and further validation of the N8R ratio as a predictor of disease severity will contribute for the management of COVID19 patients and potential development of therapies.</p>
</div>
<div id="credit-15" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-15" class="anchor"></a>Credit</h4>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019" class="anchor"></a>Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h3>
<p>Chen et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.20023903]</span></p>
<div id="keywords-16" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-16" class="anchor"></a>Keywords</h4>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
</div>
<div id="main-findings-15" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-15" class="anchor"></a>Main Findings</h4>
<p>This study retrospectively evaluated clinical, laboratory, hematological, biochemical and immunologic data from 21 subjects admitted to the hospital in Wuhan, China (late December/January) with confirmed SARS-CoV-2 infection. The aim of the study was to compare ‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old) COVID-19 cases. Disease severity was defined by patients’ blood oxygen level and respiratory output. They were classified as ‘severe’ if SpO2 93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had higher CRP and ferritin, alanine and aspartate aminotransferases, and lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells and total B cells; and lower total lymphocytes, CD4 and CD8 T cells (except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No significant differences were observed for IL-8, counts of NK cells, CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
</div>
<div id="limitations-16" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-16" class="anchor"></a>Limitations</h4>
<p>Several potential limitations should be noted: 1) Blood samples were collected 2 days post hospital admission and no data on viral loads were available; 2) Most patients were administered medications (e.g. corticosteroids), which could have affected lymphocyte counts. Medications are briefly mentioned in the text of the manuscript; authors should include medications as part of Table 1. 3) ‘Severe’ cases were significantly older and 4/11 ‘severe’ patients died within 20 days. Authors should consider a sensitivity analysis of biomarkers with the adjustment for patients’ age.</p>
</div>
<div id="significance-13" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-13" class="anchor"></a>Significance</h4>
<p>Although the sample size was small, this paper presented a broad range of clinical, biochemical, and immunologic data on patients with COVID-19. One of the main findings is that SARS-CoV-2 may affect T lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy production. Potentially, diminished T lymphocytes and elevated cytokines can serve as biomarkers of severity of COVID-19.</p>
</div>
<div id="credit-16" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-16" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development" class="anchor"></a>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h3>
<p>Sun et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.16.951723]</span></p>
<div id="keywords-17" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-17" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
</div>
<div id="main-findings-16" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-16" class="anchor"></a>Main Findings</h4>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S) protein receptor binding domain (RBD) and interactions with ACE2 using computational modeling, and interrogated cross-reactivity and cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the same human receptor ACE2, the receptor binding motif (RBM) is only 50% homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based on the previous crystal structure data of SARS-CoV, and measurement of binding affinities against human ACE2 using recombinant SARS-CoV and SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1 peptides to ACE2, explaining the similar transmissibility of SARS-CoV and SARS-CoV-2 and consistent with previous data (Wall et al Cell 2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal antibodies were about two-order of magnitude less efficient to neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing monoclonal antibodies against SARS-CoV had poor binding and neutralizing activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding monoclonal antibodies show some efficiency to bind and neutralize SARS-CoV-2. The results suggest that that antibodies to more conserved regions outside the RBM motif might possess better cross-protective neutralizing activities between two strains.</p>
</div>
<div id="limitations-17" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-17" class="anchor"></a>Limitations</h4>
<p>It would have been helpful to show the epitopes recognized by the monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to make predictions for induction of broadly neutralizing antibodies. The data on monoclonal antibody competition with ACE2 for binding to SARS-CoV RBD should have also included binding on SARS-CoV2, especially for the three monoclonal antibodies that showed neutralization activity for SARS-CoV2. Because of the less homology in RBM sequences between viruses, it still may be possible that these antibodies would recognize the ACE2 RBD in SARS-CoV-2.</p>
</div>
<div id="significance-14" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-14" class="anchor"></a>Significance</h4>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1 or RBD protein could induce monoclonal antibodies to cross-bind and cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these types of antibodies could be found in human survivors or in the asymptomatic populations as well, it might suggest that exposure to previous Coronavirus strains could have induced cross-neutralizing antibodies and resulted in the protection from severe symptoms in some cases of SARS-CoV2.</p>
</div>
<div id="credit-17" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-17" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine" class="section level3">
<h3 class="hasAnchor">
<a href="#protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine" class="anchor"></a>Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h3>
<p>Chen et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.17.951939]</span></p>
<div id="keywords-18" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-18" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
</div>
<div id="main-findings-17" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-17" class="anchor"></a>Main Findings</h4>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and week 4) with SV8000 carrying the information to express a S1-orf8 fusion protein and the N protein from the BJ01 strain of SARS-CoV-1. By week 8, immunized animals had signs of immunological protection (IgG and neutralization titers) against SARS-CoV-1 and were protected against challenge with the PUMC-1 strain, with fewer detectable symptoms of respiratory distress, lower viral load, shorter periods of viral persistence, and less pathology in the lungs compared to non-immunized animals.</p>
</div>
<div id="limitations-18" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-18" class="anchor"></a>Limitations</h4>
<p>The authors should write clearer descriptions of the methods used in this article. They do not describe how the IgG titers or neutralization titers were determined. There are some issues with the presentation of data, for example, in Figure 1a, y-axis should not be Vmax; forming cells and 1d would benefit from showing error bars. Furthermore, although I inferred that the animals were challenged at week 8, the authors did not explicitly detail when the animals were challenged. The authors should explain the design of their vaccine, including the choice of antigens and vector. The authors also do not include a description of the ethical use of animals in their study.</p>
</div>
<div id="significance-15" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-15" class="anchor"></a>Significance</h4>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of possible implications for the current SARS-CoV-2 pandemic. Could a similar vaccine be designed to protect against SARS-CoV-2 and would the concerns regarding emerging viral mutations that the authors describe as a limitation for SARS-CoV-1 also be true in the context of SARS-CoV-2?</p>
</div>
<div id="credit-18" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-18" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19" class="anchor"></a>Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h3>
<p><span class="citation">[@doi:10.1101/2020.02.18.20024364]</span></p>
<div id="keywords-19" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-19" class="anchor"></a>Keywords</h4>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
</div>
<div id="main-findings-18" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-18" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 522 COVID patients and 40 healthy controls from two hospitals in Wuhan, China, authors show both age-dependent and clinical severity-dependent decrease in T cell numbers with elderly patients and patients who are in ICU-care showing the most dramatic decrease in T cell counts. Cytokine profiling of COVID patients reveal that TNF-α, IL-6 and IL-10 are increased in infected patients with patients in the ICU showing the highest levels. Interestingly, these three cytokine levels were inversely correlated with T cell counts and such inverse relationship was preserved throughout the disease progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and flow cytometry of stained peripheral blood of 14 patients and 3 healthy volunteers demonstrate that T cells of COVID patients have increased expression of PD-1 with patients in ICU having the highest number of CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
</div>
<div id="limitations-19" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-19" class="anchor"></a>Limitations</h4>
<p>Compared to the number of patients, number of control (n= 40) is small and is not controlled for age. Additional data linking inflammatory cytokines and the quality of the adaptive response including humoral and antigen specific T cell response is much needed. T cell exhaustion study relies on marker-dependent labeling of T cell functionality of a very limited sample size (n=17)—a functional/mechanistic study of these T cells from PBMCs would have bolstered their claims.</p>
</div>
<div id="significance-16" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-16" class="anchor"></a>Significance</h4>
<p>Limited but contains interesting implications. It is already known in literature that in the context of acute respiratory viral infections CD8 T cells exhibit exhaustion-like phenotypes which further underscores the importance of mechanistic studies that can elucidate how COVID infection leads to lymphopenia and T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10) correlate with or affect effective viral immunity and what types of cells produce these cytokines? Answering that question will help us refine our targets for immune-modulatory therapies especially in patients suffering from cytokine storms.</p>
</div>
<div id="credit-19" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-19" class="anchor"></a>Credit</h4>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china" class="anchor"></a>Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.20025239]</span></p>
<div id="keywords-20" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-20" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
</div>
<div id="main-findings-19" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-19" class="anchor"></a>Main Findings</h4>
<p>Most common chronic conditions among 25 patients that died from COVID-19 related respiratory failure were hypertension (64%) and diabetes (40%). Disease progression was marked by progressive organ failure, starting first with lung dysfunction, then heart (e.g. increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN, Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88% also had lymphopenia. General markers of inflammation were also increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
</div>
<div id="limitations-20" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-20" class="anchor"></a>Limitations</h4>
<p>The limitations of this study include small sample size and lack of measurements for some tests for several patients. This study would also have been stronger with comparison of the same measurements to patients suffering from less severe disease to further validate and correlate proposed biomarkers with disease severity.</p>
</div>
<div id="significance-17" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-17" class="anchor"></a>Significance</h4>
<p>This study identifies chronic conditions (i.e. hypertension and diabetes) that strongly correlates with disease severity. In addition to general markers of inflammation, the authors also identify concomitant neutrophilia and lymphopenia among their cohort of patients. This is a potentially interesting immunological finding because we would typically expect increased lymphocytes during a viral infection. Neutrophilia may also be contributing to cytokine storm. In addition, PCT was elevated in 90.5% of patients, suggesting a role for sepsis or secondary bacterial infection in COVID-19 related respiratory failure.</p>
</div>
<div id="credit-20" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-20" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china" class="anchor"></a>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.20025288]</span></p>
<div id="keywords-21" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-21" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
</div>
<div id="main-findings-20" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-20" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over 4 weeks for correlation with renal injury. Comorbidities included chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in blood urea or creatinine, and 7.2% of patients with no prior kidney disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal replacement therapy (CRRT) alongside CoVID-19 treatment. Renal functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without progression to ARDS or worsening of CRF.</p></li>
</ul>
</div>
<div id="limitations-21" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-21" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney abnormalities not tabulated separately, making overall data hard to interpret. It will be critical to separately examine kidney function (BUN, urine creatinine and eGFR) in patients that developed any kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will be useful to correlate how treatment modality influences kidney function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low instances of kidney damage<span class="citation">[@doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.06.20020974]</span>, but those studies did not track longitudinal urine samples for acute renal injury markers and viral shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of CoVID-19 status; it will be important to compare clinical parameters of these patients (n=5) with virus-naïve CRF patients (none in this study) to make any meaningful conclusions.</p></li>
</ul>
</div>
<div id="significance-18" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-18" class="anchor"></a>Significance</h4>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and not associated with high mortaility, in stark contrast with a concurrent study <span class="citation">[@doi:10.1101/2020.02.08.20021212]</span>. If supported by further studies, this argues kidney impairment is secondary to cytokine storm/inflammation-induced organ failure, and not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of CoVID-19 patients to conclude if kidney function actively disrupted due to viral infection.</p></li>
</ul>
</div>
<div id="credit-21" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-21" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov" class="section level3">
<h3 class="hasAnchor">
<a href="#potential-t-cell-and-b-cell-epitopes-of-2019-ncov" class="anchor"></a>Potential T-cell and B-Cell Epitopes of 2019-nCoV</h3>
<p><span class="citation">[@doi:10.1101/2020.02.19.955484]</span></p>
<div id="keywords-22" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-22" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
</div>
<div id="main-findings-21" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-21" class="anchor"></a>Main Findings</h4>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and MARIA, to identify regions within the COVID-19 genome that are presentable by HLA. They identify 405 viral epitopes that are presentable on MHC-I and MHC-II and validate using known epitopes from SARS-CoV. To determine whether immune surveillance drives viral mutations to evade MHC presentation, the authors analyzed 68 viral genomes from 4 continents. They identified 93 point mutations that occurred preferentially in regions predicted to be presented by MHC-I (p=0.02) suggesting viral evolution to evade CD8 T-cell mediated killing. 2 nonsense mutations were also identified that resulted in loss of presentation of an associated antigen (FGDSVEEVL) predicted to be good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the authors used homology modeling to the SARS-CoV’s spike protein (S protein) to determine the putative structure of the CoV2 spike protein. They used Discotope2 to identify antibody binding sites on the protein surface in both the down and up conformations of the S protein. The authors validate this approach by first identifying antibody binding site in SARS-CoV S protein. In both the down and up conformation of the CoV2 S protein, the authors identified a potential antibody binding site on the S protein receptor binding domain (RBD) of the ACE2 receptor (residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike proteins may be important for antibody binding, the authors note that SARS-CoV has larger attack surfaces than CoV2. These results were later validated on published crystal structures of the CoV2 S protein RBD and human ACE2. Furthermore, analysis of 68 viral genomes did not identify any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine candidates across the viral genome that can be presented by multiple HLA alleles. Several of the peptides showed homology to SARS-CoV T-cell and B-cell epitopes.</p>
</div>
<div id="limitations-22" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-22" class="anchor"></a>Limitations</h4>
<p>While the authors used computational methods of validation, primarily through multiple comparisons to published SARS-CoV structures and epitopes, future work should include experimental validation of putative T-cell and B-cell epitopes.</p>
</div>
<div id="significance-19" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-19" class="anchor"></a>Significance</h4>
<p>The authors identified potential T-cell and B-cell epitopes that may be good candidates for peptide based vaccines against CoV2. They also made interesting observations in comparing SARS-CoV and CoV2 potential antibody binding sites, noting that SARS-CoV had larger attack surfaces for potential neutralizing antibody binding. One of the highlights of this paper was the authors’ mutation analysis of 68 viral genomes from 4 continents. This analysis not only validated their computational method for identifying T-cell epitopes, but showed that immune surveillance likely drives viral mutation in MHC-I binding peptides. The smaller attack surface may point to potential mechanisms of immune evasion by CoV2. However, absence of mutations in the RBD of CoV2 and the small number of mutations in peptides presentable to T cells suggests that vaccines against multiple epitopes could still elicit robust immunity against CoV2.</p>
</div>
<div id="credit-22" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-22" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein" class="section level3">
<h3 class="hasAnchor">
<a href="#structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein" class="anchor"></a>Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h3>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.19.956581]</span> now <span class="citation">[@doi:10.1016/j.cell.2020.02.058]</span></p>
<div id="keywords-23" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-23" class="anchor"></a>Keywords</h4>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
</div>
<div id="main-findings-22" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-22" class="anchor"></a>Main Findings</h4>
<p>The authors highlight a human angiotensin-converting enzyme 2 (hACE2), as a potential receptor used by the current Severe Acute respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that allows the virus target human cells. This virus-host interaction facilitates the infection of human cells with a high affinity comparable with SARS-CoV. The authors propose this mechanism as a probable explanation of the efficient transmission of SARS-CoV-2 between humans. Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry into VeroE6 infected cells.**</p>
</div>
<div id="limitations-23" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-23" class="anchor"></a>Limitations</h4>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>, 2020. The authors used green monkey (VeroE6) and hamster (BHK) cell lines in the experiments to drive its conclusions to humans; however, it is well known the caucasian colon adenocarcinoma human cell line (CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as well. In humans, ACE2 protein is highly expressed in the gastrointestinal tract, which again, makes the CaCo-2 cell line suitable for the following SARS-CoV-2 studies.</p>
</div>
<div id="significance-20" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-20" class="anchor"></a>Significance</h4>
<p>The results propose a functional receptor used by SARS-CoV-2 to infect humans worldwide and defining two distinct conformations of spike (S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study might help establish a precedent for initial drug design and treatment of the current global human coronavirus epidemic.</p>
</div>
<div id="credit-23" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-23" class="anchor"></a>Credit</h4>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19" class="anchor"></a>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h3>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.20.20025841]</span></p>
<div id="keywords-24" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-24" class="anchor"></a>Keywords</h4>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
</div>
<div id="main-findings-23" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-23" class="anchor"></a>Main Findings</h4>
<p>The authors characterized the immune response in peripheral blood of a 47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces samples, but not in urine, rectal swab, whole blood or throat swab. 7 days after symptom onset, the nasopharyngeal swab test turned negative, at day 10 the radiography infiltrates were cleared and at day 13 the patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of <strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK frequencies and decreased monocyte frequencies compared to healthy counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9 and was associated with higher production of cytotoxic mediators by CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3 SNP-rs12252-C/C variant</strong> in this patient, which is associated with higher susceptibility to influenza virus.</p>
</div>
<div id="limitations-24" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-24" class="anchor"></a>Limitations</h4>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their lungs <span class="citation">[@doi:10.1101/2020.02.23.20026690]</span>. Monitoring monocyte proportions in blood earlier in the disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is not sufficient to rule out NK cell activation upon SARS-CoV2 infection. In response to influenza A virus, NK cells express higher levels of activation markers CD69 and CD38, proliferate better and display higher cytotoxicity <span class="citation">[@doi:10.3389/fimmu.2019.01116]</span>. Assessing these parameters in COVID-19 patients is required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP associated with severe outcome following influenza infection. The association between this SNP and outcome upon SARS-CoV2 infection should be further investigated.</p>
</div>
<div id="significance-21" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-21" class="anchor"></a>Significance</h4>
<p>This study is among the first to describe the appearance of COVID-19-binding IgG and IgM antibodies upon infection. The emergence of new serological assays might contribute to monitor more precisely the seroconversion kinetics of COVID-19 patients <span class="citation">[@doi:10.1101/2020.03.17.20037713]</span>. Further association studies between IFITM3 SNP-rs12252-C/C variant and clinical data might help to refine the COVID-19 outcome prediction tools.</p>
</div>
<div id="credit-24" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-24" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing" class="section level3">
<h3 class="hasAnchor">
<a href="#the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing" class="anchor"></a>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h3>
<p>Liao et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.23.20026690]</span></p>
<div id="keywords-25" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-25" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-24" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-24" class="anchor"></a>Main Findings</h4>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild cases, n=3 severe cases). Data was compared to previously generated scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell organization between mild and severe disease. Specifically, they found that transcriptional clusters annotated as tissue resident alveolar macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup> inflammatory macrophages dominated the BAL of patients with severe COVID19 diseases. They show that inflammatory macrophages upregulated interferon-signaling genes, monocytes recruiting chemokines including CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine TGF-β, while alveolar macrophages expressed lipid metabolism genes, such as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients. Clonally expanded CD8 T cells were enriched in mild cases suggesting that CD8 T cells contribute to viral clearance as in Flu infection, whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but considered here as ambient contaminations.</p>
</div>
<div id="limitations-25" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-25" class="anchor"></a>Limitations</h4>
<p>These results are based on samples from 6 patients and should therefore be confirmed in the future in additional patients. Longitudinal monitoring of BAL during disease progression or resolution would have been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in patients towards inflammatory macrophages should be investigated in future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The composition of the “proliferating” cluster and how these cells differ from conventional T cell clusters should be assessed. NK and CD8 T cell transcriptomic profile, in particular the expression of cytotoxic mediator and immune checkpoint transcripts, should be compared between healthy and diseased lesions.</p>
</div>
<div id="significance-22" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-22" class="anchor"></a>Significance</h4>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that contributes to severe lung tissue damage and ARDS <span class="citation">[@doi:10/ggjfnn]</span>. Accumulation of monocyte-derived inflammatory macrophages at the expense of Alveolar macrophages known to play an anti-inflammatory role following respiratory viral infection, in part through the PPARγ pathway <span class="citation">[@doi:10.3389/fimmu.2018.01777; @doi:10.1128/JVI.00030-19]</span> are likely contributing to lung tissue injuries. These data suggest that reduction of monocyte accumulation in the lung tissues could help modulate COVID-19-induced inflammation. Further analysis of lymphoid subsets is required to understand the contribution of adaptive immunity to disease outcome.</p>
</div>
<div id="credit-25" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-25" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia" class="section level3">
<h3 class="hasAnchor">
<a href="#can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia" class="anchor"></a>Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h3>
<p>Pan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.02.25.20024711]</span></p>
<div id="keywords-26" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-26" class="anchor"></a>Keywords</h4>
<ul>
<li>Routine laboratory testing</li>
</ul>
</div>
<div id="main-findings-25" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-25" class="anchor"></a>Main Findings</h4>
<p>In an attempt to use standard laboratory testing for the discrimination between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual community acquired pneumonia (CAP), the authors compared laboratory testing results of 84 NCIP patients with those of a historical group of 316 CAP patients from 2018 naturally COVID-19 negative. The authors describe significantly lower white blood- as well as red blood- and platelet counts in NCIP patients. When analyzing differential blood counts, lower absolute counts were measured in all subsets of NCIP patients. With regard to clinical chemistry parameters, they found increased AST and bilirubin in NCIP patients as compared to CAP patients.</p>
</div>
<div id="limitations-26" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-26" class="anchor"></a>Limitations</h4>
<p>The authors claim to describe a simple method to rapidly assess a pre-test probability for NCIP. However, the study has substantial weakpoints. The deviation in clinical laboratory values in NCIP patients described here can usually be observed in severely ill patients. The authors do not comment on how severely ill the patients tested here were in comparison to the historical control. Thus, the conclusion that the tests discriminate between CAP and NCIP lacks justification.</p>
</div>
<div id="significance-23" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-23" class="anchor"></a>Significance</h4>
<p>The article strives to compare initial laboratory testing results in patients with COVID-19 pneumonia as compared to patients with a usual community acquired pneumonia. The implications of this study for the current clinical situation seem restricted due to a lack in clinical information and the use of a control group that might not be appropriate.</p>
</div>
<div id="credit-26" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-26" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia" class="anchor"></a>Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h3>
<p><span class="citation">[@doi:10.1101/2020.02.25.20025643]</span></p>
<div id="keywords-27" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-27" class="anchor"></a>Keywords</h4>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
</div>
<div id="main-findings-26" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-26" class="anchor"></a>Main Findings</h4>
<p>This study is a cross-sectional analysis of 100 patients with COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol style="list-style-type: decimal">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest; oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in arterial blood ≤ 300mmHg, progression of disease on imaging to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires mechanical ventilation; shock; other organ failure that requires treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status. Disease severity was significantly associated with levels of IL-2R, IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total WBC count, lymphocyte count, neutrophil count, and eosinophil count were also significantly correlated with disease status. Since this is a retrospective, cross-sectional study of clinical laboratory values, these data may be extrapolated for clinical decision making, but without studies of underlying cellular causes of these changes this study does not contribute to a deeper understanding of SARS-CoV-2 interactions with the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was significantly different from the mean ages of patients designated as severe or critical (p &lt; 0.001). The mean patient age was not significantly different between the severe and critical groups. However, IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B), WBC count, and neutrophil count (Figure 4A, 4B) were all significantly elevated in the critical group compared to severe. These data suggest underlying differences in COVID-19 progression that is unrelated to age.</p>
</div>
<div id="significance-24" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-24" class="anchor"></a>Significance</h4>
<p>Given the inflammatory profile outlined in this study, patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any elevations in the inflammatory biomarkers listed above, should be closely monitored for potential progression to critical status.</p>
</div>
<div id="credit-27" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-27" class="anchor"></a>Credit</h4>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design" class="section level3">
<h3 class="hasAnchor">
<a href="#an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design" class="anchor"></a>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h3>
<p>Herst et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.02.25.963546]</span></p>
<div id="keywords-28" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-28" class="anchor"></a>Keywords</h4>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
</div>
<div id="main-findings-27" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-27" class="anchor"></a>Main Findings</h4>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52 located in a conserved region of ebolavirus (EBOV) nucleocapsid protein (NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV (maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which was commonly targeted by CD8+ T cells in a group of EBOV survivors carrying the HLA-A*30:01:01 allele. To justify the testing of specific vaccine epitopes in a mouse challenge setting, the authors cite known examples of human pathogen-derived peptide antigens that are also recognized by C57BL/6 mice, as well as existing data surrounding known mouse immunogenicity of peptides related to this EBOV NP region. Testing 3 distinct 9mer peptides over an 11 amino-acid window and comparing to vaccination with the 11mer with a T-cell reactivity readout demonstrated that optimizing peptide length and position for immunogenicity may be crucial, likely due to suboptimal peptide processing and MHC-class-I loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4 ligand) was added to the injectate solution. A second peptide consisting of a predicted MHC-II epitope from the EBOV VG19 protein was added using a separate population of microspheres, and the formulation was injected by intraperitoneal administration. The vaccine was protective against a range of maEBOV doses up to at least 10,000 PFU. Survival was anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the authors state that MHC binding algorithms predict NP44-52 to be a strong binder of a set of more common HLA-A*02 alleles. The authors predict that a peptide vaccine based on the proposed formulation could elicit responses in up to 50% of people in Sudan or 30% of people in North America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for HLA-binding 9mers identified 53 peptides with predicted binding affinity &lt; 500nM, including peptides that are predicted to bind to HLA-class-I alleles of 97% of humans, 7 of which have previously been tested <em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain cytokines and disease severity, specifically IL6 which relates to multiple trial therapies. Prediction of HLA-class-I binding of SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine targeting conserved regions of SARS-CoV-2 NP although further validation in previously infected patient samples will be essential.</p>
</div>
<div id="credit-28" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-28" class="anchor"></a>Credit</h4>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2" class="anchor"></a>Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h3>
<p>Li et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.25.965434]</span></p>
<div id="keywords-29" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-29" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
</div>
<div id="main-findings-28" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-28" class="anchor"></a>Main Findings</h4>
<p>This study employs a series of bioinformatic pipelines to identify T and B cell epitopes on spike (S) protein of SARS-CoV-2 and assess their properties for vaccine potential. To identify B cell epitopes, they assessed structural accessibility, hydrophilicity, and beta-turn and flexibility which are all factors that promote their targeting by antibodies. To identify T cell epitopes, they filtered for peptides with high antigenicity score and capacity to bind 3 or more MHC alleles. Using the protein digest server, they also demonstrated that their identified T and B cell epitopes are stable, having multiple non-digesting enzymes per epitope. Epitopes were also determined to be non-allergenic and non-toxin as assessed by Allergen FP 1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the strength of epitope-HLA interaction via PepSite. Overall, they predict four B cell and eleven T cell epitopes (two MHC I and nine MHC II binding) to pass stringent computational thresholds as candidates for vaccine development. Furthermore, they performed sequence alignment between all identified SARS-CoV-2 S protein mutations and predicted epitopes, and showed that the epitopes are conserved across 134 isolates from 38 locations worldwide. However, they report that these conserved epitopes may soon become obsolete given the known mutation rate of related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the urgency of anti-viral vaccine development.</p>
</div>
<div id="limitations-27" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-27" class="anchor"></a>Limitations</h4>
<p>While spike (S) protein may have a critical role in viral entry into host cells and their epitope prediction criterion were comprehensive, this study did not examine other candidate SARS-CoV-2 proteins. This point is particularly important given that a single epitope may not be sufficient to induce robust immune memory, and recent approaches involve multi-epitope vaccine design. Furthermore, their study only included a direct implementation of various published methods, but did not validate individual bioinformatic tools with controls to demonstrate robustness. Finally, it is critical that these predicted epitopes are experimentally validated before any conclusions can be drawn about their potential as vaccine candidates or their clinical efficacy.</p>
</div>
<div id="significance-25" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-25" class="anchor"></a>Significance</h4>
<p>This study provides a computational framework to rapidly identify epitopes that may serve as potential vaccine candidates for treating SARS-CoV-2.</p>
</div>
<div id="credit-29" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-29" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis" class="anchor"></a>The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h3>
<p>Wang Jr. et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.26.20026989]</span></p>
<div id="keywords-30" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-30" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
</div>
<div id="main-findings-29" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-29" class="anchor"></a>Main Findings</h4>
<p>This study describes the occurrence of a cytokine release syndrome-like (CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time from first symptom to acute respiratory distress syndrome (ARDS) was 10 days. All patients had decreased CD3, CD4 and CD8 cells, and a significant increase of serum IL-6. Furthermore, 91% had decreased NK cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell counts. All of these parameters correlated with the area of pulmonary inflammation in CT scan images. Mechanical ventilation increased the numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and improving the immunological parameters.</p>
</div>
<div id="limitations-28" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-28" class="anchor"></a>Limitations</h4>
<p>The number of patients included in this retrospective single center study is small (n=11), and the follow-up period very short (25 days). Eight of the eleven patients were described as having CRSL, and were treated by intubation (7) or ECMO (2). Nine patients were still in the intensive care unit at the time of publication of this article, so their disease outcome is unknown.</p>
</div>
<div id="significance-26" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-26" class="anchor"></a>Significance</h4>
<p>The authors define a cytokine release syndrome-like toxicity in patients with COVID-19 with clinical radiological and immunological criteria: 1) decrease of circulating CD4, CD8 and NK cells; 2) substantial increase of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue damage. This event seems to occur very often in critically ill patients with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the peripheral blood preceded other laboratory alterations, thus, IL-6 might be an early biomarker for the severity of COVID-19 pneumonia. The manuscript will require considerable editing for organization and clarity.</p>
</div>
<div id="credit-30" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-30" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china" class="anchor"></a>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h3>
<p>Huang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.02.27.20029009]</span></p>
<div id="keywords-31" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-31" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
</div>
<div id="main-findings-30" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-30" class="anchor"></a>Main Findings</h4>
<p>This is a simple study reporting clinical characteristics of patients who did not survive COVID-19. All patients (mean age=69.22 years) had acute respiratory distress syndrome (ARDS) and their median time from onset to ARDS was 11 days. The median time from onset to death was 17 days. Most patients were older male (70% male) with co-morbidities and only 11 % were smokers. 75% patients showed bilateral pneumonia. Many patients had chronic diseases, including hypertension (58.33%). cardiovascular disease (22.22%) and diabetes (19.44%). Typical clinical feature measured in these patients includes lymphopenia and elevated markers of inflammation.</p>
</div>
<div id="limitations-29" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-29" class="anchor"></a>Limitations</h4>
<p>As noted by the authors, the conclusions of this study are very limited because this is single-centered study focusing on a small cohort of patients who did not survive. Many clinical parameters observed by the authors (such* as increase levels of serum CRP, PCT, IL-6) have also been described in other COVID19 patients who survived the infection</p>
</div>
<div id="significance-27" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-27" class="anchor"></a>Significance</h4>
<p>This study is essentially descriptive and may be useful for clinical teams monitoring COVID19 patients.</p>
</div>
<div id="credit-31" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-31" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019" class="anchor"></a>Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h3>
<p><span class="citation">[@doi:10.1101/2020.02.28.20028514]</span></p>
<div id="keywords-32" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-32" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
</div>
<div id="main-findings-31" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-31" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 85 hospitalized COVID patients in a Beijing hospital, authors showed that patients with elevated ALT levels (n = 33) were characterized by significantly higher levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia compared to their counterparts with normal ALT levels. Proportion of severe and critical patients in the ALT elevation group was significantly higher than that of normal ALT group. Multivariate logistic regression performed on clinical factors related to ALT elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count (&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a finding that led the authors to suggest cytokine storm as a major mechanism of liver damage.</p>
</div>
<div id="limitations-30" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-30" class="anchor"></a>Limitations</h4>
<p>The article’s most attractive claim that liver damage seen in COVID patients is caused by cytokine storm (rather than direct infection of the liver) hinges solely on their multivariate regression analysis. Without further mechanistic studies a) demonstrating how high levels of inflammatory cytokines can induce liver damage and b) contrasting types of liver damage incurred by direct infection of the liver vs. system-wide elevation of inflammatory cytokines, their claim remains thin. It is also worth noting that six of their elevated ALT group (n=33) had a history of liver disease (i.e. HBV infection, alcoholic liver disease, fatty liver) which can confound their effort to pin down the cause of hepatic injury to COVID.</p>
</div>
<div id="significance-28" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-28" class="anchor"></a>Significance</h4>
<p>Limited. This article confirms a rich body of literature describing liver damage and lymphopenia in COVID patients.</p>
</div>
<div id="credit-32" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-32" class="anchor"></a>Credit</h4>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients" class="anchor"></a>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h3>
<p><span class="citation">[@doi:10.1101/2020.02.29.20029520]</span></p>
<div id="keywords-33" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-33" class="anchor"></a>Keywords</h4>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
</div>
<div id="main-findings-32" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-32" class="anchor"></a>Main Findings</h4>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines for Covid19 diagnosis and treatment version 6 were included in this study. Patients were further sub-divided into three groups based on clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR with no or mild clinical symptoms (fever; respiratory; radiological abnormalities); (2) severe, at least one of the following: shortness of breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%, Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary damage); and (3) critically ill, at least one additional complicating factor: respiratory failure with need for mechanical ventilation; systemic shock; multi-organ failure and transfer to ICU. Serum samples and throat-swaps were collected from all 48 patients enrolled. SARS-CoV-2 RNA was assessed by qPCR with positive results being defined as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified using a commercially available detection kit. Briefly, patient characteristics in this study confirm previous reports suggesting that higher age and comorbidities are significant risk factors of clinical severity. Of note, 5 out of 48 of patients (10.41%), all in the critically ill category, were found to have detectable serum SARS-CoV-2 RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these patients were found to be substantially higher and this correlated with the presence of detectable SARS-CoV-2 RNA levels. The authors hypothesize that viral RNA might be released from acutely damages tissues in moribund patients during the course of Covid19 and that RNaemia along with IL-6 could potentially be used as a prognostic marker.</p>
</div>
<div id="limitations-31" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-31" class="anchor"></a>Limitations</h4>
<p>While this group’s report generally confirms some of the major findings of a more extensive study, published in early February 2020, <span class="citation">[@doi:10/ggjfnn]</span>, there are limitations that should be taken into account. First, the number of patients enrolled is relatively small; second, interpretation of these data would benefit from inclusion of information about study specifics as well as providing relevant data on the clinical course of these patients other than the fact that some were admitted to ICU (i.e. demographics on how many patients needed respiratory support, dialysis, APACHE Ii/III or other standard ICU scores as robust prognostic markers for mortality etc). It also remains unclear at which time point the serum samples were taken, i.e. whether at admission, when the diagnosis was made or during the course of the hospital stay (and potentially after onset of therapy, which could have affected both IL-6 and RNA levels). The methods section lacks important information on the qPCR protocol employed, including primers and cycling conditions used. From a technical point of view, Ct values &gt;35 seem somewhat non-specific (although Ct &lt;40 was defined as the CDC cutoff as well) indicating that serum RNA levels are probably very low, therefore stressing the need for highly specific primers and high qPCR efficiency. In addition, the statistical tests used (t-tests, according to the methods section) do not seem appropriate as the organ-specific data such as BUN and troponin T values seem to be not normally distributed across groups (n= 5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations and the differences in patient sample size, it is difficult to believe that these data are statistically significantly different.</p>
</div>
<div id="significance-29" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-29" class="anchor"></a>Significance</h4>
<p>This study is very rudimentary and lacks a lot of relevant clinical details. However, it corroborates some previously published observations regarding RNAemia and IL-6 by another group. Generally, regarding future studies, it would be important to address the question of IL-6 and other inflammatory cytokine dynamics in relation to Covid19 disease kinetics (high levels of IL-6, IL-8 and plasma leukotriene were shown to have prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med. 2015 <span class="citation">[@doi:10.1186/s40169-015-0065-2]</span>).</p>
</div>
<div id="credit-33" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-33" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study" class="section level3">
<h3 class="hasAnchor">
<a href="#lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study" class="anchor"></a>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h3>
<p><span class="citation">[@doi:10.1101/2020.03.01.20029074]</span></p>
<div id="keywords-34" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-34" class="anchor"></a>Keywords</h4>
<ul>
<li>Lymphopenia</li>
</ul>
</div>
<div id="main-findings-33" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-33" class="anchor"></a>Main Findings</h4>
<p>Based on a retrospective study of 162 COVID patients from a local hospital in Wuhan, China, the authors show an inverse correlation between lymphocyte % (LYM%) of patients and their disease severity. The authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40 moderate) throughout the disease progression with fatal cases showing no recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The temporal data of LYM % in COVID patients was used to construct a Time-Lymphocyte% model which is used to categorize and predict patients’ disease severity and progression. The model was validated using 92 hospitalized cases and kappa statistic test was used to assess agreement between predicted disease severity and the assigned clinical severity (k = 0.49).</p>
</div>
<div id="limitations-32" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-32" class="anchor"></a>Limitations</h4>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as a predictive model for clinical severity is very simple in its construction and derives from correlative data of 162 patients. In order for the model to be of use, it needs validation using a far more robust data set and possibly a mechanistic study on how COVID leads to lymphopenia in the first place. In addition, it should be noted that no statistical test assessing significance of LYM % values between disease severities was performed.</p>
</div>
<div id="significance-30" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-30" class="anchor"></a>Significance</h4>
<p>This article is of limited significance as it simply reports similar descriptions of COVID patients made in previous literature that severe cases are characterized by lymphopenia.</p>
</div>
<div id="credit-34" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-34" class="anchor"></a>Credit</h4>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019" class="anchor"></a>The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.01.20029769]</span></p>
<div id="keywords-35" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-35" class="anchor"></a>Keywords</h4>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
</div>
<div id="main-findings-34" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-34" class="anchor"></a>Main Findings</h4>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe and 11 non-severe). Patients with severe symptoms at admission (baseline) showed obvious lymphocytopenia and significantly increased interleukin-6 (IL-6) and CRP, which was positively correlated with symptoms severity. IL-6 at baseline positively correlates with CRP, LDH, ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed significant reduction of IL-6 in remission cases.</p>
</div>
<div id="limitations-33" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-33" class="anchor"></a>Limitations</h4>
<p>Limited sample size at baseline, especially for the non-severe leads to question on representativeness. The longitudinal study method is not described in detail and suffers from non-standardized treatment. Limited panel of pro-inflammatory cytokine was analyzed. Patients with severe disease show a wide range of altered blood composition and biomarkers of inflammation, as well as differences in disease course (53.6% were cured, about 10% developed acute respiratory distress syndrome). The authors comment on associations between IL-6 levels and outcomes, but these were not statistically significant (maybe due to the number of patients, non-standardized treatments, etc.) and data is not shown. Prognostic biomarkers could have been better explored. Study lacks multivariate analysis.</p>
</div>
<div id="significance-31" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-31" class="anchor"></a>Significance</h4>
<p>IL-6 could be used as a pharmacodynamic marker of disease severity. Cytokine Release Syndrome (CRS) is a well-known side effect for CAR-T cancer therapy and there are several effective drugs to manage CRS. Drugs used to manage CRS could be tested to treat the most severe cases of COVID19.</p>
</div>
<div id="credit-35" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-35" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china" class="anchor"></a>Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h3>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.01.20029785]</span></p>
<div id="keywords-36" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-36" class="anchor"></a>Keywords</h4>
<ul>
<li>Routine laboratory testing</li>
</ul>
</div>
<div id="main-findings-35" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-35" class="anchor"></a>Main Findings</h4>
<p>The authors of this study provide a comprehensive analysis of clinical laboratory assessments in 75 patients (median age 47 year old) hospitalized for Corona virus infection in China measuring differential blood counts including T-cell subsets (CD4, CD8), coagulation function, basic blood chemistry, of infection-related biomarkers including CRP, Procalcitonin (PCT) (Precursor of calcitonin that increases during bacterial infection or tissue injury), IL-6 and erythrocyte sedimentation rate as well as clinical parameters. Among the most common hematological changes they found increased neutrophils, reduced CD4 and CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and PCT levels were significantly increased in infected patients suggesting that increased IL-6 may correlate well with disease severity in COVID-19 infections</p>
</div>
<div id="limitations-34" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-34" class="anchor"></a>Limitations</h4>
<p>The authors performed an early assessment of clinical standard parameters in patients infected with COVID-19. Overall, the number of cases (75) is rather low and the snapshot approach does not inform about dynamics and thus potential relevance in the assessment of treatment options in this group of patients.</p>
</div>
<div id="significance-32" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-32" class="anchor"></a>Significance</h4>
<p>The article summarizes provides a good summary of some of the common changes in immune cells inflammatory cytokines in patients with a COVID-19 infection and. Understanding how these changes can help predict severity of disease and guide therapy including IL-6 cytokine receptor blockade using Tocilizumab or Sarilumab will be important to explore.</p>
</div>
<div id="credit-36" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-36" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome" class="section level3">
<h3 class="hasAnchor">
<a href="#exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome" class="anchor"></a>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h3>
<p>Yang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.02.20029975]</span></p>
<div id="keywords-37" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-37" class="anchor"></a>Keywords</h4>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
</div>
<div id="summary-2" class="section level4">
<h4 class="hasAnchor">
<a href="#summary-2" class="anchor"></a>Summary</h4>
<p>Plasma cytokine analysis (48 cytokines) was performed on COVID-19 patient plasma samples, who were sub-stratified as severe (N=34), moderate (N=19), and compared to healthy controls (N=8). Patients were monitored for up to 24 days after illness onset: viral load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests, and epidemiological/clinical characteristics of patients were reported.</p>
</div>
<div id="main-findings-36" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-36" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function (PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001 and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic insufficiency, renal insufficiency).</p></li>
</ul>
</div>
<div id="limitations-35" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-35" class="anchor"></a>Limitations</h4>
<p>Collection time of clinical data and lab results not reported directly (likely 4 days (2,6) after illness onset), making it very difficult to determine if cytokines were predictive of patient outcome or reflective of patient compensatory immune response (likely the latter). Small N for cytokine analysis (N=2 fatal and N=5 severe/critical, and N=7 moderate or discharged). Viral treatment strategy not clearly outlined.</p>
</div>
<div id="expanded-results" class="section level4">
<h4 class="hasAnchor">
<a href="#expanded-results" class="anchor"></a>Expanded Results</h4>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory manifestations, and radiological findings consistent with pneumonia while severe patients had one or more of the following: 1) respiratory distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300 mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3, and IL-Ra</strong> as measure by area under the curve analysis during sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif. upregulated throughout observation timecourse, and IP-10 increased and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF, MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as a potential treatment for amelioration of COVID-19 (and associated ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li>
<p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/ severe patients</strong></p>
<ul>
<li>
<strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or eosino/basophil % measured</li>
</ul>
</li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and LDH were not significantly different between groups and were not compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly different (chronic heart/lung/renal/liver disease, diabetes, or cancer) and patient time courses (onset to admission and onset to viral tx) also not significantly different – 4 days (2, 6) on average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74) vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%), myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
</div>
<div id="significance-33" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-33" class="anchor"></a>Significance</h4>
<p>Outline of pathological time course (implicating innate immunity esp.) and identification key cytokines associated with disease severity and prognosis (+ comorbidities). Anti-IP-10 as a possible therapeutic intervention (ex: Eldelumab).</p>
</div>
<div id="credit-37" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-37" class="anchor"></a>Credit</h4>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019" class="section level3">
<h3 class="hasAnchor">
<a href="#antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019" class="anchor"></a>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h3>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.02.20030189]</span></p>
<div id="keywords-38" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-38" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
</div>
<div id="main-findings-37" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-37" class="anchor"></a>Main Findings</h4>
<p>This study examined antibody responses in the blood of COVID-19 patients during the early SARS CoV2 outbreak in China. Total 535 plasma samples were collected from 173 patients (51.4% female) and were tested for seroconversion rate using ELISA. Authors also compared the sensitivity of RNA and antibody tests over the course of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody (Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. Overall, the seroconversion of Ab was significantly quicker than that of IgM (p = 0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates between critical and non-critical patients did not reveal any significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease onset, reached 90% at day 13 and 100% at later time points (15-39 days). In contrast, RNA was only detectable in 45.5% of samples at days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6% (2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable total Ab titers respectively Combining RNA and antibody tests significantly raised the sensitivity for detecting COVID-19 patients in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative COVID-19 patients (n=9) since onset of disease revealed that antibodies may not be sufficient to clear the virus. It should be noted that increases in of antibody titers were not always accompanied by RNA clearance.</p></li>
</ul>
</div>
<div id="limitations-36" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-36" class="anchor"></a>Limitations</h4>
<p>Because different types of ELISA assays were used for determining antibody concentrations at different time points after disease onset, sequential seroconversion of total Ab, IgM and IgG may not represent actual temporal differences but rather differences in the affinities of the assays used. Also, due to the lack of blood samples collected from patients in the later stage of illness, how long the antibodies could last remain unknown. For investigative dynamics of antibodies, more samples were required.</p>
</div>
<div id="significance-34" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-34" class="anchor"></a>Significance</h4>
<p>Total and IgG antibody titers could be used to understand the epidemiology of SARS CoV-2 infection and to assist in determining the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological and RNA testing in the diagnosis and clinical management of COVID-19 patients. The understanding of antibody responses and their half-life during and after SARS CoV2 infection is important and warrants further investigations.</p>
</div>
<div id="credit-38" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-38" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients" class="anchor"></a>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h3>
<p>Chen et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.03.20030437]</span></p>
<div id="keywords-39" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-39" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
</div>
<div id="main-findings-38" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-38" class="anchor"></a>Main Findings</h4>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14 women) using standard laboratory tests and flow cytometry. All patients were treated with antibiotics. Twenty-four of the 25 patients were also treated with anti-viral Umefinovir and 14 of the patients were treated with corticosteroids. 14 patients became negative for the virus after 8-14 days of treatment. The same treatment course was extended to 15-23 days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of infection. Patients with hypertension, diabetes, malignancy or chronic liver disease were all unable to clear the virus at day 14, though not statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also reported. B cell, CD4 and CD8 T cell counts were only increased upon treatment in patients who cleared the virus. NK cell frequencies remained unchanged after treatment in all the patients.</p>
</div>
<div id="limitations-37" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-37" class="anchor"></a>Limitations</h4>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup> treatment, and 43% of all patients who cleared the virus were treated with corticosteroids. Corticosteroids have strong effects on the immune compartment in blood <span class="citation">[@doi:10.1038/srep23002]</span>. The authors should have accounted for corticosteroid treatment when considering changes in T, NK and B cell frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following treatment would have provided insights into the COVID-19 cytokine storm biology. Patients with higher baseline levels of IL-6 have been reported to have lower CD8 and CD4 T cell frequencies <span class="citation">[@doi:10.1101/2020.03.01.20029785]</span>. Correlating IL-6 with cell counts before and after treatment would thus have also been of interest. The report of the laboratory measures in table 2 is incomplete and should include the frequencies of patients with increased/decreased levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data does not support NK cell restoration upon treatment in patients who cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup> treatment course in those patients.</p>
</div>
<div id="significance-35" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-35" class="anchor"></a>Significance</h4>
<p>Previous reports suggest an association between disease severity and elevated IL-6 or pro-calcitonin concentrations in COVID-19 patients <span class="citation">[@doi:10.1101/2020.02.29.20029520; @doi:10.1016/j.cca.2020.03.004]</span>. IL-6 receptor blockade is also being administered to patients enrolled in clinical trials (NCT04317092). This report thus contributes to highlight elevated concentrations of these analytes in COVID-19 patients. Mechanisms underlying the association between viral clearance and restoration of the T cell and B cell frequencies suggests viral-driven immune dysregulation, which needs to be investigated in further studies.</p>
</div>
<div id="credit-39" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-39" class="anchor"></a>Credit</h4>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study" class="anchor"></a>Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study </h3>
<p>Xu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.03.20030668]</span></p>
<div id="keywords-40" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-40" class="anchor"></a>Keywords</h4>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
</div>
<div id="main-findings-39" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-39" class="anchor"></a>Main Findings:</h4>
<p>This work analyses laboratory and clinical data from 45 patients treated in the in ICU in a single province in China. Overall, 44% of the patients were intubated within 3 days of ICU admission with only 1 death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ Failure Assessment (SOFA) score (clinical score, the higher the more critical state), LDH levels are positively correlated to SOFA score. Overall, older patients (&gt;60yo), with high SOFA score, high LDH levels and low lymphocytes levels at ICU admission are at higher risk of intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to limited sample size no conclusion can be made.</p>
</div>
<div id="limitations-38" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-38" class="anchor"></a>Limitations</h4>
<p>While the study offers important insights into disease course and clinical lab correlates of outcome, the cohort is relatively small and is likely skewed towards a less-severe population compared to other ICU reports given the outcomes observed. Analysis of laboratory values and predictors of outcomes in larger cohorts will be important to make triage and treatment decisions. As with many retrospective analyses, pre-infection data is limited and thus it is not possible to understand whether lymphopenia was secondary to underlying comorbidities or infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of convalescent plasma administration.</p>
</div>
<div id="significance-36" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-36" class="anchor"></a>Significance</h4>
<p>This clinical data enables the identification of at-risk patients and gives guidance for research for treatment options. Indeed, further work is needed to better understand the causes of the lymphopenia and its correlation with outcome.</p>
</div>
<div id="credit-40" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-40" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2" class="anchor"></a>Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h3>
<p><span class="citation">[@doi:10.1101/2020.03.03.962332]</span></p>
<div id="keywords-41" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-41" class="anchor"></a>Keywords</h4>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
</div>
<div id="main-findings-40" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-40" class="anchor"></a>Main Findings</h4>
<p>Using in silico bioinformatic tools, this study identified putative antigenic B-cell epitopes and HLA restricted T-cell epitopes from the spike, envelope and membrane proteins of SARS-CoV-2, based on the genome sequence available on the NCBI database. T cell epitopes were selected based on predicted affinity for the more common HLA-I alleles in the Chinese population. Subsequently, the authors designed vaccine peptides by bridging selected B-cell epitopes and adjacent T-cell epitopes. Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface of the virion and had a high antigenicity score. A total of 499 T-cell epitopes were predicted. Based on the 19 B-cell epitopes and their 121 adjacent T-cell epitopes, 17 candidate vaccine peptides were designed. Additionally, another 102 vaccine peptides containing T-cell epitopes only were generated. Based on the epitope counts and HLA score, 13 of those were selected. Thus, a total of 30 peptide vaccine candidates were designed.</p>
</div>
<div id="limitations-39" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-39" class="anchor"></a>Limitations</h4>
<p>While this study provides candidates for the development of vaccines against SARS-CoV-2, in vitro and in vivo trials are required to validate the immunogenicity of the selected B and T cell epitopes. This could be done using serum and cells from CoV-2-exposed individuals, and in preclinical studies. The implication of this study for the current epidemic are thus limited. Nevertheless, further research on this field is greatly needed.</p>
</div>
<div id="credit-41" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-41" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china" class="section level3">
<h3 class="hasAnchor">
<a href="#clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china" class="anchor"></a>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h3>
<p>Cao et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.04.20030395]</span></p>
<div id="keywords-42" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-42" class="anchor"></a>Keywords</h4>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
</div>
<div id="main-findings-41" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-41" class="anchor"></a>Main Findings</h4>
<p>This single-center cohort study analyzes the clinical and laboratory features of 198 patients with confirmed COVID-19 infection in Shanghai, China and correlated these parameters with clinical disease severity, including subsequent intensive care unit (ICU) admission. 19 cases (9.5%) required ICU admission after developing respiratory failure or organ dysfunction. Age, male sex, underlying cardiovascular disease, and high symptom severity (high fever, dyspnea) were all significantly correlated with ICU admission. Additionally, ICU admission was more common in patients who presented with lymphopenia and elevated neutrophil counts, among other laboratory abnormalities. Flow cytometric analysis revealed that patients admitted to the ICU had significantly reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+ leukocyte populations compared to the cohort of patients not requiring ICU admission.</p>
</div>
<div id="limitations-40" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-40" class="anchor"></a>Limitations</h4>
<p>The limitations of this study include the relatively small sample size and lack of longitudinal testing. The authors also did not assess whether respiratory comorbidity – such as asthma or chronic obstructive lung disease – in addition to immunosuppression affected ICU admission likelihood.</p>
</div>
<div id="significance-37" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-37" class="anchor"></a>Significance</h4>
<p>COVID-19 has already sickened thousands across the globe, though the severity of these infections is markedly diverse, ranging from mild symptoms to respiratory failure requiring maximal intervention. Understanding what clinical, laboratory, and immunologic factors predict the clinical course of COVID-19 infection permits frontline providers to distribute limited medical resources more effectively.</p>
</div>
<div id="credit-42" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-42" class="anchor"></a>Credit</h4>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
</div>
</div>
<div id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing" class="section level3">
<h3 class="hasAnchor">
<a href="#serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing" class="anchor"></a>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h3>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.04.20030916]</span></p>
<div id="keywords-43" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-43" class="anchor"></a>Keywords</h4>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
</div>
<div id="main-finding" class="section level4">
<h4 class="hasAnchor">
<a href="#main-finding" class="anchor"></a>Main Finding</h4>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by automated chemiluminescent immunoassay in the patients who had been already confirmed as positive by nucleic acid detection, while single positivity of IgM or IgG were detected in a very few cases in the other population including 225 non-COVID-19 cases. In addition to the increase of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was detected in three patients with COVID-19 within a very short of time (0-1 day).</p>
</div>
<div id="limitations-41" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-41" class="anchor"></a>Limitations</h4>
<p>The limitation of this study is only 3 confirmed COVID-19 cases were included, so that the relationship between anti-2019-nCov antibodies and disease progression might not be clearly defined. Another limitation is that they did not show the course of 2019-nCov specific antibodies in the cases with positive for COVID-19 but without clinical symptoms.</p>
</div>
<div id="significance-38" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-38" class="anchor"></a>Significance</h4>
<p>The detection of anti-2019-nCov antibodies can be an alternative method to diagnose and treat COVID-19 more comprehensively by distinguish non COVID-19 patients. It may be helpful to understand the course of individual cases with COVID-19 to predict the prognosis if more cases will be evaluated.</p>
</div>
<div id="credit-43" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-43" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection" class="anchor"></a>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h3>
<p><span class="citation">[@doi:10.1101/2020.03.04.20031120]</span></p>
<div id="keywords-44" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-44" class="anchor"></a>Keywords</h4>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
</div>
<div id="main-finding-1" class="section level4">
<h4 class="hasAnchor">
<a href="#main-finding-1" class="anchor"></a>Main Finding</h4>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19 patients and characterizing tissue damage and viral presence in post-mortem kidney samples from 6 Covid-19 patients, the authors conclude that significant damage occurs to the kidney, following Covid-19 infection. This is in contrast to the SARS infection from the 2003 outbreak. They determine this damage to be more prevalent in patients older than 60 years old, as determined by analysis of eGFR. H&amp;E and IHC analysis in 6 Covid-19 patients revealed that damage was in the tubules, not the glomeruli of the kidneys and suggested that macrophage accumulation and C5b-9 deposition are key to this process. </p>
</div>
<div id="limitations-42" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-42" class="anchor"></a>Limitations</h4>
<p>Severe limitations include that the H&amp;E and IHC samples were performed on post-mortem samples of unknown age, thus we cannot assess how/if age correlates with kidney damage, upon Covid-19 infection. Additionally, eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and proteinuria are amongst other measurements that could have been obtained from patient records. An immune panel of the blood was not performed to assess immune system activation. Additionally, patients are only from one hospital. </p>
</div>
<div id="significance-39" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-39" class="anchor"></a>Significance</h4>
<p>This report makes clear that kidney damage is prevalent in Covid-19 patients and should be accounted for. </p>
</div>
<div id="credit-44" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-44" class="anchor"></a>Credit</h4>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
</div>
</div>
<div id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients" class="anchor"></a>COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h3>
<p>Song et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.05.20031906]</span></p>
<div id="keywords-45" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-45" class="anchor"></a>Keywords</h4>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
</div>
<div id="main-findings-42" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-42" class="anchor"></a>Main Findings</h4>
<p>The aim of this study was to identify diagnostic or prognostic criteria which could identify patients with COVID-19 and predict patients who would go on to develop severe respiratory disease. The authors use EMR data from individuals taking a COVID-19 test at Zhejiang hospital, China in late January/Early February. A large number of clinical parameters were different between individuals with COVID-19 and also between ‘severe’ and ‘non-severe’ infections and the authors combine these into a multivariate linear model to derive a weighted score, presumably intended for clinical use.</p>
</div>
<div id="limitations-43" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-43" class="anchor"></a>Limitations</h4>
<p>Unfortunately, the paper is lacking a lot of crucial information, making it impossible to determine the importance or relevance of the findings. Most importantly, the timings of the clinical measurements are not described relative to the disease course, so it is unclear if the differences between ‘severe’ and ‘non-severe’ infections are occurring before progression to severe disease (which would make them useful prognostic markers), or after (which would not).</p>
</div>
<div id="significance-40" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-40" class="anchor"></a>Significance</h4>
<p>This paper is one of many retrospective studies coming from hospitals in China studying individuals with COVID-19. Because of the sparse description of the study design, this paper offers little new information. However, studies like this could be very valuable and we would strongly encourage the authors to revise this manuscript to include more information about the timeline of clinical measurements in relation to disease onset and more details of patient outcomes.</p>
</div>
<div id="credit-45" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-45" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease" class="section level3">
<h3 class="hasAnchor">
<a href="#ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease" class="anchor"></a>LY6E impairs coronavirus fusion and confers immune control of viral disease</h3>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.05.979260]</span></p>
<div id="keywords-46" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-46" class="anchor"></a>Keywords</h4>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
</div>
<div id="main-findings-43" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-43" class="anchor"></a>Main Findings</h4>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes for antiviral activity against endemic human coronavirus HCoV-229E (associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular infection of Huh7 cells, a human hepatoma cell line susceptible to HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em> experiments including both stable Ly6E overexpression and CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E reduces cellular infection by various other coronaviruses including human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis virus (MHV). Their experiments suggest that this effect is dependent on Ly6E inhibition of CoV strain-specific spike protein-mediated membrane fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup> mice (referred to as functional wild-type mice) with transgenic <em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type and Ly6E HSC ko mice of both sexes were infected intraperitoneally with varying doses of the natural murine coronavirus MHV, generally causing a wide range of diseases in mice including hepatitis, enteritis and encephalomyelitis. Briefly, compared to wild-type controls, mice lacking hematopoietic cell-expressed Ly6E were found to present with a more severe disease phenotype as based on serum ALT levels (prognostic of liver damage), liver histopathology, and viral titers in the spleen. Moreover, bulk RNAseq analysis of infected liver and spleen tissues indicated changes in gene expression pathways related to tissue damage and antiviral immune responses as well as a reduction of genes associated with type I IFN response and inflammation. Finally, the authors report substantial differences in the numbers of hepatic and splenic APC subsets between wild-type and knockout mice following MHV infection and show that Ly6E-deficient B cells and to a lesser extent also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
</div>
<div id="limitations-44" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-44" class="anchor"></a>Limitations</h4>
<p>Experiments and data in this study are presented in an overall logical and coherent fashion; however, some observations and the conclusions drawn are problematic and should be further addressed &amp; discussed by the authors. Methodological &amp; formal limitations include relatively low replicate numbers as well as missing technical replicates for some <em>in vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission of “outliers” in Fig. legend 2 without an apparent rationale as to why this approach was chosen; the lack of detection of actual Ly6E protein levels in Ly6E HSC ko or wild-type mice; and most importantly, missing information on RNAseq data collection &amp; analysis in the method section and throughout the paper. A more relevant concern though is that the interpretation of the experimental data presented and the language used tend to overrate and at times overgeneralize findings: for example, while the authors demonstrate statistically significant, Ly6E-mediated reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it remains unclear whether a viral titer reduction by one log decade would be of actual biological relevance in face of high viral titers <em>in vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation in the spleen (~1.5 log decades) but not liver of the ko mice (other tissue not evaluated), and while ko mice presented with only modestly increased liver pathology, both male and female ko mice exhibited significantly higher mortality. Thus, the manuscript tile statement that “Ly6E … confers immune control of viral disease” is supported by only limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E overexpression) were not performed. Of additional note here, tissue tropism and virulence differ greatly among various MHV strains and isolates whereas dose, route of infection, age, genetic background and sex of the mice used may additionally affect disease outcome and phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A, <a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/ 83/18/9258</a>). Observations attributed to hematopoietic stem cell-specific Ly6E deletion could therefore be influenced by the different genetic backgrounds of floxed and cre mice used, and although it appears that littermates wt and ko littermates were used in the experiments, the potentially decisive impact of strain differences should at least have been discussed. Along these lines, it should also be taken into account that the majority of human coronaviruses cause respiratory symptoms, which follow a different clinical course engaging other primary cellular mediators than the hepatotropic murine MHV disease studied here. It therefore remains highly speculative how the findings reported in this study will translate to human disease and it would therefore be important to test other routes of MHV infection and doses that have been described to produce a more comparable phenotype to human coronavirus disease (<em>cf.</em> Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/ 83/18/9258</a>). Another important shortcoming of this study is the lack of any information on functional deficits or changes in Ly6E-deficient immune cells and how this might relate to the phenotype observed. Overall, the <em>in vitro</em> experiments are more convincing than the <em>in vivo</em> studies which appear somewhat limited.</p>
</div>
<div id="significance-41" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-41" class="anchor"></a>Significance</h4>
<p>Despite some shortcomings, the experiments performed in this study suggest a novel and somewhat unexpected role of Ly6E in the protection against coronaviruses across species. These findings are of relevance and should be further explored in ongoing research on potential coronavirus therapies. Yet an important caveat pertains to the authors’ suggestion that “therapeutic mimicking of Ly6E action” may constitute a first line of defense against novel coronaviruses since their own prior work demonstrated that Ly6E can enhance rather than curtail infection with influenza A and other viruses.</p>
</div>
<div id="credit-46" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-46" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients" class="anchor"></a>A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.06.20031856]</span></p>
<div id="keywords-47" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-47" class="anchor"></a>Keywords</h4>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
</div>
<div id="main-findings-44" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-44" class="anchor"></a>Main Findings</h4>
<p>While RT-PCR is being used currently to routinely diagnose infection with SARS-CoV-2, there are significant limitations to the use of a nucleic acid test that lead to a high false-negative rate. This article describes ELISAs that can measure IgM and IgG antibodies against the N protein of SARS-CoV-2 to test samples from 238 patients (153 positive by RT-PCR and 85 negative by RT-PCR) at different times after symptom onset. The positivity rate of the IgM and/or IgG ELISAs was greater than that of the RT-PCR (81.5% compared to 64.3%) with similar positive rates in the confirmed and suspected cases (83% and 78.8%, respectively), suggesting that many of the suspected but RT-PCR-negative cases were also infected. The authors also found that the ELISAs have higher positive rates later after symptom onset while RT-PCR is more effective as a diagnostic test early during the infection.</p>
</div>
<div id="limitations-45" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-45" class="anchor"></a>Limitations</h4>
<p>I cannot identify any limitations to this study.</p>
</div>
<div id="significance-42" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-42" class="anchor"></a>Significance</h4>
<p>The authors make a strong case for using a combination of ELISA and RT-PCR for diagnosis of infection with SARS-CoV-2, especially considering the dynamics of positivity rates of RT-PCR and ELISA. Fewer false-negative diagnoses would improve infection control and patient management.</p>
</div>
<div id="credit-47" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-47" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Early, low-dose and short-term application of corticosteroid treatment in patients with severe COVID-19 pneumonia: single-center experience from Wuhan, China</strong></p>
<p>Corticosteroids, methylprednisolone, COVID-19 pneumonia, ARDS</p>
<p>This retrospective study evaluated the use of intravenous methylprednisolone to treat severe COVID-19 pneumonia in a cohort of 46 patients. The severity of the disease was determined according to version 5 of the Coronavirus Pneumonia Diagnosis and Treatment Plan by the National Health Committee of the People’s Republic of China. The percentage of patients with comorbidities was 32%, and it was reported similar between methylprednisolone treated and untreated groups. The results showed that the group of patients that received methylprednisolone (n=26) had a shorter number of days with fever than patients that did not received methylprednisolone (n=20); they also had a faster improvement of peripheral capillary oxygen saturation (SpO2), and better outcomes in follow-up CT scans of the lungs. The dosage of methylprednisolone was reported to be 1-2mg/kg/d for 5-7 days, although there is no information about the concrete dosages for each patient. From the 46 patients, 43 recovered and were discharged, while 3 cases were fatal. Patients without administration of methylprednisolone needed longer periods of supplemental oxygen therapy, though there is no reference to the number of patients requiring mechanical ventilation. Interestingly, there were no significant differences in leucocyte and lymphocyte counts nor in the levels of IL-2, IL-4, IL-6 or IL-10 after treatment with methylprednisolone.</p>
<p>Some of our main criticisms to this study are also pointed-out by the authors themselves: it is a retrospective single-center study with no validation cohort and without mid- and long-term follow-ups. The reported mortality was 7% (3/46) and did not appear to be affected by corticosteroid treatment: one patient died in the group that did not receive methylprednisolone, while two patients died in the methylprednisolone treatment group. Additionally, although the authors mention that patients received cotreatments, such as antiviral therapy and antibiotics, there is no mention of differences between the prevalence of other medications between the two groups. Unfortunately, there is also no indication on whether the patients receiving methylprednisolone were discharged earlier; the authors merely refer that the symptoms and signs improved faster.</p>
<p>Corticosteroid have been widely used as therapy for acute respiratory distress syndrome (ARDS), including in infections by SARS-CoV, so these findings in COVID-19 patients are not unexpected. The implications of this study for the current pandemic due to SARS-CoV-2 require evaluation in future clinical trials, especially in a randomized way and in combination with and comparison to other immunosuppressive and immunomodulatory agents, including hydroxychloroquine. Nevertheless, based on this report, the intravenous application of methylprednisolone with the intention of strengthening the immunosuppressive treatment and controlling the cytokine storm appears to be safe in COVID-19 patients, and it might successfully shorten the recovery period.</p>
<p>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</p>
</div>
</div>
<div id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2" class="anchor"></a>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.06.980037]</span></p>
<div id="keywords-48" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-48" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
</div>
<div id="main-findings-45" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-45" class="anchor"></a>Main Findings</h4>
<p>Whole genome sequencing-based comparisons of the 2003 Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019 SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with this, the authors tested cross-reactivity of murine monoclonal antibodies (mAbs) previously generated against the SARS-CoV spike (S) glycoprotein involved in viral entry. One of the screened mAb, 1A9, was able to bind and cross-neutralize multiple strains of SARS-CoV, as well as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was generated using an immunogenic fragment in the S2 subunit of SARS-CoV and binds through a novel epitope within the S2 subunit at amino acids 1111-1130. It is important to note that CD8+ T lymphocyte epitopes overlap with these residues, suggesting that S2 subunit could be involved in inducing both, humoral and cell-mediated immunity.</p>
</div>
<div id="limitations-46" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-46" class="anchor"></a>Limitations</h4>
<p>The authors used previously generated mouse mAbs against the S protein in SARS-CoV expressed in mammalian cell line. Future experimental validation using COVID-19 patient samples is needed to validate these findings. In addition, the results of these studies are predominantly based on in vitro experiments and so, evaluating the effects of the mAb 1A9 in an animal model infected with this virus will help us better understand the host immune responses in COVID-19 and potential therapeutic vaccines.</p>
</div>
<div id="significance-43" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-43" class="anchor"></a>Significance</h4>
<p>This study identified mAbs that recognize the new coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in developing diagnostic assays for COVID-19.</p>
</div>
<div id="credit-48" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-48" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>SARS-CoV-2 is sensitive to type I interferon pretreatment</strong></p>
<p>Lokugamage et al. 2020</p>
<p>Keywords: Treatment, cytokines, interferons</p>
<p>Summary</p>
<p>Lokugamage et al have found that SARS-CoV-2 has sensitivity to type I interferon treatment <em>in vitro</em>. They report a significant difference in interferon stimulated genes (ISGs) and transcription factor expression between SARS-CoV and SARS-CoV-2 infected VERO cells pretreated with IFN-α. They attribute these differences to significant changes in the viral genes ORF6 and ORF 3b, which have been reported to function as interferon antagonist in SARS-CoV. They suggest using interferon type I treatment as a potential pathway to further analyze in animal models and humans.</p>
<p>Caveats</p>
<p>The study was carried out <em>in vitro</em> in an African monkey fibroblast kidney cell line. They show sensitivity to IFN in cells infected <em>in vitro</em> and do not mention any other pathway.</p>
<p>[]{#_heading=h.gjdgxs .anchor}This is extrapolated from sequence homology of SARS-CoV and SARS-CoV-2 proteins. The study does not test the ability of ORF6 and ORF 3b to inhibit the IFN pathway directly.</p>
<p>Importance of findings</p>
<p>Interferon type I therapies are readily available and have been tested against several viral infections such as SARS-CoV without success. Nevertheless, these results suggest that the difference in homologies between viral proteins ORF6 and ORF 3b of SARS-CoV and SARS-CoV-2 is enough to disrupt this viral defense mechanism.</p>
<p>[]{#_heading=h.1b4b0g63ddt .anchor}<em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population" class="section level3">
<h3 class="hasAnchor">
<a href="#mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population" class="anchor"></a>Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h3>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.08.20031229]</span></p>
<div id="keywords-49" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-49" class="anchor"></a>Keywords</h4>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
</div>
<div id="main-findings-46" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-46" class="anchor"></a>Main Findings</h4>
<p>Retrospective study of the clinical characteristics of 752 patients infected with COVID-19 at Chinese PLA General Hospital, Peking Union Medical College Hospital, and affiliated hospitals at Shanghai University of medicine &amp; Health Sciences. This study is the first one that compares PB from healthy controls from the same regions in Shanghai and Beijing, and infected COVID-19 patients to standardize a reference range of WBCs of people at high risk.</p>
</div>
<div id="limitations-47" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-47" class="anchor"></a>Limitations</h4>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T cells- correlated with mortality (WBCs are significantly lower in severe or critical UCI patients vs mild ones). Based on 14,117 normal controls in Chinese Han population (ranging in age from 18-86) it is recommended that reference ranges of people at high risk of COVID-19 infection are CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500 cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this study also reported that the levels of D-dimer, C-reactive protein and IL-6 were elevated in COVID-19 pts., indicating clot formation, severe inflammation and cytokine storm.</p>
</div>
<div id="significance-44" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-44" class="anchor"></a>Significance</h4>
<p>This study sets a threshold to identify patients at risk by analyzing their levels of leukocytes, which is an easy and fast approach to stratify individuals that require hospitalization. Although the study is limited (only counts of WBC are analyzed and not its profile) the data is solid and statistically robust to correlate levels of lymphopenia with mortality.</p>
</div>
<div id="credit-49" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-49" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19" class="anchor"></a>Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h3>
<p>Chen et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.09.20033068]</span></p>
<div id="keywords-50" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-50" class="anchor"></a>Keywords</h4>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
</div>
<div id="main-findings-47" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-47" class="anchor"></a>Main Findings</h4>
<p>This is a retrospective study involving 101 death cases with COVID-19 in Wuhan Jinyintan Hospital. The aim was to describe clinical, epidemiological and laboratory features of fatal cases in order to identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by clinical diagnostics. Males dominated the number of deaths and the average age was 65.46 years. All patients died of respiratory failure and multiple organs failure, except one (acute coronary syndrome). The predominant comorbidities were hypertension (42.57%) and diabetes (22.77%). 25.74% of the patients presented more than two underlying diseases. 82% of patients presented myocardial enzymes abnormalities at admission and further increase in myocardial damage indicators with disease progression: patients with elevated Troponin I progressed faster to death. Alterations in coagulation were also detected. Indicators of liver and kidney damage increased 48 hours before death. The authors studied the deceased patients’ blood type and presented the following results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O (18.19%), which is inconsistent with the distribution in Han population in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%), followed by cough and dyspnea. The medium time from onset of symptoms to acute respiratory distress syndrome (ARDS) development was 12 days. Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented abnormal lung imaging at admission and most had double-lung abnormalities. Related to the laboratorial findings some inflammatory indicators gradually increased during the disease progression, such as IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive protein (CRP), while platelets numbers decreased. The authors also reported an initial lymphopenia that was followed by an increase in the lymphocytes numbers. Neutrophil count increased with disease progression.</p>
<p>The patients received different treatments such as antiviral drugs (60.40%), glucocorticoids, thymosin and immunoglobulins. All patients received antibiotic treatment and some received antifungal drugs. All patients received oxygen therapy (invasive or non-invasive ones).</p>
</div>
<div id="limitations-48" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-48" class="anchor"></a>Limitations</h4>
<p>This study involves just fatal patients, lacking comparisons with other groups of patients e.g. patients that recovered from COVID-19. The authors didn’t discuss the different approaches used for treatments and how these may affect the several parameters measured. The possible relationship between the increase of inflammatory indicators and morbidities of COVID-19 are not discussed.</p>
</div>
<div id="significance-45" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-45" class="anchor"></a>Significance</h4>
<p>This study has the largest cohort of fatal cases reported so far. The authors show that COVID-19 causes fatal respiratory distress syndrome and multiple organ failure. This study highlights prevalent myocardial damage and indicates that cardiac function of COVID-19 patients should be carefully monitored. The data suggest that Troponin I should be further investigated as an early indicator of patients with high risk of accelerated health deterioration. Secondary bacterial and fungal infections were frequent in critically ill patients and these need to be carefully monitored in severe COVID-19 patients. Differences in blood type distribution were observed, suggesting that type A is detrimental while type O is protective – but further studies are needed to confirm these findings and elucidate if blood type influences infection or disease severity. Several inflammatory indicators (neutrophils, PCT, CRP and IL-6, D-dimer) increased according to disease severity and should be assessed as biomarkers and to better understand the biology of progression to severe disease.</p>
</div>
<div id="credit-50" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-50" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Aerosol and surface stability of HCoV-19 (SARS-COV-2) compared to SARS-CoV-1</strong></p>
<p> </p>
<p>van Doremalen, N. et al. New England Journal of Medicine. <span class="math display">\[@[doi:
10.1056/NEJMc2004973](about:blank)\]</span></p>
<p><strong>Keywords</strong></p>
<p>COVID-19, aerosol, transmission, fomites</p>
<p> </p>
<p><strong>Main Findings</strong></p>
<p>The authors analyzed and compared the stability of viable SARS-COV-2 and SARS-CoV-1 inoculums in five environmental conditions (aerosol, copper, cardboard, steel, and plastic) by using</p>
<p>Bayesian regression model. It was reported that SARS-COV-2 was still detected in aerosols at 3 hours, with an exponential reduction in infectious titer that was similarly observed for SARS-CoV-1. The study also concluded that both SARS-COV-2 and SARS-CoV-1 are more stable on stainless steel and plastic than cardboard and copper. Viable SARS-CoV-2 was detected up to 72 hours on stainless steel and plastic. On copper and cardboard, SARS-COV-2 was viable up to 4 hours and 24 hours, respectively, compared to SARS-CoV-1 which could be detected up to 8 hours on both material types. The half-lives between both viruses are similar, except for on cardboard. The median half-lives are summarized in Table 1 below.</p>
<p> </p>
<p> </p>
<table class="table">
<colgroup><col width="97%"></colgroup>
<tbody>
<tr class="odd">
<td align="center">SARS-COV-2 SARS-CoV-1 difference</td>
</tr>
<tr class="even">
<td align="center">Median half-life (hrs) Median half-life (hrs) (SARS-COV-2 - SARS-Cov-1)</td>
</tr>
<tr class="odd">
<td align="center">difference (hrs)</td>
</tr>
</tbody>
</table>
<p>aerosols 1.09 1.18 -0.0913</p>
<p>copper 0.774 1.5 -0.735</p>
<p>cardboard 3.46 0.587 2.85</p>
<p>steel 5.63 4.16 1.46</p>
<p>plastic 6.81 7.55 -0.722 —————————————————————————————–</p>
<p> </p>
<p><strong>Limitation of the study</strong></p>
<p>The strain used in the study was SARS-COV-2 nCoV-WA1-2020 (MN985325.1) from the first case of 2019 novel coronavirus in the US. However, mutation throughout the course of the pandemic is inevitable and may cause unpredictable consequences on its transmissibility and disease severity. Thus, follow-up on samples from various patients in different geographic and temporal time points should be conducted.</p>
<p><strong> </strong></p>
<p><strong>Significance</strong></p>
<p>[]{#_gjdgxs .anchor}The results support that modes of SARS-COV-2 transmission can be attributed to both aerosol and fomites, due to extended viability for hours in aerosol and up to 72 hours on stainless steel surfaces. The types of plastic, cardboard, copper, and stainless materials were selected to reflect typical hospital and household situations. It is important to compare with the SARS-CoV-1 as similarities between the two suggests methods of mitigating the pandemic by abrogating transmission both in the community and hospital.</p>
<p><em>Review by Joan Shang as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of medicine, Mount Sinai.</em></p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell membrane during infection. The spike (S) protein plays an essential role in this step (1). This work sheds light on the particularities of the S protein from SARS-CoV2 and compares it to previously existing coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for hACE2 compared to SARS-CoV. Previous work by this group had established the potential of a previous version of the peptide (EK1) in reducing the infectivity of several coronavirus by blocking the S protein interactions with the host cell (2). In this work, authors present an improved EK1C4 version of this peptide. In vitro experiments on human cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2 and other coronavirus. Administration of EK1C4 in prophylaxis or shortly after exposure to the HCoV-OC43 coronavirus protected newborn mice from virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be related to its increased infectivity compared to CoV-S. However, a previous report suggested that the enhanced cell-to-cell fusion mediated by this furin cleavage might not necessarily translate into enhanced viral infectivity (3). The mutation of this site, or inhibition of furin in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if this mechanism is actually involved in SARS-CoV2 pathogenesis, and potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is based on the coronavirus HCoV-OC43. Further work should aim to establish the potential of EK1C4 for prevention of SARS-CoV-2 replication in more relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is, however, acknowledged, that experimentation in vitro included both SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been previously examined (2), the potential side effects and pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a paramount element of epidemic control, specially in the time before a vaccine can be readily available. Similar strategies have been followed in the past in response to threats of this nature (4,5). This work’s relevance is limited by the absence of relevant animal models of SARS-CoV-2 infection. Further pharmacological analyses would also be necessary to ensure the safety of EK1C4 in animal models before a potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike protein and acquisition of fusion competence. Vol. 4, Viruses. Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S. Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide That Targets Viral Attachment to Cells. J Virol. 2006 Dec 15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human antimicrobial peptides as therapeutics for viral infections. Vol. 11, Viruses. MDPI AG; 2019.</p>
</div>
</div>
<div id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility" class="section level3">
<h3 class="hasAnchor">
<a href="#relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility" class="anchor"></a>Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h3>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.11.20031096]</span></p>
<div id="keywords-51" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-51" class="anchor"></a>Keywords</h4>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
</div>
<div id="main-findings-48" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-48" class="anchor"></a>Main Findings</h4>
<p>These authors compared the ABO blood group of 2,173 patients with RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the ABO blood group distribution in unaffected people in the same cities from previous studies (2015 and 2010 for Wuhan and Shenzhen, respectively). They found that people with blood group A are statistically over-represented in the number of those infected and who succumb to death while those with blood group O are statistically underrepresented with no influence of age or sex.</p>
</div>
<div id="limitations-49" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-49" class="anchor"></a>Limitations</h4>
<p>This study compares patients with COVID-19 to the general population but relies on data published 5 and 10 years ago for the control. The mechanisms that the authors propose may underlie the differences they observed require further study.</p>
</div>
<div id="significance-46" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-46" class="anchor"></a>Significance</h4>
<p>Risk stratification based on blood group may be beneficial for patients and also healthcare workers in infection control. Additionally, investigating the mechanism behind these findings could lead to better developing prophylactic and therapeutic targets for COVID-19.</p>
</div>
<div id="credit-51" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-51" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15" class="section level3">
<h3 class="hasAnchor">
<a href="#the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15" class="anchor"></a>The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h3>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.11.987016]</span></p>
<div id="keywords-52" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-52" class="anchor"></a>Keywords</h4>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
</div>
<div id="main-findings-49" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-49" class="anchor"></a>Main Findings</h4>
<p>This study reconsiders the use of inhaled corticosteroids in the treatment of pneumonia by coronavirus. Corticosteroids were associated with increased mortality for SARS in 2003 and for MERS in 2013, probably due to that fact that systemic corticosteroids suppress the innate immune system, resulting in increased viral replication. However, some steroid compounds might block coronavirus replication. The authors screened steroids from a chemical library and assessed the viral growth suppression and drug cytotoxicity. Ciclesonide demonstrated low cytotoxicity and potent suppression of MERS-CoV viral growth. The commonly used systemic steroids cortisone, prednisolone and dexamethasone did not suppress viral growth, nor did the commonly used inhaled steroid fluticasone. To identify the drug target of virus replication, the authors conducted 11 consecutive MERS-CoV passages in the presence of ciclesonide or mometasone, and they could generate a mutant virus that developed resistance to ciclesonide, but not to mometasone. Afterwards, they performed next-generation sequencing and identified an amino acid substitution in nonstructural protein 15 (NSP15) as the predicted mechanism for viral resistance to ciclesonide. The authors were able to successfully generate a recombinant virus carrying that amino acid substitution, which overcome the antiviral effect of ciclesonide, suggesting that ciclosenide interacts with NSP15. The mutant virus was inhibited by mometasone, suggesting that the antiviral target of mometasone is different from that of ciclesonide. Lastly, the effects of ciclesonide and mometason on suppressing the replication of SARS-CoV-2 were evaluated. Both compounds were found to suppress viral replication with a similar efficacy to lopinavir.</p>
</div>
<div id="limitations-50" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-50" class="anchor"></a>Limitations</h4>
<p>Most of the experiments, including the identification of the mutation in NSP15 were conducted with MERS-CoV. This is not the closest related virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the initial experiments with SARS-CoV, or preferably SARS-CoV-2, is essential. The manuscript should address this and, therefore, it will require considerable editing for organization and clarity. Also, in terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein contains several predicted B and T cell immunogenic epitopes that are shared with other coronaviruses, some studies have shown critical differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct cytotoxic effects of viral replication. To evaluate this in other cell lines, including human airway epithelial cells, is crucial, as the infectivity of coronavirus strains greatly varies in different cell lines,</p>
</div>
<div id="significance-47" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-47" class="anchor"></a>Significance</h4>
<p>Nevertheless, these findings encourage evaluating ciclesonide and mometasone as better options for patients with COVID-19 in need of inhaled steroids, especially as an alternative to other corticosteroids that have been shown to increase viral replication in vitro. This should be evaluated in future clinical studies.</p>
</div>
<div id="credit-52" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-52" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#a-human-monoclonal-antibody-blocking-sars-cov-2-infection" class="anchor"></a>A human monoclonal antibody blocking SARS-CoV-2 infection **</h3>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.11.987958]</span></p>
<div id="keywords-53" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-53" class="anchor"></a>Keywords</h4>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
</div>
<div id="main-findings-50" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-50" class="anchor"></a>Main Findings</h4>
<p>The authors reported a human monoclonal antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same family of corona viruses. For identifying mAbs, supernatants of a collection of 51 hybridomas raised against the spike protein of SARS-CoV (SARS-S) were screened by ELISA for cross-reactivity against the spike protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S, one of which (47D11) exhibited cross-neutralizing activity for SARS-S and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1 isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml respectively. 47D11 mAb bound a conserved epitope on the spike receptor binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and SARS2-S with similar affinities. Interestingly, binding of 47D11 to SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2 receptor-expressing cells assayed by flow cytometry.</p>
</div>
<div id="limitations-51" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-51" class="anchor"></a>Limitations</h4>
<p>These results show that the human 47D11 antibody neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown mechanism that is different from receptor binding interference. Alternative mechanisms were proposed but these as yet remain to be tested in the context of SARS-CoV2. From a therapeutic standpoint and in the absence of in vivo data, it is unclear whether the 47D11 ab can alter the course of infection in an infected host through virus clearance or protect an uninfected host that is exposed to the virus. There is a precedent for the latter possibility as it relates to SARS-CoV that was cited by the authors and could turn out to be true for SARS-CoV2.</p>
</div>
<div id="significance-48" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-48" class="anchor"></a>Significance</h4>
<p>This study enabled the identification of novel neutralizing antibody against COV-that could potentially be used as first line of treatment in the near future to reduce the viral load and adverse effects in infected patients. In addition, neutralizing antibodies such as 47D11 represent promising reagents for developing antigen-antibody-based detection test kits and assays.</p>
</div>
<div id="credit-53" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-53" class="anchor"></a>Credit</h4>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies, IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples collected from suspected COVID-19 patients reduces the sensitivity of a fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an important safety precaution in laboratory manipulation of clinical samples. However, the effect of this step on downstream SARS-CoV-2-specific serum antibody assays has not been examined. The authors tested the effect of heat inactivation (56 deg C for 30 minutes) versus no heat inactivation on a fluorescence immunochromatography assay. Heat inactivation reduced all IgM measurements by an average of 54% and most IgG measurements (22/36 samples, average reduction of 50%), consistent with the lower thermal stability of IgM than that of IgG. Heat inactivation caused a subset of IgM but not IgG readings to fall below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat inactivated vs non-inactivated sera, and the use of the same baseline for heat inactivated and non-inactivated sera. The results indicate that heat inactivation affects the quantification of SARS-CoV-2-antibody response, specially IgM, but still allows to distinguish positive specific IgG. Therefore, the effect of heat inactivation should be studied when designing assays that quantitatively associate immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Immunopathological characteristics of coronavirus disease 2019 cases in</strong></p>
<p><strong>Guangzhou, China</strong></p>
<p><strong>Immunology keywords:</strong> SARS-COV-2, COVID-19, T cell, B cell, NK cell, IL-2, IL-6, IL-10, TNF<span class="math inline">\(\alpha\)</span></p>
<p><strong>The main finding of the article: </strong></p>
<p>This study analyzed immune cell populations and multiple cytokines in 31 patients with mild/moderate COVID-19 (ave. 44.5 years) and 25 with severe COVID-19 (ave. 66 years). Samples from patients with fever and negative for the SARS-COV-2 test were used as control. At inpatient admission, total lymphocytes number was decreased in severe patients but not in mild patients, whereas neutrophils were increased in severe patients. CD4+ and CD8+ T cells were diminished in all COVID-19 patients. CD19+ B cells and NK cells were decreased in both mild and severe patients, however, severe patients showed a notable reduction. These data might suggest a profound deregulation of lymphocytes in COVID-19 patients. Further analysis showed significant increases of IL-2, IL-6, IL-10 and TNF<span class="math inline">\(\alpha\)</span> in blood of severe patients at the admission. Sequential samples revealed that IL-2 and IL-6 peaked on day 15-20 and declined thereafter. A moderate increase of IL-4 was seen in mild/moderate patients. Thus, elevation of IL-2, IL-6 can be indicators of severe COVID-19.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>There is no information on when the patients were assessed as severe or mild/moderate, at inpatient admission or later. The authors could have analyzed the correlation between immune cell population and cytokine levels to see, for example, if severe lymphopenia correlated to higher elevation of IL-2.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>While similar findings have already been shown, the data corroborates alterations in circulating adaptive and innate immune cell populations and cytokines, and its correlation to disease severity. The increase of IL-2 and IL-6 at the admission might an indicator to start intensive therapies (like convalescent serum) at an early time.</p>
</div>
</div>
<div id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19" class="anchor"></a>Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h3>
<p>Zhang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.12.20035048]</span></p>
<div id="keywords-54" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-54" class="anchor"></a>Keywords</h4>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
</div>
<div id="main-findings-51" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-51" class="anchor"></a>Main Findings</h4>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were analyzed during acute and convalescent phases (up to day 35) and correlated to the diseases’ severity. The same was done with neutrophil-to-lymphocyte ratio. High IgG levels and high neutrophil-to-lymphocyte ratio in convalescence were both independently associated to the severity of the disease. The simultaneous occurrence of both of these laboratory findings correlated even stronger to the diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly higher levels of IL-6. The authors propose that a robust IgG response leads to immune-mediated tissue damage, thus explaining the worse outcome in patients with overexuberant antibody response.</p>
</div>
<div id="limitations-52" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-52" class="anchor"></a>Limitations</h4>
<p>A main criticism is that the criteria for stratifying patients in severe vs. non-severe are not described. The only reference related to this is the difference between the percentage of patients who needed mechanical ventilation, which was greater in patients with both high IgG levels and high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels and low neutrophil-to-lymphocyte ratio was treated with mechanical ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of the most interesting findings is that an early increase in anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease. However, as only median values are shown for antibody kinetics curves, the extent of variation in acute phase cannot be assessed.</p>
</div>
<div id="significance-49" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-49" class="anchor"></a>Significance</h4>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio predict severity of COVID-19 independently of each other. An additive predictive value of both variables is noticeable. Importantly, an early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict outcome.</p>
</div>
<div id="credit-54" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-54" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques" class="section level3">
<h3 class="hasAnchor">
<a href="#reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques" class="anchor"></a>Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h3>
<p>Bao et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.13.990226]</span></p>
<div id="keywords-55" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-55" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
</div>
<div id="main-findings-52" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-52" class="anchor"></a>Main Findings</h4>
<p>This study addresses the issue or acquired immunity after a primary COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were intratracheally infected with SARS-CoV-2 and two out of the four were re-infected at 28 days post initial infection (dpi) with the same viral dose after confirming the recovery by the absence of clinical symptoms, radiological abnormalities and viral detection (2 negative RT-PCR tests). While the initial infection led the viral loads in nasal and pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3 dpi in all four monkeys, viral loads in the swabs tested negative after reinfection in the two reinfected monkeys. In addition, the necropsies from a monkey (M1) at 7 days after primary infection, and another monkey (M3) at 5 days post reinfection, revealed the histopathological damages and viral replication in the examined tissues from M1, while no viral replication as well as no histological damages were detected in the tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting the production of protective neutralizing antibodies in these monkeys. Overall, this study indicates that primary infection with SARS-CoV-2 may protect from subsequent exposure to the same virus.</p>
</div>
<div id="limitations-53" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-53" class="anchor"></a>Limitations</h4>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15 days after the onset of symptoms. In the infected monkeys in this study, virus were detected from day 1 after the infection, declining to undetectable level by day 15 post infection. It may suggest that there is a faster viral clearance mechanism in monkeys, therefore the conclusions of reinfection protection for humans need to be carefully considered. In addition, only two monkeys were re-infected in this study and the clinical signs of these monkeys were not similar: M3 did not show weight loss and M4 showed relatively higher fever on the day of infection and the day of re-challenge.</p>
</div>
<div id="significance-50" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-50" class="anchor"></a>Significance</h4>
<p>This study showed clear viral clearance and no indications of relapse or viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus macaque model. These results support the idea that patients with full recovery (two negative RT-PCR results) may also be protected from secondary SARS-CoV-2 infection. Recovered patients may be able to reintegrate to normal public life and provide protective serum perhaps even if having had a mild infection. The results are also encouraging for successful vaccine development against SARS-CoV-2.</p>
</div>
<div id="credit-55" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-55" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov" class="section level3">
<h3 class="hasAnchor">
<a href="#a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov" class="anchor"></a>A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h3>
<p><span class="citation">[@doi:10.1101/2020.03.13.991570]</span></p>
<div id="keywords-56" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-56" class="anchor"></a>Keywords</h4>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
</div>
<div id="main-findings-53" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-53" class="anchor"></a>Main Findings</h4>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing antibodies isolated from convalescent SARS-CoV patients may offer insight into cross-reactive antibodies targeting SARS-CoV-2. In particular, they find that the receptor-binding domain (RBD) of SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of SARS-CoV S protein that binds to the CR3022 neutralizing antibody. CR3022 also displays increased affinity for the “up” conformation of the SARS-CoV-2 S protein compared to the “down” conformation as it does for the SARS-CoV S protein. Therefore, the authors propose that this cross-reactive antibody may confer some degree of protection <em>in vivo</em> even if it fails to neutralize <em>in vitro</em>.</p>
</div>
<div id="limitations-54" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-54" class="anchor"></a>Limitations</h4>
<p>Although the authors offer a logical rationale for identifying cross-reactive neutralizing antibodies derived from SARS-CoV, their study using only CR3022 failed to demonstrate whether this approach will be successful. After all, CR3022 failed to neutralize <em>in vitro</em> despite the binding affinity to a similar epitope on SARS-CoV-2. They would benefit from testing more candidates and using an <em>in vivo</em> model to demonstrate their claim that protection may be possible in the absence neutralization if combinations are used <em>in vivo.</em></p>
</div>
<div id="significance-51" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-51" class="anchor"></a>Significance</h4>
<p>The ability to make use of previously characterized neutralizing antibodies for conserved epitopes can expedite drug design and treatment options.</p>
</div>
<div id="credit-56" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-56" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr" class="section level3">
<h3 class="hasAnchor">
<a href="#highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr" class="anchor"></a>Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h3>
<p>Dong et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.14.20036129]</span></p>
<div id="keywords-57" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-57" class="anchor"></a>Keywords</h4>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
</div>
<div id="main-findings-54" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-54" class="anchor"></a>Main Findings</h4>
<p>The authors present a digital PCR (dPCR) diagnostic test for SARS-CoV-2 infection. In 103 individuals that were confirmed in a follow-up to be infected, the standard qPCR test had a positivity rate of 28.2% while the dPCR test detected 87.4% of the infections by detecting an additional 61 positive cases. The authors also tested samples from close contacts (early in infection stage) and convalescing individuals (late in infection stage) and were able to detect SARS-CoV-2 nucleic acid in many more samples using dPCR compared to qPCR.</p>
</div>
<div id="limitations-55" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-55" class="anchor"></a>Limitations</h4>
<p>I did not detect limitations.</p>
</div>
<div id="significance-52" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-52" class="anchor"></a>Significance</h4>
<p>The authors make a strong case for the need for a highly sensitive and accurate confirmatory method for diagnosing COVID-19 during this outbreak and present a potential addition to the diagnostic arsenal. They propose a dPCR test that they present has a dramatically lower false negative rate than the standard RT-qPCR tests and can be especially beneficial in people with low viral load, whether they are in the earlier or later stages of infection.</p>
</div>
<div id="credit-57" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-57" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein" class="anchor"></a>SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h3>
<p>Wang et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.14.988345]</span></p>
<div id="keywords-58" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-58" class="anchor"></a>Keywords</h4>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
</div>
<div id="main-findings-55" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-55" class="anchor"></a>Main Findings</h4>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through the interaction of the viral spike protein (SP) and the immunoglobulin superfamily protein CD147 (also known as Basigin). Using an in-house developed humanized antibody against CD147 (maplazumab), they show that blocking CD147 decreases viral replication in Vero E6 cells. Using surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly, they utilize immune-election microscopy to show spike protein and CD147 localize to viral inclusion bodies of Vero E6 cells.</p>
</div>
<div id="limitations-56" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-56" class="anchor"></a>Limitations</h4>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits SARS-CoV-2 replication by testing cell growth and viral load in cells infected with SARS-CoV-2, however there are key pieces of this experiment that are missing. First, the authors fail to use a non-specific antibody control. Second, the authors claim that viral replication is inhibited, and that they test this by qPCR, however this data is <strong>not shown.</strong> To further prove specificity, the authors should introduce CD147 to non-susceptible cells and show that they become permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and SP through SPR, ELISA, and Co-IP, and this data seems generally convincing. The electron microscopy provides further correlative evidence that SARS-CoV-2 may interact with CD147 as they are both found in the same viral inclusion body. A quantification of this data would make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and statistics to show that the data are reproducible and statistically significant.</p>
</div>
<div id="significance-53" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-53" class="anchor"></a>Significance</h4>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell surface protein ACE2 for cell entry, yet ACE2 is highly expressed in heart, kidney, and intestinal cells, raising the concern that blocking ACE2 would result in harmful side effects <span class="citation">[@doi:10.1093/ndt/gfu302]</span> CD147 on the other hand is highly expressed in various tumor types, inflamed tissues, and pathogen infected cells, suggesting that the inhibition of CD147 would not result in major side effects <span class="citation">[@doi:10.1016/j.yexmp.2018.05.001; @pmid:20551248]</span> The research in this paper has resulted in an ongoing clinical trail in China to test the safety and efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov identifier NCT04275245).</p>
</div>
<div id="credit-58" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-58" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19" class="anchor"></a>Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h3>
<p>Huang et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.15.20033472]</span></p>
<div id="keywords-59" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-59" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
</div>
<div id="main-findings-56" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-56" class="anchor"></a>Main Findings</h4>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of peripheral blood mononuclear cells isolated from whole blood samples of COVID-19 patients (n=10). Data was compared to scRNAseq of samples collected from patients with influenza A (n=1), acute pharyngitis (n=1), and cerebral infarction (n=1), as well as, three healthy controls. COVID-19 patients were categorized into those with moderate (n=6), severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across all COVID-19 disease levels revealed 56 different cellular subtypes, among 17 immune cell types; comparisons between each category to the normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19 patients have circulating antibodies against known viral antigens, including EBV, HIV, influenza A, and other RNA viruses. This may suggest that the immune response to SARS-CoV-2 infection elicits production of antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients with other conditions (influenza A, acute pharyngitis, and cerebral infarction), the authors identified the <strong>interferon-MAPK signaling pathway as a major response to SARS-CoV-2 infection</strong>. The authors performed quantitative real-time reverse transcriptase polymerase chain reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em> and <em>FOS</em>. These samples were collected from a separate cohort of COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild, n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19 patients and down-regulation in cured patients. <strong>The authors propose that <em>FOS</em> may be a candidate marker gene for curative COVID-19 disease</strong>.</p>
</div>
<div id="limitations-57" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-57" class="anchor"></a>Limitations</h4>
<p>The sample size of this study is limited. To further delineate differences in the immune profile of peripheral blood of COVID-19 patients, a greater sample size is needed, and longitudinal samples are needed, as well. A better understanding of the immunological interactions in cured patients, for example, would require a profile before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to potential autoimmunity and immune deficiency to distinguish different severities of COVID-19 disease. However, this requires an expanded number of samples and a more robust organization of specific immune cell subtypes that can be compared across different patients. Importantly, this criterion is likely needed to ensure greater specificity in identifying markers for COVID-19 infection and subsequent immune response.</p>
</div>
<div id="significance-54" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-54" class="anchor"></a>Significance</h4>
<p>At the single-cell level, COVID-19 disease has been characterized in the lung, but a greater understanding of systemic immunological responses is furthered in this study. Type I interferon is an important signaling molecule for the anti-viral response. The identification of the interferon-MAPK signaling pathway and the differential expression of MAPK regulators between patients of differing COVID-19 severity and compared to cured patients may underscore the importance of either immune deficiency or autoimmunity in COVID-19 disease.</p>
</div>
<div id="credit-59" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-59" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection-" class="section level3">
<h3 class="hasAnchor">
<a href="#cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection-" class="anchor"></a>Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h3>
<p>Lv et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.15.993097]</span></p>
<div id="keywords-60" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-60" class="anchor"></a>Keywords</h4>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity, cross-neutralization, antibody, human patients, mouse</p>
</div>
<div id="main-findings-57" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-57" class="anchor"></a>Main Findings</h4>
<p>The authors explore the antigenic differences between SARS-CoV-2 and SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the spike protein between SARS-CoV-2 and SARS-CoV was found to be common, mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients. Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV, with low neutralization activity. No cross-neutralization response was detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of plasma collected from mice infected or immunized with SARS-CoV-2 or SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2 S ectodomain. Cross-neutralization activity was not detected in any of the mouse plasma samples.</p>
</div>
<div id="limitations-58" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-58" class="anchor"></a>Limitations</h4>
<p>The size of each patient cohort is insufficient to accurately determine the frequency of cross-reactivity and cross-neutralization in the current SARS-CoV-2 pandemic. Recruitment of additional patients from a larger range of geographical regions and time points would also enable exploration into the effect of the genetic diversity and evolution of the SARS-CoV-2 virus on cross-reactivity. This work would also benefit from the mapping of specific epitopes for each sample. Future studies may determine whether the non-neutralizing antibody responses can confer <em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
</div>
<div id="significance-55" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-55" class="anchor"></a>Significance</h4>
<p>The cross-reactive antibody responses to S protein in the majority of SARS-CoV-2 patients is an important consideration for development of serological assays and vaccine development during the current outbreak. The limited extent of cross-neutralization demonstrated in this study indicates that vaccinating to cross-reactive conserved epitopes may have limited efficacy, presenting a key concern for the development of a more universal coronavirus vaccine to address the global health risk of novel coronavirus outbreaks.</p>
</div>
<div id="credit-60" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-60" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study" class="section level3">
<h3 class="hasAnchor">
<a href="#the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study" class="anchor"></a>The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h3>
<p>Duan et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.16.20036145]</span></p>
<div id="keywords-61" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-61" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
</div>
<div id="main-findings-58" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-58" class="anchor"></a>Main Findings</h4>
<p>This is the first report to date of convalescent plasma therapy as a therapeutic against COVID-19 disease. This is a feasibility pilot study. The authors report the administration and clinical benefit of 200 mL of convalescent plasma (CP) (1:640 titer) derived from recently cured donors (CP selected among 40 donors based on high neutralizing titer and ABO compatibility) to 10 severe COVID-19 patients with confirmed viremia. The primary endpoint was the safety of CP transfusion. The secondary endpoint were clinical signs of improvement based on symptoms and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate any potential residual virus in the plasma samples, without compromising neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete disappearance of clinical symptoms, including fever, cough, shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in neutralizing antibody titers in 5 patients. Persistence of neutralizing antibody levels in 4 patients.</p></li>
</ul>
</div>
<div id="limitations-59" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-59" class="anchor"></a>Limitations</h4>
<p>It is important to note that most recipients had high neutralization titers of antibodies before plasma transfusion and even without transfusion it would be expected to see an increase in neutralizing antibodies over time. In addition to the small sample set number (n=10), there are additional limitations to this pilot study:</p>
<ol style="list-style-type: decimal">
<li><p>All patients received concurrent therapy, in addition to the CP transfusion. Therefore, it is unclear whether a combinatorial or synergistic effect between these standards of care and CP transfusion contributed to the clearance of viremia and improvement of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect to the administration of CP transfusion. So, the definitive impact of CP transfusion on immune dynamics and subsequent viral load is not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
</div>
<div id="significance-56" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-56" class="anchor"></a>Significance</h4>
<p>For the first time, a pilot study provides promising results involving the use of convalescent plasma from cured COVID-19 patients to treat others with more severe disease. The authors report that the administration of a single, high-dose of neutralizing antibodies is safe. In addition, there were encouraging results with regards to the reduction of viral load and improvement of clinical outcomes. It is, therefore, necessary to expand this type of study with more participants, in order to determine optimal dose and treatment kinetics. It is important to note that CP has been studied to treat H1N1 influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to be effective in treating these infections.</p>
</div>
<div id="credit-61" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-61" class="anchor"></a>Credit</h4>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial" class="anchor"></a>Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.20037135]</span></p>
<div id="keywords-62" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-62" class="anchor"></a>Keywords</h4>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
</div>
<div id="main-findings-59" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-59" class="anchor"></a>Main Findings</h4>
<p>This study was a single-arm, open label clinical trial with 600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients who refused participation or patients from another center not treated with HCQ were included as negative controls (n = 16). Among the patients in the treatment arm, 6 received concomitant azithromycin to prevent superimposed bacterial infection. The primary endpoint was respiratory viral loads on day 6 post enrollment, measured by nasopharyngeal swab followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n = 8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to be synergistic with HCQ, as 6/6 patients receiving combined treatment had complete viral clearance (p &lt; 0.001).</p>
</div>
<div id="limitations-60" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-60" class="anchor"></a>Limitations</h4>
<p>Despite what is outlined above, this study has a number of limitations that must be considered. First, there were originally n = 26 patients in the treatment arm, with 6 lost to follow up for the following reasons: 3 transferred to ICU, 1 discharge, 1 self-discontinued treatment d/t side effects, and 1 patient expired. Total length of clinical follow up was 14 days, but the data beyond day 6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR are listed for the control and treatment arms from D0 – D6. However, the data are not reported in a standard way, with a mix of broadly positive or negative result delineation with Ct (cycle threshold) values, the standard output of real time PCR. It is impossible to compare what is defined as a positive value between the patients in the control and treatment arms without a standardized threshold for a positive test. Further, the starting viral loads reported at D0 in the groups receiving HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs 26.8 respectively), which could explain in part the differences observed in the response to treatment between 2 groups. Finally, patients in the control arm from outside the primary medical center in this study (Marseille) did not actually have samples tested by PCR daily. Instead, positive test results from every other day were extrapolated to mean positive results on the day before and after testing as well (Table 2, footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a promising treatment avenue for COVID-19 patients. However, the limited size of the trial, and the way in which the results were reported does not allow for other medical centers to extrapolate a positive or negative result in the treatment of their own patients with HCQ +/- azithromycin. Further larger randomized clinical trials will be required to ascertain the efficacy of HCQ +/- azithromycin in the treatment of COVID-19.</p>
</div>
<div id="significance-57" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-57" class="anchor"></a>Significance</h4>
<p>Chloroquine is thought to inhibit viral infection, including SARS-Cov-2, by increasing pH within endosomes and lysosomes, altering the biochemical conditions required for viral fusion <span class="citation">[@doi:10.1038/s41422-020-0282-0; @doi:10.1016/j.imlet.2013.07.004]</span>. However, chloroquine also has immuno-modulatory effects that I think may play a role. Chloroquine has been shown to increase CTLA-4 expression at the cell surface by decreasing its degradation in the endo-lysosome pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but in conditions of increased pH, the protein machinery required for degradation is less functional <span class="citation">[@doi:10.1126/science.aaa1663]</span>. As such, more CTLA-4 remains in endosomes and is trafficked back to the cell surface. It is possible that this may also contribute to patient recovery via reduction of cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
</div>
<div id="credit-62" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-62" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids" class="section level3">
<h3 class="hasAnchor">
<a href="#recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids" class="anchor"></a>Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.990317]</span></p>
<div id="keywords-63" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-63" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
</div>
<div id="main-findings-60" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-60" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes in healthy liver (2.92% of all cells) are infectable and support SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid barrier and bile transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation (CLDN1) and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
</div>
<div id="limitations-61" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-61" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct cholangiocyte infection or due to secondary immune/cytokine effects. This study argues for direct damage as it lacks immune contexture; but further studies needed with autopsy samples or organoid-immune cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral response and alarmins secreted upon infection, and furthermore study tropism of various immune cells to conditioned media from organoids infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or alcohol/smoking/obesity-associated liver organoids respond to SARS-CoV-2 infectivity and replication, worth pursuing to experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
</div>
<div id="significance-58" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-58" class="anchor"></a>Significance</h4>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2 infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the results from this study could explain the bile acid accumulation and consequent liver damage observed.</p></li>
</ul>
</div>
<div id="credit-63" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-63" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2" class="section level3">
<h3 class="hasAnchor">
<a href="#the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2" class="anchor"></a>The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.16.994236]</span></p>
<div id="keywords-64" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-64" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
</div>
<div id="main-findings-61" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-61" class="anchor"></a>Main Findings</h4>
<p>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infects cells through S spike glycoprotein binding angiotensin-converting enzyme (ACE2) on host cells. S protein can bind both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a decoy that neutralizes infection. Recombinant sACE2 is now being tested in clinical trials for COVID-19. To determine if a therapeutic sACE2 with higher affinity for S protein could be designed, authors generated a library containing every amino acid substitution possible at the 117 sites spanning the binding interface with S protein. The ACE2 library was expressed in human Expi293F cells and cells were incubated with medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused to GFP. Cells with high or low affinity mutant ACE2 receptor compared to affinity of wild type ACE2 for the RBD were FACS sorted and transcripts from these sorted populations were deep sequenced. Deep mutagenesis identified numerous mutations in ACE2 that enhance RBD binding. This work serves to identify putative high affinity ACE2 therapeutics for the treatment of CoV-2.</p>
</div>
<div id="limitations-62" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-62" class="anchor"></a>Limitations</h4>
<p>The authors generated a large library of mutated ACE2, expressed them in human Expi293F cells, and performed deep mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S protein. While these data serve as a useful resource, the ability of the high affinity ACE2 mutants identified to serve as therapeutics needs further validation in terms of conformational stability when purified as well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally, authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit beneficial host immune responses, which would need further design and validation.</p>
</div>
<div id="significance-59" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-59" class="anchor"></a>Significance</h4>
<p>This study identified structural ACE2 mutants that have potential to serve as therapeutics in the treatment of SARS-CoV-2 upon further testing and validation.</p>
</div>
<div id="credit-64" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-64" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA - seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1 SARS-CoV-2 isolate. The S protein was modified to allow trimerization and increase stability. The authors compared the antibody reactivity of 59 banked human serum samples (non-exposed) and 3 serum samples from confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19 patients, and observed stronger IgG3 response than IgG1. IgM and IgA responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and samples from only three different patients to test for reactivity against the RBD domain and full-length spike protein. It will be important to follow up with a larger number of patient samples to confirm the data obtained. Furthermore, it would be interesting to assess people at different age groups and determine whether unexposed control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are limited, since antibody response should start to be detectable only one to two weeks after infection. Future studies will be required to assess how long after infection this assay allow to detect anti-CoV2 antibodies. Finally, while likely, the association of seroconversion with protective immunity against SARS-Cov-2 infection still needs to be fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2. First, it is now possible to perform serosurveys and determine who has been infected, allowing a more accurate estimate of infection prevalence and death rate. Second, if it is confirmed that re-infection does not happen (or is rare), this assay can be used as a tool to screen healthcare workers and prioritize immune ones to work with infected patients. Third, potential convalescent plasma donors can now be screened to help treating currently infected patients. Of note, this assay does not involve live virus handling. experimentally, this is an advantage as the assay does not require the precautions required by manipulation of live virus. Finally, the recombinant proteins described in this study represent new tools that can be used for further applications, including vaccine development.</p>
</div>
</div>
<div id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model" class="section level3">
<h3 class="hasAnchor">
<a href="#comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model" class="anchor"></a>COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h3>
<p><span class="citation">[@doi:10.1101/2020.03.17.995639]</span></p>
<div id="keywords-65" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-65" class="anchor"></a>Keywords</h4>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
</div>
<div id="main-findings-62" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-62" class="anchor"></a>Main Findings</h4>
<p>This work assesses SARS-CoV-2 infection in young adult and aged cynomolgus macaques. 4 macaques per age group were infected with low-passage clinical sample of SARS-CoV-2 by intranasal and intratracheal administration. Viral presence was assessed in nose, throat and rectum through RT-PCR and viral culture. SARS-CoV-2 replication was confirmed in the respiratory track (including nasal samples), and it was also detected in ileum. Viral nucleocapsid detection by IHC showed infection of type I and II pneumocytes and epithelia. Virus was found to peak between 2 and 4 days after administration and reached higher levels in aged vs. young animals. The early peak is consistent with data in patients and contrasts to SARS-CoV replication. SARS-CoV-2 reached levels below detection between 8 and 21 days after inoculation and macaques established antibody immunity against the virus by day 14. There were histopathological alteration in lung, but no overt clinical signs. At day 4 post inoculation of SARS-CoV-2, two of four animals presented foci of pulmonary consolidation, with limited areas of alveolar edema and pneumonia, as well as immune cell infiltration. In sum, cynomolgus macaques are permissive to SARS- CoV-2 and develop lung pathology (less severe than SARC-CoV, but more severe than MERS-CoV).</p>
</div>
<div id="limitations-63" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-63" class="anchor"></a>Limitations</h4>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2 replication, it is unclear if the viral load reaches levels comparable to humans and there wasn’t overt clinical pathology.</p>
</div>
<div id="significance-60" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-60" class="anchor"></a>Significance</h4>
<p>The development of platforms in which to carry out relevant experimentation on SARS-CoV-2 pathophysiology is of great urgency. Cynomolgus macaques offer an environment in which viral replication can happen, albeit in a limited way and without translating into clinically relevant symptoms. Other groups are contributing to SARS-CoV2 literature using this animal model <span class="citation">[@doi:10.1101/2020.03.13.990226]</span>, potentially showing protection against reinfection in cured macaques. Therefore, this platform could be used to examine SARS-CoV2 pathophysiology while studies in other animal models are also underway <span class="citation">[@doi:10.1128/JVI.02012-06; @doi:10.1101/2020.02.07.939389]</span>.</p>
</div>
<div id="credit-65" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-65" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission" class="section level3">
<h3 class="hasAnchor">
<a href="#investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission" class="anchor"></a>Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20037994]</span></p>
<div id="keywords-66" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-66" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
</div>
<div id="main-findings-63" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-63" class="anchor"></a>Main Findings</h4>
<p>Multiple studies reported the same level of infectiousness between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that asymptomatic and undocumented carriers escape public health surveillance systems, a better mathematical model of transmission is needed to determine a more accurate estimate of the basic reproductive number (R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors developed a SEYAR dynamical model for transmission of the new coronavirus that takes into account asymptomatic and undocumented carriers. The model was validated using data reported from thirteen countries during the first three weeks of community transmission. While current studies estimate R<sub>0</sub> to be around 3, this model indicates that the value could range between 5.5 to 25.4.</p>
</div>
<div id="limitations-64" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-64" class="anchor"></a>Limitations</h4>
<p>The SEYAR model realistically depicts transmission of the virus only during the initial stages of the disease. More data is necessary to better fit the model with current trends. In addition, multiple factors (e.g. behavioral patterns, surveillance capabilities, environmental and socioeconomic factors) affect transmission of the virus and so, these factors must be taken into consideration when estimating the R<sub>0</sub>.</p>
</div>
<div id="significance-61" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-61" class="anchor"></a>Significance</h4>
<p>Public health authorities use the basic reproductive number to determine the severity of disease. An accurate estimate of R<sub>0</sub> will inform intervention strategies. This model can be applied to different locations to assess the potential impact of COVID-19.</p>
</div>
<div id="credit-66" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-66" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice" class="section level3">
<h3 class="hasAnchor">
<a href="#antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice" class="anchor"></a>Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h3>
<p>Long et al. <em>medRxiv</em> <span class="citation">[@doi:10.1101/2020.03.18.20038018]</span></p>
<div id="keywords-67" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-67" class="anchor"></a>Keywords</h4>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
</div>
<div id="main-findings-64" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-64" class="anchor"></a>Main Findings</h4>
<p>This study investigated the profile of the acute antibody response against SARS-CoV-2 and provided proposals for serologic tests in clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was used to evaluate IgM and IgG seroconversion in 285 hospital admitted patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19 suspected patients that tested negative by RT-PCR. A follow up study with 63 patients was performed to investigate longitudinal effects. In addition, IgG and IgM titers were evaluated in a cohort of close contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after symptom onset. Patients varied in the order of IgM/ IgG seroconversion and there was no apparent correlation of order with age, severity, or hospitalization time. This led the authors to conclude that for diagnosis IgM and IgG should be detected simultaneously at the early phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared to non-severe patients after controlling for days post-symptom onset. Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet the WHO serological diagnosis criterion of either seroconversion or <em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests the current serological criteria may be too stringent for COVID-19 diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test) had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close contacts” who had negative RT-PCR tests were positive for IgG and/or IgM. This highlights the usefulness of a serological assay to identify asymptomatic infections and/or infections that are missed by RT-PCR.</p>
</div>
<div id="limitations-65" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-65" class="anchor"></a>Limitations</h4>
<p>This group’s report generally confirms the findings of others that have evaluated the acute antibody response to SARS-Cov-2. However, these data would benefit from inclusion of data on whether the participants had a documented history of viral infection. Moreover, serum samples that were collected prior to SARS-Cov-2 outbreak from patients with other viral infections would serve as a useful negative control for their assay. Methodological limitations include that only one serum sample per case was tested as well as the heat inactivation of serum samples prior to testing. It has previously been reported that heat inactivation interferes with the level of antibodies to SARS-Cov-2 and their protocol may have resulted in diminished quantification of IgM, specifically <span class="citation">[@doi:10.1101/2020.03.12.20034231]</span>.</p>
</div>
<div id="significance-62" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-62" class="anchor"></a>Significance</h4>
<p>Understanding the features of the antibody responses against SARS-CoV is useful in the development of a serological test for the diagnosis of COVID-19. This paper addresses the need for additional screening methods that can detect the presence of infection despite lower viral titers. Detecting the production of antibodies, especially IgM, which are produced rapidly after infection can be combined with PCR to enhance detection sensitivity and accuracy and map the full spread of infection in communities, Moreover, serologic assays would be useful to screen health care workers in order to identify those with immunity to care for patients with COVID19.</p>
</div>
<div id="credit-67" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-67" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-specific-antibody-responses-in-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-specific-antibody-responses-in-covid-19-patients" class="anchor"></a>SARS-CoV-2 specific antibody responses in COVID-19 patients</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20038059]</span></p>
<div id="keywords-68" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-68" class="anchor"></a>Keywords</h4>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
</div>
<div id="main-findings-65" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-65" class="anchor"></a>Main findings</h4>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
</div>
<div id="limitations-66" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-66" class="anchor"></a>Limitations</h4>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
</div>
<div id="significance-63" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-63" class="anchor"></a>Significance</h4>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
</div>
</div>
<div id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19" class="anchor"></a>A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h3>
<p>Belhadi et al. <span class="citation">[@doi:10.1101/2020.03.18.20038190]</span></p>
<div id="keywords-69" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-69" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
</div>
<div id="main-findings-66" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-66" class="anchor"></a>Main Findings</h4>
<p>Summary of clinical trials registered as of March7, 2020 from U.S, Chinese, Korean, Iranian and European registries. Out of the 353 studies identified, 115 were selected for data extraction. 80% of the trials were randomized with parallel assignment and the median number of planned inclusions was 63 (IRQ, 36-120). Most frequent therapies in the trials included; 1) antiviral drugs <span class="math display">\[lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)\]</span>; 2) anti-malaria drugs <span class="math display">\[chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs \[methylprednisolone
(n=5)\]</span>; and stem cell therapies (n=23). Medians of the total number of planned inclusions per trial for these therapies were also included. Stem cells and lopunavir/ritonavir were the most frequently evaluated candidate therapies (23 and 15 trials respectively), whereas remdesivir was only tested in 5 trials but these trials had the highest median number of planned inclusions per trial (400, IQR 394-453). Most of the agents used in the different trials were chosen based on preclinical assessments of antiviral activity against SARS CoV and MERS Cov corona viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological (23%); radiological (8%); or immunological (3%). The trials were classified as those that included patients with severe disease only; trials that included patients with moderate disease; and trials that included patients with severe or moderate disease.</p>
</div>
<div id="limitations-67" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-67" class="anchor"></a>Limitations</h4>
<p>The trials evaluated provided incomplete information: 23% of these were phase IV trials but the bulk of the trials (54%) did not describe the phase of the study. Only 52% of the trials (n=60) reported treatment dose and only 34% (n=39) reported the duration. A lot of the trials included a small number of patients and the trials are still ongoing, therefore no insight was provided on the outcome of the trials.</p>
</div>
<div id="significance-64" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-64" class="anchor"></a>Significance</h4>
<p>Nonetheless, this review serves as framework for identifying COVID-19 related trials, which can be expanded upon as new trials begin at an accelerated rate as the disease spreads around the world.</p>
</div>
<div id="credit-68" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-68" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19" class="anchor"></a>ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h3>
<p>Leung et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.18.20038455]</span></p>
<div id="keywords-70" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-70" class="anchor"></a>Keywords</h4>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
</div>
<div id="main-findings-67" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-67" class="anchor"></a>Main Findings</h4>
<p>In bronchial epithelial samples from 3 different cohorts of individuals, ACE-2 gene expression was found to be significantly increased in both COPD patients and smokers relative to healthy controls. Across all test subjects, ACE-2 gene expression was also highly correlated with decreased forced expiratory volume in 1 second (FEV1), which may explain the increased COVID-19 disease severity in COPD patients. Former smokers were also found to show decreased ACE2 expression relative to current smokers and had no significant difference when compared to non-smokers.</p>
</div>
<div id="limitations-68" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-68" class="anchor"></a>Limitations</h4>
<p>While the upregulation of ACE-2 is an interesting hypothesis for COVID-19 disease severity in COPD patients, this study leaves many more unanswered questions than it addresses. Further studies are required to show whether the specific cell type isolated in these studies is relevant to the pathophysiology of COVID-19. Furthermore, there is no attempt to show whether that increased ACE-2 expression contributes to greater disease severity. Does the increased ACE-2 expression lead to greater infectivity with SARS-CoV-2? There is no mechanistic explanation for why ACE-2 levels are increased in COPD patients. The authors could also have considered the impact of co-morbidities and interventions such as corticosteroids or bronchodilators on ACE-2 expression. Finally, given the extensive sequencing performed, the authors could have conducted significantly more in-depth analyses into gene signature differences.</p>
</div>
<div id="significance-65" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-65" class="anchor"></a>Significance</h4>
<p>This study attempts to address an important clinical finding that both smokers and COPD patients show increased mortality from COVID-19. The novel finding that ACE-2 expression is induced in smokers and COPD patients suggests not only a mechanism for the clinical observation, but also highlights the potential benefit of smoking cessation in reducing the risk of severe COVID-19 disease.</p>
</div>
<div id="credit-69" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-69" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="dynamic-profile-of-severe-or-critical-covid-19-cases" class="section level3">
<h3 class="hasAnchor">
<a href="#dynamic-profile-of-severe-or-critical-covid-19-cases" class="anchor"></a>Dynamic profile of severe or critical COVID-19 cases</h3>
<p><span class="citation">[@doi:10.1101/2020.03.18.20038513]</span></p>
<div id="keywords-71" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-71" class="anchor"></a>Keywords</h4>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
</div>
<div id="main-findings-68" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-68" class="anchor"></a>Main Findings</h4>
<p>Authors evaluate clinical correlates of 10 patients (6 male and 4 female) hospitalized for severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and received broad spectrum antibiotics and 6 patients received anti-viral drugs. Additionally, 40% of patients were co-infected with influenza A. All 10 patients developed lymphopenia, two of which developed progressive lymphopenia (PLD) and died. Peripheral blood (PB) lymphocytes were analyzed – low CD4 and CD8 counts were noted in most patients, though CD4:CD8 ratio remained normal.</p>
</div>
<div id="limitations-69" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-69" class="anchor"></a>Limitations</h4>
<p>The authors evaluated a small cohort of severe SARS-CoV-2 cases and found an association between T cell lymphopenia and adverse outcomes. However, this is an extremely small and diverse cohort (40% of patients were co-infected with influenza A). These findings need to be validated in a larger cohort. Additionally, the value of this data would be greatly increased by adding individual data points for each patient as well as by adding error bars to each of the figures.</p>
</div>
<div id="significance-66" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-66" class="anchor"></a>Significance</h4>
<p>This study provides a collection of clinical data and tracks evolution of T lymphocyte in 10 patients hospitalized for SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
</div>
<div id="credit-70" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-70" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients" class="anchor"></a>Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h3>
<p>Fu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.19.20038315]</span></p>
<div id="keywords-72" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-72" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
</div>
<div id="study-description" class="section level4">
<h4 class="hasAnchor">
<a href="#study-description" class="anchor"></a>Study description</h4>
<p>Data analyzed from 52 COVID-19 patients admitted and then discharged with COVID-19. Clinical, laboratory, and radiological data were longitudinally recorded with illness timecourse (PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up positive test (PCR+) within two weeks of discharge.</p>
</div>
<div id="main-findings-69" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-69" class="anchor"></a>Main Findings</h4>
<ul>
<li>
<p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular enlargement (42.3%), and reticulation (11.5%) were most commonly observed.</p></li>
</ul>
</li>
<li>
<p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging w different extents of inflammatory exudation in lungs, with overall tendency for improvement (except 2/7 patients that were readmitted after discharge with re-positive test) after negative PCR.</p></li>
</ul>
</li>
<li>
<p>Seven patients repeated positive RT-PCR test and were readmitted to the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery and patients with lesion progression should be given more attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in lymphocytes in one patient), but was not correlated with new lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs. re-exposure.</p></li>
</ul>
</li>
</ul>
</div>
<div id="limitations-70" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-70" class="anchor"></a>Limitations</h4>
<p>Patients sampled in this study were generally younger (65.4% &lt; 50 yrs) and less critically ill/all discharged. Small number of recovered patients (N=18). Time of follow up was relatively short.. Limited clinical information available about patients with re-positive test (except CRP and lymph tracking).</p>
</div>
<div id="extended-results" class="section level4">
<h4 class="hasAnchor">
<a href="#extended-results" class="anchor"></a>Extended Results</h4>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4% &lt; 50 yrs) and less critically ill/all discharged. After two consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37 cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+ and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in lungs, with overall tendency for improvement (except 2/7 patients that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of segmental progression CT in 2 patients (2/18 or 11.1% - re-positive 9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in lymphocytes in one patient), but was not correlated with new lesions or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
</div>
<div id="significance-67" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-67" class="anchor"></a>Significance</h4>
<p>Study tracked key clinical features associated with disease progression, recovery, and determinants of clinical diagnosis/management of COVID-19 patients.</p>
</div>
<div id="credit-71" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-71" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus" class="section level3">
<h3 class="hasAnchor">
<a href="#an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus" class="anchor"></a>An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h3>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.19.997890]</span></p>
<div id="keywords-73" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-73" class="anchor"></a>Keywords</h4>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
</div>
<div id="main-findings-70" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-70" class="anchor"></a>Main Findings</h4>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an orally bioavailable ribonucleoside with antiviral activity against various RNA viruses including Ebola, Influenza and CoV. NHC activity introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a dose dependent <em>manner</em> that directly correlated with a decrease in viral titers. Authors show that NHC inhibited multiple genetically distinct Bat-CoV viruses in human primary epithelial cells <em>without affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>. Prophylactic oral administration of NHC in C57BL/6 mice reduce lung titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic administration of NHC in C57BL/6 mice 12 hours post infected with SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage. The degree of clinical benefit was dependent on the time of treatment initiation post infection. The authors also demonstrate that NHC reduces MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic and therapeutic settings.</p>
</div>
<div id="limitations-71" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-71" class="anchor"></a>Limitations</h4>
<p>Most of the experiments were conducted using MERS-CoV, and SARS-CoV and a few experiments were conducted using other strains of CoV as opposed to SARS-CoV-2. The authors note the core residues that make up the RNA interaction sites (which constitutes the NHC interaction sites) are highly conserved among CoV and because of this conservation their understanding is that NHC can inhibit a broad-spectrum of CoV including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have adverse effects if mutations cause the virus to become drug resistant, more infectious or speed-up immune evasion. <em>In addition, the temporal diminishing effectiveness of NHC on clinical outcome when NHC was used therapeutically is concerning. However, the longer window (7-10 days) for clinical disease onset in human patients from the time of infection compared to that of mice (24-48 hours), may associate with increased NHC effectiveness in the clinic.</em></p>
</div>
<div id="significance-68" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-68" class="anchor"></a>Significance</h4>
<p>Prophylactic or therapeutic oral administration of NHC reduces lung titers and prevents acute lung failure in C57B\6 mice infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC could turn out to be a useful drug for treating current, emerging and future corona virus outbreaks.</em>   #### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title:</strong> Anti-hypertensive Angiotensin II receptor blockers associated to mitigation of disease severity in elderly COVID-19 patients</p>
<p><strong>Keywords:</strong> SARS-CoV-2, ACE-2, Renin-angiotensin system, Hypertension</p>
<p><strong>Main findings:</strong> The authors analyzed clinical data obtained from COVID-19 patients and categorized them based on the antihypertensive drugs they were taking. They then investigated its association with morbidity and mortality of pneumonic COVID-19 patients. ARBs were found to be associated with a reduced risk of pneumonia morbidity in a total of 70,346 patients in three studies. They found that in the elderly (age&gt;65) group of COVID-19 patients with hypertension comorbidity, the risk of severe disease was significantly lower in patients who were on ARB anti-hypertensive drugs prior to hospitalization compared to patients who took no drugs. Also, through their meta-analysis of the literature, the authors reported that ARB anti-hypertensive drugs were associated with a decreased risk of severe disease in elderly COVID-19 patients.</p>
<p><strong>Critical Analyses</strong>:</p>
<ol style="list-style-type: decimal">
<li><p>Retrospective study with large potential for confounder bias.</p></li>
<li><p>Their inference that ARB is better than other anti-hypertensive drugs is based on literature met-analysis.</p></li>
<li><p>P-values could not be computed for some subsets because of very low/no patients in these categories(ref to table-1;ACEI, thiazide and BB)</p></li>
</ol>
<p><strong>Relevance:</strong> Anti-hypertensive ARB drugs taken by COVID-19 patients prior to entering the hospital may be associated with improved morbidity and mortality of pneumonia in elderly COVID-19 patients although confounders may bias results.</p>
</div>
</div>
<div id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs" class="section level3">
<h3 class="hasAnchor">
<a href="#identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs" class="anchor"></a>Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h3>
<p>Sangeun Jeon et al. <span class="citation">[@doi:10.1101/2020.03.20.999730]</span></p>
<div id="keywords-74" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-74" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
</div>
<div id="main-findings-71" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-71" class="anchor"></a>Main Findings</h4>
<p>A panel of ~3,000 FDA- and IND-approved antiviral drugs were previously screened for inhibitory efficacy against SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2 (79.5%) homology. 35 of these drugs along with another 15 (suggested by infectious disease specialists) were tested in vitro for their ability to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell viability. The infected cells were scored by immunofluorescence analysis using an antibody against the N protein of SARS CoV-2. Chloroquine, lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50 values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti helminthic drug niclosamide which exhibited potent antiviral activity against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect of niclosamide against SARS and MERS-CoV have been previously documented and recent evidence suggests that in may inhibit autophagy and reduce MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat asthma and allergic rhinitis, also exhibited antiviral efficacy but with a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of ciclesonide were directed against NSP15, a viral riboendonuclease which is the molecular target of this drug.</p>
</div>
<div id="limitations-72" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-72" class="anchor"></a>Limitations</h4>
<p>The drugs were tested against SARS CoV-2 infectivity in vitro only, therefore preclinical studies in animals and clinical trials in patients will be need for validation of these drugs as therapeutic agents for COVID-19. In addition, niclosamide exhibits low adsorption pharmatokinetically which could be alleviated with further development of drug formulation to increase effective delivery of this drug to target tissues. Nonetheless, niclosamide and ciclesonide represent promising therapeutic agents against SARS CoV-2 given that other compounds tested in the same study including favipiravir (currently used in clinical trials) and atazanavir (predicted as the most potent antiviral drug by AI-inference modeling) did not exhibit antiviral activity in the current study.</p>
</div>
<div id="credit-72" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-72" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2" class="section level3">
<h3 class="hasAnchor">
<a href="#respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2" class="anchor"></a>Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h3>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.001628]</span></p>
<div id="keywords-75" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-75" class="anchor"></a>Keywords</h4>
<p>animal model, pulmonary infiltrates, dynamic of antibody response, cytokine</p>
</div>
<div id="main-findings-72" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-72" class="anchor"></a>Main Findings</h4>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which all showed clinical signs of infection (respiratory pattern, reduced appetite, weight loss, elevated body temperature) resulting in moderate, transient disease. Four animals were euthanized at 3dpi, the 4 others at 21 dpi. Study of viral loads in different organs showed that nose swab and throat swabs were the most sensitive, with bronchio-alveolar lavage. Interstitial pneumonia was visible in radiographies and at the histological scale too. Clinically, the macaques had similar symptoms as described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs immediately after infection but less consistent thereafter which could reflect virus administration route (intranasal, oral). Bronchoalveolar lavages performed as a measure of virus replication in the lower respiratory tract on animals maintained for 21 days, contained high viral loads in 1 and 3dpi. The majority of the animals exhibited pulmonary edema and mild to moderate interstitial pneumonia on terminal bronchioles. In addition to the lung, viral RNA could also be detected throughout the respiratory track where viral replication mainly occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and through 10dpi after which they started to recover. After infection, serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10, IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody response started around 7dpi, and the antibody titers stayed elevated until 21dpi</strong> (day of animal euthanasia).</p>
</div>
<div id="limitations-73" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-73" class="anchor"></a>Limitations</h4>
<p>The macaques were inoculated via a combination of intratracheal, intranasal, ocular and oral routes, which might not reproduce how humans get infected. Maybe this can lead to different dynamics in the host immune response. Also, the authors noted that the seroconversion was not directly followed by a decline in viral loads, as observed in covid19 patients.</p>
</div>
<div id="significance-69" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-69" class="anchor"></a>Significance</h4>
<p>This work confirms that rhesus macaques can be a good model to study Covid-19, as it has been shown by other groups <span class="citation">[@doi:10.1101/2020.03.13.990226; @doi:10.1101/2020.03.13.990036; @doi:10.1101/2020.03.17.995639]</span>. While these experiments recapitulate moderate COVID-19 in humans, the mode of inoculation via a combination of intratracheal, intranasal, ocular and oral routes, might not reproduce how humans get infected and may lead to different dynamics in the host immune response. For example, the authors noted that the seroconversion was not directly followed by a decline in viral loads, as observed in COVID-19 patients. Therefore, it will be interesting to follow their antibody titers longer and further assess the possibility/effect of reinfection in these macaques. It is essential to be able to understand the dynamic of the disease and associated immune responses, and to work on vaccine development and antiviral drug testing.</p>
</div>
<div id="credit-73" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-73" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19" class="anchor"></a>ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h3>
<p><span class="citation">[@doi:10.1101/2020.03.21.20040261]</span></p>
<div id="keywords-76" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-76" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
</div>
<div id="main-findings-73" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-73" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and network analysis of existing literature to determine ACE2 regulation in patients who have frequent COVID-19 comorbidities <span class="math display">\[eg-
cardiovascular diseases, familial pulmonary hypertension, cancer\]</span>.</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation, metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2 higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
</div>
<div id="limitations-74" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-74" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations in this study need to be confirmed by analyzing upcoming transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and autoimmunity as risk factors in CoVID-19 patients due to lack of data; would be useful to extend such analyses to those datasets when available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs validation-experiments especially in CoVID-19 lung samples to mean anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of histone marks does not necessarily mean occupancy. Would be pertinent to check cell-line data (CCLE) or actual CoVID-19 patient samples to confirm ACE2 epigenetic control.</p></li>
</ul>
</div>
<div id="significance-70" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-70" class="anchor"></a>Significance</h4>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that could promote CoVID-19 severity. Shows important data-mining strategy to find gene-networks associated with ACE2 upregulation in co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might play an important role in CoVID-19 and can be preliminary targets for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2 expression could be a new target for CoVID-19 therapy with strategies such as KDM5 demethylases.</p></li>
</ul>
</div>
<div id="credit-74" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-74" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial" class="anchor"></a>Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h3>
<p>Bian et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.20040691]</span></p>
<div id="keywords-77" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-77" class="anchor"></a>Keywords</h4>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
</div>
<div id="main-findings-74" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-74" class="anchor"></a>Main Findings</h4>
<p>This work is based on previous work by the same group that demonstrated that SARS-CoV-2can also enter host cells via CD147 (also called Basigin, part of the immunoglobulin superfamily, is expressed by many cell types) consistent with their previous work with SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17 patients receiving Meplazumab, a humanized anti-CD147 antibody, in addition to all other treatments. 11 patients were included as a control group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group (e.g. at day 14 47% vs 17% improvement rate) compared to the control patients. Also, virological clearance was more rapid with median of 3 days in the Meplazumab group vs 13 days in control group. In laboratory values, a faster normalization of lymphocyte counts in the Meplazumab group was observed, but no clear difference was observed for CRP levels.</p>
</div>
<div id="limitations-75" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-75" class="anchor"></a>Limitations</h4>
<p>While the results from the study are encouraging, this study was non-randomized, open-label and on a small number of patients, all from the same hospital. It offers evidence to perform a larger scale study. Selection bias as well as differences between treatment groups (e.g. age 51yo vs 64yo) may have contributed to results. The authors mention that there was no toxic effect to Meplazumab injection but more patient and longer-term studies are necessary to assess this.</p>
</div>
<div id="significance-71" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-71" class="anchor"></a>Significance</h4>
<p>These results seem promising as for now there are limited treatments for Covid-19 patients, but a larger cohort of patient is needed. CD147 has already been described to facilitate HIV <span class="citation">[@doi:10.1073/pnas.111583198]</span>, measles virus <span class="citation">[@doi:10.1128/JVI.02168-09]</span>, and malaria <span class="citation">[@doi:10.1038/nature10606]</span> entry into host cells. This group was the first to describe the CD147-spike route of SARS-Cov-2 entry in host cells <span class="citation">[@doi:10.1101/2020.03.14.988345]</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov could enter host cells via this transmembrane protein <span class="citation">[@doi:10.1086/427811]</span>. Further biological understanding of how SARS-CoV-2 can enter host cells and how this integrates with ACE2R route of entry is needed. Also, the specific cellular targets of the anti-CD147 antibody need to be assessed, as this protein can be expressed by many cell types and has been shown to involved in leukocytes aggregation <span class="citation">[@doi:10.1371/journal.pone.0215557]</span>. Lastly, Meplazumab is not a commercially-available drug and requires significant health resources to generate and administer which might prevent rapid development and use.</p>
</div>
<div id="credit-75" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-75" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection" class="section level3">
<h3 class="hasAnchor">
<a href="#potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection" class="anchor"></a>Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h3>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.21.990770]</span></p>
<div id="keywords-78" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-78" class="anchor"></a>Keywords</h4>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
</div>
<div id="main-findings-75" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-75" class="anchor"></a>Main Findings</h4>
<p>In this study the authors report the affinity, cross reactivity (with SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206 monoclonal antibodies engineered from isolated IgG memory B cells of patients suffering from SARS-CoV-2 infection in Wuhan, China. All patients but one recovered from disease. Interestingly, the patient that did not recover had less SARS-CoV-2 specific B cells circulating compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore, plasma from patients recognized the receptor binding domain (RBD) from SARS-CoV-2 but had little to no cross-reactivity against the RBD of related viruses SARS-CoV and MERS-CoV, suggesting significant differences between the RBDs of the different viruses. Negligible levels of cross-neutralization using pseudoviruses bearing Spike proteins of SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV specific IgG memory B cells were single cell sorted to sequence the antibody genes that were subsequently expressed as recombinant IgG1 antibodies. From this library, 206 antibodies with different binding capacities were obtained. No discernible patterns of VH usage were found in the 206 antibodies suggesting immunologically distinct responses to the infection. Nevertheless, most high-binding antibodies were derived by clonal expansion. Further analyses in one of the patient derived clones, showed that the antibodies from three different timepoints did not group together in phylogenetic analysis, suggesting selection during early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have 10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in competition assays. Thus, low Kd values alone did not predict ACE2 competing capacities. Consistent with competition assays the two antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using SPR the neutralizing antibodies were found to recognize both overlapping and distinct epitopes of the RBD of SARS-CoV-2.</p>
</div>
<div id="limitations-76" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-76" class="anchor"></a>Limitations</h4>
<ol style="list-style-type: decimal">
<li>
<p>Relatively low number of patients</p>
<ol style="list-style-type: lower-alpha">
<li>No significant conclusion can be drawn about the possible &gt; correlation between humoral response and disease severity</li>
</ol>
</li>
<li>
<p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol style="list-style-type: lower-alpha">
<li><p>Huh7 cells were used in neutralization assays with &gt; pseudoviruses, and they may not entirely mimic what happens in &gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol>
</li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
</div>
<div id="significance-72" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-72" class="anchor"></a>Significance</h4>
<p>This paper offers an explanation as to why previously isolated antibodies against SARS-CoV do not effectively block SARS-CoV-2. Also, it offers important insight into the development of humoral responses at various time points during the first weeks of the disease in small but clinically diverse group of patients. Furthermore, it provides valuable information and well characterized antibody candidates for the development of a recombinant antibody treatment for SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have variability in its effectiveness, depending on the donor’s antibody repertoire at the time of donation.</p>
</div>
<div id="credit-76" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-76" class="anchor"></a>Credit</h4>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site-" class="section level3">
<h3 class="hasAnchor">
<a href="#characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site-" class="anchor"></a>Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h3>
<p>Davidson et al. <span class="citation">[@doi:10.1101/2020.03.22.002204]</span></p>
<div id="keywords-79" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-79" class="anchor"></a>Keywords</h4>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
</div>
<div id="main-findings-76" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-76" class="anchor"></a>Main Findings</h4>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore MinION as well as tandem mass spec (MS) on Vero cells (a cell line derived from kidney cells of the African green monkey that is deficient in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike (S) protein. The deleted sequence overlaps a predicted furin cleavage site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but not SARS-CoV or the closely related bat coronavirus RaTG13, which are cleaved at S1 / S2 by other proteases <span class="citation">[@doi:10.1126/science.abb2507]</span>. Furin cleavage sites at similar positions in other viruses have been linked to increased pathogenicity and greater cell tropism <span class="citation">[@doi:10.1128/JVI.00797-10]</span>. Loss of this site in SARS-CoV-2 has also already been shown to increase viral entry into Vero but not BHK cells (which are also interferon deficient) <span class="citation">[@doi:10.1016/j.cell.2020.02.058]</span>. The authors therefore make an important contribution in demonstrating that passage in Vero cells may lead to spontaneous loss of a key pathogenicity-conferring element in SARS-CoV-2.</p>
</div>
<div id="limitations-77" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-77" class="anchor"></a>Limitations</h4>
<p>As the authors note, a similar study posted earlier by Kim et al., which also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the S protein furin cleavage site <span class="citation">[@doi:10.1101/2020.03.12.988865]</span>. It therefore remains to be determined how likely it is that this mutation spontaneously arises. A quantitative investigation using multiple experimental replicas to understand the spontaneous viral mutation rate at this site and elsewhere would be informative. Also, the mechanistic basis for the higher viral fitness conferred by loss of the furin cleavage site in Vero cells – but, evidently, not in vivo in humans, as this site is maintained in all currently sequenced circulating isolates - remains to be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’ bioinformatic pipeline required aligning transcripts to a reference to correct sequencing artifacts. This presumably made it difficult or impossible to identify other kinds of mutations, such as single nucleotide substitutions, which may occur even more frequently than the deletions identified in this work. Pairing long read sequencing with higher-accuracy short-read sequencing may be one approach to overcome this issue.</p>
</div>
<div id="significance-73" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-73" class="anchor"></a>Significance</h4>
<p>As the authors suggest, animal studies using live virus challenge may need to periodically verify the genomic integrity of the virus, or potentially risk unknowingly using a likely less-pathogenic variant of the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the SARS-CoV-2 viral transcriptome and proteome. For example, the authors found multiple versions of transcripts encoding the nucleocapsid (N) protein, each with different small internal deletions, some of which were verified for translation by MS. A number of peptides arising from translation of unexpected rearrangements of transcripts were also detected. Additionally, the authors identified phosphorylation of a number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For any cases where these have functional consequences, targeting the kinases responsible could be an avenue for drug development. Understanding the functional consequences of the mutations, transcript variations, and post translational modifications identified in this study will be important future work.</p>
</div>
<div id="credit-77" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-77" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing" class="section level3">
<h3 class="hasAnchor">
<a href="#a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing" class="anchor"></a>A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h3>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.22.002386]</span></p>
<div id="keywords-80" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-80" class="anchor"></a>Keywords</h4>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
</div>
<div id="main-findings-77" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-77" class="anchor"></a>Main Findings</h4>
<p>Gordon et al cloned, tagged and expressed 26 of the 29 SARS-CoV-2 proteins individually in HEK293T cells and used mass spectrometry to identify protein-protein interactions. They identified 332 viral-host protein-protein interactions. Furthermore, they used these interactions to identify 66 existing drugs known to target host proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact with proteins regulated by the mTOR pathway, so mTOR inhibitors Silmitasertib and Rapamycin are possible drug candidates).</p>
</div>
<div id="limitations-78" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-78" class="anchor"></a>Limitations</h4>
<p>The main limitation of the study stems from the reductionist model: overexpression of plasmids encoding individual viral proteins in HEK293T cells. This precludes any interactions between the viral proteins, or the combined effects of multiple proteins on the host, as they are expressed individually. Moreover, HEK293T cells come from primary embryonic kidney and therefore might not reflect how SARS-CoV-2 interacts with its primary target, the lung. However, the authors found that the proteins found to interact with viral proteins in their experiments are enriched in lung tissue compared to HEK293Ts.</p>
</div>
<div id="significance-74" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-74" class="anchor"></a>Significance</h4>
<p>The authors provide a “SARS-CoV-2 interaction map,” which may provide potential hypotheses as to how the virus interacts with the host. Further, they identified existing drugs that could disrupt these host-viral interactions and curb SARS-CoV-2 infection. Although these interactions have not been validated, this paper acts as a valuable resource.</p>
</div>
<div id="credit-78" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-78" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients" class="section level3">
<h3 class="hasAnchor">
<a href="#first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-na%C3%AFve-and-experienced-covid-19-patients" class="anchor"></a>First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h3>
<p>Chen et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.22.20034041]</span></p>
<div id="keywords-81" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-81" class="anchor"></a>Keywords</h4>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
</div>
<div id="main-findings-78" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-78" class="anchor"></a>Main Findings</h4>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a commercialized HCV protease inhibitor <span class="citation">[@doi:10.1128/AAC.00699-08]</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation">[@doi:10.14218/JCTH.2019.00033]</span>, a CYP3A4 inhibitor (which enhances the plasma concentration and bioavailabilty of Danoprevir). Two patients had never received anti-viral therapy before (=naïve), whereas nine patients were on Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir (=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a decreased hospitalization time. Patients treated with Lopinavir/Ritonavir did not have a negative PCR test, while after switching to Danoprevir/Ritonavir treatment, the first negative PCR test occurred at a median of two days.</p>
</div>
<div id="limitations-79" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-79" class="anchor"></a>Limitations</h4>
<p>The results of the study are very hard to interpret as there is no control group not receiving Danoprevir/Ritonavir treatment. This was especially true in naïve patients who seemed to have more mild symptoms before the start of the study and were younger (18 and 44yo) compared to the experienced patients (18 to 66yo). The possibility that the patients would have recovered without Danoprevir/Ritonavir treatment cannot be excluded.</p>
</div>
<div id="significance-75" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-75" class="anchor"></a>Significance</h4>
<p>The authors of the study treated patients with Danoprevir, with the rational to that this is an approved and well tolerated drug for HCV patients <span class="citation">[@doi:10.14218/JCTH.2019.00033]</span>, and that it could also target the protease from SARS-CoV-2 (essential for viral replication and transcription). Indeed, homology modelling data indicated that HCV protease inhibitors have the highest binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation">[@doi:10.1101/2020.02.05.936013]</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir might be beneficial for Covid-19 patients, additional clinical trials with more patients and with better methodology (randomization and control group) are needed to make further conclusions.</p>
</div>
<div id="credit-79" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-79" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial" class="section level3">
<h3 class="hasAnchor">
<a href="#efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial" class="anchor"></a>Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h3>
<p><span class="citation">[@doi:10.1101/2020.03.22.20040758]</span></p>
<div id="keywords-82" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-82" class="anchor"></a>Keywords</h4>
<ul>
<li>hydroxychloroquine</li>
</ul>
</div>
<div id="study-description-1" class="section level4">
<h4 class="hasAnchor">
<a href="#study-description-1" class="anchor"></a>Study Description</h4>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy in the treatment of COVID-19. From February 4 – February 28, 2020 142 COVID-19 positive patients were admitted to Renmin Hospital of Wuhan University. 62 patients met inclusion criteria and were enrolled in a double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol style="list-style-type: decimal">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li>
<p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note: relevant clinical references described below.)</p>
<ol style="list-style-type: lower-alpha">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt; 85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction of inspired oxygen) is used to classify the severity of acute respiratory distress syndrome (ARDS). Mild ARDS has a PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe &lt; 100.</p></li>
</ol>
</li>
<li><p>Willing to receive a random assignment to any designated treatment group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol style="list-style-type: decimal">
<li><p>Severe or critical respiratory illness (not explicitly defined, presumed to be respiratory function worse than outlined in inclusion criteria); or participation in trial does not meet patient’s maximum benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt; twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral agents, antibacterial agents, and immunoglobulin, with or without corticosteroids. Patients in the HCQ treatment group received additional oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from days 1-5 of the trial. The primary endpoint was 5 days post enrollment or a severe adverse reaction to HCQ. The primary outcome evaluated was time to clinical recovery (TTCR), defined as return to normal body temperature and cough cessation for &gt; 72h. Chest CT were imaged on days 0 and 6 of the trial for both groups; body temperature and patient reports of cough were collected 3x/day from day 0 – 6. The mean age and sex distribution between the HCQ and control arms were comparable.</p>
</div>
<div id="main-findings-79" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-79" class="anchor"></a>Main Findings</h4>
<p>There were 2 patients showing mild secondary effects of HCQ treatment. More importantly, while 4 patients in the control group progressed to severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p = 0.0008); time to cessation of cough was similarly reduced (3.1 days control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19 has a modest effect on clinical recovery (symptom relief on average 1 day earlier) but may be more potent in reducing the progression from mild to severe disease.</p>
</div>
<div id="limitations-80" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-80" class="anchor"></a>Limitations</h4>
<p>This study is limited in its inclusion of only patients with mild disease, and exclusion of those on any treatment other than the standard of care. It would also have been important to include the laboratory values of positive RT-PCR detection of SARS-CoV-2 to compare the baseline and evolution of the patients’ viral load.</p>
</div>
<div id="limitations-81" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-81" class="anchor"></a>Limitations</h4>
<p>Despite its limitations, the study design has good rigor as a double blind RCT and consistent symptom checks on each day of the trail. Now that the FDA has approved HCQ for treatment of COVID-19 in the USA, this study supports the efficacy of HCQ use early in treatment of patients showing mild symptoms, to improve time to clinical recovery, and possibly reduce disease progression. However, most of the current applications of HCQ have been in patients with severe disease and for compassionate use, which are out of the scope of the findings presented in this trial. Several additional clinical trials to examine <a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a> are now undergoing; their results will be critical to further validate these findings.</p>
</div>
<div id="credit-80" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-80" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>10.1101 2020.03.22.20040832 </strong></p>
<p><strong>Title:</strong> Factors associated with prolonged viral shedding and impact of Lopinavir/Ritonavir treatment in patients with SARS-CoV-2 infection<strong> </strong></p>
<p><strong>Keywords</strong>: retrospective study – lopinavir/ritonavir – viral shedding</p>
<p><strong>Main findings:</strong></p>
<p>The aim of this retrospective study is to assess the potential impact of earlier administration of <strong>lopinavir/ritonavir (LPV/r)</strong> treatment on the duration of viral shedding in hospitalized non-critically ill patients with SARS-CoV-2.</p>
<p>The analysis shows that administration of LPV/r treatment reduced the duration of viral shedding (22 vs 28.5 days). Additionally, if the treatment was started within 10 days of symptoms onset, an even shorter duration of virus shedding was observed compared to patients that started treatment after 10 days of symptoms s onset (19 vs 27.5 days). Indeed, patients that started LPV/r treatment late did not have a significant median duration of viral shedding compared to the control group (27.5 vs 28.5 days). Old age and lack of LPV/r administration independently associated with prolonged viral shedding in this cohort of patients.</p>
<p><strong>Limitations:</strong></p>
<p>In this non-randomized study, the group not receiving LPV/r had a lower proportion of severe and critical cases (14.3% vs 32.1%) and a lower proportion of patients also receiving corticosteroid therapy and antibiotics, which can make the results difficult to interpret.</p>
<p>The endpoint of the study is the end of viral shedding (when the swab test comes back negative), not a clinical amelioration. The correlation between viral shedding and clinical state needs to be further assessed to confirm that early administration of LPV/r could be used in treating COVID-19 patients.</p>
<p><strong>Relevance</strong>:</p>
<p>Lopinavir/ritonavir combination has been previously shown to be efficient in treating SARS <sup>1,2</sup>. While this article raises an important point of early administration of LPV/r being necessary to have an effect, the study is retrospective, contains several sources of bias and does not assess symptom improvement of patients. A previously published randomized controlled trial including 200 severe COVID-19 patients did not see a positive effect of LPV/r administration<sup>3</sup>, and treatment was discontinued in 13.8% of the patients due to adverse events. Similarly, another small randomized trial did not note a significant effect of LPV/r treatment<sup>4</sup> in mild/moderate patients. A consequent European clinical trial, “Discovery”, including among others LPV/r treatment is under way and may provide conclusive evidence on the effect and timing of LPV/r treatment on treating COVID-19.</p>
<p>1. Treatment of severe acute respiratory syndrome with lopinavir/ritonavir: a multicentre retrospective matched cohort study. - PubMed - NCBI. <a href="https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14660806" class="uri">https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14660806</a>. Accessed March 30, 2020.</p>
<p>2. Role of lopinavir/ritonavir in the treatment of SARS: initial virological and clinical findings. - PubMed - NCBI. <a href="https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14985565" class="uri">https://www-ncbi-nlm-nih-gov.docelec.univ-lyon1.fr/pubmed/14985565</a>. Accessed March 30, 2020.</p>
<p>3. Cao B, Wang Y, Wen D, et al. A Trial of Lopinavir–Ritonavir in Adults Hospitalized with Severe Covid-19. <em>New England Journal of Medicine</em>. March 2020. <a href="doi:10.1056/NEJMoa2001282" class="uri">doi:10.1056/NEJMoa2001282</a></p>
<p>4. Li Y, Xie Z, Lin W, et al. <em>An Exploratory Randomized, Controlled Study on the Efficacy and Safety of Lopinavir/Ritonavir or Arbidol Treating Adult Patients Hospitalized with Mild/Moderate COVID-19 (ELACOI)</em>. Infectious Diseases (except HIV/AIDS); 2020. <a href="doi:10.1101/2020.03.19.20038984" class="uri">doi:10.1101/2020.03.19.20038984</a></p>
<p><strong>Emma Risson Edited by K Alexandropoulos</strong></p>
<p><strong>Clinical features and the maternal and neonatal outcomes of pregnant women with coronavirus disease 2019</strong></p>
<p><strong><em>Keywords</em></strong></p>
<p>Pregnancy, SARS-CoV2, neonatal and maternal Covid-19 outcome</p>
<p><strong><em>Key findings</em></strong></p>
<p>33 pregnant woman and 28 newborns were included in this retrospective multicenter study, conducted at 5 hospitals in Wuhan and Hubei province, China, between January 1 and February 20, 2020. All women were diagnosed with Covid-19 by qPCR or viral gene sequencing based on the Chinese New Corona Pneumonia Prevention and Control Program, 6<sup>th</sup> edition, and were further subdivided into four groups based on clinical severity: (1) mild, presence of mild clinical symptoms without radiological abnormalities; (2) moderate, fever or upper respiratory symptoms as well as radiological signs of pneumonia; (3) severe, at least one of the following: shortness of breath/respiratory rate &gt;30/min, resting oxygen saturation SaO<sub>2</sub>&lt;93%, Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary damage); and (4) severe-acute, acute respiratory distress with need for mechanical ventilation; systemic shock; multi-organ failure and transfer to ICU. Maternal admission to ICU, mechanical ventilation or death were defined as primary outcomes; secondary study outcomes comprised clinical Covid-19 severity in both mothers and newborns, including development of ARDS, neonatal ICU admission as well as mortality.</p>
<p><em>Maternal characteristics and outcome:</em> 3 out of 33 women were in their second trimester of pregnancy (17, 20 and 26 weeks), and 15/33 (45.5%) had a previous history of underlying chronic health disorders including cardiovascular, cerebrovascular or nervous system disease. Common Covid-19 symptoms at presentation were fever (63.6%), dry cough (39.4%), fatigue (21.2%), and shortness of breath (21.2%). Less common symptoms included diarrhea, post-partum fever, muscle ache, sore throat and chest pain. 4 (12.1%) pregnant women had no apparent symptoms. The majority of cases were classified as mild (39.4%) or moderate (57.6%); however, one woman developed severe Covid-19. 40.6% of women were diagnosed with bilateral pneumonia, 43.8% presented with unilateral pneumonia, and 15.6% showed radiological ground-glass opacity. 87.9% of women required oxygen administration, and one (3%) woman had to be put on non-invasive mechanical ventilation (primary outcome). 81.5% of women had a C-section and only 5% had vaginal deliveries. Obstetrical complications were seen in 22.2% of women, including three cases of preterm rupture of membranes, two cases of hypertensive disorders of pregnancy, and one case of spontaneous preterm labor. Five pregnancies were ongoing at the end of the observation point of this study; one woman decided to have her pregnancy terminated. <em>Neonatal outcome:</em> Out of 28 newborns included in this study, 35.7% were born preterm at &lt;37 weeks of gestation with Apgar scores ranging from 8-10/10 at 1 min and from 9-10/10 after 5 min, indicating normal heart and respiratory rates. 17.9% of newborns were of low birth weight (not specified) and 14.3% showed signs of fetal distress (also not specified). According to the authors of this study, none of the newborns presented with clinical Covid-19 symptoms. However, one newborn, delivered at 34 weeks of gestation, was diagnosed with (apparently Covid-19 unrelated?) ARDS and transferred to NICU (secondary outcome). Of 26 newborns tested for SARS-CoV2, only one was found positive and showed radiological signs of pneumonia, but no clinical symptoms of Covid-19. It remains unclear whether this was the same case as the newborn diagnosed with ARDS. The affected newborn did not require any treatment and was discharged at 16 days post birth. In summary, the primary outcome “mechanical ventilation” in pregnant women was rare (3%), no other primary outcomes were reached. Most Covid-19 cases in pregnant women were described as mild to moderate. Only one of 28 (3.57%) newborns was diagnosed with ARDS (secondary outcome).</p>
<p><strong><em>Potential limitations</em></strong></p>
<p>Major limitations of this study are its small size and the rudimentary and at times inadequate description of patient specifics. For example, underlying health conditions that might be affecting Covid-19 outcome in pregnant women should have been clearly specified (other than being of be listed (not just &lt;37 weeks). Given that maternal infection status seemed mostly unknown at the time of birth and, more importantly, that the majority of cases in this study were clinically asymptomatic or mild to moderate, it remains unclear whether the C-sections performed were a medical necessity or elective procedures. This is of importance and should have been discussed. With regard to neonatal outcome, it is also not apparent whether the newborn found to be infected with SARS-CoV2 and the case diagnosed with ARDS were the same individual. If this was the case, it would be incorrect to refer to all newborns as asymptomatic. Additionally, it seems somewhat unlikely that a newborn with a near-perfect Apgar score would present with ARDS immediately after birth. Likewise, any individual diagnosed with ARDS would certainly be expected to receive supportive treatment including (invasive) mechanical ventilation. While it is highly relevant that overall clinical outcome in pregnant women diagnosed with Covid-19 seems better than in SARS or MERS (as discussed by the authors), it nevertheless needs to be stressed that more than 37% of newborns in this study were delivered preterm and that the obstetric complication rate of 22% seems higher than non-Covid-19 average.</p>
<p><strong><em>Overall relevance for the field</em></strong></p>
<p>Observations in this study confirm some of the findings published in a case series by Yu N et al. (Lancet Infect Dis 2020; <a href="https://doi.org/10.1016/" class="uri">https://doi.org/10.1016/</a> S1473-3099(20)30176-6). However, due to the relatively small study size of 33 pregnant women and 28 newborns, this study lacks statistical power and final conclusions on Covid-19 outcomes in pregnant women and newborns cannot be drawn. Yet, the data collected here are important and should be incorporated into larger data sets for more insight. Understanding the clinical course and effects of Covid19 in both pregnant women and newborns is essential, and while there are some recent publications on vertical SARS-CoV2 transmission between mothers and newborns (Dong L et al, JAMA March 26, 2020, <a href="doi:10.1001/jama.2020.4621" class="uri">doi:10.1001/jama.2020.4621</a>; Zeng H et al, JAMA March 26, 2020, <a href="doi:10.1001/jama.2020.4861" class="uri">doi:10.1001/jama.2020.4861</a>) as well as on neonatal infection at birth (Zeng L et al, JAMA March 26, 2020, <a href="doi:10.1001/jamapediatrics.2020.0878" class="uri">doi:10.1001/jamapediatrics.2020.0878</a>), our knowledge of how these patient subsets are affected is still very limited.</p>
<ol style="list-style-type: upper-alpha">
<li><p>Elevated serum IgM levels indicate poor outcome in patients with coronavirus disease 2019 pneumonia: A retrospective case-control study</p></li>
<li><p><strong>Keywords:</strong> SAR-CoV-2, pneumonia, immunoglobulin, IgM, IgG.</p></li>
<li><p><strong>Key findings:</strong> The authors wanted to better understand the dynamics of production SARS-CoV-2-specific IgM and IgG in COVID-19 pneumonia and the correlation of virus-specific antibody levels to disease outcome in a case-control study paired by age. The retrospective study included 116 hospitalized patients with COVID-19 pneumonia and with SAR-CoV-2 specific serum IgM and IgG detected. From the study cohort, 15 cases died. SARS-CoV-2 specific IgG levels increased over 8 weeks after onset of COVID-19 pneumonia, while SARS-CoV-2 specific IgM levels peaked at 4 weeks. SARS-CoV-2 specific IgM levels were higher in the deceased group, and correlated positively with the IgG levels and increased leucocyte count in this group, a indication of severe inflammation. IgM levels correlated negatively with clinical outcome and with albumin levels. The authors suggest that IgM levels could be assessed to predict clinical outcome.</p></li>
<li><p><strong>Potential limitations:</strong> There are limitations that should be taken into account. First, the sample: small size, patients from a single-center and already critically ill when they were admitted. Second, the authors compared serum IgM levels in deceased patients and mild-moderate patients and found that the levels were higher in deceased group, however even if the difference is statistically significant the number of patients in the two groups was very different. Moreover, receiving operating characteristics (ROC) curves were used to evaluate IgM and IgG as potential predictors for clinical outcome. Given the low number of cases, especially in the deceased group, it remains to be confirmed if IgM levels could be predictive of worst outcome in patients with COVID-19 pneumonia. The study did not explore the role of SARS-CoV-2-specific IgM and IgG in COVID-19 pneumonia.</p></li>
<li><p><strong>Overall relevance for the field:</strong> Some results of this study have been supported by subsequent studies that show that older age and patients who have comorbidities are more likely to develop a more severe clinical course with COVID-19, and severe SARS-CoV-2 may trigger an exaggerated immune response. The study seems to demonstrate that the increase of SARS-CoV-2-specific IgM could indicate poor outcome in patients with COVID-19 pneumonia, however given the very small sample size, the results are not yet conclusive.</p></li>
</ol>
<p>Review by Meriem Belabed as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
<p><strong>Title:</strong> scRNA-seq reveals ACE2 and TMPRSS2 expression in TROP2+ Liver Progenitor Cells: Implications in COVID-19 associated liver dysfunction</p>
<p><strong>Keywords:</strong> SARS-CoV-2, COVID-19, ACE2, TMPRSS2, TROP2, Liver, scRNA-seq</p>
<p><strong>Main findings:</strong> The authors performed scRNA-seq on human liver tissue and identified an epithelial progenitor cell type that co-express ACE2 and TMPRSS2. ACE2 receptor was highly expressed in cholangiocyte-biased liver progenitors (EPCAM<sup>+</sup>) which also had a high expression for TROP2 gene. TMPRSS2 was expressed by both TROP2<sup>high</sup> and TROP2<sup>int</sup> liver cells. The authors reported that these cholangiocyte-biased progenitor cells had a high expression of TROP2 that correlated with high levels of ACE2 and TMPRSS2 in liver. This suggests that SARS-CoV-2 can infect TROP2<sup>high</sup> cells via ACE2 and TMPRSS2, contributing to liver dysfunction by compromising the ability of the human liver to regenerate cholangiocytes. The authors also note that ACE2 and TMPRSS2 positive cells are absent in human fetal liver.</p>
<p><strong>Critical Analyses</strong>:</p>
<p>1. Normal tissue from hepatocellular carcinoma patients are tumor-adjacent tissues from these patients and might not be truly ‘normal’.</p>
<p>2. Tissues from liver cancer patients may have an altered phenotype from healthy controls.</p>
<p>3. ACE2 expression may be variable and inducible in other cell types in the context of inflammation.</p>
<p><strong>Relevance: </strong></p>
<p>The apparent only cell population in liver that expresses ACE2 and TMPRSS2 are EPCAM+ liver progenitor cells, making them suitable hosts for viral infection and potentially resulting in liver dysfunction. This cell population of liver progenitors with a cholangiocyte fate bias may favor SARS-CoV-2 entry into liver, affecting cholangiocyte precursors, thus leading to liver comorbidities in COVID-19 patients.</p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide, modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC complexes on Protein Data Bank, substitute homologous peptides with SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding energy for further study and publish models in an online database (<a href="https://rosettamhc.chemistry.ucsc.edu" class="uri">https://rosettamhc.chemistry.ucsc.edu</a>).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted SARS-CoV-2 peptides must be validated experimentally for immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high resolution X-ray structures on PDB, authors only simulate 9-mer and 10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a starting point, backbone conformations that deviate significantly between test and template peptides are not captured. Furthermore, Rosetta modeling protocols do not capture all possible structures and binding energy scoring does not fully recapitulate fundamental forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides that may direct a targeted, experimental validation approach toward a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results from NetMHCpan 4.0, supporting machine learning prediction with structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method of leveraging existing X-ray structures for identification of immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol style="list-style-type: decimal">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D., Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34), 4748-4763. <a href="doi:10.1021/acs.biochem.6b00444" class="uri">doi:10.1021/acs.biochem.6b00444</a></p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J., DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design. J Chem Theory Comput, 13(6), 3031-3048. <a href="doi:10.1021/acs.jctc.7b00125" class="uri">doi:10.1021/acs.jctc.7b00125</a></p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset" class="section level3">
<h3 class="hasAnchor">
<a href="#serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset" class="anchor"></a>Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h3>
<p>Lou et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.23.20041707]</span></p>
<div id="keywords-83" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-83" class="anchor"></a>Keywords</h4>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
</div>
<div id="main-findings-80" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-80" class="anchor"></a>Main Findings</h4>
<p>Currently, the diagnosis of SARS-CoV-2 infection entirely depends on the detection of viral RNA using polymerase chain reaction (PCR) assays. False negative results are common, particularly when the samples are collected from upper respiratory. Serological detection may be useful as an additional testing strategy. In this study the authors reported that a typical acute antibody response was induced during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8% (79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first detectible serology marker was total antibody followed by IgM and IgG, with a median seroconversion time of 15, 18 and 20 days-post exposure (d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was first detected at day 7d.p.e in 98.9% of the patients. Interestingly they found that viral load declined as antibody levels increased. This was in contrast to a previous study <span class="citation">[@doi:10.1101/2020.03.02.20030189]</span>, showing that increased antibody titers did not always correlate with RNA clearance (low number of patient sample).</p>
</div>
<div id="limitations-82" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-82" class="anchor"></a>Limitations</h4>
<p>Current knowledge of the antibody response to SAR-CoV-2 infection and its mechanism is not yet well elucidated. Similar to the RNA test, the absence of antibody titers in the early stage of illness could not exclude the possibility of infection. A diagnostic test, which is the aim of the authors, would not be useful at the early time points of infection but it could be used to screen asymptomatic patients or patients with mild disease at later times after exposure.</p>
</div>
<div id="significance-76" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-76" class="anchor"></a>Significance</h4>
<p>Understanding the antibody responses against SARS-CoV2 is useful in the development of a serological test for the diagnosis of COVID-19. This manuscript discussed acute antibody responses which can be deducted in plasma for diagnostic as well as prognostic purposes. Thus, patient-derived plasma with known antibody titers may be used therapeutically for treating COVID-19 patients with severe illness. <span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
</div>
<div id="credit-81" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-81" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems" class="section level3">
<h3 class="hasAnchor">
<a href="#sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems" class="anchor"></a>SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h3>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.004655]</span></p>
<div id="keywords-84" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-84" class="anchor"></a>Keywords</h4>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
</div>
<div id="main-findings-81" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-81" class="anchor"></a>Main Findings</h4>
<p>Given the high mortality rate of SARS-CoV-2 relative to other respiratory viruses such seasonal IAV and RSV, there may be underlying host-pathogen interactions specific to SARS-CoV-2 that predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and <em>in vitro</em> systems, the authors profiled host cell transcriptional responses to SARS-CoV-2 and to other common respiratory viruses (seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an induction of type I interferon response signaling and the upregulation of cytokine/chemokines transcripts. In comparison with IAV and RSV infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I and type III interferon expression and higher levels of two cytokines previously implicated in respiratory inflammation. Lastly, <em>in vivo</em> data from ferrets showed a reduced induction of cytokines and chemokines by SARS-CoV-2 infection relative to IAV infection.</p>
</div>
<div id="limitations-83" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-83" class="anchor"></a>Limitations</h4>
<p>While these results are promising, there are several key weaknesses of this paper. 1) As the authors point out, there is an undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease (TMPRSS2) expression in the lung epithelial cell line used for the <em>in vitro</em> studies. This raises the important question of whether viral replication actually occurs in any of the models used, which may explain the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2 treated cells. Further studies characterizing viral titers across timepoints are needed. 2) Furthermore, these studies only characterize the host response at a single dose and timepoint per virus, and it is unclear why these doses/timepoints were chosen. This leaves open the possibility that the observed differences between viruses could be due to differences in dose, timing, host response, or a combination of all of these. 3) It is unclear whether ferrets are productively infected, which cell types are infected, and the extent/timing of the clinical course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not strongly correlate and the reasons for this are unclear.</p>
</div>
<div id="significance-77" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-77" class="anchor"></a>Significance</h4>
<p>This paper describes potentially unique transcriptional signatures of host cells exposed to SARS-CoV-2. If validated, these findings may help explain clinical outcomes and could be targeted in future therapeutic interventions.</p>
</div>
<div id="potential-conflicts-of-interest-disclosure" class="section level4">
<h4 class="hasAnchor">
<a href="#potential-conflicts-of-interest-disclosure" class="anchor"></a>Potential Conflicts of Interest Disclosure</h4>
<p>The reviewers are also researchers at the Icahn School of Medicine at Mount Sinai.</p>
</div>
<div id="credit-82" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-82" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Analysis of Serologic Cross-Reactivity Between Common Human Coronaviruses and SARS-CoV-2 Using Coronavirus Antigen Microarray</strong></p>
<p><strong><em>Keywords</em></strong>: serum antibodies, cross-reactivity, antigen microarray</p>
<p><strong><em>Key Findings</em></strong></p>
<p>The authors of this study describe a common respiratory viral antigens microarray that can be used to assess the cross-reactivity of antibodies against the novel SARS-CoV-2 antigens and common human coronaviruses. Sixty-seven (67) antigens from across subtypes of coronaviruses, influenza viruses, adenoiruses, respiratory syncytial viruses and other viruses, were printed onto microarrays, probed with human sera, and analyzed. Five (5) serum samples collected before the SARS-CoV-2 outbreak were tested as part of a study that monitored acute respiratory infection cases. Four out of five (4/5) sera samples showed high IgG seroreactivity across the 4 common human coronaviruses. All sera showed low IgG seroreactivity to SARS-CoV-2.</p>
<p>The S1 domain of the spike protein is suggested to be subtype specific as it demonstrated very low cross-reactivity among the novel coronaviruses (SARS-CoV-2, SARS-CoV and MERS-CoV) and between the novel coronal viruses and common human coronaviruses. In contrast, the S2 domain of the spike protein and the nucleocapsid (NP) protein showed low levels of cross-reactivity between the coronavirus subtypes, suggesting the presence of more conserved antigenic domains. Therefore, the authors conclude that the S1 domain is an ideal candidate for virus-specific serologic assays.</p>
<p><strong><em>Importance</em></strong></p>
<p>This study provides insights into the potential cross-reactivity of common human coronavirus antibodies for SARS-CoV-2 antigens. Understanding cross-reactivity is useful because it can help in the development of sensitive and specific serological assays that can test for COVID-19. Additionally, studies such as this can help inform vaccine development and indicate which antigens would be optimal to target.</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=5) of serum samples that all came from students who were part of the same college resident community at The University of Maryland. A greatest limitation is the lack of serum samples from COVID-19 patients. Additionally, the MFI of 2019-nCov from these naïve samples (in particular sera 4) is not zero. This indicates that while relatively low, there is some cross-reactivity of common human coronavirus antibodies for SARS-CoV-2 antigens, including the S1 domain. A larger sample size and samples from SARS-CoV2 infected patients may be needed to confirm the sensitivity and specificity of the assay.</p>
<p><strong>Lactate dehydrogenase - A Risk Factor of Severe COVID-19 Patients: A Retrospective and Observational Study</strong></p>
<p>Han et al., medRxiv <span class="math display">\[10.1101:2020.03.24.20040162\]</span></p>
<p>Keywords:</p>
<ul>
<li><p>lactate dehydrogenase</p></li>
<li><p>COVID-19</p></li>
<li><p>risk factor</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report is a retrospective analysis of clinical data collected from 47 patients with confirmed COVID-19 disease (median age 64.91 years; 26 males; 24 severe cases). Upon admission, patients were assessed with an APACHE II score (mortality risk) and SOFA score (organ system dysfunction). Among other clinical parameters, including lymphocyte count, CRP, and enzyme levels, serum lactate dehydrogenase (LDH) level was most positively correlated with APACHE II and SOFA. Additionally, strong positive correlations were found between LDH and observational assessment of severity of pneumonia and of underlying coronary disease. Notably, serum LDH was also positively correlated with other univariate parameters, including troponin and CRP. Meanwhile, LDH was negatively correlated with lymphocyte count across different subsets, including CD4<sup>+</sup> and CD8<sup>+</sup> T cells.</p>
<p><em>Limitations</em></p>
<p>Several studies have previously identified serum LDH as a predictive biomarker for COVID-19 progression (see references). Therefore, it is important to note that while LDH correlates with disease severity, it does not suffice as a reason for severe disease, so whether it serves as a robust <strong>predictive</strong> factor is not thoroughly supported; an explanation is still needed between the release of LDH from infected cells and disease severity.</p>
<p><em>Significance</em></p>
<p>The identification of serum LDH as a predictive biomarker does glean insight into possible links between its release into the circulation and immune response to SARS-CoV-2 pathogenesis. It is important to note that LDH release itself requires significant damage to and permeabilization of plasma membranes, as observed in pyroptosis and necrotic cell death. Therefore, this report, and others like it, warrants investigation into the link between SARS-CoV-2 infection and potential necrosis or pyroptosis of infected host cells. It is important to note that previous studies concerning biomarkers for SARS-CoV-1 disease have also identified serum LDH as an important correlative factor with disease severity<sup>1</sup>. And subsequent investigations have studied the activation of inflammasome structures involved in pyroptosis in the presence of SARS-CoV<sup>2-3</sup>.</p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><em>References</em></p>
<p>1. Tsui PT, Kwok ML, Yuen H, Lai ST. Severe acute respiratory syndrome: clinical outcome and prognostic correlates. <em>Emerg Infect Dis</em>. 2003;9(9):1064–1069. <a href="doi:10.3201/eid0909.030362" class="uri">doi:10.3201/eid0909.030362</a></p>
<p>2. Yue, Y, Nabar, NR, Shi, C. <em>et al.</em> SARS-Coronavirus Open Reading Frame-3a drives multimodal necrotic cell death. <em>Cell Death Dis</em> 2018;<strong>9, </strong>904. doi: 10.1038/s41419-018-0917-y</p>
<p>3. Shi, C., Nabar, N.R., Huang, N. <em>et al.</em> SARS-Coronavirus Open Reading Frame-8b triggers intracellular stress pathways and activates NLRP3 inflammasomes. <em>Cell Death Discov.</em> 2019;<strong>5, </strong>101. doi: 10.1038/s41420-019-0181-7</p>
</div>
</div>
<div id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china" class="section level3">
<h3 class="hasAnchor">
<a href="#a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china" class="anchor"></a>A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h3>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042119]</span></p>
<div id="keywords-85" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-85" class="anchor"></a>Keywords</h4>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
</div>
<div id="main-findings-82" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-82" class="anchor"></a>Main Findings</h4>
<p>377 hospitalized patients were divided into two groups: severe and non-severe pneumonia. The laboratory results of their first day of admission were retrospectively analyzed to identify predictors of disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such as high blood pressure, type 2 diabetes, coronary heart disease, and chronic obstructive pulmonary disease), the independent risk factors identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers, they showed that their multiplication <strong><span class="math display">\[(Neutrophil/lymphocyte count)
\* CRP \* D-dimer\]</span></strong> was a better predictor of disease severity, with higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value of 2.68.</p>
</div>
<div id="limitations-84" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-84" class="anchor"></a>Limitations</h4>
<p>This study included 377 hospitalized patients. Among them, 45.6% patients tested positive for SARS-Cov-2 nucleic acid test results, and others were included in the study based on clinically diagnosis even if the molecular diagnosis was negative. Thus, additional studies are needed to verify this on a larger number of covid-19 certified patients and the cutoff value might be adjusted. Also, all the patients that did not have the clinical characteristics of severe pneumonia were included in the non-severe pneumonia group, but usually patients are also divided into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more specific predictor. Another study showed that the neutrophils to CD8+ T cells ratio was a strong predictor of disease severity <span class="citation">[@doi:10.1101/2020.02.16.20023671]</span>. Another more precise study showed that the percentage of helper T cells and regulatory T cells decrease but the percentage of naïve helper T cells increases in severe cases <span class="citation">[@doi:10.1093/cid/ciaa248]</span>. Taking these subpopulations into account might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity and LDH <span class="citation">[@doi:10.1101/2020.03.03.20030668]</span> or IL6 <span class="citation">[@doi:10.1101/2020.03.08.20031229]</span> levels, but the authors here do not discuss LDH nor IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of admission, studying the dynamic of the changes in patients might also be interesting to better predict disease severity.</p>
</div>
<div id="significance-78" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-78" class="anchor"></a>Significance</h4>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a predictor of disease severity as shown by many others <span class="citation">[@doi:10.1093/cid/ciaa248; @doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.10.20021584]</span>. The novelty here is that they show that a combination with other markers can enhance the specificity and sensibility of the predictor, although the study could be improved by taking into account sub-populations of lymphocytes and more biological factors from patients such as LDH and IL6.</p>
</div>
<div id="credit-83" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-83" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study" class="section level3">
<h3 class="hasAnchor">
<a href="#metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study" class="anchor"></a>Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h3>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042283]</span></p>
<div id="keywords-86" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-86" class="anchor"></a>Keywords</h4>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
</div>
<div id="main-findings-83" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-83" class="anchor"></a>Main Findings</h4>
<p>Retrospective Study on 97 COVID-19 hospitalized patients (25 severe and 72 non-severe) analyzing clinical and laboratory parameter to predict transition from mild to severe disease based on more accessible indicators (such as fasting blood glucose, serum protein or blood lipid) than inflammatory indicators. In accordance with other studies, age and hypertension were risk factors for disease severity, and lymphopenia and increased IL-6 was observed in severe patients. The authors show that fasting blood glucose (FBG) was altered and patients with severe disease were often hyperglycemic. Data presented support that hypoproteinaemia, hypoalbuminemia, and reduction in high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease severity.</p>
</div>
<div id="limitations-85" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-85" class="anchor"></a>Limitations</h4>
<p>In this study non-severe patients were divided in two groups based on average course of the disease: mild group1 (14 days, n=28) and mild group 2 (30 days, n=44). However mild patients with a longer disease course did not show an intermediate phenotype (between mild patients with shorter disease course and severe patients), hence it is unclear whether this was a useful and how it impacted the analysis. Furthermore, the non-exclusion of co-morbidity factors in the analysis may bias the results (e.g. diabetic patients and glucose tests) It is not clear at what point in time the laboratory parameters are sampled. In table 3, it would have been interesting to explore a multivariate multiple regression. The correlation lacks of positive control to assess the specificity of the correlation to the disease vs. correlation in any inflammatory case. The dynamic study assessing the predictability of the laboratory parameter is limited to 2 patients. Hence there are several associations with disease severity, but larger studies are necessary to test the independent predictive value of these potential biomarkers.</p>
</div>
<div id="significance-79" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-79" class="anchor"></a>Significance</h4>
<p>As hospital are getting overwhelmed a set of easily accessible laboratory indicators (such as serum total protein) would potentially provide a triage methodology between potentially severe cases and mild ones. This paper also opens the question regarding metabolic deregulation and COVID-19 severity.</p>
</div>
<div id="credit-84" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-84" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#viral-kinetics-and-antibody-responses-in-patients-with-covid-19" class="anchor"></a>Viral Kinetics and Antibody Responses in Patients with COVID-19</h3>
<p><span class="citation">[@doi:10.1101/2020.03.24.20042382]</span></p>
<div id="keywords-87" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-87" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
</div>
<div id="main-findings-84" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-84" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset in sputum, nasal swabs, and stool samples. Shedding ranged from 12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as ‘strong vs weak vs non-responders’. Strong responders (with higher IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
</div>
<div id="limitations-86" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-86" class="anchor"></a>Limitations</h4>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in intestine/liver or from swallowed sputum. Transmission electron microscopy (TEM) carried out on sputum samples as proof of concept, but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver disease) that were not accounted for when tracking antibody responses. Viral kinetics and IgM/IgG titers in subsets of patients with underlying conditions/undergoing certain medication would be informative.</p></li>
</ul>
</div>
<div id="significance-80" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-80" class="anchor"></a>Significance</h4>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different disease progression and kinetics is useful to stratify patients, and for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and CoVID-19 disease severity <span class="citation">[@doi:10.1101/2020.03.20.20039495]</span>, similar to macaque studies in SARS <span class="citation">[@doi:10.1172/jci.insight.123158]</span>. It would be critical to see if anti-NP or anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar detrimental effects before clinical use.</p></li>
</ul>
</div>
<div id="credit-85" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-85" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome" class="section level3">
<h3 class="hasAnchor">
<a href="#covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome" class="anchor"></a>COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h3>
<p><span class="citation">[@doi:10.1101/2020.03.24.20042655]</span></p>
<div id="keywords-88" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-88" class="anchor"></a>Keywords</h4>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
</div>
<div id="main-findings-85" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-85" class="anchor"></a>Main Findings</h4>
<p>This study is based on flow cytometry immunophenotyping of PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The authors identify a population of abnormally large (FSC-hi) monocytes, present in COVID19 patients, but absent in PBMCs of healthy volunteers (n=16) or patients with different infections (AIDS, malaria, TB). This FSC-hi monocytic population contains classical, intermediate and non-classical (monocytes (based on CD14 and CD16 expression) that produce inflammatory cytokines (IL-6, TNF and IL-10). The authors suggest an association of FSC-hi monocytes with poor outcome and correlate a high percentage of FSC-low monocytes, or higher ratio of FSC-low/hi monocytes, with faster hospital discharge.</p>
</div>
<div id="limitations-87" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-87" class="anchor"></a>Limitations</h4>
<p>While identification of the monocytic population based on FSC is rather robust, the characterization of these cells remains weak. A comprehensive comparison of FSC-hi monocytes with FSC-low monocytes from patients and healthy controls would be of high value. It is unclear if percentages in blood are among CD45+ cells. Furthermore, it would have been important to include absolute numbers of different monocytic populations (in table 1 there are not enough samples and it is unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the monocytes, and highlight these cells as potential targets of SARS-Cov2. However, appropriate controls are needed. CD16 has high affinity to rabbit IgG and it is unclear whether the authors considered unspecific binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on monocytes needs to be assessed. Furthermore, it would be important to confirm infection of monocytes by presence of viral proteins or viral particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development of the disease and its severity, some data expected at this level are neither present nor addressed. Although the cohort is small, it does include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi monocytes in comparison to other patients? Was this apparent early in the disease course? Does this population of FSC-hi monocytes differ between ICU patients and others in terms of frequency, phenotype or cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example, in Fig. 5, according to the legend it seems that patients with FSC-high monocytes are discharged faster from the hospital. However according to description in the text, patients were grouped in high or low levels of FSC-low monocytes.</p>
</div>
<div id="significance-81" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-81" class="anchor"></a>Significance</h4>
<p>Despite the limitations of this study, the discovery of a FSC-high monocyte population in COVID-19 patients is of great interest. With similar implication, a the recent study by Zhou et al. <span class="citation">[@doi:10.1101/2020.02.12.945576]</span> identified a connection between an inflammatory CD14+CD16+ monocyte population and pulmonary immunopathology leading to deleterious clinical manifestations and even acute mortality after SARS-CoV-2 infections. Although the presence of these monocytes in the lungs has yet to be demonstrated, such results support the importance of monocytes in the critical inflammation observed in some COVID19 patients.</p>
</div>
<div id="credit-86" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-86" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study" class="section level3">
<h3 class="hasAnchor">
<a href="#correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study" class="anchor"></a>Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h3>
<p>Miller et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.24.20042937]</span></p>
<div id="keywords-89" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-89" class="anchor"></a>Keywords</h4>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
</div>
<div id="main-findings-86" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-86" class="anchor"></a>Main Findings</h4>
<p>The authors compared middle and high income countries that never had a universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium) and countries with a current policy (low income countries were excluded from the analysis as their number of cases and deaths might be underreported for the moment). <strong>Countries that never implement BCG vaccination have a higher mortality rate than countries which have a BCG vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next, <strong>the authors show that an earlier start of vaccination correlates with a lower number of deaths per million inhabitants</strong>. They interpret this as the vaccine protecting a larger fraction of elderly people, which are usually more affected by COVID-19. Moreover, higher number of COVID-19 <strong>cases</strong> were presented in countries that never implemented a universal BCG vaccination policy.</p>
</div>
<div id="limitations-88" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-88" class="anchor"></a>Limitations</h4>
<p>While this study aims to test an intriguing hypothesis unfortunately, the data is not sufficient at this time to accurately make any determinations. Several caveats must be noted including: not all countries are in the same stage of the pandemic, the number of cases/deaths is still changing very rapidly in a lot of countries and thus the association may only reflect exposure to the virus. This analysis would need to be re-evaluated when all the countries are passed the pandemic and more accurate numbers are available. Additionally, very few middle and high-income countries ever implemented universal BCG vaccination, which can be a source of bias (5 countries, vs 55 that have a BCG vaccine policy). Effective screening and social isolation policies also varied considerable across the countries tested and may reflect another important confounder. The authors could consider analyzing the Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more correct for exposure although testing availability will still bias this result. Variability in mortality within countries or cities with variable vaccination and similar exposure could also be appropriate although confounders will still be present.</p>
</div>
<div id="significance-82" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-82" class="anchor"></a>Significance</h4>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has been proven to be efficient in preventing childhood meningitis TB, but doesn’t prevent adult TB as efficiently. For this reason, several countries are now only recommending this vaccine for at-risk population only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has been shown to protect against several viruses and enhance innate immunity <span class="citation">[@doi:10.1016/j.cmi.2019.04.020]</span>, which could explain why it could protect against SARS-CoV-2 infection, but the exact mechanism is still unknown. <strong>Moreover, the efficiency of adult/older people vaccination and protection against Covid-19 still needs to be assessed.</strong> Regarding this, Australian researchers are starting a clinical trial of BCG vaccine for healthcare workers <span class="citation">[@url:https://www.mcri.edu.au/BRACE]</span>, to assess if it can protect them against Covid-19.</p>
</div>
<div id="credit-87" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-87" class="anchor"></a>Credit</h4>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium" class="section level3">
<h3 class="hasAnchor">
<a href="#non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium" class="anchor"></a>Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h3>
<p><span class="citation">[@doi:10.1101/2020.03.25.009084]</span></p>
<div id="keywords-90" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-90" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
</div>
<div id="main-findings-87" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-87" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry) are expressed in olfactory epithelial (OE) cells, basal stem cells and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE sustentacular cells was similar to expression in the remainder of the non-nasal respiratory tract.</p></li>
</ul>
</div>
<div id="limitations-89" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-89" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is not sufficient to confirm infectivity of nasal passage, or to indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial injury leads to reduced sensitivity or increased inflammation and altered immune contexture drives neural/epithelial dysfunction. Will be critical to test this in CoVID-19 patient samples or mouse models.</p></li>
</ul>
</div>
<div id="significance-83" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-83" class="anchor"></a>Significance</h4>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells, and olfactory epithelia may serve as a viral reservoir after initial SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1, ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in other coronaviruses.</p></li>
</ul>
</div>
<div id="credit-88" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-88" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome peptide microarray. The peptide library was constructed with 966 linear peptides, each 15 amino acids long with a 5 amino acid overlap, based on the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1 nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides. Polyclonal antibodies showed some crossreactivity to other epitopes from membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a, and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies in more than 80 % COVID-19 patients were present in N, S and Orf3. It is shown that the receptor binding domain (RBD) resides on S protein and RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among six epitopes on S protein, structural analysis predicted that three epitopes were located at the surface and three epitopes were located inside of the protein. Furthermore, some IgM antibodies from 1 patient and IgG antibodies from 2 patients bound to the same epitope (residue 456-460, FRKSN) which resided within the RBD, and structural analysis determined that this epitope was located in the region of the RBD loop that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would have been informative to do the analysis over the course of the disease. The pattern of antibody recognition, especially on S protein, and the course of antibodies of different isotypes recognizing the same peptide might correlate to the clinical course in these patients. It would alos have been informative to analyze the presence of cross-reactive antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2, Wuhan-Hu-1 strain. This is a valuable information to design vaccines that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe lymphocytopenia, but the mechanism through which this depletion occurs has not been described. In order to characterize the cause and process of lymphocyte depletion in COVID-19 patients, the authors performed gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens and lymph nodes (hilar and subscapular) obtained from post-mortem autopsies of 6 patients with confirmed positive viremia and 3 healthy controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy, hemorrhaging, and necrosis with congestion of interstitial blood vessels and large accumulation of mononuclear cells and massive lymphocyte death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and subscapular LN, and capillaries of these secondary lymphoid organs expressed the ACE2 receptor and stain positive for the SARS-CoV-2 nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup> CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ) and in the marginal sinuses of LN, which suggests that these macrophages were positioned to encounter invading pathogens first and may contribute to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for persistent activation of lymphocytes via Fas::FasL interactions that would mediate activation-induced cell death (AICD). Indeed, the expression of Fas was significantly higher in virus-infected tissue than that of healthy controls, and TUNEL staining showed significant lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and TNF-α can also engage cellular apoptosis and necrosis, the authors interrogated the cytokine expression of the secondary lymphoid organs from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected splenic and lymph node tissues, compared to those of healthy controls, and immunofluorescent staining showed that IL-6 is primarily produced by the infected macrophages. <em>In vitro</em> infection of THP1 cells with SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em> expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively, the authors concluded that a combination of Fas up-regulation and IL-6 production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic MZ, subscapular LN, and the lining capillaries of the secondary lymphoid tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The findings point to the potential role of these macrophages in viral dissemination, immunopathology of these secondary lymphoid organs, hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6); moreover, the analysis of these samples using multiplexed immunohistochemistry and immunofluorescence do not necessarily provide the depth of unbiased interrogation needed to better identify the cell types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including single-cell experiments of spleen and LN from organ donors, do not indicate that ACE2 is expressed by macrophages; however, it remains possible that ACE2 expression may be triggered by type I IFN in COVID-19 patients. Importantly, the SARS-CoV-2 NP staining of the macrophages does not necessarily reflect direct infection of these macrophages; instead, positive staining only indicates that these macrophages carry SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct viral culture of macrophages isolated from the secondary lymphoid organs with SARS-CoV-2 is required to confirm the potential for direct infection of macrophages by SARS-CoV-2. Additionally, it is important to note that the low to negligible viremia reported in COVID-19 patients to-date does not favor a dissemination route via the blood, as suggested by this study, which would be necessary to explain the presence of virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized by cytokine storm in many COVID-19 patients. The contribution of this pathology to the overall fatality rate due to COVID-19, not even necessarily directly due to SARS-CoV-2 infection, is significant. A better understanding of the full effect and source of some of these major cytokines, like IL-6, as well as the deficient immune responses, like lymphocytopenia, is urgently needed. In this study, the authors report severe tissue damage in spleens and lymph nodes of COVID-19 patients and identify the role that CD169<sup>+</sup> macrophages may play in the hyperinflammation and lymphocytopenia that are both characteristic of the disease. It may, therefore, be important to note the effects that IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on splenic and LN function. It is important to note that similar observations of severe splenic and LN necrosis and inflammation in patients infected with SARS-CoV-1 further support the potential importance and relevance of this study.</p>
</div>
</div>
<div id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2" class="section level3">
<h3 class="hasAnchor">
<a href="#cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2" class="anchor"></a>Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h3>
<p><span class="citation">[@doi:10.1101/2020.03.28.013672]</span></p>
<div id="keywords-91" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-91" class="anchor"></a>Keywords</h4>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
</div>
<div id="main-findings-88" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-88" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture experiments to show that cigarette smoke causes a dose-dependent upregulation of ACE2 in mouse and human lungs (transplantation, tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+ goblet and SCGB1A1+ club cells; ACE2 presence in these cells is increased upon smoke exposure.</p></li>
</ul>
</div>
<div id="limitations-90" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-90" class="anchor"></a>Limitations:</h4>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including immune dysfunction, which can affect interpretation of CoV-2 susceptibility in these datasets. Ideally, analyses can control for major co-morbidities across smokers and non-smokers (immune suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral entry into goblet cells; future studies with air-liquid interface cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells from human epithelial lines, ex vivo samples or hACE2 mice will be very informative.</p></li>
</ul>
</div>
<div id="significance-84" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-84" class="anchor"></a>Significance</h4>
<ul>
<li><p>This study may partially explain why smokers are more likely to develop severe SARS-CoV-2 infections. Also, the reversibility of ACE2 expression upon smoking cessation suggests that quitting smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies implies CoV-2 susceptibility might be smoking-specific, and not fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of patients with CoV-2 co-morbidities such as hypertension <span class="citation">[@doi:10.1101/2020.03.21.20040261]</span>; these studies collectively paint a better picture of CoV-2 susceptibility before actual experiments can be carried out.</p></li>
</ul>
</div>
<div id="credit-89" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-89" class="anchor"></a>Credit</h4>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Structural Analysis of SARS-CoV-2 and Predictions of the Human Interactome</strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v4.full.pdf"><em>https://www.biorxiv.org/content/10.1101/2020.03.28.013789v4.full.pdf</em></a></p>
<p><strong>Keywords:</strong></p>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>Computational structural analysis</p></li>
<li><p>RNA-protein interactions</p></li>
<li><p>Binding partner predictions</p></li>
</ul>
<p><strong>Main findings:</strong></p>
<p>To elucidate mechanisms of viral replication within human hosts, the authors used computational approaches to analyze the structural properties of SARS-CoV-2 RNA and predict human proteins that bind to it. They compared 2800 coronaviruses and 62 SARS-CoV-2 strains using <em>CROSS</em> (Computational Recognition of Secondary Structure) and <em>CROSS</em>align algorithms, which predict RNA structure using sequence information and evaluate structural conservation, respectively. From these structural comparisons, they found that the spike S protein that interacts with the human receptor angiotensin-converting enzyme 2 (ACE2) is highly conserved amongst coronaviruses.</p>
<p>The study also identified over 100,000 human protein interactions with SARS-CoV-2 utilizing <em>cat</em>RAPID, an algorithm that determines binding potentials of proteins for RNA using secondary structure, van der Waals, and hydrogen bonding contributions. They found that the 5’ of SARS-CoV-2 is highly structured and has a strong propensity to bind to human proteins with known involvement in viral RNA processing. Amongst the proteins they identified can bind to 5’, there was significant enrichment in proteins associated with HIV infection and replication, including ATP-dependent RNA Helicase (DDX1), A-kinase anchor protein 8-like (AKAP8L), and dsRNA-specific Editase 1 (ADARB1).  </p>
<p><strong>Limitations:</strong></p>
<p>The study was based off of computational structural predictions and subsequent gene ontology enrichment analyses. Computational predictions are less accurate than experimental observations, and while their predicted binding partners prompt interesting hypotheses, they must be experimentally confirmed. Furthermore, the gene ontology annotations used to create their candidate list are based off prior studies and inherently miss novel biological implications. Targeted biochemical and structural studies that can be built off their identified targets will be essential for elucidating viral-host complexes and informing potential targeted drug designs.</p>
<p><strong>Significance:</strong></p>
<p>Their computational approach demonstrated findings confirmatory of other studies in many respects. Their structural analysis findings of the high conservation of the spike S protein amongst analyzed coronaviruses and SARS-CoV-2 strains suggest that the spike S evolved to specifically interact with host ACE2, supporting that the human engineering of SARS-CoV2 is very unlikely and implicating spike S as a potential therapeutic target. Their RNA-protein interaction predictions suggest several relevant host-virus interactions that warrant further investigation. If proved experimentally, their identified links to proteins studied in the context of HIV and other viruses may be relevant for the repurposing of existing antiviral drugs for SARS-CoV-2.</p>
<p><em>Review by Michelle Tran as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<p>Review of paper titled:</p>
<p><strong>Virus shedding patterns in nasopharyngeal and fecal specimens of1COVID-19 patients</strong></p>
<p>Zhang et al. 2020</p>
<p>Keywords: Viral shedding, testing specimens.</p>
<p><strong>Summary </strong></p>
<p>The authors tested feces, urine, plasma, nasal/throat swabs (respiratory samples) for SARS-CoV-2 virus load analysis by qPCR several days after illness onset (DAO).</p>
<ul>
<li><p>All plasma samples were negative (n=56).</p></li>
<li><p>Two urine samples were positive from patients that had severe disease and were positive at least 16-21 DAO (n=56).</p></li>
<li><p>10 of 12 cases (83.3%) tested positive for fecal samples.</p></li>
<li><p>14 of 21 cases (66.7%) were positive for respiratory samples.</p></li>
<li><p>Respiratory samples were positive for 21 DAO, nevertheless fecal samples were still positive.</p></li>
<li><p>Median duration of virus shedding was 10.0 days in respiratory samples</p></li>
<li><p>For fecal samples median shedding was 22.0 days for the feces</p></li>
<li><p>Viral titers of respiratory samples peaked at six to nine DAO. and at 14-18 DAO for fecal samples, and the highest virus titers at the peak was significantly higher for feces (105.8 copies/ml, mean 5623 copies/ml) than of respiratory samples (106.363 copies/ml, mean 2535 copies/ml).</p></li>
</ul>
<p>Importance of findings and caveats</p>
<p>Notably, these data reflect viral RNA and there is mention of infectious particles. Without these data, it is difficult to ascertain the significance of this paper. While these data demonstrate that the duration of fecal viral RNA shedding is significantly prolonged when compared to viral RNA shedding from the respiratory tract, it is critical for the field to ascertain if this represents infectious virions or not. If viable virus were to be isolated from the GI tract, that would indeed be very concerning for potential feco-oral transmission of SARS-COV2. As such, this paper does not establish this important fact.</p>
<p><em>Review by Jovani Catalan-Dibene and validated by Saurabh Mehandru as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis" class="section level3">
<h3 class="hasAnchor">
<a href="#the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis" class="anchor"></a>The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h3>
<p>Liu et al. <em>medRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.28.20045765]</span></p>
<div id="keywords-92" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-92" class="anchor"></a>Keywords</h4>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
</div>
<div id="main-findings-89" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-89" class="anchor"></a>Main Findings</h4>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin Hospital (Wuhan University, China) were analyzed. The positive ratio was 78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N qPCR detection kit). There were no differences in the sensitivity of SARS-CO-V2 diagnosis in patients grouped according to disease severity. For example, IgG responses were detected in 93.18% of moderate cases, 100% of severe cases and 97.3% of critical cases. In sum, positive ratios were higher in antibody testing compared to RT-PCR detection, demonstrating a higher detection sensitivity of IgM-IgG testing for patients hospitalized with COVID-19 symptoms.</p>
</div>
<div id="limitations-91" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-91" class="anchor"></a>Limitations</h4>
<p>This analysis only included one-time point of 133 hospitalized patients, and the time from symptom onset was not described. There was no discussion about specificity of the tests and no healthy controls were included. It would be important to perform similar studies with more patients, including younger age groups and patients with mild symptoms as well as asymptomatic individuals. It is critical to determine how early after infection/symptom onset antibodies can be detected and the duration of this immune response.</p>
</div>
<div id="significance-85" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-85" class="anchor"></a>Significance</h4>
<p>The IgM-IgG combined testing is important to improve clinical sensitivity and diagnose COVID-19 patients. The combined antibody test shows higher sensitivity than individual IgM and IgG tests or nucleic acid-based methods, at least in patients hospitalized with symptoms.</p>
</div>
<div id="credit-90" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-90" class="anchor"></a>Credit</h4>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Clinical observations of low molecular weight heparin in relieving inflammation in COVID-19 patients: a retrospective cohort study</strong></p>
<p><strong><em>Key words</em></strong></p>
<p>Heparin, IL-6, cytokine storm, anti-inflammatory, anti-coagulant</p>
<p><strong><em>Key findings</em></strong></p>
<p>The authors investigated the use of a commercially available form of heparin, low molecular weight heparin (LMWH), as a therapeutic drug for patients with COVID-19. Previous studies showed that in addition to its anticoagulant properties, LMWH exerts anti-inflammatory effects by reducing the release of IL-6 and counteracting IL-6.</p>
<p>This was a retrospective single-center study conducted in Wuhan, China. Forty-two (42) hospitalized patients with coronavirus pneumonia were included, of which 21 underwent LMWH treatment (heparin group) and 21 did not (control). The general characteristics of the two groups of patients were statistically comparable. Both control and LMWH had the same hospitalization time and there were no critical cases in either group.</p>
<p>This study found that treatment with LMWH significantly reduced IL-6 levels in patients in the heparin group compared to the control group. However, LMWH treatment did not have an effect on the levels of other inflammatory factors: CRP, IL-2, IL-4, IL-10, TNF-α, and IFN-γ. Compared with the control group, patients in the heparin group had a significantly increased percentage of lymphocytes after treatment, further suggesting that LMWH treatment has anti-inflammatory effects and can reduce the lymphopenia associated with COVID-19.</p>
<p>Consistent with other studies in COVID-19 patients, they found that LMWH treatment can improve hypercoagulability. D-dimer and FDP levels (biomarkers of coagulation) in the heparin group significantly decreased from baseline after treatment, whereas there was no significant change in levels for the control group. Of note however, patients in the heparin group had a significantly higher level of D-dimer and FDP at baseline compared to the control group.</p>
<p><strong><em>Importance</em></strong></p>
<p>Many studies have shown that severely ill COVID-19 patients have significantly higher levels of IL-6 compared to patients with mild cases and it has been proposed that progression to severe disease may be caused by lymphopenia and cytokine storms. The anti-inflammatory effects of heparin may help prevent or reverse a cytokine storm caused by the virus and thus delay COVID-19 progression and improve overall condition in patients. The pleiotropic effects of heparin may have a greater therapeutic effect than compounds that are directed against a single target, such as an anti-IL-6 therapy. This is because COVID-19 patients commonly have complications such as coagulopathy and endothelial dysfunction leading to cardiac pathology that may be mitigated by heparin treatment (Li J, et.al; Wojnicz et.al).</p>
<p><strong><em>Limitations</em></strong></p>
<p>This study is limited by a small sample size (n=44) and a single-center design. Double-blinded, randomized, placebo controlled clinical trials of LMWH treatment are needed to understand the possible benefit of the treatment. Additionally, this study was unable to control for the dose and days of treatment of LMWH. Identifying the correct dose and timing of LMWH is a matter of immediate interest. Of note, patients in the heparin group received two types of LMWH, enoxaparin sodium or nadroparin calcium, which have been reported to have differing anticoagulant activity. The use of different LMWHs in the heparin group warrants further explanation.</p>
<p>Another caveat of this study is that the levels of D-dimer and fibrinogen degradation products were significantly higher at baseline for patients in the heparin group compared to those in the control group. Therefore, it is difficult to decipher whether some of the positive effects of heparin treatment were due to its anti-coagulation effects or direct anti-inflammatory effects. Future studies are that delineate the anti-inflammatory functions of heparin independently of its anticoagulant properties in cases of COVID-19 would be useful.</p>
<p>Lastly, this study did not discuss any side-effects of heparin, such as the risk of bleeding. Moreover, coagulation can help to compartmentalize pathogens and reduce their invasion, therefore anticoagulant treatments like heparin may have risks and it remains to be determined which patients would benefit from this therapy.</p>
<p><strong>References:</strong></p>
<p>Li J, Li Y, Yang B, Wang H, Li L. Low-molecular-weight heparin treatment for acute lung injury/acute respiratory distress syndrome: a meta-analysis of randomized controlled trials. Int J Clin Exp Med 2018;11(2):414-422</p>
<p>Wojnicz R, Nowak J, Szyguła-Jurkiewicz B, Wilczek K, Lekston A, Trzeciak P, et al. Adjunctive therapy with low-molecular-weight heparin in patients with chronic heart failure secondary to dilated cardiomyopathy: oneyear follow-up results of the randomized trial. Am Heart J. 2006;152(4):713.e1-7</p>
<p><strong>Title</strong>: “Azithromycin and Ciprofloxacin have a chloroquine-like effect on respiratory epithelial cells” Jens F. Poschet &amp; al, Posted: 2020-03-31</p>
<p><strong>Key words</strong>: Azithromycin (AZT) and ciprofloxacin (CPX), hydroxychloroquine (HCQ), chloroquine (CQ), cystic fibrosis, lung epithelial cells, pH, NFkB, IL-8, Furin, TGF-β</p>
<p><strong>Summary</strong>: The study assesses the effect of azithromycin (AZT) and its potential mechanism of action in cystic fibrosis (CF) epithelial cells beyond anti-bacterial activity and through changes in the pH of intracellular organelles of lung epithelial cells. The authors also discuss potential implications for COVID-19 therapy, as they compare AZT to hydroxychloroquine (HCQ) and chloroquine (CQ), and further suggest the use of ciprofloxacin (CPX) based on acido-tropic lipophilic weak base properties of these drugs. The study examines AZT’s mechanism of action in vitro on cystic fibrosis respiratory epithelial cells and find:</p>
<ul>
<li><p>AZT elicits pH normalization in trans-Golgi network (TGN) and endosomes from their subtly increased acidification in CF epithelial cells.</p></li>
<li><p>AZT reduces inflammatory mediators in CF lung epithelial cells with or without bacterial products challenge: basal NFκB activation was reduced by 50% and by 40% in bacterial challenge; basal IL-8 secretion was reduced by almost 70 %, and by approximatively 33% after bacterial product challenge.</p></li>
<li><p>AZT corrects Furin activity in CF cells’ intracellular organelles and leads to a decrease in abundance of the profibrotic mediator TGF-beta.</p></li>
</ul>
<p><strong>Limitations</strong>:</p>
<p>This study is conducted in-vitro on respiratory epithelial cells affected by cystic fibrosis, and even though normal human epithelial cells were included in some experiments, they were not treated with AZT or CPX. The AZT results are derived from only 3 independent in-vitro experiments. The number of experiments supporting the CPX is not clear, nor why different cell types were used in AZT and CPX experiments. Very little is shown, except for TGN pH normalization, to suggest replication of AZT’s results with CPX.</p>
<p>The biological mechanism linking pH increase in the TGN and the endosome and the decrease in pro-inflammatory and pro-fibrotic effect in CF epithelial cell would need to be further explored.</p>
<p>The hypothesis of similar effect in pH shift in normal cells or in a SARS-CoV2 affected cells would need to be substantiated by experimental data.</p>
<p><strong>Findings implications</strong>: As we are desperately looking for a COVID-19 therapy, an hic and nunc approach identifying FDA-approved drugs with known safety profiles, is appealing. The preliminary in-vitro findings of this study warrant further in-vivo studies regarding these drugs.</p>
<p>Moreover, the biological mechanism underscored in this study could provide insights in the link between SARS-Cov2 viral infection and hyperinflammatory reaction as well as anti-viral effects. In particular, the Furin activity reduction by AZT may result in hindering SARS-CoV2 entry, as its Spike protein possess a novel Furin cleavage site.</p>
<p><em>Review by Jaime Mateus-Tique as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar epithelial cells and small intestine enterocytes, respiratory problems are the most common symptom after viral infection while intestinal symptoms are much less frequent. Thus, the authors here investigate the biology behind the observed protection of the intestinal epithelium from SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the small intestine, was shown to interact with human ACE2, with a binding affinity of 39.3 nM by biolayer interferometry (BLI). A blocking experiment using different doses of HD5 coating ACE2 showed that HD5 lowered viral spike protein S1 binding to ACE2. Further, a molecular dynamic simulation demonstrated a strong intermolecular interaction between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells were preincubated with HD5. Preincubation strongly reduced adherence of S1 to surface of cells. HD5 was effective at a concentration as low as 10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could have been tested to block ACE2-S1 binding in human lung epithelial cells as a potential treatment strategy. It would be useful to know whether HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency of viral entry and infection among ACE2-expressing cells and tissues. Specifically, the authors show that human defensin 5 produced in the small intestine is able to block binding between S1 and ACE2 necessary for viral entry into cells. The study provides a plausible explanation on why few patients show intestinal symptoms and suggests that patients with intestinal disease that decrease defensins’ production may be more susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a molecule to be exogenously administered to patients to prevent viral infection in lung epithelial cells.</p>
</div>
</div>
<div id="highly-pathogenic-coronavirus-n-protein-aggravates-lung-injury-by-masp-2-mediated-complement-over-activation" class="section level1">
<h1 class="hasAnchor">
<a href="#highly-pathogenic-coronavirus-n-protein-aggravates-lung-injury-by-masp-2-mediated-complement-over-activation" class="anchor"></a>Highly pathogenic coronavirus N protein aggravates lung injury by MASP-2-mediated complement over-activation</h1>
<p><em>Keywords:</em></p>
<p>Nucleocapsid protein, MASP-2, complement, pneumonia</p>
<p><em>Summary</em></p>
<p>In this manuscript, the authors describe direct complement system activation through the SARS-CoV-2-N-protein as a common denominator of coronavirus-induced lung injury.</p>
<p>The authors refer to previous evidence on SARS-CoV and MERS-CoV in which they showed that the nucleocapsid (N-) protein binds to a variant of MBL-associated serine protease-2 (MASP-2) that under physiologic conditions initiates the lectin pathway of the complement cascade. The authors hypothesized that SARS-CoV-2 activates the complement cascade in severely ill patients through viral N-protein-mediated dimerization and activating auto-cleavage of MASP-2 leading to respiratory failure in some patients.</p>
<p>Following confirmation of MASP-2 binding to the N protein of SARS-CoV, MERS-CoV and SARS-CoV2, the authors demonstrate the subsequent complement activation in vitro as measured by C4b deposition in a complement deposition assay and enhanced phagocyte activity. Translating these findings into an in vivo model, the authors pre-infected mice with an adenovirus expressing SARS CoV, MERS-CoV and control virus and challenged the animals with LPS to activate the lectin pathway. This effect was abolished when the N-protein was truncated or anti-MASP-2 antibodies or inhibitors were used. In mice infected with SARS- or MERS-CoV, the authors found significantly higher C4b and C4 deposits in the lungs and showed tremendously worse survival. However, when MASP-2 knockout mice were challenged with the above-mentioned viruses, survival was substantially better as compared to virus-infected wildtype mice.</p>
<p>The authors next analyzed lung tissues from deceased COVID-19 patients and found strong IHC stainings of complement components. In the final step, the authors applied a clinically used C5-inhibitor to two critically ill COVID-19 patients as a part of a clinical trial. Both patients showed clinical improvement of their severe pneumoniae.</p>
<p><em>Critical analysis</em></p>
<p>The question if and in how far coronaviruses are capable of activating the complement system has been controversial so far with conflicting data published.</p>
<p>However, the authors performed a thorough and convincing analysis with well-designed in vitro an in vivo experiments showing the ability of the nucleocapsid to activate the complement cascade through the lectin pathway.</p>
<p>However, the authors performed key experiments using adenoviruses only containing SARS-CoV and MERS-CoV but not SARS-CoV-2. While the assumption that due to the high sequence homology in the N-protein-binding motif and the proven binding of the SARS-CoV-2 N-protein to MASP-2 strongly suggests that the N-protein in SARS-CoV-2 also plays a major role in complement activation in vivo remains to be proven.</p>
<p><em>Implications of the findings in the context of current epidemics</em></p>
<p>Anti-inflammatory treatment in patients with severe viral pneumonia is established as a clinical standard procedure. However, most pathophysiologic aspects of immune hyperreactivity in these patients remains unclear. The study reviewed here shows one mechanism by which coronaviruses activate the complement cascade causing or at least aggravating inflammation in the lung. Showing the successful treatment of two patients with a complement C5 inhibitor should not be overrated since these patients are enrolled in an ongoing clinical trial assessing the significance of complement C5 inhibition in severely ill COVID-19 patients.</p>
<p><em>Additional resources used:</em></p>
<p>doi: 10.1038/emi.2015.28. Epub 2015 May 6.</p>
<p>doi: 10.1128/mBio.01753-18</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal, domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory animals (ferrets), as well as companion (cats and dogs), and domestic animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection with two SARS-CoV2 isolates, one from an environmental sample collected in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two ferrets from each group were euthanized on day 4 post infection (p.i.). AT day 4 p.i., viral RNA and infectious viruses were detected only in upper respiratory tract (nasal turbinate, upper palate, tonisls, but not in the trachea, lungs or other tissues. Viral RNA and virus titer in the remaining ferrets were monitored in nasal washes and rectal swabs on days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were detected in nasal washes until day 8 p.i. One ferret in each group developed fever and loss of appetite on days 10 and 12 p.i., however, viral RNA was practically undetactable. These two ferrets showed severe lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with CTan-h virus and three uninfected cats were placed in a cage adjacent to each of the infected cats to monitor respiratory droplet transmission. Viral RNA was detected in the upper respiratory organs from all infected cats and in one out of three exposed cats. All infected (inoculated and exposed) cats developed elevated antibodies against SARS-CoV2. Viral replication studies with juvenile cats (70-100 days) revealed massive lesions in the nasal and tracheal mucosa epithelium and lungs of two inoculated cats which died or were euthanized on day 3 p.i., and infection in one out of three exposed cats. These results indicated SARS-CoV2 could replicate in cats, that juvenile cats were more susceptible that adults, and theat SARS-CoV2 could be transmit via respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and housed with two uninoculated beagles in a room. Two virus inoculated dogs seroconverted, but others including two contact dogs were all seronegative for SARS-CoV2 and infectious virus was not detected in any swabs collected. Viral RNA was not detected in swabs from pigs, chickens, and ducks inoculated or contacted. These results indicated that dogs, pigs, chickens, and ducks might have low or no susceptibility to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and transmission in cats. Clinical and pathological analysis was not performed in cats, therefore the correlation of virus titer with symptoms severity in the adult and juvenile cats could not be determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously reported. It should be noted that this manuscript did not evaluate the transmission from cats to human. Nevertheless, it clearly showed higher susceptibility of ferrets and domestic cats to SARS-CoV-2. This data strongly indicates the need for surveillance of possible infection and transmission of SARS-CoV-2 by domestic cats.</p>
<div id="prediction-of-sars-cov-2-epitopes-across-9360-hla-class-i-alleles" class="section level3">
<h3 class="hasAnchor">
<a href="#prediction-of-sars-cov-2-epitopes-across-9360-hla-class-i-alleles" class="anchor"></a>Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles</h3>
<p>Campbell et al. <em>bioRxiv</em> <span class="citation">[@doi:10.1101/2020.03.30.016931]</span></p>
<div id="keywords-93" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-93" class="anchor"></a>Keywords</h4>
<ul>
<li><p>MHC class I</p></li>
<li><p>HLA</p></li>
<li><p>Haplotype</p></li>
<li><p>CD8+ T cells</p></li>
<li><p>CTL</p></li>
<li><p>SARS-CoV-2 peptidome</p></li>
</ul>
</div>
<div id="main-findings-90" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-90" class="anchor"></a>Main Findings</h4>
<p>In this study, Campbell et al. investigate the immunogenic potential of SARS-CoV-2-derived epitopes in the context of eliciting CD-8 T cell responses. Using pVAC, a computational toolkit developed by the Parker Institute for Cancer Immunotherapy to predict cancer neoantigens presented by human leukocyte antigen (HLA) class I, they analyzed the binding affinity of 9360 HLA-I alleles to 9-mer epitopes derived from all 11 viral proteins spanning the SARS-CoV-2 proteome. They identified 6,748 epitope – HLA-I pairs, which consisted of 1,103 unique 9-mer epitopes and 1,022 HLA class I alleles, with a predicted binding affinity Kd &lt; 500 nM. Each viral protein was predicted to generate immunogenic epitopes (12 to 684 epitopes, with a median of 31) and some of these epitopes could bind to multiple HLA-I alleles (1 to 55 alleles, with a median of 3). The study tested 2987 HLA-A, 3707 HLA-B and 2666 HLA-C alleles, of which majority of viral epitopes were predicted to bind HLA-B alleles (295 HLA-A, 614 HLA-B and 113 HLA-C). Of note, 10 of the 30 experimentally validated immunogenic SARS-CoV HLA-I epitopes were identified among current SARS-CoV-2 predicted epitopes. Together, this work shows that a diverse set of HLA alleles can potentially bind to a wide variety of viral peptides, suggesting the presence of strong cytotoxic T cell (CTL) responses against SARS-CoV-2 in COVID-19 patients and serves as a valuable resource for SARS-CoV-2 epitope selection and experimental validation efforts, which would guide the design of T-cell based vaccination strategies against COVID-19.<span id="limitations" class="anchor"></span></p>
</div>
<div id="significance-86" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-86" class="anchor"></a>Significance</h4>
<p>A limitation of this study is that it identifies only 9-mers peptides with &lt;500 nM binding affinity, but it does not look for 8-mer or 10-mer peptides and excludes peptides with &gt;500 nM affinity but within statistical rank-cutoff of 2%, which could still generate an immune response to the virus. Another limitation is that the authors did not evaluate epitope - HLA-II pairs, even though HLA class II is also important in the context of viral immune responses. Studies have shown that convalescent sera of recovered COVID-19 patients contain neutralizing antibodies, which can be used as an early treatment for recipient patients or as a prophylactic for at-risk or exposed patients. Generation of these antibodies require viral detection by B cells, aided by antigen presentation by HLA II molecules to CD4 T cells, which stimulates B cells to differentiate into antibody-producing plasma cells. Thus, further study could provide insight into the ability to potentially initiate potent antibody responses by identifying longer peptides and their affinity for class II variants.</p>
</div>
<div id="significance-87" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-87" class="anchor"></a>Significance</h4>
</div>
<div id="this-study-predicts-6748-unique-pmhc-complexes-across-the-sars-cov-2-peptidome-and-global-variety-of-hla-alleles-highlighting-that-a-wide-variety-of-patients-can-generate-a-ctl-mediated-response-to-infection--these-results-can-help-explain-the-heterogeneity-of-covid-19-disease-phenotypes-and-can-be-used-to-monitor-cd8-t-cell-responses-across-patients-with-many-different-haplotypes--since-sars-cov-2-proteome-is-quite-large-to-be-screened-experimentally-for-immunogenic-epitopes-in-an-unbiased-fashion-these-in-silico-findings-guide-the-selection-of-immunogenic-epitopes-for-functional-validation-of-sars-cov-2-epitopes-which-can-be-used-in-vaccine-development-and-pre-clinical-treatment-studies--moreover-all-data-generated-in-this-study-is-publicly-available-for-further-investigation-which-is-critical-for-clinical-translation-in-the-evolving-pandemic-" class="section level4">
<h4 class="hasAnchor">
<a href="#this-study-predicts-6748-unique-pmhc-complexes-across-the-sars-cov-2-peptidome-and-global-variety-of-hla-alleles-highlighting-that-a-wide-variety-of-patients-can-generate-a-ctl-mediated-response-to-infection--these-results-can-help-explain-the-heterogeneity-of-covid-19-disease-phenotypes-and-can-be-used-to-monitor-cd8-t-cell-responses-across-patients-with-many-different-haplotypes--since-sars-cov-2-proteome-is-quite-large-to-be-screened-experimentally-for-immunogenic-epitopes-in-an-unbiased-fashion-these-in-silico-findings-guide-the-selection-of-immunogenic-epitopes-for-functional-validation-of-sars-cov-2-epitopes-which-can-be-used-in-vaccine-development-and-pre-clinical-treatment-studies--moreover-all-data-generated-in-this-study-is-publicly-available-for-further-investigation-which-is-critical-for-clinical-translation-in-the-evolving-pandemic-" class="anchor"></a>This study predicts 6,748 unique pMHC complexes across the SARS-CoV-2 peptidome and global variety of HLA alleles, highlighting that a wide variety of patients can generate a CTL-mediated response to infection. These results can help explain the heterogeneity of COVID-19 disease phenotypes and can be used to monitor CD8+ T cell responses across patients with many different haplotypes. Since SARS-CoV-2 proteome is quite large to be screened experimentally for immunogenic epitopes in an unbiased fashion, these in silico findings guide the selection of immunogenic epitopes for functional validation of SARS-CoV-2 epitopes, which can be used in vaccine development and pre-clinical treatment studies. Moreover, all data generated in this study is publicly available for further investigation, which is critical for clinical translation in the evolving pandemic.</h4>
</div>
<div id="credit-91" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-91" class="anchor"></a>Credit</h4>
<p><em>Reviewed </em>by Miriam Saffern* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><strong>Neutralizing antibody responses to SARS-CoV-2 in a COVID-19 recovered patient cohort and their implications</strong></p>
<p>Fan Wu et al.; medRxiv 2020.03.30.20047365; <a href="doi:https://doi.org/10.1101/2020.03.30.20047365" class="uri">doi:https://doi.org/10.1101/2020.03.30.20047365</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>Neutralizing antibodies</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>pseudotype neutralization assay</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this study, plasma obtained from 175 convalescent patients with laboratory-confirmed mild COVID-19 was screened for SARS-CoV-2-specific neutralizing antibodies (nABs) by pseudotype-lentiviral-vector-based neutralization assay as well as for binding antibodies (Abs) against SARS-CoV-2 RBD, S1 and S2 proteins by ELISA. Kinetics of neutralizing and binding Ab titers were assessed during the acute and convalescent phase in the context of patient age as well as in relation to clinical markers of inflammation (CRP and lymphocyte count at the time of hospitalization). Across all age groups, SARS-CoV-2-specific nAbs titers were low within the first 10 days of symptom onset, peaked between days 10-15, and persisted for at least two weeks post discharge. In contrast to spike protein binding Abs, nAbs were not cross-reactive to SARS-CoV-1. Moreover, nAb titers moderately correlated with the amount of spike protein binding antibodies. Both neutralizing and binding Ab titers varied across patient subsets of all ages, but were significantly higher in middle-aged (40-59 yrs) and elderly (60-85) <em>vs</em>. younger patients (15-39 yrs). However, plasma nAb titers were found to be below detection level in 5.7% (10/175) of patients, i.e. a small number of patients recovered without developing a robust nAb response. Conversely, 1.14% (2/175) of patients had substantially higher titers than the rest. Notably, in addition to patient age, nAb titers correlated moderately with serum CRP levels but were inversely related to lymphocyte count on admission. In summary, the authors show that patients with clinically mild COVID-19 disease mount a strong humoral response against the SARS-CoV-2 spike protein. Compared to younger patients, middle-aged and elderly patients had both higher neutralizing and binding Ab titers, accompanied by increased CRP levels and lower lymphocyte counts. These patients are usually considered at higher risk of severe disease. Therefore, robust neutralizing and binding Ab responses may be particularly important for recovery in this patient subset. Conversely, patients who failed to produce high nAb/binding Ab titers against spike protein did not progress to severe disease, indicating that binding Abs against other viral epitopes as well as cellular immune responses are equally important.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This study provides valuable information on the kinetics of spike protein-specific nAb as well as binding Ab titers in a cohort of convalescent mild COVID-19 patients of all ages. However, similar studies enrolling larger patient numbers, including those diagnosed with moderate and severe disease as well as survivors and non-survivors, especially in the elderly group (to rule out potential bias for more favorable outcome), are warranted for reliable assumptions on the potentially protective role of Abs and nAbs in COVID-19. Moreover, longitudinal observation beyond the acute and convalescent phase in addition to stringent clinical and immunological characterization is urgently needed.</p>
<p>In their study, Wu et al. did not measure binding Abs against non-S viral proteins, which are also induced in COVID-19 and therefore could have added valuable diagnostic information with regard to patients who seemingly failed to mount both binding and neutralizing Ab responses against the SARS-CoV-2 spike protein. Likewise, while this study excluded cross-reactivity of nAbs against SARS-CoV-1, no other coronaviruses were tested. Of additional note, neutralizing activity of plasma Abs was only assessed by pseudotype neutralization assay, not against live SARS-CoV-2. Generally, while these are widely used and reproducible assays, <em>in vitro</em> neutralization of pseudotyped viruses does not necessarily translate to effective protection against the respective live virus <em>in vivo</em> (<em>cf</em>. review by Burton, D. Antibodies, viruses and vaccines. <em>Nat Rev Immunol</em> <strong>2, </strong>706–713 (2002)). Further studies are therefore needed to assess the specificity and neutralizing characteristics of these Abs to test whether they could be candidates for prophylactic and therapeutic interventions. In this context, setting arbitrary cut-off values (ID<sub>50</sub> &lt;500 <em>vs</em>. detection limit o ID<sub>50</sub> &lt; 40) and thus classifying up to 30% of patients in this study as “weak” responders does not take into account our currently limited knowledge regarding protective capacity of these nAbs and should therefore have been avoided by the authors.</p>
<p><strong><em>Significance</em></strong></p>
<p>This preprint is arguably the first report on neutralizing and binding Ab titers in a larger cohort of mild COVID-19 patients. Assessing Ab titers in these patients is not only important in order to confirm whether mild COVID-19 infection elicits robust nAb responses, but also adds further information regarding the use of plasma from mild disease patients for convalescent plasma therapy as well as vaccine design in general. Future studies will need to address now whether the nAb responses generated in mild disease will be protective or (functionally) different from nAbs generated in moderate and severe disease. The findings in this study are therefore of great relevance and should be further explored in ongoing research on potential coronavirus therapies and prevention strategies.</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em> </p>
<p> </p>
<p>Title: Association of BCG vaccination policy with prevalence and mortality of</p>
<p>COVID-19</p>
<p>Immunology Keywords</p>
<p>Bacillus Calmette–Guérin (BCG) Immunization, COVID-19 prevalence, COVID-19 deaths</p>
<p>Main findings</p>
<p>Previously reported immunization programs using BCG vaccines have demonstrated heterologous protection against other unrelated pathogens that associated with lower mortality and morbidity risks <span class="math display">\[1\]</span>. Therefore this study investigated the possible correlation between COVID-19 death cases or prevalence with BCG vaccination. The authors used publicly available COVID-19 data from 136 countries as well as vaccination demographics from the BCG World Atlas to perform a linear regression modeling.</p>
<p>After correcting for life expectancy and the onset of the spread of the virus (n=40), the analyses revealed a positive effect of current BCG vaccination programs and controlling the number of COVID-19 cases and deaths.</p>
<p>The amount of variance explained by BCG vaccination was 20% for number of cases and significant for both groups of countries, the ones that used to have a BCG immunization program in the past (b = 0.6122, p = .0024) and the ones that never have it (b = 0.6511, p = .0326).</p>
<p>Only the group of countries that never vaccinated against BCG showed significance in deaths/cases ratio but explains only 3.39% of the observed variance.</p>
<p>The authors concluded that BCG immunization may provide protection against COVID-19 probably due to the infection spread reduction. BCG immunization doesn’t have a significant impact in the mortality induced by COVID-19.</p>
<p>Limitations:</p>
<p>As acknowledged by the authors of this study, there are large number of unexplained potential confounding variables such as BCG immunization coverage, and onset of virus spread in different countries.</p>
<p>The authors cite that BCG immunization coverage could be variable among countries, but they didn’t explore it. Further, vaccination coverage changes at different rates over time across countries for different reasons <span class="math display">\[2\]</span>. Additionally, the authors did not consider the variable immunization coverage within countries, where unequal access to healthcare is frequently observed <span class="math display">\[3, 4\]</span>.</p>
<p>The authors do not adequately control for time of spread in infection for each country <span class="math display">\[5\]</span>.</p>
<p>The authors discuss the importance of validating experimentally the results observed and claim that BCG vaccination could provide non-specific protection against COVID-19. A stronger discussion of the use of BCG vaccine would have included known considerations on efficacy considering route of administration (intravenous, intradermal), vaccine strains which are known to differ in the number of viable bacteria and duration of protection.</p>
<p>Relevance:</p>
<p>This study presented preliminary data on possible non-specific protection by BCG immunization on COVID-19 infection.</p>
<p>References</p>
<p>1. Aaby, P., T.R. Kollmann, and C.S. Benn, <em>Nonspecific effects of neonatal and infant vaccination: public-health, immunological and conceptual challenges.</em> Nat Immunol, 2014. <strong>15</strong>(10): p. 895-9.</p>
<p>2. Nuffieldtrust. <em>Vaccination coverage for children and mothers</em>. 2020 [cited 2020; Available from: <a href="https://www.nuffieldtrust.org.uk/resource/vaccination-coverage-for-children-and-mothers-1?gclid=CjwKCAjw1cX0BRBmEiwAy9tKHhtZ6v3YmGA_N9MSd5IGM0rPHkb3vv4MViuWHpJjmBnKFJ0OAvNJtxoCCQAQAvD_BwE" class="uri">https://www.nuffieldtrust.org.uk/resource/vaccination-coverage-for-children-and-mothers-1?gclid=CjwKCAjw1cX0BRBmEiwAy9tKHhtZ6v3YmGA_N9MSd5IGM0rPHkb3vv4MViuWHpJjmBnKFJ0OAvNJtxoCCQAQAvD_BwE</a>.</p>
<p>3. WHO. <em>10 facts on health inequities and their causes</em>. 2017; Available from: <a href="https://www.who.int/features/factfiles/health_inequities/en/" class="uri">https://www.who.int/features/factfiles/health_inequities/en/</a>.</p>
<p>4. Balance. <em>Health Care Inequality in America</em>. 2020; Available from: <a href="https://www.thebalance.com/health-care-inequality-facts-types-effect-solution-4174842" class="uri">https://www.thebalance.com/health-care-inequality-facts-types-effect-solution-4174842</a>.</p>
<p>5. Statista. <em>Rate of coronavirus (COVID-19) tests performed in select countries worldwide as of April 8, 2020 (per thousand population)*</em>. 2020; Available from: <a href="https://www.statista.com/statistics/1104645/covid19-testing-rate-select-countries-worldwide/" class="uri">https://www.statista.com/statistics/1104645/covid19-testing-rate-select-countries-worldwide/</a>.</p>
<p>Review by Alessandra Soares Schanoski as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
<p><strong>Title</strong>: Fatal toxicity of chloroquine or hydroxychloroquine with metformin in mice</p>
<p><strong>Summary</strong>: Chloroquine (CQ) and hydroxychloroquine (HCQ) have been presented as potential therapeutic strategies for acute respiratory syndrome, a serious complication sometimes observed in COVID-19 patients. The drug was rapidly pushed to clinical testing and approved within weeks as a recommended COVID-19 treatment by the National Health Commission of the People’s Republic of China<span class="math display">\[1\]</span>. However, CQ or HCQ, sometimes used in combination with anti-diabetic drug metformin, may have serious toxic side-effects that have not been appropriately addressed.</p>
<p>The authors, who study metformin with CQ or HCQ as potential anti-cancer drugs in mice, study their toxicity. They found that the combination was lethal in at least 40% of nude mice, tumour-bearing or not. Similar results were obtained in non-immunocompromised mice. They observed that nude mice treated with HCQ+metformin presented an increased number of autophagosomes in the heart, liver and kidneys. Lactate dehydrogenase (LDH) and creatine kinase (CK) were also elevated in treatment groups compared to control vehicle-treated mice.</p>
<p><strong>Critical analysis</strong>: The authors of this article perform studies exclusively in mice. They note that the lethal doses of CQ or HCQ are similar to those in humans with allometric scaling. However, the side-effects of the combinations observed in mice must yet be confirmed in humans. The article acts as a word of caution for individuals and physicians who have been taking and prescribing these drug combinations prior to adequate clinical trials.</p>
<p><strong>Relevance to current epidemic</strong>: The authors note that both CQ and HCQ can lead to the development of hypoglycemia, cardiomyopathy and retinopathy. The dose of 500mg twice per day of CQ recommended in China can rapidly approach a dangerous dose in humans<span class="math display">\[1\]</span>. CQ poisoning has already been reported in Nigeria and resulted in at least one death in Arizona; the clinical relevance of treating COVID-19 patients with CQ+metformin or HCQ+metformin is yet to be confirmed or rejected.</p>
<p><strong>References</strong>:</p>
<p>1. Wong et al. Caution and clarity required in the use of chloroquine for COVID-19, <em>The Lancet,</em> April 02 2020</p>
<p><strong>Title</strong>: Fatal toxicity of chloroquine or hydroxychloroquine with metformin in mice</p>
<p><strong>Summary</strong>: Chloroquine (CQ) and hydroxychloroquine (HCQ) have been presented as potential therapeutic strategies for acute respiratory syndrome, a serious complication sometimes observed in COVID-19 patients. The drug was rapidly pushed to clinical testing and approved within weeks as a recommended COVID-19 treatment by the National Health Commission of the People’s Republic of China<span class="math display">\[1\]</span>. However, CQ or HCQ, sometimes used in combination with anti-diabetic drug metformin, may have serious toxic side-effects that have not been appropriately addressed.</p>
<p>The authors, who study metformin with CQ or HCQ as potential anti-cancer drugs in mice, study their toxicity. They found that the combination was lethal in at least 40% of nude mice, tumour-bearing or not. Similar results were obtained in non-immunocompromised mice. They observed that nude mice treated with HCQ+metformin presented an increased number of autophagosomes in the heart, liver and kidneys. Lactate dehydrogenase (LDH) and creatine kinase (CK) were also elevated in treatment groups compared to control vehicle-treated mice.</p>
<p><strong>Critical analysis</strong>: The authors of this article perform studies exclusively in mice. They note that the lethal doses of CQ or HCQ are similar to those in humans with allometric scaling. However, the side-effects of the combinations observed in mice must yet be confirmed in humans. The article acts as a word of caution for individuals and physicians who have been taking and prescribing these drug combinations prior to adequate clinical trials.</p>
<p><strong>Relevance to current epidemic</strong>: The authors note that both CQ and HCQ can lead to the development of hypoglycemia, cardiomyopathy and retinopathy. The dose of 500mg twice per day of CQ recommended in China can rapidly approach a dangerous dose in humans<span class="math display">\[1\]</span>. CQ poisoning has already been reported in Nigeria and resulted in at least one death in Arizona; the clinical relevance of treating COVID-19 patients with CQ+metformin or HCQ+metformin is yet to be confirmed or rejected.</p>
<p><strong>References</strong>:</p>
<p>1. Wong et al. Caution and clarity required in the use of chloroquine for COVID-19, <em>The Lancet,</em> April 02 2020</p>
</div>
</div>
<div id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis" class="section level3">
<h3 class="hasAnchor">
<a href="#virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis" class="anchor"></a>Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h3>
<p>Li et al. <em>bioRxiv</em>. <span class="citation">[@doi:10.1101/2020.03.31.019216]</span></p>
<div id="keywords-94" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-94" class="anchor"></a>Keywords</h4>
<ul>
<li>PBMC</li>
<li>virulence factors – interaction network – nsp9</li>
<li>nsp10 – NKRF</li>
</ul>
</div>
<div id="main-findings-91" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-91" class="anchor"></a>Main findings</h4>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong> (PPI) with two different approaches: genome wide yeast-two hybrid (Y2H) and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were characterized. A screen of <strong>viral-host PPI</strong> was also established by overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293 cells and purifying each protein complex. Interacting host proteins were then identified by liquid chromatography and tandem mass spectrometry. 251 cellular proteins were identified, such as subunits of ATPase, 40S ribosomal proteins, T complex proteins and proteasome related proteins, for a total of 631 viral-host PPI. Several interactions suggesting protein-mediated modulation of the immune response were identified, highlighting the multiple ways SARS-CoV-2 might reprogram infected cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs from healthy donors (n=6) with PBMC from COVID-19 patients with mild (n=22) or severe (n=13) symptoms. 220 proteins were found to be differentially expressed between <em>healthy donors and mild COVID-19 patients</em>, and a pathway analysis showed <strong>a general activation of the innate immune response.</strong> 553 proteins were differentially expressed between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them (95%) being downregulated in severe patients<em>.</em> Functional pathway analysis indicated a defect of T cell activation and function in severe COVID-19. There was also evidence suggesting reduced antibody secretion by B cells. Together, these results suggest a <strong>functional decline of adaptive immunity</strong>. A FACS analysis of PBMC from severe patients indicated higher levels of IL6 and IL8 but not IL17 compared to mild patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6 synthesis that was previously identified as interacting with SARS-Cov-2 nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459 cells, indicating that nsp9 and nsp10 may be directly involved in the induction of these pro-inflammatory cytokines. The authors finally argue that nsp9 and nsp10 represent potential drug targets to prevent over-production of IL6 and IL8 in infected cells, and reducing the over-activation of neutrophils.</p>
</div>
<div id="limitations-92" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-92" class="anchor"></a>Limitations</h4>
<p>First, the authors seem to have forgotten to include the extended data in the manuscript, and their proteomic data does not seem to be publicly available for the moment, which limits greatly our analysis of their results.</p>
<p>While this work provides important data on host and viral PPI, only 19 interactions were identified by Y2H system but 52 with co-IP. The authors do not comment about what could lead to such differences between the two techniques and they don’t specify whether they detected the same interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk PBMC. Consequently, protein differences likely reflect differences in cell populations rather than cell-intrinsic differences in protein expression. While this analysis is still interesting, a similar experiment performed on pre-sorted specific cell populations would allow measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to another SARS-CoV-2 interactome of host-viral PPI that had been published previously<sup>1</sup>. This study reported 332 host-virus PPI, but no interaction of viral proteins with NKRF was found. Some interactions were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However, the time point used to lyse the cells were different (40h previously vs 72h here), which could explain some of the differences.</p>
</div>
<div id="relevance" class="section level4">
<h4 class="hasAnchor">
<a href="#relevance" class="anchor"></a>Relevance</h4>
<p>The identification of many interactions between intra-viral and host-virus PPI provides an overview of host protein and pathways that are modulated by SARS-CoV-2, which can lead to the identification of potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2 induce an over-expression of IL6 and IL8 by lung epithelial cells, which recruits neutrophils and could lead to an excess in lung infiltration. Nsp9 has been shown to be essential for viral replication for SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>. Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help to develop drug inhibitors if this protein is further confirmed as being important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386. <a href="doi:10.1101/2020.03.22.002386" class="uri">doi:10.1101/2020.03.22.002386</a></p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW. Severe acute respiratory syndrome coronavirus nsp9 dimerization is essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018. <a href="doi:10.1128/JVI.01505-08" class="uri">doi:10.1128/JVI.01505-08</a></p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020. <a href="doi:10.1101/2020.03.28.013920" class="uri">doi:10.1101/2020.03.28.013920</a></p>
<p><strong>Structural basis to design multi-epitope vaccines against Novel Coranvirus 19 (COVID19) infection, the ongoing pandemic emergency: an in silico approach</strong></p>
<p>S Srivastava et. al. 2020 <a href="https://www.biorxiv.org/content/10.1101/2020.04.01.019299v1.full.pdf" class="uri">https://www.biorxiv.org/content/10.1101/2020.04.01.019299v1.full.pdf</a></p>
<p><strong>Keywords: </strong></p>
<ul>
<li><p>In silico</p></li>
<li><p>Epitope prediction</p></li>
<li><p>Immunoinformatics</p></li>
</ul>
<p><strong>Main Findings:</strong></p>
<p>This study reports the identification of in-silico screened epitopes capable of binding MHCI (CTL), MHCII (HTL), and B cells with high immunogenicity that can be formulated with Ochocerca volvulus activation-associated secreted protein-1 (Ov-ASP-1) adjuvant into two multi-epitope vaccines (MEVs) for SARS-CoV-2. CTL, HTL, and B cell linear epitopes were identified, scored, and percentile-ranked utilizing respective IEDB server tools. SARS-CoV-2 polyprotein, surface (S) glycoprotein, envelope (E) protein, membrane (M) protein, nucleocapsid (N) protein, and several open reading frame proteins were screened in silico for potential CTL, HTL, and B cell epitopes. CTL epitopes were identified by the “MHC-I Binding Predictions” IEDB tool with default parameters of 1<sup>st</sup>, 2<sup>nd</sup>, and C-term amino acids; epitopes were ranked by total score combining proteasomal cleavage, TAP transport, and MHC scores combined. HTL epitopes were identified by the “MHC-II Binding Predictions” IEDB tool, which gives a percentile rank by combining 3 methods (viz. combinatorial library, SMM_align &amp; Sturniolo, score comparison with random five million 15-mer peptides within SWISSPROT). B cell linear epitopes were identified by the “B Cell epitope Prediction” IEDB tool, which searches continuous epitopes based on propensity scales for each amino acid.</p>
<p>From these proteins, 38 CTL top percentile ranked epitopes, 42 HTL top scorers, and 12 B cell top scorers were used for further analysis. Candidates were then analyzed for epitope conservation analysis (number of protein sequences containing that particular epitope), toxicity, population coverage, and overlap with one another. 9 epitopes that overlapped among all three types (CTL, HTL, and B cell linear) were then analyzed for interaction with HLA binders, showing stable binding with A*11:01, A*31:01, B3*01:01, and B1*09:01, and TAP, demonstrating ability to pass from cytoplasm into the ER. Two MEVs were formulated using the top CTL and HTL epitopes, which were then analyzed for physicochemical properties, allergenicity, and potential to induce IFN-gamma production. Final 3D modeling, refinement, and discontinuous B cell epitope analysis were completed to optimize the space-occupancy of the MEVs. This rendering was used to assess docking with TLR3, the major domain used by Ov-ASP-1. Codon adaptation optimization yielded cDNA capable of high expression in mammalian host cells. Taken together, this in-silico study produced two MEVs containing CTL, HTL, and B cell epitopes capable of eliciting potent cell-mediated and humoral responses for HLA types representing up to 96% (SD 31.17) of the population. Further in vitro study is warranted to confirm its clinical potential.</p>
<p><strong>Limitations:</strong></p>
<p>In silico approaches are based upon models, however accurate, that make certain assumptions and contain biases inherent to training data. Synthesizing and testing a few candidates alongside their initial findings would make this method far more robust. It remains to be seen the efficacy of screened epitopes and corresponding multi-epitope formulations function in vitro and in vivo models.</p>
<p><strong>Significance:</strong></p>
<p>This study reports an in silico approach to producing multi-epitope vaccines that can produce potent adaptive immune responses. Utilizing protein databases, established protein modeling, folding, and docking algorithms, as well as population analysis, the team identifies 38 MHCI-binding, 42 MHCII-binding, and 12 B cell epitopes that can be linked with Ov-ASP-1 adjuvant to form stable proteins. These proteins are shown to dock well with HLA-alleles, TAP, TLR3, and to induce IFN-gamma responses.</p>
<p><em>Review by Matthew Lin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
</div>
<div id="notes" class="section level1">
<h1 class="hasAnchor">
<a href="#notes" class="anchor"></a>Notes</h1>
<p>11 Protein sequences</p>
<p>CTL, HTL, B cell epitopes</p>
<ul>
<li><p>MHCI (IC50s, percentile ranked; immunogenicity) 38 shortlisted</p></li>
<li><p>MHCII (affinity, percentile ranked) 42 shortlisted</p></li>
<li><p>Population coverage (score is % of population express HLA to which epitope binds)</p></li>
<li><p>B cell epitope (hydrophilicity, flexibility, accessibility, turns, exposed surface, polarity, antigenic propensity correlated with location in protein score) 12 shortlisted</p></li>
</ul>
<p>Overlapping (partial and complete)</p>
<ul>
<li><p>Conservation analysis (number of protein sequences containing particular epitope)</p></li>
<li><p>Toxicity (highly toxic vs non-toxic short peptides algorithm derived from datasets from Swiss-Prot)</p></li>
<li><p><strong>Overlap of all three (multiple sequence alignment) 9 shortlisted (2 CTL, 4 HTL, 3 B cell)</strong></p></li>
</ul>
<p><em>ONLY PARTIAL OR COMPLETE OVERLAP EPITOPES ARE NOW SCREENED BELOW</em></p>
<p>CTL/HTL HLA allele binders molecular docking</p>
<ul>
<li><p>Tertiary structure modeling (HLA structures based off IPD-IMGT/HLA; de novo structure prediction for epitopes)</p></li>
<li><p>Molecular docking (PatchDock) for respective HLA binders</p></li>
</ul>
<p>CTL TAP interaction molecular docking</p>
<p><em>NOW GOING BACK TO USING ALL THE TOP HITS NOT JUST THE OVERLAPPING SEQUENCES?</em></p>
<p>MEV – 38 CTL and 42 HTL then used to design MEV with EAAK and GGGS linkers, with adjuvant at N-term (Fig 2)</p>
<ul>
<li><p>ProtParam used for physicochemical property analysis of designed MEV (table 5)</p></li>
<li><p>IFNg screen (will the epitope induce IFNg?) 20 CTL and 20 HTL shortlisted</p></li>
<li><p>Allergenicity (AlgPred analysis) both CTL and HTL MEVs non-allergenic</p></li>
<li><p>Physiochemical analysis</p></li>
</ul>
<p>3D refinement</p>
<ul>
<li><p>Validation (RAMPAGE to show that residues are optimized for favored confirmation and minimized for outlier regions)</p></li>
<li><p>Linear and Discontinuous B cell epitope prediction (from refined structure) CTL MEV has 17 linear and 2 discontinuous; HTL MEV has 17 and 4 respectively</p></li>
</ul>
<p>TLR3 interaction molecular docking with ectodomain of TLR3 (PatchDock) stable complex formation tendency for both MEVs</p>
<p>cDNA analysis of sequence for cloning and expression</p>
<ul>
<li>Codon adaptation tool cDNA predicted to have high expression in mammalian host cell line</li>
</ul>
<p><strong>Title</strong>: <span id="page-title" class="anchor"></span>Level of IL-6 predicts respiratory failure in hospitalized symptomatic COVID-19 patients</p>
<p><strong>Summary</strong>: Hospitals around the world being overwhelmed due the COVID-19 pandemic calls for determination of predictive markers which would accurately predict patient outcome at an early stage. Severe COVID complications often include acute respiratory distress syndrome.</p>
<p>The authors analyse clinical and laboratory findings in 40 COVID patients. 13 required mechanical ventilation, and 27 did not. Age, comorbidities, radiological findings, respiratory rate or qSofa score (mortality prediction score upon sepsis) did not discriminate between both groups. However, all patients requiring intubation were male, and the pulse as well as several laboratory parameters correlated with the risk of respiratory failure. The strongest correlation was found to be with serological IL-6. Elevated IL-6 was found to increase the risk of respiratory failure by a factor of 22.</p>
<p><strong>Critical analysis</strong>: The value of 80pg/mL of IL-6 is identified as decisive, as 92% of patients who reached this cut-off required intubation 0-4 days afterwards. The study must be extended to a larger cohort, however, to perhaps identify a more precise cut-off.</p>
<p>Moreover, the causality between IL-6 and respiratory distress remains unclear. Understanding the mechanisms behind these correlations could pinpoint new therapies to prevent respiratory failure.</p>
<p><strong>Relevance to current epidemic</strong>: Foreseeing respiratory distress at an early stage in admitted COVID patients could allow hospitals to better manage their resources, both human and material.</p>
<p><strong>Title</strong>: <span id="page-title" class="anchor"></span>Level of IL-6 predicts respiratory failure in hospitalized symptomatic COVID-19 patients</p>
<p><strong>Summary</strong>: Hospitals around the world being overwhelmed due the COVID-19 pandemic calls for determination of predictive markers which would accurately predict patient outcome at an early stage. Severe COVID complications often include acute respiratory distress syndrome.</p>
<p>The authors analyse clinical and laboratory findings in 40 COVID patients. 13 required mechanical ventilation, and 27 did not. Age, comorbidities, radiological findings, respiratory rate or qSofa score (mortality prediction score upon sepsis) did not discriminate between both groups. However, all patients requiring intubation were male, and the pulse as well as several laboratory parameters correlated with the risk of respiratory failure. The strongest correlation was found to be with serological IL-6. Elevated IL-6 was found to increase the risk of respiratory failure by a factor of 22.</p>
<p><strong>Critical analysis</strong>: The value of 80pg/mL of IL-6 is identified as decisive, as 92% of patients who reached this cut-off required intubation 0-4 days afterwards. The study must be extended to a larger cohort, however, to perhaps identify a more precise cut-off.</p>
<p>Moreover, the causality between IL-6 and respiratory distress remains unclear. Understanding the mechanisms behind these correlations could pinpoint new therapies to prevent respiratory failure.</p>
<p><strong>Relevance to current epidemic</strong>: Foreseeing respiratory distress at an early stage in admitted COVID patients could allow hospitals to better manage their resources, both human and material.</p>
<div id="reduction-of-lymphocyte-at-early-stage-elevates-severity-and-death-risk" class="section level3">
<h3 class="hasAnchor">
<a href="#reduction-of-lymphocyte-at-early-stage-elevates-severity-and-death-risk" class="anchor"></a>Reduction of lymphocyte at early stage elevates severity and death risk</h3>
<p>of COVID-19 patients: a hospital-based case-cohort study</p>
<div id="keywords-95" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-95" class="anchor"></a>Keywords</h4>
<ul>
<li>COVID-19</li>
<li>lymphocyte</li>
<li>severity</li>
<li>organ injuries</li>
</ul>
</div>
<div id="main-findings-92" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-92" class="anchor"></a>Main findings</h4>
<p>The aim of this study was to assess an association between reduced blood lymphocyte counts at hospital admission and prognosis of COVID-19 patients (n=192). The authors found:</p>
<ul>
<li><p>Patients with lymphopenia are more likely to progress to severe disease or succumb to COVID-19 (32.1% of COVID-19 patients with lymphocyte reduction died).</p></li>
<li><p>Reduction of lymphocytes mainly affects the elderly (&gt; 70 years old).</p></li>
<li><p>Lymphocyte reduction is more prevalent in COVID-19 patients with cardiac disease and pulmonary disease, patients with increase in the chest CT score (key marker of lung injury) and a decrease in several respiratory function markers (PaCO2, SpO2, oxygenation index).</p></li>
</ul>
</div>
<div id="limitations-of-the-study" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-of-the-study" class="anchor"></a>Limitations of the study</h4>
<p>Reduced blood lymphocyte counts with aging have been known (<a href="https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2</em></a>) <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413"><em>https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413</em></a>). Therefore, it is not unexpected that a larger fraction of COV ID-19 patients above 70 years old have lower lymphocytes counts. Since age has been reported to be a major factor that determines outcome for COVID-19, lymphocyte counts and prognosis should have been adjusted by age. Multivariate analysis to identify independent risk factors is lacking.</p>
</div>
<div id="relevance-1" class="section level4">
<h4 class="hasAnchor">
<a href="#relevance-1" class="anchor"></a>Relevance</h4>
<p>Previous studies demonstrated that SARS-CoV-2 infection leads to a decrease of the T cell count. This study confirms these results and shows that lymphocyte reduction mainly affects the elderly. Lymphopenia was associated with disease severity as well as worse prognosis. Future studies need to address if lymphopenia is a negative predictive factor independent from age.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2 genomes in order to identify conserved linear and conformational B cell epitopes found on the spike (S), envelope (E), and membrane (M) proteins. The characterization of the surface proteins in this study began with an assessment of the peptide sequences in order to identify hydrophilicity indices and protein instability indices using the Port-Param tool in ExPASy. All three surface proteins were calculated to have an instability score under 40 indicating that they were stable. Linear epitopes were identified on the basis of surface probability and antigenicity, excluding regions of glycosylation. Using BepiPred 2.0 (with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51), 4 linear B cell epitopes were predicted for the S protein, 1 epitope for the E protein, and 1 epitope for the M protein. For structural analysis, SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for input into the SWISS-MODEL server in order to generate three-dimensional structural models for the assessment of conformational epitopes. Using Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1 conformational epitope was identified for the S protein and 1 epitope was identified for the E protein, both of which are accessible on the surface of the virus. Finally, the Consurf Server was used to assess the conservation of these epitopes. All epitopes were conserved across the published SARS-CoV-2 genomes and one epitope of the spike protein was predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential linear and conformational B cell epitopes, the translational value of the epitopes described still needs extensive experimental validation to ascertain whether these elicit a humoral immune response. The conformational epitope analyses are also limited by the fact that they are based off of predicted 3D structure from homology comparisons and not direct crystal structures of the proteins themselves. Additionally, since there was not a published M protein with a high homology to SARS-CoV-2, no conformational epitopes were assessed for this protein. Finally, while evolutionary conservation is an important consideration in understanding the biology of the virus, conservation does not necessarily imply that these sites neutralize the virus or aid in non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes induce effective antibody responses to the virus, the epitopes described can be used for the development of treatments and vaccines as well as better characterize the viral structure to more deeply understand pathogenesis.</p>
<p><em>Review of paper titled:</em></p>
<p><strong>Atazanavir inhibits SARS-CoV-2 replication and pro-inflammatory cytokine production</strong></p>
<p>N Fintelman-Rodrigues et. al. 2020 <a href="https://www.biorxiv.org/content/10.1101/2020.04.04.020925v1"><em>https://www.biorxiv.org/content/10.1101/2020.04.04.020925v1</em></a></p>
<p><strong>Keywords: </strong></p>
<ul>
<li><p>Repurpose drug</p></li>
<li><p>Major protease inhibitor</p></li>
<li><p>Antiretroviral</p></li>
</ul>
<p><strong>Summary:</strong></p>
<p>Major protease (Mpro) is a targetable viral protease important in processing polyprotein during the coronavirae replication phase. Atazanavir (ATV), an FDA-approved HIV protease inhibitor with fewer side effects than liponavir (currently in COVID-19 clinical trial: NCT04307693), can reach lung tissue through intravenous administration and suggests functional improvement in pulmonary fibrosis patients. In this study, in vitro assays with VeroE6 and A549 cell lines demonstrate ATV’s binding specificity for and inhibitory action on Mpro. Elevated lactate dehydrogenase (LDH) levels have previously been shown to correlate with COVID-19 mortality, and measured levels were reduced in ATV-treated cells infected with SARS-CoV-2. In addition, co-treatment with ATV and ritonavir prevented induction of IL-6 and TNF-a in monocytes, which was as good or better than chloroquine treatment. These data suggest it may be possible to repurpose ATV for SARS-CoV-2.</p>
<p><strong>Limitations:</strong></p>
<p>The study was performed entirely <em>in vitro</em> in VeroE6 and A549 cells.</p>
<p>Comparisons were done against chloroquine, a relatively toxic compound yet to demonstrate efficacy during in vivo testing<sup>1–3</sup>. Molecular docking stimulations are limited by previous data and training sets; while unlikely, there is a possibility that the binding properties modeled are not reflective of the true interactions.</p>
<p><strong>Importance of findings:</strong></p>
<p>This study identifies atazanavir, a HIV protease inhibitor, as an alternative to liponavir/ritonavir to treat SARS-CoV-2 infection. A safer toxicity profile, localization to lung tissue, and demonstrated in vitro efficacy to reduce cell mortality markers and inflammatory markers in monocytes make the case that ATV is a potential candidate to prevent SARS-CoV-2 viral replication.</p>
<p>References</p>
<p>1. Vigerust, D. J. &amp; McCullers, J. A. Chloroquine is effective against influenza A virus in vitro but not in vivo. <em>Influenza Other Respi. Viruses</em> <strong>1</strong>, 189–192 (2007).</p>
<p>2. Dowall, S. D. <em>et al.</em> Chloroquine inhibited ebola virus replication in vitro but failed to protect against infection and disease in the in vivo guinea pig model. <em>J. Gen. Virol.</em> <strong>96</strong>, 3484–3492 (2015).</p>
<p>3. Wang, J., Yu, L. &amp; Li, K. Benefits and Risks of Chloroquine and Hydroxychloroquine in The Treatment of Viral Diseases: A Meta-Analysis of Placebo Randomized Controlled Trials. <em>medRxiv</em> 2020.04.13.20064295 (2020). <a href="doi:10.1101/2020.04.13.20064295" class="uri">doi:10.1101/2020.04.13.20064295</a></p>
<p><em>Review by Matthew Lin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
<div id="title-on-the-interactions-of-the-receptor-binding-domain-of-sars-cov-1-and-sars-cov-2-spike-proteins-with-monoclonal-antibodies-and-the-receptor-ace2" class="section level4">
<h4 class="hasAnchor">
<a href="#title-on-the-interactions-of-the-receptor-binding-domain-of-sars-cov-1-and-sars-cov-2-spike-proteins-with-monoclonal-antibodies-and-the-receptor-ace2" class="anchor"></a>Title: <strong>On the interactions of the receptor-binding domain of SARS-CoV-1 and SARS-CoV-2 spike proteins with monoclonal antibodies and the receptor ACE2</strong>
</h4>
<p>Correa Giron et al., <em>bioRxiv</em> <span class="math display">\[@doi: []{#_Hlk37590784
.anchor}10.1101/2020.04.05.026377\]</span></p>
</div>
<div id="keywords-96" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-96" class="anchor"></a>Keywords</h4>
<ul>
<li><p>Monoclonal antibodies</p></li>
<li><p>Binding affinities</p></li>
<li><p>RBD/ACE2 interaction</p></li>
</ul>
</div>
<div id="main-findings-93" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-93" class="anchor"></a>Main Findings</h4>
<p>This study combined a theoretical approach, with structural and computational methods to identify antibody epitopes on the ACE2 (angiotensin-converting enzyme 2) receptor-binding-domains of the Spike (S) proteins of SARS CoV1 and SARS CoV2. The authors used constant-pH Monte Carlo (MC) simulations and the PROCEEDpKa method, to determine binding affinities of four SARS-Cov monoclonal antibody fragments (CR3022, 80R, m39, F26G29<strong>)</strong> to SARS-Cov2, as well as the titratable amino acids involved in the interactions. They also compared antibody binding affinities to the binding affinity of S RBD of SARS-Cov1 and 2 to the ACE2 receptor to ensure that antibody binding affinities were higher compared to those of S RBD/ACE2. They identified CR3022 as the Ab having the highest affinity with SARS-Cov2 RBD. Based on their simulations they were able to suggest some 3 amino acid modifications on CR3022, to enhance its binding affinity to SARS-Cov2 RBD. The computational analysis of interactions between SARS-CoV-1 S RBD proteins and the S RBD-specific neutralizing mAbs (80R, F26G19, m396, CR3022) recapitulated previous experimental results suggesting that the approach was valid. The only mAb with measured affinity for the SARS-CoV-2 S RBD protein by BLI assay was CR3020, which also exhibited the highest theoretical binding affinity determined in the present study.</p>
</div>
<div id="limitations-93" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-93" class="anchor"></a>Limitations</h4>
<p>Simulations were performed on S RBD proteins alone. However, given that the S protein is trimeric, structure modifications and/or conformational changes in the S protein trimer could alter antibody binding affinity. In addition, while CR3022 has been shown to bind S RBD, it did not neutralize SARS-Cov2 in vitro. The CR3022 modifications suggested by the authors could enhance the binding affinitys and the neutralizing potential of CR3022, but this is currently hypothetical and remains to be tested.</p>
</div>
<div id="significance-88" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-88" class="anchor"></a>Significance</h4>
<p>These results are in agreement with previous experimental studies where CR3022 was shown to bind SARS-Cov2 RBD (10.1101:2020.01.28.923011, 10.1126/science.abb7269) and confirm the validity of the approach. A human monoclonal antibody against SARS-Cov2 RBD could be used as therapeutics to treat patients with severe COVID-19, by inhibiting ACE2/RBD interaction.</p>
</div>
<div id="credit-92" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-92" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>Edited by K Alexandropoulos</p>
<p><em>.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Genomic determinants of pathogenicity in SARS-CoV-2 and other human coronaviruses</strong></p>
<p><strong>Immunology keywords: SARS-CoV-1, SARS-CoV-2, MERS-CoV, pathogenicity sequences, comparative genomics</strong></p>
<p><strong>The main findings of the article: </strong></p>
<p>In this study, potential genomic determinants of pathogenicity of the high case fatality rates (CFR) coronavirus strains were identified using integrated comparative genomics and advanced machine learning methods.</p>
<p>A total 3001 coronavirus genomes were compared. Of those, 944 belong to viruses that infect humans, including both viruses with low CFR (NL63, 229E, OC43 and HKU1) and those with high CFR (MERS, SARS-CoV-1 and SARS-CoV-2). The full genomes of these coronaviruses were aligned to detect high-confidence genomic features that were predictive of the high-CFR. Eleven regions were detected in total, and the nucleocapsid (N) protein and the spike (S) glycoprotein were significantly enriched with those predictive regions. Further analysis revealed that 4 out of 11 differences identified were reflected in the protein alignment, 3 of which resided in the N phosphoprotein and one in the S glycoprotein. The deletions and insertions found in N proteins mapped to monopartite nuclear localization signals (NLS), one bipartite NLS and a nuclear export signal (NES). They resulted in the accumulation of positive charges in the monopartite NLS, bipartite NLS and NES of SARS-CoV-1 and SARS-CoV-2, whereas in MERS-CoV, positive charges accumulated primarily in the first of the two monopartite NLS. The accumulation of positive charges is known to result in the enhancement of these signals, thus implying that the localization pattern of the N proteins differs between high-CFR and low-CFR strains, and this may be a factor contributing to the increased pathogenicity of high-CFR strains. The analysis of S protein revealed a unique 4 amino acid insertion upstream of the heptad repeat region in all high-CFR viruses but not in any of the low-CFR ones. The structural analysis of SARS-CoV confirmed that this insertion increased the length and flexibility of the region connecting the fusion peptide and the first heptad repeat, likely affecting the membrane fusion process of the high-CFR coronaviruses.</p>
<p>By the alignment of all 3001 collected coronavirus strain sequences, they also identified a unique insert in the RBD and the RBM in the cases of SARS-CoV and SARS-CoV-2, and within the subdomain that binds to DPP4 in the case of MERS-CoV. This unique insert was also found in the most proximal zoonotic strains before the zoonotic strains jumped to human. The unique insertions occurred independently in three high-CFR strains and resulted in a larger hydrophobic surface and additional interactions, which might contribute not only to the zoonotic transmission of the high-CFR CoV strains to humans but also their high CFR.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This study provides predictions on unique sequences present in coronaviruses of high pathogenicity, and in highly related zoonotic species. Functional analysis of the unique elements is not carried out.</p>
<p>There is an inconsistency related to description of positive/negative charged proteins between figure legends of 2 and 3, and the main text.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>The identification of unique sequences presents in the high-CFR but not in the low-CFR human coronaviruses can be experimentally tested to investigate the importance of each unique motive in the pathogenicity and transmissibility of human coronaviruses. This information will be useful to identify viruses with a potential to cause future pandemic, and to develop novel therapeutic strategies.</p>
<p>Title: The phenotypic changes of γδ T cells in COVID-19 patients</p>
<p><em>Keywords</em></p>
<p>T cells, Innate immune response, CD25, γδ T cells.</p>
<p><em>Main findings</em></p>
<p>γδ T cells represent 1-10% of total circulating lymphocytes, and are able to promptly respond to infections and other stimuli. To address the effects of SARS-CoV-2 infection on γδ T cells, the authors collected blood samples from 18 healthy donor (HD) s and 38 COVID-19 patients and analyzed the frequency and phenotype of γδ T cells.</p>
<p>Unlike other viral infections, COVID-19 patients presented lower frequency of circulating γδ T cells than HD. Among γδ T cells, there was an increase in the CD4+ subset and CD25 expression compared to non-infected volunteers. The authors also report that γδ T cells of COVID-19 patients didn’t present higher frequencies of early activation/exhaustion markers such as CD69 and PD-1, in comparison to the control group.</p>
<p><em>Limitations</em></p>
<p>There is very little information on COVID-19 patients or healthy volunteers (age and sex should be reported). Furthermore, since the time after the onset of symptoms and viral load may be important parameters in the innate immune response to SARS-CoV-2 the authors should have shown this information in a table.</p>
<p>Since lymphopenia is an important characteristic of COVID-19 patients <span class="math display">\[1\]</span> and the authors observed lower frequencies of γδ T cells, it would have been important to show frequencies and cell numbers for different T cell populations.</p>
<p><em>Relevance </em></p>
<p>This study indicates, for the first time, that γδ CD4+ CD25+ T cells can be involved in the acute phase of COVID-19. The function of γδ T cells in the context of COVID-19 should be further investigated.</p>
<p><em>References</em></p>
<p>1. Tan, L., et al., <em>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study.</em> Signal Transduct Target Ther, 2020. <strong>5</strong>(1): p. 33.</p>
<p>Review by Alessandra Soares Schanoski as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
</div>
<div id="title-structural-and-functional-analysis-of-a-potent-sarbecovirus-neutralizing-antibody" class="section level4">
<h4 class="hasAnchor">
<a href="#title-structural-and-functional-analysis-of-a-potent-sarbecovirus-neutralizing-antibody" class="anchor"></a><strong>Title</strong> <strong>Structural and functional analysis of a potent sarbecovirus neutralizing</strong> <strong>antibody </strong>
</h4>
<p>Pinto et al. <em>bioRxiv</em> <span class="math display">\[@doi: 10.1101/2020.04.07.023903\]</span></p>
</div>
<div id="keywords-97" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-97" class="anchor"></a>Keywords</h4>
<ul>
<li><p>Neutralizing antibodies</p></li>
<li><p>ACE2/RBD binding</p></li>
<li><p>SARS-CoV-1</p></li>
</ul>
</div>
<div id="main-findings-94" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-94" class="anchor"></a>Main Findings</h4>
<p><span id="limitations" class="anchor"></span>Antibodies were isolated from memory B cells of a recovered SARS-CoV-1 infected individual. 8 out of 25 isolated antibodies bound the SARS-CoV-2 Spike (S) protein, and 4 bound the S receptor binding domain (RBD) from SARS-CoV-1 and SARS-CoV-2. One of them (s309) neutralized SARS-CoV-1 and SARS-CoV-2 pseudoviruses with similar potencies by binding to S RBD, as well as authentic SARS CoV2 (IC50‑69ng/ml). Single particle cryoEM structural mapping showed that s309 bound to a strictly conserved protein/glycan epitope between SARS-CoV-1 and SARS-CoV-2, which is distinct from the RBD. Biolayer interferometry indeed confirmed that s309 did not interfere with ACE2 and S binding. The authors also showed that weakly neutralizing antibodies when used together with s309, enhanced its ability to neutralize SARS CoV2.</p>
<p>The authors further explored the effect of the different antibodies on NK-mediated antibody-dependent cell toxicity (ADCC) and antibody-dependent cellular phagocytosis (ADCP), both effector mechanisms that can contribute to viral control in infected individuals. s309 showed the highest ADCC and ADCP responses whereas other antibodies showed limited or no activity. The authors propose that s309 besides neutralization, may regulate protective mechanisms to mitigate viral risk as shown for other antiviral antibodies. (ref 41,42 in the paper)</p>
</div>
<div id="limitations-1" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-1" class="anchor"></a>Limitations</h4>
<p>Although s309 seems to be neutralizing in vivo, more studies need to be performed with pre-clinical models. Also, it would have been interesting to assess ADCC and ADCP in SARS-CoV-2 infected co-cultures of NK-cells and ACE2-expressing cells. Although the mechanism of neutralization by s309 is not understood, the authors postulate that S trimer crosslinking, steric hindrance or aggregation of virions may all contribute to the ability of the antibody to neutralize the virus.</p>
<p><em>(Of note, legend and panels of figure 3 are</em> <em>mismatched)</em></p>
</div>
<div id="significance-89" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-89" class="anchor"></a>Significance</h4>
<p><span id="credit" class="anchor"></span>This study identifies s309 as a potential human monoclonal neutralizing antibody. It doesn’t neutralize SARS-CoV-2 by inhibiting ACE2/RBD binding, as shown in other neutralizing antibodies<sup>1,2,3</sup>. While clinical data is still needed, it is a promising candidate for further development and <em>in</em> <em>vivo</em> testing. Moreover, s309 binds to a conserved epitope between SARS-CoV-1 and SARS-CoV-2, so it could possibly also neutralize other zoonotic sarbecovirus (sub-genus of coronavirus).</p>
</div>
<div id="credit-1" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-1" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><em>Edited by K Alexandropoulos</em></p>
<p>1. Sun C, Chen L, Yang J, et al. SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development. Microbiology; 2020. <a href="doi:10.1101/2020.02.16.951723" class="uri">doi:10.1101/2020.02.16.951723</a></p>
<p>2. Wang C, Li W, Drabek D, et al. A Human Monoclonal Antibody Blocking SARS-CoV-2 Infection. Microbiology; 2020. <a href="doi:10.1101/2020.03.11.987958" class="uri">doi:10.1101/2020.03.11.987958</a></p>
<p>3. Wu Y, Li C, Xia S, et al. Fully Human Single-Domain Antibodies against SARS-CoV-2. Microbiology; 2020. <a href="doi:10.1101/2020.03.30.015990" class="uri">doi:10.1101/2020.03.30.015990</a></p>
<p><strong>Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis</strong></p>
<p>Guo et al., bioRxiv <span class="math display">\[10.1101:2020.04.08.029769\]</span></p>
<p>Keywords:</p>
<ul>
<li><p>Tocilizumab</p></li>
<li><p>COVID-19</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>monocytes</p></li>
</ul>
<p><em>Main findings</em></p>
<p>Single-cell RNA sequencing was performed on PBMCs from two COVID-19 patients treated with Tocilizumab, an interleukin (IL)-6 receptor antagonist. One blood sample was taken from each patient within 12 hours post-treatment (severe stage of disease), and a total of three blood samples were taken 5 and 7 days post-treatment (recovery stage of disease). A total of 13,289 cells were analyzed post-quality control and compared to 55,948 previously published profiles of healthy PBMCs.</p>
<p>The authors identified a subpopulation of pro-inflammatory CD14<sup>+</sup> monocyte/macrophages enriched in the severe stage of COVID-19, characterized by the upregulation of TNF, IL-6, IL-1β, and CXCL8. In addition, a cluster of plasma cells and subpopulations of effector CD8<sup>+</sup> T cells were found to be enriched in both the severe and recovery stages. The authors attribute this finding to the fact that Tocilizumab therapy does not necessarily compromise the anti-viral immune response.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>Only two patients were included in this analysis. Though both patients were classified under severe COVID-19 disease, at the time of therapy, patients PZ and PW were at significantly different stages of the disease; PZ had an Sp<sub>O2</sub> &lt; 93%, and PW presented with respiratory failure, multiple organ dysfunction, and Sp<sub>O2</sub> &lt; 93% under high flow oxygen.</p>
<p>Of note, PBMCs collected from patients were compared to previously published healthy controls. However, this is not a proper control; cells collected from COVID-19 patients that had not been treated with Tocilizumab are needed. Therefore, this comparison is not sufficient to attribute significant differences in clustering patterns to Tocilizumab treatment. Moreover, without additional details concerning improvement of clinical symptoms after Tocilizumab therapy at days 5 and 7, there is a lack of data supporting the claim that these timepoints (only a week into therapy) necessarily represent a stage of recovery in COVID-19 patients.</p>
<p>Finally, PBMCs attributed to severe disease (presumably to define the pre-treatment stage) were collected after administration of Tocilizumab, instead of before administration to establish an untreated baseline control. In addition, PBMCs attributed to the recovery stage were collected at timepoints that were not necessarily the same for both patients. Recovery stage cells from patient PZ were collected 5 days post-treatment, whereas recovery stage cells from patient PW were collected 5 and 7 days post-treatment. Because it was not clarified, it must be assumed that the 5 and 7 day samples from patient PW were combined to represent the recovery stage for that patient, but this yields an inconsistent comparison between the two patients.</p>
<p><em>Biological </em></p>
<p>Enrichment of pro-inflammatory monocyte/macrophages in patients with severe infections is expected. The study is missing a more detailed characterization of cluster 9 (identified as the group of myeloid cells unique to the severe stage) to potentially establish a novel gene program specific to COVID-19. This is furthered by the fact that despite authors’ claims, cluster 9 technically belongs to both the severe and recovery stages rather than being stage-specific. It is notable that transcriptionally, the acute inflammatory response is reduced in the defined recovery stage cells.</p>
<p>Other findings are expected, including the persistence of plasma B cells and effector CD8<sup>+</sup> T cells in response to a viral infection. In addition to the technical limitations aforementioned, to better assess the impact of Tocilizumab on the inflammatory response, both transcriptional and translational outcomes must be investigated. It is not sufficient to account the downregulation of pro-inflammatory cytokines and chemokines to Tocilizumab treatment without the proper controls.</p>
<p><em>Significance</em></p>
<p>IL-6R antagonists, like Tocilizumab and Sarilumab, have been raised as potential therapeutics for COVID-19 as a means of reducing the hyper-inflammation seen in patients. Understanding the cellular changes that occur during and after therapy is important to determining the efficacy of such therapy and whether inhibition of IL-6 signaling compromises other adaptive immune mechanisms needed for a complete anti-viral response. This study contributes to that analysis at the single-cell level.</p>
<p><em>Reviewed by Matthew D. Park and Assaf Magen as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>Comparison of SARS-CoV-2 infections among 3 species of non-human primates</strong></p>
<p><strong>Immunology keywords: SARS-CoV-2, animal model</strong></p>
<p><strong>The main finding of the article: </strong></p>
<p>In this study, two species of old world monkeys, <em>Macaca mulatta</em> and <em>Macaca facicularis</em>, and one of new world monkeys, <em>Callithrix jacchus</em>, were used as potential animal models for SARS-CoV-2 infection. 4 adults and 4 old <em>M. mulatta</em> and 6 <em>M. facicularis</em> were inoculated with 4.75 x 10<sup>6</sup> pfu of virus intratracheally (4.0 ml), intranasally (0.5 ml) and on conjunctiva (0.25 ml), and half dosage of virus was given to 4 young <em>M. Mulatta</em>. 6 <em>C. jacchus</em> were inoculated with 1.0 x 10<sup>6</sup> pfu intranasally.</p>
<p>The authors described clinical symptoms of fever and weight loss were observed in 12/12 and 9/12 of <em>M. mulatta</em> and 2/6 and 5/6 of <em>M. facicularis</em> post infection, respectively. Chest radiographs of <em>M. mulatta</em> and <em>M. facicularis</em> revealed abnormalities in lungs in all animals from 10 days post infection (dpi), and progressive pulmonary infiltration was observed until 21 dpi. Along with clinical symptoms, viral RNA was detected in nasal swabs, pharyngeal swabs, anal swabs, feces, blood and tissues from <em>M. mulatta</em> and <em>M. facicularis</em> on 2 dpi at high level. Most of the swabs showed a second peaks on 6-8 dpi and viral RNA was detectable on 14 dpi in some swab samples. The analysis of viral load in the tissues revealed higher level of viral genomes in bronchus and trachea. Although no viral particles were detected by ultrastructural examination, the authors described that histopathological analysis revealed severe gross lesions on lung, heart and stomach of <em>M. mulatta</em> and <em>M. facicularis</em>, but not <em>C. jacchus.</em> While <em>C. jacchus</em> did not show any symptoms or viral RNA in the tissues, lower levels of viral RNA were detected in the nasal swab samples until 12 dpi. Virus-specific antibodies became detectable as early as 4 dpi with gradual increase in most of the animals, however, young <em>M. mulatta</em> showed overall lower levels of antibodies. Increase of IL-2, IL-6 was not detected, but G-CSF, IL-1A, IL-8, IL-15, IL-18, MCP-1, MIP-1B and sCD40L were detected in the serum from <em>M. mulatta</em> and <em>M. facicularis</em>.</p>
<p>Although they refer to a decline in CD4+ T cells, CD8+ T cells and monocytes in peripheral blood, the graphics shown are inconsistent with their statements.</p>
<p>Overall, the study shows that, among the macaques tested, <em>M. mulatta</em> has the highest susceptibility to SARS-CoV-2 infection, followed by <em>M. facicularis</em> and <em>C. jacchus</em>.</p>
<p><strong>Critical analysis of the</strong> <strong>study:</strong></p>
<p>The manuscript needs clearer annotation and consistency among the main text, methods and figure legends, and overall better scientific writing. Because of the lack of uninfected control or baseline data for immune cell numbers in healthy monkeys, neither the decline of T cells nor stronger B cell response in young animals can be assessed from the data. Cytokine levels in infected animals could also have been compared with uninfected animals, or pre-infection levels. The histopathology and TEM figures don’t have enough resolution and lack comparison with uninfected controls. The authors report that they did not find virus perticles in the TEM sections.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>While some of the observations of SARS-Cov2 infection in macaques were not consistent with those of human patients, non-human primates may still be a useful model for preclinical studies.</p>
<p>The Role of Vitamin D in Suppressing Cytokine Storm in COVID-19 Patients and Associated Mortality</p>
<p><strong>Keywords:</strong> Vitamin D, cytokine storm, CRP, COVID-19</p>
<p><strong>Key findings:</strong> Using clinical data of patients and statistical analysis, the authors wanted to explore the potential role of vitamin D in decreased cytokine levels which may lead to the reduction of disease severity in COVID-19 cases. The study shows that Spain and Italy present a severe vitamin D deficiency. Concerning COVID-19, the two countries have a higher mortality rate and a higher case fatality rate (CFR) for patients age ≥ 70. The authors have also shown that subjects with a severe deficiency of Vitamin D have 1.4 times higher risk for production of high C reactive protein (CRP), used to indirectly evaluate cytokine storm. They hypothesized that Vitamin D deficiency increases the risk for cytokine storm in COVID-19 patients and they estimated a 15% reduction in the number of severe COVID-19 cases after vitamin D deficiency elimination.</p>
<p><strong>Potential limitations:</strong> The results of this study remain hypothetical. Future studies should assess vitamin D and pro-inflammatory cytokine levels in COVID-19 patients upon hospital admission and the role of vitamin D in decreasing cytokine levels should be evaluated. In addition, the authors did not take into account the medical history of COVID-19 patients in their analysis, as previous studies have shown that patients who have comorbidities (cardiovascular disease, hypertension, chronic pulmonary disease, hepatic disease, diabetes and other chronic disease) are more likely to develop a severe COVID-19 <em><a href="https://www.medrxiv.org/content/10.1101/2020.03.11.20030957v1" class="uri">https://www.medrxiv.org/content/10.1101/2020.03.11.20030957v1</a></em></p>
<p><a href="https://www.medrxiv.org/content/10.1101/2020.03.30.20043919v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.30.20043919v1</em></a></p>
<p>Furthermore, age has been reported to be a major factor that determines outcome for COVID-19 and it has been demonstrated that aging affects the formation of the active form of vitamin D.<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4399494/pdf/nihms662547.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4399494/pdf/nihms662547.pdf</em></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3782116/pdf/nihms466447.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3782116/pdf/nihms466447.pdf</em></a> It would be interesting to see if vitamin D deficiency in the elderly associates with severe forms of COVID-19. Finally, the authors have chosen to use CRP to evaluate the intensity of cytokine storm which is an indirect approach. However, even if CRP has been regarded as a pro-inflammatory molecule, it has some anti-inflammatory functions (recruitment of complement inhibitors and release of anti-inflammatory cytokines such as IL-10 and IL-1ra) and could be replaced by more specific inflammatory markers. <a href="https://www.fasebj.org/doi/pdf/10.1096/fj.11-186460"><em>https://www.fasebj.org/doi/pdf/10.1096/fj.11-186460</em></a> <a href="https://www.sciencedirect.com/science/article/pii/S0889159118300266"><em>https://www.sciencedirect.com/science/article/pii/S0889159118300266</em></a></p>
<p><strong>Overall relevance for the field:</strong> The study results did not demonstrate a direct and obvious link between vitamin D deficiency and severe cases of COVID-19 that feature cytokine storm. The study is based only on analysis of the reported clinical data from multiple studies, the results are therefore hypothetical and require a more direct demonstration to see if vitamin D supplementation could really suppress cytokine storm in COVID-19 patients.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
</div>
</div>
<div id="type-2-and-interferon-inflammation-strongly-regulate-sars-cov-2-related-gene-expression-in-the-airway-epithelium" class="section level3">
<h3 class="hasAnchor">
<a href="#type-2-and-interferon-inflammation-strongly-regulate-sars-cov-2-related-gene-expression-in-the-airway-epithelium" class="anchor"></a>Type 2 and interferon inflammation strongly regulate SARS-CoV-2 related gene expression in the airway epithelium</h3>
<p>Sajuthi S.P. et al.  <span class="math display">\[@doi: 10.1101/2020.04.09.034454\]</span></p>
<div id="keywords-98" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-98" class="anchor"></a>Keywords</h4>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>scRNAseq</p></li>
<li><p>Type 2 Inflammation</p></li>
<li><p>Nasal airway epithelium</p></li>
<li><p>Interleukin</p></li>
</ul>
</div>
<div id="main-findings-immunological" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-immunological" class="anchor"></a>Main Findings (Immunological)</h4>
<ul>
<li><p>Analyzed nasal airway epithelial transcriptome data from 695 asthmatic and healthy children (GALA II study, no COVID-19) to determine WGCNA networks. TMPRSS2 gene was contained in a set of three highly correlated networks exhibiting strong enrichments for IL13-induced mucus secretory cell genes and canonically Type-2 inflammation pathways. ACE2 gene was correlated with interferon response and cytotoxic immune signaling eigengene enrichment.</p></li>
<li><p>TMPRSS2 upregulated and ACE2 downregulated upon rIL13 treatment in muco-ciliary air-liquid interface (ALI) cell culture; supporting transcriptome analyses. Also, scRNAseq data from tracheal airway epithelial cultures chronically stimulated with IL-13 also showed congruent changes in TMPRSS2 and ACE2.</p></li>
<li><p>Used metagenomic analysis from dataset to find 18 asymptomatic children with four different coronaviral sequences (CoV; OC43, JKU1, 229E, NL63). Compared 11 subjects with highest CoV infection to 571 CoV-naïve subjects, with 37 rhinovirus (HRV)-infected subjects serving as ‘basal/normal respiratory virus’ comparators.</p></li>
<li><p>Induction of cytotoxic immune response was considerably higher in CoV-infected subjects, compared to HRV-infected individuals. IL10, IL1B, IFNG, IFNA2, STAT1, and ACE2 upregulation shared for CoV or HRV infections. IL6 upregulated and genes associated with CD8 T cells, DCs and NK cells specifically enriched in CoV infections.</p></li>
<li><p>Identified eQTLs for ACE2 and TMPRSS2 using GALA II dataset and used multi-variable modeling to estimate the relative contribution of these factors to population variation in these genes (not reviewed here).</p></li>
</ul>
</div>
<div id="limitations-94" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-94" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Dataset is large, but having asthmatic young subjects as part of GALA II study might skew WGCNA analyses to enrich for T2 biomarkers and secretory phenotype genes. Will be important to replicate analyses in another independent non-COPD/allergy/asthma dataset and reconcile ACE2+TMPRSS2 correlation with T2 emergence.</p></li>
<li><p>Provides rationale for increased cytotoxic responses in CoVID-19 cases, but no explanation for ACE2 and TMPRSS2 anti-correlative trends in epithelia upon T2 inflammation.</p></li>
<li><p>Will need ALI experiments with IL13 addition in CoV-2 infection models to ascertain changes in ACE2/TMPRSS2 to be consistent with existing patient data for COVID-19.</p></li>
</ul>
</div>
<div id="significance-90" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-90" class="anchor"></a>Significance</h4>
<ul>
<li><p>Study strongly suggests airway epithelial TMPRSS2 expression is highly upregulated upon Type 2 inflammation, specifically by IL13 stimulation of epithelia. Also, ACE2 expression (and therefore viral infectivity) is inextricably linked to interferon response and could sabotage initial inflammation.</p></li>
<li><p>Changes in immune contextures observed in asymptomatic CoV+/HRV+ subjects suggest remodeling of airway epithelia even after viral resolution, which can impact future infections with COVID-19 and outcomes.</p></li>
<li><p>Results showing IL10, IL1B, IL6 enrichment and cytotoxic programs (CD8, NK, DC) in CoV+ subjects supports other concurrent studies showing their importance in cytokine storm and points to broader conserved inflammatory pathways worth targeting in COVID-19.</p></li>
</ul>
</div>
<div id="section" class="section level4">
<h4></h4>
</div>
<div id="credit-93" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-93" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Single-cell atlas of a non-human primate reveals new pathogenic mechanisms of COVID-19</strong></p>
<p>Han L. et al.  <span class="math display">\[@doi: 10.1101/2020.04.10.022103\]</span></p>
</div>
<div id="keywords-99" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-99" class="anchor"></a>Keywords</h4>
<ul>
<li><p>scRNAseq</p></li>
<li><p>ScATACseq</p></li>
<li><p>ACE2</p></li>
<li><p>TMPRSS2</p></li>
<li><p>Non-human primate</p></li>
<li><p>interleukin</p></li>
</ul>
</div>
<div id="main-findings-95" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-95" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study profiled nine tissues from cynomolgus monkey (NHP model with no COVID19 infection) by scRNAseq and analyzed ACE2 and TMPRSS2 expression findings to existing human scRNA datasets. Also conducted scATACseq on kidney tissue specifically to map chromatin accessibility relevant to above genes.</p></li>
<li><p>ACE2 and TMPRSS2 predominantly detected in ciliated, club cells and type2 alveolar cells, as well as proximal tubule cells of kidney, and cholangiocytes in liver (agreeing with human data). Some differences exist in ACE2 and TMPRSS2 expression between human and NHP samples, especially in liver but also in lung.</p></li>
<li><p>Correlation analyses show immune-modulatory genes such as TMEM27, IDO2, DNAJC12, and ANPEP co-expressed with ACE2 upregulation in kidney. Also, IL6R gene expression correlated well with ACE2 in proximal tubule cells (agreeing with human data).</p></li>
<li><p>scATACseq of kidney cells determined open chromatin regions with discrete ACE2 peaks in proximal tubule cells S3, and TF motifs for these regions were enriched in STAT1/STAT3 and IRF1 binding sites.</p></li>
</ul>
</div>
<div id="limitations-95" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-95" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Variations in ACE2/TMPRSS2 gene expression between NHP and human tissues could arise due to differences in digestion and/or processing protocols hence biasing reads. Will be important to analyze protein expression for relevant genes in NHP tissue to confirm such differences.</p></li>
<li><p>Study requires ex vivo co-culture experiments with kidney tubule epithelial cells and COV-2 to ascertain IL6R signaling axis is linked to ACE upregulation and/or impacts downstream alarmin/PAMP release.</p></li>
</ul>
</div>
<div id="significance-91" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-91" class="anchor"></a>Significance</h4>
<ul>
<li><h4 id="single-cell-nhp-profiling-is-relevant-and-useful-considering-monkeys-are-a-preferred-model-for-studying-the-effectiveness-of-drug-treatments-and-of-vaccines-against-covid-19">Single-cell NHP profiling is relevant and useful considering monkeys are a preferred model for studying the effectiveness of drug treatments and of vaccines against COVID-19</h4></li>
<li><h4 id="scatacseq-results-with-irf1stat1-accessibility-suggests-a-link-between-paracrine-interferon-signaling-il6-and-enhanced-ace2-expression-in-kidney-that-can-exacerbate-covid-19-severity-due-to-increased-viral-entry-and-dissemination.">scATACseq results with IRF1/STAT1 accessibility suggests a link between paracrine interferon signaling + IL6 and enhanced ACE2 expression in kidney that can exacerbate COVID-19 severity due to increased viral entry and dissemination.</h4></li>
<li><h4 id="findings-supports-other-reports-of-cytokine-induced-tissue-damage-in-kidneys-of-covid-19-patients-as-well-as-provide-additional-support-for-anti-il6r-strategies-such-as-tocilizumab.">Findings supports other reports of cytokine-induced tissue damage in kidneys of COVID-19 patients, as well as provide additional support for anti-IL6R strategies such as Tocilizumab.</h4></li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<blockquote>
<p>Validated by: Robert M Samstein</p>
<p>The SARS-CoV-2 receptor-binding domain elicits a potent neutralizing response without antibody-dependent enhancement</p>
<p>Quinlan et al <strong>doi:</strong> <a href="https://doi.org/10.1101/2020.04.10.036418" class="uri">https://doi.org/10.1101/2020.04.10.036418</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.04.10.036418v1" class="uri">https://www.biorxiv.org/content/10.1101/2020.04.10.036418v1</a></p>
<p><em>Keywords:</em> antibody-dependent enhancement, ACE2, RBD-based vaccine.</p>
<p><span id="OLE_LINK4" class="anchor"><span id="OLE_LINK5" class="anchor"></span></span><em>Main findings:</em> <strong><em>The author’s data suggest that a receptor-binding domain-based (RBD) vaccine for SARS-Cov2 could be safe and effective</em></strong>. Similar to SARS-CoV-1, SARS-CoV-2 requires expression of the cellular receptor angiotensin-converting enzyme 2 (ACE2) to infect cells. The spike (S) protein mediates entry of CoV-2 engages ACE2 through its receptor-binding domain (RBD), an independently folded 197-amino acid fragment of the 1273-amino acid S-protein promoter.</p>
<p>The authors immunized mice using RBD fusion protein, mixed with adjuvant. Blood were collected at day 0(before injection), day 5 and day10 and analyzed neutralization of viral entry. The data suggested that antibodies to the RBD domain of SARS-CoV-1 potently neutralize SARS-CoV-1 S-protein-mediated entry in rat, and the presence of anti-RBD antibodies correlates with neutralization in SARS-CoV-2 convalescent sera.</p>
<p>Antibody dependent enhancement (ADE) can contribute to the pathogenicity of viral infection a major concern observed with other viruses but not with SARS-Cov-2 to date<em>.</em> Importantly, <strong>the RBD-elicited antibodies may mediate ADE less efficiently than those recognizing other neutralizing S-protein epitopes</strong>. Furthermore, anti-RBD anti-serum did not promote ADE at serum dilutions and under a condition in which is the <strong>ZIKV ADE could be observed.</strong></p>
<p>Relevance: The study highlights a potential strategy for SARS-CoV-2 vaccine, using RBD protein to promote neutralization and protection. The work demonstrated limited evidence of ADE, that has been shown to contribute to pathogenesis in zika and dengue.</p>
<p>Limitation: Data show that the RBD alone is sufficient to elicit a potent neutralizing antibody response. It remains to be determined whether trimer-based vaccines, or trimer/RBD combinations, can be more effective, and whether they carry additional safety concerns.</p>
<p><span id="OLE_LINK1" class="anchor"><span id="OLE_LINK2" class="anchor"></span></span><strong>Credit:</strong></p>
<p><em>Reviewed by Zafar Mahmood as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</blockquote>
<p><strong>Title: </strong></p>
<p><strong>Homologous protein domains in SARS-CoV-2 and measles, mumps and rubella viruses:</strong></p>
<p><strong>preliminary evidence that MMR vaccine might provide protection against COVID-19</strong></p>
<p><strong>Immunology keywords: SARS-CoV-2, COVID-19, MMR vaccine, protection </strong></p>
<p><strong>The main findings of the article: </strong></p>
<p>This work aimed to determine whether measles, mumps and rubella (MMR) vaccination might provide protection against COVID-19. The authors examined: 1) sequence homologies between SARS-CoV2 and measles, mumps and rubella viruses; 2) correlations between MMR vaccination coverage, rubella antibody titers, and COVID-19 case fatality in European countries.</p>
<p>Sequences of measles, mumps and rubella virus, which are component of MMR vaccine, were aligned to SARS-CoV-2 to identify homologous domains at the amino acid level. The Macro domain of rubella virus p150, a protease) aligned with SARS-CoV-2 Macro domain of non-structural protein 3 (NSP3), also a protease, at 29 % amino acids identity, suggest similarity in protein folding. Residues conserved in both strains include surface-expressed residues and residues required for ADP-ribose binding, and ADP-ribose 1” phosphatase (ADRP) enzymatic activity. Although the Macro domains are within a cytoplasmic non-structural protein, the authors speculate that they could contribute to vaccine antigenicity if released upon cell lysis. Measles and mumps, both paramyxoviruses, showed structural homology between their F proteins and SARS-CoV-2 spike protein. Both F proteins and spike proteins are responsible for fusion of viral and cellular membrane. The sequence identity was 20 % over a 369-amino acid region and surface-exposed residues were well conserved.</p>
<p>The examination of historic vaccination schedules or recommendations for MMR vaccination in Italy, Spain and Germany revealed that populations who are currently in the age group of 40-49 years old in Germany, 30-39 years old in Spain, and 20-29 years old in Italy were vaccinated. However, the rubella vaccine was introduced for pre-adolescent girls and campaigns for women in child bearing age were conducted early 1970s to 90s in each countries. The latter might cover the women who are currently in the age group of 59-69 years old. If MMR is indeed protected for COVID-19 fatality, the above analysis would suggest that older populations and males are both more likely to die from Covid-19, and less likely to be seropositive for rubella specific immunity.</p>
<p>On the other hand, analysis of anti-rubella IgG titers in moderate and severe COVID-19 patients showed increased levels of rubella IgG in severe patients. To argue that the increase in rubella antibodies in severe COVID-19 was not due to a generalized increased antibody response, the authors mentioned that there was no increase in varicella zoster virus antibody titers in a small subset of patients analyzed. While increase of anti-rubella IgM was not clearly observed in both severe and mild patients, anti-rubella IgG antibody titers were increased in patients who had been admitted for a period of less than 7 days. The authors suggest that IgG titers trend with disease burden on the basis of the shared homology between SARS-CoV2 and rubella virus.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This study demonstrated shared homologies between SARS-CoV2 and MMR viruses that could support the hypothesis that previous MMR vaccination protects against fatality in COVID-19 patients. The authors suggested that older populations and males were less likely to be seropositive for rubella and this might be related to their higher mortality rate. On the other hand, they found that anti-rubella IgG was higher in severe patients than mild patients with COVID-19. Since there was no information about the demographics of severe and mild patients, especially the percentage of male patients and average age to analyze the relationship between severity and MMR vaccination history, the data appears inconclusive. Because of the homology in spike protein of SARS-CoV-2 and F protein of paramyxoviruses, which are important for virus entry in the host cells, measuring cross-reactive anti-measles and anti-mumps antibody titers may provide more information on whether MMR vaccination has the potential to protect against COVID-19.</p>
<p><strong>The importance and implications for the current epidemics</strong></p>
<p>The homology search for conserved domains among different virus strains and vaccine antigens may provide helpful information to develop vaccine antigens that elicit cross-reactive immunity to several viruses. While it is not clear at present if MMR vaccination reduces or not the severity of COVID-19, given the high coverage of MMR vaccination and the potential for vaccines to modulate innate immunity, this question deserves further investigation.</p>
<p><strong>Presence of SARS-CoV-2 reactive T cells in COVID-19 patients and healthy donors</strong></p>
<p>Braun J et al.; medRxiv  2020.04.17.20061440; <a href="https://doi.org/10.1101/2020.04.17.20061440" class="uri">https://doi.org/10.1101/2020.04.17.20061440</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 specific CD4 T cells</p></li>
<li><p>Human endemic coronaviruses</p></li>
<li><p>COVID-19</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, Braun et al. report quantification of virus-specific CD4 T cells in 18 patients with mild, severe and critical COVID-19, including 10 patients admitted to ICU. Performing <em>in vitro</em> stimulation of PBMCs with two sets of overlapping SARS-CoV-2 peptide pools – the S I pool spanning the N-terminal region (aa 1-643) of the S protein, including 21 predicted SARS-CoV-1 MHC-II epitopes, and the C-terminal S II pool (aa 633-1273) containing 13 predicted SARS-CoV-1 MHC-II epitopes – the authors detected S-protein-specific CD4 T cells in up to 83% of COVID-19 patients based on intracellular 4-1BB (CD137) and CD40L (CD154) induction. Notably, peptide pool S II shares higher homology with human endemic coronaviruses (hCoVs) 229E, NL63, OC43, and HKU1 that may cause the common cold, but it does not include the SARS-CoV-2 receptor-binding domain (RBD), which has been identified as a critical target of neutralizing antibodies in both SARS-CoV-1 and SARS-CoV-2. S I-reactive CD4 T cells were found in 12 out of 18 (67%) patients, whereas CD4 T cells against S II were detected in 15 patients (83%). Intriguingly, S-specific CD4 T cells could also be found in 34% (n=23) of 68 SARS-CoV-2 seronegative donors, referred to as reactive healthy donors (RHD), with a preference for S II over S I epitopes. Only 6 of 23 RHDs also had detectable frequencies of S I-specific CD4 T cells, overall suggesting S II-reactive CD4 T cells had likely developed in response to prior infections with hCoVs. Of 18 out of 68 total healthy donors tested, all were found to have anti-hCoV antibodies, although this was independent of concomitant anti-S II CD4 T cell frequencies detected. This finding mirrors observations of declining numbers of specific CD4 T cells, but persistent humoral memory after certain vaccinations such as against yellow fever. The authors further speculate that these pre-existing virus-specific T cells against hCoVs might be one of the reasons why children and younger patients, usually considered to have a higher incidence of hCoV infections per year, are seemingly better protected against SARS-CoV-2. Unlike specific CD4 T cells found in RHDs, most S-specific CD4 T cells in COVID-19 patients displayed a phenotype of recent <em>in vivo</em> activation with co-expression of HLA-DR and CD38, as well as variable expression of Ki-67. In addition, a substantial fraction of peripherally found HLA-DR<sup>+</sup>/CD38 bulk CD4 T cells was found to be refractory to peptide stimulation, potentially indicating cellular exhaustion.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This is one of the first preprints reporting the detection of virus-specific CD4 T cells in COVID-19 (also cf. Dong et al., <a href="https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1</em></a>; Weiskopf et al., <a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info"><em>https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info</em></a>). While it generally adds to our current knowledge about the potential role of T cells in response to SARS-CoV-2, a few limitations, some of which are discussed by the authors themselves, should be addressed. Findings in this study pertain to a relatively small cohort of patients of variable clinical disease. To corroborate the observations made here, larger studies including greater numbers of both healthy donors and patients of all clinical stages are needed to better assess the function of virus-specific CD4 T cells in COVID-19. Specifically, the presence of pre-existing, potentially hCoV-cross-reactive CD4 T cells in healthy donors needs to be explored in the context of COVID-19 immunopathogenesis. While the authors suggest a potentially protective role based on higher incidence of hCoV infection in children and younger patients, and therefore a presumably larger pool of pre-existing virus-specific memory T cells, the opposite could also be the case given cumulatively increased number of hCoV infections in older patients. In this context, it would therefore have been interesting to also measure anti-hCoV antibodies in COVID-19 patients. Furthermore, this study did not quantify virus-specific CD8 T cells. Based on observations in SARS-CoV-1, virus-specific memory CD8 T cells are more likely to persist long-term and confer protection than CD4 T cells, which were detected only at lower frequencies six years post recovery from SARS-CoV-1 (<em>cf</em>. []{#_ENREF_18 .anchor}Li CK et al., Journal of immunology <em>181</em>, 5490-5500.) Morover, no other specifities such as against the N or M epitopes were evaluated. Robust generation of virus-specific T cells against the N protein was shown to be induced by SARS-CoV-2 in another pre-print by Dong et al. (Dong et al., <a href="https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1"><em>https://www.medrxiv.org/content/10.1101/2020.03.17.20036640v1</em></a>), while Weiskopf et al. recently reported preference of both CD8 and CD4 T cells for S epitopes <a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info"><em>https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.article-info</em></a>). Moreover, the authors seem to suggest that some of the virus-specific CD4 T cells detected could be potentially cross-reactive to predicted SARS-CoV-1 epitopes present in the peptide pools used. Indeed, this has been recently established for several SARS-CoV-2 binding antibodies, while it was found not to be the case for RBD-targeting neutralizing antibodies (<em>cf.</em> Wu et al., <a href="https://www.medrxiv.org/content/10.1101/2020.03.30.20047365v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.30.20047365v2</em></a>.; Ju et al., <a href="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2"><em>https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2</em></a>). A similar observation has not been made for T cells so far and should be evaluated. Finally, since reactive healthy donors were only tested for anti-S1 IgG, however not for other more ubiquitous binding antibodies, e.g. against M, and only a fraction of these donors was additionally confirmed to be negative by PCR, though unlikely, there is the possibility that some of the seronegative reactive donors were previously exposed to SARS-CoV-2.</p>
<p><strong><em>Significance</em></strong></p>
<p>Quantification of virus-specific T cells in peripheral blood is a useful tool to determine the cellular immune response to SARS-CoV-2 both in acute disease and even more so post recovery. Ideally, once immunogenic T cell epitopes are better characterized, tetramer assays will allow for faster and more efficient detection of their frequencies. Moreover, assessing the potential role of pre-existing virus-specific CD4 T cells in healthy donors in the context of COVID-19 pathogenesis will be of particular importance. The observations made here are also highly relevant for the design and development of potential vaccines and should therefore be further explored in ongoing research on potential coronavirus therapies and prevention strategies.</p>
<p><em>This review was undertaken by Verena van der Heide as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em> </p>
<p> </p>
<p>DOI: 10.1101/2020.04.11.20062349</p>
<p>Title: Phenotype of SARS-CoV-2-specific T-cells in COVID-19 patients with acute respiratory distress syndrome</p>
<p>Keywords: SARS-CoV-2 specific T cells; Acute Respiratory Distress Syndrome; Th1 response</p>
<p>Summary: Using peripheral blood samples collected from 8 COVID-19 patients with moderate to severe acute respiratory distress syndrome (ARDS), the authors showed that COVID-19 patients exhibit lower CD3<sup>+</sup> T cell counts as well as increased CD4:CD8 ratio. In addition to population analysis, PBMCs were stimulated with three pools of either overlapping peptides of SARS-CoV-2 spike (S) protein or HLA Class II or I predicted epitopes covering all viral proteins except S designed to activate CD4 and CD8 T cells, respectively. While stimulation of PBMCs with all three peptide pools led to detection and activation of SARS-CoV-2 specific CD4 and CD8 T cells, spike (S) peptide pool elicited the strongest response, indicating that the S surface glycoprotein is a strong inducer of both CD4 and CD8 T-cell responses. Phenotyping of activated T cells (CD4<sup>+</sup>CD69<sup>+</sup>CD137<sup>+</sup> or CD8<sup>+</sup>CD69<sup>+</sup>CD137<sup>+</sup>) showed that the majority of activated CD4 T cells were central memory T-cells (CD45RA<sup>-</sup> and CCR7<sup>+</sup>) while activated CD8 T cells were mostly effector memory T cells (CCR7<sup>-</sup>) and terminally differentiated effector T cells. Cytokine analysis of cell culture supernatants from PBMCs stimulated by S-peptide pool led to a strong production of Th1 cytokines IFNγ, TNF⍺ and IL-2. Lastly, the authors profiled the kinetics of both humoral and cell-mediated response in four different time points using ELISA of virus-specific serum IgG and quantifying expression of cell surface markers induced by S peptide pool activation. Throughout the patients’ stay at the ICU, both levels of virus-specific IgG antibodies and frequencies of virus-specific CD4 cells increased significantly over time.</p>
<p>Limitations: A couple of additional assays done in parallel could have further strengthened the paper’s findings. Simultaneous profiling of cytokines from PBMCs could have easily answered whether these T cells which are capable of producing Th1 cytokines upon activation are indeed producing them in patients. Furthermore, it would have been informative to have added a couple of functional exhaustion markers (i.e. PD-1, Tim-3, etc.) and compare their expression between pre- and post-activation by S peptide pool—thereby addressing the effect of functional exhaustion in T cells reported in severe COVID-19 patients. Follow-up studies investigating which epitopes out of the S protein peptide pool elicited the most potent T-cell response would have yielded informative results for possible vaccine design efforts against the spike protein. Lastly, comparing the quality of virus specific T cells immunity between patients in ICU (the focus of the study) and patients with mild /moderate disease would have been very informative.</p>
<p>Significance of the finding: For the most part, the study design has been well established to answer the following questions: a) are there T cells reactive to spike (and other HLA-reactive) protein of SARS-CoV2 even in cases of COVID-19 with moderate to severe ARDS (which has been characterized with lymphopenia)? b) what are the phenotype and cytokine profile of CD4 and CD8 T cells upon activation? Answering these questions do advance the field’s understanding of T cell response to COVID-19 and add to much-needed effort to devise a vaccine against SARS-CoV2. Building on this article’s findings, perhaps future studies could perform mechanistic assays the function of T cells from COVID-19 patients in the context of systemic inflammation (i.e. adding IL-1, IL-6, TNF⍺ in the culture media) as well as correlating epitope-specific immune response of patients with their clinical severity.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the<br>
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p>DOI: 10.1101/2020.04.15.20065623</p>
<p>Title: Neutralizing Antibodies Responses to SARS-CoV-2 in COVID-19 Inpatients and Convalescent Patients</p>
<p>Keywords: Neutralizing Antibodies; Convalescent plasma</p>
<p>Summary: Based on peripheral blood samples from 117 COVID-19 inpatients and convalescent patients, the authors demonstrate that all patients sampled became seropositive with neutralizing antibodies within 20 days since onset of symptoms and stayed seropositive until day 41-53. Seropositivity of neutralizing antibodies was defined as a geometric mean titer (GMT) of 1:4 higher and the titer was calculated using a modified <em>in vitro</em> cytopathogenic assay where the dilution number of 50% protective condition from cytopathic effect of the virus represented the titer. The GMT of neutralizing antibodies (average: 1:271.2) was the highest at 31-40 days since onset and multivariate Generalized Estimating Equations (GEE) model controlling for clinical variables (i.e. gender, age, clinical severity, etc.) showed that antibody titer at 31-40 days was significantly higher than 10-20 days past onset. In addition, their multivariate GEE analysis showed that age- and clinical severity-dependent rise in antibody titers with the youngest (age 16-30) and patients with mild or asymptomatic conditions having a lower antibody tier than its elderly and moderately-sick counterparts.</p>
<p>Limitations: Several shortcomings limit the impact of this study. While it has been the intent of the authors to sample PBMCs from patients at various time points in order to establish a robust profile of antibody response against SARS-CoV2, in reality, sampling has been limited and inconsistent across different time points. For instance, PBMCs of only 12 out of 117 patients have been collected three or more times and it is not clear from the data whether samples from patients whose blood has been collected only once (n = 37) are evenly distributed across the time frames under analysis. Furthermore, the authors have tried to show differences in kinetics of antibody response between patients with mild and moderate conditions by sampling their blood at four different time points. However, not only do two of eight patients sampled in this study have only two data points, but also the authors have found that the antibody response varies considerably across individuals—further underscoring the need to have PBMCs sampled from each patient at multiple time points and normalizing their response before comparing the titers across individuals. In addition, due to the fact that patients were enrolled using convenient sampling instead of random sampling methods, it’s evident that the authors could not control for disease severity as they only had four patients in severe condition. Beyond the sampling issues, the modified cytopathic assay used to calculate the neutralizing antibody titers may be less sensitive and specific than ELISA-based assays that use purified antigens from the virus.</p>
<p>Significance of the finding: Limited. While it is informative to have descriptive studies like this one showing the dynamics of the antibody response against COVID-19, the failure of the study to collect samples in controlled manner prevents the reader from using the data to answer key questions regarding the humoral immune response against COVID-19: do differences in clinical severity manifest in different kinetics of antibody response? When controlled for age, is higher antibody titer predictive of their clinical severity and prognosis? Future studies may address those questions with more controlled experimental setup.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the<br>
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
<p><strong>Heparin therapy improving hypoxia in COVID-19 patients - a case series</strong></p>
<p> </p>
<p>Negri et al. medRxiv preprint doi: <a href="https://" class="uri">https://</a> <a href="https://doi.org/10.1101/2020.04.15.20067017">doi.org/10.1101/2020.04.15.20067017</a></p>
<p><em>Keywords </em></p>
<ul>
<li><p>Heparin </p></li>
<li><p>COVID-19</p></li>
</ul>
<!-- --><ul>
<li><p>Hypoxia</p></li>
<li><p>Acute Respiratory Distress Syndrome</p></li>
<li><p>Critical Care</p></li>
</ul>
<p><em>Main Findings</em> </p>
<p>The authors describe a small cohort of 27 COVID-19 patients treated with enoxaparin or heparin in escalating doses (corresponding to clinical severity) at Sirio-Libanes Hospital in Sao Paolo, Brazil. Importantly, no control patients are included in this study. Additionally, all patients received concomitant azithromycin and a subset received methylprednisolone therapy. Patients had mean WHO score of 4.0 ± 1.2 (corresponding to moderate clinical severity) upon admittance. PaO<sub>2</sub>/FIO<sub>2</sub> was significantly increased from 254(±90) to 325(±80) after 72 hours after the initiation of heparin therapy. 56% of patients were discharged within 7.3 (±4.0) days. 50% of mechanically ventilated patients were extubated within 10.3 (±1.5) days. The study reported no fatal events or bleeding complications due to anticoagulation. The authors suggest that early heparin therapy significantly improves hypoxemia and may be beneficial in the management of such patients.</p>
<p><em>Limitations</em> </p>
<p>This is a small, single arm, retrospective study without controls and with concomitant confounding treatments. Therefore, no definitive conclusions can be made here.</p>
<p><em>Significance</em> </p>
<p>This article adds anecdotal evidence regarding coagulability in COVID-19 patients and points to the potential for anticoagulation in the right clinical study. Given the multiple limitation, evidence herein can only corroborate previous reports demonstrating associations between elevated D-dimer and disease severity <span class="math display">\[1-3\]</span>. Additionally, this study may add to the evidence regarding mortality benefits of heparin therapy in severe COVID-19 <span class="math display">\[2, 3\]</span>.</p>
<p><em>Credit</em> </p>
<p><em>Reviewed by Joan Shang as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em> </p>
<p><em>References</em></p>
<p>1. Han H, Yang L, Liu R, et al. Prominent changes in blood coagulation of patients with SARS- CoV-2 infection. Clin Chem Lab Med 2020 doi: 10.1515/cclm-2020-0188 <span class="math display">\[published Online First: 2020/03/17\]</span></p>
<p>2. Shi, C., Wang, C., Wang, H., et al. The potential of low molecular weight heparin to mitigate cytokine storm in severe COVID-19 patients: a retrospective clinical study. medRxiv. (2020). Doi: <a href="https://doi.org/10.1101/2020.03.28.20046144" class="uri">https://doi.org/10.1101/2020.03.28.20046144</a></p>
<p>3. Tang N, Bai H, Chen X, et al. Anticoagulant treatment is associated with decreased mortality in severe coronavirus disease 2019 patients with coagulopathy. J Thromb Haemost 2020 doi: 10.1111/jth.14817 <span class="math display">\[published Online First: 2020/03/29\]</span></p>
<p>DOI: 10.1101/2020.04.16.045690</p>
<p>Title: The scRNA-seq expression profiling of the receptor ACE2 and the cellular protease TMPRSS2 reveals human organs susceptible to COVID-19 infection</p>
<p>Keywords: ACE2, TMPRSS2, scRNA-seq expression profiling</p>
<p>Summary: Using publicly available scRNA-seq data of healthy human samples from 31 organs, the authors compare the gene expression levels of ACE2 and TMPRSS2 of each organ. The authors categorized each organ as susceptible to SARS-CoV infection based on its expression of ACE2 and further stratified 11 susceptible organs into three levels of risk for infection based on each organ’s TMPRSS2 expression (i.e. TMPRSS2 expression ratio &gt;20% was defined as level 1). The authors claimed that for the first time, their scRNA-seq analysis showed the brain, gall bladder and fallopian tube as vulnerable to COVID-19 infection in addition to confirming previous molecular and clinical data implicating other organs susceptible to SARS-CoV2 (i.e. nose, heart, intestines, etc.).</p>
<p>Limitations: The article advances its risk stratification strategy based on a couple of naïve assumptions that are being actively contested in literature: a) ACE2/TMPRSS2-mediated viral entry is the only route used by SARS-CoV2 to infect cells and b) ACE2/TMPRSS2 expression is stable in systemic inflammatory contexts such as COVID-19. Furthermore, their risk stratification does not consider the immune contexture of each organ. For instance, while several papers have suggested that reproductive organs (i.e. testes) are also susceptible to SARS-CoV2 based on their ACE2/TMPRSS2 expression, there are no reported clinical manifestations of COVID-19 in those organs. Lastly, even though they were using scRNA-seq data available from 13 different publications, their paper does not discuss how they accounted for variations in scRNA-seq protocols as well as how they normalized each data set for comparative analysis.</p>
<p>Significance of the finding: Mostly confirmatory. While it is nice to see that there were clinical cases describing the adverse effect of COVID-19 for most of the eleven organs identified as SARS-CoV2 susceptible in this study, previous studies have already shown most of these organs to be susceptible to SARS-CoV2 using various approaches (including scRNA-seq analysis). Furthermore, their risk stratification strategy is simplistic as it doesn’t seem to take account of recent findings on SARS-CoV2’s alternative mechanisms of entry as well as reported clinical manifestations of COVID-19.</p>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the<br>
Immunology Institute of the Icahn school of medicine, Mount Sinai.</em></p>
</div>
</div>
</div>
<div id="title-rapid-development-of-an-inactivated-vaccine-for-sars-cov-2" class="section level1">
<h1 class="hasAnchor">
<a href="#title-rapid-development-of-an-inactivated-vaccine-for-sars-cov-2" class="anchor"></a>Title Rapid development of an inactivated vaccine for SARS-CoV-2</h1>
<p>Gao et al. bioRxiv <span class="math display">\[@doi: 10.1101/2020.04.17.046375\]</span></p>
<div id="keywords-100" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-100" class="anchor"></a>Keywords</h4>
<ul>
<li><p>Inactivated vaccine</p></li>
<li><p>Neutralizing antibodies</p></li>
<li><p>Non-human primates</p></li>
</ul>
</div>
<div id="main-findings-96" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-96" class="anchor"></a>Main Findings</h4>
<p>This study describes the development and pre-clinical tests on mice, rats and rhesus macaques of a SARS-CoV-2 inactivated vaccine candidate. Vaccine preparation (PiCoVacc) was generated from a viral strain (CN2) isolated from a COVID-19 patient and cultured on Vero cells during 10 passages, before being inactivated with Beta-propiolactone, an alkylating agent that damages nucleic acids and inhibits viral membrane fusion.</p>
<p>Balb/c mice and Wistar rats were injected at d0 and d7 with PiCoVacc (0, 1.5 or 3 or 6 ug per dose) and alum adjuvant. SARS-CoV-2 S and RBD specific IgG were detected after 1 week and continued to increase until the end of the study (week 6). Micro-neutralization assays showed a progressive augmentation of neutralizing antibodies (nAbs) against the strain used to make the PiCoVacc from week 1 to week 6. These antibodies (from week 3) also neutralized 9 others SARS-CoV-2 strains.</p>
<p>Next, rhesus macaques (n=4 per group) were immunized intramuscularly three times at d0, d7 and d14 with 0 (sham), 3ug (medium dose) or 6ug (high dose) of PiCoVacc with alum adjuvant, or only physiological saline. S-specific IgG and nAbs were measured at week 2 in both vaccinated groups. No fever or weight loss was observed after immunization. Biochemical blood tests, lymphocytes subset percent and cytokines showed no particular changed. Histopathological analysis of several organs (kidney, brain, liver, heart, spleen) did not show any changes either.</p>
<p>Macaques were then challenged by intra-tracheal inoculation of 10<sup>6</sup> TCID<sub>50</sub> viral preparation of a different strain through the intra-tracheal route, 8 days after the third vaccine inoculation. All non-vaccinated controls showed important amount of viral RNA in pharynx, crissum and lung until day 7 post inoculation (dpi) and had severe interstitial pneumonia, whereas vaccinated macaques showed a quick decrease in throat viral loads and small histopathological changes in the lungs. Macaque vaccinated with the high dose had undetectable viral loads in lungs and anal swab at any point of the study. No Antibody Dependent Enhancement (ADE) was observed.</p>
</div>
<div id="limitations-96" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-96" class="anchor"></a>Limitations</h4>
<p>The macaques were challenge at the peak of the antibody response induced by the vaccine and no data was collected to assess the length of vaccine protection. Furthermore, the macaques were euthanized as early as seven days after the virus challenge and longer time of analysis will be required to evaluate potential immune-pathological effects of the vaccination and the dynamics of the infection in the different groups. Finally, the size of the study is very limited with 4 animals vaccinated per group.</p>
<p>While SARS-CoV-2 appears to be relatively stable, it would be interesting to test cross reactivity and protection with heterologous viruses, to confirm that the nAb induced by the vaccine can neutralize many other SARS-CoV-2 strains, as it was performed in the mice and rat studies. Indeed, although two different strains were used for the immunization and the challenge in rhesus macaques, the S protein sequences of those strains are identical, and the Genbank accession number to compare the rest of the sequence was not made available yet.</p>
<p>Furthermore, as most of COVID-19 severe patients are of advanced age, testing if this inactivated vaccine can also elicit antibody response in aged macaques would be of great interest.</p>
<p>The timing of vaccination and challenge is not clear with contradictory timelines indicated in the figures and main text. The authors do not describe the inactivation process of the virus enough, nor show any evidence of inactivation. More description and evidence of the inactivation (by showing that the virus does not grow in Vero cells nor in animals) is required to assess efficacy of vaccination</p>
<p><em>Finally, the resolution of the figures is really low and makes it hard to analyze</em></p>
</div>
<div id="section-1" class="section level4">
<h4></h4>
</div>
<div id="significance-92" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-92" class="anchor"></a>Significance</h4>
<p>Development of a vaccine against SARS-CoV-2 is strongly needed as no effective treatment has been found yet. Among all the vaccine clinical trials already started<sup>2</sup>, this pre-clinical study shows that this inactivated vaccine could represent a good candidate for further development as it protected rhesus macaques from viral challenge without immediate side-effects. Although more pre-clinical data seem to be needed before starting human trials, this is the first study to our knowledge to show effectiveness in non-human primates. The biopharmaceutical firm, Sinovac, has registered phase I and II clinical trials <sup>1</sup>. Moreover, the inactivation technique for vaccine development is well known and can be adapted for production in other facilities.</p>
</div>
<div id="section-2" class="section level4">
<h4></h4>
</div>
<div id="credit-94" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-94" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Emma Risson as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>1. Safety and Immunogenicity Study of 2019-nCoV Vaccine (Inactivated) for Prophylaxis SARS CoV-2 Infection (COVID-19) - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04352608" class="uri">https://clinicaltrials.gov/ct2/show/NCT04352608</a>. Accessed April 22, 2020.</p>
<p>2. <a href="http://paperpile.com/b/F9bKuX/LCEi">World Health Organization (2020). Draft of the landscape of COVID-19 candidate vaccines. World Health Organization 4–4.</a></p>
<p><strong>COVID-19 Critical Illness Pathophysiology Driven by Diffuse Pulmonary Thrombi and Pulmonary Endothelial Dysfunction Responsive to Thrombolysis</strong></p>
<p>Poor et al., medRxiv <span class="math display">\[10.1101/2020.04.17.20057125\]</span></p>
<p>Keywords:</p>
<ul>
<li><p>COVID-19</p></li>
<li><p>pneumonia</p></li>
<li><p>thrombolysis</p></li>
<li><p>endothelium</p></li>
<li><p>platelet</p></li>
<li><p>D-dimer</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report describes the use of systemic tissue plasminogen activator (tPA) to treat venous thromboembolism (VTE) seen in four critically ill COVID-19 patients with respiratory failure. These patients all exhibited gas exchange abnormalities, including shunt and dead-space ventilation, despite well-preserved lung mechanics. A pulmonary vascular etiology was suspected.</p>
<p>All four patients had elevated D-dimers and significant dead-space ventilation. All patients were also obese, and 3/4 patients were diabetic.</p>
<p>Not all patients exhibited an improvement in gas exchange or hemodynamics during the infusion, but some did demonstrate improvements in oxygenation after treatment. Two patients no longer required vasopressors or could be weaned off them, while one patient became hypoxemic and hypotensive and subsequently expired due to a cardiac arrest. Echocardiogram showed large biventricular thrombi.</p>
<p><em>Limitations</em></p>
<p>In addition to the small sample size, all patients presented with chronic conditions that are conducive to an inflammatory state. It is unclear how this would have impacted the tPA therapy, but it is likely not representative of all patients who present with COVID-19-induced pneumonia. Moreover, each patient had received a different course of therapy prior to receiving the tPA infusion. One patient received hydroxychloroquine and ceftriaxone prior to tPA infusion, two patients required external ventilator support, and another patient received concurrent convalescent plasma therapy as part of a clinical trial. Each patient received an infusion of tPA at 2 mg/hour but for variable durations of time. One patient received an initial 50 mg infusion of tPA over two hours. 3/4 patients were also given norepinephrine to manage persistent, hypotensive shock. Of note, each patient was at a different stage of the disease; One patient showed cardiac abnormalities and no clots in transit on an echocardiogram, prior to tPA infusion.</p>
<p><em>Significance</em></p>
<p>The study describes emphasizes the importance of coagulopathies in COVID-19 and describes clinical outcomes for four severe, COVID-19 patients, who received tPA infusions to manage poor gas exchange. While the sample size is very limited and mixed benefits were observed, thrombolysis seems to warrant further investigation as a therapeutic for COVID-19-associated pneumonia that is characterized by D-dimer elevation and dead-space ventilation. All four patients had normal platelet levels, which may suggest that extrinsic triggers of the coagulation cascade are involved.</p>
<p>The authors suspect that endothelial dysfunction and injury contribute to the formation of pulmonary microthrombi, and these impair gas exchange. Pulmonary thrombus formation has also been reported by other groups; post-mortem analyses of 38 COVID-19 patients’ lungs showed diffuse alveolar disease and platelet-fibrin thrombi (Carsana et al., 2020). Inflammatory infiltrates were macrophages in the alveolar lumen and lymphocytes in the interstitial space (Carsana et al., 2020). Endothelial damage in COVID-19 patients has also been directly described, noting the presence of viral elements in the endothelium and inflammatory infiltrates within the intima (Varga et al., 2020). One hypothesis may be that the combination of circulating inflammatory monocytes (previously described to be enriched among PBMCs derived from COVID-19 patients) that express tissue factor, damaged endothelium, and complement elements that are also chemotactic for inflammatory cells may contribute to the overall pro-coagulative state described in COVID-19 patients.</p>
<p><em>References</em></p>
<p>Carsana, L., Sonzogni, A., Nasr, A., Rossi, R.S., Pellegrinelli, A., Zerbi, P., Rech, R., Colombo, R., Antinori, S., Corbellino, M., et al. (2020) Pulmonary post-mortem findings in a large series of COVID-19 cases from Northern Itality. medRxiv. 2020.04.19.20054262.</p>
<p>Varga, Z., Flammer, A.J., Steiger, P., Haberecker, M., Andermatt, R., Zinkernagal, A.S., Mehra, M.R., Schuepbach, R.A., Ruschitzka, F., Moch, H. (2020) Endothelial cell infection and endotheliitis in COVID-19. Lancet. 10.1016/S0140-6736(20)30937-5.</p>
<p><em>The study described in this review was conducted by physicians of the Divisions of Pulmonary, Critical Care, and Sleep Medicine, Cardiology, Nephrology, Surgery, and Neurosurgery and Neurology at the Icahn School of Medicine at Mount Sinai. </em></p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
<div id="title-a-single-cell-atlas-of-the-peripheral-immune-response-to-severe-covid-19" class="section level3">
<h3 class="hasAnchor">
<a href="#title-a-single-cell-atlas-of-the-peripheral-immune-response-to-severe-covid-19" class="anchor"></a>Title: A single-cell atlas of the peripheral immune response to severe COVID-19</h3>
</div>
<div id="wilk-a-j--et-al--medrxiv-doi10-11012020-04-17-20069930" class="section level3">
<h3 class="hasAnchor">
<a href="#wilk-a-j--et-al--medrxiv-doi10-11012020-04-17-20069930" class="anchor"></a>Wilk, A.J. et al. MedRxiv <span class="math display">\[@doi:10.1101/2020.04.17.20069930\]</span>
</h3>
<p><strong>Keywords:</strong> scRNAseq; Interferon-Stimulating Genes (ISGs);Activated granulocytes</p>
<p><strong>Main Findings:</strong></p>
<p>The authors performed single-cell RNA-sequencing (scRNAseq) on peripheral blood from 6 healthy donors and 7 patients, including 4 ventilated and 3 non-ventilated patients. 5 of the patients received Remdesivir.</p>
<p>scRNAseq data reveal 30 gene clusters, distributed among granulocytes, lymphocytes (NK, B, T cells), myeloid cells (dendritic cells DCs, monocytes), platelets and red blood cells. Ventilated patients specifically display cells containing neutrophil granule proteins that appear closer to B cells than to neutrophils in dimensionality reduction analyses. The authors named these cells “Activated Granulocytes” and suggest them to be class-switched B cells that have lost the expression of CD27, CD38 and BCMA and acquired neutrophil-associated genes, based on RNA velocity studies.</p>
<p>SARS-CoV2 infection leads to decreased frequencies of myeloid cells, including plasmacytoid DCs and CD16+ monocytes. CD14+ monocyte frequencies are unchanged in the patients, though their transcriptome reveals an increased activated profile and a downregulation of HLAE, HLAF and class II HLA genes. NK cell transcriptomic signature suggests lower CD56<sup>bright</sup> and CD56<sup>dim</sup> NK cell frequencies in COVID-19 patients. NK cells from patients have increased immune checkpoint (Lag-3, Tim-3) and activation marker transcripts and decreased maturation and cytotoxicity transcripts (CD16, Ksp-37, granulysin). Granulysin transcripts are also decreased in CD8 T cells, yet immune checkpoint transcripts remain unchanged in both CD8 and CD4 T cells upon SARS-CoV2 infection. The frequencies of memory and naïve CD4 and CD8 T cell subsets seem unchanged upon disease, though γδT cell proportions are decreased. SARS-CoV2 infection also induces expansion of IgA and IgG plasmablasts that do not share Ig V genes.</p>
<p>Interferon-signaling genes (ISGs) are upregulated in the monocyte, the NK and the T cell compartment in a donor-dependent manner. ISG transcripts in the monocytes tend to increase with the age, while decreasing with the time to onset disease. No significant cytokine transcripts are expressed by the circulating monocytes and IFNG, TNF, CCL3, CCL4 transcript levels remain unchanged in NK and T cells upon infection.</p>
<p><strong>Limitations:</strong></p>
<p>The sample size of the patients is limited (n=7) and gender-biased, as all of them are men.</p>
<p>The activating and resting signatures in monocytes should be further detailed. The authors did not detect IL1B transcripts in monocytes from the patients, though preliminary studies suggest increased frequencies of CD14+ IL1B+ monocytes in the blood of convalescent COVID-19 patients<span class="math display">\[1\]</span>.</p>
<p>Decreased NK cells, B cells, DCs, CD16+ monocytes and γδT cells observed in peripheral blood might not only reflect a direct SARS-CoV2-induced impairment, but also the migration of these cells to the infected lung, in line with preliminary data suggesting unchanged NK cell frequencies in the patient lungs<span class="math display">\[2\]</span>.</p>
<p>The authors identified platelets in their cluster analyses. Recent reports of pulmonary complications secondary to COVID-19 describe thrombus formation that is probably due, in part, to platelet activation<span class="math display">\[3, 4\]</span>. A targeted characterization of the platelet transcriptome may thus benefit an increased understanding of this phenomenon.</p>
<p>The transcriptome of the Activated Granulocytes should be further detailed. As discussed by the authors, IL24 and EGF might be involved in the generation of the Activated Granulocytes, though these cytokines are poorly represented in the blood of the patients. The generation of these cells should therefore be further investigated in future studies.</p>
<p><strong>Significance:</strong></p>
<p>The authors show a SARS-CoV2-induced NK cell dysregulation, in accordance with previous studies<span class="math display">\[5\]</span>. Alongside the upregulation of ISGs in NK cells, these findings suggest an impaired capacity of the NK cells to respond to activating signals in COVID-19 patients. The unchanged expression of immune checkpoints on CD4 and CD8 T cells suggest distinct SARS-CoV2 dysregulation pathways in the NK and the T cell compartments. In particular, the downregulation of transcripts encoding for class II HLA but not for the HLA-A, -B, -C molecules in monocytes suggest an impaired antigen presentation capacity to CD4 T cells, which should be further investigated.</p>
<p>The authors provide preliminary results suggesting an age-related activation of the monocytes in the COVID-19 patients. Future studies will be needed to evaluate if the age impacts the involvement of the monocytes in the cytokine storm observed in COVID-19 patients.</p>
<div id="references" class="section level4">
<h4 class="hasAnchor">
<a href="#references" class="anchor"></a><strong>References:</strong>
</h4>
<p>1. Wen, W., et al., <em>Immune Cell Profiling of COVID-19 Patients in the Recovery Stage by Single-Cell Sequencing.</em> MedRxiv, 2020.</p>
<p>2. Liao, M., et al., <em>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</em>. 2020.</p>
<p>3. Giannis, D., I.A. Ziogas, and P. Gianni, <em>Coagulation disorders in coronavirus infected patients: COVID-19, SARS-CoV-1, MERS-CoV and lessons from the past.</em> J Clin Virol, 2020. <strong>127</strong>: p. 104362.</p>
<p>4. Dolhnikoff, M., et al., <em>Pathological evidence of pulmonary thrombotic phenomena in severe COVID-19.</em> J Thromb Haemost, 2020.</p>
<p>5. Zheng, M., et al., <em>Functional exhaustion of antiviral lymphocytes in COVID-19 patients.</em> Cell Mol Immunol, 2020.</p>
<p><strong>Credit:</strong></p>
<p><em>Reviewed by Bérengère Salomé and Zafar Mahmood as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>﻿<strong>National Consumption of Antimalarial Drugs and COVID-19 Deaths Dynamics: An Ecological Study</strong></p>
<p>M. Izoulet, <em>medRxiv</em> <span class="math display">\[@doi: 10.1101/2020.04.18.20063875\]</span></p>
<p><span id="keywords" class="anchor"></span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>hydroxychloroquine</p></li>
<li><p>modeling</p></li>
<li><p>ecological fallacy</p></li>
</ul>
<p><span id="main-findings" class="anchor"></span></p>
<p><em>Main Findings</em></p>
<p>This preprint sought to compare the daily deaths in countries using CQ/HCQ as a treatment from the beginning of the COVID-19 pandemic to those that did not. From a list of 60 countries in descending order by number of confirmed cases, 16 countries were selected for inclusion into either the high CQ/HCQ production or use group, versus not. Countries were included if they met the criteria for having data from the day of the 3<sup>rd</sup> death in the entire country and the daily deaths for the 10 days immediately following, until both groups were populated with a list of 16 (Figure 1: Table with the CQ/HCQ group list; Figure 2: Table with the “control” group list). For each group of countries, the average daily deaths were determined, and the curves projected to illustrate trajectories. In Figure 3, the author suggests that the deaths in the countries belonging to the control group follow an exponential curve, while the progression of average daily deaths in the countries with greater use of CQ/HCQ follow a polynomial curve.</p>
<p>The author then applies Auto Regressive Integrated Moving Average (ARIMA), a modeling tool used for time-series forecasting (i.e., predicting the future trajectory of data over time using the data from previous time points as predictors in a linear regression). The Auto Regressive component refers to each difference between two previous time points that make the model “stationary” (current – previous); the Moving Average is the number of forecast errors from calculating these differences that should go in the model. The author uses ARIMA to predict the next 10 days of mean deaths for the CQ/HCQ list (Figure 6) and the control countries (Figure 8). In figures 9 and 10, autocorrelations of residuals are performed to determine internal validity of the model, here defined as no significant autocorrelations.</p>
<p>In conjunction, the author argues that these findings support major differences in death rates between countries that use/mass produce CQ/HCQ versus those that do not.</p>
<p><span id="limitations" class="anchor"></span><em>Limitations</em></p>
<p>The title of this study refers to itself as an ecological study, an observational study in which the data are defined at the population level, rather than individual. Although this study design allows for rapid hypothesis testing in large datasets, a robust ecological study should account for as many known risk-modifying factors or confounders as possible. Subsequently, any results should be reviewed under strict criteria for causality, since there is high probability of the outcomes falling under the definition of ecological fallacy, which occurs when inferences about individuals are determined from inferences about a group to which they belong.</p>
<p>This study conflated the use and mass production of CQ/HCQ at the start of the COVID-19 pandemic in each respective country, with that country’s direct pandemic response. It is never explained whether use or production is the key output for any given country, which are vastly different metrics. The author fails to consider other reasons for having existing infrastructure for the mass production of drugs like hydroxychloroquine, whether the country was a global supplier of the medication (India), or is a region where malaria is endemic (India, Pakistan, Indonesia, Malaysia, South Korea), which may correlate to both chloroquine production and use. Notably, the countries from which studies of HCQ in the treatment of COVID-19 have been predominantly performed (China, France, USA), are all in the control list of countries. Additionally, the data for cases and deaths were collected from reports accessed from <a href="https://www.worldometers.info/coronavirus/" class="uri">https://www.worldometers.info/coronavirus/</a>; data were not selected from the top countries using a methodological approach, but rather skipping certain countries to use only the most complete death data for the timeframe of interest, allowing for bias introduced by the reporting of each individual country.</p>
<p>With regards to the statistical methods applied, namely ARIMA, they are non-standard practices for interpreting the results of an ecological study. The first problem with this, in my opinion, is that the message will be difficult to interpret and criticize for many scientists, as ARIMA will be unfamiliar to most in the biological sciences. Further, the models applied (Table 4) do not take into account any confounders, which is a requirement for robust analysis of an ecological study. There are only 3 variables in this type of model: p, the autoregressive coefficient, q, the moving average coefficient, and d, the difference between points in the time-series. Any flaws or bias inherent to the input data are then upheld and propagated by the model, which does not allow for any other variable that would contribute to the risk of death (say, honest and complete reporting by country).</p>
<p><em>Significance</em></p>
<p>The faults of the stratification of countries into the groups proposed in this study, together with the unorthodox application of ARIMA modeling, make it challenging to accept the conclusion that the author draws in this study. The apparent decrease in death rate in countries with a high production/use/either/or of CQ/HCQ could be due to any number of other factors for which this study did not account. The top 5 countries in both confirmed cases and reported deaths are all in the control list, which has no relationship to the amount of CQ/HCQ production within those countries yet skews the data to make the dynamics of death rate appear more dramatic.</p>
<p><em>Reviewed by Rachel Levantovsky as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<p><strong>Title:</strong> Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2 Infection Reveals Unique Host Responses, Viral Diversification, and Drug Interactions</p>
<p>Butler <em>et al.</em> <span class="math display">\[bioRxiv, @doi: 10.1101/2020.04.20.048066\]</span></p>
<p><strong>Key words:</strong> loop-mediated isothermal amplification (LAMP), RNA-Seq, ACE2 expression, qRT-PCR</p>
<p><strong>Summary:</strong></p>
<p>The SARS-CoV-2 pandemic has created an urgent need for widely available diagnostic tools to detect and understand host immune responses. This study introduces a fast SARS-CoV-2 detection test from swabs, based on colorimetric reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) assay technology, which amplifies viral RNA. Results were validated using both qRT-PCR and total RNA sequencing. The study also introduces a new large-scale total RNA sequencing platform which enables comprehensive transcriptome profiling of the virus, host and other microbiomes. Both technologies were used to analyze samples from 735 confirmed/suspected COVID-19 patients, as well as 62 environmental swabs from NYC subway.</p>
<p>Through a viral evolution map based on the total RNA-sequencing data, the study identified a novel subclade (A2-25563) enriched in NYC cases with likely origin in western Europe. Overall, the host transcriptome profiles revealed 216 qRT-PCR positive and 519 negative samples. Integrative analysis together with the viral load levels revealed that samples with high viral load exhibit increased expression of ACE2 (SARS-CoV-2 receptor), interferon pathway (SHFL, IFI6, IFI27, and IFIT1) and HERC6 gene and decreased expression of olfactory receptor pathway and TMPRSS-11B gene. Since ACE2 expression can be increased by pharmacologic intake of angiotensin enzyme inhibition (ACEI) or angiotensin receptor blockers (ARB), additional analysis of an observational cohort of 8,856 suspected COVID-19 patients showed a strong positive association between ACEI/ARB usage and SARS-CoV-2 infection.</p>
<p><strong>Limitations:</strong></p>
<p>While the host transcriptome analyses were performed on individuals who tested positive and negative for SARS-CoV-2 infection and other subgroups based on viral load, the influence of other clinical characteristics/phenotypes (e.g. age, symptoms, severity) were not discussed. It would also been useful to have the characteristics of the 735 confirmed/suspected COVID-19 patients.</p>
<p><strong>Findings implication:</strong></p>
<p>During this pandemic, large-scale rapid testing is key in tracking, surveillance and containment of the viral spread. To aid diagnostics, this study identified a fast LAMP assay that was applied to both naso/ oropharyngeal swabs and environmental samples and validated against gold standard qRT-PCR approach. Furthermore, total RNA-Seq profiling enabled the pursuit of multiple aspects of virus genetic features and evolution, host response, co-infections and other possible correlations between virus/host genotype and clinical phenotype. This is one of the first studies based on sequencing data from SAR-CoV-2 patients and will serve as an important resource to the research community.</p>
<p><em>Reviewed by Myvizhi Esai Selvan as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>TMPRSS2 and TMPRSS4 mediate SARS-CoV-2 infection of human small intestinal enterocytes</strong></p>
<p>Zang R. et al.  <span class="math display">\[@doi: 10.1101/2020.04.21.054015\]</span></p>
</div>
<div id="keywords-101" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-101" class="anchor"></a>Keywords</h4>
<ul>
<li><p>TMPRSS2</p></li>
<li><p>scRNAseq</p></li>
<li><p>tropism</p></li>
<li><p>enteroid</p></li>
<li><p>intestine</p></li>
<li><p>fecal</p></li>
</ul>
</div>
<div id="main-findings-97" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-97" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study uses scRNAseq to profile viral-entry genes in intestinal cell types and show confirmatory result of predominant ACE2 expression in CD26+EPCAM+ mature enterocytes. The transcriptomics also map TMPRSS2, TMPRSS4 and ST14 protease gene expression across mature enterocytes and other enteric cells. They additionally show ACE2 protein localizing to the brush border of the intestinal epithelium.</p></li>
<li><p>Researchers use VSV chimera reporter virus with SARS-CoV-2 S protein to assay viral tropism in culture and demonstrate active replication in human duodenal enteroids as well as apical mature enterocytes (ACE2+) in monolayer system.</p></li>
<li><p>Used HEK293 with ectopic expression of proteases to show that TMPRSS2 or TMPRSS4 co-expression with ACE2 enhanced viral infectivity by enhancing endocytosis. CRISPR-Cas9 based deletion or pharmacological inhibition of TMPRSS proteases in human primary enteroids limited viral replication, more so upon TMPRSS4 deletion.</p></li>
<li><p>Further, a co-culture system designed with ACE2/TMPRSS2/4 expressing host cells and S-protein expressing donor cells, to mimic enteroid cell population interactions. Importantly found that TMPRSS2 can influence viral entry in trans through paracrine interaction, unlike TMPRSS4 that functions in cis.</p></li>
<li><p>SARS-CoV-2 chimera virus were very unstable in colonic fluids, compared to rotaviruses that were stable in gastric and colonic fluids. Supporting these data, CoVID-19 patient fecal sample analysis also consistently showed viral RNA load, but no infectious particles.</p></li>
</ul>
</div>
<div id="limitations-97" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-97" class="anchor"></a>Limitations</h4>
<ul>
<li><p>The studies use pseudotyped-VSV reporter virus instead of WT virus, which could potentially have an impact on tropism. It will be important to replicate some of the co-culture studies using active SARS-CoV-2 virus as confirmation. Similarly, ectopic expression and co-culture experiments can be strengthened by replicating studies in primary cells.</p></li>
<li><p>Study mentions lack of infectious particle in fecal samples, but no conclusive data presented to support this important point.</p></li>
</ul>
</div>
<div id="significance-93" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-93" class="anchor"></a>Significance</h4>
<ul>
<li><h4 id="tmprss4-important-for-influenza-virus-tropism-but-not-previously-shown-to-be-important-in-cov-2-infectivity-in-the-gut-context.">TMPRSS4 important for influenza virus tropism, but not previously shown to be important in CoV-2 infectivity in the gut context.</h4></li>
<li><h4 id="co-culture-experiments-imply-that-paracrine-interaction-of-tmprss2-from-goblet-cells-for-example-on-neighboring-tmprss4ace2-expressing-enteric-cells-can-drive-synergistic-increase-in-viral-infectivity-in-gut-epithelia.-whether-such-gene-expression-distribution-and-protein-interactions-exist-in-other-tissue-contexts-lung-kidney-remains-to-be-seen.">Co-culture experiments imply that paracrine interaction of TMPRSS2 (from goblet cells for example) on neighboring TMPRSS4/ACE2 expressing enteric cells can drive synergistic increase in viral infectivity in gut epithelia. Whether such gene expression distribution and protein interactions exist in other tissue contexts (lung, kidney) remains to be seen.</h4></li>
<li>Fecal-oral transmission of SARS-CoV-2 has not been broadly observed, but intestinal infection of ACE2++ cells has been reported clinically; this study reconciles these paradoxical observations by demonstrating that despite active infection and replication in enterocytes the virus might not remain intact in colonic fluids. Conversely, observed SARS-CoV-2 stability in small intestine might play a role in clinically presented GI pathologies.</li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>SARS-CoV-2 serological analysis of COVID-19 hospitalized patients, pauci-symptomatic individuals and blood donors. </strong></p>
<p>Grzelak et al. <em>medRxiv</em> <span class="math display">\[@doi.org/10.1101/2020.04.21.20068858\]</span></p>
</div>
<div id="keywords-102" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-102" class="anchor"></a><em>Keywords</em>
</h4>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>COVID-19</p></li>
<li><p>Serological assays</p></li>
<li><p>NAbs</p></li>
</ul>
</div>
<div id="main-findings-98" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-98" class="anchor"></a><em>Main Findings</em>
</h4>
<p>The prevalence of the SARS-CoV2-specific antibody responses in large symptomatic and asymptomatic populations has been of great interest to understand the incidence of and immunity against SARS-CoV-2 infection. To analyze the virus-specific antibody response, Grzelak et al. designed several assays: anti-nucleocapsid (N) and anti-trimerized spike (S) ELISAs, S-Flow cellular assay, LIPS (luciferase immunoprecipitation assay) and pseudovirus neutralization. They profiled several different populations that included 491 pre-endemic individuals, 51 hospitalized CIVID-19 patients, 209 pauci-symptomatic individuals reporting mild signs compatible with COVID-19, and 200 sera from blood donors.</p>
<p>The full-length N protein or the extracellular domain of S in a trimerized form, which existed naturally on the viral surface, were used as ELISA antigens. In addition, to allow the detection of antibodies binding to various conformations and domains of viral glycoprotein by flowcytometry, S protein was expressed on the surface of 293T cells (S-Flow). Luciferase immunoprecipitation assay (LIPS) was also established with a panel of 10 different S and N-derived antigens, and N and S1 proteins were selected as the antigens for the further analysis based on their sensitivity.</p>
<p>With the four assays, signals were consistently negative or low in the pre-epidemic samples, suggesting that a pre-exposure to human seasonal coronaviruses did not induce obvious cross-reactive antibodies to S and N protein from SARS-CoV-2. The analysis of 161 samples that combined different time point samples from 51 hospitalized patients, detected positives that ranged between 65 to 72 %. On the other hand, positivity rates in pauci-symptomatic individuals varied from 27 to 36 % between assays, suggesting that pauci-symptomatic individuals either had lower viral loads or the reported symptoms were not caused by SARS-Co-V2. Correlation analysis revealed that sera with high antibody levels were well detected by all the assays, and that the highest correlation were observed between ELISA tri-S and S-Flow.</p>
<p>Lastly, the presence of neutralizing antibodies (Nabs) was evaluated by microneutralization (MNT) and pseudovirus neutralization assay using the sera from 9 hospitalized patients and 12 pauci-symptomatic individuals. A neutralization activity &gt;80% was associated with ELISA N (&gt;2.37; OD<sub>405</sub> values), ELISA tri-S (&gt;2.9; AUC values determined by plotting the log<sub>10</sub> of dilution factor required to obtain OD<sub>405</sub>), S-Flow (&gt;60%; anti-Spike IgG+ cells) and LIPS-N (&gt;0.049; Signal-to-Noise ratio).</p>
</div>
<div id="limitations-98" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-98" class="anchor"></a><em>Limitations</em>
</h4>
</div>
<div id="as-described-in-the-text-the-pauci-symptomatic-population-appears-to-be-a-mix-of-the-individuals-with-sars-cov-2-infection-and-individuals-with-other-conditions-that-had-covid-19-like-symptoms-fever-cough-or-dyspnea-" class="section level4">
<h4 class="hasAnchor">
<a href="#as-described-in-the-text-the-pauci-symptomatic-population-appears-to-be-a-mix-of-the-individuals-with-sars-cov-2-infection-and-individuals-with-other-conditions-that-had-covid-19-like-symptoms-fever-cough-or-dyspnea-" class="anchor"></a>As described in the text, the pauci-symptomatic population appears to be a mix of the individuals with SARS-CoV-2 infection and individuals with other conditions that had COVID-19-like symptoms (fever, cough or dyspnea).</h4>
</div>
<div id="it-will-be-informative-to-investigate-the-relationship-between-the-presence-of-anti-viral-antibodies-and-neutralization-activity-and-severity-of-covid-19-" class="section level4">
<h4 class="hasAnchor">
<a href="#it-will-be-informative-to-investigate-the-relationship-between-the-presence-of-anti-viral-antibodies-and-neutralization-activity-and-severity-of-covid-19-" class="anchor"></a>It will be informative to investigate the relationship between the presence of anti-viral antibodies and neutralization activity, and severity of COVID-19.</h4>
</div>
<div id="significance-94" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-94" class="anchor"></a><em>Significance</em>
</h4>
</div>
<div id="while-some-of-the-assays-described-here-cannot-be-routinely-performed-in-clinical-settings-their-higher-specificity-and-sensitivity-to-detect-antibodies-and-their-neutralizing-activity-is-important-to-understand-the-protection-mechanism-against-sars-cov-2-" class="section level4">
<h4 class="hasAnchor">
<a href="#while-some-of-the-assays-described-here-cannot-be-routinely-performed-in-clinical-settings-their-higher-specificity-and-sensitivity-to-detect-antibodies-and-their-neutralizing-activity-is-important-to-understand-the-protection-mechanism-against-sars-cov-2-" class="anchor"></a>While some of the assays described here cannot be routinely performed in clinical settings, their higher specificity and sensitivity to detect antibodies and their neutralizing activity is important to understand the protection mechanism against SARS-CoV-2.</h4>
</div>
<div id="credit-95" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-95" class="anchor"></a><em>Credit</em>
</h4>
<p><em>Reviewed by Miyo Ota as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p>Heterogeneous expression of the SARS-Coronavirus-2 receptor ACE2 in the human respiratory tract</p>
<p><strong>Doi:</strong> 10.1101/2020.04.22.056127</p>
<p><strong>Keywords: </strong></p>
<p>SARS-CoV-2, ACE2, TMPRSS2, Lung, alveolar type II cells</p>
<p><strong>The main finding of the article: </strong></p>
<p>In this study, distribution of angiotensin-converting enzyme 2 (ACE2) expression in the human cells of airways and its relation with clinical and demographic characteristics were identified using available databases of scRNA-seq, and immunohistochemistry. Samples from conducting airways and lung parenchyma from donors having chronic diseases, such as asthma and diabetes, were included (n = 15). Autopsy tissues were used as control. Through the analysis of scRNAseq, and staining of ACE2 in sections of lung parenquima, the authors demonstrated that most of the cells expressing ACE2 were alveolar type II (AT2). Of the AT2 cells, only 1.2% expressed ACE2 mRNA and 35.5% expressed the protease TMPRSS2 mRNA in the scRNAseq analysis. TMPRSS2 mRNA was detected in 50% of ACE2+ AT2 cells. ACE2 mRNA was not detected in alveolar type I (AT1) cells, macrophages, monocytes or dendritic cells. In lung sections, atelectasis areas showed a higher percentage of AT2 ACE2+ cells than non-collapsed alveoli. Interestingly, ACE2+ AT2 cells were hypertrophic compared to ACE2- AT2 cells.</p>
<p>In relation to the conducting airways, ACE2 protein expression was abundant in ciliated cells of the thinner nasal epithelium, while few ACE2+ cells were found in the trachea and bronchi where expression was restricted to ciliated cells. scRNAseq of nasal tissue however demonstrated ACE2 mRNA expression in all epithelial cell types, which suggests a different regulation between transcription and translation of ACE2.</p>
<p>When analyzing the regional distribution of the ACE2 in the airways among donors a large variability was found. Despite that, it was demonstrated that the nasal epithelium and alveoli exhibit greater presence of this protein. Sex and chronic diseases were not associated with ACE2 expression, but its detection was higher in young children than adults.</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The manuscript needs better consistency between the main text, methods and figure legends. Comparisons with a control group are confusing and not always present. The different comorbidities presented by donors may have interfered with the results presented, since each disease has a pathophysiology. Furthermore, since the immunohistochemistry analyzes were analyzed qualitatively, there should have been an evaluation by at least two different pathologists and not one. The authors could have analyzed the correlation between some immune markers with the presence of the ACE2 in the AT2 cells, since the tissues came from chronic patients. ACE2 functions in the respiratory system could have been better discussed.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>These results demonstrating ACE2 expression in nasal epithelium and lung alveolar AT2 cells, corroborates the potential of these tissues as sites involved in viral replication and transmission, as well involvement in severe lung failure. The mapping of ACE2 and TMPRSS2 expression in cells of the respiratory system may help to define news therapeutic targets.</p>
<p><strong>Long period dynamics of viral load and antibodies for SARS-CoV-2 infection: an observational cohort study</strong></p>
<p>Huang et al. <em>medRxiv</em> <span class="math display">\[@doi.org/10.1101/2020.04.22.20071258\]</span></p>
</div>
<div id="keywords-103" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-103" class="anchor"></a><em>Keywords</em>
</h4>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>Specific IgM and IgG</p></li>
<li><p>Viral load</p></li>
</ul>
</div>
<div id="main-findings-99" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-99" class="anchor"></a><em>Main Findings</em>
</h4>
<p><span id="limitations" class="anchor"></span>The presence of serum IgM and IgG against SARS-CoV2 has been shown in several studies, however, a limited number of studies have shown the longitudinal relationship between viral RNA levels and antibody titers. This retrospective, observational study evaluated the dynamics of viral RNA, IgM and IgG specific for SARS-CoV2 proteins in patients with confirmed SARS-CoV-2 pneumonia over an 8-week period.</p>
<p>Throat swabs, sputum, stool and blood samples from 33 patients with laboratory confirmed SARS-CoV-2 pneumonia were collected to analyze viral load and specific IgM and IgG against spike protein (S), spike protein receptor binding domain (RBD), and nucleocapsid (N). The demographics of the patients showed that 24 had respiratory symptom, two had symptoms in both the respiratory and the gastrointestinal tracts, one had gastrointestinal symptoms and six were asymptomatic. Chest CT revealed 27 patients had bilateral infiltrates and six had unilateral infiltrates. All the patients received antiviral treatment and atomized interferon during hospitalization.</p>
<p>While viral load in throat swabs and sputum was higher at the symptom onset and undetectable by three weeks and five weeks respectively, viral load in stool started low but remained detectable for more than five weeks in many patients. The viral loads in sputum declined significantly slower compare to throat, so that the patients were divided into two groups based on load in sputum: short-persistence (viral RNA undetectable within 22 days, n=17) and long-persistence (viral RNA persists more than 22 days, n=16). The relationship between the persistence of sputum viral RNA and antibodies showed that short-persistence group had higher anti-S IgM, anti-RBD IgM and anti-RBD IgG levels compare to long-persistence group suggesting a potential protection by anti-RBD antibodies. The length of time from symptom onset to hospital admission was also associated with SARS-CoV-2 viral clearance. In addition, the higher seropositive rate for anti-S and anti-RBD IgM was seen in long-persistence patients. They suggest that delayed admission to the hospital resulted in higher seropositive and longer infection in patients with COVID-19.</p>
</div>
<div id="limitations-1" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-1" class="anchor"></a><em>Limitations</em>
</h4>
<p>The separation of ‘low persistence’ vs ‘high persistence’ groups seems quite arbitrary, as only virus RNA levels for in sputum were considered, and viral loads in throat swabs and stool were not considered (these were no significant difference between the low vs high groups. The manuscript needs better and more detailed description of methods and figure legends. The same color codes for each patient could be used in figure 1, so that the readers could see if there was a trend in viral load between specimens in a same patient, and graphics for each patient will be useful to understand viral dinamics in the three types of samples for each person. The graphs on figure 5 seemed to correspond to one time point data, but there was no explanation which time point was used. In the results section, it was not clear which figures or tables were related to the text. The correlation between severity, viral load and persistency, and antibody titers could be analyzed.</p>
</div>
<div id="significance-95" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-95" class="anchor"></a><em>Significance</em>
</h4>
</div>
<div id="it-is-important-to-understand-the-relationship-between-viral-loads-disease-progression-and-viral-specific-antibodies-in-covid-19-disease--more-studies-are-necessary-" class="section level4">
<h4 class="hasAnchor">
<a href="#it-is-important-to-understand-the-relationship-between-viral-loads-disease-progression-and-viral-specific-antibodies-in-covid-19-disease--more-studies-are-necessary-" class="anchor"></a>It is important to understand the relationship between viral loads, disease progression and viral-specific antibodies in COVID-19 disease. More studies are necessary.</h4>
</div>
<div id="credit-96" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-96" class="anchor"></a><em>Credit</em>
</h4>
<p><em>Reviewed by Miyo Ota as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>STAT2 signaling as double-edged sword restricting viral dissemination but driving severe pneumonia in SARS-CoV-2 infected hamsters</strong></p>
<p>Boudewijns et al., bioRxiv <span class="math display">\[10.1101/2020.04.23.056838\]</span></p>
<p>Keywords:</p>
<ul>
<li><p>STAT2</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>animal model</p></li>
<li><p>transgenic hACE2</p></li>
<li><p>Syrian hamster</p></li>
</ul>
<p><em>Main findings</em></p>
<p>In this report, the authors provide an evaluation of available animal models, based on their ability to recapitulate characteristics of COVID-19 as seen in humans. As of now, both non-human primate models and transgenic hACE2 mice have limitations. An immediate type I IFN response, for one, has been notable in these models, even though the dysregulated inflammatory response seen in SARS-CoV-2 infection in humans is significant in its delayed IFN response.</p>
<p>Here, the authors describe the efficiency of SARS-CoV-2 viral replication and viral load in wild-type (WT), <em>Ifnar1</em><sup>-/-</sup>, and <em>Il28r</em><sup>-/-</sup> C57BL/6 mice. Results showed that wild-type mice that do not bear the human ACE2 receptor can be infected with SARS-CoV-2, “although inefficiently and likely transiently.” Comparison with mice without the type I IFN receptor and the type III IFN receptor showed that all strains presented with a mild lung pathology that did not resemble COVID-19 bronchopneumonia; notably, though, <em>Ifnar1</em><sup>-/-</sup> mice did present with increased levels of inflammatory infiltration, compared to the WT strain, suggesting that the active and functional IFN response in WT mice restricts pathogenesis. Hence, the disease is not fully replicable in mice.</p>
<p>The authors then propose that Syrian hamsters may be a better model, given previous reports that they are highly susceptible to SARS-CoV-2. WT, <em>STAT2</em><sup>-/-</sup> (loss of type I and III IFN signaling), and <em>Il28r</em><sup>-/-</sup> (loss of type III IFN signaling) hamsters were intranasally inoculated with SARS-CoV-2. WT hamsters carried high viral loads and infectious titers in the lungs, and histological analysis revealed necrotizing bronchiolitis, massive immune infiltration, and edema. These findings reflect the histopathological characteristics of human bronchopneumonia seen in COVID-19.</p>
<p>Interestingly, while viral RNA levels were not significantly different in the lungs of these hamsters, titers of infectious virus in the lung and viremia were highest in the <em>STAT2</em><sup>-/-</sup> hamsters, which suggests STAT2 plays a role in deterring viral dissemination. In fact, these hamsters had the highest levels of viral RNA in the spleen, liver, and upper and lower GI tracts. Still, <em>Il28r</em><sup>-/-</sup> hamsters exhibited bronchopneumonia and lung inflammation, actually to a greater extent than <em>STAT2</em><sup>-/-</sup> hamsters, suggesting that type III IFN indeed plays an important, protective role in restricting pathogenesis and the inclusion of a type I IFN exacerbates bronchopneumonia. Though, IL-6, IL-10, and IFN-ɣ were not remarkably high in the serum of infected hamsters, IL-6 and IL-10 were elevated in the lungs of infected <em>STAT2</em><sup>-/-</sup> and <em>Il28r</em><sup>-/-</sup> hamsters.</p>
<p><em>Limitations</em></p>
<p>Without comparing the aforementioned three murine strains to a transgenic hACE2 mouse strain as a “control”, it is difficult to properly assess the significance of viral replication and load in the non-transgenic mice. A similar limitation can be identified in the experiments involving the Syrian hamsters. A transgenic model with hamsters would provide additional evidence to suggest that viral replication in these animals in the absence of a human version of the ACE2 receptor is not happenstance and is truly sufficient to reflect COVID-19 disease. Or, alternatively, a transient knockout of the ACE2 receptor in hamsters before challenging them with SARS-CoV-2 to assess for signs of reduced viral infection may be appropriate.</p>
<p>Another experiment that is absent in this study is the use of antibodies against SARS-CoV-2 components required for viral entry as a means of immunizing Syrian hamsters before challenging them with the virus. This, followed by immunofluorescence or immunohistochemistry and flow cytometry for ACE2-expressing cells, would provide further validity to the concept of SARS-CoV-2 actually invading and replicating inside ACE2<sup>+</sup> cells in Syrian hamsters, which has been described for SARS-CoV-1, but not yet for SARS-CoV-2.</p>
<p><em>Significance</em></p>
<p>Ongoing studies have identified notable limitations to the use of mice to recapitulate human COVID-19 following SARS-CoV-2 infection. Lung pathology is not significant. Moreover, viral dissemination to extrapulmonary organs, including the GI tract or the liver, and viremia are not observed. Therefore, to address this unmet need for an improved animal model for SARS-CoV-2 infection, the present study proposed the use of golden Syrian hamsters, which have been previously described to better reflect SARS-CoV-1 infection than mice. The histopathological findings do provide convincing evidence to suggest that these animals may indeed be superior than mice to also study SARS-CoV-2 infection and COVID-19, but the role of STAT2 - as suggested by the experiments involving Syrian hamsters - remains to be further elucidated. The precise cellular mechanism for viral dissemination throughout the vasculature and to other peripheral organs is not clear, so the supposed dual impact of STAT2 deletion in mice or in hamsters is still phenotypic observation.</p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>SUCCESSFUL MANUFACTURING OF CLINICAL-GRADE SARS-COV-2 SPECIFIC T CELLS FOR ADOPTIVE CELL THERAPY</strong></p>
<p>Leung Wing et al.; medRxiv  2020.04.24.20077487; <a href="https://doi.org/10.1101/2020.04.24" class="uri">https://doi.org/10.1101/2020.04.24</a>. 20077487</p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 specific T cells</p></li>
<li><p>Adoptive T cell transfer</p></li>
<li><p>COVID-19</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, Leung et al. report the isolation of SARS-CoV-2-specific T cells from two convalescent COVID-19 donors (n=1 mild, n=1 severe; both Chinese Singapore residents), using Miltenyi Biotec’s fully automated CliniMACS Cytokine Capture System: convalescent donor PBMCs were stimulated with MHC class I and class II peptide pools, covering immunodominant sequences of the SARS-CoV-2 S protein as well as the complete N and M proteins; next, PBMCs were labeled with a bi-specific antibody against human CD45, a common leukocyte marker expressed on white blood cells, as well as against human IFN-γ, capturing T cell-secreted IFN-γ in response to stimulation with SARS-CoV-2 peptides. Post stimulation, IFN-γ<sup>+</sup> CD45<sup>+</sup> cells were identified by a mouse anti-human IFN-γ antibody, coupled to ferromagnetic dextrane microbeads, and magnetically labeled cells were subsequently isolated by positive immunomagnetic cell separation. Enriched IFN-γ<sup>+</sup> CD45<sup>+</sup> cells were mostly T cells (58%-71%; CD4&gt;CD8), followed by smaller fractions of B (25-38%) and NK cells (4%). Up to 74% of T cells were found to be IFN-γ<sup>+</sup>, and 17-22% of T cells expressed the cytotoxic effector marker CD56. Very limited phenotyping based on CD62L and CD45RO expression identified the majority of enriched CD4 and CD8 T subsets as effector memory T cells. TCR spectratyping of enriched T cells further revealed an oligoclonal TCR ß distribution (vs. a polyclonal distribution pre-enrichment), with increased representation of Vß3, Vß16 and Vß17. Based on limited assumptions about HLA phenotype frequencies as well as estimated haplotype sharing among Chinese Singaporeans, the authors suggest that these enriched virus-specific T cells could be used for adoptive cell therapy in severe COVID-19 patients.</p>
<p><strong><em>Limitations </em></strong></p>
<p>This preprint reports the technical adaptation of a previously described approach to isolate virus-specific T cells for targeted therapy in patients with CMV and EBV (reviewed by Houghtelin A et al.: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf</em></a>) to PBMCs obtained from two convalescent COVID-19 patients. However, not only is the title of this preprint misleading - no adoptive cell transfer was performed -, but this study also lacks relevant information - among others - on technical details such as the respective S epitopes studied, on the precise identification of immune cell subsets (e.g. NK cells: CD56<sup>+</sup> CD3<sup>-</sup>?), data pertaining to technical stimulation controls (positive/negative controls used for assay validation and potentially gating strategies), as well as on the percentage of live enriched IFN-γ<sup>+</sup> CD45<sup>+</sup> cells. Generally, a more stringent phenotypical and functional characterization (including coexpression data of CD56 and IFN-γ as well as activation, effector, proliferation and other markers) would be advisable. Similarly, in its current context, the TCR spectratyping performed here remains of limited relevance. Most importantly, though, as noted by the authors themselves, this study is substantially impaired based on the inclusion of only two convalescent donors from a relatively homogenous genetic population as well as lack of any potential recipient data. In related terms, clinical criteria, implications and potential perils of partially HLA-matched cell transfers are generally not adequately addressed by this study and even less so in the novel COVID-19 context.</p>
<p><strong><em>Significance</em></strong></p>
<p>Adoptive cell therapy with virus-specific T cells from partially HLA-matched third-party donors into immunocompromised recipients has been successfully performed in the past (cf. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5641550/pdf/fimmu-08-01272.pdf</em></a>; <a href="https://www.jci.org/articles/view/121127"><em>https://www.jci.org/articles/view/121127</em></a>). However, whether this approach might be clinically feasible for COVID-19 therapy remains unknown. Therefore, larger, more extensive studies including heterogeneous patient populations are needed to assess and balance potential risks vs. outcome in the new context of COVID-19.</p>
<p><em>This review was undertaken by V. van der Heide as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em> </p>
<p> </p>
<p><strong>Title: </strong></p>
<p>Vitamin D insufficiency is prevalent in severe COVID-19</p>
<p><strong>Doi:</strong> 10.1101/2020.04.24.20075838</p>
<p><strong>Keywords: </strong></p>
<p>SARS-CoV-2, Covid-19, Vitamin D insufficiency, African American</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study analyzed the prevalence of vitamin D insufficiency (VDI) among SARS-CoV-2 positive subjects hospitalized in a medical center situated in New Orleans. Through medical records, the authors performed a retrospective review of 20 Covid-19 patients with serum 25-hydroxycholecalciferol (25ODH) levels determined. Vitamin D insufficiency (VDI) was defined as a serum 25OHD &lt; 30 ng/mL. Among these patients, 65% required intensive care unit (ICU). When they analyzed the 25ODH levels, 84.6% of ICU subjects presented VDI in compare to 57.1% of floor subjects. Curiously, 100% of ICU patients less than 75 years old had VDI. In addition, the authors demonstrated that lymphocytopenia was present in 92.3% of the Covid-19 patients admitted to ICU. Among the clinical and demographic characteristics of SARS-Cov-19 ICU patients of this study we can highlight: 84% of the subjects were African American, hypertension and diabetes was presented in 76.9% and 46.2% ICU and non-ICU patients respectively, the body mass index of 100% of the patients was considered overweight or obesity, and all had high levels of lactate dehydrogenase.</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The manuscript data and patient description is well presented. However, the number of patients analyzed is very small to extract meaningful conclusions. The manuscript has a comprehensive discussion on the effects of VDI in prothrombosis and inflammation. The presence of vitamin D receptors in lung epithelial cells and macrophages suggests that the influences of vitamin D (VD) is relevant for respiratory health. It is known that active VD acts by reducing inflammation in human airway smooth muscle cells and influences airway barrier integrity. In addition to increasing the study population, the authors could have analyzed the direct correlation between VDI and lung damage (using specifics parameters like tomography and arterial blood gas analysis), since these exams are often prescribed to patients with severe acute respiratory syndrome.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>VDI is highly prevalent in dark-skinned individuals, and in the Italian and Spanish populations. VDI could thus contribute to Covid-19 severity disease in these populations, as VD plays an essential role in modulating the innate and adaptative immune response. This manuscript suggests that healthy levels of VD, through controlled supplementation, can help the immune system to protect against respiratory diseases like SARS-CoV-19.</p>
<p><strong>Title: </strong></p>
<p>Hypertension and Renin-Angiotensin-Aldosterone System Inhibitors in Patients with Covid-19</p>
<p><strong>Doi:</strong> 10.1101/2020.04.24.20077388</p>
<p><strong>Keywords: </strong></p>
<p>Covid-19, ACE2, Hypertension, Renin-Angiotensin-Aldosterone, Angiotensin II, Angiotensin-receptor blockers, ACE1 inhibitor</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study analyzed the effects of the arterial hypertension and of the use of renin-angiotensin-aldosterone system (RAAS) inhibitors on mortality and recovery in patients with Covid-19. Through medical records, the authors performed a multicenter retrospective study of 3017 COVID-19 patients hospitalized within the Hackensack Meridian Health network in New Jersey. Among these patients, 52.5% presented a diagnosis of hypertension. The authors showed a significantly increase (2.7 times) of the mortality in patients with hypertension compared to Covid-19 patients without hypertension. However, when adjusted for age, the effect of hypertension in mortality decreased, as the incidence of hypertension was higher in older populations. In addition, when other clinical or demographic conditions were taken into account, no effect of hypertension on mortality was found.</p>
<p>In relation to the RAAS inhibitors, angiotensin converting enzyme 1 (ACE1) inhibitors and angiotensin-receptor blockers (ARBs) were used in 22.8% and 18% of hypertensive patients. The use of ACE1 inhibitors and ARBs were found not to have detrimental effects and perhaps offer some protection to hypertensive patients in comparison with other anti-hypertensive agents. Hospital discharge rates were 9% higher for hypertensive patients prescribed RAAS inhibitors compared to other anti-hypertensive agents.</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The manuscript needs a better scientific writing, especially more in-depth details on the description of the patient population, clinical parameters, treatments used, other co-morbidities. The implications for COVID-19 disease of the upregulated cascade of vasoactive peptides belonging to RAAS on hypertensive patients, the relationship between the use of RAAS inhibitors on cytokine storm, plasma angiotensin II and ACE2 activity, could be better discussed. There is no information on which ARBs or other anti-hypertensive agents were used, despite being an important information given the different pharmacological characteristics of each one.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>While there is still uncertainty on the effect of RAAS inhibitors on Covid-19 severity in hypertensive patients, this manuscript demonstrates that ACE1 inhibitors and ARBs therapy are not detrimental, and can even be protective in hypertensive individuals. These results thus support the recommendations of the guidelines for maintaining therapy with these classes of drugs in hypertensive SARS-CoV-19 patients.</p>
</div>
</div>
<div id="title" class="section level3">
<h3 class="hasAnchor">
<a href="#title" class="anchor"></a>Title</h3>
<p><strong>Disparate temperature-dependent virus – host dynamics for SARS-CoV-2 and SARS-CoV in the human respiratory epithelium</strong><span id="keywords" class="anchor"></span></p>
<p>V’kovski et al., bioRxiv/2020.04.27.062315 <span class="math display">\[https://doi.org/10.1101/2020.04.27.062315\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>Upper and lower respiratory tracts</p></li>
<li><p>Temperature</p></li>
<li><p>Interferons</p></li>
</ul>
<p><span id="main-findings" class="anchor"></span><em>Main Findings</em></p>
<p>The anatomical distance between the upper and lower respiratory tracts, as well as their different ambient temperatures (32-33°C and 37°C, respectively) could influence viral replication and subsequent immune responses, as previously reported in studies with influenza and rhinovirus infection <span class="math display">\[1, 2\]</span>. By employing an <em>in vitro</em> model of SARS-CoV-2 and SARS-CoV infection of human airway epithelial cells (hAEC) originated from three different donors, this study aimed to investigate the effect of temperature on virus-host innate immune response dynamics in the epithelial lining of the human respiratory tract.</p>
<p>The authors show comparable SARS-CoV and SARS-CoV-2 replication at 37°C. However, when assessing viral replication efficiency at 33°C, SARS-CoV-2 infection resulted in 100-fold higher viral titers compared SARS-CoV, or SARS-CoV-2 replication at 37°C. Data in transcriptional signature after infection revealed that at 37°C, SARS-CoV-2-infection triggered earlier and greater changes in the transcriptional program compared to infection at 33°C. Among the differentially expressed genes (DEGs) after SARS-CoV-2-infection, the authors identified immunological mediators, such as chemokines and cytokines (<em>CXCL10, CXCL11, TNFSF13B, and CCL5/RANTES</em>), Interferon Stimulating Genes (ISG, such as <em>OASL</em>), as well as type I and III interferon genes. The authors tested SARS-CoV-and SARS-CoV-2 replication sensitivity to exogenous type I or type III interferon (IFN-I/III), and found a drastic reduction in viral load after hAEC infection.</p>
<p>Altogether, these results suggest a temporal, temperature-dependent gene expression profile that is inversely associated with SARS-CoV-2 replication which may account for the high replicative capacity of SARS-CoV-2 in the upper airways.</p>
<p><span id="limitations" class="anchor"></span><em>Limitations</em></p>
<p>There is considerable variability between donors with regards to viral replication at 37°C and 33°C (specially in Figure 1a), and it is unclear how many times experiments were performed.</p>
<p>The observation that temperature influences both viral replication and innate immune responses is essentially correlative. Further functional studies evaluating whether the temperature-dependent viral replication is a direct consequence of a delayed interferon response will be required. The variability between samples may also explain the fact that authors didn’t detect any DEGs in hAEC infected with SARS-CoV.</p>
<p>It was recently proposed that ACE2 can be an ISG <span class="math display">\[3\]</span> induced specifically by IFNa, favoring SARS-CoV-2 cell entry. It would be interesting to indicate the level of ACE2 expression after IFNs exogeneous treatment of hAEC in the different culture temperatures.</p>
<p>Moreover, it would be important to perform immunofluorescence analysis showing colocalization of ACE2 and SARS nucleocapsid protein.</p>
<p><em>Significance</em></p>
<p>This study provides insights about the impact of microenvironmental temperature on virus-host interactions. Enhanced replication of SARS-CoV-2 at 33°C, may influence its replication in the upper respiratory tract and transmissibility in comparison to SARS-CoV.</p>
<p><span id="credit" class="anchor"></span><em>Credit</em></p>
<p><em>Reviewed </em>by Alessandra Soares-Schanoski* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><em>References</em></p>
<p>1. Holwerda, M., et al., <em>Determining the Replication Kinetics and Cellular Tropism of Influenza D Virus on Primary Well-Differentiated Human Airway Epithelial Cells.</em> Viruses, 2019. <strong>11</strong>(4).</p>
<p>2. Foxman, E.F., et al., <em>Temperature-dependent innate defense against the common cold virus limits viral replication at warm temperature in mouse airway cells.</em> Proc Natl Acad Sci U S A, 2015. <strong>112</strong>(3): p. 827-32.</p>
<p>3. Carly G.K. Ziegler, S.J.A., Sarah K. Nyquist, Ian M. Mbano, Vincent N., et al., <em>SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues.</em> Cell, 2020.</p>
<p><strong>A possible role of immunopathogenesis in COVID-19 progression</strong></p>
<p>Anft M., Paniskaki K, Blazquez-Navarro A t al.; medRxiv  2020.04.28.20083089; <a href="https://doi.org/10.1101/2020.04.28.20083089" class="uri">https://doi.org/10.1101/2020.04.28.20083089</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 spike protein-specific T cells</p></li>
<li><p>COVID-19</p></li>
<li><p>adaptive immunity</p></li>
</ul>
<p><strong><em>Main findings </em></strong></p>
<p>In this preprint, 53 hospitalized COVID-19 patients, enrolled in a prospective study at a tertiary care center in Germany, were assigned to moderate (n=21; light pneumonia), severe (n=18; fever or respiratory tract infection with respiratory rate &gt;30/min, severe dyspnea, or resting SpO<sub>2</sub> &lt;90%), and critical subgroups (n=14; ARDS, sepsis, or septic shock) according to clinical disease. Moderately and severely ill patients with a PCR-confirmed diagnosis were recruited within four days of clinical onset, whereas critically ill patients were enrolled on average within 14 days of diagnosis on admission to ICU. To account for the overall longer hospital stay in ICU cases prior to inclusion, repeated blood samples were obtained from moderately and severely ill donors within eight days post recruitment. For 10 out of 14 ICU patients, no follow up blood samples were collected. At recruitment as well as on follow-up, circulating lymphocyte counts were below reference range in the majority of enrolled COVID-19 patients. Relative frequencies were significantly reduced in critically <em>vs</em>. moderately, but not <em>vs</em>. severely ill individuals, with substantially lower NK as well as CD8 T cells counts, and a concomitant increase of the CD4:CD8 T cell ratio in ICU patients. Basic phenotypic and immune cell subset analysis by flow cytometry detected lower frequencies of central memory CD4 T cells as well as reduced terminally differentiated CD8 Temra cells in critical COVID-19. Moreover, a decrease in activated HLA-DR<sup>+</sup> CD4 and CD8 T cells as well as in cytolytic CD57<sup>+</sup> CD8 T cells was observed in critical <em>vs</em>. severe/moderate disease. Similarly, frequencies of CD11a<sup>+</sup> CD4 and CD8 T cells as well as CD28<sup>+</sup> CD4 T cells were lower in critically ill donors, indicating a general loss of activated bulk T cells in this subgroup. In addition, a reduction of both marginal and transitional CD19<sup>+</sup> B cells was seen in patients with severe and critical symptoms. Of note, on follow-up, recovering severe COVID-19 patients showed an increase in bulk T cell numbers with an activated phenotype. Importantly, SARS-CoV-2 spike (S)-protein-specific CD4 and CD8 T cells, identified following stimulation of PBMCs with 15-mer overlapping S protein peptide pools by flow-cytometric detection of intracellular CD154 and CD137, respectively, were found in the majority of patients in all COVID-19 subgroups at the time of recruitment and further increased in most subjects by the time of follow-up (antiviral CD4 &gt;&gt; CD8 T cells). Most notably, frequencies of both antiviral CD4 and CD8 T cells were substantially higher in critically ill patients, and virus specific CD4 and CD8 T cells in both critically and severely ill subgroups were shown to produce more pro-inflammatory Th1 cytokines (TNFa, IFNg, IL-2) and the effector molecule GzmB, respectively, suggesting an overall increased magnitude of virus-specific T cell inflammation in the context of more severe disease courses. Furthermore, frequencies of antiviral CD4 T cells correlated moderately with anti-S-protein IgG levels across all patient groups.</p>
<p><strong><em>Limitations </em></strong></p>
<p>In general, this is a well executed study and most of the observations reported here pertaining to overall reduced bulk T cell frequencies (along with lower NK and other immune cell counts) as well as diminished numbers of T cells with an activated phenotype in ICU <em>vs</em>. non ICU COVID-19 corroborate findings in several previous publications and preprints (cf. <a href="https://www.jci.org/articles/view/137244" class="uri">https://www.jci.org/articles/view/137244</a>; <a href="https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa150/5813618" class="uri">https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa150/5813618</a>; <a href="https://www.nature.com/articles/s41423-020-0401-3" class="uri">https://www.nature.com/articles/s41423-020-0401-3</a>; <a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf</a>; <a href="https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf</a>). Notably, in contrast to many previous reports, the prospective study by Anft et al. enrolled a relatively larger number of COVID-19 patients of variable clinical disease (with the exception of mild cases). However, there are a few weaknesses that should be addressed. Most importantly, the choice of statistical tests applied should be carefully revised: e.g. comparison of more than two groups, as seems to be the case for most of the figures, requires ANOVA testing, which should ideally be followed by post-hoc testing (despite the somewhat confusing statement that this was conceived as an exploratory study). Given the overall limited case numbers per clinical subgroup, trends even though they might not reach statistical significance are equally important. Similarly, some statements are overgeneralized and should to be adjusted based on the actual data shown (e.g. the authors continue to refer to gradual reductions of activated T cell subset numbers in moderately <em>vs.</em> severely <em>vs</em>. critically ill patients, but for the majority of data shown substantial differences are apparent only in ICU <em>vs</em>. non-ICU patients). Moreover, it would be helpful to include exemplary FACS plots in addition to explanatory gating strategies provided in the supplemental document. There are also several inconsistencies regarding the order of data presented here (e.g. in the main manuscript, Fig S5 is chronological referred to before Fig S4) as well as pertaining to relevant technical details (according to both the main manuscript and the gating strategy in Figure S5, virus-specific CD4 T cells were identified by CD154 expression; however, in figure legend S5 virus-specific CD4 T cells are defined as CD4<sup>+</sup> CD154<sup>+</sup> CD137<sup>+</sup>). Additionally, from a technical point of view, it is somewhat intriguing that the percentages of virus-specific T cells identified by expression of CD154 and CD137, respectively, following peptide simulation seem to differ substantially from frequencies of CD154<sup>+</sup> or CD137<sup>+</sup> INFg<sup>+</sup> virus-specific T cells. Assuming a somewhat lower extent of cellular exhaustion in the moderate COVID-19 group, one would expect these cell subsets to mostly overlap/match in frequencies, therefore suggesting slight overestimation of actual virus-specific T cell numbers. In this context, inclusion of positive controls, such as CMV pp65 peptide stimulation of PBMCs from CMV seropositive donors, in addition to the already included negative controls would also be helpful. Moreover, in view of the observation that virus-specific T cells were found to be increased in critically ill ICU over non-ICU patients, a more stringent characterization of these patients as well as assessment of potential associations with clinical characteristics such as mechanical ventilation or death would add further impact to the findings described here. Finally, this study is limited to anti-S protein specific T cells. However, evaluation of N and also M-protein specific CD8 T cell responses are likely of great interest as well based on current knowledge about persistent M-protein specific memory CD8 T cells following SARS-CoV-1 infection (cf. <a href="https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.82839-0" class="uri">https://www.microbiologyresearch.org/content/journal/jgv/10.1099/vir.0.82839-0</a>).</p>
<p><strong><em>Significance</em></strong></p>
<p>In addition to reduced frequencies of activated bulk T cell numbers, the authors report an enhanced virus-specific T cell response against S protein epitopes in critically ill COVID-19 patients compared to severely and moderately ill individuals, which correlated with anti-S protein antibody titers (also cf. Ni et al.: <a href="https://doi.org/10.1016/j.immuni.2020.04.023" class="uri">https://doi.org/10.1016/j.immuni.2020.04.023</a>).</p>
<p>This is an important observation that mirrors previous data about SARS-CoV-1 (cf. Ka-fai Li C et al.: <a href="https://www.jimmunol.org/content/jimmunol/181/8/5490.full.pdf" class="uri">https://www.jimmunol.org/content/jimmunol/181/8/5490.full.pdf</a>). Furthermore, in accordance with a recent preprint by Weiskopf et al. (<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.11.20062349v1.full.pdf</a>), virus-specific CD4 T cells were found to increase in most patients over time regardless of clinical disease, whereas antiviral CD8 T cell kinetics seemed slightly less pronounced. Moreover, in the majority of moderately and severely ill cases, virus-specific T cells against the S protein could be detected early on - on average within 4 days of symptom onset. Longitudinal studies including larger numbers of COVID-19 patients across all clinical subgroups are therefore needed to further evaluate the potential impact of this observation, in particular in the context of previously described pre-existing memory T cells cross-reactive against human endemic coronaviruses (cf. <a href="https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.17.20061440v1.full.pdf</a>; <a href="https://journals.sagepub.com/doi/pdf/10.1177/039463200501800312" class="uri">https://journals.sagepub.com/doi/pdf/10.1177/039463200501800312</a>).</p>
<p><em>This review was undertaken by V. van der Heide as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</em> </p>
<p> </p>
</div>
</div>
<div id="cross-talk-between-the-airway-epithelium-and-activated-immune-cells-defines-severity-in-covid-19" class="section level1">
<h1 class="hasAnchor">
<a href="#cross-talk-between-the-airway-epithelium-and-activated-immune-cells-defines-severity-in-covid-19" class="anchor"></a><strong>Cross-talk between the airway epithelium and activated immune cells defines severity in COVID-19</strong>
</h1>
<p>Chua R.L., Lukassen S, Trump S, Hennig B.P., et al.; medRxiv 2020.04.29.20084327;</p>
<p><a href="https://doi.org/10.1101/2020.04.29.20084327"><em>https://doi.org/10.1101/2020.04.29.20084327</em></a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>Single-cell RNA sequencing</p></li>
<li><p>COVID-19</p></li>
<li><p>ACE2</p></li>
<li><p>Chemokines</p></li>
</ul>
<p><em>Main findings</em></p>
<p>The need for improved cellular profiling of host immune responses seen in COVID-19 has required the use of high-throughput technologies that can detail the immune landscape of these patients at high granularity. To fulfill that need, Chua et al. performed 3’ single-cell RNA sequencing (scRNAseq) on nasopharyngeal (or pooled nasopharyngeal/pharyngeal swabs) (NS), bronchiolar protected specimen brush (PSB), and broncheoalveolar lavage (BAL) samples from 14 COVID-19 patients with moderate (n=5) and critical (n=9, all admitted to the ICU; n=2 deaths) disease, according to WHO criteria. Four patients (n=2 with moderate COVID-19; n=2 with critical disease, n=1 on short-term non-invasive ventilation and n=1 on long-term invasive ventilation), were sampled longitudinally up to four times at various time points post symptom onset. In addition, multiple samples from all three respiratory sites (NS, PSB, BAL) were collected from two ICU patients on long-term mechanical ventilation, one of whom died a few days after the sampling procedure. Moreover, three SARS-CoV-2 negative controls, one patient diagnosed with Influenza B as well as two volunteers described as “supposedly healthy”, were included in this study with a total of n=17 donors and n=29 samples.</p>
<p>Clustering analysis of cells isolated from NS samples identified all major epithelial cell types, including basal, scretory, ciliated, and <em>FOXN4+</em> cells as well as ionocytes; of particular note, a subset of basal cells was found to have a positive IFNγ transcriptional signature, suggesting prior activation of these cells by the host immune system, likely in response to viral injury. In addition to airway epithelial cells, 6 immune cell types were identified and further subdivided into a total of 12 different subsets. These included macrophages (moMacs, nrMacs), DCs (moDCs, pDCs), mast cells, neutrophils, CD8 T (CTLs, lytic T cells), B, and NKT cells; however, seemingly neither NK nor CD4 T cells were detected and the Treg population lacked canonical expression of <em>FoxP3,</em> so it is unclear whether this population is truly represented.</p>
<p>Interestingly, secretory and ciliated cells in COVID-19 patients were shown to have upregulated ACE2 and coexpression with at least one S-priming protease indicative of viral infection; <em>ACE2</em> expression on respiratory target cells increased by 2-3 fold in COVID-19 patients, compared to healthy controls. Notably, ciliated cells were mostly <em>ACE2+/TMRPSS+</em>, while secretory and <em>FOXN4</em>+ cells were predominantly <em>ACE2+/TMRPSS+/FURIN+</em>; accordingly, secretory and ciliated cells contained the highest number of SARS-CoV-2 infected cells. However, viral transcripts were generally low 10 days post symptom onset (as would be expected based on reduced viral shedding in later stages of COVID-19). Similarly, the authors report very low counts of immune cell-associated viral transcripts that are likely accounted for by the results of phagocytosis or surface binding. However, direct infection of macrophages by SARS-CoV-2 has previously been reported [^1,2^](<a href="https://paperpile.com/c/0qRp0H/qZnZ+hPdH" class="uri">https://paperpile.com/c/0qRp0H/qZnZ+hPdH</a>). Here, it is possible that these differences could be due to the different clinical stages and non-standardized gene annotation.</p>
<p>Pseudotime mapping of the obtained airway epithelial data suggested a direct differentiation trajectory from basal to ciliated cells (in contrast to the classical pathway from basal cells via secretory cells to terminally differentiated ciliated cells), driven by interferon stimulated genes (ISGs). Moreover, computational interaction analysis between these <em>ACE2<sup>+</sup></em> secretory/ciliated cells and CD8 CTLs indicated that upregulation of ACE2 receptor expression on airway epithelial cells might be induced by IFNγ, derived from these lymphocytes. However, while IFN-mediated <em>ACE2</em> upregulation in response to viral infections may generally be considered a protective component of the antiviral host response, the mechanism proposed here may be particularly harmful in the context of critical COVID-19, rendering these patients more susceptible to SARS-CoV-2 infection.</p>
<p>Moreover, direct comparisons between moderate and critical COVID-19 patient samples revealed fewer tissue-resident macs and monocyte-derived dendritic cells but increased frequencies of non-resident macs and neutrophils in critically ill COVID-19 patients. Notably, neutrophil infiltration in COVID-19 samples was significantly greater than in those obtained from healthy controls and the Influenza B patient. In addition, patients with moderate disease and those on short-term non-invasive ventilation had similar gene expression profiles (each n=1),; whereas, critical patients on long-term ventilation expressed substantially higher levels of pro-inflammatory and chemoattractant genes including <em>TNF, IL1B, CXCL5, CCL2</em>, and <em>CCL3</em>. However, no data on potentially decreasing gene expression levels related to convalescence were obtained. Generally, these profiles support findings of activated, inflammatory macrophages and CTLs with upregulated markers of cytotoxicity in critically ill COVID-19 patients. These inflammatory macrophages and CTLs may further contribute to pathology via apoptosis suggested by high <em>CASP3</em> levels in airway epithelial cells. Interestingly, the CCL5/CCR5 axis was enriched among CTLs in PSB and BAL samples obtained from moderate COVID-19 patients; recently, a disruption of that axis using leronlimab was reported to induce restoration of the CD8 T cell count in critically ill COVID-19 patients [^3^](<a href="https://paperpile.com/c/0qRp0H/f82N" class="uri">https://paperpile.com/c/0qRp0H/f82N</a>).</p>
<p>Lastly, in critically ill COVID-19 patients, non-resident macrophages were found to have higher expression levels of genes involved in extravasation processes such as <em>ITGAM, ITGAX</em> and others. Conversely, endothelial cells were shown to express <em>VEGFA</em> and <em>ICAM1</em>, which are typical markers of macrophage/immune cell recruitment. This finding supports the notion that circulating inflammatory monocytes interact with dysfunctional endothelium to infiltrate damaged tissues. Of note, in the patient with influenza B, cellular patterns and expression levels of these extravasation markers were profoundly different from critically ill COVID-19.</p>
<p>Importantly, the aforementioned immune cell subsets were found equally in all three respiratory site samples obtained from two multiple-sample ICU donors, and there were no differences, with regards to upper <em>vs</em>. lower respiratory tract epithelial ACE2 expression. However, viral loads were higher in BAL samples as compared to NS samples, and lower respiratory tract macrophages showed overall greater pro-inflammatory potential, corresponding to higher <em>CASP3</em> levels found in PSB and BAL samples. In general, the interactions between host airway epithelial and immune cells described in this preprint likely contribute to viral clearance in mild and moderate disease but might be excessive in critical cases and may therefore contribute to the observed COVID-19 immunopathology. Based on these findings and the discussed immune cell profiles above, the authors suggest the use of immunomodulatory therapies targeting chemokines and chemokine receptors, such as blockade of CCR1 by itself or in combination with CCR5, to treat COVID-19 associated hyperinflammation.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>In addition to the small sample size, it is unclear whether samples were collected at similar time points throughout the disease course of each patient, even with time since diagnosis normalized across patients. While sampling dates in relation to symptom onset are listed, it remains somewhat unclear what kind of samples were routinely obtained per patient at given time points (with the exception of the two patients with multiple sampling). Moreover, it would have been of particular interest (and technically feasible) to collect additional swabs from the convalescent ICU patient to generate a kinetic profile of chemokine gene expression levels, with respect to disease severity as well as onset of recovery. Again, with an n=1, the number of cases per longitudinal/multiple sampling subgroup is very limited, and, in addition to the variable sampling dates, overall time passed since symptom onset as well as disease symptoms and potential treatment (e.g. invasive vs non-invasive ventilation, ECMO therapy…) across all clinical subgroups, makes a comparative analysis rather difficult.</p>
<p>It is important to note that a lack of standardized gene annotation across different studies contributes to a significant degree of variability in characterizations of immune landscapes found in COVID-19 patients. As a result, inter-study comparisons are difficult to perform. For instance, an analysis of single-cell RNA sequencing performed on bronchoalveolar lavage samples by Bost et al. identified lymphoid populations that were not found in the present study. These include several enriched subtypes of CD4<sup>+</sup> T cells and NK cells, among others. Ultimately, these transcriptomic descriptions will still need to be furthered with additional follow-up studies, including proteomic analysis, to move beyond speculation and towards substantive hypotheses.</p>
<p><em>Biological</em></p>
<p>One additional limitation involved the use of the influenza B patient. Given that the patient suffered a rather mild form of the disease (no ICU admission or mechanical ventilation required, patient was discharged from hospital after 4 days) as opposed to the to authors’ assessment as a severe case, this patient may have served as an acceptable positive control for mild and some moderate COVID-19 patients. However, this approach should still be viewed cautiously, since the potential differences of pulmonary epithelial and immune cell pathologies induced by influenza compared to critical COVID-19 patients are still unclear. Moreover, it seems that one of the presumably healthy controls was recovering from a viral infection. Since it is unclear how a recent mild viral infection might have changed the respiratory cellular compartment and immune cell phenotype, this donor should have been excluded or not used as a healthy reference control.</p>
<p><em>Significance</em></p>
<p>In general, this is a well-conducted study and provides a number of corroborative and interesting findings that contribute to our understanding of immune and non-immune cell heterogeneity in COVID-19 pathogenesis. Importantly, observations on ACE2 and ACE2 coexpression in airway epithelial cells generally corroborate previous reports. In addition, direct differentiation of IFNγ<sup>+</sup> basal cells to ACE2-expressing ciliated cells, as suggested by trajectory analysis, is a very interesting hypothesis, which, if confirmed, might contribute to progression of disease severity. The findings described in this preprint further suggest an important role for chemokines and chemokine receptors on immune cells, most notably macrophages and CTLs, which is highly relevant.</p>
<p><em>This review was undertaken by Matthew D. Park and Verena van der Heide as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong><em>References</em></strong></p>
<p>1. <a href="http://paperpile.com/b/0qRp0H/qZnZ">Chen, Y. <em>et al.</em> The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Directly Decimates Human Spleens and Lymph Nodes. <em>Infectious Diseases (except HIV/AIDS)</em> (2020) doi:</a><a href="http://dx.doi.org/10.1101/2020.03.27.20045427">10.1101/2020.03.27.20045427</a><a href="http://paperpile.com/b/0qRp0H/qZnZ">.</a></p>
<p>2. <a href="http://paperpile.com/b/0qRp0H/hPdH">Bost, P. <em>et al.</em> Host-viral infection maps reveal signatures of severe COVID-19 patients. <em>Cell</em> (2020) doi:</a><a href="http://dx.doi.org/10.1016/j.cell.2020.05.006">10.1016/j.cell.2020.05.006</a><a href="http://paperpile.com/b/0qRp0H/hPdH">.</a></p>
<p>3. <a href="http://paperpile.com/b/0qRp0H/f82N">Patterson, B. K. <em>et al.</em> Disruption of the CCL5/RANTES-CCR5 Pathway Restores Immune Homeostasis and Reduces Plasma Viral Load in Critical COVID-19. <em>medRxiv</em> (2020).</a></p>
<p><strong>SARS-CoV-2 infection of the placenta</strong></p>
<p>Hosier et al.; medRxiv <span class="math display">\[10.1101/2020.04.30.20083907\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>SARS-CoV-2</p></li>
<li><p>pregnancy</p></li>
<li><p>placenta</p></li>
<li><p>syncytiotrophoblast</p></li>
<li><p>inflammation</p></li>
</ul>
<p><em>Main findings</em></p>
<p>The impact of SARS-CoV-2 infection and subsequent COVID-19 disease in pregnant women at different trimesters is not well described. The precise influence of a potentially dysregulated antiviral response to this pathogen during pregnancy is unclear, so a better understanding would guide management for pregnant women, who may be more susceptible to infection. Here, Hosier et al. profile a case of a woman with COVID-19 in the second trimester of pregnancy with severe hypertension, elevated liver enzymes, and coagulopathy. These symptoms led to a diagnosis of severe preeclampsia.</p>
<p>The patient initially presented with a high fever, non-productive cough, nausea, diarrhea, diffuse myalgias, anorexia, and malaise. A RT-PCR test for SARS-CoV-2 RNA in a nasopharyngeal swab of the patient was positive. Upon admission, the patient was treated for hypertension, disseminated intravascular coagulopathy, and she eventually elected to terminate via dilation and evacuation. During surgical management and recovery, the patient developed lymphopenia, though she was able to be extubated and weaned to room air on post-operative day 1. The patient was given hydroxychloroquine. Two days later, her coagulation markers improved, and she was discharged.</p>
<p><strong>RT-PCR and sequencing and phylogenetic analyses</strong> showed that the placenta and umbilical cord were positive for SARS-CoV-2 RNA, but no other major fetal tissues tested positive. Saliva and urine, collected from the patient, also tested positive, although the oral and nasal swabs did not. Whole genome sequencing of the viral genome isolated from the placenta was phylogenetically similar to those isolated from local cases of SARS-CoV-2 infection and those identified in Europe and Australia.</p>
<p><strong>Serologic testing</strong> for patients’ antibodies revealed high titers of anti-SARS-CoV-2 IgG and IgM antibodies. These levels were reportedly the highest of the 56 COVID-19 patients admitted to the Yale New Haven Hospital, suggesting that the patient was not unsuccessful in eliciting a humoral response.</p>
<p><strong>Gross pathological examination</strong> revealed a marginally adherent blood clot, presenting as a focal placental infarct, while histological analyses using CD3 and CD68 as markers revealed an infiltration of T cells and macrophages, which suggests that fibrin-dense intervillositis may have contributed to the coagulopathy observed in the patient. No necrotic tissue was seen. Immunohistochemistry staining for the SARS-CoV-2 spike protein and <em>in situ</em> hybridization for the SARS-CoV-2 RNA revealed that the infection of fetal tissue was localized preferentially to the syncytiotrophoblasts of the placenta.</p>
<p><strong>Electron microscopy</strong> confirmed the presence of viral particles (75-100 nm in diameter) in the cytosol of placental cells.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>This report profiles a single patient to describe the impact of COVID-19 in pregnant women. Without a larger sample size, it is difficult to assess, however, how infection or peak disease at a given trimester differentially influence prognosis and clinical outcome. It is also important to note that this patient was diagnosed with an underlying autoimmune disease, psoriasis. It will also be important for future studies to consider the stage of pregnancy and whether COVID infection leads to other pregnancy-related disorders, such as recurring miscarriage, fetal growth restriction and possibly even increase in size of fetus creating challenges for delivery.</p>
<p><em>Biological</em></p>
<p>Finally, it is unclear whether these placental cells express the ACE2 receptor. Though electron microscopy, among other methods, identified viral particles in placental cells and indicated SARS-CoV-2 infection of the placenta, the authors did not demonstrate that the syncytiotrophoblasts expressed the ACE2 receptor, which has been shown to be the target of SARS-CoV-2 viral entry into host cells.</p>
<p><em>Additional considerations</em></p>
<p>The immunology of pregnancy is not static - the myeloid and T cell repertoire of the placental micro-environment is dynamic throughout the different trimesters. For instance, during the first trimester, trophoblastic cells secrete cytokines and chemokines that promote the recruitment and infiltration of circulating monocytes, neutrophils, NK cells, and T cells<sup>1,2</sup>. This trafficking is essential, and disruption of any elements of this signaling axis results in poor pregnancy outcomes<sup>1</sup>. Notably, NK cells and monocyte-derived macrophages are responsible for decidual vascular and tissue remodeling<sup>1,2</sup>.</p>
<p>The second trimester, however, is described as an anti-inflammatory stage, characterized by the induction of regulatory T cells by decidual CD56<sup>bright</sup>CD16<sup>-</sup> NK cells and monocyte-derived macrophages. Interestingly, a population of T<sub>H</sub>17 cells are also present and expand during the second trimester<sup>1</sup>. The third trimester is then marked by a return to an inflammatory phenotype<sup>1</sup>. It is unclear how these differential states are influenced by an antiviral response to SARS-CoV-2 infection. The authors reported the presence of macrophages and T cells, but a lack of more specific stains and analyses make it difficult to precisely characterize the immunological anomalies of the placental micro-environment in pregnant women with COVID-19. The role of decidual NK cells is likely to be especially important, given their role in trophoblast-mediated immune modulation during the different trimesters of pregnancy and their role in the antiviral response to viral infections.</p>
<p><em>Significance</em></p>
<p>Hypertensive disorders, like preeclampsia, in pregnant women increase the likelihood for complicated pregnancies, and recent studies of the field suggest that dysregulated immune activity may partially be responsible for these outcomes. The trimesters of pregnancy exhibit different immune landscapes, so the presence of certain microbes, including viral pathogens, is likely to perturb homeostatic immune processes that are required for a normal pregnancy. The impact of SARS-CoV-2 infection, therefore, warrants its own investigation, as the health outcomes of COVID-19 are especially poor. The authors report direct infection of the syncytiotrophoblast by SARS-CoV-2. These cells are derived from trophoblasts, which play an important immuno-modulatory role in all three trimesters of pregnancy. So, collectively, given the role of the ACE2 receptor as the target of SARS-CoV-2 viral entry and the involvement of ACE2 in the physiology of preeclampsia, the two pathologies likely share altered immune states and a dysfunctional renin-angiotensin-aldosterone system (RAAS) as etiologies.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>References</strong></p>
<p>1 Mor, G., Aldo, P., &amp; Alvero, A.B. The unique immunological and microbial aspects of pregnancy.</p>
<p><em>Nat. Rev. Immunol.</em> 17, 469-482 (2020).</p>
<p>2 Yockey, L.J., Lucas, C., &amp; Iwasaki, A. Contributions of maternal and fetal antiviral immunity in</p>
<p>congenital disease. <em>Science</em>. 368, 608-612 (2020).</p>
<div id="early-phases-of-covid-19-are-characterized-by-a-reduction-of-lymphocyte-populations-and-the-presence-of-atypical-monocytes" class="section level3">
<h3 class="hasAnchor">
<a href="#early-phases-of-covid-19-are-characterized-by-a-reduction-of-lymphocyte-populations-and-the-presence-of-atypical-monocytes" class="anchor"></a>Early phases of COVID-19 are characterized by a reduction of lymphocyte populations and the presence of atypical monocytes</h3>
<p>Lombardi A et al. <em>medRxiv</em> <span class="math display">\[@doi: 10.1101/2020.05.01.20087080\]</span></p>
<div id="keywords-104" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-104" class="anchor"></a>Keywords</h4>
<ul>
<li><p>COVID-19</p></li>
<li><p>Lymphocytes</p></li>
<li><p>Monocytes</p></li>
<li><p>PBMC</p></li>
<li><p>inflammation</p></li>
</ul>
</div>
<div id="main-findings-100" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-100" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study evaluated PBMCs of 63 italian patients with COVID-19 early after diagnosis (~5 days post-symptom start) using flow cytometry, and determined any association of inflammatory biomarkers with 28-days mortality.</p></li>
<li><p>Observed reduction in circulating lymphocytes (CTL, NK and NKT) and phenotypic changes in CD4+ T cells towards Th2 polarization, compared to reference intervals. Serum IL-6 levels not associated with monocyte counts, but correlated with Th2-polarized CD4+ T cell prevalence.</p></li>
<li><p>Detected enrichment of high-scatter atypical monocytes (in 11/14 subgrouped patients) with low CD14 and low HLA-DR MFI, but classical/non-classical monocytes abundance within normal reference range.</p></li>
<li><p>Lymphopenia, but also activation of T lymphocytes (defined as CD38+HLA-DR+ CTLs) reported to correlate with death within 28 days (more severe progression).</p></li>
</ul>
</div>
<div id="limitations-99" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-99" class="anchor"></a>Limitations</h4>
<ul>
<li><p>All the analysis is using peripheral blood, which provides superficial global immune context but might not necessarily inform local inflammation in the alveoli or other organs. Concordant studies in BAL or rapid-autopsy tissue would be helpful</p></li>
<li><p>Activation of lymphocytes defined by CD38 and HLA-DR would need to be assayed functionally, potentially using IFN-g or Gzm-B ELISpot.</p></li>
<li><p>There is no breakdown of inflammatory markers across 28-day patient survival stratification in Table 3. Additionally, there is no correlation analysis shown for IL-6 levels (or other biomarkers) with the density of atypical monocytes. Such a multi-variate correlation study could tease out inflammatory biomarkers useful for prioritizing care on the hospital frontline.</p></li>
</ul>
</div>
<div id="significance-96" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-96" class="anchor"></a>Significance</h4>
<ul>
<li><p>Provides on-the-ground clinical report of immunological characteristics from Milanese patients (despite being single-center small cohort of patients).</p></li>
<li><p>Results are in line with confirmed lympho-dysfunction observed in COVID-19 patients by multiple groups, and also support observations of atypical HLA-DR-low monocytes in severe disease (Giamarellos-Bourboulis et al. 2020).</p></li>
<li><p>Lymphocytopenia linked strongly to 28-day mortality stratification, in addition to standard clinical variates such as age, P/F ratio, LDH and CRP/Ferritin levels. Whether lymphocytopenia and associated innate-immune dysfunction is causal to adverse outcome is unclear, and will hopefully be revealed by prospective longitudinal clinical studies and potentially animal models.</p></li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><em>Immune alterations during SARS-CoV-2-related acute respiratory distress syndrome</em></p>
<p>Bouadma et al. <em>bioRxiv</em> <span class="math display">\[@doi:
https://doi.org/10.1101/2020.05.01.20087239\]</span></p>
</div>
<div id="keywords-105" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-105" class="anchor"></a>Keywords</h4>
<ul>
<li><p>T cells</p></li>
<li><p>NK cells</p></li>
<li><p>monocytes</p></li>
</ul>
</div>
<div id="main-findings-101" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-101" class="anchor"></a>Main Findings</h4>
<p>In this study, Bouadma and authors longitudinally profiled multiple immune parameters of a fatal case of Covid-19 that quickly developed multiorgan failure. An 80-year old male patient presented with fever and diarrhea that developed into multiorgan failure and hemoptysis over the course of 24 days that resulted in death. During this time, he was treated with broad-spectrum antibacterial agents, Remdesivir, and interferon beta-1a. Peripheral naive CD4+ and CD8+ T cells remained stable throughout, but effector memory T cells continually increased. Exhausted and senescent CD4+ and CD8+ T cells, and gamma delta T cells increased following day 14. Activated and exhausted B cells peaked on day 20. After day 16, NK cells and monocytes generally declined possibly due to lung trafficking. These fluctuations in immune populations were accompanied by induction of pro-inflammatory cytokines and Th1/Th2 factors that increased on day 14. Although some cytokines decreased following day 14, cytokines associated with T cell activation, exhaustion, and apoptosis continued to increase.</p>
</div>
<div id="limitations-100" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-100" class="anchor"></a>Limitations</h4>
<p>It is difficult to draw broad conclusions from one patient and this longitudinal study did not start at the onset of infection and symptoms. Furthermore, these observations were done on the peripheral blood without complementary analysis of the lung where they suspect NK cells and monocytes have trafficked to.</p>
</div>
<div id="significance-97" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-97" class="anchor"></a>Significance</h4>
<p>This shows that immune cells proportion, functional state, and soluble factors fluctuate throughout disease progression. This is a broad overview of potential blood biomarkers that can be used to assess progression and severity.</p>
</div>
<div id="credit-97" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-97" class="anchor"></a>Credit</h4>
<p><em>Reviewed by </em>Dan Fu Ruan, Evan Cody and Venu Pothula* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><strong>Identification of Drugs Blocking SARS-CoV-2 Infection using Human Pluripotent Stem Cell-derived Colonic Organoids</strong></p>
<p>Duan, X. <em>et al</em>. 2020. Biorxiv. DOI: 10.1101/2020.05.02.073320</p>
<p><strong>Keywords</strong>:</p>
<ul>
<li><p>SARS-CoV-2 drug screening</p></li>
<li><p>colonic organoids</p></li>
<li><p>gastrointestinal complications</p></li>
<li><p>MPA</p></li>
<li><p>QNHC</p></li>
</ul>
<p><strong>Main findings</strong>:</p>
<p>Gastrointestinal complications have been reported in almost 25% of patients infected with the SARS-CoV-2 virus<sup>1</sup>. This study illustrates a high-throughput drug screening platform using Human Pluripotent Stem Cell-derived Colonic Organoids (hPSC-COs) infected with the SARS-CoV-2 pseudo-entry virus. Using single-cell RNA sequencing and phenotypic characterization of hPSC-COs, the authors show endogenous expression of ACE2 in 5 characterized colonic cellular subsets, with the highest expression of ACE2 in KRT20+ enterocytes. hPSC-COs were inoculated with a SARS-CoV-2 pseudo-entry virus, which led to a significant decrease in both KRT20+ enterocytes and ACE2+ cells. The authors also used humanized <em>NOD-scid IL2Rg<sup>null</sup></em> mice transplanted with hPSC-COs to serve as a unique <em>in vivo</em> system for modeling COVID-19. Consistent with <em>in vitro</em> findings, ACE2 was detected in KRT20+ enterocytes from organoid xenografts, and luciferase expression was most highly detected in these populations after local infection with the SARS-CoV-2 pseudo-entry virus.</p>
<p>The authors next used the hPSC-CO system to perform high-throughput screening of drugs from the Prestwick FDA-approved library; they identified eight drugs that could block SARS-CoV-2 pseudo-virus infection, as measured by a reduction of luciferase activity by at least 75%. Of note, mycophenolic acid (MPA) and quinacrine dihydrochloride (QNHC) were drugs that showed specificity to the SARS-CoV-2 pseudo-entry virus, and were also at least 5 times more efficacious than chloroquine at inhibiting viral entry. Additionally, mice treated with MPA prior to local SARS-CoV-2 infection had significantly lower luciferase+ cells in xenografts compared to vehicle-treated mice. Lastly, the authors showed an enrichment of specific chemokines that were upregulated in hPSC-COs infected with the SARS-CoV-2 pseudo-entry virus, consistent with the “cytokine storm” that has been reported in COVID19 patients<sup>2</sup>.</p>
<p><strong>Limitations:</strong></p>
<p>While the authors tested efficacy of MPA and QNHC prior to SARS-CoV-2 virus infection, they did not show whether these drugs would have beneficial effects after viral infection. In addition, the effects of QNHC were not investigated <em>in vivo</em>, which would need to be conducted before proceeding to clinical trials. While this study focuses on using MPA and QNHC in colonic organoid-derived systems, it would be important to understand if similar therapeutic mechanisms also exist in infected respiratory epithelial cells, such as lung alveolar epithelial cells and nasal epithelial cells<sup>3</sup>.</p>
<p><strong>Significance:</strong></p>
<p>This study reveals advancements in utilizing translational hPSC-COs and humanized mouse models to elucidate key cell types that might allow for enhanced SARS-CoV-2 infection in the gastrointestinal system. The authors’ <em>in vivo</em> system allowed them to study biological effects of cells naturally expressing ACE2, instead of relying on transgenic mouse models expressing the human ACE2 receptor. Moreover, the <em>in vitro</em> hPSC-CO system enabled rapid, high-throughput screening of FDA-approved drugs, which uncovered MPA and QNHC to be promising candidates for SARS-CoV-2 entry inhibition, with greater efficacy than drugs currently being investigated for therapeutic use in COVID-19.</p>
<p><strong>References:</strong></p>
<p>1. Cheung, K.S., Hung, I.F., Chan, P.P., <em>et al.</em> Gastrointestinal Manifestations of SARS-CoV-2 Infection and Virus Load in Fecal Samples from the Hong Kong Cohort and Systematic Review and Meta-analysis, Gastroenterology (2020), <a href="https://doi.org/10.1053/j.gastro.2020.03.065" class="uri">https://doi.org/10.1053/j.gastro.2020.03.065</a>.</p>
<p>2. Ricardo, J.J. and Manuel .A. COVID-19 cytokine storm: the interplay between inflammation and coagulation. The LANCET (2020), <a href="https://doi.org/10.1016/S2213-2600(20)30216-2" class="uri">https://doi.org/10.1016/S2213-2600(20)30216-2</a></p>
<p>3. Sungnak, W., Huang, N., Bécavin, C. <em>et al.</em> SARS-CoV-2 entry factors are highly expressed in nasal epithelial cells together with innate immune genes. <em>Nat Med</em> <strong>26, </strong>681–687 (2020). <a href="https://doi.org/10.1038/s41591-020-0868-6" class="uri">https://doi.org/10.1038/s41591-020-0868-6</a></p>
<p><strong>Credit:</strong></p>
<p><em>Reviewed by Shikha Nayar as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Functional alteration of innate T cells in critically ill Covid-19 patients</strong></p>
<p>Jouan Y et al., medRxic 2020.05.03.20089300; doi: <a href="https://doi.org/10.1101/2020.05.03.20089300" class="uri">https://doi.org/10.1101/2020.05.03.20089300</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>Innate T cells (MAIT, iNT, γδ T cells)</p></li>
<li><p>Critically ill COVID-19 patients</p></li>
<li><p>SARS-CoV-2</p></li>
</ul>
<p><strong><em>Main findings</em></strong></p>
<p>Innate T cells are a heterogeneous group of immune cells, including mucosal-associated invariant T cells (MAIT), NKT cells (NKT) and γδ T cells, which differ from conventional T cells based on their unique features considered characteristic of both the innate and adaptive immune system. In this preprint, Jouan et al. aimed to address the potential contribution of innate T cells to COVID-19 lung immunopathology. SARS-CoV-2 PCR-positive ICU patients, diagnosed with severe COVID-19 according to clinical symptoms, were enrolled in a prospective study on average within 10 days of disease onset, and were immunologically followed up for two weeks. Of 30 patients, 24 presented with ARDS with the need for invasive mechanical ventilation either at the time of ICU admission (n=20) or later on (n=4); to maintain sufficient systemic oxygenation, one patient further required extracorporeal membrane oxygenation (ECMO), whereas another patient succumbed to disease two days post study inclusion. By the end of the observation period, 15 individuals could be discharged from ICU, while 9/14 ICU patients remained on invasive mechanical ventilation. Similar to several recent reports, the authors observed COVID-19-associated mild to severe lymphopenia, which correlated with disease severity, as well as concomitantly higher neutrophil-to-lymphocyte ratios when compared to age- and sex-matched healthy controls (n=20). Likewise, albeit relatively low overall, plasma levels of the proinflammatory cytokines IL-1β, IL-6, IL-1Rα and IFN-α2, a marker of the antiviral type I IFN response, were found to be higher in COVID-19 patients <em>vs</em>. healthy individuals (n=10). Of particular note, these cytokines were further increased in endotracheal aspirate (ETA) supernatants relative to plasma samples obtained from the same patients (n=20), suggesting locally enhanced airway inflammation at the time of ICU admission. Moreover, analyzing relative frequencies of circulating innate T cells in COVID-19 ICU patients (n=30) <em>vs</em>. age-/sex-matched healthy controls (n=20), the authors report a profound decrease in MAIT (ca. 6-fold; identified by 5-OP-RU loaded MR1 tetramer <em>vs</em>. unloaded control tetramer) as well as in iNKT cells (ca. 7 fold; identified by PBS-57 glycolipid-loaded CD1d tetramer <em>vs</em>. unloaded control tetramer), while γδ T cell frequencies remained largely stable (with the exception of a slight decrease in the Vδ<sup>2</sup> subset as well as a likely increase in the Vδ<sup>3</sup> subset). Interestingly, both MAIT and γδ T cell were detected in early ETA samples of 12/21 ICU patients on invasive mechanical ventilation, whereas iNKT cells were seemingly absent. Of importance, relative MAIT but not γδ T numbers were increased in ETA specimens when compared to plasma samples obtained from the same donors, suggesting enhanced airway recruitment in contrast to potential extravasation processes as a result of inflammation-induced capillary leakage. Accordingly, the presence of innate T cells was restricted to ETA samples containing high levels of chemo-attractive CXCL10 and CXCL12. Moreover, very basic phenotypic analysis of blood-circulating innate T cells by flow cytometry revealed increased activation based on enhanced expression of CD69 and PD-1 in COVID-19 <em>vs</em>. healthy controls, as well as a potential trend towards exhaustion based on continuous PD-1 expression observed after repeated sampling. Enhanced levels of CD69 and PD-1 were further associated with high levels of plasma Il-18, which has been previously linked to innate T cell activation in viral infections (<a href="https://www.jimmunol.org/content/194/8/3890" class="uri">https://www.jimmunol.org/content/194/8/3890</a>). In line with the above observations, innate T cell-expressed activation markers were found to be higher at the presumable site of inflammation as compared to peripheral circulation. Additionally, functional analysis of circulating MAIT, iNKT, and γδ T cell in COVID-19 <em>vs</em>. healthy controls showed substantially reduced IFNγ along with slightly increased production of IL-17a following stimulation with PMA/Ionomycin, corroborating a potential tendency of peripheral cellular exhaustion. Conversely, both IFNγ and IL-17a were increased in ETA over plasma samples in COVID-19 ICU patients and levels of both cytokines were equally higher in those ETA specimen containing innate T cells suggesting that these cells contribute to local airway proinflammatory cytokine production. Most importantly, while multiple sampling revealed a further decrease of CD69+ activated circulatory MAIT, iNKT, and γδ T in COVID-19 over time, the authors observed that higher frequencies of activated circulatory MAIT and iNKT cells on day 7 correlated moderately with increased PaO<sub>2</sub>/FiO<sub>2</sub> ratios, indicative of preserved lung oxygenation, thus suggesting that activation of innate T cells in early COVID-19 may be used as a predictor of disease severity.</p>
<p><strong><em>Limitations</em></strong></p>
<p><em>Technical/biological</em></p>
<p>In addition to the relatively small study size, it remains somewhat unclear how many ICU patients were included in some experiments pertaining to ETA sample collection. Furthermore, a more stringent clinical characterization of enrolled patients (in addition to basic information presented in table 1) would add further impact to the observations made here regarding the potential role of innate T cells in critical COVID-19 as well as regarding any further association with ARDS severity, death or clinical course in general. To adequately address some of these questions, larger studies including patients across all clinical stages as well as early-onset and longitudinal sampling will be needed. Similarly, it would be of great interest to further examine kinetics of both potential innate T cell migration to inflamed airways and general function by means of repeated sampling. Additionally, extended flow-cytometric phenotyping of both circulatory and airway innate T cells will be needed to further characterize these populations beyond their basic activation status. In this context, general gating strategies should be included in the supplemental.</p>
<p><em>General</em></p>
<p>Throughout the manuscript, some references pertaining to specific figures or their content are incorrect. For example, supplementary figure 1A shows a COVID-19 lung CT scan, but no healthy control as suggested by the complementary figure legend. Data on plasma and ETA IL-1Rα levels are displayed in figure 1 E &amp; F, not in supplementary figure 1B as stated in the manuscript. Supplementary figure 1B does not exist. Similarly, some of the literature references seem to have been mixed up (cf. reference 16 on the role of MAIT cells in diabetes and obesity <em>vs</em>. type I IFN response in manuscript; reference 23 is missing).</p>
<p><strong><em>Significance</em></strong></p>
<p>Innate T cells have been previously shown to mediate potent antiviral immune mechanisms in both mouse and human studies (reviewed in <a href="https://link.springer.com/content/pdf/10.1007/s11684-017-0606-8.pdf"><em>https://link.springer.com/content/pdf/10.1007/s11684-017-0606-8.pdf</em></a>). Observations made here generally corroborate limited data reported in two recent preprints (<a href="https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1"><em>https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1</em></a>; <a href="https://www.medrxiv.org/content/10.1101/2020.04.05.20046433v1.full.pdf"><em>https://www.medrxiv.org/content/10.1101/2020.04.05.20046433v1.full.pdf</em></a>) on activation status and frequencies of γδ T in COVID-19 patients. Notably, this preprint by Jouan et al. is first to assess basic functional changes of circulatory innate T cells (including MAIT, iNKT and γδ T subsets) as well as frequencies and limited phenotypic analysis of airway innate T cell in critical COVID-19. Further studies including larger cohorts of patients of variable disease are now needed to conclusively determine the role of innate T cells in COVID-19 immunopathology.</p>
<p><em>This review was undertaken by Verena van der Heide as part of a project by students, postdocs and faculty at the Precision Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<p><strong>Title</strong></p>
<p>Evidence of a significant secretory-IgA-dominant SARS-CoV-2 immune response in human milk following recovery from COVID-19</p>
<p>Fox Alisa et al., preprint <em>Medvix</em> <span class="math display">\[doi:
https://doi.org/10.1101/2020.05.04.20089995\]</span></p>
<p><strong>Keywords</strong></p>
<p>Secretory IgA; breast milk; antibodies</p>
<p><span id="main-findings" class="anchor"></span></p>
<p><strong>Main Findings</strong></p>
<p>The immunity of the mucosa between the mother and the newborn against COVID-19 was tested. The secretory antibody -IgA of breastfeeding milk, shown immune response to the Receptor Binding Domain (RBD) of SARS-CoV-2 Spike protein.</p>
<p><span id="limitations" class="anchor"></span><strong>Limitations</strong></p>
<p>Further studies are needed to understand the types of vaccines and routes of administrations in terms of protective antibodies against SARS-CoV-2 in human milk.</p>
<p><strong>Significance</strong></p>
<p>Immune-modulating factors in breast milk may exert a significant impact on the infant’s developing immune system preventing or mitigating SARS-Cov-2 infection.</p>
<p><em>Reviewed</em> Martinez-Delgado Gustavo <em>as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
</div>
<div id="title-bulk-and-single-cell-gene-expression-profiling-of-sars-cov-2-infected-human-cell-lines-identifies-molecular-targets-for-therapeutic-intervention" class="section level1">
<h1 class="hasAnchor">
<a href="#title-bulk-and-single-cell-gene-expression-profiling-of-sars-cov-2-infected-human-cell-lines-identifies-molecular-targets-for-therapeutic-intervention" class="anchor"></a>Title Bulk and single-cell gene expression profiling of SARS-CoV-2 infected human cell lines identifies molecular targets for therapeutic intervention</h1>
<p>Wyler et al. biorXiv <span class="math display">\[@doi: 10.1101/2020.05.05.079194\]</span></p>
<div id="section-3" class="section level4">
<h4></h4>
</div>
<div id="keywords-106" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-106" class="anchor"></a>Keywords</h4>
<ul>
<li><p>scRNAseq</p></li>
<li><p>Interferon-Stimulated-Genes (ISGs)</p></li>
<li><p>HSP90</p></li>
</ul>
</div>
<div id="main-findings-102" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-102" class="anchor"></a>Main Findings</h4>
<p>Wyler et al. performed bulk and single cell RNA sequencing of three human cell lines at different time points after infection with SARS-CoV-1 or SARS-CoV-2. The cell lines used were H1299 and Calu-3, both epithelial lung cancer cell lines, and Caco-2, a colorectal adenocarcinoma cell line. Permissiveness to SARS-CoV-1/2 was different among cell lines: H1299, which express low ACE2 levels, produced less viral RNA and lower yield of infectious virus than Caco-2 and Calu-3.</p>
<p><strong>Bulk RNA-sequencing</strong> showed important differences in host transcriptome responses between the Caco-2 and Calu-3 cell lines. Caco-2 cells exhibited an increase in ER stress genes. In contrast, Calu-3 exhibited a strong induction of Interferon-Stimulated-Genes (ISGs), such as IFNB1, CXCL10, HLA-B, HLA-C. This ISG induction was 2-fold higher for SARS-CoV-2 infection compared to SARS-CoV-1. <strong>scRNAseq from Calu-3 cells confirmed the differential ISGs expression.</strong> Sars-CoV-2 induced higher expression of IFIT1 and IFIT2 than SARS-CoV-1. Only a cluster of SARS-CoV-2 infected cells showed strong IFNB1 induction. RNA velocity analysis, which can measure the amount of nascent RNA, showed that the induction of ISG was short and transient during viral infection, and preceded Nf-kB signaling target genes activation (IL6, TNF, NFKB1A). A minor increase of ACE2 expression was also noted.</p>
<p>To detect subtler transcriptomic changes not related to the IFN response, the authors analyzed <strong>scRNAseq from H1299 cell line</strong>, which seem less permissive to infection. HSP90 expression correlates with the amount of viral SARS-CoV-2 RNA, but not with SARS-CoV-1 RNA. A similar induction was found in Calu-3 scRNAseq at early time point. Chemical blocking of the HSP-90 pathway in Calu-3 cells upon viral infection led to a strong reduction of viral replication and expression of the pro-inflammatory genes IL1B and TNF, but interestingly, not of IFIT-2.</p>
</div>
<div id="limitations-101" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-101" class="anchor"></a>Limitations</h4>
<p>Although increased transcription of ER stress genes was identified in Caco-2 cells, the authors did not report changes in HSP90 expression in this cell line. This could further indicate whether HSP90 induction is a lung-specific mechanism and could explain COVID19 pathology. Moreover, the relevance of these findings would benefit from the confirmation of HSP90 upregulation in more physiological systems such as primary cells or tissue derived from patients. Furthermore, validation of the role of HSP90AA1 and specificity of 17-AAG using HSP90AA1 knock-out cell would further strengthen these results.</p>
<p>The authors correlate the low susceptibility of H1299 with lower expression of ACE2, but scRNAseq data of H1299 indicates that the majority of cells are infected. Therefore, it is unclear what factors are responsible of H1299 relative resistance to infection.</p>
<p>The authors state that the lack of ISGs induction in Caco-3 could be due to a reduced expression of Pattern Recognition Receptors (PRRs) is this cell line. There might be other differences between cells lines that would explain the contrasting results, rather than the PRR expression. To confirm the role of RNA sensors, the authors could perform targeted experiments, such as genetic deletion of PRR pathway for example.</p>
<p><em>Figure 1 and Suppl Figure 1 reference in the text seems to have been mixed (S1D wrongly referred as Fig 1B, S1C as 1C, etc.)</em></p>
</div>
<div id="significance-98" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-98" class="anchor"></a>Significance</h4>
<p>Although this model uses epithelial cancer cell lines, it is of great interest to understand the effect of SARS-CoV-2 infection on lung epithelial cells. Indeed, this study identifies a potential drugable target (HSP90) against SARS-CoV-2 infection, although these findings remains to be confirmed in primary tissues, animal models, or patients. The effect of blocking HSP90 identified here is of important clinical relevance, as it decreases viral replication and the production of cytokines that could be involved in the ARDS pathogenicity.</p>
<p>Analysis of patients with severe COVID-19 showed impaired IFN-I responses compared to mild or moderate cases<sup>1</sup>, and, together with studies using animal models<sup>2</sup>, suggest a central role for the dysregulation of IFN-I signaling in COVID-19 pathology. Activation of IFN signaling upon SARS-CoV-2 infection has been observed in lung epithelial organoids<sup>3</sup>, but other studies indicate a lack of robust IFN type I/III signaling upon SARS-CoV-2 infection compared to influenza A or RSV infection<sup>4</sup>. The possible discrepancies between the results presented in this preprint and other studies could be explained by different experimental settings, such as the use of different time points, cell lines or MOIs, and warrant further investigations.</p>
</div>
<div id="credit-98" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-98" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Emma Risson and Bérangère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p>1. Impaired type I interferon activity and exacerbated inflammatory responses in severe Covid-19 patients | medRxiv. Accessed May 12, 2020. <a href="https://www.medrxiv.org/content/10.1101/2020.04.19.20068015v1" class="uri">https://www.medrxiv.org/content/10.1101/2020.04.19.20068015v1</a></p>
<p>2. Boudewijns R, Thibaut HJ, Kaptein SJF, et al. STAT2 signaling as double-edged sword restricting viral dissemination but driving severe pneumonia in SARS-CoV-2 infected hamsters. <em>bioRxiv</em>. Published online April 24, 2020:2020.04.23.056838. <a href="doi:10.1101/2020.04.23.056838" class="uri">doi:10.1101/2020.04.23.056838</a></p>
<p>3. Ravindra NG, Alfajaro MM, Gasque V, et al. <em>Single-Cell Longitudinal Analysis of SARS-CoV-2 Infection in Human Bronchial Epithelial Cells</em>. Microbiology; 2020. <a href="doi:10.1101/2020.05.06.081695" class="uri">doi:10.1101/2020.05.06.081695</a></p>
<p>4. Blanco-Melo D, Nilsson-Payant BE, Liu W-C, et al. SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems. <em>bioRxiv</em>. Published online March 24, 2020:2020.03.24.004655. <a href="doi:10.1101/2020.03.24.004655" class="uri">doi:10.1101/2020.03.24.004655</a></p>
<p>5. Zheng H-Y, Zhang M, Yang C-X, et al. Elevated exhaustion levels and reduced functional diversity of T cells in peripheral blood may predict severe progression in COVID-19 patients. <em>Cell Mol Immunol</em>. 2020;17(5):541-543. <a href="doi:10.1038/s41423-020-0401-3" class="uri">doi:10.1038/s41423-020-0401-3</a></p>
<p><em>Single-cell longitudinal analysis of SARS-CoV-2 infection in human bronchial epithelial cells</em></p>
<p>Ravindra et al. <em>bioRxiv</em> <span class="math display">\[@doi:
https://doi.org/10.1101/2020.05.06.081695\]</span></p>
</div>
<div id="keywords-107" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-107" class="anchor"></a>Keywords</h4>
<ul>
<li><p>single-cell RNAseq,</p></li>
<li><p>bronchial epithelial cells</p></li>
<li><p>IL-6</p></li>
<li><p>ACE2</p></li>
</ul>
</div>
<div id="main-findings-103" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-103" class="anchor"></a>Main Findings</h4>
<p><span id="limitations" class="anchor"></span>To study the cell tropism of SARS-CoV-2 and the changes in molecular program of infected cells, the authors infected organotypic human bronchial epithelial cells and analysed them by single-cell RNAseq and electron microscopy over a course of three days. In their organotypic model consisting of ciliated cells, basal cells, club cells, goblet cells, neuroendocrine cells, ionocytes, and tuft cells, they found that ciliated cells were the primary target although basal and club cells were also susceptible. All susceptible cells expressed the ACE2 receptor, but the expression level of ACE2 did not correlate with the increased susceptibility of ciliated cells. scRNAseq also revealed that while only infected cells produced type I and III IFN, this IFN induced ISG production in uninfected bystander cells as well. Pro-inflammatory IL-6 was specifically produced by infected cells supporting a recent study from Xu et al. (PNAS, 2020) showing the efficacy of tocilizumab in relieving COVID-19 symptoms. Lastly, analysis of the transcriptome revealed that infection of ciliated cells upregulated genes involved in apoptosis and viral gene expression while downregulating genes involved in cilia function and homeostasis.</p>
</div>
<div id="limitations-1" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-1" class="anchor"></a>Limitations</h4>
<p>The authors did not address how well the organotypic bronchial epithelial model reflects pathogenesis in vivo. The relevance of this model would be strengthened with a comparison of clinical samples even if the early time points are not captured. Additionally, cells were observed for up to 3 days post infection so further transcriptional changes that may occur at the height of infection would be missed.</p>
</div>
<div id="significance-99" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-99" class="anchor"></a>Significance</h4>
<p><span id="credit" class="anchor"></span>This study offers in-depth view into tropism of SARS-CoV-2 beyond the expression of ACE2. The identification of ciliated cells as primary targets has implications in the treatment of respiratory symptoms of Covid-19 patients. The organotypic model may be a promising tool for future studies as it recapitulates the findings from scRNAseq of primary human tissues showing the expression of ACE2 in various cells like nasal ciliated cells (<em>Nat Med,</em> 2020). Thus far, human blood vessel and kidney organoids have proven useful for demonstrating the potential of soluble recombinant ACE2 to inhibit SARS-CoV-2 infection (Monteil et al., <em>Cell</em> 2020).</p>
</div>
<div id="credit-1" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-1" class="anchor"></a>Credit</h4>
<p><em>Reviewed by </em>Dan Fu Ruan, Evan Cody and Venu Pothula* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><strong>Title:</strong> OpenSAFELY: factors associated with COVID-19-related hospital death in the linked electronic health records of 17 million adult NHS patients</p>
<p>OpenSAFELY Collaborative <em>et al.</em> <span class="math display">\[medRxiv, @doi:
10.1101/2020.05.06.20092999\]</span></p>
<p><strong>Key words:</strong> risk factors, ethnicity, electronic health records</p>
<p><strong>Main Findings:</strong></p>
<p>During the unprecedented COVID-19 pandemic, identifying patients at high risk for mortality is critical so as to guide clinical decisions on early intervention and patient care. To identify factors associated with risk of death from COVID-19, the study developed a secure and pseudonymized analytics platform, OpenSAFELY, that links the UK National Health Service (NHS) patient electronic health records (EHR) with COVID-19 in-hospital death notifications. This platform enabled the rapid analysis of by far the largest cohort to date from any country, comprising 17,425,445 multi-ethnic adults and 5,683 COVID-19 deaths. The analyses were based on hazard ratio generated by cox-regression and were adjusted for demographics and co-morbidities.</p>
<p>Increased risk of COVID-19 hospital death was associated with male gender, older age, certain clinical conditions (uncontrolled diabetes, severe asthma, other respiratory diseases, history of haematological malignancy or recent non-haematological cancer, obesity, cardiovascular disease, kidney, liver, neurological diseases, autoimmune conditions, organ transplant and splenectomy). Notably, the association of asthma with higher risk of COVID-19 related death is contradictory to previous findings of no increased risk of death or even protective association. This effect was even stronger with recent use of oral corticosteroids (i.e. more severe asthma). In addition, people of lower socio-economic background (i.e. deprivation) or black and Asian origin were identified at high risk. However, this association could not be entirely attributed to pre-existing health conditions or other risk factors, which warrants further exploration into drivers of these associations. The open source analytics code is available at OpenSAFELY.org.</p>
<p><strong>Limitations:</strong></p>
<p>1) There are few drawbacks in data source and collection. The study did not account for COVID-19 deaths in false-negative/ untested individuals, relied on EHR from specific software and dealt with incomplete EHR information.</p>
<p>2) Additional discussion regarding the reasons behind the associations would be insightful. Specifically, recent studies have shown that risk factors including asthma, hypertension, and diabetes impact the expression of ACE2 gene, which is the entry receptor for SARS-CoV-2. However, while asthma with type 2 inflammation has been associated with lower ACE2 expression and thereby potentially protective effect, this association has not been observed for nonatopic asthma in these studies. In the current study, asthma is only categorized in terms of severity (recent oral corticosteroid use vs not). Further categorization in terms of subtype would have been helpful.</p>
<p>3) Understanding the underlying causes of high risk in people of black and Asian origin is important for public health and mitigation of the spread. In this study, the most common assumptions of high burden of underlying comorbidities and lower socio-economic status are shown to contribute only partly to the risk. However, other factors, such as occupational exposure, neighborhood and household-density and possible influence of genetic or other biological factors still need to be explored.</p>
<p>4) The study suggests increased risk in former smokers and slight protective effect in current smokers. More in-depth analyses into whether the protective effect of current smoker status is an artifact of over-adjustment, selection protocol of healthy controls or a true correlation are needed.</p>
<p>5) It would be helpful to have the <em>p</em>-value along with the reported hazards ratio and 95% confidence intervals.</p>
<p><strong>Significance:</strong></p>
<p>Overall, the OpenSAFELY platform allows secure and real-time analyses of clinical data stored <em>in situ</em>. As this global pandemic progresses, outcomes and data are expected to expand, revealing more insights to the effects of medical treatments and less common risk factors on COVID-19 infection, spread and death. This approach can help better identify additional factors that affect disease severity and immune response. Finally, this rapid and massive study was only possible because of the detailed longitudinal data already available through General Practitioners within the UK National Health Service (NHS), replication of which at a similar massive scale would be daunting within the highly fragmented healthcare system of the USA. While even within UK NHS many data integration issues remain, the findings from this study is a testament to the global model we need to follow to increase our power to rapidly answer crucial questions related to COVID-19 epidemiology. Such an approach will also open new avenues for increased understanding of other diseases.</p>
<p><em>Reviewed by Myvizhi Esai Selvan as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai</em></p>
<p><strong>The RBD of the spike protein of SARS-group coronaviruses is a highly specific target of SARS-CoV-2 antibodies but not other pathogenic human and animal coronavirus antibodies</strong></p>
<p>Premkumar L et al., medRxiv 2020.05.06.20093377; doi: <a href="https://doi.org/10.1101/2020.05.06.2009337" class="uri">https://doi.org/10.1101/2020.05.06.2009337</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>SARS-CoV-2 receptor binding domain (RBD) binding antibodies</p></li>
<li><p>Endemic human coronaviruses</p></li>
<li><p>Cross-reactive abs/ELISA </p></li>
</ul>
<p><strong><em>Main findings</em></strong></p>
<p> </p>
<p>There is an urgent need for both sensitive and specific SARS-CoV-2 serological testing to not only reliably identify all infected individuals regardless of clinical symptoms, but to determine the percentage of convalescent individuals on population level.</p>
<p>In this preprint, Premkumar et al. study the performance of the SARS-CoV-2 receptor binding domain (RBD), which has been found to be largely unique across individual coronaviruses, as a target to specifically detect antibodies against SARS-CoV-2. By generation of recombinant RBDs of SARS-CoV-1, SARS-CoV-2 and human endemic coronaviruses (hCoV HKU-1, OC-43, NL63 and 229E), antigen cross-reactivity of these targets was evaluated by ELISA, using sera obtained from both infected and convalescent COVID-19 patients, healthy control individuals as well as pooled sera collected from various animals immunized with either SARS-CoV-1, SARS-CoV-2 or animal coronaviruses. While sera from mice and rabbits previously exposed to SARS-CoV-1 spike protein were found to be cross-reactive, recognizing both the SARS-CoV-1 and SARS-CoV-2 RBDs (yet none of the hCoVs), serum from SARS-CoV-2-immune mice predominantly reacted with SARS-CoV-2. Importantly, control sera obtained from healthy donors without a prior history of either SARS-CoV-1 or SARS-CoV-2 were found to only detect hCoV-RBDs. Additionally, assessment of highly concentrated sera collected from 20 healthy donors in the US prior to emergence of the SARS-CoV-2 pandemic confirmed high levels of antibodies against hCoVs in the majority of subjects, whereas cross-reactive antibodies against the SARS-CoV1 and SARS-CoV-2 RBD could not be detected. Furthermore, serological testing of sera obtained from convalescent Dengue and Zika virus patients (n=40) as well as from recently recovered patients with influenza A (n=2) and respiratory syncytial virus (n=1) demonstrated frequent antibodies against hCoV RBDs, but a lack of cross-reactive antibodies against both the SARS-CoV-1 and SARS-CoV-2 RBD as opposed to positive controls of pooled sera from SARS-CoV-1 immunized guinea pigs. Notably, sera from recently recovered, PCR-confirmed hCoV patients (NL-63, OC-43, HKU-1; n=2 each) were equally not cross-reactive against either the SARS-CoV-1 or SARS-CoV-2 RBDs (again using guinea pigs immunized with SARS-CoV-1, SARS-CoV-2 or various animal coronaviruses as positive and negative controls), suggesting that the SARS-CoV-2 RBD is a highly specific target for serological SARS-CoV-2 testing. Furthermore, assessment of total Ig as well as IgM binding to recombinant SARS-CoV and hCoV RBDs in 77 samples obtained from 70 PCR-confirmed COVID-19 patients of variable clinical disease revealed high sensitivity (Ig: 98%, IgM: 81%) for specimens collected at least 9 days post symptom onset. Of note, 67% (Ig) and 30% (IgM) of these samples were also found to be cross-reactive against the SARS-CoV-1 RBD. Repeated sampling of 14 of these 77 patients suggested that seroconversion had occurred between day 7 and day 9 post symptom onset. In addition, 19/77 patients were tested for development and kinetics of neutralizing antibodies (nAbs). Notably, 14% of these 19 patients had detectable levels of nAbs by day 7, whereas 95% of them were positive for nAbs by day 9. One patient failed to elicit both anti-RBD binding and nAbs. Finally, a robust correlation between levels of RBD binding Ig and IgM as well as nAbs was detected, suggesting that levels of RBD binding Abs in COVID-19 patients might be used as a correlate for the development of potentially protective nAbs.           </p>
<p> </p>
<p><strong><em>Limitations</em></strong></p>
<p> </p>
<p>While this is generally a well-conducted study, interrogating a relatively large number of COVID-19 patient and healthy control samples as well as sera from immunized animals, one limitation pertains to the patient cohort enrolled. Given that clinical disease might directly relate to Ab titers, as has been observed in SARS-CoV-1 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683413/pdf/nihms109289.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683413/pdf/nihms109289.pdf</em></a>) and has also been suggested in SARS-CoV-2, a more stringent characterization of these patients would add further impact to the observations made here. Moreover, inclusion of COVID-19 patients across all clinical stages and after convalescence as well as longitudinal sampling over several months and longer are needed to further assess serological testing sensitivity and specificity of RBD-binding Abs and whether the latter may be used as a correlate for potentially protective nAb titers. Additionally, detection of other binding Abs against N, M, S1, S2 may add valuable information, in particular with respect to individuals who seemingly fail to develop humoral anti-SARS-CoV-2 RBD responses. Likewise, evaluation of and comparison to other highly specific epitopes such as ORF3b and ORF8, as recently suggested by another preprint (<a href="https://www.medrxiv.org/content/10.1101/2020.04.30.20085670v1"><em>https://www.medrxiv.org/content/10.1101/2020.04.30.20085670v1</em></a>), might be helpful to rule out seroconversion failure. Notably, the authors report SARS-CoV-2 RBD binding Ab cross-reactivity against SARS-CoV-1 in some COVID-19 patients, an observation that mirrors previous findings about S-binding Abs in several preprints/publications. Given the rather low number of SARS-CoV-1 convalescent patients in the general population, this is likely not a major issue. However, for future clinical application, additional use of potentially even more specific Abs e.g. against ORFs might be favorable. </p>
<p> </p>
<p><strong><em>Significance</em></strong></p>
<p>In general, this study corroborates previous reports and observations about enhanced specificity of the SARS-CoV-2 RBD over other binding ab epitopes (cf. <a href="https://www.nature.com/articles/s41591-020-0913-5"><em>https://www.nature.com/articles/s41591-020-0913-5</em></a>; <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7189648/pdf/eurosurv-25-16-3.pdf"><em>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7189648/pdf/eurosurv-25-16-3.pdf</em></a>; <a href="https://wwwnc.cdc.gov/eid/article/26/7/20-0841_article"><em>https://wwwnc.cdc.gov/eid/article/26/7/20-0841_article</em></a>). Most importantly, these data suggest that pre-existing binding Abs against endemic human coronaviruses seem to be not cross-reactive against the SARS-CoV-2 RBD and that titers of anti-RBD binding Abs robustly correlate with nAb levels. These observations are of great relevance but need further assessment in larger studies of hCoV seropositive and SARS-CoV-2 negative healthy donors. </p>
<p><strong><em>References</em></strong></p>
<p>1. Chris Ka-fai Li <em>et al.</em> T Cell Responses to Whole SARS Coronavirus in Humans. The Journal of Immunology. 2008, 181 (8) 5490-5500; DOI: 10.4049/jimmunol.181.8.5490</p>
<p>2. Yap <em>et al.</em> Patient-derived mutations impact pathogenicity of SARS-CoV-2. PrePrint DOI:</p>
<p><a href="https://doi.org/10.1101/2020.04.14.20060160"><em>https://doi.org/10.1101/2020.04.14.20060160</em></a></p>
<p>3. Perera <em>et al</em> . Serological assays for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), March 2020. Euro Surveill. 2020;25(16):pii=2000421. <a href="https://doi.org/10.2807/1560-7917" class="uri">https://doi.org/10.2807/1560-7917</a>. ES.2020.25.16.2000421</p>
<p>4. Amanat <em>et al.</em> A serological assay to detect SARS-CoV-2 seroconversion in humans. <em>Nat Med</em> (2020). <a href="https://doi.org/10.1038/s41591-020-0913-5"><em>https://doi.org/10.1038/s41591-020-0913-5</em></a></p>
<p>5. Okba <em>et al.</em> Severe acute respiratory syndrome coronavirus 2−specific antibody responses in coronavirus disease 2019 patients. Emerg Infect Dis. 2020 Jul <span class="math display">\[*date cited*\]</span>. <a href="https://doi.org/10.3201/eid2607.200841"><em>https://doi.org/10.3201/eid2607.200841</em></a></p>
<p><em>This review was undertaken by Verena van der Heide and Zafar Mahmood as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
<div id="title-1" class="section level3">
<h3 class="hasAnchor">
<a href="#title-1" class="anchor"></a>Title</h3>
<p><span id="keywords" class="anchor"></span><strong>Serum protein profiling reveals a landscape of inflammation and immune signaling in early-stage COVID-19 infection</strong></p>
<p><em>Keywords</em></p>
<ul>
<li><p><strong>Serum Profiling</strong></p></li>
<li><p><strong>Cytokines</strong></p></li>
<li><p>Protein array</p></li>
<li><p><strong>CCL2</strong></p></li>
<li><p><strong>CXCL10</strong></p></li>
</ul>
<p><span id="main-findings" class="anchor"></span><em>Main Findings</em></p>
<p>In this preprint Hou <em>et al.</em>, analyze serological immune mediators and other proteins from individuals with early COVID-19 symptoms using an antibody microarray that detects 532 target proteins. Patients were classified as COVID-19 (n=13) or influenza (n=15) based on positive RT PCR test for SARS-CoV-2 RNA (COVID-19) or FluA, FluB, RSV RNA (all classified as influenza group).</p>
<p>88 up-regulated and 37 down-regulated proteins were identified by comparing COVID-19 and influenza patient groups (p-value &lt; 0.05). Some of those up-regulated ytyl34proteins were reported before, such as IFN-𝜸, IL-6, CXCL8, CCL2, CXCL10 and some that were not previously associated with COVID-19, such as IL-20, CCL27 and IL-21. Complement proteins C1R and C7, as well as PLG were found to be reduced in COVID-19 patients.</p>
<p>After performing a correlation analysis of the differentially upregulated proteins and clinical data, the authors found a positive correlation between expression of several proteins in the CCL2 and CXCL10 signaling pathways and clinical parameters typically related to liver and renal function, myocardial injury, inflammation and infection, as well as neutrophils counts. Conversely, most of these same proteins show a negative correlation with lymphocyte counts.<span id="limitations" class="anchor"></span></p>
<p><em>Limitations</em></p>
<p>The control “influenza group” had patients negative for influenza virus but positive for RSV, and thus the nomenclature should be revised. As noted by the authors, a larger cohort should be used to validate findings. There is no multivariate analysis, and identification of independent or confounding variables. Even though several proteins were differentially expressed between the patient groups, there was significant overlap between the groups, which may preclude the use of any single protein as a biomarker. Data on clinical variables should have been included in main figure. It is unclear why the authors annotated serum proteins in cellular components. Proteins in CCL2 and CXCL10 interaction network are largely overlapping, but the authors do not emphasize it.</p>
<p>Authors didn’t show lymphocyte and neutrophil counts. Since it is known that severe COVID-19 patients can present lymphopenia and neutrophilia, it would be important to have this information in their cohort as they are correlating protein levels with neutrophils and lymphocytes.</p>
<p>Authors claim that CCL2 can act as an autocrine factor that promotes viral replication in infected macrophages, and cite one paper with HIV, but authors should discuss that this chemokine recruits monocytes to the site of infection and then could contribute to the increased inflammatory response related to COVID-19.</p>
<p><em>Significance</em></p>
<p>This study shows the potential use protein array to simultaneously identify many different proteins in serum of COVID-19 patients.</p>
<p><span id="credit" class="anchor"></span>The authors identify several differentially expressed proteins (potential biomarkers) and correlate them with clinical indices that give insights on possible therapy targets in COVID-19.</p>
<p><em>Credit</em></p>
<p><em>Reviewed </em>by Alessandra Soares-Schanoski* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><strong>Title: </strong></p>
<p>Higher Air Temperature, Pressure, and Ultraviolet Are Associated with Less Covid-19 Incidence</p>
<p><strong>Doi:</strong> 10.1101/2020.05.09.20096321</p>
<p><strong>Keywords: </strong></p>
<p>Covid-19 incidence, air temperature, air pressure, ultraviolet index</p>
<p><strong>The main finding of the article: </strong></p>
<p>Recent studies have diverged as to weather conditions are allied or not with the spreading of Covid-19. Through random-effects meta-regression analysis, this work aimed was to determine if elements linked to meteorology can influence SARS-CoV-2 incidence and the speed of its propagation. The number of Covid-19 patients and meteorological conditions at each Japanese prefectural capital city from January to April 2020 were collected.</p>
<p>The results demonstrated a negative association between Covid-19 incidence and monthly mean air temperature (C°) (coefficient -0.351), sea level air pressure (hPa) (coefficient -0.001) and the monthly mean daily maximum UV index (UV) (coefficient -0.001).</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The manuscript would benefit from a more thorough introduction and discussion of the results in the context of previous studies. The authors could explore more the results of the supplementary table 1 (wind speed, relative humidity and sunshine). The figure caption should be better detailed, explaining the characteristics of each graph.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>The transmission dynamics of SARS-CoV-2 depends on different factors, such as population density, demographic and clinical characteristics of the population, hygiene, local ventilation, etc., and the seasonality of SARS-CoV-2 is not yet known.</p>
<p>The data of this manuscript suggest that higher air temperature, air pressure, and ultraviolet are associated with a lower incidence of Covid-19. Certainly, this study is a step in identifying which environmental factors can favor viral transmission.</p>
<p><strong>Outbreak of Kawasaki disease in children during COVID-19 pandemic: a prospective observational study in Paris, France</strong></p>
<p>Julie Toubiana et al. <em>medRxiv</em> <span class="math display">\[@doi: 10.1101/2020.05.10.20097394\]</span></p>
<div id="keywords-108" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-108" class="anchor"></a>Keywords</h4>
<ul>
<li><p>Kawasaki disease</p></li>
<li><p>COVID-19</p></li>
<li><p>Myocarditis</p></li>
</ul>
</div>
<div id="main-findings-104" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-104" class="anchor"></a>Main Findings</h4>
<p>An unusually high incidence of Kawasaki disease was reported in a pediatric center for infectious diseases in France. This is a rare post-viral vasculitis that was been associated with several viruses in the past, including coronaviruses. The authors reported 17 cases over a period of 11 days, in contrast to a mean of 1 case per 2-week period in 2018-2019.</p>
<p>Polymorphous skin rash and bulbar conjunctival injection were the most frequent criteria for diagnosis of Kawasaki disease. The patients had a median age of 7.5 years (range 3-16); 65% (n=11) presented with shock syndrome, and 70% of the patients (n=12) had concomitant myocarditis. All patients had high inflammatory parameters, including leukocytosis with a predominance of neutrophils, and high levels of C-reactive protein, procalcitonin and interleukin-6. Compared to past descriptions of Kawasaki disease, this cohort had an 8-fold increase in procalcitonin level, what suggests a particularly strong post-viral immunological reaction to SARS-CoV-2 as compared with other viral agents.</p>
<p>Remarkably, although the study was conducted in France, 59% of the patients were originally from sub-Saharan Africa or Caribbean islands, and 12% from Asia, pinpointing a possible genetic predisposition or a travel-associated exposure.</p>
<p>In 82% of the cases, IgG antibodies for SARS-CoV-2 were detected, suggestion an association with coronavirus disesase 2019 (COVID-19). RT-PCR testing for SARS-CoV-2 was positive in 41% of the patients. Although only 6 patients had recent history of an acute respiratory infection, in 9 cases there was history of recent contact with family members displaying respiratory symptoms. However, all patients had gastrointestinal symptoms prior to the onset of Kawasaki disease signs.</p>
<p>All patients were treated with intravenous immunoglobulin (IVIG) and aspirin. Some received concomitant corticosteroids (n=3) and/or broad-spectrum antibiotics (n=14). Admission to intensive care unit (ICU) was necessary in 13 cases. A total of 5 patients had IVIG resistance. Regarding the outcome, 5 patients had not yet been discharged by the time the manuscript was published.</p>
</div>
<div id="limitations-102" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-102" class="anchor"></a>Limitations</h4>
<p>This was a single-center study with a very short follow-up period of 11 days. The information about the total number of paediatric patients that tested positive for SARS-CoV-2 in this center/region during the reported period is missing. That could help to draw conclusions about the incidence of Kawasaki disease-like inflammatory syndromes in children after SARS-CoV-2 infection. Additional to the genetic predisposition hypothesis, information about potential travel-associated exposures should be discussed in the manuscript due to the apparent difference in incidence between racial groups. Furthermore, although the prevalence of COVID-19 in Europe is currently very high, an association between SARS-CoV-2 and the reported outbreak of Kawasaki disease needs further studies to determine causality.</p>
</div>
<div id="significance-100" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-100" class="anchor"></a>Significance</h4>
<p>The temporal association between the COVID-19 pandemic and the results of RT-PCR and antibody testing suggest a causal link between Kawasaki disease and COVID-19. At the time of this writing, while this is not the first description of Kawasaki disease-like inflammatory syndromes in association with COVID-19, it is the largest published cohort. Kawasaki disease should be evaluated as part of the spectrum of post-viral immunological reactions in COVID-19 convalescent children. These findings should prompt a high degree of vigilance among all physicians, and preparedness in countries with a high proportion of children of African and Asian ancestry during the COVID-19 pandemic. The World Health Organization (WHO) has recently developed a case report form and encouraged physicians to report all suspected cases.</p>
</div>
<div id="credit-99" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-99" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p>Low doses of radiation therapy increase the immunosuppressive profile of lung macrophages via IL-10 production and IFNy/IL-6 suppression: a therapeutic strategy to counteract lung inflammation?</p>
<p><strong>Doi:</strong> 10.1101/2020.05.11.077651</p>
<p><strong>Keywords: </strong></p>
<p>Acute respiratory distress syndrome, lung inflammation, low dose irradiation, chest radiation therapy, nerve- and airway-associated macrophages, lung macrophages.</p>
<p><strong>The main finding of the article: </strong></p>
<p>The pathophysiology of severe pneumonia and acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-induced is related with an overproduction of early response proinflammatory cytokines, leading to an increased vascular permeability and high risk of death. Chest radiation therapy using low doses (&lt;1 Gy) was beneficial in the past to treat pneumonia, exerting anti-inflammatory effects in the lung, however, the involved mechanism remained uncertain. The authors proposed that macrophages could be involved in the counteracting lung inflammation after irradiation, and investigated this through in vitro studies with human and mouse lung macrophages, and in vivo pneumonia models in mice.</p>
<p>Normal human lung cells were collected from 3 donors (n=2 to lung cancer, n=1 to benign disease). From these cells, macrophages were isolated and stimulated for 6 hours with Poly(I:C) (1µg/mL) and were irradiated using 0.5 or 1 Gy doses, or non-irradiated. In relation to the murine model, C57BL/6 mice received intratracheally two doses (100µg and 50µg – two consecutive days) of lipopolysaccharide (LPS) or toll-like receptor 3 ligand Poly(I:C). Six hours after the second intranasal dose, 0.5 or 1 Gy of irradiation was applied to the animal thorax. After this procedure, spectral computed tomography (CT) of the mice chest at different time points was performed. Flow cytometry was performed for human lung macrophages and for mouse lung cells. Cytokines concentrations in culture supernatants from <em>in vitro</em> experiments were profiled.</p>
<p>The quantification of supernatants demonstrated that both doses of irradiation, 0.5 and 1Gy, significantly decreased IFNγ and increased IL-10 secretion in human lung macrophages stimulated with Poly(I:C) in comparison to non-irradiated Poly (I:C)-stimulated cells. The flow cytometry analysis showed higher percentage of human lung macrophages producing IL-10 after 0.5 Gy irradiation dose when compared with the other conditions, and decrease IL-6 production after both doses of irradiation in comparison to non-irradiation cells. In the in vivo pneumonia mouse model, the authors showed that low dose thorax irradiation of mice treated with LPS or Poly(I:C), resulted in increased IL-10 production by a distinct group of macrophages dubbed nerve- and airway-associated macrophages (NAMs) compared to non-irradiated mice. In addition, the lung CT scans revealed that lungs irradiated at 1Gy presented less tissue density, indicating lower pulmonary inflammatory process in the lung.</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The data presented in this manuscript is quite limited, its mostly restricted to production of IL-10, IFN-g and IL-6 (one experiment) by lung macrophages without any other functional analysis. The only data on lung inflammation are the CT scans, adding tissue histology would have improved the analysis. The description of experimental details is incomplete, and the number of human tissues analyzed in the <em>in vitro</em> culture is very small. Given its potential importance in the mechanisms of SARS-CoV-2 lung pathology, the role of alveolar macrophages and interstitial macrophages could have been explored in more depth. The authors could had analyzed others pro- and anti-inflammatory markers, like TNFα, IL-1β, IL-1Ra, IL-8, as well as mediators involved in lung inflammation resolution such as VEGF-A. The manuscript has a comprehensive discussion on the effects of chest irradiation on lung inflammation.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>Research indicates that “cytokine storm”, an uncontrolled over-production of inflammatory markers which, in turn, sustain a systemic inflammatory response, is mostly responsible for the occurrence of severe acute respiratory syndrome. This manuscript shows that low dose thorax radiation is able to direct the pro-inflammatory lung cell responses into an anti-inflammatory response. Therefore, this therapy could be useful to mitigate lung inflammatory process in Covid-19. A point to highlight is that side and adverse effects must be well evaluated before a possible irradiation treatment in an infectious condition such as COVID-19.</p>
<p><strong><em>Title:</em></strong> Rapid isolation and profiling of a diverse panel of human monoclonal antibodies targeting the SARS-CoV-2 spike protein</p>
<p><strong><em>Keywords:</em></strong> Coronavirus; SARS-CoV-2; SARS-CoV; COVID-19; Monoclonal antibodies; Adaptive Immunity</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Zost et al. describe the methodology used to efficiently generate a large library of highly-functional monoclonal antibodies directed against the SARS-CoV-2 spike (S) protein. Several different approaches were used to select the antibodies characterized in this study. Briefly, plasma or serum was obtained from four patients infected with SARS-CoV-2, and ELISA binding assays were used to confirm the presence of reactive antibodies to the prefusion ectodomain of either the SARS-CoV-2 or SARS-CoV S protein. Additional screens were used to assess the presence of antibodies capable of binding to the receptor binding domain (RBD) as well as the entire N-terminal domain (NTD) of the SARS-CoV-2 spike protein. The highest reactivity was seen in binding assays when the antigenic targets were the SARS-CoV-2 spike S2P ectodomain or RBD. SARS-CoV-2 S-specific class-switched memory B cells were isolated from peripheral blood mononuclear cells (PBMCs) via flow cytometry. The two patients whose blood was collected at later stages of convalescence displayed higher frequencies of antigen-specific memory B cells and greater levels of neutralizing antibodies. S2P ectodomain- and RBD-specific memory B cells sorted from these two patients PBMCs were pooled and cultured for one week in wells containing a feeder layer of cells expressing CD40L, IL-21, and BAFF. Approximately 50% of these cells were single-cell sequenced for antibody gene synthesis. The other half were placed in a Berkeley Lights Beacon Optofluidic instrument to further identify, select, and export antigen-reactive B cells prior to single cell antibody sequencing and cloning into immunoglobulin expression vectors. Both approaches yielded a combined total of 386 recombinant SARS-CoV-2 reactive human monoclonal antibodies. Subsequent ELISA and neutralization assays were used to separate these antibodies into five classes based on their cross-reactivity with SARS-CoV and the specific binding domains on the SARS-CoV-2 S protein. Bioinformatic analysis of the immunoglobulin sequences revealed a high degree of relatedness to the inferred unmutated ancestor immunoglobulin genes.</p>
<p><strong><em>Critical Analysis:</em></strong></p>
<p>This study characterizes a robust repertoire of SARS-CoV-2 spike-specific antibodies. The authors begin to shed light on the binding sites of these antibodies by describing the domains on the spike protein to which these antibodies react. However, in order to more fully capture the mechanism of neutralization for the leading therapeutic candidates, it will be important to further characterize the specific epitopes and structural binding modes. This is especially important since many of the antibodies identified in this study will not directly interfere with the RBD/ACE2 interaction and therefore likely act through another mechanism such as destabilizing the spike prefusion conformation. Another interesting observation raised in this study is that, as seen with Ebola, patients do not possess a high frequency of memory B cells expressing neutralizing antibodies until later in convalescence. However, given the small number of patients in the study, a larger sample is needed to confirm this conclusion. While this study presents a comprehensive class of candidate antibodies for therapeutic development there is still much needed data describing the protective potential of these antibodies in animal models challenged with SARS-CoV-2, as the authors assert as well. Finally, as synergy has been observed in strong B cell response for other coronaviruses and the fact that antibody cocktails are an effective treatment platform to prevent mutation escape, it would be helpful to know whether specific combinations of these monoclonal antibodies enhance neutralization and in vivo protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>In conclusion, this study presents a robust analysis of the specific B cell response to SARS-CoV-2 in a small number of individuals, and describes practical techniques to isolating a large and diverse panel of human monoclonal antibodies. In addition to revealing potential therapeutic antibody candidates for COVID-19, the authors provide additional information as to the complicated and inconsistent observations of antibody cross-reactivity and cross-neutralization in the context of SARS-CoV and SARS-CoV-2. Information on conserved and highly potent neutralizing targets of antibody responses will be critical down the road as we evaluate the immunogenicity of vaccine candidates. Meanwhile, the information in this study can be directly applied to the therapeutic antibody pipeline for SARS-CoV-2 and the methodologies described here can be adapted for similar emerging pathogens in the future.</p>
<p><strong>Novel ACE2-Independent Carbohydrate-Binding of SARS-CoV-2 Spike Protein to Host Lectins and Lung Microbiota</strong></p>
<p>Chiodo et al.; bioRxiv <span class="math display">\[10.1101/2020.05.13.092478\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>Spike protein</p></li>
<li><p>Lectin</p></li>
<li><p>Siglec</p></li>
<li><p>ACE2</p></li>
</ul>
<p><em>Main findings</em></p>
<p>In addition to its primary target, the ACE2 receptor on host cells, the SARS-CoV-2 spike protein and its glycosylated residues may interact with other carbohydrate-binding receptors on innate immune cells. The results of those interactions have not been well described; activation of myeloid cells via these pathways may contribute to hyper-inflammation and thrombogenesis. In order to describe potential interactions between the glycosylated spike protein and conventional carbohydrate-binding proteins on myeloid cells, Chiodo et al. used solid-phase immunoassays to determine the binding specificity of the spike protein to C-type lectins and Siglecs found on immune cells.</p>
<p>The authors surveyed spike protein binding to dendritic cell-specific intercellular adhesion molecule-3-grabbing non-integrin (DC-SIGN), Langerin, macrophage galactose binding lectin (MGL), the mannose receptor (MR), dectin-1 (used as a negative control), macrophage inducible calcium-dependent lectin receptor (MBL), and the sialic acid-binding immunoglobulin-type lectins (Siglecs-3, 5, 7, 9, 10). The SARS-CoV-2 spike protein showed strong binding to DC-SIGN and MGL, but not to Langerin, MR, and MBL. Of note, DC-SIGN bears an immune-receptor tyrosine-based activation motif (ITAM), suggesting that it is capable of direct signaling. The spike protein also demonstrated selective binding to Siglec-3, 9, and 10, which bear immune-receptor tyrosine-based inhibition motif (ITIM) and have direct immuno-suppressive effects.</p>
<p>The authors extended this method and concept to assess potential interactions between SARS-CoV-2 spike protein and structural components of pathogens that have been cited to contribute to infections secondary to COVID-19. Additional assays using capsular polysaccharides (CPS) of <em>Streptococcus pneumoniae</em> and lipopolysaccharides (LPS) from <em>Pseudomonas aeruginosa</em>, <em>Salmonella typhimurium</em>, and <em>Shigella flexneri</em> revealed that the spike protein can strongly recognize CPS from <em>S. pneumoniae</em> serotypes Sp19F and Sp23F and LPS from <em>P. aeruginosa</em> but not that of the two enteric pathogens.</p>
<p>In summary, these findings point to ACE2-independent recognition of carbohydrate-binding elements on myeloid cells by SARS-CoV-2 spike protein. While the downstream effects of these interactions remain unclear, this study demonstrates that SARS-CoV-2 viral particles have the potential to directly interact with myeloid cells, without using the ACE2 receptor or intracellular TLRs.</p>
<p><em>Limitations</em></p>
<p>All experiments described in this report were <em>in vitro</em> experiments that utilized solid-phase immunoassays. No cellular responses were assessed. Moreover, it is unclear if these interactions are different, if the S protein were derived from infected cells <em>in vivo</em>. Therefore, it is difficult to conclude whether recognition of the SARS-CoV-2 spike protein and its carbohydrate residues by lectins or Siglecs typically found on macrophages and dendritic cells results in an immune response that significantly contributes to COVID-19 immuno-pathology. Therefore, the conclusions concerning the immuno-modulatory impact that these ACE2-independent, carbohydrate-based interactions may have are speculative.</p>
<p><em>Significance</em></p>
<p>It is known that carbohydrate-binding proteins on myeloid cells facilitate the internalization of pathogens for lysosomal degradation. So, the experiments described here may provide some context for how monocytes and their myeloid derivatives, including monocyte-derived DCs and macrophages, could harbor SARS-CoV-2 particles or viral proteins, despite the lack of evidence of ACE2 receptor expression in these cells. Albeit, it has been hypothesized that ACE2 receptor expression can be induced by interferon in mononuclear phagocytes.</p>
<p>In addition to demonstrating that the spike protein is capable of recognizing certain C-type lectins and Siglecs found on monocyte-derived DCs and macrophages, the authors interrogated the ability for the spike protein to interact with polysaccharide components derived from <em>S. pneumoniae</em>, which has been documented as the most common cause of secondary pneumonia in COVID-19 patients. Additional mechanistic studies in cell culture and in <em>in vivo</em> models are required to verify whether these interactions, in fact, contribute to the immuno-pathology of COVID-19 and viral dissemination.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>Differential Redistribution of Activated Monocyte and Dendritic Cell Subsets to the Lung Associates with Severity of COVID-19</strong></p>
<p>Sánchez-Cerrillo et al.; medRxiv <span class="math display">\[10.1101/2020.05.13.20100925\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>disease severity</p></li>
<li><p>monocyte</p></li>
<li><p>dendritic cells</p></li>
<li><p>T cell exhaustion</p></li>
</ul>
<p><em>Main findings</em></p>
<p>While the growing scientific literature on the immune responses to SARS-CoV-2 infection has highlighted several immunological markers for COVID-19, molecular or cellular determinants of disease severity have not yet been well-described. In this report, Sánchez-Cerrillo et al. profiled myeloid and T cell subsets across mild (G1, n=19; whole blood), severe (G2, n=21; whole blood), and critical COVID-19 cases (G3, n=23; whole blood and paired bronchoscopy samples), and healthy controls (n=22). Clinical parameters, including serum IL-6, procalcitonin (PCT), C-reactive protein (CRP), D-dimer levels, and serum LDH, increased with worsening disease severity.</p>
<p>Using high-dimensional flow cytometry, the authors assessed changes in classical monocytes (C Mo; CD14<sup>+</sup>CD16<sup>-</sup>), transitional monocytes (T Mo; CD14<sup>+</sup>CD16<sup>+</sup>), and non-classical monocytes (NC Mo; CD14<sup>lo</sup>CD16<sup>+</sup>), CD14<sup>-</sup>CD16<sup>hi</sup>HLA-DR<sup>-</sup> granulocytes, CD141<sup>+</sup> dendritic cells (cDC1), CD1c<sup>+</sup> dendritic cells (cDC2), and CD123<sup>hi</sup> dendritic cells (pDC) in blood and bronchoscopy samples. While almost all myeloid subsets in COVID-19 patients were significantly reduced in the blood circulation compared to healthy controls (with the exception of T Mo), no statistically significant correlations between these myeloid subsets and disease severity were observed. Of note, the overall sparsity of C and T Mo subsets corresponded to high levels of serum IL-6; otherwise, there were no remarkable correlations between the frequencies of the aforementioned subsets and inflammatory markers. Importantly, in the bronchoscopy samples, an unpaired analysis identified an enrichment of granulocytes and inflammatory T and NC Mo. Importantly, a paired analysis of blood and lung samples demonstrated that T, NC, and CD1c<sup>+</sup> DCs are significantly enriched in the lung. Collectively, these results reflect a notable recruitment of monocytes to the lung. The authors used CD40 expression as a marker of myeloid activation. While CD40 expression decreased with increasing disease severity, this trend was not significant, and expression was comparable to the cells isolated from healthy controls. Lastly, a survey of markers associated with compromised effector function of T cells isolated from blood and bronchoscopy samples of G3 patients showed that CD38<sup>+</sup>CXCR5<sup>+</sup> T cells are significantly more prevalent in the lungs than in the blood, and differences to healthy controls were significant.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>One notable limitation are superinfections as a confounding variable; their effects need to be accounted for with careful multi-variate analysis and should be replicated in larger, multicenter studies. Moreover, flow cytometry markers used in the present study can present a biased view of cell populations, so future studies using higher-dimensional, unbiased techniques may provide a more inclusive view of myeloid heterogeneity in COVID-19 patients with differing severities of disease.</p>
<p><em>Biological</em></p>
<p>It is important to note that almost all patients across the different groups had been receiving concurrent therapy, including antivirals, antibiotics, steroids, and immuno-modulators (anti-IL-6); it is unclear when these treatments were administered, relative to the collection of samples. Furthermore, the DC subsets defined in this report comprised significantly small proportions (&lt; 5%) of total CD45<sup>+</sup> immune cells isolated from blood and bronchoscopy samples of COVID-19 patients. Lastly, while T cell exhaustion was evaluated based on expression of CD38 and CXCR5, the expression of other, more prominent co-inhibitory receptors, including PD-1 or Tim-3, was not evaluated. Therefore, this report would benefit from a better study of myeloid activation and T cell exhaustion using additional markers that define activation of the myeloid subsets, including an analysis of cytokine production, and markers for T cell exhaustion.</p>
<p><em>Significance</em></p>
<p>In summary, this report offers some insight into the profiling of different circulating cell subpopulations across various degrees of COVID-19 severity. However, interpretations of the results should be approached with caution, given the lack of statistical significance and of detailed analyses of important cell groups, including better-defined exhausted T cells. However, thus far, the findings outlined in this report support the notion that monocyte dysfunction, involving a pro-inflammatory state and an overall recruitment from the peripheral blood to disease-afflicted tissues like the lung, characterizes the immune response to COVID-19.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>Pre-existing and de novo humoral immunity to SARS-CoV-2 in humans</strong></p>
<p>Ng, K., Nikhil, F., Cornish, G., Rosa, A., et al.,; bioRxiv 2020.05.14.095414; doi: <a href="https://doi.org/10.1101/2020.05.14.095414" class="uri">https://doi.org/10.1101/2020.05.14.095414</a></p>
<p><strong><em>Keywords</em></strong></p>
<ul>
<li><p>Humoral immune response</p></li>
<li><p>Human endemic coronaviruses</p></li>
<li><p>COVID-19</p></li>
</ul>
<p><strong><em>Main findings</em></strong></p>
<p>Using a novel flow-cytometry-based assay, Ng et al. analyzed anti-SARS-CoV-2 S-protein binding antibodies in sera collected from COVID-19 patients between 2 to 43 days after symptom onset (SARS-CoV-2+; pilot cohort n=35, extended cohort n=135; i.e. n=170 total), from SARS-CoV-2-negative individuals with recent PCR-confirmed human endemic coronavirus infection (SARS-CoV-2-/HCoV+; n=34) or SARS-CoV-2-negative donors of unknown HCoV-status (SARS-CoV-2-; n=30 unspecified and n=50 pregnant donors; i.e. n=80 total) as well as from both SARS-CoV-2- and HCoV-negative healthy controls (SARS-CoV-2-/HCoV-; n=31). With the exception of 6 patients in the pilot group, presumably sampled prior to seroconversion, the majority of COVID-19 sera at later collection time points contained uniformly high levels of concurrent IgG, IgA, and IgM antibodies binding to HEK293T cells transiently transfected with the SARS-CoV-2 S-protein. Of note, anti-S IgG, but not IgA or IgM, binding antibodies could also be detected, albeit at lower levels, in 8 of 95 samples from SARS-CoV-2-/HCoV+ and SARS-CoV-2- donors both by FACS assay and using a standard anti-SARS-CoV-2 S-protein ELISA for validation. Importantly, in accordance with a recent preprint by Premkumar et al. (<a href="https://doi.org/10.1101/2020.05.06.2009337" class="uri">https://doi.org/10.1101/2020.05.06.2009337</a>) none of these samples were found to be cross-reactive against the highly specific S1 protein or anti-receptor-binding domain (RBD) as determined by respective ELISA assays. Additionally, 7 of these specimen (n=3 anti-S IgG-positive and n=4 anti-S IgG-negative by FACS assay) were confirmed by specific ELISA to also react with the SARS-CoV-2 N-protein, suggesting that a total of 12/95 SARS-CoV-2-negative donors with previous or unknown HCoV history produced cross-reactive IgG binding antibodies against conserved SARS-CoV-2 epitopes. This observation was additionally corroborated by similar findings in a second cohort of SARS-CoV-2- patients with unknown HCoV history (n=50), revealing that approximately 10% of these donors had pre-existing antibodies that were cross-reactive against conserved SARS-CoV-2 proteins. Notably, according to the authors, FACS-based anti-S IgG detection, in particular at earlier time points prior to seroconversion, was seemingly more sensitive compared to standard ELISA assays, and simultaneous expression of anti-S IgG, IgA, and IgM binding antibodies, exclusively limited to COVID-19 patients, was found to persist for at least 43 days. Finally, Ng et al. demonstrate that anti-S IgG binding antibody-containing sera from donors with recent HCoV infections were equally capable of neutralizing lentivirus-pseudotyped SARS-CoV-2 as serum samples obtained from seroconverted COVID-19 patients, suggesting that cross-reactive neutralizing antibodies may interfere with SARS-CoV-2 target cell entry likely mediated by the alternative receptor CD147 (unfortunately, no data shown). Furthermore, levels of neutralizing antibody in SARS-CoV-2-/HCoV+ sera correlated particularly robustly with anti-S IgG binding antibodies detected by FACS assay.</p>
<p><strong><em>Limitations</em></strong></p>
<p>A general limitation of this preprint by Ng et al. is that extended data repeatedly referred to throughout the manuscript is not accessible. It is therefore impossible to assess some of the information provided in text, e.g. about potential advantages of the FACS assay developed here over conventional ELISA assays or regarding ACE2 and CD147 expression on HEK293T cells as well as potential implications for SARS-CoV-2 cellular entry. Furthermore, in addition to the relatively small sample size in Figure 1 (cf. n=2-5 donors per panel), the exact number of donors enrolled per study group as well as of specimens used per individual experiment is somewhat unclear and should be specified: according to the methods section, a total of n=114 SARS-CoV-2 negative donors with recent or unknown HCoV infection were included; yet, throughout the manuscript, the authors repeatedly refer to n=95 donors with diagnosed/potential HCoV-infection as well n=50 pregnant donors of uncertain HCoV-status. Similarly, numbers for both the SARS-CoV-2 negative, potentially HCoV-positive, and the COVID-19 donor groups displayed in Figure 2a-d may seem misleading within the general context of the manuscript (n=35 COVID-19 displayed vs. n=170 COVID-19 mentioned in text, n=64 potentially HCoV-positive donors displayed vs. n=114 mentioned in text). This also seems to be the case for some of the SARS-CoV2-negative donors testing positive for pre-existing cross-reactive antibodies. Given that this is one of the central observations in this preprint, one would expect the respective numbers mentioned in the manuscript to match the data displayed in Figure 2a (n=8 by FACS assay vs. n=5 on display). Moreover, information on clinical course and symptoms of COVID-19 patients would be of particular interest with respect to potential differences in antibody composition related to disease severity as well as in the context of sample timing and viral persistence as based on qPCR detection (in addition to limited information on two donors of interest already discussed in the manuscript). Similarly, to confirm that simultaneous detection of IgG, IgA, and IgM co-expression is indeed highly specific for active/convalescent COVID-19 as suggested by the authors, this should be validated in donors with current HCoV infection or in those recently recovered (i.e. earlier than 3-9 months post HCoV diagnosis as was the case here) to exclude sample bias due to waning HCoV humoral immunity with time in comparison to the very recent COVID-19 patients studied here. Given that the clinical course of HCoV infections might be similar to mild COVID-19 it is critical to determine whether the observations about concurrent expression of IgG, IgA, and IgM will reliably discriminate between active HCoV and SARS-CoV-2 infection. Finally, neutralizing capacity of cross-reactive IgG antibodies eventually needs to be confirmed using live SARS-CoV-2 to better assess the potential for antiviral protection. </p>
<p><strong><em>Significance</em></strong></p>
<p>In this preprint, Ng et al report the flow-cytometry-based detection of pre-existing binding antibodies against conserved SARS-CoV-2 epitopes in around 10% of SARS-CoV-2 negative donors with confirmed or potential previous history of HCoV infections. Notably, binding antibody-containing sera from the same donors were found to possess neutralizing activity comparable to samples obtained from seroconverted COVID-19 patients, suggesting that pre-existing cross-reactive antibodies may interfere with SARS-CoV-2 entry into target cells, likely via alternative receptors such as CD147. Assessing the potential role of both pre-existing binding and neutralizing antibodies in healthy donors in the context of COVID-19 pathogenesis will be of particular importance. The observations made here are also highly relevant for the design and development of reliable diagnostic assays as well as of potential vaccine candidates, and should therefore be further explored in ongoing research on potential coronavirus therapies and prevention strategies.</p>
<p><em>This review was undertaken by Verena van der Heide as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai. </em></p>
</div>
</div>
<div id="deep-immune-profiling-of-covid-19-patients-reveals-patient-heterogeneity-and-distinct-immunotypes-with-implications-for-therapeutic-interventions" class="section level3">
<h3 class="hasAnchor">
<a href="#deep-immune-profiling-of-covid-19-patients-reveals-patient-heterogeneity-and-distinct-immunotypes-with-implications-for-therapeutic-interventions" class="anchor"></a>Deep immune profiling of COVID-19 patients reveals patient heterogeneity and distinct immunotypes with implications for therapeutic interventions</h3>
<p>Mathew D et al. <em>bioRxiv</em> <span class="math display">\[@doi: 10.1101/2020.05.20.106401\]</span></p>
<div id="keywords-109" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-109" class="anchor"></a>Keywords</h4>
<ul>
<li><p>COVID-19</p></li>
<li><p>Lymphocytes</p></li>
<li><p>plasmablasts</p></li>
<li><p>Profiling</p></li>
<li><p>cytometry</p></li>
</ul>
</div>
<div id="main-findings-105" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-105" class="anchor"></a>Main Findings</h4>
<ul>
<li><p>Study profiled immune dysregulation in peripheral blood of COVID-19 patients (n=71) along with recovered donors (RD; n=25) and healthy subjects (HD; n=37) using high dimensional cytometry. Integrated correlation mapping of clinical features (CRP/Ferritin etc) with immune phenotype was carried out, and findings in specific cellular compartments were parsed against pre-existing conditions or treatment modalities.</p></li>
<li><p>CD8 T cell profiling indicated an increase in effector memory 2 (EM2) CD45RA- CD27- CCR7+, and CD45RA+ effector memory (EMRA) CD45RA+ CD27- CCR7- CD8 T cell populations as well as increased frequencies of PD1+ CD39+ non-naïve T cells in ~2/3<sup>rd</sup> of COVID-19 patients compared to HD. This was associated with higher proliferation and HLA-DR, CD38 expression. FlowSOM clustering indicated similarly that ~60% subset of COVID-19 patients had increased TEMRA-like and activated CD8 T cell populations, while the CD45RA-CCR7+ central memory (CM)-like clusters were decreased compared to HD.</p></li>
<li><p>Most major CD8 T cell populations in RDs were comparable to HDs but non-naïve CD8+ T cells in RD expressed higher PD1. High-dimensional tSNE mapping indicated subtle differences in CX3CR1 and Tbet expression.</p></li>
<li><p>CD4 T cell analyses indicated similar increases in EM2/3 and EMRA populations, but no major changes in cTfh cells in COVID-19 patient blood when compared to controls. FlowSOM analyses showed decreases in CD45RA+CCR7+ naïve CD4 T cells but also cTfh-like cells in another subset of COVID-19 patients. In a subgroup of COVID-19 patients and most RD patients, activated cTfh were increased suggesting ongoing or residual germinal center interactions and links to humoral immunity.</p></li>
<li><p>Luminex assays of plasma and paired stimulated-PBMC supernatant showed a subset of COVID-19 patients to have higher CXCL9, CXCL10, CCL2 and ILRA compared to HD. Eosinophil or T cell chemokines such as Eotaxin and CCL5 were decreased in COVID-19 plasma.</p></li>
<li><p>A subset of COVID-19 patients (~2/3) had lower class-switched (IgD-) and non-class-switched (IgD+) memory B cells, increased frequencies of CD27+CD38+ plasmablasts, and higher proliferation in certain memory B cell subsets. There was no clear association between increased plasmablast frequency and cTfh activation status. Here, COVID-19 subjects segregated into 2 groups: one with high plasmablast responses and low CD4 activation, and a second with lower plasmablasts and higher CD4 T cell activation. Clustering COVID-19 patients according to global B cell response resulted majorly in 2 distinct EMD clusters (matching aforementioned categories), and the remainder 30% clustered with HD/RD controls.</p></li>
<li><p>Study observed stable tSNE distributions of immune response (CD8, CD4 and B cell differentiation) over days of hospitalization, but subpopulation analyses indicated a sustained CD8 T cell proliferative response (not CD4, nor plasmablasts) in COVID-19 patients over 7 days in clinic. Using a previously-described dataset to define typical HD-like stability and then mapping variation over time, the study additionally showed ~60% of patients had increases in HLA-DR+CD38+ non-naive CD4 T cells temporally, and ~42% patients displayed sustained plasmablast responses.</p></li>
<li><p>Data clustering based on immune features overlaid with disease severity score over time revealed “immunotype” groups with similar composite signatures, and an UMAP-based meta-analysis showed ‘stepwise’ increase in principal features with progressive disease. These computational approaches underlined 3 archetypes of COVID-19 patients – an immunotype 1, comprised of CD4 T cell activation/proliferation and Tbet+ plasmablasts and linked to disease severity , an immunotype 2 of CD8 T cell EM/EMRA activation and CD138+ki67+ plasmablasts, and finally an immunotype 3 with undetectable T/B cell responses which may represent ~30% of COVID19 patients.</p></li>
</ul>
</div>
<div id="limitations-103" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-103" class="anchor"></a>Limitations</h4>
<ul>
<li><p>Whether all of the observed changes in the T and B cell compartment, including potential germinal center responses, are due to SARS-COV-2 directed responses is unclear and could be verified using either COV-2-tetramers, COV-2-specific ELISAs, or TCR/BCR sequencing analyses. Associative studies of changes in myeloid compartment would also be extremely valuable here.</p></li>
<li><p>It should be noted that the COVID-19 cohort included various pre-existing comorbidities, as well as different treatments with HCQ, steroids or remdesivir. Care was taken to parse many of these variables, but readers should not interpret interventional efficacy from this data.</p></li>
<li><p>As stated by the authors, the model used to define ‘immunotypes’ may benefit from adding additional immune features, such as comprehensive serum cytokine measurements as well as viral titers. Additionally, information on how/if these immunotypes associate with specific COVID-19 symptoms and complications (eg- endothelial dysfunction or myeloid activation) might be enlightening.</p></li>
</ul>
</div>
<div id="significance-101" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-101" class="anchor"></a>Significance</h4>
<ul>
<li><p>This study presents extensive longitudinal profiling of T and B cell compartments in COVID-19 patients and convalescent donors, as well as provide useful correlation-frameworks for defining ‘immunotypes’ or ‘immune trajectory archetypes’ in this disease. Basically, COVID-19 manifests in many different ways immunologically.</p></li>
<li><p>Importantly, this study demonstrates some patients displayed dramatic changes in T cell or B cell activation over 1 week in the hospital, but there were also other patients who remained stable lymphocyte-wise. A deeper dive into characteristics of these ‘lympho-stable’ COVID-19 patients would be critical, especially in terms of the myeloid cell correlates and differential inflammatory cascades.</p></li>
<li><p>There was no clear link between plasmablast responses and follicular CD4 T cell dynamics, suggesting there might not be extensive T cell-dependent humoral immunity in COVID-19. However, more carefully-controlled studies honing on this question are needed to comprehensively resolve this.</p></li>
<li><p>Some ‘lympho-stable’ COVID-19 patients have high and ever-increasing CD8, CD4 T proliferation response, potentially as an emergency mechanism to keep numbers up amidst lymphopenia. The observed activation in almost all non-naïve CD8 subtypes suggests potential bystander activation and/or emergency proliferation globally in addition to the antigen-driven activation of SARS-COV-2-specific CD8 T cells.</p></li>
<li><p>The durable lymphocytic response in COVID-19 contrasts with the short transient peak characteristic of other viral infections (influenza, HIV, yellow fever etc). This incessant T cell proliferation in COVID-19 might give rise to anergy/exhaustion, and might be linked to the delayed Type I/III IFN dynamics in SARS-COV and also SARS-COV-2. Hence, analyzing the temporal pattern of COVID-19 remains critical.</p></li>
</ul>
<p><em>Reviewed by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>The ORF8 protein of SARS-CoV-2 mediates immune evasion through potentially down-regulating MHC-I</strong></p>
<p>Zhang et al.; bioRxiv <span class="math display">\[10.1101/2020.05.24.111823\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>ORF8</p></li>
<li><p>MHC class I</p></li>
<li><p>autophagy</p></li>
<li><p>immune evasion</p></li>
<li><p>T cell</p></li>
</ul>
<p><em>Main findings</em></p>
<p>In this pre-print, Zhang et al. hypothesized that one possible mechanism used by SARS-CoV-2 to participate in efficacious viral spread is an evasion of the immune response by interfering with antigen presentation by infected cells via MHC-I. Of note, this behavior has been previously characterized in other viruses, including HIV-1 and HSV.</p>
<p>The authors identified ORF8 as one such possible mediator of this mechanism. Interestingly, an overexpression of ORF8 in HEK293T cells resulted in a significant down-regulation of MHC I (HLA-A2) expression, as determined by flow cytometry. Using GFP as a molecular marker and HIV-1-derived Nef protein as a positive control, the authors evaluated MHC-I heavy chain and β2M expression as a functional readout of overall MHC-I expression on the control and ORF8-expressing 293T cells. Flow cytometric analysis on these cells, as well as other cell lines that were similarly tested (human fetal colon, human bronchial epithelial, and human liver), showed significant down-regulation of MHC-I, in response to ORF8 overexpression via plasmid transfection. The authors confirmed that 293T cells could harbor ORF8 production by exposing ACE2-expressing 293T cells to a SARS-CoV-2 strain. Importantly, sequences of ORF8 from SARS-CoV-2 and SARS-CoV-1 showed the least homology, suggesting that this may be a potentially novel immune evasion mediator.</p>
<p>To determine the cellular pathways underlying this relationship between ORF8 and MHC-I, the authors used small molecule inhibitor analysis to determine that ORF8-mediated down-regulation of MHC-I expression is facilitated by lysosomal degradation. The authors performed western blot analyses following in-tact lysosomal extraction to show that an enrichment of MHC-I protein is seen in the lysosomes of 293T cells that had been transfected with the ORF8 plasmid. In fact, confocal microscopy of the ORF8-expressing cells showed a co-localization of the MHC-I proteins with LAMP1, a marker for lysosomal membranes. In fact, the authors demonstrated that ORF8 expression also co-localized with MHC-I. Subsequent co-immunoprecipitation experiments confirmed binding of ORF8 with MHC-I complexes in these compartments.</p>
<p>The authors further elucidated this mechanism by determining the route of the endolysosomal pathway used by this mechanism. A series of co-localization/confocal microscopy experiments demonstrated significant co-localization of ORF8 with ER components (CALNEXIN staining) and lysosomes (LAMP1 staining), but not the Golgi or early endosomal vesicles. The inability to counteract the ORF8-mediated down-regulation of MHC-I via knockdown of vesicle-trafficking-related proteins supported this finding. Importantly, the authors also evaluated MHC-I protein ubiquitination as a cause of reduced surface expression; however, knockdown of ER-associated protein degradation genes did not reverse the phenotype. The authors identified, instead, that selective knockdown of autophagy-associated proteins restored MHC-I expression, implicating a role for autophagy in this process.</p>
<p>Finally, the authors tested the potential role of this relationship between ORF8 and MHC-I in immune evasion. Using SSp-1, a predicted potential SARS-CoV-2 epitope, the authors pulsed 293T cells (transfected with empty vector or the ORF8 plasmid) with SSp-1 and exposed these cells to SSp-1-specific CTLs via sensitization of HLA-A2 healthy donor PBMCs with autologous DCs pre-pulsed with SSp-1. The killing assay showed reduce elimination of target ORF8-expressing 293T cells by these CTLs. The authors performed similar subsequent experiments by using CTLs isolated from a recovering COVID-19 patient (selected from among other patients, based on ability for their CTLs to secrete IFN-ɣ, when exposed to S and N protein-derived peptides). When exposed to 293T cells that were pulsed with the same peptide mixture, the SARS-CoV-2-specific CTLs eliminated ORF8-expressing 293T cells at a lower efficacy.</p>
<p><em>Limitations</em></p>
<p>First, many of the experiments involved use of a cell line (HEK293T) that had been transfected with a plasmid to overexpress ORF8. It remains to be seen whether productive infection with a SARS-CoV-2 strain of cells that naturally express human ACE2 results in the same down-regulation of MHC-I.</p>
<p>Of note, the knockdown siRNA experiments investigating the role of the autophagy pathway in ORF8-mediated down-regulation of MHC-I could be bolstered with the use of additional controls for the Western blot analyses, specifically cellular and lysosomal membrane proteins. (Moreover, to provide context, the authors could have also included a list of hits produced by the mass spectrometry analysis.)</p>
<p>Second, the authors do not provide a kinetics analysis that includes an assessment of overall cell death of the transfected or infected 293T cells. Though it is unlikely that we would see significant cell death in the <em>in vitro</em> experiments described in this report (given that the 293T cells were usually transfected by a plasmid), the kinetics of cell turnover may have an impact on the overall contribution of ORF8-mediated down-regulation of MHC-I to successful immune evasion.</p>
<p>Third, the T cell experiments could have been better supplemented by analyses of MHC-I and ORF8 expression by infected cells and non-infected cells of these patients by using FACS-sorted epithelial cells collected from BAL. Moreover, the non-reactive T cells from COVID-19 patients could have been used as controls, alongside the reactive T cells that were used in this study to better assess killing in a controlled experimental design.</p>
<p>Finally, without an <em>in vivo</em> experiment that demonstrates the aforementioned observations, it is difficult to assess whether additional signaling pathways active during viral infection, such as interferon signaling, may influence the relationship between ORF8 and MHC-I.</p>
<p>Generally, the aforementioned experiments could have compared the results of using SARS-CoV-2 ORF8 with using SARS-CoV-1 ORF8 to further demonstrate the specificity of this mechanism to SARS-CoV-2 infection (in addition to the evidence that SARS-CoV-2 and SARS-CoV-1 ORF8 proteins are the least homologous <span class="math display">\[Fig. 1\]</span>).</p>
<p><em>Significance</em></p>
<p>In summary, this report provides a direct mechanistic link between SARS-CoV-2 infection and immune evasion that involves a role for ORF8, a seemingly SARS-CoV-2-specific protein, and MHC-I expression. The results warrant further investigations in an <em>in vivo</em> model to validate the proposed relationship between ORF8 expression in infected cells and MHC-I down-regulation and the subsequent impaired ability for CTLs to kill these infected targets.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>Single-cell RNA-seq and V(D)J profiling of immune cells in COVID-19 patients</strong></p>
<p>Fan et al.; medRxiv <span class="math display">\[10.1101/2020.05.24.20101238\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>single-cell RNA sequencing</p></li>
</ul>
<p><em>Main findings</em></p>
<p>To improve understanding of the cellular changes in the T and B cell compartments of COVID-19 patients, both during and after disease, Fan et al. analyzed lymphocytes isolated from the PBMCs of 4 severe COVID-19 patients (n=4), 6 COVID-19 recovered patients (n=6), and 3 healthy controls (n=3). Of note, 3 recovered patients’ samples were collected 7 days after a negative SARS-CoV-2 test (recovery-early stage; RE) and absence of clinical symptoms, whereas the other 3 samples were collected 20 days after these criteria (recovery-late stage; RL). The authors used single-cell RNA sequencing and single-cell V(D)J sequencing to perform their analysis.</p>
<p>The authors identified 9 classes of T cells, which included 4 sub-classes of CD4<sup>+</sup> T cells and 5 sub-classes of CD8<sup>+</sup> T cells. Not surprisingly, across severe COVID-19 patients, the proportion of T cells was reduced, compared to healthy controls. However, differential gene expression analysis revealed that T cells from severe COVID-19 patients highly expressed inflammatory markers, including <em>IFNG</em> and <em>GZMA</em>. Interestingly, when compared to these patients with active disease, RE samples showed significant enrichment of <em>ICOS</em>+ T<sub>H</sub>2-like follicular helper T cells (T<sub>FH</sub>), whereas RL samples showed a reportedly significant enrichment of a cluster identified as T<sub>H</sub>1 cells, though this result should be revisited for review (See <em>biological</em> limitations). These cell types were, in fact, reduced in severe COVID-19 patients. Generally, these T cells from recovering patients continued to indicate persistent activation and counter-regulation, based on expression of TCR activation-associated genes, including <em>RNF125</em> and <em>PELI1</em>. Subsequent trajectory analyses of transcriptional dynamics indicated transition of effector CD8<sup>+</sup> T cells to central memory T cells in RL patients. Ligand-receptor analysis revealed potential interactions between T<sub>H</sub>1 cells and CD14<sup>+</sup> monocytes in severe COVID-19 patients. Finally, TCR sequencing identified several VJ combinations in high frequencies in severe COVID-19 patients, but not others.</p>
<p>Within the B cell compartments across patients, the authors identified 9 clusters of naive B cells, 2 clusters of memory B cells, 2 clusters of plasma B cells, and a cluster of plasmablasts. Of these clusters, one, in particular, expressed genes characteristic of <em>FCRL5<sup>+</sup></em> atypical memory B cells, which have been described to be induced by viral infections. Interestingly, ligand-receptor analyses of the clusters in each group of patient samples identified different degrees of T<sub>FH</sub> cell and B cell interactions, suggesting different stages of T cell help for B cell activation. Subsequent BCR characterizations revealed the presence of homogenous monoclonal and heterogeneous clonally expanded B cell populations; the latter population exhibited an enrichment of B cell activation genes. The authors, then, compare across patients to evaluate T and B cell clonality based on V(D)J recombination analyses of RE and RL patient samples (See <em>technical</em> limitations).</p>
<p>Interestingly, cytokine expression analysis revealed IL-6 expression by B cells. In contrast, B cells expressed <em>IL12A</em> in RE patients, while effector memory CD8, proliferative CD8, and CD4 T cells and plasma B cells highly expressed <em>IL16</em> in RL patients. The authors report additional cytokine (and cellular) characteristics that distinguish severe COVID-19 patients and recovering patients.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A primary technical limitation is the sample size of this study for each group. There is little clinical information about the patients and no details about disease severity in patients recruited after viral clearance. For example, age and CMV status have a huge impact on the TCR repertoire, therefore clinical data on the different groups should be presented. Moreover, without additional information on the clinical management of the severe COVID-19 patients and what therapies were given to the recovering COVID-19 patients, it is difficult to compare the cellular changes in the immune landscapes of the COVID-19 patients across samples. Longitudinal analysis would have been more informative especially with regards to repertoire analysis and how expanded clones during active infections might differentiate into particular phenotypes after viral clearance.CD8 expression should have been included in the violin plots, as it is usually more robust and reliable than CD4 expression.</p>
<p><em>Biological</em></p>
<p>An immediate concern is whether the authors mis-characterized cluster 13 as a T<sub>H</sub>1 cell cluster. The cluster exhibits a low expression of CD3G and CD4. It’s neighboring clusters within the hierarchy belong to monocyte groups, so it is unexpected that a T cell subtype would be belong to their branch of the hierarchy tree. Consider also cluster 38, which shows more robust expression of CD3G and NKG7 and is arranged with the B cell group.</p>
<p>In addition, the authors did not highlight or discuss expression of co-inhibitory receptors that could elucidate the heterogeneity of T cell differentiation during COVID-19. As a result, it is difficult to truly assess the activation status of the CD8+ cytotoxic T cells and whether there are features of T cell exhaustion.</p>
<p>Finally, the distinction between naïve and some subsets of memory T cells by scRNA analysis can be challenging. It would be important for the authors to explore whether cluster 26, classified as a naïve CD8 T cell cluster predominant in RL group could be actually memory cells. It would have been important to show clonal diversity of the different clusters.</p>
<p><em>Significance</em></p>
<p>In summary, Fan et al. provide a comparative analysis of lymphocyte changes between PBMCs of patients with ongoing COVID-19 progression and of patients recovering from the disease. Using a combination of single-cell RNA sequencing and V(D)J recombination sequencing, the authors describe specific changes in T and B cell subpopulations over the course of early and late-stage recovery.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
</div>
</div>
<div id="title-2" class="section level3">
<h3 class="hasAnchor">
<a href="#title-2" class="anchor"></a>Title</h3>
<p><strong>Mouse model of SARS-CoV-2 reveals inflammatory role of type I interferon signaling</strong></p>
<p>Benjamin Israelow et al. <em>preprint server</em> doi.org/10.1101/2020.05.27.118893</p>
<div id="keywords-110" class="section level4">
<h4 class="hasAnchor">
<a href="#keywords-110" class="anchor"></a>Keywords</h4>
<ul>
<li><p>SARS-CoV-2</p></li>
<li><p>adeno associated virus</p></li>
<li><p>human ACE2 transgenic mouse</p></li>
</ul>
</div>
<div id="section-4" class="section level4">
<h4></h4>
</div>
<div id="main-findings-106" class="section level4">
<h4 class="hasAnchor">
<a href="#main-findings-106" class="anchor"></a>Main Findings</h4>
</div>
<div id="the-manuscript-describes-a-versatile-mouse-model-of-sars-cov-2-based-on-adeno-associated-virus-aav-mediated-expression-of-hace2--the-aav-hace2-model-supported-productive-sars-cov-2-infection-in-the-lung-of-mice-inducing-leukocyte-recruitment-and-mounting-antibody-response-igg-against-the-spike-s-protein-showing-neutralization-capacity-within-4-and-7-days-of-infection--rna-sequencing-analysis-data-reveal-signatures-of-cytokines-and-interferon-stimulated-genes-isgs-with-up-regulation-compared-with-control--neither-type-i-nor-type-ii-nor-type-iii-interferons-were-up-regulated-indicating-a-resemblance-with-published-data-from-the-lungs-of-patients-infected-with-sars-cov-2-blanco-melo-et-al--2020--further-exploration-of-type-i-interferon-signaling-either-interferon-alpha-receptor-ifnar---and-irf37---interferon-regulatory-transcription-factor-37-knout-out-mice-were-transduced-with-aav-hace2-and-infected-with-sars-cov-2--the-results-exhibited-that-despite-similar-kinetics-of-viral-clearance-with-aav-hace2-model-it-was-a-decrease-in-isg-response-suggesting-that-viral-sars-cov-2-replication-is-resistant-to-interferon-signaling--the-authors-concluded-the-aav-hace2-infected-sars-cov-2-model-largely-recapitulated-the-transcriptome-changes-observed-in-the-lungs-of-sars-cov-2-infected-patients-" class="section level4">
<h4 class="hasAnchor">
<a href="#the-manuscript-describes-a-versatile-mouse-model-of-sars-cov-2-based-on-adeno-associated-virus-aav-mediated-expression-of-hace2--the-aav-hace2-model-supported-productive-sars-cov-2-infection-in-the-lung-of-mice-inducing-leukocyte-recruitment-and-mounting-antibody-response-igg-against-the-spike-s-protein-showing-neutralization-capacity-within-4-and-7-days-of-infection--rna-sequencing-analysis-data-reveal-signatures-of-cytokines-and-interferon-stimulated-genes-isgs-with-up-regulation-compared-with-control--neither-type-i-nor-type-ii-nor-type-iii-interferons-were-up-regulated-indicating-a-resemblance-with-published-data-from-the-lungs-of-patients-infected-with-sars-cov-2-blanco-melo-et-al--2020--further-exploration-of-type-i-interferon-signaling-either-interferon-alpha-receptor-ifnar---and-irf37---interferon-regulatory-transcription-factor-37-knout-out-mice-were-transduced-with-aav-hace2-and-infected-with-sars-cov-2--the-results-exhibited-that-despite-similar-kinetics-of-viral-clearance-with-aav-hace2-model-it-was-a-decrease-in-isg-response-suggesting-that-viral-sars-cov-2-replication-is-resistant-to-interferon-signaling--the-authors-concluded-the-aav-hace2-infected-sars-cov-2-model-largely-recapitulated-the-transcriptome-changes-observed-in-the-lungs-of-sars-cov-2-infected-patients-" class="anchor"></a>The manuscript describes a versatile mouse model of SARS-CoV-2 based on adeno-associated virus (AAV)-mediated expression of hACE2. The AAV-hACE2 model supported productive SARS-CoV-2 infection in the lung of mice, inducing leukocyte recruitment and mounting antibody response (IgG) against the spike (S) protein, showing neutralization capacity within 4 and 7 days of infection. RNA sequencing analysis-data reveal signatures of cytokines and interferon-stimulated genes (ISGs) with up-regulation compared with control. Neither type I nor type II, nor type III interferons were up-regulated, indicating a resemblance with published data from the lungs of patients infected with SARS-CoV-2 (Blanco-Melo et al., 2020). Further exploration of type I interferon signaling, either interferon-alpha receptor IFNAR-/- and IRF3/7-/- interferon regulatory transcription factor 3/7 knout out mice were transduced with AAV-hACE2 and infected with SARS-CoV-2. The results exhibited that despite similar kinetics of viral clearance with AAV-hACE2 model, it was a decrease in ISG response, suggesting that viral SARS-Cov-2 replication is resistant to interferon signaling. The authors concluded the AAV-hACE2 infected SARS-CoV-2 model largely recapitulated the transcriptome changes observed in the lungs of SARS-CoV-2 infected patients.</h4>
</div>
<div id="limitations-104" class="section level4">
<h4 class="hasAnchor">
<a href="#limitations-104" class="anchor"></a>Limitations</h4>
<p>The results displayed by the authors suggest the hACE2-AAV mouse model infected with SARS-CoV-2 generates symptoms that partly resembled human infection symptoms; however, neither express real tissue tropism of SARS-CoV-2 in infected mice or different susceptibility with the infection based on gender or age.  Additionally, because no dead mice were reported, the model does not fully reflect the pathogenesis of SARS-CoV-2 as in other transgenic models. Future studies should figure out the persisting time of virus in infected mice based on strain, gender, and age.</p>
</div>
<div id="significance-102" class="section level4">
<h4 class="hasAnchor">
<a href="#significance-102" class="anchor"></a>Significance</h4>
</div>
<div id="the-hace2-aav-mouse-model-partially-simulated-the-pathology-of-covid-19-with-a-focused-and-robust-lung-sars-cov-2-infection--the-hace2-aav-mouse-model-would-allow-test-the-patient-sars-cov-2-derived-virus-in-mice-of-diverse-genetic-backgrounds-as-a-platform-to-study-therapeutic-strategies-to-combat-covid-19-disease-" class="section level4">
<h4 class="hasAnchor">
<a href="#the-hace2-aav-mouse-model-partially-simulated-the-pathology-of-covid-19-with-a-focused-and-robust-lung-sars-cov-2-infection--the-hace2-aav-mouse-model-would-allow-test-the-patient-sars-cov-2-derived-virus-in-mice-of-diverse-genetic-backgrounds-as-a-platform-to-study-therapeutic-strategies-to-combat-covid-19-disease-" class="anchor"></a>The hACE2-AAV mouse model partially simulated the pathology of COVID-19 with a focused and robust lung SARS-CoV-2 infection. The hACE2-AAV mouse model would allow test the patient-SARS-CoV-2 derived-virus in mice of diverse genetic backgrounds as a platform to study therapeutic strategies to combat COVID-19 disease.</h4>
</div>
<div id="section-5" class="section level4">
<h4></h4>
</div>
<div id="credit-100" class="section level4">
<h4 class="hasAnchor">
<a href="#credit-100" class="anchor"></a>Credit</h4>
<p><em>Reviewed by Martinez-Delgado, G as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Targeted Immunosuppression Distinguishes COVID-19 from Influenza in Moderate and Severe Disease</strong></p>
<p>Mudd et al.; medRxiv <span class="math display">\[10.1101/2020.05.28.20115667\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>influenza</p></li>
<li><p>single-cell RNA sequencing</p></li>
<li><p>monocytes</p></li>
<li><p>cytokines</p></li>
</ul>
<p><em>Main findings</em></p>
<p>Despite similarities in presentation at onset, differences in the underlying immunopathology of SARS-CoV-2 infections and other respiratory infections, like influenza, remain largely unknown. In this pre-print, Mudd et al. performed single-cell RNA sequencing (scRNAseq) of PBMCs from COVID-19 patients and influenza patients, in order to delineate potential key differences between the aforementioned respiratory infections. Analyses were performed using a cohort of 79 COVID-19 patients (n=79; 35 of whom developed acute respiratory failure), 26 influenza patients (n=26; 7 of whom developed acute respiratory failure), and 15 healthy controls (n=15</p>
<p>First, plasma cytokine levels were evaluated in the 79 COVID-19 patients, 26 influenza patients, and 8 of the healthy controls. Cytokine analyses identified a reduced production of GM-CSF, IFN-ɣ. and IL-9 but a significant elevation of IL-6 and IL-8 across all COVID-19 patients, compared to influenza patients. In fact, certain chemokines and others were more up-regulated in influenza patients, as opposed to COVID-19 patients. The authors subsequently performed a computational assessment of whether certain groups (or modules) of cytokines were predictive of one of two infections.</p>
<p>Interestingly, two modules, containing G-CSF, IFN-ɣ, IL-2R, IL-6, IL-8, and MCP-1 (among several others), were inversely correlated with an increased likelihood of being SARS-CoV-2-positive. The authors observe that while higher generalized inflammation is characteristic of influenza patients, COVID-19 patients exhibit a marked elevation of a distinct subset of cytokines. Using intubation status and expiration as end-markers of disease severity, the authors found that IL1-RA and IL-6 were associated with COVID-19 disease severity and predictive of poor outcome, both with and without comparison to influenza patients. Collectively, these results suggest that only a selection of inflammatory cytokines are predictive of disease severity, while cytokine storm syndrome is not necessarily descriptive of all COVID-19 patients; these characterizations distinguish COVID-19 immunopathology from that of influenza.</p>
<p>PBMCs had been collected from 79 COVID-19 patients (n=79; 35 of whom developed acute respiratory failure), 26 influenza patients (n=26; 7 of whom developed acute respiratory failure), and 15 healthy controls (n=15). A comparison of the peripheral immune landscape identified several primary differences. Though both groups of patients exhibited pan-lymphopenia, generally, COVID-19 patients had more antibody-secreting plasmablasts and activated CD4<sup>+</sup> T cells than influenza patients or controls. However, COVID-19 patients showed significantly reduced numbers of circulating monocytes, in line with previous reports that inflammatory monocytes are recruited to the lung and reduced in the periphery in COVID-19 patients. Notably, both these peripheral monocytes and CD4<sup>+</sup> T cells in COVID-19 patients showed reduced HLA-DR expression, indicative of reduced activation.</p>
<p>A closer interrogation of potential immuno-regulatory cell types (as a compensatory response to the hyper-inflammation observed in COVID-19 patients) via scRNAseq (of 3 COVID-19 patients <span class="math display">\[n=3\]</span>, 3 influenza patients <span class="math display">\[n=3\]</span>, and 1 healthy control) revealed a significantly suppressed type I interferon (IFN) response among B cells, CD8<sup>+</sup> T cells, regulatory T cells, plasmacytoid dendritic cells (pDCs), and especially among monocytes. In contrast, his pathway and its associated downstream cascades were enriched in influenza patients. Notably, pathways enriched in COVID-19 patients were glucocorticoid and metabolic stress pathways across multiple cell types, but most significantly in monocytes.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>The limited patient sample size of the scRNAseq analysis should be noted.</p>
<p><em>Biological</em></p>
<p>Without additional clinical information, it is difficult to know whether relative time-points (at which blood samples were collected and cytokine analyses were performed) may have been different, so an analysis of patients at different stages of their disease course may be a confounding factor. Indeed, the authors make some reference to this potential limitation, in addition to age, in their linear regression models.</p>
<p>In addition, the authors use HLA-DR expression to evaluate myeloid cell activation; other markers should be used to validate the observation of reduced HLA-DR expression. This reduced activation phenotype, in combination with the fewer number of monocytes in the periphery and down-regulated IFN response, provides the basis for the authors’ conclusion that an overall suppressed monocyte response underscores COVID-19 immunopathology, when compared to the immune profile of influenza patients. However, it is important to consider the recruitment of the inflammatory subset of monocytes to the lung or other extrapulmonary organs as a reason for the reduced number in the vasculature.</p>
<p><em>Significance</em></p>
<p>Through a much needed comparison, Mudd et al. provide a closer look at the cellular differences between the immune response to COVID-19 and influenza. Using scRNAseq, the authors identify notable changes in monocyte transcriptional activity and number and in cytokine profiles that suggest potential associations with disease severity of COVID-19, but not influenza. In particular, the identification of a glucocorticoid response in monocytes is worth further investigation, given previous claims towards the use of immunosuppressive agents to treat COVID-19.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>Sex differences in immune responses to SARS-CoV-2 that underlie disease outcomes</strong></p>
<p>Takahashi et al.; medRxiv <span class="math display">\[10.1101/2020.06.06.20123414\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>Sex</p></li>
<li><p>Immune response</p></li>
<li><p>Monocytes</p></li>
<li><p>Cytokines</p></li>
<li><p>T cells</p></li>
</ul>
<p><em>Main findings</em></p>
<p>Sex-based differences in the immune response have been reported for various types of infections. There is a growing body of epidemiological evidence that supports the finding that men experience more severe COVID-19 disease than women do, but the immune mechanisms underscoring such a difference remain unknown.</p>
<p>Here, Takahashi et al. analyze PBMCs, plasma, and nasopharyngeal swabs or saliva from 93 mild-to-moderate COVID-19 patients (n=93), comprised of 48 women (n=48) and 45 men (n=45), to characterize potential sex-based differences in the immune response to SARS-CoV-2 infection. It is important to note that patients on hydroxychloroquine and Remdesivir were not excluded from a sub-cohort of patients (n=39) evaluated as baseline measures for untampered immune responses to SARS-CoV-2 (these patients were not treated prior to first sample collection). In a second sub-cohort, 54 patients were assessed longitudinally for an undisclosed amount of time. Samples from uninfected healthcare workers were used as controls.</p>
<p><em>Viral Load (nasopharyngeal or saliva samples)</em></p>
<p>No significant differences were identified between male and female patients. Still, median viral RNA was higher in male patients at first sample collection and generally throughout disease course.</p>
<p><em>Antibody production (plasma samples)</em></p>
<p>Anti-SARS-CoV-2 S1 protein-specific IgG and IgM antibodies were measured in the plasma of male and female patients. Though anti-S1-IgG antibodies were higher in female patients, compared to male patients, no significant differences could be identified either in the baseline cohort or in longitudinal patients.</p>
<p><em>Cytokine analysis</em> <em>(plasma samples)</em></p>
<p>Among baseline patients, who had not received immunomodulatory therapy prior to sample collection (except hydroxychloroquine), type I/II/III IFN levels were not significantly different between male and female patients. However, IL-8 was significantly higher in male than in female patients. Of note, among longitudinally evaluated patients, CCL5 levels were significantly higher in male than in female patients. CXCL10 levels show a similar trend, though this was not significant.</p>
<p><em>Immune cell landscape (PBMCs)</em></p>
<p>Both male and female patients exhibited a reduction among T cells and an increase in B cells. No significant differences in T cell subtypes (naïve, central/effector memory, follicular, regulatory) were observed between male and female patients. Of note, however, female patients showed (1) a significantly greater proportion of CD38<sup>+</sup>HLA-DR<sup>+</sup> activated CD8<sup>+</sup> T cells and (2) a concomitant enrichment of PD-1<sup>+</sup>TIM-3<sup>+</sup> terminally differentiated T cells, compared to male patients. Otherwise, no other significant differences were identified between male and female patients.</p>
<p>The authors subsequently interrogated the peripheral myeloid compartment. Female patients showed a greater increase in CD14<sup>+</sup>CD16<sup>+</sup> intermediate monocytes than male patients, while both patients exhibited a marked increase in total monocytes, compared to the controls. However, male patients showed higher levels of CD14<sup>lo</sup>CD16<sup>+</sup> non-classical monocytes than female patients and their uninfected, healthy counterparts. The authors noted that this enrichment of non-classical monocytes was correlated with CCL5 levels only in male patients.</p>
<p><em>Clinical comparison</em></p>
<p>Clinical outcomes were tracked for both male and female patients. Clinical scoring was used to separate each group into two sub-groups: patients that had remained stable throughout hospital stay (stabilized) and patients that had worsened since the first sample collection (deteriorated). Deteriorated male patients were significantly older than stabilized male patients; there was no significant difference in age between stabilized and deteriorated female patients. In terms of BMI, both deteriorated male and female patients tended to be higher in BMI than their respective stabilized counterparts. Interestingly, anti-S1-IgG antibodies were higher in stabilized female patients than their deteriorated counterparts, though this trend was not seen with male patients. Otherwise, no other significant differences in clinical parameters were observed.</p>
<p>Additional comparisons between deteriorated and stabilized patients of each sex revealed that certain innate cytokine mediators (TNFSF10 and IL-15) associated with worse outcome in female patients but not in male patients. In contrast, the proportion of CD38<sup>+</sup>HLA-DR<sup>+</sup> activated CD8<sup>+</sup> T cells was significantly reduced in deteriorated male patients compared to their stable counterparts, but this was not true for female patients. Indeed, poor CD8+ T cell activation and IFNɣ production were both negatively correlated with age in male patients, but not in female patients.</p>
<p><em>Limitations</em></p>
<ul>
<li><p>A significant number of patients were diagnosed with underlying chronic conditions that have been previously described to associate with poorer COVID-19 outcomes or with a compromised immune system.</p></li>
<li><p>Approximately two-thirds of each group (men and women) were treated with tocilizumab, and nearly a sixth of each group were treated with corticosteroids. While these patients were excluded from the baseline cohort, it is unclear whether or not these patients contributed to the second cohort that was longitudinally examined.</p></li>
<li><p>The mean age for patients is notably higher than the mean age for the HCW control group.</p></li>
<li><p>Duration of hospital stay was not considered, so it is unclear how quickly certain subsets of male and female patients deteriorated. This may be a confounding variable, or at the very least, the kinetics of disease course in male and female patients is a parameter that warrants investigation.</p></li>
</ul>
<p><em>Significance</em></p>
<p>In summary, Takahashi et al. provide the first report-to-date that delineates immunological differences between male and female patients with mild-to-moderate COVID-19 disease during the initial stages of infection. For example, male patients deteriorate due to less robust T cell-mediated antiviral immunity, compared to their female counterparts. Several of the other findings substantiate previous reports, such as those of significant neutrophil chemotaxis in the lung of COVID-19 patients (and its association with poorer prognosis). This study, therefore, provides an important platform for additional inquiries into key signaling pathways and transcriptional programs that are differentially regulated between male and female COVID-19 patients by specific cell types (i.e. intermediate and non-classical monocytes, CD38<sup>+</sup>HLA-DR<sup>+</sup> CD8<sup>+</sup> T cells) identified in this report. These studies, alongside others, are warranted to better tailor therapies for male and female COVID-19 patients.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai. </em></p>
<p><strong><em>Title:</em></strong> Rapid isolation and profiling of a diverse panel of human monoclonal antibodies targeting the SARS-CoV-2 spike protein</p>
<p><strong><em>Keywords:</em></strong> Coronavirus; SARS-CoV-2; SARS-CoV; COVID-19; Monoclonal antibodies; Adaptive Immunity</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Zost et al. describe the methodology used to efficiently generate a large library of highly-functional monoclonal antibodies directed against the SARS-CoV-2 spike (S) protein. Several different approaches were used to select the antibodies characterized in this study. Briefly, plasma or serum was obtained from four patients infected with SARS-CoV-2, and ELISA binding assays were used to confirm the presence of reactive antibodies to the prefusion ectodomain of either the SARS-CoV-2 or SARS-CoV S protein. Additional screens were used to assess the presence of antibodies capable of binding to the receptor binding domain (RBD) as well as the entire N-terminal domain (NTD) of the SARS-CoV-2 spike protein. The highest reactivity was seen in binding assays when the antigenic targets were the SARS-CoV-2 spike S2P ectodomain or RBD. SARS-CoV-2 S-specific class-switched memory B cells were isolated from peripheral blood mononuclear cells (PBMCs) via flow cytometry. The two patients whose blood was collected at later stages of convalescence displayed higher frequencies of antigen-specific memory B cells and greater levels of neutralizing antibodies. S2P ectodomain- and RBD-specific memory B cells sorted from these two patients PBMCs were pooled and cultured for one week in wells containing a feeder layer of cells expressing CD40L, IL-21, and BAFF. Approximately 50% of these cells were single-cell sequenced for antibody gene synthesis. The other half were placed in a Berkeley Lights Beacon Optofluidic instrument to further identify, select, and export antigen-reactive B cells prior to single cell antibody sequencing and cloning into immunoglobulin expression vectors. Both approaches yielded a combined total of 386 recombinant SARS-CoV-2 reactive human monoclonal antibodies. Subsequent ELISA and neutralization assays were used to separate these antibodies into five classes based on their cross-reactivity with SARS-CoV and the specific binding domains on the SARS-CoV-2 S protein. Bioinformatic analysis of the immunoglobulin sequences revealed a high degree of relatedness to the inferred unmutated ancestor immunoglobulin genes.</p>
<p><strong><em>Critical Analysis:</em></strong></p>
<p>This study characterizes a robust repertoire of SARS-CoV-2 spike-specific antibodies. The authors begin to shed light on the binding sites of these antibodies by describing the domains on the spike protein to which these antibodies react. However, in order to more fully capture the mechanism of neutralization for the leading therapeutic candidates, it will be important to further characterize the specific epitopes and structural binding modes. This is especially important since many of the antibodies identified in this study will not directly interfere with the RBD/ACE2 interaction and therefore likely act through another mechanism such as destabilizing the spike prefusion conformation. Another interesting observation raised in this study is that, as seen with Ebola, patients do not possess a high frequency of memory B cells expressing neutralizing antibodies until later in convalescence. However, given the small number of patients in the study, a larger sample is needed to confirm this conclusion. While this study presents a comprehensive class of candidate antibodies for therapeutic development there is still much needed data describing the protective potential of these antibodies in animal models challenged with SARS-CoV-2, as the authors assert as well. Finally, as synergy has been observed in strong B cell response for other coronaviruses and the fact that antibody cocktails are an effective treatment platform to prevent mutation escape, it would be helpful to know whether specific combinations of these monoclonal antibodies enhance neutralization and in vivo protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>In conclusion, this study presents a robust analysis of the specific B cell response to SARS-CoV-2 in a small number of individuals, and describes practical techniques to isolating a large and diverse panel of human monoclonal antibodies. In addition to revealing potential therapeutic antibody candidates for COVID-19, the authors provide additional information as to the complicated and inconsistent observations of antibody cross-reactivity and cross-neutralization in the context of SARS-CoV and SARS-CoV-2. Information on conserved and highly potent neutralizing targets of antibody responses will be critical down the road as we evaluate the immunogenicity of vaccine candidates. Meanwhile, the information in this study can be directly applied to the therapeutic antibody pipeline for SARS-CoV-2 and the methodologies described here can be adapted for similar emerging pathogens in the future.</p>
<p>10.1101/2020.03.09.983247</p>
<p><strong>Title:</strong> Inhibition of SARS-CoV-2 infection (previously 2019-nCoV) by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</p>
<p><strong>Keywords:</strong> SARS-CoV2, S protein, fusion inhibitor, hACE2</p>
<p><strong>Main findings:</strong></p>
<p>Members of the coronavirus family rely on fusion with the host’s cell membrane during infection. The spike (S) protein plays an essential role in this step (1). This work sheds light on the particularities of the S protein from SARS-CoV2 and compares it to previously existing coronaviral S proteins. SARS-CoV2 S protein showed stronger affinity for hACE2 compared to SARS-CoV. Previous work by this group had established the potential of a previous version of the peptide (EK1) in reducing the infectivity of several coronavirus by blocking the S protein interactions with the host cell (2). In this work, authors present an improved EK1C4 version of this peptide. In vitro experiments on human cells showed EK1C4-mediated reduction of the infectivity of SARS-CoV-2 and other coronavirus. Administration of EK1C4 in prophylaxis or shortly after exposure to the HCoV-OC43 coronavirus protected newborn mice from virus-induced lethality.</p>
<p><strong>Limitations:</strong></p>
<p>This work suggests that the furin cleavage site from CoV2-S could be related to its increased infectivity compared to CoV-S. However, a previous report suggested that the enhanced cell-to-cell fusion mediated by this furin cleavage might not necessarily translate into enhanced viral infectivity (3). The mutation of this site, or inhibition of furin in 293T cells expressing SARS-CoV or CoV-2 S proteins, could inform if this mechanism is actually involved in SARS-CoV2 pathogenesis, and potentially suggest additional routes for therapeutic intervention.</p>
<p>The in vivo model used to address the protective potential of EK1C4 is based on the coronavirus HCoV-OC43. Further work should aim to establish the potential of EK1C4 for prevention of SARS-CoV-2 replication in more relevant animal models (e.g. Rhesus Macaques or hACE2 mice). It is, however, acknowledged, that experimentation in vitro included both SARS-CoV2 and the same HCoV-OC43 used in the in vivo infection model.</p>
<p>Even though the safety of the unmodified EK1 peptide in mice had been previously examined (2), the potential side effects and pharmacokinetics/dynamics of EK1C4 in the in vivo setting are not addressed in this report.</p>
<p><strong>Relevance:</strong></p>
<p>The development of prophylactic drugs against viral infection is a paramount element of epidemic control, specially in the time before a vaccine can be readily available. Similar strategies have been followed in the past in response to threats of this nature (4,5). This work’s relevance is limited by the absence of relevant animal models of SARS-CoV-2 infection. Further pharmacological analyses would also be necessary to ensure the safety of EK1C4 in animal models before a potential safety analysis in human subjects.</p>
<p>1. Heald-Sargent T, Gallagher T. Ready, set, fuse! the coronavirus spike protein and acquisition of fusion competence. Vol. 4, Viruses. Multidisciplinary Digital Publishing Institute (MDPI); 2012. p. 557–80.</p>
<p>2. Xia S, Yan L, Xu W, Agrawal AS, Algaissi A, Tseng CTK, et al. A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike. Sci Adv. 2019 Apr 1;5(4):eaav4580.</p>
<p>3. Follis KE, York J, Nunberg JH. Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology. 2006 Jul 5;350(2):358–69.</p>
<p>4. Jones JC, Turpin EA, Bultmann H, Brandt CR, Schultz-Cherry S. Inhibition of Influenza Virus Infection by a Novel Antiviral Peptide That Targets Viral Attachment to Cells. J Virol. 2006 Dec 15;80(24):11960–7.</p>
<p>5. Ahmed A, Siman-Tov G, Hall G, Bhalla N, Narayanan A. Human antimicrobial peptides as therapeutics for viral infections. Vol. 11, Viruses. MDPI AG; 2019.</p>
<p>Reduction of lymphocyte at early stage elevates severity and death risk of COVID-19 patients: a hospital-based case-cohort study</p>
<p><strong>Keywords:</strong> COVID-19, lymphocyte, severity, organ injuries.</p>
<p><strong>Main findings:</strong> The aim of this study was to assess an association between reduced blood lymphocyte counts at hospital admission and prognosis of COVID-19 patients (n=192). The authors found:</p>
<ul>
<li><p>Patients with lymphopenia are more likely to progress to severe disease or succumb to COVID-19 (32.1% of COVID-19 patients with lymphocyte reduction died).</p></li>
<li><p>Reduction of lymphocytes mainly affects the elderly (&gt; 70 years old).</p></li>
<li><p>Lymphocyte reduction is more prevalent in COVID-19 patients with cardiac disease and pulmonary disease, patients with increase in the chest CT score (key marker of lung injury) and a decrease in several respiratory function markers (PaCO2, SpO2, oxygenation index).</p></li>
</ul>
<p><strong>Limitations of the study:</strong> Reduced blood lymphocyte counts with aging have been known (<a href="https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2"><em>https://www.medrxiv.org/content/10.1101/2020.03.08.20031229v2</em></a>) <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413"><em>https://onlinelibrary.wiley.com/doi/epdf/10.1111/sji.12413</em></a>). Therefore, it is not unexpected that a larger fraction of COV ID-19 patients above 70 years old have lower lymphocytes counts. Since age has been reported to be a major factor that determines outcome for COVID-19, lymphocyte counts and prognosis should have been adjusted by age. Multivariate analysis to identify independent risk factors is lacking.</p>
<p><strong>Relevance:</strong> Previous studies demonstrated that SARS-CoV-2 infection leads to a decrease of the T cell count. This study confirms these results and shows that lymphocyte reduction mainly affects the elderly. Lymphopenia was associated with disease severity as well as worse prognosis. Future studies need to address if lymphopenia is a negative predictive factor independent from age.</p>
<p>Review by Meriem Belabed as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai.</p>
<p><strong>COVID-19 Critical Illness Pathophysiology Driven by Diffuse Pulmonary Thrombi and Pulmonary Endothelial Dysfunction Responsive to Thrombolysis</strong></p>
<p>Poor et al., medRxiv <span class="math display">\[10.1101/2020.04.17.20057125\]</span></p>
<p>Keywords:</p>
<ul>
<li><p>COVID-19</p></li>
<li><p>pneumonia</p></li>
<li><p>thrombolysis</p></li>
<li><p>endothelium</p></li>
<li><p>platelet</p></li>
<li><p>D-dimer</p></li>
</ul>
<p><em>Main findings</em></p>
<p>This report describes the use of systemic tissue plasminogen activator (tPA) to treat venous thromboembolism (VTE) seen in four critically ill COVID-19 patients with respiratory failure. These patients all exhibited gas exchange abnormalities, including shunt and dead-space ventilation, despite well-preserved lung mechanics. A pulmonary vascular etiology was suspected.</p>
<p>All four patients had elevated D-dimers and significant dead-space ventilation. All patients were also obese, and 3/4 patients were diabetic.</p>
<p>Not all patients exhibited an improvement in gas exchange or hemodynamics during the infusion, but some did demonstrate improvements in oxygenation after treatment. Two patients no longer required vasopressors or could be weaned off them, while one patient became hypoxemic and hypotensive and subsequently expired due to a cardiac arrest. Echocardiogram showed large biventricular thrombi.</p>
<p><em>Limitations</em></p>
<p>In addition to the small sample size, all patients presented with chronic conditions that are conducive to an inflammatory state. It is unclear how this would have impacted the tPA therapy, but it is likely not representative of all patients who present with COVID-19-induced pneumonia. Moreover, each patient had received a different course of therapy prior to receiving the tPA infusion. One patient received hydroxychloroquine and ceftriaxone prior to tPA infusion, two patients required external ventilator support, and another patient received concurrent convalescent plasma therapy as part of a clinical trial. Each patient received an infusion of tPA at 2 mg/hour but for variable durations of time. One patient received an initial 50 mg infusion of tPA over two hours. 3/4 patients were also given norepinephrine to manage persistent, hypotensive shock. Of note, each patient was at a different stage of the disease; One patient showed cardiac abnormalities and no clots in transit on an echocardiogram, prior to tPA infusion.</p>
<p><em>Significance</em></p>
<p>The study describes emphasizes the importance of coagulopathies in COVID-19 and describes clinical outcomes for four severe, COVID-19 patients, who received tPA infusions to manage poor gas exchange. While the sample size is very limited and mixed benefits were observed, thrombolysis seems to warrant further investigation as a therapeutic for COVID-19-associated pneumonia that is characterized by D-dimer elevation and dead-space ventilation. All four patients had normal platelet levels, which may suggest that extrinsic triggers of the coagulation cascade are involved.</p>
<p>The authors suspect that endothelial dysfunction and injury contribute to the formation of pulmonary microthrombi, and these impair gas exchange. Pulmonary thrombus formation has also been reported by other groups; post-mortem analyses of 38 COVID-19 patients’ lungs showed diffuse alveolar disease and platelet-fibrin thrombi (Carsana et al., 2020). Inflammatory infiltrates were macrophages in the alveolar lumen and lymphocytes in the interstitial space (Carsana et al., 2020). Endothelial damage in COVID-19 patients has also been directly described, noting the presence of viral elements in the endothelium and inflammatory infiltrates within the intima (Varga et al., 2020). One hypothesis may be that the combination of circulating inflammatory monocytes (previously described to be enriched among PBMCs derived from COVID-19 patients) that express tissue factor, damaged endothelium, and complement elements that are also chemotactic for inflammatory cells may contribute to the overall pro-coagulative state described in COVID-19 patients.</p>
<p><em>References</em></p>
<p>Carsana, L., Sonzogni, A., Nasr, A., Rossi, R.S., Pellegrinelli, A., Zerbi, P., Rech, R., Colombo, R., Antinori, S., Corbellino, M., et al. (2020) Pulmonary post-mortem findings in a large series of COVID-19 cases from Northern Itality. medRxiv. 2020.04.19.20054262.</p>
<p>Varga, Z., Flammer, A.J., Steiger, P., Haberecker, M., Andermatt, R., Zinkernagal, A.S., Mehra, M.R., Schuepbach, R.A., Ruschitzka, F., Moch, H. (2020) Endothelial cell infection and endotheliitis in COVID-19. Lancet. 10.1016/S0140-6736(20)30937-5.</p>
<p><em>The study described in this review was conducted by physicians of the Divisions of Pulmonary, Critical Care, and Sleep Medicine, Cardiology, Nephrology, Surgery, and Neurosurgery and Neurology at the Icahn School of Medicine at Mount Sinai. </em></p>
<p><em>Reviewed by Matthew D. Park as part of a project by students, postdocs, and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
<div id="title-3" class="section level3">
<h3 class="hasAnchor">
<a href="#title-3" class="anchor"></a>Title</h3>
<p><span id="keywords" class="anchor"></span><strong>Serum protein profiling reveals a landscape of inflammation and immune signaling in early-stage COVID-19 infection</strong></p>
<p><em>Keywords</em></p>
<ul>
<li><p><strong>Serum Profiling</strong></p></li>
<li><p><strong>Cytokines</strong></p></li>
<li><p>Protein array</p></li>
<li><p><strong>CCL2</strong></p></li>
<li><p><strong>CXCL10</strong></p></li>
</ul>
<p><span id="main-findings" class="anchor"></span><em>Main Findings</em></p>
<p>In this preprint Hou <em>et al.</em>, analyze serological immune mediators and other proteins from individuals with early COVID-19 symptoms using an antibody microarray that detects 532 target proteins. Patients were classified as COVID-19 (n=13) or influenza (n=15) based on positive RT PCR test for SARS-CoV-2 RNA (COVID-19) or FluA, FluB, RSV RNA (all classified as influenza group).</p>
<p>88 up-regulated and 37 down-regulated proteins were identified by comparing COVID-19 and influenza patient groups (p-value &lt; 0.05). Some of those up-regulated ytyl34proteins were reported before, such as IFN-𝜸, IL-6, CXCL8, CCL2, CXCL10 and some that were not previously associated with COVID-19, such as IL-20, CCL27 and IL-21. Complement proteins C1R and C7, as well as PLG were found to be reduced in COVID-19 patients.</p>
<p>After performing a correlation analysis of the differentially upregulated proteins and clinical data, the authors found a positive correlation between expression of several proteins in the CCL2 and CXCL10 signaling pathways and clinical parameters typically related to liver and renal function, myocardial injury, inflammation and infection, as well as neutrophils counts. Conversely, most of these same proteins show a negative correlation with lymphocyte counts.<span id="limitations" class="anchor"></span></p>
<p><em>Limitations</em></p>
<p>The control “influenza group” had patients negative for influenza virus but positive for RSV, and thus the nomenclature should be revised. As noted by the authors, a larger cohort should be used to validate findings. There is no multivariate analysis, and identification of independent or confounding variables. Even though several proteins were differentially expressed between the patient groups, there was significant overlap between the groups, which may preclude the use of any single protein as a biomarker. Data on clinical variables should have been included in main figure. It is unclear why the authors annotated serum proteins in cellular components. Proteins in CCL2 and CXCL10 interaction network are largely overlapping, but the authors do not emphasize it.</p>
<p>Authors didn’t show lymphocyte and neutrophil counts. Since it is known that severe COVID-19 patients can present lymphopenia and neutrophilia, it would be important to have this information in their cohort as they are correlating protein levels with neutrophils and lymphocytes.</p>
<p>Authors claim that CCL2 can act as an autocrine factor that promotes viral replication in infected macrophages, and cite one paper with HIV, but authors should discuss that this chemokine recruits monocytes to the site of infection and then could contribute to the increased inflammatory response related to COVID-19.</p>
<p><em>Significance</em></p>
<p>This study shows the potential use protein array to simultaneously identify many different proteins in serum of COVID-19 patients.</p>
<p><span id="credit" class="anchor"></span>The authors identify several differentially expressed proteins (potential biomarkers) and correlate them with clinical indices that give insights on possible therapy targets in COVID-19.</p>
<p><em>Credit</em></p>
<p><em>Reviewed </em>by Alessandra Soares-Schanoski* as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*</p>
<p><strong>Title: </strong></p>
<p>Higher Air Temperature, Pressure, and Ultraviolet Are Associated with Less Covid-19 Incidence</p>
<p><strong>Doi:</strong> 10.1101/2020.05.09.20096321</p>
<p><strong>Keywords: </strong></p>
<p>Covid-19 incidence, air temperature, air pressure, ultraviolet index</p>
<p><strong>The main finding of the article: </strong></p>
<p>Recent studies have diverged as to weather conditions are allied or not with the spreading of Covid-19. Through random-effects meta-regression analysis, this work aimed was to determine if elements linked to meteorology can influence SARS-CoV-2 incidence and the speed of its propagation. The number of Covid-19 patients and meteorological conditions at each Japanese prefectural capital city from January to April 2020 were collected.</p>
<p>The results demonstrated a negative association between Covid-19 incidence and monthly mean air temperature (C°) (coefficient -0.351), sea level air pressure (hPa) (coefficient -0.001) and the monthly mean daily maximum UV index (UV) (coefficient -0.001).</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The manuscript would benefit from a more thorough introduction and discussion of the results in the context of previous studies. The authors could explore more the results of the supplementary table 1 (wind speed, relative humidity and sunshine). The figure caption should be better detailed, explaining the characteristics of each graph.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>The transmission dynamics of SARS-CoV-2 depends on different factors, such as population density, demographic and clinical characteristics of the population, hygiene, local ventilation, etc., and the seasonality of SARS-CoV-2 is not yet known.</p>
<p>The data of this manuscript suggest that higher air temperature, air pressure, and ultraviolet are associated with a lower incidence of Covid-19. Certainly, this study is a step in identifying which environmental factors can favor viral transmission.</p>
<p><strong>Title: </strong></p>
<p>Low doses of radiation therapy increase the immunosuppressive profile of lung macrophages via IL-10 production and IFNy/IL-6 suppression: a therapeutic strategy to counteract lung inflammation?</p>
<p><strong>Doi:</strong> 10.1101/2020.05.11.077651</p>
<p><strong>Keywords: </strong></p>
<p>Acute respiratory distress syndrome, lung inflammation, low dose irradiation, chest radiation therapy, nerve- and airway-associated macrophages, lung macrophages.</p>
<p><strong>The main finding of the article: </strong></p>
<p>The pathophysiology of severe pneumonia and acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-induced is related with an overproduction of early response proinflammatory cytokines, leading to an increased vascular permeability and high risk of death. Chest radiation therapy using low doses (&lt;1 Gy) was beneficial in the past to treat pneumonia, exerting anti-inflammatory effects in the lung, however, the involved mechanism remained uncertain. The authors proposed that macrophages could be involved in the counteracting lung inflammation after irradiation, and investigated this through in vitro studies with human and mouse lung macrophages, and in vivo pneumonia models in mice.</p>
<p>Normal human lung cells were collected from 3 donors (n=2 to lung cancer, n=1 to benign disease). From these cells, macrophages were isolated and stimulated for 6 hours with Poly(I:C) (1µg/mL) and were irradiated using 0.5 or 1 Gy doses, or non-irradiated. In relation to the murine model, C57BL/6 mice received intratracheally two doses (100µg and 50µg – two consecutive days) of lipopolysaccharide (LPS) or toll-like receptor 3 ligand Poly(I:C). Six hours after the second intranasal dose, 0.5 or 1 Gy of irradiation was applied to the animal thorax. After this procedure, spectral computed tomography (CT) of the mice chest at different time points was performed. Flow cytometry was performed for human lung macrophages and for mouse lung cells. Cytokines concentrations in culture supernatants from <em>in vitro</em> experiments were profiled.</p>
<p>The quantification of supernatants demonstrated that both doses of irradiation, 0.5 and 1Gy, significantly decreased IFNγ and increased IL-10 secretion in human lung macrophages stimulated with Poly(I:C) in comparison to non-irradiated Poly (I:C)-stimulated cells. The flow cytometry analysis showed higher percentage of human lung macrophages producing IL-10 after 0.5 Gy irradiation dose when compared with the other conditions, and decrease IL-6 production after both doses of irradiation in comparison to non-irradiation cells. In the in vivo pneumonia mouse model, the authors showed that low dose thorax irradiation of mice treated with LPS or Poly(I:C), resulted in increased IL-10 production by a distinct group of macrophages dubbed nerve- and airway-associated macrophages (NAMs) compared to non-irradiated mice. In addition, the lung CT scans revealed that lungs irradiated at 1Gy presented less tissue density, indicating lower pulmonary inflammatory process in the lung.</p>
<p><strong>Critical analysis of the study: </strong></p>
<p>The data presented in this manuscript is quite limited, its mostly restricted to production of IL-10, IFN-g and IL-6 (one experiment) by lung macrophages without any other functional analysis. The only data on lung inflammation are the CT scans, adding tissue histology would have improved the analysis. The description of experimental details is incomplete, and the number of human tissues analyzed in the <em>in vitro</em> culture is very small. Given its potential importance in the mechanisms of SARS-CoV-2 lung pathology, the role of alveolar macrophages and interstitial macrophages could have been explored in more depth. The authors could had analyzed others pro- and anti-inflammatory markers, like TNFα, IL-1β, IL-1Ra, IL-8, as well as mediators involved in lung inflammation resolution such as VEGF-A. The manuscript has a comprehensive discussion on the effects of chest irradiation on lung inflammation.</p>
<p><strong>The importance and implications for the current epidemics: </strong></p>
<p>Research indicates that “cytokine storm”, an uncontrolled over-production of inflammatory markers which, in turn, sustain a systemic inflammatory response, is mostly responsible for the occurrence of severe acute respiratory syndrome. This manuscript shows that low dose thorax radiation is able to direct the pro-inflammatory lung cell responses into an anti-inflammatory response. Therefore, this therapy could be useful to mitigate lung inflammatory process in Covid-19. A point to highlight is that side and adverse effects must be well evaluated before a possible irradiation treatment in an infectious condition such as COVID-19.</p>
<p><strong>Differential Redistribution of Activated Monocyte and Dendritic Cell Subsets to the Lung Associates with Severity of COVID-19</strong></p>
<p>Sánchez-Cerrillo et al.; medRxiv <span class="math display">\[10.1101/2020.05.13.20100925\]</span></p>
<p><em>Keywords</em></p>
<ul>
<li><p>COVID-19</p></li>
<li><p>disease severity</p></li>
<li><p>monocyte</p></li>
<li><p>dendritic cells</p></li>
<li><p>T cell exhaustion</p></li>
</ul>
<p><em>Main findings</em></p>
<p>While the growing scientific literature on the immune responses to SARS-CoV-2 infection has highlighted several immunological markers for COVID-19, molecular or cellular determinants of disease severity have not yet been well-described. In this report, Sánchez-Cerrillo et al. profiled myeloid and T cell subsets across mild (G1, n=19; whole blood), severe (G2, n=21; whole blood), and critical COVID-19 cases (G3, n=23; whole blood and paired bronchoscopy samples), and healthy controls (n=22). Clinical parameters, including serum IL-6, procalcitonin (PCT), C-reactive protein (CRP), D-dimer levels, and serum LDH, increased with worsening disease severity.</p>
<p>Using high-dimensional flow cytometry, the authors assessed changes in classical monocytes (C Mo; CD14<sup>+</sup>CD16<sup>-</sup>), transitional monocytes (T Mo; CD14<sup>+</sup>CD16<sup>+</sup>), and non-classical monocytes (NC Mo; CD14<sup>lo</sup>CD16<sup>+</sup>), CD14<sup>-</sup>CD16<sup>hi</sup>HLA-DR<sup>-</sup> granulocytes, CD141<sup>+</sup> dendritic cells (cDC1), CD1c<sup>+</sup> dendritic cells (cDC2), and CD123<sup>hi</sup> dendritic cells (pDC) in blood and bronchoscopy samples. While almost all myeloid subsets in COVID-19 patients were significantly reduced in the blood circulation compared to healthy controls (with the exception of T Mo), no statistically significant correlations between these myeloid subsets and disease severity were observed. Of note, the overall sparsity of C and T Mo subsets corresponded to high levels of serum IL-6; otherwise, there were no remarkable correlations between the frequencies of the aforementioned subsets and inflammatory markers. Importantly, in the bronchoscopy samples, an unpaired analysis identified an enrichment of granulocytes and inflammatory T and NC Mo. Importantly, a paired analysis of blood and lung samples demonstrated that T, NC, and CD1c<sup>+</sup> DCs are significantly enriched in the lung. Collectively, these results reflect a notable recruitment of monocytes to the lung. The authors used CD40 expression as a marker of myeloid activation. While CD40 expression decreased with increasing disease severity, this trend was not significant, and expression was comparable to the cells isolated from healthy controls. Lastly, a survey of markers associated with compromised effector function of T cells isolated from blood and bronchoscopy samples of G3 patients showed that CD38<sup>+</sup>CXCR5<sup>+</sup> T cells are significantly more prevalent in the lungs than in the blood, and differences to healthy controls were significant.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>One notable limitation are superinfections as a confounding variable; their effects need to be accounted for with careful multi-variate analysis and should be replicated in larger, multicenter studies. Moreover, flow cytometry markers used in the present study can present a biased view of cell populations, so future studies using higher-dimensional, unbiased techniques may provide a more inclusive view of myeloid heterogeneity in COVID-19 patients with differing severities of disease.</p>
<p><em>Biological</em></p>
<p>It is important to note that almost all patients across the different groups had been receiving concurrent therapy, including antivirals, antibiotics, steroids, and immuno-modulators (anti-IL-6); it is unclear when these treatments were administered, relative to the collection of samples. Furthermore, the DC subsets defined in this report comprised significantly small proportions (&lt; 5%) of total CD45<sup>+</sup> immune cells isolated from blood and bronchoscopy samples of COVID-19 patients. Lastly, while T cell exhaustion was evaluated based on expression of CD38 and CXCR5, the expression of other, more prominent co-inhibitory receptors, including PD-1 or Tim-3, was not evaluated. Therefore, this report would benefit from a better study of myeloid activation and T cell exhaustion using additional markers that define activation of the myeloid subsets, including an analysis of cytokine production, and markers for T cell exhaustion.</p>
<p><em>Significance</em></p>
<p>In summary, this report offers some insight into the profiling of different circulating cell subpopulations across various degrees of COVID-19 severity. However, interpretations of the results should be approached with caution, given the lack of statistical significance and of detailed analyses of important cell groups, including better-defined exhausted T cells. However, thus far, the findings outlined in this report support the notion that monocyte dysfunction, involving a pro-inflammatory state and an overall recruitment from the peripheral blood to disease-afflicted tissues like the lung, characterizes the immune response to COVID-19.</p>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn school of medicine, Mount Sinai. </em></p>
<p><strong>Title</strong></p>
<p>Evidence of a significant secretory-IgA-dominant SARS-CoV-2 immune response in human milk following recovery from COVID-19</p>
<p>Fox Alisa et al., preprint <em>Medvix</em> <span class="math display">\[doi:
https://doi.org/10.1101/2020.05.04.20089995\]</span></p>
<p><strong>Keywords</strong></p>
<p>Secretory IgA; breast milk; antibodies</p>
<p><span id="main-findings" class="anchor"></span></p>
<p><strong>Main Findings</strong></p>
<p>The immunity of the mucosa between the mother and the newborn against COVID-19 was tested. The secretory antibody -IgA of breastfeeding milk, shown immune response to the Receptor Binding Domain (RBD) of SARS-CoV-2 Spike protein.</p>
<p><span id="limitations" class="anchor"></span><strong>Limitations</strong></p>
<p>Further studies are needed to understand the types of vaccines and routes of administrations in terms of protective antibodies against SARS-CoV-2 in human milk.</p>
<p><strong>Significance</strong></p>
<p>Immune-modulating factors in breast milk may exert a significant impact on the infant’s developing immune system preventing or mitigating SARS-Cov-2 infection.</p>
<p><em>Reviewed</em> Martinez-Delgado Gustavo <em>as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by First Last.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.0.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
